{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Collecting numpy\n",
      "  Using cached numpy-1.24.1-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (17.3 MB)\n",
      "Collecting pandas\n",
      "  Using cached pandas-1.5.3-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (12.2 MB)\n",
      "Collecting scipy\n",
      "  Using cached scipy-1.10.0-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (34.5 MB)\n",
      "Collecting matplotlib\n",
      "  Using cached matplotlib-3.6.3-cp38-cp38-manylinux_2_12_x86_64.manylinux2010_x86_64.whl (9.4 MB)\n",
      "Collecting seaborn\n",
      "  Using cached seaborn-0.12.2-py3-none-any.whl (293 kB)\n",
      "Collecting py3Dmol\n",
      "  Using cached py3Dmol-2.0.0.post2-py2.py3-none-any.whl (6.7 kB)\n",
      "Requirement already satisfied: ipywidgets in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (7.6.5)\n",
      "Requirement already satisfied: python-dateutil>=2.8.1 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from pandas) (2.8.2)\n",
      "Requirement already satisfied: pytz>=2020.1 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from pandas) (2022.7)\n",
      "Requirement already satisfied: packaging>=20.0 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from matplotlib) (22.0)\n",
      "Collecting fonttools>=4.22.0\n",
      "  Using cached fonttools-4.38.0-py3-none-any.whl (965 kB)\n",
      "Collecting pillow>=6.2.0\n",
      "  Using cached Pillow-9.4.0-cp38-cp38-manylinux_2_28_x86_64.whl (3.4 MB)\n",
      "Collecting cycler>=0.10\n",
      "  Using cached cycler-0.11.0-py3-none-any.whl (6.4 kB)\n",
      "Collecting kiwisolver>=1.0.1\n",
      "  Using cached kiwisolver-1.4.4-cp38-cp38-manylinux_2_5_x86_64.manylinux1_x86_64.whl (1.2 MB)\n",
      "Collecting contourpy>=1.0.1\n",
      "  Using cached contourpy-1.0.7-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (300 kB)\n",
      "Collecting pyparsing>=2.2.1\n",
      "  Using cached pyparsing-3.0.9-py3-none-any.whl (98 kB)\n",
      "Requirement already satisfied: widgetsnbextension~=3.5.0 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from ipywidgets) (3.5.2)\n",
      "Requirement already satisfied: ipython-genutils~=0.2.0 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from ipywidgets) (0.2.0)\n",
      "Requirement already satisfied: nbformat>=4.2.0 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from ipywidgets) (5.7.0)\n",
      "Requirement already satisfied: ipykernel>=4.5.1 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from ipywidgets) (6.19.2)\n",
      "Requirement already satisfied: traitlets>=4.3.1 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from ipywidgets) (5.7.1)\n",
      "Requirement already satisfied: jupyterlab-widgets>=1.0.0 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from ipywidgets) (1.0.0)\n",
      "Requirement already satisfied: ipython>=4.0.0 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from ipywidgets) (8.7.0)\n",
      "Requirement already satisfied: tornado>=6.1 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from ipykernel>=4.5.1->ipywidgets) (6.2)\n",
      "Requirement already satisfied: matplotlib-inline>=0.1 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from ipykernel>=4.5.1->ipywidgets) (0.1.6)\n",
      "Requirement already satisfied: nest-asyncio in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from ipykernel>=4.5.1->ipywidgets) (1.5.6)\n",
      "Requirement already satisfied: jupyter-client>=6.1.12 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from ipykernel>=4.5.1->ipywidgets) (7.4.8)\n",
      "Requirement already satisfied: psutil in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from ipykernel>=4.5.1->ipywidgets) (5.9.0)\n",
      "Requirement already satisfied: pyzmq>=17 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from ipykernel>=4.5.1->ipywidgets) (23.2.0)\n",
      "Requirement already satisfied: debugpy>=1.0 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from ipykernel>=4.5.1->ipywidgets) (1.5.1)\n",
      "Requirement already satisfied: comm>=0.1.1 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from ipykernel>=4.5.1->ipywidgets) (0.1.2)\n",
      "Requirement already satisfied: pygments>=2.4.0 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from ipython>=4.0.0->ipywidgets) (2.11.2)\n",
      "Requirement already satisfied: backcall in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from ipython>=4.0.0->ipywidgets) (0.2.0)\n",
      "Requirement already satisfied: jedi>=0.16 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from ipython>=4.0.0->ipywidgets) (0.18.1)\n",
      "Requirement already satisfied: pexpect>4.3 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from ipython>=4.0.0->ipywidgets) (4.8.0)\n",
      "Requirement already satisfied: decorator in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from ipython>=4.0.0->ipywidgets) (5.1.1)\n",
      "Requirement already satisfied: prompt-toolkit<3.1.0,>=3.0.11 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from ipython>=4.0.0->ipywidgets) (3.0.36)\n",
      "Requirement already satisfied: pickleshare in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from ipython>=4.0.0->ipywidgets) (0.7.5)\n",
      "Requirement already satisfied: stack-data in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from ipython>=4.0.0->ipywidgets) (0.2.0)\n",
      "Requirement already satisfied: fastjsonschema in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from nbformat>=4.2.0->ipywidgets) (2.16.2)\n",
      "Requirement already satisfied: jupyter-core in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from nbformat>=4.2.0->ipywidgets) (5.1.1)\n",
      "Requirement already satisfied: jsonschema>=2.6 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from nbformat>=4.2.0->ipywidgets) (4.16.0)\n",
      "Requirement already satisfied: six>=1.5 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from python-dateutil>=2.8.1->pandas) (1.16.0)\n",
      "Requirement already satisfied: notebook>=4.4.1 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from widgetsnbextension~=3.5.0->ipywidgets) (6.5.2)\n",
      "Requirement already satisfied: parso<0.9.0,>=0.8.0 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from jedi>=0.16->ipython>=4.0.0->ipywidgets) (0.8.3)\n",
      "Requirement already satisfied: attrs>=17.4.0 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from jsonschema>=2.6->nbformat>=4.2.0->ipywidgets) (22.1.0)\n",
      "Requirement already satisfied: pyrsistent!=0.17.0,!=0.17.1,!=0.17.2,>=0.14.0 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from jsonschema>=2.6->nbformat>=4.2.0->ipywidgets) (0.18.0)\n",
      "Requirement already satisfied: importlib-resources>=1.4.0 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from jsonschema>=2.6->nbformat>=4.2.0->ipywidgets) (5.2.0)\n",
      "Requirement already satisfied: pkgutil-resolve-name>=1.3.10 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from jsonschema>=2.6->nbformat>=4.2.0->ipywidgets) (1.3.10)\n",
      "Requirement already satisfied: entrypoints in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from jupyter-client>=6.1.12->ipykernel>=4.5.1->ipywidgets) (0.4)\n",
      "Requirement already satisfied: platformdirs>=2.5 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from jupyter-core->nbformat>=4.2.0->ipywidgets) (2.5.2)\n",
      "Requirement already satisfied: Send2Trash>=1.8.0 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from notebook>=4.4.1->widgetsnbextension~=3.5.0->ipywidgets) (1.8.0)\n",
      "Requirement already satisfied: terminado>=0.8.3 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from notebook>=4.4.1->widgetsnbextension~=3.5.0->ipywidgets) (0.17.1)\n",
      "Requirement already satisfied: jinja2 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from notebook>=4.4.1->widgetsnbextension~=3.5.0->ipywidgets) (3.1.2)\n",
      "Requirement already satisfied: nbconvert>=5 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from notebook>=4.4.1->widgetsnbextension~=3.5.0->ipywidgets) (6.5.4)\n",
      "Requirement already satisfied: argon2-cffi in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from notebook>=4.4.1->widgetsnbextension~=3.5.0->ipywidgets) (21.3.0)\n",
      "Requirement already satisfied: prometheus-client in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from notebook>=4.4.1->widgetsnbextension~=3.5.0->ipywidgets) (0.14.1)\n",
      "Requirement already satisfied: nbclassic>=0.4.7 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from notebook>=4.4.1->widgetsnbextension~=3.5.0->ipywidgets) (0.4.8)\n",
      "Requirement already satisfied: ptyprocess>=0.5 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from pexpect>4.3->ipython>=4.0.0->ipywidgets) (0.7.0)\n",
      "Requirement already satisfied: wcwidth in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from prompt-toolkit<3.1.0,>=3.0.11->ipython>=4.0.0->ipywidgets) (0.2.5)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Requirement already satisfied: pure-eval in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from stack-data->ipython>=4.0.0->ipywidgets) (0.2.2)\n",
      "Requirement already satisfied: asttokens in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from stack-data->ipython>=4.0.0->ipywidgets) (2.0.5)\n",
      "Requirement already satisfied: executing in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from stack-data->ipython>=4.0.0->ipywidgets) (0.8.3)\n",
      "Requirement already satisfied: zipp>=3.1.0 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from importlib-resources>=1.4.0->jsonschema>=2.6->nbformat>=4.2.0->ipywidgets) (3.11.0)\n",
      "Requirement already satisfied: jupyter-server>=1.8 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from nbclassic>=0.4.7->notebook>=4.4.1->widgetsnbextension~=3.5.0->ipywidgets) (1.23.4)\n",
      "Requirement already satisfied: notebook-shim>=0.1.0 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from nbclassic>=0.4.7->notebook>=4.4.1->widgetsnbextension~=3.5.0->ipywidgets) (0.2.2)\n",
      "Requirement already satisfied: tinycss2 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from nbconvert>=5->notebook>=4.4.1->widgetsnbextension~=3.5.0->ipywidgets) (1.2.1)\n",
      "Requirement already satisfied: jupyterlab-pygments in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from nbconvert>=5->notebook>=4.4.1->widgetsnbextension~=3.5.0->ipywidgets) (0.1.2)\n",
      "Requirement already satisfied: lxml in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from nbconvert>=5->notebook>=4.4.1->widgetsnbextension~=3.5.0->ipywidgets) (4.9.1)\n",
      "Requirement already satisfied: mistune<2,>=0.8.1 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from nbconvert>=5->notebook>=4.4.1->widgetsnbextension~=3.5.0->ipywidgets) (0.8.4)\n",
      "Requirement already satisfied: beautifulsoup4 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from nbconvert>=5->notebook>=4.4.1->widgetsnbextension~=3.5.0->ipywidgets) (4.11.1)\n",
      "Requirement already satisfied: defusedxml in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from nbconvert>=5->notebook>=4.4.1->widgetsnbextension~=3.5.0->ipywidgets) (0.7.1)\n",
      "Requirement already satisfied: nbclient>=0.5.0 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from nbconvert>=5->notebook>=4.4.1->widgetsnbextension~=3.5.0->ipywidgets) (0.5.13)\n",
      "Requirement already satisfied: pandocfilters>=1.4.1 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from nbconvert>=5->notebook>=4.4.1->widgetsnbextension~=3.5.0->ipywidgets) (1.5.0)\n",
      "Requirement already satisfied: MarkupSafe>=2.0 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from nbconvert>=5->notebook>=4.4.1->widgetsnbextension~=3.5.0->ipywidgets) (2.1.1)\n",
      "Requirement already satisfied: bleach in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from nbconvert>=5->notebook>=4.4.1->widgetsnbextension~=3.5.0->ipywidgets) (4.1.0)\n",
      "Requirement already satisfied: argon2-cffi-bindings in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from argon2-cffi->notebook>=4.4.1->widgetsnbextension~=3.5.0->ipywidgets) (21.2.0)\n",
      "Requirement already satisfied: websocket-client in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from jupyter-server>=1.8->nbclassic>=0.4.7->notebook>=4.4.1->widgetsnbextension~=3.5.0->ipywidgets) (0.58.0)\n",
      "Requirement already satisfied: anyio<4,>=3.1.0 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from jupyter-server>=1.8->nbclassic>=0.4.7->notebook>=4.4.1->widgetsnbextension~=3.5.0->ipywidgets) (3.5.0)\n",
      "Requirement already satisfied: cffi>=1.0.1 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from argon2-cffi-bindings->argon2-cffi->notebook>=4.4.1->widgetsnbextension~=3.5.0->ipywidgets) (1.15.1)\n",
      "Requirement already satisfied: soupsieve>1.2 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from beautifulsoup4->nbconvert>=5->notebook>=4.4.1->widgetsnbextension~=3.5.0->ipywidgets) (2.3.2.post1)\n",
      "Requirement already satisfied: webencodings in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from bleach->nbconvert>=5->notebook>=4.4.1->widgetsnbextension~=3.5.0->ipywidgets) (0.5.1)\n",
      "Requirement already satisfied: idna>=2.8 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from anyio<4,>=3.1.0->jupyter-server>=1.8->nbclassic>=0.4.7->notebook>=4.4.1->widgetsnbextension~=3.5.0->ipywidgets) (3.4)\n",
      "Requirement already satisfied: sniffio>=1.1 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from anyio<4,>=3.1.0->jupyter-server>=1.8->nbclassic>=0.4.7->notebook>=4.4.1->widgetsnbextension~=3.5.0->ipywidgets) (1.2.0)\n",
      "Requirement already satisfied: pycparser in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from cffi>=1.0.1->argon2-cffi-bindings->argon2-cffi->notebook>=4.4.1->widgetsnbextension~=3.5.0->ipywidgets) (2.21)\n",
      "Installing collected packages: py3Dmol, pyparsing, pillow, numpy, kiwisolver, fonttools, cycler, scipy, pandas, contourpy, matplotlib, seaborn\n",
      "Successfully installed contourpy-1.0.7 cycler-0.11.0 fonttools-4.38.0 kiwisolver-1.4.4 matplotlib-3.6.3 numpy-1.24.1 pandas-1.5.3 pillow-9.4.0 py3Dmol-2.0.0.post2 pyparsing-3.0.9 scipy-1.10.0 seaborn-0.12.2\n"
     ]
    }
   ],
   "source": [
    "!pip install numpy pandas scipy matplotlib seaborn py3Dmol ipywidgets "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Collecting prody\n",
      "  Using cached ProDy-2.3.1-cp38-cp38-linux_x86_64.whl\n",
      "Requirement already satisfied: numpy>=1.10 in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from prody) (1.24.1)\n",
      "Collecting biopython\n",
      "  Using cached biopython-1.80-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (3.1 MB)\n",
      "Requirement already satisfied: scipy in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from prody) (1.10.0)\n",
      "Requirement already satisfied: pyparsing in /home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages (from prody) (3.0.9)\n",
      "Installing collected packages: biopython, prody\n",
      "Successfully installed biopython-1.80 prody-2.3.1\n"
     ]
    }
   ],
   "source": [
    "!pip install prody"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "#!pip install pydca\n",
    "#not working use carlos version"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages/Bio/pairwise2.py:278: BiopythonDeprecationWarning: Bio.pairwise2 has been deprecated, and we intend to remove it in a future release of Biopython. As an alternative, please consider using Bio.Align.PairwiseAligner as a replacement, and contact the Biopython developers if you still need the Bio.pairwise2 module.\n",
      "  warnings.warn(\n"
     ]
    }
   ],
   "source": [
    "import numpy as np\n",
    "import pandas as pd\n",
    "\n",
    "import pickle\n",
    "import scipy.signal as scs\n",
    "\n",
    "import matplotlib.pyplot as plt\n",
    "import seaborn as sns\n",
    "\n",
    "import py3Dmol\n",
    "from matplotlib import pyplot as plt, colors\n",
    "from matplotlib.colors import Normalize\n",
    "\n",
    "from matplotlib import cm\n",
    "from matplotlib.colors import BoundaryNorm\n",
    "from matplotlib.ticker import MaxNLocator\n",
    "import matplotlib\n",
    "import ipywidgets as widgets\n",
    "from ipywidgets import interact, interact_manual\n",
    "\n",
    "import os\n",
    "import dca_frustratometer\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "path_='/home/ezequiel/Desktop/LFP_local/foldon_data/'\n",
    "\n",
    "results_table_pdb=pd.read_csv(path_+'results_table_pdb.csv',index_col=0)\n",
    "results_table_pdb=results_table_pdb.drop_duplicates(subset=['pfam'],ignore_index=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "def view_3d_exon(pdb_filename,color):\n",
    "\n",
    "    view = py3Dmol.view(js='https://3dmol.org/build/3Dmol.js')\n",
    "    view.addModel(open(pdb_filename,'r').read(),'pdb')\n",
    "        \n",
    "\n",
    "        \n",
    "    view.setBackgroundColor('white')\n",
    "\n",
    "    view.setStyle({'cartoon':{'color':color}})\n",
    "    \n",
    "    view.zoomTo(viewer=(0,0))\n",
    "    return view\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "def view_3d_exon_hist(ali_seq,colors,pdb,file=False,pdb_filename=''):\n",
    "\n",
    "    if file:\n",
    "        view = py3Dmol.view(js='https://3dmol.org/build/3Dmol.js')\n",
    "        view.addModel(open(pdb_filename,'r').read(),'pdb')\n",
    "        \n",
    "    else:\n",
    "        view = py3Dmol.view(query='pdb:'+pdb,width=800, height=600)\n",
    "        \n",
    "    view.setBackgroundColor('white')\n",
    "\n",
    "    view.setStyle({'cartoon':{'color':'white'}})\n",
    "    view.setStyle({'chain':'B'},{'opacity':0 })\n",
    "    view.setStyle({'chain':'C'},{'opacity':0 })\n",
    "    view.setStyle({'chain':'D'},{'opacity':0 })\n",
    "    view.setStyle({'chain':'E'},{'opacity':0 })\n",
    "    view.setStyle({'chain':'F'},{'opacity':0 })\n",
    "    view.setStyle({'chain':'G'},{'opacity':0 })\n",
    "\n",
    "    if len(ali_seq.shape)>1: #choose the instance with less gaps\n",
    "        count_aa=[]\n",
    "        for seqs in ali_seq:\n",
    "            count_aa.append(len(seqs[seqs>0]))\n",
    "        print(np.argmax(count_aa))\n",
    "        ali_seq=ali_seq[np.argmax(count_aa)]\n",
    "    \n",
    "    #change residue color\n",
    "    for i,res in enumerate(ali_seq):\n",
    "        if res>0: #gaps are res=-1 and we dont want them\n",
    "            view.addStyle({'chain':'A','resi':[str(res)]},{'cartoon':{'color':colors[i]}})\n",
    "        #view.addStyle({'chain':'A','resi':[str(alinum.no[e_])]},{'cartoon':{'color':'white'}})\n",
    "    view.zoomTo(viewer=(0,0))\n",
    "    return view\n",
    "\n",
    "    # esto no toma en cuenta si hay varias columnas juntas chicas (problemas de alineamiento)\n",
    "def find_common_bs(exon_freq,order,thresh,border,npos):\n",
    "    final_bs=scs.argrelmax(exon_freq,order=order)[0]\n",
    "    final_bs=final_bs[exon_freq[final_bs]>thresh]\n",
    "    final_bs=final_bs[final_bs>=border]\n",
    "    final_bs=final_bs[final_bs<=(npos-border)]\n",
    "    final_bs=np.hstack([0,final_bs,npos])\n",
    "    return final_bs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "colors= sns.color_palette().as_hex() + list(pd.read_csv('/home/ezequiel/Desktop/LFP_local/alignments/colors.csv')['0'].values)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Cytochrom_C\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "array([ 0, 23, 50, 67, 84])"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "order=10\n",
    "threshold=0.01\n",
    "border=order\n",
    "f=results_table_pdb.name[1]\n",
    "print(f)\n",
    "path_f=path_+f+'/'\n",
    "with open (path_+'exon_pdb_ali/exon_pdb_ali_'+f, 'rb') as fp:\n",
    "    exon_freq_old, ali_seq_num_pdb, pdb = pickle.load(fp)\n",
    "npos=len(exon_freq_old)\n",
    "exon_table=pd.read_csv(path_f+'exon_table.csv')\n",
    "exon_freq=np.histogram(exon_table.exon_bs_aa,bins=np.arange(-0.5,npos+0.5),density=True,\n",
    "                       weights=exon_table.exon_rew)[0]\n",
    "final_bs=find_common_bs(exon_freq,order,threshold,border,npos)\n",
    "final_bs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_16742557277086334\"  style=\"position: relative; width: 800px; height: 600px\">\n        <p id=\"3dmolwarning_16742557277086334\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    //this is to ignore the existence of requirejs amd\n    var savedexports, savedmodule;\n    if (typeof exports !== 'undefined') savedexports = exports;\n    else exports = {}\n    if (typeof module !== 'undefined') savedmodule = module;\n    else module = {}\n\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n        exports = savedexports;\n        module = savedmodule;\n        resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://cdn.jsdelivr.net/npm/3dmol@latest/build/3Dmol-min.min.js');\n}\n\nvar viewer_16742557277086334 = null;\nvar warn = document.getElementById(\"3dmolwarning_16742557277086334\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_16742557277086334 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_16742557277086334\"),{backgroundColor:\"white\"});\n$3Dmol.download(\"pdb:1cyc\", viewer_16742557277086334, {}, function() {\nviewer_16742557277086334.zoomTo();\n\tviewer_16742557277086334.setBackgroundColor(\"white\");\n\tviewer_16742557277086334.setStyle({\"cartoon\": {\"color\": \"white\"}});\n\tviewer_16742557277086334.setStyle({\"chain\": \"B\"},{\"opacity\": 0});\n\tviewer_16742557277086334.setStyle({\"chain\": \"C\"},{\"opacity\": 0});\n\tviewer_16742557277086334.setStyle({\"chain\": \"D\"},{\"opacity\": 0});\n\tviewer_16742557277086334.setStyle({\"chain\": \"E\"},{\"opacity\": 0});\n\tviewer_16742557277086334.setStyle({\"chain\": \"F\"},{\"opacity\": 0});\n\tviewer_16742557277086334.setStyle({\"chain\": \"G\"},{\"opacity\": 0});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"5\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"6\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"7\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"8\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"9\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"10\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"11\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"12\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"13\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"14\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"15\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"16\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"17\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"18\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"19\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"20\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"21\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"22\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"23\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"24\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"25\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"26\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"27\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"28\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"29\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"30\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"31\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"32\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"33\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"34\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"35\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"36\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"37\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"38\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"39\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"40\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"41\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"42\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"43\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"44\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"45\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"46\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"47\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"48\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"49\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"50\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"51\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"52\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"53\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"54\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"55\"]},{\"cartoon\": {\"color\": \"#2ca02c\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"56\"]},{\"cartoon\": {\"color\": \"#2ca02c\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"57\"]},{\"cartoon\": {\"color\": \"#2ca02c\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"58\"]},{\"cartoon\": {\"color\": \"#2ca02c\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"59\"]},{\"cartoon\": {\"color\": \"#2ca02c\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"60\"]},{\"cartoon\": {\"color\": \"#2ca02c\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"61\"]},{\"cartoon\": {\"color\": \"#2ca02c\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"62\"]},{\"cartoon\": {\"color\": \"#2ca02c\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"63\"]},{\"cartoon\": {\"color\": \"#2ca02c\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"64\"]},{\"cartoon\": {\"color\": \"#2ca02c\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"65\"]},{\"cartoon\": {\"color\": \"#2ca02c\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"66\"]},{\"cartoon\": {\"color\": \"#2ca02c\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"67\"]},{\"cartoon\": {\"color\": \"#2ca02c\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"68\"]},{\"cartoon\": {\"color\": \"#2ca02c\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"69\"]},{\"cartoon\": {\"color\": \"#2ca02c\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"70\"]},{\"cartoon\": {\"color\": \"#2ca02c\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"71\"]},{\"cartoon\": {\"color\": \"#2ca02c\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"72\"]},{\"cartoon\": {\"color\": \"#d62728\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"85\"]},{\"cartoon\": {\"color\": \"#d62728\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"86\"]},{\"cartoon\": {\"color\": \"#d62728\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"87\"]},{\"cartoon\": {\"color\": \"#d62728\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"88\"]},{\"cartoon\": {\"color\": \"#d62728\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"89\"]},{\"cartoon\": {\"color\": \"#d62728\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"90\"]},{\"cartoon\": {\"color\": \"#d62728\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"91\"]},{\"cartoon\": {\"color\": \"#d62728\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"92\"]},{\"cartoon\": {\"color\": \"#d62728\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"93\"]},{\"cartoon\": {\"color\": \"#d62728\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"94\"]},{\"cartoon\": {\"color\": \"#d62728\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"95\"]},{\"cartoon\": {\"color\": \"#d62728\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"96\"]},{\"cartoon\": {\"color\": \"#d62728\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"97\"]},{\"cartoon\": {\"color\": \"#d62728\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"98\"]},{\"cartoon\": {\"color\": \"#d62728\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"99\"]},{\"cartoon\": {\"color\": \"#d62728\"}});\n\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"100\"]},{\"cartoon\": {\"color\": \"#d62728\"}});\n\tviewer_16742557277086334.zoomTo();\nviewer_16742557277086334.render();\n})\n});\n</script>",
      "text/html": [
       "<div id=\"3dmolviewer_16742557277086334\"  style=\"position: relative; width: 800px; height: 600px\">\n",
       "        <p id=\"3dmolwarning_16742557277086334\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n",
       "        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n",
       "        </div>\n",
       "<script>\n",
       "\n",
       "var loadScriptAsync = function(uri){\n",
       "  return new Promise((resolve, reject) => {\n",
       "    //this is to ignore the existence of requirejs amd\n",
       "    var savedexports, savedmodule;\n",
       "    if (typeof exports !== 'undefined') savedexports = exports;\n",
       "    else exports = {}\n",
       "    if (typeof module !== 'undefined') savedmodule = module;\n",
       "    else module = {}\n",
       "\n",
       "    var tag = document.createElement('script');\n",
       "    tag.src = uri;\n",
       "    tag.async = true;\n",
       "    tag.onload = () => {\n",
       "        exports = savedexports;\n",
       "        module = savedmodule;\n",
       "        resolve();\n",
       "    };\n",
       "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
       "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
       "});\n",
       "};\n",
       "\n",
       "if(typeof $3Dmolpromise === 'undefined') {\n",
       "$3Dmolpromise = null;\n",
       "  $3Dmolpromise = loadScriptAsync('https://cdn.jsdelivr.net/npm/3dmol@latest/build/3Dmol-min.min.js');\n",
       "}\n",
       "\n",
       "var viewer_16742557277086334 = null;\n",
       "var warn = document.getElementById(\"3dmolwarning_16742557277086334\");\n",
       "if(warn) {\n",
       "    warn.parentNode.removeChild(warn);\n",
       "}\n",
       "$3Dmolpromise.then(function() {\n",
       "viewer_16742557277086334 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_16742557277086334\"),{backgroundColor:\"white\"});\n",
       "$3Dmol.download(\"pdb:1cyc\", viewer_16742557277086334, {}, function() {\n",
       "viewer_16742557277086334.zoomTo();\n",
       "\tviewer_16742557277086334.setBackgroundColor(\"white\");\n",
       "\tviewer_16742557277086334.setStyle({\"cartoon\": {\"color\": \"white\"}});\n",
       "\tviewer_16742557277086334.setStyle({\"chain\": \"B\"},{\"opacity\": 0});\n",
       "\tviewer_16742557277086334.setStyle({\"chain\": \"C\"},{\"opacity\": 0});\n",
       "\tviewer_16742557277086334.setStyle({\"chain\": \"D\"},{\"opacity\": 0});\n",
       "\tviewer_16742557277086334.setStyle({\"chain\": \"E\"},{\"opacity\": 0});\n",
       "\tviewer_16742557277086334.setStyle({\"chain\": \"F\"},{\"opacity\": 0});\n",
       "\tviewer_16742557277086334.setStyle({\"chain\": \"G\"},{\"opacity\": 0});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"5\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"6\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"7\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"8\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"9\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"10\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"11\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"12\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"13\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"14\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"15\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"16\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"17\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"18\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"19\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"20\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"21\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"22\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"23\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"24\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"25\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"26\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"27\"]},{\"cartoon\": {\"color\": \"#1f77b4\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"28\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"29\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"30\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"31\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"32\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"33\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"34\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"35\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"36\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"37\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"38\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"39\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"40\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"41\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"42\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"43\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"44\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"45\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"46\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"47\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"48\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"49\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"50\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"51\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"52\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"53\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"54\"]},{\"cartoon\": {\"color\": \"#ff7f0e\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"55\"]},{\"cartoon\": {\"color\": \"#2ca02c\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"56\"]},{\"cartoon\": {\"color\": \"#2ca02c\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"57\"]},{\"cartoon\": {\"color\": \"#2ca02c\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"58\"]},{\"cartoon\": {\"color\": \"#2ca02c\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"59\"]},{\"cartoon\": {\"color\": \"#2ca02c\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"60\"]},{\"cartoon\": {\"color\": \"#2ca02c\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"61\"]},{\"cartoon\": {\"color\": \"#2ca02c\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"62\"]},{\"cartoon\": {\"color\": \"#2ca02c\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"63\"]},{\"cartoon\": {\"color\": \"#2ca02c\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"64\"]},{\"cartoon\": {\"color\": \"#2ca02c\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"65\"]},{\"cartoon\": {\"color\": \"#2ca02c\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"66\"]},{\"cartoon\": {\"color\": \"#2ca02c\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"67\"]},{\"cartoon\": {\"color\": \"#2ca02c\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"68\"]},{\"cartoon\": {\"color\": \"#2ca02c\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"69\"]},{\"cartoon\": {\"color\": \"#2ca02c\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"70\"]},{\"cartoon\": {\"color\": \"#2ca02c\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"71\"]},{\"cartoon\": {\"color\": \"#2ca02c\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"72\"]},{\"cartoon\": {\"color\": \"#d62728\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"85\"]},{\"cartoon\": {\"color\": \"#d62728\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"86\"]},{\"cartoon\": {\"color\": \"#d62728\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"87\"]},{\"cartoon\": {\"color\": \"#d62728\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"88\"]},{\"cartoon\": {\"color\": \"#d62728\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"89\"]},{\"cartoon\": {\"color\": \"#d62728\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"90\"]},{\"cartoon\": {\"color\": \"#d62728\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"91\"]},{\"cartoon\": {\"color\": \"#d62728\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"92\"]},{\"cartoon\": {\"color\": \"#d62728\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"93\"]},{\"cartoon\": {\"color\": \"#d62728\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"94\"]},{\"cartoon\": {\"color\": \"#d62728\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"95\"]},{\"cartoon\": {\"color\": \"#d62728\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"96\"]},{\"cartoon\": {\"color\": \"#d62728\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"97\"]},{\"cartoon\": {\"color\": \"#d62728\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"98\"]},{\"cartoon\": {\"color\": \"#d62728\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"99\"]},{\"cartoon\": {\"color\": \"#d62728\"}});\n",
       "\tviewer_16742557277086334.addStyle({\"chain\": \"A\", \"resi\": [\"100\"]},{\"cartoon\": {\"color\": \"#d62728\"}});\n",
       "\tviewer_16742557277086334.zoomTo();\n",
       "viewer_16742557277086334.render();\n",
       "})\n",
       "});\n",
       "</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "<py3Dmol.view at 0x7f57222c9460>"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "view=view_3d_exon_hist(ali_seq_num_pdb,\n",
    "                  np.hstack([np.repeat(colors[i],final_bs[i+1]-final_bs[i]) for i in range(len(final_bs)-1)]),\n",
    "                  pdb)\n",
    "view"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "def make_exon_pdb_files(pdb_file,final_bs,bs_name,path_,prg_dir):\n",
    "\n",
    "    path_bs=path_f+'pdb_files_'+bs_name+'/'\n",
    "    os.system('mkdir '+path_bs)\n",
    "    exon_start=[ali_seq_num_pdb[final_bs[i]] for i in range(len(final_bs[:-1]))]\n",
    "    exon_end=[ali_seq_num_pdb[final_bs[i]-1] for i in range(1,len(final_bs))]\n",
    "    for i in range(len(exon_start)):\n",
    "        output_1=path_bs+'exon_'+str(i)+'_aux.pdb'\n",
    "        output_2=path_bs+'exon_'+str(i)+'.pdb'\n",
    "        os.system('python3 '+prg_dir+'pdb_delres.py -:'+str(exon_start[i]-1)+' '+pdb_file+' > '+output_1)\n",
    "        os.system('python3 '+prg_dir+'pdb_delres.py -'+str(exon_end[i]+1)+': '+output_1+' > '+output_2)\n",
    "        os.system('rm '+output_1)\n",
    "        print(output_2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "mkdir: cannot create directory ‘/home/ezequiel/Desktop/LFP_local/foldon_data/Cytochrom_C/pdb_files_all/’: File exists\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/home/ezequiel/Desktop/LFP_local/foldon_data/Cytochrom_C/pdb_files_all/exon_0.pdb\n",
      "/home/ezequiel/Desktop/LFP_local/foldon_data/Cytochrom_C/pdb_files_all/exon_1.pdb\n",
      "/home/ezequiel/Desktop/LFP_local/foldon_data/Cytochrom_C/pdb_files_all/exon_2.pdb\n",
      "/home/ezequiel/Desktop/LFP_local/foldon_data/Cytochrom_C/pdb_files_all/exon_3.pdb\n"
     ]
    }
   ],
   "source": [
    "prg_dir='/home/ezequiel/anaconda3/lib/python3.8/site-packages/pdbtools/'\n",
    "pdb_code='1cyc'\n",
    "chain='A'\n",
    "bs_name='all' \n",
    "pdb_file=path_+'pdb/'+pdb_code+'.pdb'\n",
    "pdb_file_chain=path_+'pdb/'+pdb_code+'_'+chain+'.pdb'\n",
    "os.system('python3 '+prg_dir+'pdb_selchain.py -'+chain+' '+pdb_file+'> '+pdb_file_chain)\n",
    "make_exon_pdb_files(pdb_file_chain,final_bs,bs_name,path_,prg_dir)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "2312bd91acfb4db19c115d656be44eb4",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "interactive(children=(Dropdown(description='exon', options=(0, 1, 2, 3), value=0), Output()), _dom_classes=('w…"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "@interact\n",
    "def plot_single_exon(exon=range(len(final_bs)-1)):\n",
    "    path_bs=path_f+'pdb_files_'+bs_name+'/'\n",
    "    return view_3d_exon(path_bs+'exon_'+str(exon)+'.pdb',colors[exon])\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# frustration"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages/Bio/PDB/StructureBuilder.py:89: PDBConstructionWarning: WARNING: Chain A is discontinuous at line 1984.\n",
      "  warnings.warn(\n",
      "/home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages/Bio/PDB/StructureBuilder.py:89: PDBConstructionWarning: WARNING: Chain B is discontinuous at line 2027.\n",
      "  warnings.warn(\n",
      "/home/ezequiel/anaconda3/envs/dca-env/lib/python3.8/site-packages/Bio/PDB/StructureBuilder.py:89: PDBConstructionWarning: WARNING: Chain A is discontinuous at line 2070.\n",
      "  warnings.warn(\n",
      "@> 1677 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    }
   ],
   "source": [
    "model_all=dca_frustratometer.AWSEMFrustratometer(pdb_file,chain='A',sequence_cutoff=3,  distance_cutoff=10)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<AxesSubplot: >"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAg4AAAGuCAYAAAD4T45KAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjYuMywgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy/P9b71AAAACXBIWXMAAA9hAAAPYQGoP6dpAABgPElEQVR4nO3deXxM1/8/8NdMdpFFQjZkIWkiSqKWVKg1FaqW0qJVVBfVWopW0dbSoqFVS9Un2v5apXTRFlWtqF0VQVJKW7taErGU7BKROb8/fA1nMpNt7mzm9Xw87uMx77n3nHsyJDk559zzVgkhBIiIiIgqQW3pBhAREZHtYMeBiIiIKo0dByIiIqo0dhyIiIio0thxICIiokpjx4GIiIgqjR0HIiIiqjR2HIiIiKjS2HEgIiKiSmPHgYiIiCrNoh2HRYsWITQ0FK6uroiLi8PevXst2RwiIiKqgMU6Dt9++y3GjRuHqVOnIj09HTExMUhMTMSlS5cs1SQiIiKqgMpSSa7i4uLQsmVLfPTRRwAAjUaD+vXrY9SoUZg4caIlmkREREQVcLTETW/cuIG0tDRMmjRJ+55arUZCQgJ2795dqTpKCgurdM/sI39LcdbuXdrX9TonSOdqBodWqW4iIntxcc8uKfZ/MN7gtQdPXJTimHD/Kt2roPim9rW7i3G/rpxq1DCqfGWsfLCtIvX027NTkXpMxSIdhytXrqC0tBT+/vJ/In9/fxw5cqTM9cXFxSguLpbeU5eWwsXFxaTtJCIiIplNPFWRlJQELy8v6Zg9Z46lm0VERGR3LDLiULt2bTg4OODiRXkY6+LFiwgICChz/aRJkzBu3DjpPXVpqUnbSEREVCVqlaVbYBYW6Tg4OzujefPm2Lx5M3r37g3g1uLIzZs3Y+TIkWWud3FxKTMtUdU1Dt5R0VJ87KuvtK+v/PmndI5rHIjoXnXki8+lOOqZZ8u9/vg3K6RY7eAgxfIaB3mtfVRoHSkWxUVSrHJxLffeV7dv0r5279K13GvJfCzScQCAcePGYciQIWjRogVatWqF+fPno6CgAEOHDrVUk4iIiKpNpbKJ2X+jWazj0L9/f1y+fBlTpkxBVlYWYmNjkZKSUmbBJBERkS1Q2clUhcX2cTBWVacqynMy45oUFxaXSHGTBn6K3YuIyJbp/rxsWLeWwWtnfZ0uxd7uTlIc7Cc/Irl4k7wBYNN6d64f81hj6VxtL7eKG3sXczyO+f1DHRSp5/HftilSj6lYbMSBiIjoXmIvIw7sOBARESmBaxyIiIiosjjiYCazZs3CpEmT8Morr2D+/PkWaYPuHN3xr5dLccbZICmu26GTydtERGSN3M/+Jb9R1/A2y4MejpDikpvy/juhAd5S/MiDDaX4v9zr2te+nlVb00CmY9GOw759+/Dxxx+jadOmlmwGERGR0VQq+xhxsNiETH5+PgYOHIhPP/0UtWoZXpVLRERkC1RqtSKHtbNYC0eMGIHu3bsjISGh4ouJiIjIKlhkquKbb75Beno69u3bV6nrzZ0dM+LJp6X4wAfvSbFH/fpSXOovx7Vq3tlG9diKZdK5+wYOVqKJRETVorvltEdIiBTX7dhZik+s/FqKdYfj9055S/u61TszpHP+PjWl2LGKiwdtbl2DnSyONPuIw7lz5/DKK69gxYoVcHUtf5/y25gdk4iIrJ1KpVLksHZmH3FIS0vDpUuX8MADD2jfKy0txY4dO/DRRx+huLgYDjpJVJgdk4iIyDqYfcvpvLw8nDlzRnpv6NChiIqKwoQJE3D//fdXqh4lt5yustKbUnghW874VjPvzrapzh4e0rmL++XpmfqJj0ix9fc1ieheUnQpS4pd/QKkOO/fU1J8U+dn74WdO7WvvfoMlOtylv82ddGJi2/IP0t1pyZKCwu0r7NPnpCvbRKDqjDHltNrH+muSD09f/lZkXpMxewjDh4eHmU6B+7u7vD19a10p4GIiMja2MI0gxKs/7kPIiIishoW3zkSALZt22bpJhARERnHTp6qsIqOg81xkD82jc4ykfObNmpfuwUESudCH+0p16WzXuL4ym+kWPfR0Kq4lC3PRfp5m36Oj4hsi+6aBl0eoQ2kWPfxzOhhwyt9r9zCG1Jc0eOWV4/8o31dkp9f6ftYiopJroiIiKiy7CXJlX10j4iIiEgRHHFQQN3a8iOX2SV3ph88goOlcwfnyRtXxYx9TYqdaso7rRmDUxNEVJGS3GwpdvL0Lvd6Nz//at+rsKhEij1rOJd7fZ0HWlT7XpZgC3kmlGCRr7K0tBSTJ09GWFgY3Nzc0LBhQ0yfPh1m3lKCiIiIqsgiIw6zZ89GcnIyli5disaNG2P//v0YOnQovLy8MHr0aEs0iYiIiCrBIh2HXbt2oVevXuje/dYuW6Ghofj666+xd+9eSzSHiIjIeHayAZRFOg7x8fH45JNPcOzYMdx33304ePAgdu7ciblz51qiOYpr/NII7evj569K5+p3SZTibzYfkeL28UwzTkQV0HmMW/cR8fLoPqbt5CgnGyzYtEGK6yXIP7O8IyKk+O4MwBVl/824nCvFAT7u5TfWxtjLUxUW6ThMnDgRubm5iIqKgoODA0pLSzFz5kwMHDhQ7/XmTqtNRERE+llkceTKlSuxYsUKfPXVV0hPT8fSpUsxZ84cLF26VO/1TKtNRETWTqVSK3JYO4uMOIwfPx4TJ07EgAEDAABNmjTBmTNnkJSUhCFDhpS5nmm1iYjI6nGqwnQKCwuh1nne1cHBARqNRu/1Li4uZaYlLJpWuwoi6vlIcZG/lxTnnPpbiv85c0WK9x+VU972iA+X4u0Hzmpft4+V94z4349/SvHLvZpWosVEZPWqsKZBV0X7u9TSWdOgK89N/pnm07jyWY0PnZbXfJ2/LG8j3autvH7ibtfyi6S4Vk1XA1eSqVmk49CjRw/MnDkTwcHBaNy4Mf744w/MnTsXzz77rCWaQ0REZDR7SattkY7DwoULMXnyZLz88su4dOkSgoKC8OKLL2LKlCmWaA4REZHR7GXnSJWw0e0abWWqoiKL1sjTCV1L5KmLBk8MkGL76M8SESnLqYbpt+D/9Wn9TwZWVZflKxSpx1Tso3tEREREimCSKyIiIgXYy1QFOw5EREQK4OLIatqxYwfef/99pKWl4cKFC1i9ejV69+4NACgpKcFbb72FX375BadOnYKXlxcSEhIwa9YsBAUFKd0UmzCit/yIpIAcp/6VIcUPNq5r8jYRkf3Q6Kxys5OtCMgIio+rFBQUICYmBosWLSpzrrCwEOnp6Zg8eTLS09OxatUqHD16FD179lS6GURERGalUqsUOayd4iMO3bp1Q7du3fSe8/LywsaNG6X3PvroI7Rq1Qpnz55FcHCw3nJERERWzwa2i1aCxdc45OTkQKVSwdvb29JNsQq6fU3/E3Kq8SslIVJcO/YBE7eIiExt59hXpFh3rjx8gPxYdklenhTX6/xwte+dubn8bJgVSZ38hvZ13PR3q90Osh0W7TgUFRVhwoQJePLJJ+Hp6WnwOmbHJCIia2cL0wxKsNi4SklJCfr16wchBJKTk8u9ltkxiYjI2jE7pgnd7jScOXMGW7ZsKXe0AWB2TCIiImth9o7D7U7D8ePHsXXrVvj6+lZYxpazYxorrNdjUnwy45oUnzl6Qfu6eWSgdG72N39I8YQBzRRunXJuXP1Pip19Kv5/QXSvaPbaa1LsXrd+udfrPkJpjKquadAV0Lp1pa9d89txKa7vV1OKdX+G3e1mXq4UO3qU/wenRdjJVIXiHYf8/HycOHFCG58+fRoHDhyAj48PAgMD8fjjjyM9PR3r1q1DaWkpsrJupY328fGBs7Oz0s0hIiIyC24AVU379+9Hx44dtfHtKYYhQ4Zg2rRpWLt2LQAgNjZWKrd161Z06NBB6eYQERGZBbecrqYOHTqgvISbNpqMk4iIiGAF+zhQ1TSsW0uKy+uGNQm1wjlAA7imgexZRWsadFnTVHrIIz0qfW3vhyKqfR+rXNNgBZKSkrBq1SocOXIEbm5uiI+Px+zZsxEZGWmye9rHuAoREZGpqVTKHFWwfft2jBgxAnv27MHGjRtRUlKCLl26oKCgwERfJEcciIiIbFZKSooUf/HFF/Dz80NaWhratWtnknuaNTvmbf/88w8mTJiA7du34+bNm4iOjsYPP/zAXBXVUF7f9JEHG5rsvtnHjshvaDRS6B0VbbJ7ExFZI6UWR+rbLVnftgT65OTkALj1pKKpmDU7JgCcPHkSbdu2RVRUFLZt24Y///wTkydPhqurq9JNISIiMh+Fpir07ZaclJRU4e01Gg3GjBmDNm3a4P777zfdlylM+JiDSqUqM+IwYMAAODk54csvvzSqbnvZAMpaccSBiGyJU40aJr/HjtEjFakn7v0PqjXi8NJLL2H9+vXYuXMn6tWrp0hb9DHr4kiNRoOff/4Z9913HxITE+Hn54e4uDisWbPGnM0gIiJSnEqtVuRwcXGBp6endFTUaRg5ciTWrVuHrVu3mrTTAJh5ceSlS5eQn5+PWbNmYcaMGZg9ezZSUlLQp08fbN26Fe3btzdnc8gIXvdFSbFK58HQc5fl7WHr1yn/Uaqc40fl+iNM9ygRkSXc/R1SmHFOOnfj/+alb6sVbbphZjIdS+wcKYTAqFGjsHr1amzbtg1hYWEmv6dZOw6a/xvO7tWrF8aOHQvg1g6Su3btwuLFiw12HJhWm4iIrJ4FNtgYMWIEvvrqK/z444/w8PDQpnHw8vKCm5ubSe5p1qmK2rVrw9HREdHR8vx3o0aNcPbsWYPlmFabiIiorOTkZOTk5KBDhw4IDAzUHt9++63J7mnWEQdnZ2e0bNkSR4/Kw9LHjh1DSEiIwXJMq20e12/In6mbs4MUFxTf1L52vXldOldafEOKz2bJi1fr+8pZ8C7u3SPFju7uVWsskQ3T3SmyqjtH3iuOnZOz4t5X37Z3kFWpzL+noiXSOJg1O2ZwcDDGjx+P/v37o127dujYsSNSUlLw008/Ydu2bQbrtOe02kREZBtU1rQXuAmZNTvmF198gcceewyLFy9GUlISRo8ejcjISPzwww9o27at0k0hIiIihZk9OyYAPPvss3j22WeVvjUREZHlWGCqwhKYq8KOnVr1nRT7xz0oxdk6SVJcPO88UnnpxPFy6/YNbizF//11SIoLMjOk2D2orhQXXcrSvnb1Cyj3XkS2wD4GsavG1tc06LKXqQr76B4RERGRIjjiQEREpABLPFVhCew42LEGfZ6Q4lU7jkmxq7OTFD9yX5D2tVtAEMoTWOad2lLk1bipFDvqDPHtOnxe+7rgzGnp3MMtTb8zGhGZ3rY/zkhxh2aGH8vPLZQf+f7jWJYUt4+1guzKnKqonqSkJLRs2RIeHh7w8/ND7969y+zbUFRUhBEjRsDX1xc1a9ZE3759cfHiRaWbQkRERApTvOOwfft2jBgxAnv27MHGjRtRUlKCLl26oOCuhXZjx47FTz/9hO+++w7bt29HZmYm+vTpo3RTiIiIzEalUityWDvFpypSUlKk+IsvvoCfnx/S0tLQrl075OTk4LPPPsNXX32FTp06AQCWLFmCRo0aYc+ePXjwwQf1VUtERGTd7GSqwuRrHHL+L+ubj48PACAtLQ0lJSVISEjQXhMVFYXg4GDs3r2bHQcL6tPuPim+cfU/A1ca7/TKr6Q4YsBAKW4UemdNRK2ariZrBxFZTnlrGnR51nCWYqtY06DDFkYLlGDSjoNGo8GYMWPQpk0b3H//rTSxWVlZcHZ2hre3t3Stv7+/NquXLmbHJCIisg4m7R6NGDEChw8fxjfffGNUPcyOSURE1k6lVilyWDuTjTiMHDkS69atw44dO1CvXj3t+wEBAbhx4ways7OlUYeLFy8iIED/DoHMjklERFZPZf2/9JWgeMdBCIFRo0Zh9erV2LZtG8LC5GfumzdvDicnJ2zevBl9+/YFABw9ehRnz55F69at9dbJ7JiW4ewjbwd7ftMG7et6CYlVquvEyq+lOO/0v1L8709rpDgwvo329cXD6dI5/wfjq3RvJR1dtkSKa9aX51nrduxszuYQEZmd4h2HESNG4KuvvsKPP/4IDw8P7boFLy8vuLm5wcvLC8899xzGjRsHHx8feHp6YtSoUWjdujUXRhIREVk5xTsOycnJAG5lybzbkiVL8MwzzwAA5s2bB7Vajb59+6K4uBiJiYn43//+p3RTiIiIzEalto+nKlSiohzYVopTFZZ1dsMvUhyc+Ei51xddviTF5zZvlOLaTWOk2CH0zqOhuo9hERFVlVONGia/x/6Z7yhST4s3pyhSj6kwVwUREZECVFwcSURERJVmJ1MV9vFVEhERkSIUH3FISkrCqlWrcOTIEbi5uSE+Ph6zZ89GZGRkmWuFEHjkkUeQkpKC1atXo3fv3ko3h0ykojUNuor+uyLFpdevS3Gt6PuNbhMRkSXZy1SFRbJj3jZ//ny7+aCJiOgep1Irc1g5s2fHvO3AgQP44IMPsH//fgQGBirdDCIiIrOyhe2ilWD27JgAUFhYiKeeegqLFi0yuM002bZtf5yR4vgQfyl2qlnTnM1RzOW0fVJcu3lLKVaV3pTi7Ycyta+tMZsfEVFVmT07JgCMHTsW8fHx6NWrV6XqYXZMIiKydvaSVtvs2THXrl2LLVu2YP78+ZWuh9kxiYjI6qlVyhxWzuzZMbds2YKTJ09KmTEBoG/fvnjooYewbdu2MnUxOyYREZF1UHzLad3smBEREdL5rKwsXLkiP5rXpEkTLFiwAD169CiTTdMQbjltW07/uFqKc44fk+LY1yaYszlEZGfMseX0wflzFaknZsy4ii+yILNnxwwICNC7IDI4OLjSnQYiIiJrYy9PVSi+xiE5ORk5OTno0KEDAgMDtce3336r9K2IiIjIzBQfcajOzIeNJugkIiK6w06eqmCSKzKLsF6PWboJRGaV9+8pKS65a/fcw//7n3Su3aJks7SJTItTFUREREQ6OOJARESkBE5VVE9lsmNmZWVh/Pjx2LhxI/Ly8hAZGYk333wTffv2Vbo5ZCPEDXlnUJWzde4KWnxTI8UujuX/oLh76+0OzUJM0iayTgdy5B+vTo53tt3n1MS9iVMV1VSZ7JiDBw/G0aNHsXbtWhw6dAh9+vRBv3798McffyjdHCIiIrNQqdSKHNbOItkxd+3aheTkZLRq1QoA8NZbb2HevHlIS0tDs2bNlG4SERERKcTkXRt92THj4+Px7bff4urVq9BoNPjmm29QVFSEDh06mLo5REREpsFcFcYzlB1z5cqV6N+/P3x9feHo6IgaNWpg9erVCA8PN2VzyIrprmm4mZerfe3o4Wm2dhRmZkhxjaC6Unxt3y4pDohrLcU375qSA4BW/ne+xdLenS6da/7G5Gq3k6zfQzFMo073JpN2HG5nx9y5c6f0/uTJk5GdnY1Nmzahdu3aWLNmDfr164fffvsNTZo0KVMP02oTERFZB5NNVdzOjrl161YpO+bJkyfx0Ucf4fPPP0fnzp0RExODqVOnokWLFli0aJHeuphWm4iIrJ1KrVbksHYm2XL67uyYuomrCv8vq6Va58NxcHCARiM/6nYb02rbn7unJ85v3iidq9f5YZPdV3dqQtcVnSd/iq9dk+KQR3pI8d1fB6cmiBSk+/vCGn7hqqx/fYISzJ4dMyoqCuHh4XjxxRcxZ84c+Pr6Ys2aNdi4cSPWrVunt04XF5cy0xJMq01ERGR+incckpNvbWyi+4TEkiVL8Mwzz8DJyQm//PILJk6ciB49eiA/Px/h4eFYunQpHnnkEaWbQ0REZBa2MM2gBItkx4yIiMAPP/yg9K2JiIgsRsWpCiLroLumIWu3/JROQOu2ZmtLrUbRUly3Y2ez3duSLvyXL8WBvjUt1BKyVxd2bpfiwPiHLNQSYseBiIhICZyqICIiosriVAURERFVGhdHVlNycjKSk5Px77//AgAaN26MKVOmoFu3brh69SqmTp2KX3/9FWfPnkWdOnXQu3dvTJ8+HV5eXko3he5RumsaFq89JMXBfjWkON6nRIq974uq9r1rReqW1V0MfG/+xcE1DWRpgW3bS/GRs1ekOOhmjhR7Nmho8jbZK8U7DvXq1cOsWbMQEREBIQSWLl2KXr164Y8//oAQApmZmZgzZw6io6Nx5swZDB8+HJmZmfj++++VbgoREZH52EBKbCUo3nHo0UPeOW/mzJlITk7Gnj178Nxzz0mPYTZs2BAzZ87E008/jZs3b8LRkTMnRERkm1Q2kNlSCSb9TV1aWorvvvsOBQUFaN26td5rcnJy4OnpyU4DVdvwnnJitA17T0mx2tFJsXvdyM+T4hr36NSErh+2H5Xivu0jLdQSupccOnVJiv86fVWKB3S+MzWYe+qkdK6Ge20p9qwbonDryBCT/LY+dOgQWrdujaKiItSsWROrV69GdHR0meuuXLmC6dOnY9iwYeXWx+yYRERk9exkqsIkX2VkZCQOHDiA1NRUvPTSSxgyZAj+/vtv6Zrc3Fx0794d0dHRmDZtWrn1MTsmERFZO5Vapchh7Uwy4uDs7Izw8HAAQPPmzbFv3z4sWLAAH3/8MQAgLy8PXbt2hYeHB1avXg0np/KHkpkdk4iIyDqYZWGBRqPRTjXk5uYiMTERLi4uWLt2LVxdXSssz+yYVBWJrRpI8aV9qVJszGNaxdnZ1S5rS4ouy3PP8ffrpBsvuaF9efGPdOmUf6sHpTj35HEpFjrpkL0iyl8vcTl9v/Z1nQdalHstWVZVH06ODqsjxcH+hh/L1/2+9dA5/9+fB6TYt2lsBXdXnspOpioU7zhMmjQJ3bp1Q3BwMPLy8vDVV19h27Zt2LBhA3Jzc9GlSxcUFhZi+fLlyM3NRW5uLgCgTp06cHBwULo5RERE5mED0wxKULzjcOnSJQwePBgXLlyAl5cXmjZtig0bNuDhhx/Gtm3bkJp666+/21MZt50+fRqhoaFKN4eIiIgUpBKVyYNthThVQWQ+YxbvleK6te6sS2oaJg8v604V6Tp44qIUx4T7G9k6slW6u74eybwuxfOHtzJY9vP1h6U4xE/e3bRz81Apdqoh7yhrCqfWrFKknga9+1T62h07duD9999HWloaLly4gNWrV6N3796KtMMQ+5iQISIiMjFLPFVRUFCAmJgYLFq0yERfVVncdYmIiEgJFlgc2a1bN3Tr1s2s92THgYiIyIro2/RQ39OFlmLW7Ji37d69G2+++SZSU1Ph4OCA2NhYbNiwAW5ubko3h4iqYf4PB6S4tof8xFPDQHft64rWNOjimga6TXe7+Kp4ttv9Ulx8U2PgSvNRKq12UlIS3n77bem9qVOnVrhZormYNTtm48aNsXv3bnTt2hWTJk3CwoUL4ejoiIMHD0JtJ3nMiYiIyqNv00NrGW0AzJwds3Hjxhg7dixGjx6NiRMnaq+JjGTCHCIiIsC6piX0MWt2zEuXLiE1NRUDBw5EfHw8Tp48iaioKMycORNt27Y1ZVOIKuW/Qwel2EFnO/QcnQx9IY/IHWVblX1EziUzorOcaTA9U378Oa5RoMnbRFQVLo5WMGqt4gZQ1WYoO+aePXsAANOmTcOcOXMQGxuLZcuWoXPnzjh8+DAiIiL01sfsmEREZO2UWuNQFfn5+Thx4oQ2Pn36NA4cOAAfHx8EBweb5J5mzY6p+b896l988UUMHToUzZo1w7x58xAZGYnPP//cYH3MjklERFTW/v370axZMzRr1gwAMG7cODRr1gxTpkwx2T3Nmh3z9rqG6Oho6fpGjRrh7NmzButjdkwiIrJ2lkiJ3aFDB5h7A2izZscMDQ1FUFAQjh49Kp0/duxYuRtYMDsmmYtvkxgp1s2saU1rGjK2bpbfuGt+tW6HTuWWPfy/hVIc2PYhKc49J3fkg4N0MoraSRZAoiqxk+8Ls2bHVKlUGD9+PKZOnYqYmBjExsZi6dKlOHLkCL7//nulm0JEREQKM2t2TAAYM2YMioqKMHbsWFy9ehUxMTHYuHEjGjZsWEHNRERE1ssSUxWWoHjH4bPPPqvwmokTJ0r7OBAREdk6FacqiMivZZwUbxv+ohR3WPxxueVT35ykfR03M8motmx+5hkprllX3kvBqeadtMLnN26Uzmlu3pTivHMZUlyq87iz0Mjb98aOfa1Kbb1b6uQ3pDhu+rtVKr912Ava1x0/+bTa7aiqfdMmS3HLadPLvT5txjtS3Pwt061qJytlJzsg28dXSURERIrgiAMREZECVHayc6RKmPsBUIXwcUyyRrrfTCrpHeN+qPz9yWIpDuvZU4odnO88sqxylLNZOnl6S3F60kwpbjbuVSlWubhK8S975K22H3mQi5nJtjjVqGHye2T+tl2ReoIeaq9IPaZi8qmKWbNmQaVSYcyYMdr3ioqKMGLECPj6+qJmzZro27cvLl68aOqmEBERkZFM2nHYt28fPv74YzRt2lR6f+zYsfjpp5/w3XffYfv27cjMzESfPn1M2RQiIiLTUqmUOaycyToO+fn5GDhwID799FPUqlVL+35OTg4+++wzzJ07F506dULz5s2xZMkS7Nq1S5sEi4iIyNao1GpFDmtnssWRI0aMQPfu3ZGQkIAZM2Zo309LS0NJSQkSEhK070VFRSE4OBi7d+/Ggw8+aKomEZnc6h3HpPjuJUR920caVXf0sOHlni/+77L2tbOnb7nXPjDpzSrdm2saiOg2k3QcvvnmG6Snp2Pfvn1lzmVlZcHZ2Rne3t7S+/7+/sjKytJbH9NqExGRtbOF0QIlKP5Vnjt3Dq+88gpWrFgBV1fXigtUAtNqExGR1bOTNQ6KjzikpaXh0qVLeOCBB7TvlZaWYseOHfjoo4+wYcMG3LhxA9nZ2dKow8WLFxEQEKC3TqbVJlvRp919UqybXVNJGdu2SHFFGTGJiJSgeMehc+fOOHTokPTe0KFDERUVhQkTJqB+/fpwcnLC5s2b0bdvXwDA0aNHcfbsWbRu3VpvnUyrTURE1s5epioU7zh4eHjg/vvvl95zd3eHr6+v9v3nnnsO48aNg4+PDzw9PTFq1Ci0bt2aCyOJiMhm2cvOkRbZcnrevHlQq9Xo27cviouLkZiYiP/973+WaAoREZEi7GXEgVtOExHRPc8cW05fTt+vSD11HmihSD2mYh/dIyIiIlIEs2MSEREpwU6mKuzjqyQiIiJFWCQ75m1CCHTr1g0qlQpr1qwxdVOIiIhMRqVSKXJYO5NOVRjKjnnb/PnzbeJDIiIiqoi9PFVh9uyYtx04cAAffPABPv/8c1M1gYiIiBRmso7D3dkxdRUWFuKpp57CokWLDG4zTUTlK8m+Jh0383K1BxFZgEqtzGHlzJ4dEwDGjh2L+Ph49OrVq1L1MTsmERFZO5XaPqbezZ4dc+3atdiyZQvmz59f6TqZHZOIiMg6KL5z5Jo1a/DYY4/BwcFB+15paSlUKhXUajVeeuklLFq0COq7FpGUlpZCrVbjoYcewrZt28rUyREHorJKsq9Jsequ7zlHD09zN4fIqplj58hrR/5WpJ5aUdGK1GMqincc8vLycObMGem9u7Nj1q5dG1euXJHON2nSBAsWLECPHj0QFhZWqftwy2myd6d/XC3FYb0es1BLSJ+Cc/LPQff6IRZqCQHm6ThkH/1HkXq8IxspUo+pWCQ7pr4FkcHBwZXuNBAREVkdPo5JREREJGN2TCIb8c//+1iKI/o/KcV3r+h2cPcwS5vojus3SqXY1dlBiu1jvb31MsdURc6J44rU4xUeoUg9psIkV0RERArg45hEREREOjjiQEREpADmqlCAvsyYWVlZGDRoEAICAuDu7o4HHngAP/zwgymbQXRPaPT8i9JRnJsjHzm5d47/LksHmZ6bs4N0qADpIDtgJ1tOm6yFhjJjDh48GEePHsXatWtx6NAh9OnTB/369cMff/xhqqYQERGRQkzScSgvM+auXbswatQotGrVCg0aNMBbb70Fb29vpKWlmaIpREREZqFSqxQ5rJ1JOg7lZcaMj4/Ht99+i6tXr0Kj0eCbb75BUVEROnToYIqmEBERmYVKpVbksHaKL46sKDPmypUr0b9/f/j6+sLR0RE1atTA6tWrER4ebrBO5qogKsu9bn0pztq9U/taU3JTOleSnyfF6XMWSvFjW35VuHW27+KeXVJ89a+/pDi87+NSXJybI8U1g0NN0i4iS1O0a1NRZkwAmDx5MrKzs7Fp0ybs378f48aNQ79+/XDo0CGD9TI7JhERWTuVWq3IYe0U3TmyosyYR48eRXh4OA4fPozGjRtrr0lISEB4eDgWL16st16OOBBVjCMOyuKIw73FHDtHFmZlKlJPjYAgReoxFUWnKjp37lxm5ODuzJiF/7dNtFqnR+Xg4ACNRmOwXhcXlzKdBG45TSQLaN1W+/rQwvnSuZvXi6TYzdfbDC2yvNTJb0hx3PR3K102P+O8FBdfuyrFJ75fKcUqRycpdnDV+ZmVe6fzFj1seKXbQbbDFtYnKEHRjkNFmTFLSkoQHh6OF198EXPmzIGvry/WrFmDjRs3Yt26dUo2hYiIiEzArN0jJycn/PLLL6hTpw569OiBpk2bYtmyZVi6dCkeeeQRczaFiIiIqsHkW05v27ZNiiMiIrhTJBER3XNsYQ8GJTCtNpEduLQvVYpPr1kjxc1efU2KnX18Td0kq5f37ykp9ghtIMU383LLLe/g7CzFxbl3rnet42dk66iqzLE4sujyRUXqca3jr0g9psIkV0RERErg4kgiIiKqLHuZqmDHgcgO+LWMk+LjX38txSpHB5As//w5KdadqnD08KxSfa519G+KR2RrFB9XmTZtGlQqlXRERUUBAK5evYpRo0YhMjISbm5uCA4OxujRo5GTk1NBrURERNaNuSqM0LhxY2zatOnOTRxv3SYzMxOZmZmYM2cOoqOjcebMGQwfPhyZmZn4/vvvTdEUIiIi81BxqqL6lTo6IiAgoMz7999/v/QoZsOGDTFz5kw8/fTTuHnzpraDQURERNbJJL+pjx8/jqCgILi6uqJ169ZISkpCcHCw3mtzcnLg6enJTgORGbWZO1+KxY1i/RfaMY/gECnOOX603Ou9wiPkN2xgyJmUpeKIQ/XExcXhiy++QGRkJC5cuIC3334bDz30EA4fPgwPDw/p2itXrmD69OkYNmxYuXUyyRUREVk9G8hsqQTFv8pu3brhiSeeQNOmTZGYmIhffvkF2dnZWLlSTgiTm5uL7t27Izo6GtOmTSu3TqbVJiIiq6dSKXNYObPsHNmyZUskJCQgKSkJAJCXl4fExETUqFED69atg6tr+Y8pccSBiEwtY+tmKXbzk3d31H0806tBQzmOiDRNw0gR5tg5Ujd9fXU51fSo+CILMvm4Sn5+Pk6ePInAwEAAt0YaunTpAmdnZ6xdu7bCTgNwK622p6endLDTQERE1kWl0GHdFO84vPbaa9i+fTv+/fdf7Nq1C4899hgcHBzw5JNPajsNBQUF+Oyzz5Cbm4usrCxkZWWhtLRU6aYQERGZj330G5TvOJw/fx5PPvkkIiMj0a9fP/j6+mLPnj2oU6cO0tPTkZqaikOHDiE8PByBgYHa49y5cxVXTkRERGUsWrQIoaGhcHV1RVxcHPbu3WuyezE7JhGRHgfmzJbi2NcmWKglpASzrHEoLFCkHqca7lW6/ttvv8XgwYOxePFixMXFYf78+fjuu+9w9OhR+Pkpn4nVPp4dISIiukfNnTsXL7zwAoYOHYro6GgsXrwYNWrUwOeff26S+7HjQEREZEWKi4uRm5srHbpPFt5248YNpKWlISEhQfueWq1GQkICdu/ebZL2seNARESkCGVWR+rbu+j2dga6rly5gtLSUvj7+0vv+/v7IysrywRfowl2jpw2bRrefvtt6b3IyEgcOXJEG+/evRtvvvkmUlNT4eDggNjYWGzYsAFubm5KN4eITOBS9p01Rn7eFcwdl96UY4eq/djJPvK39rV3VHSVylZFQYa8QLtBn8dNdi+i8kyaNAnjxo2T3rOmLQjMmh0TuNVp6Nq1KyZNmoSFCxfC0dERBw8ehNpOtuokIiIqj4uLS6U7CrVr14aDgwMuXrwovX/x4kW9ySaVYNbsmAAwduxYjB49GhMnTtS+FxnJHdeIiMi2WeIRRWdnZzRv3hybN29G7969AQAajQabN2/GyJEjTXJPs2bHvHTpElJTUzFw4EDEx8fj5MmTiIqKwsyZM9G2bVtTNIWITKC86QndIf/rNWtLcW2vqv3YMeX0xN3c69Y3y32IlDZu3DgMGTIELVq0QKtWrTB//nwUFBRg6NChJrmfWbNjnjp1CsCtdRBz5sxBbGwsli1bhs6dO+Pw4cOIiIjQWydzVRAREenXv39/XL58GVOmTEFWVhZiY2ORkpJSZsGkUky+AVR2djZCQkIwd+5cNGrUCG3atMGkSZPw7rvvaq9p2rQpunfvbnDVqL4Fl2+98QamvPmmKZtORNVQ8YgDF0GT+ZljA6gbCm1M6GyGthrDJFMVd/P29sZ9992HEydOoFOnTgCA6Gh56LFRo0Y4e/aswTr0rTBVM7cFERFZEdvch7nqTN5xuJ0dc9CgQQgNDUVQUBCOHj0qXXPs2DF069bNYB36Vphyy2ki63Twmpyl5/Lp81LcqlGgFAf61pTiH7Yf1Tl/Z/vd+PvrKdFEshI/7jwuxb3a6p+u1mfNb3LZ3g9VvqypCIssjzQ/xTsOr732Gnr06IGQkBBkZmZi6tSp2uyYKpUK48ePx9SpUxETE4PY2FgsXboUR44cwffff690U4iIiEhhinccbmfH/O+//1CnTh20bdtWmx0TAMaMGYOioiKMHTsWV69eRUxMDDZu3IiGDRsq3RQiIiKzsZepCmbHJCKTuvBfvhQHejpL8b6TV6S4tpe8MCws0Nsk7SL7Yo7FkYV5ymTHrOFRteyY5sbtGomIiKjSTL44koiIyB5obHMAv8rYcSAiIlKAjc78V5lJOg4ZGRmYMGEC1q9fj8LCQoSHh2PJkiVo0aIFgFsf7tSpU/Hpp58iOzsbbdq0QXJyssGdI4nIduk+bnnt78NS3LxBAylWu1r35jdk2LX8IimuVdO1SuUzruRJcd3aHka3iZSn+BqHa9euoU2bNnBycsL69evx999/44MPPkCtWrW017z33nv48MMPsXjxYqSmpsLd3R2JiYkoKioqp2YiIiLrJYQyh7VTfMRh9uzZqF+/PpYsWaJ9LywsTPtaCIH58+fjrbfeQq9evQAAy5Ytg7+/P9asWYMBAwYo3SQiIiKTs5c1DoqPOKxduxYtWrTAE088AT8/PzRr1gyffvqp9vzp06eRlZWFhIQE7XteXl6Ii4vD7t27lW4OERGRWQghFDmsneIjDqdOnUJycjLGjRuHN954A/v27cPo0aPh7OyMIUOGICsrCwDKZO3y9/fXntPF7JhE945a0fdLse6PyTO//CTFOceOaV+HDhslnfOsIe8JQZZ1JVveX6eqaxy4psE2KD7ioNFo8MADD+Ddd99Fs2bNMGzYMLzwwgtYvHhxtetMSkqCl5eXdMyeM0fBVhMRERlHoxGKHNZO8Y5DYGBgudkvAwICAAAXL16Urrl48aL2nK5JkyYhJydHOia89prSTSciIqo2TlVUU5s2bfRmvwwJCQFwa6FkQEAANm/ejNjYWABAbm4uUlNT8dJLL+mtk9kxie5dKp045OFEKT7vfGc64r8c+fv++Ln/pLh5pJx5syou7tklt8vBQYr9WsZVu257UVhUYukmkBko3nEYO3Ys4uPj8e6776Jfv37Yu3cvPvnkE3zyyScAAJVKhTFjxmDGjBmIiIhAWFgYJk+ejKCgIPTu3Vvp5hAREZmFvTxVoXjHoWXLlli9ejUmTZqEd955B2FhYZg/fz4GDhyoveb1119HQUEBhg0bhuzsbLRt2xYpKSlwda3aQhoiIiJrYQvrE5TA7JhEZF1Kbkjh+e1b75xqLE8XXM29LsWcqrCsgyfktWsx4f4GrjQ/c2THzMr6r+KLKiEgwFeRekyFuSqIyLo4yY9YeoSEal976qTYDvP3VOy2/g/GK1aXLTv3a4oU1+/Stdzrj3+zQvs66vEndc7q/l0qr2jJLZQ7iU6OcmfNzVmOK1+zZdjmn+FVx44DERGRAuxljYPij2MSERHRvYsjDkRk1bwiIrWvU//OkM5dyZHXOHRvHV7t++gOm1vzrpQ3dRbhOaoND9QfOC7vyBsboX+/nNvc69aV4n+zsqU4NMBbvrfbnbUDO/44I517uGUYymPMZ2wNUxO67GVxpElGHDIyMvD000/D19cXbm5uaNKkCfbv36/32uHDh0OlUmH+/PmmaAoREZFZcAOoarqdVrtjx45Yv3496tSpg+PHj0tptW9bvXo19uzZg6CgIKWbQURERCZg9rTat2VkZGDUqFHYsGEDunfvrnQziIiIyAQU7zisXbsWiYmJeOKJJ7B9+3bUrVsXL7/8Ml544QXtNRqNBoMGDcL48ePRuHFjpZtARPeouGh5/n3jvtNSfHrND1Ic1rtvufV9uu6Q9nVPvwLpXGnNmlKsm9XTkspb06CrojUNunwaN5HjCq4P6/XYnddVutO9h09VVNPttNoRERHYsGEDXnrpJYwePRpLly7VXjN79mw4Ojpi9OjRlaqzuLgYubm50qGbZpuIiMiSNEIoclg7s6fVTktLw4IFC/DFF19Apapcr5lptYmIiKyD2dNq//bbb7h06RKCg4Ph6OgIR0dHnDlzBq+++ipCQ0P11sm02kREZO2ERihyWDuzp9UeNGgQEhISpPOJiYkYNGgQhg4dqrdOptUmIn109wk48tdWKf7ns0+luNFzL0hxz9p52tdXD/0lnXPzl/MsVLTG4cqBdCmuHfuAFF/YuV37OrBte+nc7gnjpbj17PfLvdfldPnx9joPtCj3+qr4+5PFUhw9bHi516cnzdS+9n9Bnn6uW9tDsXbZAluYZlCC2dNq+/r6wtdXTuDh5OSEgIAAREZG6quSiIiIrITiUxW302p//fXXuP/++zF9+vQyabWJiIjuNUIoc1g7ptUmontWYaa8RXWNoLoGrgQu/JcvxYG+NQ1cqbxSnR/DDjoLx4+dk9M1B/jKUwC6j6X2bV/50dvj569KcT3HIinOc/WWYo8a8rTxzcw720zv/U/+W7Rz81Ap/u3gWSl2cJCvj7+/XoXtrS5zpNU+cjyj4osqISrC8P9Ta8AkV0RERFRpTHJFRESkAC6OJCIiokqz0Zn/KjNJxyEjIwMTJkzA+vXrUVhYiPDwcCxZsgQtWtx6ZCg/Px8TJ07EmjVr8N9//yEsLAyjR4/G8OHlP/ZDRFQVZdY0lN6Uwh9331kb0CzCzxxN0kt3TYOu0xdypNjT3VWK867LX9f2A3fWErSPDS637oh68qbSV/86JMV+jctPQngy7c6joZ379iv32odiym+LreOIQzVVJjvmuHHjsGXLFixfvhyhoaH49ddf8fLLLyMoKAg9e/ZUuklERESkEItkx9y1axeGDBmCDh06AACGDRuGjz/+GHv37mXHgYiIbJLGBnZ9VIJFsmPGx8dj7dq1ePbZZxEUFIRt27bh2LFjmDdvntLNISK6w0H+kRdz7c5ukUWbdsvXDhwshce/WSHFEQOU25vm4Dw5947QaKS40f3yrpXFe+RdLvtERkmxZ8PqTwmc+3WDFOtmy9TVsILpCXtiJzMVlsmOuXDhQkRHR6NevXpwdnZG165dsWjRIrRr105vncyOSUREZB3Mnh0TuNVx2LNnD9auXYu0tDR88MEHGDFiBDZt2qS3TmbHJCIia2cvabUVn6owlB3zhx9+AABcv34db7zxBlavXo3u3bsDAJo2bYoDBw5gzpw5ZRJgAbeyY44bN056T11aqnTTiYiIqo1rHKqpouyYJSUlKCkpgVotD3Y4ODhAozOvdxuzYxKRKYT26K19fWzFMunc0WVLpFjt7CzF/65bK8UqtfxIpdrRSYpvFhZoX1/aJ2e39ApvKMWFWRfLraukSJ6q9WwYAaXEjH1Nsbro3mT27Jienp5o3749xo8fDzc3N4SEhGD79u1YtmwZ5s6dq3RziIiIzIIbQFXT7eyYkyZNwjvvvIOwsLAy2TG/+eYbTJo0CQMHDsTVq1cREhKCmTNncgMoIiKyWXYyU8HsmERElaE7lVGrkbyWS3NT3r3Rv9WDla5798TXpVjt4CDFjZ59Vop1pyYKiuV7u7swm4Auc2THTD1wSpF64mIbKFKPqfB/FxERkQJs9O/wKmNabSIiIqo0jjgQEREpwBb2YFCC4mscQkNDcebMmTLvv/zyy5g+fTqmTp2KX3/9FWfPnkWdOnXQu3dvTJ8+HV5eXlW6D9c4EJE5lWRfk2In71oGriRrZI41DrvSTyhST/wD4YrUYyqKjzjs27cPpXdtznT48GE8/PDDeOKJJ5CZmYnMzEzMmTMH0dHROHPmDIYPH47MzEx8//33SjeFiIjIbOxlAyiTP1UxZswYrFu3DsePH4dKT8757777Dk8//TQKCgrg6Fj5fgxHHIjInDjiYNvMMeLw277jitTzUEvlNvQyBZMujrxx4waWL1+OZ599Vm+nAQBycnLg6elZpU4DERERWYZJf1uvWbMG2dnZeOaZZ/Sev3LlCqZPn45hw4aVW09xcXGZbJjq0tIy21ATEZmK7gjD9axMKVY7ydtCu/jWMXmbyLrYy+JIk444fPbZZ+jWrRuCgoLKnMvNzUX37t0RHR2NadOmlVsPs2MSEZG1E0Ioclg7k61xOHPmDBo0aIBVq1ahV69e0rm8vDwkJiaiRo0aWLduHVxdXcutiyMORGRtOOJgW8yxxmFb6tGKL6qEDnGRitRjKiabqliyZAn8/Py0qbNvy83NRWJiIlxcXLB27doKOw0As2MSkfVxC5BHUnNPnZRidhzsj708VWGSjoNGo8GSJUswZMgQadFjbm4uunTpgsLCQixfvhy5ubnIzc0FANSpUwcOOvuzExER2Qo76TeYpuOwadMmnD17Fs/qJGZJT09HamoqACA8XN7g4vTp0wgNDTVFc4iIiEghJuk4dOnSRe8Cjw4dOtjEwg8iIqKqsvbfbzNnzsTPP/+MAwcOwNnZGdnZ2dWqh5snEBEpwLNBQ0s3gSzM2h/HvHHjBp544gm0bt0an332WbXrYceBiIhIAUr1G/Q9SajvIYGqevvttwEAX3zxhVH1MK02ERGRFdG3d1FSUpKlm6XFEQciIjMQN+78Baly5h409yKlHsecNGkSxo0bJ71nTfsWKT7iEBoaCpVKVeYYMWKE9prdu3ejU6dOcHd3h6enJ9q1a4fr168r3RQiIiKz0QihyOHi4gJPT0/pMNRxmDhxot7fuXcfR44cUfTrNGtabeBWp6Fr166YNGkSFi5cCEdHRxw8eBBqNWdNiIiIquLVV181mA/qtgYNGih6T8U7DnXqyLulzZo1Cw0bNkT79u0BAGPHjsXo0aMxceJE7TWRkda9vSYREVFFLPFQRZ06dcr83jU1k65xuJ1We9y4cVCpVLh06RJSU1MxcOBAxMfH4+TJk4iKisLMmTPRtm1bUzaFiMii7l7XcOCD96RzPtGNpTi4m7xVP9kGa38c8+zZs7h69SrOnj2L0tJSHDhwAMCtDRlr1qxZ6XrMmlb71KlTAIBp06Zhzpw5iI2NxbJly9C5c2ccPnwYEREReuthkisiIiLjTJkyBUuXLtXGzZo1AwBs3boVHTp0qHQ9Zk2rrdFoAAAvvvgihg4dimbNmmHevHmIjIzE559/brAeptUmIiJrZ+1ptb/44gu996tKpwEw4YjDmTNnsGnTJqxatUr7XmBgIAAgOjpaurZRo0Y4e/aswbr0PZqivmsBJhERkaUxyZWR9KXVDg0NRVBQEI4elXOWHzt2DN26dTNYF9NqE9G9JPbV1y3dBKJqM2tabZVKhfHjx2Pq1KmIiYlBbGwsli5diiNHjuD77783RVOIiIjMQqkNoKydWdNqA8CYMWNQVFSEsWPH4urVq4iJicHGjRvRsCETxBAREVk7lbD2PKAGcKqCiIgqy6lGDZPf47sNBxWp54nEGEXqMRVu10hERESVxiRXRERECrD2DaCUwo4DERGRAmx05r/KFJ+qKC0txeTJkxEWFgY3Nzc0bNgQ06dPlz5QIQSmTJmCwMBAuLm5ISEhAcePH1e6KURERKQwxTsOs2fPRnJyMj766CP8888/mD17Nt577z0sXLhQe817772HDz/8EIsXL0Zqairc3d2RmJiIoqIipZtDRERkFhqhzGHtFJ+q2LVrF3r16qXd+Ck0NBRff/019u7dC+DWaMP8+fPx1ltvoVevXgCAZcuWwd/fH2vWrMGAAQOUbhIREZHJ2csaB8VHHOLj47F582YcO3YMAHDw4EHs3LlTuzPk6dOnkZWVhYSEBG0ZLy8vxMXFYffu3Uo3h4jonnPu1xTpIDInxUccJk6ciNzcXERFRcHBwQGlpaWYOXMmBg4cCADIysoCAPj7+0vl/P39ted0MTsmERFZO6GxdAvMQ/ERh5UrV2LFihX46quvkJ6ejqVLl2LOnDlSKs+qYnZMIiKydhohFDmsneIjDuPHj8fEiRO1axWaNGmCM2fOICkpCUOGDEFAQAAA4OLFi9psmbfj2NhYvXUyOyYREVk7G/idrwjFOw6FhYVQq+WBDAcHB2g0t8ZwwsLCEBAQgM2bN2s7Crm5uUhNTcVLL72kt05mxyQiuqN+l65SrPv7SmW+ppAdUrzj0KNHD8ycORPBwcFo3Lgx/vjjD8ydO1eb8EqlUmHMmDGYMWMGIiIiEBYWhsmTJyMoKAi9e/dWujlERERmYQvTDEpQvOOwcOFCTJ48GS+//DIuXbqEoKAgvPjii5gyZYr2mtdffx0FBQUYNmwYsrOz0bZtW6SkpMDV1VXp5hAREZmFLezBoARmxyQisnGcqqiYObJj/r/V+xWp5/nHWihSj6kwVwURkY3T7SiUlN7pSjg5lN+NuH5DXmh+ebO8L0Rwt+5Gtc2e2Ojf4VXGjgMREZEC7GWNg+L7OBAREdG9S/ERh9LSUkybNg3Lly9HVlYWgoKC8Mwzz+Ctt96CSlV2yGz48OH4+OOPMW/ePIwZM0bp5hAR2Z3ypieuZ2VKsaPO3P+NvNxy675yIF2KM3ds175uOnpsZZt4T7KTAQflOw63s2MuXboUjRs3xv79+zF06FB4eXlh9OjR0rWrV6/Gnj17EBQUpHQziIiIzEpjJ49VmD075m0ZGRkYNWoUNmzYoL2WiIiIrJvZs2MCgEajwaBBgzB+/Hg0btxY6SYQERGZnUYoc1g7s2fHBG5NZzg6OpaZuiAiItNyC5Cnhn/adUKKH2oaU2752rEPlBvbM65xqKa7s2M2btwYBw4cwJgxYxAUFIQhQ4YgLS0NCxYsQHp6ut7FkvowrTYREZF1UHyq4u7smE2aNMGgQYMwduxYJCUlAQB+++03XLp0CcHBwXB0dISjoyPOnDmDV199FaGhoXrrZFptIiIi62D27JiDBg1CQkKCdD4xMRGDBg3C0KFD9dbJtNpERKbRIz5cik+s/FqKXby9pTg/I0OKM7Zt1b6OffV1ZRtnY+xlAyizZ8f09fWFr6+vVMbJyQkBAQGIjIzUWyfTahMRkbWzk36DZbJjEhERkW1SvOPg4eGB+fPnY/78+ZUu8++//yrdDCIiIrPiVAUREdkdtaP8a0F3jUNxdrYU2/u6hrvZwh4MSmCSKyIiIqo0jjgQEREpQHCqgoiI7E2DPk9IcUHGOSm+kZcnxQc+eE/72t6nLThVUU2lpaWYPHkywsLC4ObmhoYNG2L69OlSTyw/Px8jR45EvXr14ObmhujoaCxevFjpphAREZHCLJJWe9y4cdiyZQuWL1+O0NBQ/Prrr3j55ZcRFBSEnj17Kt0kIiIik7OXqQrFRxzuTqsdGhqKxx9/HF26dJHSau/atQtDhgxBhw4dEBoaimHDhiEmJqZM6m0iIiJbodEoc1g7i6TVjo+Px9q1a5GRkQEhBLZu3Ypjx46hS5cuSjeHiIiM4F63vnS4eHtLh4OLq/awd0yrXU2VSau9cOFCDBs2DPXq1YOjoyPUajU+/fRTtGvXTm+dzI5JRERkHRQfcbg7rXZ6ejqWLl2KOXPmYOnSpdprFi5ciD179mDt2rVIS0vDBx98gBEjRmDTpk1662R2TCIisnZCCEUOa6cSCreyfv36mDhxIkaMGKF9b8aMGVi+fDmOHDmC69evw8vLC6tXr0b37t211zz//PM4f/48UlJSytTJEQciIuuQc/yoFJ/dsEH7usnI0eZuTqU51ahh8nu8/f92KlLP1OfbKlKPqZg9rXZJSQlKSkrKvUYXs2MSEVkHrwg5i3Hexx/fef3vKemcR2gDs7SJzMvsabU9PT3Rvn17jB8/Hm5ubggJCcH27duxbNkyzJ07V+nmEBERmQWTXFVTZdJqf/PNN5g0aRIGDhyIq1evIiQkBDNnzsTw4cOVbg4REZFZ2Em/Qfk1DubCqQoiIuty/PxVKXZ1kf82rV/H05zNkZhjjcOUT35TpJ53hj2kSD2mwlwVRERECrCFPRiUwI4DERGRAuxljYPi+zgQERHRvcskHYe8vDyMGTMGISEhcHNzQ3x8PPbt2wfg1uOYEyZMQJMmTeDu7o6goCAMHjwYmZmZpmgKERGZSUQ9H+moV8dTOopvaqTjXiOEMoe1M0nH4fnnn8fGjRvx5Zdf4tChQ+jSpQsSEhKQkZGBwsJCpKenY/LkyUhPT8eqVatw9OhRZsUkIiKbZi+5KhR/quL69evw8PDAjz/+KO0M2bx5c3Tr1g0zZswoU2bfvn1o1aoVzpw5g+Dg4Erdh09VEBFZN91fLjd0RhlcHM03W26Opyom/G+HIvXMfll/3iZrofjiyJs3b6K0tBSurnKmNDc3N+zcqX87zpycHKhUKnh7eyvdHCIishCVTnzlt61SXLNePSkuvJglxYFt25uiWWQkxTsOHh4eaN26NaZPn45GjRrB398fX3/9NXbv3o3w8PAy1xcVFWHChAl48skn4emp/xlf5qogIiJrZ6PbIlWZScaJvvzySwghULduXbi4uODDDz/Ek08+WSY/RUlJCfr16wchBJKTkw3Wx+yYRERk7exljYNJ9nFo2LAhtm/fjoKCAuTm5iIwMBD9+/dHgwZ3Ep7c7jScOXMGW7ZsMTjaAACTJk3CuHHjpPfUpaWmaDoRERGVw6QbQLm7u8Pd3R3Xrl3Dhg0b8N577wG402k4fvw4tm7dCl9f33LrYXZMIiLbV7djZyn+7eBZKf7zSi0pHmHyFinLFkYLlGCSjsOGDRsghEBkZCROnDiB8ePHIyoqCkOHDkVJSQkef/xxpKenY926dSgtLUVW1q0FMT4+PnB2djZFk4iIiEyKO0caIScnByNGjEBUVBQGDx6Mtm3bYsOGDXByckJGRgbWrl2L8+fPIzY2FoGBgdpj165dpmgOERGRXfv333/x3HPPISwsDG5ubmjYsCGmTp2KGzduVLkuk4w49OvXD/369dN7LjQ01G5WnhIRkf2w5l9tR44cgUajwccff4zw8HAcPnwYL7zwAgoKCjCnig8bMK02ERFZhenL06T4gYZe2tfd48Lki9UOVarbHBtAjZq3TZF65rzcuswWBPrW+hnr/fffR3JyMk6dOlWlckxyRUREZEX0bUGQlJSk+H1ycnLg4+NT5XJMq01ERKQApZ6q0LcFgdKjDSdOnMDChQurPE0BWCA75m3//PMPevbsCS8vL7i7u6Nly5Y4e/asgRqJiOheN/np5tLhWcNZe0DtIB1pRy9IhzUQQihyuLi4wNPTUzoMdRwmTpwIlUpV7nHkyBGpTEZGBrp27YonnngCL7zwQpW/TpOMODz//PM4fPgwvvzySwQFBWH58uVISEjA33//jbp16+LkyZNo27YtnnvuObz99tvw9PTEX3/9VSa/BRERka2wxD4Or776Kp555plyr7l788XMzEx07NgR8fHx+OSTT6p1T4tkxxwwYACcnJzw5ZdfVvs+XBxJRHRvu3uDqIdi5MzJuqMMzSMDy63LHIsjh8/ZWvFFlbD4tY6K1KMrIyMDHTt2RPPmzbF8+XI4OFRtgeltik9VVJQdU6PR4Oeff8Z9992HxMRE+Pn5IS4uDmvWrFG6KURERGZjzbkqMjIy0KFDBwQHB2POnDm4fPkysrKytBswVoXZs2NeunQJ+fn5mDVrFmbMmIHZs2cjJSUFffr0wdatW9G+PdOoEhGRPMpQfFNT7rUFxTel2N3F/Gv/rXlzg40bN+LEiRM4ceIE6umkM6/qxIPZs2NqNLf+8Xv16oWxY8ciNjYWEydOxKOPPorFixfrra+4uBi5ubnSofuMKxEREen3zDPPGFyMWVUm6Tjczo6Zn5+Pc+fOYe/evSgpKUGDBg1Qu3ZtODo6Ijo6WirTqFEjg09VMK02ERFZO40QihzWzuzZMZ2dndGyZUscPXpUuvbYsWMICQnRWw/TahMR2TcXR/nv3IvX5AXyIQElUmyJqQpmxzRCedkxAWD8+PHo378/2rVrh44dOyIlJQU//fQTtm3bprc+ptUmIiKyDibpOOTk5GDSpEk4f/48fHx80LdvX8ycORNOTk4AgMceewyLFy9GUlISRo8ejcjISPzwww9o27atKZpDRERkcvYy4sAkV0REZHN+2XNSils1CpLi2l5uUmyOfRwGvbtFkXq+fKOTIvWYCnNVEBGRzXnkwYZSfOPqfzpXuIFMgx0HIiIiBdjm+H3VMa02ERERVRpHHIiIiBRgL4sjqzzisGPHDvTo0QNBQUFQqVRlckwIITBlyhQEBgbCzc0NCQkJOH78uHTN1atXMXDgQHh6esLb2xvPPfcc8vPzjfpCiIjIfjn7+EoHmU6VOw4FBQWIiYnBokWL9J5/77338OGHH2Lx4sVITU2Fu7s7EhMTUVRUpL1m4MCB+Ouvv7Bx40asW7cOO3bswLBhw6r/VRAREVmYvewcadTjmCqVCqtXr0bv3r0B3BptCAoKwquvvorXXnsNwK09Hfz9/fHFF19gwIAB+OeffxAdHY19+/ahRYsWAICUlBQ88sgjOH/+PIKCggzdTsLHMYmIqLLM8Thmv3c2K1LPyimdFanHVBRdHHn69GlkZWUhISFB+56Xlxfi4uKwe/duAMDu3bvh7e2t7TQAQEJCAtRqNVJTU5VsDhERESlM0cWRt/N6+/v7S+/7+/trz2VlZcHPz09uhKMjfHx8DOYFLy4uLpMNU11aWmYbaiIiIkvh4kgrwuyYRERk7TRCmcPaKTriEBAQAAC4ePEiAgMDte9fvHgRsbGx2msuXboklbt58yauXr2qLa+L2TGJiIisg6IjDmFhYQgICMDmzXcWiOTm5iI1NRWtW7cGALRu3RrZ2dlIS0vTXrNlyxZoNBrExcXprdfFxQWenp7SwWkKIiKyJkIoc1i7Ko845Ofn48SJE9r49OnTOHDgAHx8fBAcHIwxY8ZgxowZiIiIQFhYGCZPnoygoCDtkxeNGjVC165d8cILL2Dx4sUoKSnByJEjMWDAgEo/UUFERGRtbGGaQQlV7jjs378fHTt21Ma3pxCGDBmCL774Aq+//joKCgowbNgwZGdno23btkhJSYGrq6u2zIoVKzBy5Eh07twZarUaffv2xYcffqjAl0NERESmxLTaRER0zzPHPg49Jiuzj8NP0617HwfmqiAiIlIApyqIiIio0jQaS7fAPGxiHwciIiKyEsKGFRUVialTp4qioiKzlrXXe9tqu+313rbabkve21bbbcl722q7qfpsuuOQk5MjAIicnByzlrXXe9tqu+313rbabkve21bbbcl722q7qfo4VUFERESVxo4DERERVRo7DkRERFRpNt1xcHFxwdSpU6uVt8KYsvZ6b1ttt73e21bbbcl722q7LXlvW203VZ/N7hxJRERE5mfTIw5ERERkXuw4EBERUaWx40BERESVxo4DERERVZpddxy4LpSIiKhqbCo75pUrV/D5559j9+7dyMrKAgAEBAQgPj4ezzzzDOrUqVOl+lxcXHDw4EE0atTIFM0lIiK659jM45j79u1DYmIiatSogYSEBPj7+wMALl68iM2bN6OwsBAbNmxAixYtypQdN26c3joXLFiAp59+Gr6+vgCAuXPnGrz/Rx99hL179+KRRx7BgAED8OWXXyIpKQkajQZ9+vTBO++8A0dH6+yH7d27t0xnq3Xr1mjVqlWV6+rUqROWLFmCkJCQCq89ePAg0tLS0KFDBzRo0AB//fUXFi1aBI1Gg8ceewyJiYlVvr+58DOrmhs3bmDNmjV6O/W9evWCs7Nzlets0KABNmzYgIiIiHKvW7duHfbu3YvExES0adMGW7ZswZw5c7Tfm8OGDavW12RqtvyZnT9/Ht7e3qhZs6b0fklJCXbv3o127doZLOfq6oratWsDAH777TcsXrwYZ8+eRUhICEaMGIHWrVtX4SsmS7CZjsODDz6ImJgYLF68GCqVSjonhMDw4cPx559/Yvfu3WXKqtVqxMTEwNvbW3p/+/btaNGiBdzd3aFSqbBlyxa9954xYwbee+89dOnSBb///jvGjBmD999/H2PHjoVarca8efPw0ksv4e233y73a6jON5sx32iXLl1C37598fvvvyM4OFjqbJ09exZt2rTBDz/8AD8/vzJl165dq7fOPn36YMGCBahfvz4AoGfPnnqvW7VqFfr16wdvb28UFxdj9erVeOKJJ9CiRQs4ODhg06ZNWLZsGZ566ikDnxag0WigVpedTdNoNDh//jyCg4P1lisuLoZarYaTkxMA4OTJk/j888+1n9lzzz2HsLAwvWX5mVX9Mztx4gQSExORmZmJuLg46TNLTU1FvXr1sH79eoSHh+st/+GHH+p9f9y4cXj99dcREBAAABg9enSZaz7++GOMHDkSMTExOH78OBYtWoSXX34Z/fv3h4ODA5YtW4akpCS88soreu9x9+dT1c/NHj+zCxcuoFevXkhLS4NKpcJTTz2F//3vf9qfaRcvXkRQUBBKS0v1ti8uLg6TJ0/Go48+ih9//BF9+vTBo48+ikaNGuHYsWNYt24dVq1ahUcffVRv+duysrKQmpoqdbji4uK0XzeZmOXya1WNq6ur+Oeffwye/+eff4Srq6vec0lJSSIsLExs3rxZet/R0VH89ddfFd67YcOG4ocffhBCCHHgwAHh4OAgli9frj2/atUqER4ebrB8ZmamaNmypVCr1cLBwUEMGjRI5OXlac9nZWUJtVqtt2yrVq3ETz/9JIQQYs2aNUKtVouePXuKCRMmiMcee0w4OTlpz+vq27evaN26tThy5EiZc0eOHBHx8fHi8ccf11tWpVIJtVotVCqVwcNQm4UQ4oEHHhAzZswQQgjx9ddfC29vb/HOO+9oz8+ZM0fExsbqLZuTkyOeeOIJ4erqKvz8/MTkyZPFzZs3tefL+7yEEKJ9+/biu+++E0IIsXPnTuHi4iKaNm0q+vfvL5o1ayZq1Kghdu3apbcsP7Oqf2YJCQmiV69eejMU5uTkiF69eokuXboYvLdKpRL16tUToaGh0qFSqUTdunVFaGioCAsL01s2OjpafPLJJ0IIIbZs2SJcXV3FokWLtOeXLFkiGjVqZPDexnxu9viZDR48WMTFxYl9+/aJjRs3iubNm4sWLVqIq1evCiFufV4qlcpgu93d3cWpU6eEEELExcWJWbNmSecXLlwomjVrZrB8fn6+GDhwoHBwcBCOjo7Cz89P+Pn5CUdHR+Hg4CCefvppUVBQYLA8KcNmOg6hoaFi6dKlBs8vXbpUhISEGDy/d+9ecd9994lXX31V3LhxQwhR+Y6Dm5ubOHPmjDZ2cnIShw8f1sb//vuvqFGjhsHyxnyzGfONVrNmTZGenm6wXfv37xc1a9bUe65r166ie/fu4uLFi9L7lf3M3N3dxenTp4UQQmg0GuHk5CT+/PNP7fmTJ08avPfo0aPFfffdJ7777jvx6aefipCQENG9e3dRXFwshKj4h5Onp6c4duyYEOLWD/exY8dK59966y3Rpk0bvWX5mVX9M3NzcxOHDh0yWPeff/4p3NzcDJ5/8cUXRWxsrPj777+l9yvzuen73ry7LadPny73e9OYz80eP7OgoCCRmpqqjYuKikSPHj1EbGys+O+//yrsoHp5eYmDBw8KIYTw8/PTvr7txIkT5f57PffccyIiIkKkpKRIHbybN2+KDRs2iPvuu088//zzBsuTMmym4/DRRx8JFxcXMXr0aPHjjz+KPXv2iD179ogff/xRjB49Wri5uUm9Zn3y8vLE4MGDRdOmTcWhQ4eEk5NTpX6gh4WFifXr1wshhDh27JhQq9Vi5cqV2vM///yzCA0NNVjemG82Y77RfH19xbZt2wy2a+vWrcLX19fg+blz54r69etLIxqV/SUYEBAg9u/fL4QQ4urVq0KlUomtW7dqz+/du1cEBAToLRscHCxde/nyZdGqVSvRpUsXUVRUVOEPJ3d3d+3olL+/vzhw4IB0/sSJEwZ/AfMzq/pnFhgYaHDUSwgh1q5dKwIDAw2eF+LWqF39+vXFwoULte9V5nOrV6+e2LFjhxBCiIyMDKFSqcTPP/+sPb9t2zZRr149g+WN+dzs8TNzd3fXdpZuKykpEb179xZNmzYVf/75Z7n/z3r27CkmTpwohBAiMTFRLFiwQDr/6aefioiICIPlvb29xe+//27w/M6dO4W3t7fB86QMm+k4CCHEN998I+Li4oSjo6N26NfR0VHExcWJb7/9ttL1fP3118Lf31+o1epK/UB/6623RJ06dcTzzz8vwsLCxMSJE0VwcLBITk4WixcvFvXr1y/z18bdjPlmM+Yb7eWXXxYhISFi1apV0pBoTk6OWLVqlQgNDRUjR44s92v/448/RHR0tBg2bJgoKCio9C/Bp59+WsTFxYnly5eLHj16iMTERPHggw+Kf/75Rxw5ckS0b9/e4JC/m5ubdpTlttzcXNG6dWvRqVMncerUqXJ/OHXq1Em89957Qggh4uPjy4xUff/99yI4OFhvWX5mVf/MJk+eLGrVqiXmzp0rDh48KLKyskRWVpY4ePCgmDt3rvDx8RFTp06t8Os/f/686NSpk+jatau4cOFCpT63ESNGiIiICDFjxgzRqlUrMWTIEBEVFSXWr18vUlJSRJMmTcSzzz5rsLwxn5s9fmZNmjQR33//fZn3b/88Cw4OLvf/2d9//y18fX3F4MGDxfTp00XNmjXF008/LWbOnCkGDx4sXFxcxJIlSwyW9/T0FPv27TN4fu/evcLT09PwF0+KsKmOw203btwQmZmZIjMzUzvtUFXnzp0Ta9asEfn5+RVeW1paKmbOnCkeffRR8e677wqNRiO+/vprUb9+feHr6yueeeaZcusx5pvNmG+0oqIiMXz4cOHs7CzUarVwdXUVrq6uQq1WC2dnZ/HSSy+JoqKiCr/+wsJC8eKLL4qIiAjh4OBQqV+CWVlZ4uGHHxY1a9YUiYmJIjs7W4wcOVI7zx8RESFOnDiht2xkZKT0F9BteXl5onXr1iImJqbcH067du0SXl5eYurUqWLhwoWidu3a4q233hIrVqwQU6ZMEd7e3mL27Nl6yxr6zFQqFT8zA5+ZEELMmjVLBAYGatt6e61HYGBgueV0aTQa8e6774qAgIBKfW75+fnihRdeEPfff78YNmyYKC4uFu+//75wdnYWKpVKdOjQoczU0d2M+dzs8TN7/fXXDa69KCkpET179iz3/5kQt0Zi+vfvLzw8PLR/ADo5OYn4+HixevXqcss+9dRTolmzZnqnE9PT00Xz5s3FwIEDy62DjGeTHQdbU5lvtvLmn0+cOCEGDBhQrW80IW79tbxlyxbx1Vdfia+++kps2bJF76Ksivz4449izJgx5f4grsjJkyfFoUOHRElJicFrRo0aZfAv69zcXBEXF1fhD6ddu3aJBx98sMzixLp164r58+dX2M6cnByxefNm7We2efPman9mo0ePtovPTAghTp06JXbt2iV27dpV5i/5qti/f7+YP3++dh1QVV2/fl3k5uZWeN3IkSON+tzs7TMrKSkp9/ugpKRE/Pvvv5W6n0ajEVlZWVX6A/Dq1auia9euQqVSCR8fHxEVFSWioqKEj4+PUKvVolu3buLatWuVqouqz2Yex7RlN2/eRGFhITw9PQ2ez8jIqPA5fyEELl26BI1Gg9q1a2sfA7vXXLt2DZmZmWjcuLHe83l5eUhPT0f79u0rrOvy5cs4deoUNBoNAgMDERoaWq02OTs7V3uzMGPKVra8NX5mtkCpz82ePrMLFy4gOTkZO3fuxIULF6BWq9GgQQP07t0bzzzzDBwcHExaHgD++ecf7Nmzp8w+K1FRUYp8jVQ+dhyswLlz5zB16lR8/vnnipe9fv060tLS4OPjg+joaOlcUVERVq5cicGDByte1tjyt38w3P5hcOTIESxYsADFxcV4+umn0alTJ4P3vbt8fHw8IiMjK13emM3CjN1oTImNym4rKCjAypUrceLECQQGBuLJJ5/Ulq+Mu8sHBQVhwIABBsunp6ejVq1a2j0LvvzyS2mvkZEjR2LAgAEG72VMeWPvPWrUKPTr1w8PPfRQpT4XpcoCxm8qZ0z56pbdv38/EhISEB4eDjc3N+zevRtPPfUUbty4gQ0bNiA6OhopKSnw8PDQe19jy5OVsOh4Bwkhbu0NUdEwcnXKHj16VISEhGjnUNu1aycyMjK058tbMa6vbGZmZqXKGlt+/fr1wtnZWfj4+AhXV1exfv16UadOHZGQkCA6deokHBwcyuzJoVR5lUolYmNjRYcOHaRDpVKJli1big4dOoiOHTsqXtbY8o0aNRL//fefEEKIs2fPitDQUOHl5SVatmwpfHx8hJ+fX7nD4MaUb9q0qdi4caMQ4tZiXTc3NzF69GiRnJwsxowZI2rWrCk+++wzg/c2pryx9757/cisWbPEhQsXDF6rZNnp06cLDw8P0bdvXxEQECBmzZolfH19xYwZM8S7774r6tSpI6ZMmWKS8saUbdOmjZg2bZo2/vLLL0VcXJwQ4tY0QmxsrBg9erTBdhtbXgghiouLxbfffivGjBkjBgwYIAYMGCDGjBkjVq5cqX2MlkyLHQcz+PHHH8s95s2bZ/CXqDFle/fuLbp37y4uX74sjh8/Lrp37y7CwsK0z3CX98vbmLLGlm/durV48803hRC3noCpVauWeOONN7TnJ06cKB5++GGD9zamvDGbhRm70Zgx5VUqlXYdxcCBA0V8fLzIzs4WQtxa6JeQkCCefPJJk5R3c3PTzms3a9ZMu7nQbStWrBDR0dEG721MeWPvrVKpxKZNm8Qrr7wiateuLZycnETPnj3FTz/9JEpLSw2WM7assZvKGVPemLJubm7i5MmT2ri0tFQ4OTmJrKwsIYQQv/76qwgKCjLYbmPLHz9+XDRo0EC4urqK9u3bi379+ol+/fqJ9u3bC1dXVxEeHi6OHz9usDwpgx0HMzBmR0Fjyvr5+UkbCGk0GjF8+HARHBwsTp48We4vb2PKGlve09NT+81fWloqHB0dpVXUhw4dEv7+/gbvbWx5YzYLM6asMeXv/sXfoEED8euvv0rnf//9d1G/fn2TlPf19dXuP+Hn56d3P4PyNjMypryx9777675x44b49ttvRWJionBwcBBBQUHijTfeMPiLyJiyxm4qZ0x5Y8qGhISInTt3auPMzEyhUqlEYWGhEOLW5lGGdvBVoryxO26SMuw6rba5BAYGYtWqVdBoNHqP9PR0k5S9fv26NE+pUqmQnJyMHj16oH379jh27JhJyipR/nY+ErVaDVdXV3h5eWnPeXh4ICcnx2TlW7ZsibS0NFy+fBktWrTA4cOHy+RHMUVZY8vfvq6oqAiBgYHSubp16+Ly5csmKd+tWzckJycDANq3b4/vv/9eOr9y5UqDOReMLW/sve/m5OSEfv36ISUlBadOncILL7yAFStWIDIyUvGyAQEB+PvvvwEAx48fR2lpqTYGgL/++ktvPhQlyhtTtnfv3hg+fDhSUlKwdetWDBw4EO3bt4ebmxsA4OjRo6hbt67Bdhtb/vfff8eMGTP0LjT39PTE9OnT8dtvvxksTwqxdM/FHvTo0UNMnjzZ4PkDBw4YfBzTmLItW7YUy5Yt03tuxIgRwtvb2+Bf/caUNbZ806ZNtTt1CiHKPIq4Y8cOg/vwK1H+blXdLEypslUtr1KpRJMmTUSzZs1EzZo1y+wbsn37dlG3bl2TlM/IyBChoaGiXbt2Yty4ccLNzU20bdtWvPDCC6Jdu3bC2dlZ714JSpQ39t53jxroo9Foyoy+KFHW2E3ljClvTNm8vDzRr18/7SZ88fHx0tqXDRs2SLvqKl1eiR03yXjsOJjBjh07pF9kuvLz8w1uc2xM2XfffVd069bNYNmXXnrJYKfDmLLGlk9OThbr1q0zWHbSpEniueeeM3je2PK6qrJZmJJlq1J+2rRp0pGSkiKdf+2118SAAQNMVv7atWtiwoQJIjo6Wri6ugpnZ2cREhIinnrqqXJ3+lOivDFlQ0NDxZUrVypsn9Jljd1Uzpjyxt5biFt7PtydqK+qqlteqR03yTh8HJOIiGzG7NmzsWDBAmRlZWmn14QQCAgIwJgxY/D6669buIX3PnYciIjI5pw+fVraAOr2Xh5keuw4EBHRPcGYzfSo8thxICKie8LBgwfxwAMPoLS01NJNuacZ3s+UiIjIiqxdu7bc86dOnTJTS+wbRxyIiMgmqNVqqFQqlPdrS6VSccTBxLgBFBER2QRjNsQj5bDjQERENqF58+ZIS0szeL6i0QhSBtc4EBGRTRg/fjwKCgoMng8PD8fWrVvN2CL7xDUOREREVGmcqiAiIqJKY8eBiIiIKo0dByIiIqo0dhyIiIio0thxICIiokpjx4GIiIgqjR0HIiIiqrT/D8X4gHFN6lhDAAAAAElFTkSuQmCC\n",
      "text/plain": [
       "<Figure size 640x480 with 2 Axes>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "model=model_all\n",
    "\n",
    "contact_map=np.triu(model.mask[:len(model.sequence),:len(model.sequence)])\n",
    "pair_frustration=model.frustration('mutational')*contact_map\n",
    "sns.heatmap(pair_frustration,cmap='vlag',center=0)"
   ]
  },
  {
   "attachments": {
    "image.png": {
     "image/png": "iVBORw0KGgoAAAANSUhEUgAAAfQAAAH0CAIAAABEtEjdAAAgAElEQVR4Ae2dD8gfxZ3/h0I5KIdQDkrZQwqlcBThOFgKRQQpFBGCIIIUIQgiWUQQIZRACGQlEIQgQhBCIAQXEUIIhEAQggS+jzVqrH9rrPH0qtV2bX+25+/Xu+vv7vq7u/3xPJ8nn8wzs5/5zu7Ozs7u834Yktmdmc985jUz72ee2X+qwQ8IgAAIgMDiCKjFtQgNAgEQAAEQaCDuGAQgAAIgsEACEPcFdiqaBAIgAAIQd4wBEAABEFggAYj7AjsVTQIBEAABiDvGAAiAAAgskADEfYGdiiaBAAiAAMQdYwAEQAAEFkgA4r7ATkWTQAAEQADijjEAAiAAAgskAHFfYKeiSSAAAiAAcccYAAEQAIEFEoC4L7BT0SQQAAEQgLhjDIAACIDAAglA3BfYqWgSCIAACEDcMQZAAARAYIEEIO4L7FQ0CQRAAAQg7hgDIAACILBAAhD3BXYqmgQCIAACEHeMARAAARBYIAGI+wI7FU0CARAAAYg7xgAIgAAILJAAxH2BnYomgQAIgADEHWMABEAABBZIAOK+wE5Fk0AABEAA4o4xAAIgAAILJABxX2CnokkgAAIgAHHHGAABEACBBRKAuC+wU9EkEAABEIC4YwyAAAiAwAIJQNwX2KloEgiAAAhA3DEGQAAEQGCBBCDuC+xUNAkEQAAEIO4YAyAAAiCwQAIQ9wV2KpoEAiAAAhB3jAEQAAEQWCABiPsCOxVNAgEQAAGIO8YACIAACCyQAMR9gZ2KJoEACIAAxB1jAARAAAQWSADivsBORZNAAARAAOKOMQACIAACCyQAcV9gp6JJIAACIABxxxgAARAAgQUSgLgvsFPRJBAAARCAuGMMgAAIgMACCUDcF9ipaBIIgAAIQNwxBkAABEBggQQg7gvsVDQJBEAABCDuGAMgAAIgsEACEPcFdiqaBAIgAAIQd4wBEAABEFggAYj7AjsVTQIBEAABiDvGAAiAAAgskADEfYGdiiaBAAiAAMQdYwAEQAAEFkgA4r7ATkWTQAAEQADijjEAAiAAAgskAHFfYKeiSSAAAiAAcccYAAEQAIEFEoC4L7BT0SQQAAEQgLhjDIAACIDAAglA3BfYqWgSCIAACEDcMQZAAARAYIEEIO4L7FQ0CQRAAAQg7hgDIAACILBAAhD3BXYqmgQCIAACEHeMARAAARBYIAGI+wI7FU0CARAAAYg7xgAIgAAILJAAxH2BnYomgQAIgADEHWMABEAABBZIAOK+wE5Fk0AABEAA4o4xAAIgAAILJABxX2CnokkgAAIgAHHHGAABEACBBRJITtw3VquiKNTWj8GbkrIsU0rleV5VlZGhrmsuWxRFXddGBhyCAAiAQCgCuiJlWdYqSqHq6mEnOXHPsqwsy7qulTJ9I03fWK2aptlYrbIsM/Sdy9Z1XZZllmU9iKAICIAACPgQIDWnRWRd1xurVZ7nZVn6lI2QxxTQCFV6VmGLu0GtrmtdvsuyLIpCN14UhaH+eiriIAACIBCWQOuqNGwV/tbmJO52q/RfAHme06Kes9EvUj5EBARAAARGJWCsOEeta63xGYt7VVX6yj3LMmOTPSnQa3sCGUAABGZNANsyvt2nr8rtMiTc+lK9NX/rSdsazoAACIBAPwJ09wf9m+d5PyNjlJrlyr2ua3sTplXHW0+OwRE2QQAEdjOBuq5pL8G4NDghk/mJu71mJ3zBt2X0X8iIgwAIJEugVUB7eNtqp9NJuouvU5HxMs9M3CVlb5rGXssPvKCa8qofvvWbEuC2e7gppb72P1/zD6HGRig7/XpKLzUncXcoe9M0wW+FTKeT9A6jOHyzmficATcfSnaeOXKbRNyxcrcHT8sZezytfQAs7ENMtgMtXk50Cr71Aw9uu4dbBHGn3QJ+iIn23PW7PPrRDlUquZV7604ZtbY1Sb/9MezrByAE/QYZuIFbPwL9SknjTSn19f/+un+Q7Di8Ml4/UBRFOsreNE1y4u5AGTmpR2dH8xC+9UMNbruHWwRx7wczWimIu4gaQiCicSaAmxOPmAhuIhpngsRNKfVX//1X/kGy46w86USIu9g9y+tssalIAIHZEpDmKcQd4i4OamnQiAWQAAIgEJ2ANE8h7hB3cTBKg0YsgAQQAIHoBKR5qpT6xn99wz9IdqI3KFiFEHcRZcqdnfKrjOGbOKScCeDmxCMmSvMU4g5x7zxoxAIREyAE/WCD2/K4OcT9lr/c4h8kO/2IpVAK4i72wvI6W2wqEkBgtgSkeaqU8lf2W/5yi2RntmBwn7vcdcvrbLmtSAGBuRKQ5inEHSt3cUxLg0YsgAQQAIHoBKR5qpT65n9+0z9IdqI3KFiFEHcRZcqdjb1jsducCeDmxCMmpsxNmqcQd4i7OKClQSMWiJiQ8mSDb/0GArj14ybNU6XU3/zn3/gHyU4/r1IoBXEXe2F5nS02FQkgMFsC0jyFuEPcxUEtDRqxABJAAASiE5DmaYLiXhSF/hZbQkXvsh0DG8RdpCoNGrEAEkAABKITkOapUupb//Et/yDZCdigsiyzLNP33+jjHiN9dhXiLvZdhM4W616XoI+PdXljp8O3fsTBrR83aZ4mKO5N09C3P/M8pwU7Rfo1fG0piLuISBo0YoGICRCCfrDBbXncpHmqlPr2v3/bP0h2+hFzl7KX8O78/VIh7iK3mJ0tOoEEEAABJwFpnkLcIe7iwJEGjVgACSAAAtEJSPNUKXXrn2/1D5KdsA2ibRm6sloUBbZlwuL1tRans329QT4QAIE2AtI8TVDcaTdG/84qLqi2den456RBM37N62vA3vF6Rm05wK2NyvpzKXOT5mmC4o5bIdcPtTg5pEETp3Z3LSlPNvjm7jspFdwkMu7z0jxVSn3nz9/xD5Idd+0pp2LPXeyd5XW22FQkgMBsCUjzFOIOcRcHtTRoxAJIAAEQiE5AmqdKqe/+23f9g2QnbIPogmqWZVxdnuf6LnzA6iDuIkymL+ZAAgiAwNQEpHmaoLhXVZVl2cZqVdc1u01yPwZFiLtIlemLOaZLwP5sP/bgtjxu0jxNUNxJ2akLdLf1eL8Oai0FcW/FsnlyJOJifV0SIFJdaN3MC243WXSJpcxNmqdKqe/96/f8g2SnC6c1efUqOK6v4teU75gMcReBMX0xBxJAAASmJiDN0wTFPc9z/jXJbldVlef5GBQh7iJVpi/mQAIIgMDUBKR5qpT6/r983z9IdgK2jx9ZotV6Xde0C2+/BzhIpRB3EWOEzhbrRgIIgIAfAWmeJijuTdPQyyDpbpksy1ofa/Jr9/pcEHeRkTRoxAIRE/iPu4h1+lYF33xJ7cwHbjt5+B5J8zRNcfdtVYh8EHeRojRoxAIREyAE/WCD2/K4SfNUKXXbn27zD5KdfsRSKAVxF3theZ0tNhUJIDBbAtI8TUfclcfPGPgh7iJVadCIBZAAAiAQnYA0T5VSf/9//t4/SHbCNqgoirIs6QpqXddlWRZFEbYKtgZxZxRmJE5nm7XiGARAoAsBaZ4mKO5FUdgbg+PpO8RdHEfSoBELREywh0jEytdUBd/WABKSwU0As+a0NE+VUvlXuX+Q7KypvktylmX2XY91XWdZ1sWMb16Iu0gqQmeLda9LgBCsI9SeDm7tXNadTZmbNE8TFHelVKu4S01Y1y1r0iHuIqCRiIv1pZTw8+eeo/DG6dNvnD79xC+foHD8neN6SMll+BKGwL5P9u37ZN+1o0cpvHXyJIXnX39eD2EqC2FFmqcJirv+hCo3HU+oMop4EWnQxPNgupog7tOxn7jmJYn7D776gX+IMN9pB8a4oNq6VxNkEGDlLmKM0Nli3VMnQNyn7oHJ6oe4j4p+Vz+hurFaFUVBN4balAkNpdpP7rpTbWvuM2mK+zPvPPPMO89UVUURdxO6pm7/Bf7WySHintr+7NmrZzmk5pveQdP6dubqGQqt4l5Vlb4n8/zrz+ueTxuX5qlS6of//EP/INlxtI7Eit4l0Lrl4igbISm5lXuWZfRnSytrTqVbRI2rzO7UrjRbHehqJHh+iHtXpKzsZ6+enVZA3Z5P6xvEvcd839zWv/EdJXop2LSdaAyw5MSd/bNZ2zeE6veNulPZrH/EduCxjx/TA5si+dh5wenm0uaBzx/QA5fqFyFx53/7GZFK8cr9ysWLFGg9d+LtExSefeNZPUh2kjqvi/vZq2eT8m1yZw5dP0SBxf3AhwcOfHhg6+/DzVH22rlzepjcYdsBe55Snggr97IsdX/Gu6lRr8U/Pidx51+S3Dz9C1XuVC7iH7EHja7sj338GJuCuDOKBCMQd0enLFvcb//j7f7Bnu8OblLSWiN0bwzt5OhvJZAMDjk/J3G3LyvrvyrdqT0Y2f0EcdeX7c++8WwPqvGLQNwdzCHurP72fHdwa02il7O3JtFJ/RuqjmyhkuYk7q30+SRHdDStJ/UMjrhSiv44ff/IEQok7quNlR6apom5ct/72d69n+2tqooiDv97JD346wf10DTNjr/JXzvnYzOpbUfDYfhmAGFx58i99b331vdeeeEFPaTMTZrjSqk7v7zTP0h2DGLSIS00N1YrKUPTNPo3VB3ZQiVB3EWSEHeIuzg4RkiYREBZ0zkCce/Rt3Vd29vCtp2Bvz9sg+4zcxJ398aLO9VNoTVV3xHj+IVXLtCyvdr6WW1s/qJ+/fnn6fDThx6iQIdsFoe6coFGOjT4Oip1yrNvPEt/ntIhX1hKp8t4JuoRnmh6RM9AcWMVb2fQi/vHfdbsZM3WKP9aeuSck7jbvxvHvqBqbMtceOUCizvvzJC4v/7886zsFOnRGckW2b515srFnTtSrr9A02zL1qP1dCf35r/s5LH3jumBzy84curNU6fePMXizldTpAtLyaKQlsNKqR99+SP/INlxN9xf2ZumKbd+3AYDps5J3N03O7pTeyCzt2Ug7hD3HgMpzSIQd0P3e4h7J2WnYaC/z33sgTEncacrEvSI0yQPMek3/L527ubVxf0f7deD0WfG2y2MVOMJOiNVOtT/rpfy9DhvDPemaR7/+PHHP36808p9JN96NIeK3P+b+zlUVaUv3tmmvmw/9t4xPk+Ru35/lx6M1FCHcbhxF5O481XTF3/2oh6MRsXxzajU81ASZaXUj//Xj/2DZMfhRtenUu2NID7jqKV3UnLizq3VI9w89wsG3KlsxDNidzbE3WflnpoQsLLf/5v7Ie4Qd0nu7fm+Vih0jeK4/VLftXZGypCcuI/Uzh5m7c5OR9x7NMenCM98ivRbuftUFDOPLu76st1/z11ftt/1+7tiOh+8Lu7iTiv34G4ENGjPUzIeYeUesBVjmIK4i1R50PBFJ7pdbPtF19eO8j3gtCfz8qVLFC6/dPnyS5f5kf0xtmVEp4cl8Mw3xJ0XO3u+2KOHYbVFKq2L+/2/ub+1VncfLUPc6a8u7mJ6eOeeL+7RQyucxE/yPDX8VEoZHec+lOwYZmd0CHEXO4s7G+IOcdd1QRwxaSdA3PVOtOM834N3I+/YOCLBK22aBuIuUuXOXivudBuZtHK/eefd1qaAUd+j//SoHoxU6TC1fW3dzzn65rO619s4RnxsbiTu/ASy/jK7Bz5/wN2isX1z1+5O5XlqZFNK3f27u/2DZMcwO6NDiLvYWUop2lqBuIuM2hJSFgLJtwWLu3GnE8RdknuIe9tsXui5lMV9ocgna1YK4j5S44OI+0i+BTEribJSSr+csDYu2Qni5CRGsHIXsbO482sF6eG9n/7jTynwPgwv7SkiWkTC1AS23wD3/hF+lQpF8q9yCvf99j49kL93/OEOPUzdCK/6+Y3/LO70gv7l/RqTRBniDnEXpwrEXUQz2wSIO8RdWsJLvyRmO9hxQVXuupTFXdo7llsTLyVx30jfE1y5h+UWduUe1rewY1ESZaUUveHS81/JTlhvY1rDyl2kvbaz+Zl1QynIIt8Dw/cRUoTrO/7O8ePvHE/q9QPs25BIykJQVVUncad1rr4nc8cf7hgCx1F2LTd9v+i+395nmHrkV49QOPzB4cMfHGZx5z9W6Kkl/tco7j5c65u7+Kip0jyFuEPcxYEnDRouMKG4sw+IdCLASmf8Ppb23GOK+9qGTCjua32bMIM0T5VSBjH3oWRnwqYNrBriLgLkzr7xDeFD9Als/nYwizutlfhfelvRqCt30Wkk9CLAz20ad3+TuPf766qXI65ChjbxV7PoFdPGyp3+Ljz+znHj15irgnmm8Tw13E9T3PENVaObpjnkQQNxn6YDItYKcY8IO3BVPE8NuwmKO76havTRZIc8aBIU95T3QOfoWwrivpbbhCv3tb5NNkubhuep4YNSyrg1yH0o2THMDjnEN1SH0AtZljtbEnf+u5g3aihC2zJvn+BXh21GQnrWNClPtln7dv7V83qgXnvil0/oIWxXsrW13Pjd6xTh4UfbMluXUTe3Bulj7jxoKcK19Ius9a2f2SCleJ4a1hIUd8lVw/NQh9hzF0lyT/A8MfbceXbFF3fRaSQMI6Ar+/lXz5MxXdmf+OUTw2roX3pCce/v9PgleZ4aVSmljCso7kPJjmF2yCG+oTqEXsiy3NnXDx7UQ8g6YCsxAimLu4GK301Pr5579+mnKez8M2OyX0WGt+Md8jw1qkhQ3PENVaOPJjvkQaMr+/WDBydzCBWPRoDf0e8Wd/6c4s7vaZuf5Qvr5hvPbn+8mj6DR7su7x85AnEnzjxPDewJinvTNPiGqtFN0xzyoElQ3FPeA52jbymIu8QtBXGXfJtmZu6slefpztObF1r3frbXP0h2DLNDDvE+9yH0QpZd29lb347e/IL0z597Tg/kBG3Q87+2Z3wFjCJ2BseZlCfbHH1jcV/t/EosdQFfGR915S5xk8Sd/3r4xVNP/eKppzgbRRyDp0eS5FsPU8GLSPM0QXEP3na3QVxQFflIg4YLTCju7AMiQQikIO5SQ1i1jW2ZmOIu+ZbCeWmeKqX45fU+EclOCm3s5wPEXeS2trMh7iK7uSVA3OfWYzf9leZpmuK+sVoVRZFlmVIqy7KiKDZWq5uNCRqDuIs4edDQn738LxeAuDOKZUemvRXyzVPbL/uivT8edXwnLm8WUWTZfWG3juepkaSUevjTh/2DZMcwO+RwY7XKsqyqqrqum6ap65qfWR1iVioLcZfI3HzyjWWdIlyAp5m+4f7z556jDLzbThEuxRHsuTOKaJF+e8dxxF3yLQVxl3yL1nGOiiRRTlDc8zy3SdLbZhwN7J0EcRfR8aCRxP3GF5m2P80kGhohwR4iI1TS0yR86wduLTfaO/rqBz+gsPPL6o/2q9Sz1FrfPO2MkY3nqWE8QXFXStGaXXe1rmupCXq2HnGIuwiNiSco7qLTSFgugQnFPWWoPE8NJ5VS+/hZAI+IZMcwO+QQ4j6EXsiy3NkQ95BYYasvAYh7Kzmep0ZqguKObRmjjyY7lAbNZA6h4l1J4OPH6MPsj9HTs7wtQ5FdiWRHo6V5qpTid9z7RCQ7OyobdkCXT3FBdRjFEKUjdHZvN1PeA4Vv/bpV4paCuEu+9Wtp2FLSPE1Q3Jumwa2QYXu/pzVp0PQ0F7RYypMNvvXraombIe789pt+tfQrJfnWz1rYUtI8TVPcw7bdbQ0XVEU+0qARCyBhoQT4y6u/u/tuCh8eOKCHkdpNt2OlIO4jNTCIWWmeKqX4U5c+EclOECcnMQJxF7Evr7PFpiLBSQDi7sQzcaI0TyHuEHdxaEqDRiywM4G/UEyRnYmbR63vqLKz4cyEBGjFJ4n7lYsX9RDKT/6cHq3c9T8RPjxwIFQti7EjzVOl1OZb/byDZGc4KHoZZLP1RUDpxZDDa7EtQNxtJttnBnb2qOKe8h7oknyLKe7MLUFxZ9/E2TJdgjRP0xH3qdhA3EXy0qARC+xMgLjv5JHEUVeRgrhTt3XlFrOzpXkKcYe4i+NQGjRigZ0Jo4r7zqpwNBYBEndeSr927hyF57deGMR7MnzNkyIDveHq4rzTZqC3kxeX5qlSar/xTjXnoWRn8gb2dgDiLqIb2NkQd5HsfBIg7un3lTRPExR3h6tjcIa4i1SlnhALdEy4dpTfIr4Z6Vga2eMR4OcbeeVHb2rkHgyycr/9j7dTMN6IEq+d86xJmqdKqe1X+hlv+BMOJTsBqUhVSOcHVg1xFwGORJzrY2mgCJ/3iaS8B7o83+KIe1VVyYp7yn0qzdO5iDu9k8Bn1nfNA3EXiUmDRizQMQHi3hFYgOz9RAri3o9bgA7zMCHN06TEXboDkr7HNNLHmGYm7vRieyJlv2KtruuiKCi1KAr71ckeQ+VmFmnQ3MwxLHbtySf1YBi79c+36sFIxeG0BGhb5v0j23fAr72+onv7zDvP6KFpmoc+feihTx/ibRmK6EUQdxCQ5qlSasdjxMbzAtahZMdRNb8rhjTHnZPuc1+bJ2CGOYl7URR5nvNvuY3VKs/zoigYR5ZlZVnWWz9lWWZZxkk9Iv06278iXdmvPfmkUVBX9lv/fKuRisNpCUDcp+Wv1y7N0zjizpojuaG7Gjk+J3G3X3Wvf8SkLEtd6JumKYpiyJ+TY/cWfRKT/zU6HuJuAEnqkJbeL1x5gUKQlTut3x/69KGkWpq+M9I8VUodvH7QP0h2PAkMLO5ZS6dscxL3LMuMnZa6rnl5ri/qCQEt7Tvh0DOP3Vss6xTRq26axi3uvX9pqUbpwag0yGFv34LU7jYSyrcg4s43S5GsV1VFEXcTJkkNxW0M56V5mqa4046CvQU/CpkxjI5ksyxLXcFJu8uypOrc0t/DJWnQ9DDVWgTi3opl1JOhRAriPmo3dTIuzVOl1GHjI/TOQ8mOpzM+xWl3gfcb6rq29xs8q/PJNqeVO+206L/09H2YVritJ324xL/6YXilr69VE6yb2Cy/FpwiRu1JHfLdyq+e3+H15E52ffUbvWbs5FsnKfDKnSKTN2emDkhzPEFx5wWo7nNVVbqOBeyFYKoR0CfJFP1FQx+pquua7g/llbvOiy20nuRUd2RIWbdln1RWYYr4FPHJw2Z3yOSr533KTpUH4j4V+VnUK83TBMWdXeUIEea95bDA5yTuWZbxrTJEYWO1Yi78W5EB6TvyfNI/ov+JwHEuXm39jHfIKhy8IrJ8/tXzZJlX7sErCgiH9P3V89s+0xK+aZppfeaVu6cbtHKnzCffOkkLdjo8em37EWVPU8R2F2bmmahHeKTpET0DxY139dgZ9OJd44ZetxbnWzx0KRsoU60V0ck5iXsrPj6pb8dT2xK/oOrolaZpvvtv39WDkbn33jHf+Tveyr2Hb796ZPM5oS9/9CM9cJMDrtx7+MZu2JG3T5zQg53BOEPi/uLPXtQD5wnrG5sNEknZN1YAo6VKKUPN3YeSHcOsdOhTnHWcLh/STdvF1o9kdsj5OYm7/uuO2qyv3O1LE/x7sh8gn97qZ9mnlK7s3/237xpFek82iLtBcsihruxvnzix1hTEfS2iHhmkeaqU4q+s+EQkO54udS1eFEW29TP8cUvJwzmJO22yb6xW9BuPDnWZ4wcK6DI079hIjXef79pbbmtIdRBwr9wvvHKBgvHGJ4fBNJP4iVaKpOnk7LyS5mni4h6B85zEnfZY8zynzbK5v34gQu9GroJfd04R/9qTFXf37arvHTtG4cb9L9s3wvD9MPTad17gQ9z9h4R/zmnF3d64l/yJfwPezMTdv8uH53R00nDji7QAcSeVh7jHHN7SPFVKGTebug8lOwHbMnAvoasnEHeRWITOFutel6BvRq3LGy+dxL2qqq4rd3KRV7gU4euOfO3XWCBfPXNGDz7t7Mot7Mqd2iX52dU3yc4Y51P2TZqnCYr7wKuAXXsW4i4SkwaNWCBiQpqTDeLuXrlD3MeYItI8VUpt75rx9pkzItkJ63NRFPRyw7BmW61B3FuxbJ6M09li9TNM6L0tQ22NsHLvCjXmyr2rb8hPBKR5mqC4t27Q08kxehPiLlKVBo1YAAlBCfC2zNPvPk3BWHjRnkx97716COqCaIy+t8fvHPz04Yc/ffjh3963/WnrQ9cPUXjqF0/pQTSHhAEEpHmaoLgPaGWfohB3kZo0aMQCSAhKAOIeFOdijUnzVCml/2ZdG5fszBccxF3su5Q7O80999NvnD79xumqqigikvVL4Ddxb6y2n/On66dXLtLDQBdpOc/Ldn4FID/RShG9NoPbpZcv6YFzvvHssxQe2/npa85AK3e6g/NXjzzyu7vv/t3dd/PK/ezVs3rgUu6I4Zs7c+TUlH2T5inEHeIuThNp0IgFIiakOdkg7qTvurKfvXrWc1yk2afkfMq+SfNUKcUbej4RyY5n9/lkk6qQzvvYdOSBuItwRiIu1jf/BBJ3/ndgg8ZYuRsu6cv2Sy9f4tRJVu5cOyKdCEjzFOIOcRcHkjRoxAK7PoFlnSIDebC47/tkH4XWbRm+C95nW8ZwyVPc93yxhwJvENGjp7xBdGrrm6ruW2uMqnEYioA0T5VS+ofI18YlO6H8lG7Ao9eoBKyFTUHcGYUZidDZZpUzP4a46/o+886cjfvSPE1K3B03QdrvQwyFHuIukpQGjVggYkLKe6CRfdu+9HljJ6Vpmvt/c78e9G4xfKMFOP/LOW98+PoF+qipuHLf/j72C7Ry53/ZTqeI4VunsmNnTtk3aZ4mJe7UQZKrI3UfxF0EG7knRD/aElKebJF9g7i3DZDA5yL3aSfvpXmqlNrxun3jGTnrULLTyRl3ZuMLE0VRKKXyPHeX6p0aRtz5ISvHXx/UDONTSr39jlAwQmdHaMXiq+gk7gYNXrNThFMnWblz7Yh0IiDN0wTFXf+gUFVVeZ7Ty8n5W6GdGr42cxhxX1sNZdC/reFZZMJs0qCZ0CVUHYcA3ztPEa6Uz394YPOrJ3yfO0U4GyIxCUjzVCnFr+f0iUh2ArZFKVXXNRnkl4jx55kCVkSmooq7dL04eB9NKm4AACAASURBVKuCGIzQ2UH8hJHgBFjEIe7B2QY3KM3TNMWdm69/81lqAmfuF4kt7v28nKTUSMSDtCXlPdAF+DaJuC+AW5Cx3dWINE8TFHcW9Lqu2e3lrNy79tyE+Zn+hD5IVUMIJDLu8125vXT58kuXL/O2/u/vuovCzrcSPOau1DO1q2+eZoNkS9k3aZ4qpfj+JZ+IZCcIQDJSbv00TaN/FLuqqqIoAtbCprByZxRmJEJnm1XiODECMcU9sabPxh1pniYo7iTrdGsJb77rV1nDQoe4izylQSMWQMLiCBji/tnevXpYXHNn2SBpniql+E8un4hkZ5ZQtpyGuIt9t7zOFpuKBIEAxF0Ak9BpaZ5C3CHu4jCVBo1YIGJCynugS/ItprgviVvEqSB+MS1BcXc8BjQGsfDiTk9hZVnG4jjeptIYRNgm+89n0olACPr1hcTt+DvHKbxy4cIrFy7w+wvorWRPXntSD/2qXltK8m1twQgZUvZNmqdKKf1VP2vjkp3x8NITTCNdTd287zys6/SGs43VSr/Xx3joNmyN41mL39njtQWW3QRSEHe3h0iVCEjzNH1xpxbxLTRSA3ufDyzu+hvOdOh6vLevkQvO0efIiBZTnSTutC3D/y6mvUtqiDRPlVLG6yXch5KdsVnN5j53HRDH9VX82KQC2mf/A9qEqaQI8Du+Ie5J9UsnZ6R5OhdxH++5/cAr9zzPeXuOodMrcjp1WAqZ2f8UnDF8YMjG+RQOZ+RbUuI+I24pDDP2QZqncxH38d64FVjcydGyLGm1Xtc17cLzHfvcJelHpEGTgucQAv9e2HzF143A3OijTizud/zhDgr0Anfj4pt/XUNysm9DjIxUNmXfpHmqlKJe9vxXsjMS0qZp6romwRwJb2BxJ4+LoqC7ZbIsK4pijso+3t9K440VWG4lwMpOEcqToLi3Oo+TawlIopyguNu3QupbHWtb2jVDeHHv6kGy+aVBk6zDcKyVAMS9FctiTkrzVCl19upZ/yDZmS8oiLvYd8vrbLGpy0p46+SO13eTuH/+k5/ogVbuPPPv+eIeCoeuH9LDssAsszXSPIW4BxZ3++8OPjO7kSUNmhQaMtImXZCmTe6bQ9yrqiKJT1DcJ+fm6P2UfZPmqVKKnkHz/Fey48DSNSlCFbpLgcVdN03772VZjvcIllFd2MPIPdHJ+ZQn2+S+GeJOD52SpldVRVP9k337Ptm3j5fpxsc2O/VFqMyTc3M0JGXfpHmaoLhnWeaAHDxpXHEnd8d7BCs4Dt2gNGj0PIgnSMAh7ryIS1DcEyQ5C5ekeZqguPOn9eKAjSHu4z2CNSojadCMWimMDycAcR/OcEYWpHmqlNp8T5B3kOyERVEUBd0pHtZsq7UY4j7T2wrjdHZrr6R28s4v76Rw784P0KXmZ6s/92+9BuzI+0eOvH+EV+605/7ClRf00FocJxMnIM3TBMWdL0DakTEgxxD38R7BGoMI25QGDWeYMBJ5D7STuEf2bW0v6OLOe+4Jintq3HSwKfsmzVOl1MUrF/2DZEfnMK/4uOI+9iNYo7JOubMjT7YliTvdGfnTf/ypHkYdSJ7GI/epp1eULWXfpHmajrhLHnbqgh6ZA4u7/efGqI9g9Wiwf5GpusTfw2g5PcX99ed3vHcvmntGRZvftH7p8suXLlHQV+78QJOu7D/9x58aFnA4IwLSPE1W3CWHgzMPLO7B/ZvQYLQ+mLCNnlVD3D1BIVt8AtI8VUrpF1TWxiU7w1tkWDYOh9uXLMxP3OlNZEqpLMuMvxbrui6Kgv56GP5Om2h9IPVNOueN7wtLjqW5cj/32rlzr51755nNt4S9+/TTepAagvMzIiDN0zji7qM5hofG4Xiow4i7vRtjnwnShmLrh95EZn/gKcsyus2Ivl818JGBaH3Qg4zxW62HhU5FOol7VVWk8p2qCJjZ2JbRxb2qqmTFPXKfdgKesm/SPFVKvfizF/2DZMcNykdzDMvGodv+kNQw4q57oN/IGfYjgbaa6/Xaj8IOfGQgWh/orfCMR55sEHfPfhmSLXKfdnI1Zd+keRpB3D01x/DQOOzUEZ0yBxb3Vj21EXRykTMXRbGxWvGhEbE/w+3+ZWAUtw+j9YFddWpn+Ol8ftMWRVLzs9UfcvXjxx/XQ2tOnJwjAWmeRhB3T82xtzHsM2OQDyzuWZbZb28P9YQqbbLneU5oDLJ21QPrlQbNGN2QuE2Ie+IdtJvdk+apUoo26Dz/lew42AbXHEddPZICi7tSqlXce4CzG8MXUeutH7qyymv51ipaT9qWW88MKdtqcL4nDXG/dvQohQ8OH9ZDIg2kjdaty6ib++3GYyyJOAk3QhGQ5mkEcW+tuvVkqMZ2shNY3Fvvag/1DVWllLH3p1tuZdp60hPQkLKeVfTOZnDobcezYCdxj+yb3QSHuE/um+0tn4FvjKJTRJqnSqnVxso/SHYczrQWaT3pMDJeUmBxp50QfjMO37ViL+d7NKn1jyBG2Zo65IYZe1+M62qaptr64VZEPmQhCF7vg79+kAJZpqeSTrx9gg5p//ra0aN0yMt2OqRPj1KcPYzPisWdPOGVOx1O1WVr6yViiTtJrZjQSfesZMj0PisjsyH0Rqo+u3U7jnhwzXHU1SMpsLiP+g3VPM+NXxL0GW5qtrEF3zQNLqh2HRCs7BRpmmbHI6evP0/Tw96WoXd4TPJdabuNLO68M/PauXN2NpxZBgFJlJVSW48qe265X5bsOCgF1xxHXT2Swot7Dyc8ixjrBVoS5nlOxe17clpv3fGsa6ZvsvRvXWtOiHsrFpxMmYAkyhHEPbjmhOU8J3Fvmob29FsvqDZN4/NAgT8+adD4W5hdTkncH/2nRykcvH7w4PWDp948RYHerXjm6pkUVu787CneGzO7gTfEYWmeRhD34JozhINddmbiTg/7ZlmmlLL/JvJ5FNhGIJ2RBo2UP+Z5fUc7YL1BxL2qKtqfCeiYjykfcR+Jm497a/PAt7WIWjNI81QpRRt0nv9Kdlor5ZNhNYfNBonMTNyDtNnTSL/O9jQ+MNu0QsCXKHna0At06XxVVayzFBnYWC5OO/5Xz24/SkXPzfJVMp9Kp+XGDWmNwLdWLGtPSvM0jrivdW/CDGHEna47t16h5kvSEzayX9XSoOlnbUml3OJ+8cpFH53tAWS4uPeoFEUSJyDNU6UUv/bZJyLZGd581kBHZHgttoUw4m7bXcCZ8Tp77nAg7nPvwSX5L83TdMR9KtqBxd24+5BewMs3tEzVyH71SoOmn7VFlmKVpyus29slV88GWbnzp43p86dH3j/iXrlT6iI5o1EOAtI8hbgHFnf9Iic9PkrPMZVl6eieNJOkQZOCt4nsz7aKe6g99zHEPRFurUMIvrViWXtSmqdKqZ3fP1/zuQ7JzloHks0QWNz1d8vwbeYDX+A1FbuUOzs1IaBbJWlns6qq7Q/c3djp7NeDtri/dfLkWydPXj90iMIN89u1+dSSGjfdZ/im0/CPS/M0TXEvy5Ju9jP23/3b658zvLhz3fqzuVIHcOYEI3P0eSqMuri/fOnG10tvqG8/r8YQ936eoFTKBKR5qpR65QIPovURyU7AttNDT/xcfdjPXdh+BhZ3FnRuAL2QYMg7Xmyn45yJ0NlxGhKhFoh7BMioopWANE8TFHeWR93nqqqKomht2sCTgcW93Pppmoa+h0fOjef9wMa7i+sd4M6JVCLw3rFjeuiK5dOHHqJAd7LzW8n4DSG/euSRXz3yyP/68Y/10LUW5F8YAWmeJiju7CpHqC9GWvsGFneSdXp8lF/ypV9lndHAMjogKc/T3J8lZa+qiiJdiUUQ9zS5ESj41nXAUH5pniqlXj1/3j9Idvp51VqKL0NmWcYfohjvkmR4cW9t1RxPRujs3ljSFAKIe+8OpbfgDSk+atk0xxs1WZqnCYo763hZlvSOW3p7wTy2ZUYdYZGNS4MmshuzqO6rPP8qz6+e2X6TGO+o/Ob++/VgtOWPt9+uB2Plzu824E+FXHvyST0Y1nC4OwlI81Qppb/zeW1csjMS1aIosq2foih4kyNsXVi5izwjd7boxxwSIO5z6KVl+ijN08TFPUJnhBd3ekiV7uWkBmDPPUJHTlsFiTsv2DlCy3a6P53/ZVf1Zfsfb7/9T7fdRuHzBx74/IEHPtq/nwK/KYwiXBwREHB8dwHiHljc+aPV+q2QxjsJ5jIipRVBCv6ntgeqi3tVVcmKe2rc9LEE33Qa/nFpniqljGWB+1Cy4++JT056bt9+jsmnbNc8gcVdvwqsw9LjXV2cKn/KPqcmBBD34aM0tT7VW5Syb9I8TVDc9bWvjnekeGBx10FzXF/Fj9SMMcyy/2MYX5hNuvH83ae33xj29gm+CLoZ4Q0ZinDbafvl5r8/+cnnW+GpXzz11C+ekkpxcURAwL0tw1f4fSIR5ru+9o3Qd4HFnT6DR34zLPpLJEJjwlbB/oc1u0hrEPdFdussGiXN082V+43bt3wikp2AECJUoXsbWNw3Viv+kCm9RIz+EhnpXh+9JcHjkXsiuP/xDfIrfynS1QF+xySt3PnfrnaQf1cRkOapUurnz9EHH73+lewEhMmvHwho02EqsLjTm2ToFk6lVJZl493F6WhVkKQInd3bzzT3QEnTq6pKVtzT5EbDAL71mw7SPE1Q3PntLP1a2rVUeHHv6kGy+aVBk4LDaQoBxH3I2EizT6lFKfsmzdMExZ3ezlKWZZydjEjinvLgkCakNGik/Djfm8C1o0evHT26dRmVN2Oe6m0NBXcVAWmeKqXeeJY+ou71r2QnIEzjHe76YcBa2NTo4k577nP80l6EzuZu2OURiPsuHwBDmi/N0wTFfUgze5QNJu78hZEsy2idzrLO7z/r4d+ERaRBM6FLS60a4r7Uno3QLmmebor76dP+QbIToQkjVRFG3Pnm/KZp6IaZPM9Z5UdyfWyzKXd2yttcVVW9f4Q/ar0ZEXtKqWYrkLjzv2L+wQmJcxvcvrEMpMxNmqdpivvGamXcbzLe2jeMuBtvj6mqSimV8oDwmQTSoPEpO3aelNlC3Pv1fuJ92q9REUpJ8zRBcaeFb1VVdEG1rmt9WRycVRhx17+LTXdDGmeC+x3BoDRoIlQ99yrcK3f+wjUt22/+O/dmw/8pCEjzVCn15qlT/kGyE7BN+jOebHa8ZzyDiTv7SpEIpIwagx8uoAnBmXgahLh7gkK24QSkeZqguLcuecd7OwvEXRxd0qARCyDhBgGI+w0S+H90AtI8VUoZ7zhyH0p2AjYA4h4Q5iBTETq7t3+h9md/e999eujtDxWkK6L8DdX3jh37bO/ez/bufeeZZyjQIW/LUGRgpZ2Kh+LWqVLPzPDNE5SRTZqnCYr7XLdl9BvyW+NGl6R/KA2aFDwPJQS6sv/2vvsGNg3iPgRgqD4d4oNUNmXfpHmaoLjT5dOZXVCVxsSsz0uDZtaNMpwfQ9yvHT1KX8qWVu60fv9s717DGRyCQA8C0jxVSvGfjD4RyU4PlxxF5ncrpKMx802K09lj8zG+PmNUB3E3gOBwdgSkeZqmuMfEG+aCakyPo9UlDZpoDgSpCOIeBCOMJEtAmqeb4n78uH+Q7CTb8LWOQdxFRCl3tr0HKn23yC3u/BFq4/4W/qYSRURGbQlVVV154QUKtD9z+aXLemgrFOmczS1SxR7VwDcPSC1ZpHkKcYe4twwXOiUNGrFAxARbCCDuPvhtbj6l4uSBb/04S/NUKWW8Z9R9KNnp55Veim4woS8Ctt5sMlLVEHe9F3bERyK+o45wB+mIe9M0ya7cw/GGpVQISPM0HXGfihTEXSQvDRqxwKQJkrgbe46Gj8a2zGMfP0aBt2WuXOSP321GjOLGns/Lly5RuPDKBQofHjigB6M4DkFgOAFpnkLcIe7i6JIGjVhg0gSI+6T4UflkBKR5qpTiW3J9IpKdyRo2uGKIu4gw5c6292e318w3Fs/cqtXGSg98niL8QDZ9SphX7nz903Pl/sWePV/s2UOVV1XFK3eKGJVOeGhzm9AZo2r4ZgDxPJTmaYLi7nDVs7Gdss1V3Ou6zrLMgFXXdVEUdMli+Ie5DeOdsI6d2RYCiLsPc5ubT6k4eeBbP87SPFVKXXvySf8g2XF7RQ8l8SVTd2apCum829ra1LmKO72lwYCSZRl9fLaua/oy1Nr2OzIYxh05U0hKR9xfvnQp2ZV7Cj0FH8ISkOZpHHFnzZHc0BvbmofeSaBnCxWfpbiXZVkUBd1axCD4JJ8pimLIaqi1J9g4Ik3T8FYmbcu8/vzzegAiEIhAQJqnccSdGyi5QRmkOyCVUlmWjfQxpvmJO33NhJExXONrUPTBvyEf5nb3Fte7myMQ993c+4m0XZqnm+J+9Kh/kOx4NtOnuE8ez+p8ss1P3PVfdDqsLMvo41XcbNqX58OuEd1417Jj5x/yF0lA3/hWSL6l8qP9+xPxrbWZ8K0Vy9qTKXOT5mmC4r6Wc9gMMxP3oijKsmQEer/q8dYMfNIz0mrQs+zY2RKZbBD3gB2dSJ+2tihl36R5qpT65RNP+AfJTisQ+6RncboWaO/S2AaHn5mTuG+sVsY2iw5UjzOX1pOc6o4MKeu2vJjUVnH/aP/+xTQQDUmfgDRPExR3ui7I39Wj+z7o8uEYnOck7vbGi96vdurwbRnHL9hq64e7ZFcd/vrBB6m9b5w+TWFXNb9pGrRXX8vHpGFPSV0EeD62vsjFWMXbpvTirXG9iJFBckPPxhqlZ66qaiR9n5O462SNeNM0uKCqD6NR479+8EEKLO4UGbVSGAeBVgK6UOoZlFIfHD7sHyQ7uk1H3Kc45+EIGcyyzGG5d9KcxN1upM5oV90Kqa+bbCxjn3GL+7S+udsO39x8pNSUuekioPufoLjzzdn6XSEDNxj0Jhvx5Yh70zT8QMHiH2Iae7IZnyV76sbLUs9cPXPm6hkWd4oYQ2ps34zqOh3Ct064OHPK3BzibnyK3X0o2WEI7ohPcdbxsizzPK+3foqtH7fxfqmLEvfd8/qBfp3tX2qIuPvXgpwgMJyApKpKKbeaG6mSHbeHxv4wHbqLUGpRFNnWz/AXpUjVzVvcpVYFOd+vs4NUPbkRt7jT+v3M1TOT+wkHQECap3HEPWX+EHexd6RBIxZYUALEfUGdufCmSPN0U9wPHvQPkp354oO4i32XcmePvQc6RNzH9k3sMI8E+OYBqSVLytykeQpxh7i3DGU6JQ0asUDEhGiTje5xPLLz+9nuhkbzze1Gayp8a8Wy9mTK3KR5qpTSvwK2Ni7ZWQvHP4NUhXTe33JrToh7K5bNkyMRF+tLMqGHuCfZDji1WALSPIW4Q9zFQS8NGrHAEhMg7kvs1UW1SZqnSin9fXZr45KdgLBaq8D73AMS9jXV2hO+heeZ7+7f3U3h0I3bxE7u/DbrPJsFr5dMQJqnSYl76x2TdFJ/oClsP2HlLvKUBo1YIGLCSHugQcR9JN+C0IVv/TCmzE2ap0mJO2GXXO3XKWtLQdxFRJF7QvSjLSH4ZHv36affffppW9xpCd/mgnguuG9iTd0T4Ft3ZpslUuYmzVOl1MePP+4fJDv9iKVQCuIu9sLyOltsatMEFHdHLUgCgeAEpHmaoLg7NmeCY9m8JWQMo8uwKQ2aZbTOaAXE3QCCw7kQkObpprg/9ph/kOyMxwHvcx+P7RrL8Tt7jUNbyfx35jvHj79z/Pibp05RuPLCCxRWGys9uG3y0Cdxf/Lak3pwl0UqCKRAQJqn6Ys70Su3fsYgiZW7SFUaNGKBKAkk7lVVJSvuKe/Pwrd+gzRlbtI8nYu486si+3WNoxTEXYQjDRqxQJSEkcR95x2PJ4c0JWUhgG/9ejZlbtI8VUr906OP+gfJTj9inUqNVDXEXeyFkYgb9f3wn3+oByO1aRr6xC/vHYbalqHbYHhbJqC4203AGRAYj4A0T+ci7hurFb7ENN7waLcsDZr23H3P6sr+w3/+oW0G4m4zwRkQYALSPFVKfbJvn3+Q7HBFwSN1XZOyj/SHEVbuYpfF6eyu4s756V1e7x07RoGX9rQG579G6Urppw8/rIemaWjlfvbqWT2ILJAAAqkSkOZpguJu3wqZ5/lIyo5bIV0DVho0rjLd01isKWIbMFbulK2qqmTFfbzBasPpega+dSVG+VPmJs3TBMW9H/zepbByF9FJg0Ys0CsB4t4LW89CKYsUfOvXqdI8VUrpf62ujUt2+nmVQimIu9gLiXQ23b3+2rlzFPhNXsbKnW94d2/LvHr+PAXOTxGRAhJAIG0C0jyFuEPcxZErDRqxwDgJEPdxuMLqQghI83RT3B96yD9IdsJiqqoqz3N78z1sLWQN4i5SjdPZUvWP/OoRClcuXrxy8eLaWxX5W0l0ffU3999Pgd7d+/u77qLAK3eKSLXjPAjMhYA0TxMU96Io8jzfWK3isIW4i5ylQSMWCJrgFnd7fzYdcbd9CwpmkDH41g9fytykeaqU+mzvXv8g2elHrLWUUqqu69akMU5C3EWqETpbrLtpIO4OOL2TUhYp+NavW6V5mqC4Z1kGce/Xy4FLSYMmcDWCORb3C69cuPDKhUsvX9KDUKjl9EuXL790+fIHhw/roSUfToHAPAlI8zRBcS+KIuavcKzcxREtDRqxQNAEiHtQnDC2WALSPFVKff7AA/5BshMWHOl7nPU7xF3suzidLVWvGkXhmXee0YOUXzq//QDrjSdZpWw4DwIzJSDN0zTFvSxL+1YZqQkDewTiLgIcibhY384EQ9xviPO2Vjv+uKO7bL7Kcwrxxd3h284mTnAE3/pBT5mbNE83xf0nP/EPkp1+xFpLlWWJu2VaycQ+GaGzHU2CuDvg9E5KWaTgW79uleZpguKOC6r9ujh8KWnQhK+pzSJvxVw9c+bqmTPGyt0uwRdcjZU7rd/t/DgDAssgIM3TBMUdt0KmMuSkQRPHP4h7HM6oZe4EpHmqlOJH+Xwikp2AfHC3TECYg0xF6GyHfxB3BxwkgQATkOZpguLeNA3uluGOmzIiDZrIPtHt7dtfv77xGeyqqowrRbwt87MXX9RDZG+bpsHecT/m4NaPmzRPN/dA7r3XP0h2+nnVWqr1Phk62Zp/4EncLSMCjNDZYt1aAsRdgzE0CgHtRzBlbtI8TVDc+8HvXQriLqKTBo1YYJwE+lLSz597jgI/lUErdz6vP4D6weHD4/gCqyCQHAFpnkLcIe7iYJUGjVhgnASI+zhcYXUhBKR5qpT64p57/INkZ76YIO5i3yXS2RB3sYeQAAJNI81TiDvEXZwf0qARC4yToH/A+uzVs7QtU1XVG6dPv3H6NG/LUGQcF7pZTXl/Fr5168sbuVPmJs3TTXHfs8c/SHZuMJjf/xB3sc8S6WyIu9hD3RNSFin41r0/N0tI8xTiDnEXR5Q0aMQCYyZsf0H1tXP83bAE1+xjAoBtEGgnIM1TiPucxH1jtSqKIssypVSe5/ZKp67roijovtGiKAa+V1MaNO1DbOSzEPeRAcP8XAlI81Qp9bu77/YPkh0Hl7WK5CgbIWlO4k6aTl8g3Fitsiwz9D3LsrIs662fsiyzLBtCsEdnD6nOXRbi7uaD1F1LQJqnEcR9rSJRpzieXeKkMbpvTuJelqWOoK5rXb7LsiyKQs8w8E0O0qDRq4gWp9eHXT1zZv9H+/d/tL+qKopEc8C/IuM3rn/BCDnhWz/IKXOT5qlSir8L7xOR7DiIuRXJUTBO0pzE3Sai94f9ouSN1SrPc7uU5xnduGeR8bJB3IOwTVmk4Fu/LpbmaQRxtx2WnLFzRjgzY3GvqkpfudvvSjaW9l1pJtVPhrjTsn3/R/u7Ngr5QWBhBKR5qpT68kc/8g+SHX9chiL5Fxwp51zFnYSb9t8JTWvftJ70RDmkrF7FnV/eqQc9qWma2/50GwVjGHI2+rI1iztFOBURENjlBKR5GlncbUVq7Re6Fshb7RxpzTzw5CzFva5rexOmtY9bT3oiG1JWr0JX9ju/vFNPgrgbNHAIAl0JSPM0pri3KpLdELouWNc1+VzXtX2l0C7V+8z8xF36DTnGtgz/XuUIg662fnwO7/zyTspMKk8vxeUN1tv+dBul8sq91TKt3w9/cJhSuXhrZh+vDDcCHhoeBrS8bFPUpwl2aFLYeSbqER7wekTPQPGdf0LfaWfQi7fG9SKcQVIkzsAR1ij9F1JVVcadIJx/YGRm4u7gaK/lE7mg2m/lzlrPsk4R6m8W94HdP0Zx+NaPKrj146YLpW5hc+W+Y0PUmIjmoWRHt2nHHYpkZ+YqOEJ59GuHdqneZ+Yk7m6O9h84idwKaQwio6ukPXe3uBtGcAgCu5aAIZTMQSn1hzvu8A+SHTZoR9yKZOdnRcqyjK8XkhE78/AzcxL31qdSdQRpPsR08PpBPegON01z4MMDFPi7evR1DhZ3ihilcAgCIEAEJFGOIO5rFcnoI9bxsizzPKfHLYutHyNnkMM5ibu+4cVx/R0Dab5+QFf2g9cPGt0GcTeA4BAEOhGYUNxZhfSIrkiOhtCbVLIsG/6iFKmWOYm71IaRzkuDpmt1Y4g79me79gLlB7flcZPmqVLqj7ff7h8kO/2IpVAK4i72QqjOlsT91JunTr15Slq50+bMpZcvtfoHkWrFsvYkuK1F1JohZW7SPIW4Q9xbB/PmSWnQiAWEhDHEXagKp0Fg1xGQ5qlS6qsf/MA/SHYCAm2965GvsgasiExB3EWkoTr78Y8fp/Dq+fOvnj9PX1B64/RpWrnf+udbKXznz9/Rg+gWEkAABDQC0jxNUNz1m2S4BQPv2GY7dgTibjPZPiMNGrGAkABxF8DgNAgEICDN0wTF3eFqABCWCYi7heTGCaknbqT7/j+GuKe8BwrffEfGznzgtpOH75E0TzfFPc/9g2TH1w+PfPyEqp6X74/UTwaJQ9xFjF07+92nn6Zw7ejRcZkq9gAAENZJREFUa0ePPnfjO3iSuFO2Q9cP6UH0ZmcChGAnD98jcPMltTNfytykeZqguBdFYbwCvmka++nLnez7H0HcRXbSoJEKxBR3yQecB4HdRkCap0qp//0P/+AfJDsBedIinb4W1zQNvTisdTkfpFKIu4ixa2dL4s6fcaRnUPmDeVfPntWD6AcSQAAEZALSPE1Q3EnQ+UPQoz7BtHm/nwxtt6dIg8bmQtsxEHebDM6AwNgEpHmqlLrxrQR+gZMrItkZ2//x7EPcRbb+nR1f3FPeA4Vv4pByJoCbE4+YKM1TiDvEvfOgsQuQuF945YIe+NFTfgL6vt/ed99v79Mvnx66fsi25nMGQuBDyc4DbjYTnzMpc5uXuG+sVsa2DL8e0qcjOuWBuIu4pEFjF4gv7rYPOAMCu5OANE+VUv/y/e/7B8lOQKobq1WWZVVV0cvF6rqmz66OpO8Qd7Hv/Dsb4i5CRAIIjExAmqcJinvrK4KrqsrzfAxIEHeRqjRo7AKvndu8BebGWwYe/2Tfvk/27ePXDJy+EXv/yBE92HZwBgRAoCsBaZ5uivvf/Z1/kOx09ceRXyllvxCYP6nqKNgvCeIucvPv7PjinvIeKHwTh5QzAdyceMREaZ5C3CHunQeNXaBV3K+88AKFJ375hB7s4j3OQAh6QKMPPfcrGKEU+rQf5BmJO7Zl+nVx+FLSoLFrii/utg84AwK7k4A0T5VS//q97/kHyU5AqnT5FBdUAyLtacq/syHuPRGjGAgMJiDN0wTFvWka3Ao5uMNDGJAGDds++dZJCrT98ulDD1H4Ys+eL/bsefFnL+qBSyECAiAQkIA0T5VSOz+RoH8uoSUu2QnoamRT2HMXga/t7AnFHfuzYrc5E8DNiUdMTJmbNE8h7hB3cUBLg4YLsLhvvuH32tGLVy5S2PE+sKtnOX/ASMqTDb7162hw68dNmqczEnepCf2AcCmIO6MwI2uJTyjupq84BoHdSkCap5vivv0JS/6WpSsi2YnAdaSqIe5i360lDnEX2SEBBGIRkOYpxB3iLo5BadBcP3iQwpun6BvXp16+dOnlS5euPfmkHkS7SAABEAhHQJqnSqn/+7d/6x8kO+E8FS2NVDXEvTPxFMQd+7NitzkTwM2JR0xMmZukjBB3iLs4oKVBA3EXkW0lpCwE8M3dd1JqytykeQpxh7hL47mxB83bJ068feIEi/sX99xDgT51zf+KFpEAAiAQmoA9T6kGpdS/f/vb/kGyM9xf5fEzvBbbAsTdZrJ9xu5siLsICwkgMBEBe56SI+mI+0Rg8A1VGbxSir6t9PPnnqNA4s4rdHoSlf+VLSEFBEBgLAIOcf+Pb33LP0h2xvJ7fLtYuYuMUxb3lPdA4Zs4pJwJ4ObEIyZKoqyU8lf2//jWtyQ7YsXJJ0DcxS6CuItonAkQKSceMRHcRDTOBEmUlVL/+c1v+gfJjrPypBMh7mL32OK++b2l187prwN78WcviuWRAAIgMD4BSZQh7hB3cfRB3EU0SACBZAhA3KWugLhLZDZvhfzwwIEPDxzgd/nS9VX+VyyJBBAAgVgEHOL+l1tu8Q+SnVjtCF8PxF1kmrK4Y39W7DZnArg58YiJKXOTRFkp5a/sf7nlFsmOCCX5BIi72EUQdxGNMyFlIYBvzq4TE1PmJomyUur//fVf+wfJjggl+QSIu9hFSqlj7x079t6xly5f1oNYAAkgAALRCUiiDHGHuIuDEeIuokECCCRDAOIudQXEXSKzeUEVK3eRDhJAIA0CDnH/r298wz9IdtJoZR8vlibudV0XRUEv6imKoq7rPlS2yiilHvv4MQ697YxRMOU9UPjWr8fBrR83SZSVUv7K/l/f+IZkx+HVxmpVFEWWZUqpLMvyPE+qE5cm7lmWlWVZb/2UZZllmaNv3Ekpi3uPgehubMBU+NYPJriF5aaU+u+vf90/9OBPak4ryLquN1arPM/LsuzXkOClFiXuZVkWRaEzKoqi9+/SHp2tVz1qHL71wwtuu4dbBHG3YdZ1nc4YW5S453m+sVrpxOl3qX7GP55OJ9k+wzebic8ZcPOhZOeZI7epxH3IboFNfsiZRYl7lmXGJntd171Zz3FADxkKocqCWz+S4BaWm1Lqf772Nf8wnD+2Zfr1oFep1u5pPeljrndBH+MD88C3fgDBbfdwiybu+neW8jzvR3iMUotaubdO3daTPih7F/QxPjAPfOsHENx2D7fNvu4SBo6Nuq6rqqIbOvpBDl5qUeIefFtG/52MOAiAQJoEWmWxh6utdvSTuk39PMc3Vqve+8BsJFRkUeIe9oJqKMSwAwIgsHsIDPwLICCoRYl72FshA1KGKRAAgd1AACv3EXs54ENMI3oJ0yAAAvMnQFsF/BAT7bkbd2NP2MpFrdybpgn4+oEJewVVgwAIpE/AeP1AURTpKHvTNEsT9/QHBDwEARAAgQgEIO4RIKMKEAABEIhNAOIemzjqAwEQAIEIBCDuESCjChAAARCITQDiHps46gMBEACBCAQg7hEgowoQAAEQiE0A4h6bOOoDARAAgQgEIO4RIKMKEAABEIhNAOIemzjqAwEQAIEIBCDuESCjChAAARCITQDiHps46gMBEACBCAQg7hEgowoQAAEQiE0A4h6bOOoDARAAgQgEIO4tkBN5taT+zrk8z6uqMnxNxE/6CrnxjYLJfaP3ryqlsiwz0E3rW1VVeZ7TN33sbo3sG40xcsYYXWvfsTq2qw7f5jI1bKQxz0DcW2gn8lJ4pRR/W4o+AmCIVCJ+kkIZ4j6tb8XWD71om75Jr3fzhL4VRcF92jQN+VYUBbsX2Teuzug+8odT67ouy9L4gJw7lVvUO8L2bd/mMjV6tz1IQYi7iTGdzzmVZak7RwtkPpOIn+yGPgP5JHtbFIXxm4mTgkdsNdermNY3pRT9ymGX6rpmdBP6xj6wY25n3KlsJEik1TfdcppTQ/dwkjjE3cSuL6wozS0WZvkxj/VRnoKf+kfF0vHN/c2Eabm5v+E+oW9699EQdjvjTg07CWzfbPt6npi+2Z6kcwbibvaFe/qZuSMe0yYyV5iCn1mW8adn9Nk1rW+0ya7va7OTTdNM61tZlrr00LqB/0Sb0De9+2iMuZ1xp/IoDRKxfTPMJjg1DA8nOYS4m9hbR1LrSbPkmMf0h6cuUq0utZ4cya+iKFiVNr/ppW6OJT3Otbee5NSAEb6IWm/9GF+2bHWj9WRAl3RTRVHQBUz6V99wb3Wj9aRuMEjcrsU+o/eyOzWIS2yktS5OTXBqsG/TRm5OyGn9SKf21pHUejKaz3Vd68s9qrfVpdaTY/hpb1XpVetxrr31JKcGjCiljP19ukGFqmh1o/VkQJfYFF2WrKpK/8XDvyNb3Wg9yQZDRexa7DMJinuCUyNUjwy3A3E3Gcb8e9Osu+3YXphQrmn9tGvXtcBONS55tTU02LnW2tm91lTjPpBgrliG9I0sStSvW0zoG/Nhl93OuFPZSJCI7RuZTXNqBGlyECMQdxOjvUa2V6lmmdGOpeHbNM20fuobC0Z8ct/yPHfckTI5N3uwsHhN6Bv7wO65nXGnspEgEds3ugff/k1J1cX0LUgDRzICcTfBxrzHy6x757FD2ZumScdP8lqfgdP6Vm396Cz1bZlpfbP1SF+5T+ib3n2Ezu2MO1WHPzxu+zavqTGcQD8LEPcWbvz0ROuzGy0FxjllP75o1JOIn+SVMQOn9Y3Q6fva+rXoCX3jq7u6b/oVgql8M7qP+tTtjDvVGKtDDm3f5jU1hrR9SFmIewu9sZ+rbqmy7ZSx3UGH+oZDIn6S78YMnNY3qj3LMv1RRmY8rW9JvX6gdYx5ghobo8O31qRkpwbzjByBuEcGjupAAARAIAYBiHsMyqgDBEAABCITgLhHBo7qQAAEQCAGAYh7DMqoAwRAAAQiE4C4RwaO6kAABEAgBgGIewzKqAMEQAAEIhOAuEcGjupAAARAIAYBiHsMyqgDBEAABCITgLhHBo7qQAAEQCAGAYh7DMqoAwRAAAQiE4C4RwaO6kAABEAgBgGIewzKqAMEQAAEIhOAuEcGjupAAARAIAYBiHsMyqgDBEAABCITgLhHBo7qQAAEQCAGAYh7DMqoAwRAAAQiE4C4RwaO6kwCxocX1n5kxyzfdmx8OaQtS+OTp7Wg4+QYNh3VIQkEHAQg7g44SIpBQBfEuq7pm6L6l+d6OKHblIr75JHKSufHsCnVhfMg4CYAcXfzQeroBGxB1L8ZPV71dr3j1QXLIBCfAMQ9PnPUuINAq8jqJzdWqzzPlVJZltkr+rIs6Vup+udS9eJUWV3XbKQoiqa5uS1jZ9ZTm6ZxO6A3ptWUngFxEIhGAOIeDTUqaidgC6K+cq/rmjWdzm+sVmyoKIqyLOnLyLSlk+e5Ic1N07CReuunLMuiKLhejrBZ3QKXJZXPskx3QC+ilzLO4xAE4hOAuMdnjhp3ENC1ta7rqqpYzZumIfnmArSI5kO9LJ+0RdYw0jRNWZZcliOtFoyyhgN6EbteIxWHIBCTAMQ9Jm3U1ULAuFuGdlc4X5ZltDCnM7SO5tQ8z4ui0DNQkqHXhhFay3MejrBZXaaNsoYDehG9lHEehyAQnwDEPT5z1LiDgK6t9srdln4jP22wZFlWFAVvmOh5JM3lPBzR3eKTbgf0IlJFRh4cgkAcAhD3OJxRi0iAZZRz0M4MHRoLZ85jROi3Ai3kbZG1jdR1zfUqpYy1v55qlzWq1g/Zpn4ScRCYhADEfRLsqPQmgVZB5EeZiqKw75C5WXhnjEXZsGnsmxt77nme85Kf7Ok78p0cMOrd6R2OQCAqAYh7VNyozCbQKoh8wwzfrEKLa+N6Jv0OoKS6rsuy9Llbpqoq/W4Z+kOB9J2M6KluB4zmtLbFyINDEIhDAOIehzNqEQlIgsiLd77NXL+TncxtrFZFUdB97lmWlWVJ522bjvvcm6Yhfadb6cmIbsHhgNEqvZSRhEMQiEwA4h4ZOKoDARAAgRgEIO4xKKMOEAABEIhMAOIeGTiqAwEQAIEYBCDuMSijDhAAARCITADiHhk4qgMBEACBGAQg7jEoow4QAAEQiEwA4h4ZOKoDARAAgRgEIO4xKKMOEAABEIhMAOIeGTiqAwEQAIEYBCDuMSijDhAAARCITADiHhk4qgMBEACBGAQg7jEoow4QAAEQiEwA4h4ZOKoDARAAgRgEIO4xKKMOEAABEIhMAOIeGTiqAwEQAIEYBCDuMSijDhAAARCITADiHhk4qgMBEACBGAQg7jEoow4QAAEQiEwA4h4ZOKoDARAAgRgEIO4xKKMOEAABEIhMAOIeGTiqAwEQAIEYBCDuMSijDhAAARCITADiHhk4qgMBEACBGAQg7jEoow4QAAEQiEwA4h4ZOKoDARAAgRgEIO4xKKMOEAABEIhMAOIeGTiqAwEQAIEYBCDuMSijDhAAARCITADiHhk4qgMBEACBGAQg7jEoow4QAAEQiEwA4h4ZOKoDARAAgRgEIO4xKKMOEAABEIhMAOIeGTiqAwEQAIEYBCDuMSijDhAAARCITADiHhk4qgMBEACBGAQg7jEoow4QAAEQiEwA4h4ZOKoDARAAgRgEIO4xKKMOEAABEIhMAOIeGTiqAwEQAIEYBCDuMSijDhAAARCITADiHhk4qgMBEACBGAQg7jEoow4QAAEQiEwA4h4ZOKoDARAAgRgEIO4xKKMOEAABEIhMAOIeGTiqAwEQAIEYBCDuMSijDhAAARCITOD/A0/B6uvCRW6nAAAAAElFTkSuQmCC"
    }
   },
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "![image.png](attachment:image.png)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    "def view_frustration_(pdb_filename,model,contact_map):\n",
    "    \n",
    "    pair_frustration=model.frustration('mutational')*contact_map\n",
    "    residues=np.arange(len(model.sequence))\n",
    "    r1, r2 = np.meshgrid(residues, residues, indexing='ij')\n",
    "    sel_frustration = np.array([r1.ravel(), r2.ravel(), pair_frustration.ravel()]).T\n",
    "    minimally_frustrated = sel_frustration[sel_frustration[:, -1] < -0.78]\n",
    "    frustrated = sel_frustration[sel_frustration[:, -1] > 1]\n",
    "    \n",
    "    \n",
    "    view = py3Dmol.view(js='https://3dmol.org/build/3Dmol.js')\n",
    "    view.addModel(open(pdb_filename,'r').read(),'pdb')\n",
    "\n",
    "    view.setBackgroundColor('white')\n",
    "    view.setStyle({'cartoon':{'color':'white'}})\n",
    "    \n",
    "\n",
    "\n",
    "    for i,j,f in frustrated:\n",
    "        view.addLine({'start':{'chain':'A','resi':[str(i+1)]},'end':{'chain':'A','resi':[str(j+1)]},\n",
    "                      'color':'red', 'dashed':False,'linewidth':3})\n",
    "    \n",
    "    for i,j,f in minimally_frustrated:\n",
    "        view.addLine({'start':{'chain':'A','resi':[str(i+1)]},'end':{'chain':'A','resi':[str(j+1)]},\n",
    "                      'color':'green', 'dashed':False,'linewidth':3})\n",
    "\n",
    "  \n",
    "    view.zoomTo(viewer=(0,0))\n",
    "\n",
    "    return view"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_16742557823931737\"  style=\"position: relative; width: 640px; height: 480px\">\n        <p id=\"3dmolwarning_16742557823931737\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    //this is to ignore the existence of requirejs amd\n    var savedexports, savedmodule;\n    if (typeof exports !== 'undefined') savedexports = exports;\n    else exports = {}\n    if (typeof module !== 'undefined') savedmodule = module;\n    else module = {}\n\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n        exports = savedexports;\n        module = savedmodule;\n        resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://3dmol.org/build/3Dmol.js');\n}\n\nvar viewer_16742557823931737 = null;\nvar warn = document.getElementById(\"3dmolwarning_16742557823931737\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_16742557823931737 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_16742557823931737\"),{backgroundColor:\"white\"});\nviewer_16742557823931737.zoomTo();\n\tviewer_16742557823931737.addModel(\"HEADER    ELECTRON TRANSPORT                      01-AUG-76   1CYC              \\nTITLE     THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C AT 2.3     \\nTITLE    2 ANGSTROMS RESOLUTION. II. STRUCTURE AND FUNCTION                     \\nCAVEAT     1CYC    THR A 40 HAS WRONG CHIRALITY AT ATOM CB THR A 78 HAS WRONG   \\nCAVEAT   2 1CYC    CHIRALITY AT ATOM CB THR B 40 HAS WRONG CHIRALITY AT ATOM    \\nCAVEAT   3 1CYC    CB THR B 78 HAS WRONG CHIRALITY AT ATOM CB                   \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: FERROCYTOCHROME C;                                         \\nCOMPND   3 CHAIN: A, B;                                                         \\nCOMPND   4 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 ORGANISM_SCIENTIFIC: KATSUWONUS PELAMIS;                             \\nSOURCE   3 ORGANISM_COMMON: SKIPJACK TUNA;                                      \\nSOURCE   4 ORGANISM_TAXID: 8226                                                 \\nKEYWDS    ELECTRON TRANSPORT                                                    \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    N.TANAKA,T.YAMANE,T.TSUKIHARA,T.ASHIDA,M.KAKUDO                       \\nREVDAT  17   03-MAR-21 1CYC    1       CAVEAT COMPND REMARK HET                 \\nREVDAT  17 2                   1       HETNAM HETSYN FORMUL LINK                \\nREVDAT  17 3                   1       ATOM                                     \\nREVDAT  16   29-NOV-17 1CYC    1       HELIX                                    \\nREVDAT  15   24-FEB-09 1CYC    1       VERSN                                    \\nREVDAT  14   01-APR-03 1CYC    1       JRNL                                     \\nREVDAT  13   27-JAN-84 1CYC    1       REMARK                                   \\nREVDAT  12   30-SEP-83 1CYC    1       REVDAT                                   \\nREVDAT  11   19-APR-83 1CYC    1       COMPND SOURCE REMARK                     \\nREVDAT  10   15-JAN-82 1CYC    1       REMARK                                   \\nREVDAT   9   06-JAN-82 1CYC    3       SCALE                                    \\nREVDAT   8   31-DEC-80 1CYC    1       REMARK                                   \\nREVDAT   7   07-APR-80 1CYC    3       SOURCE SEQRES ATOM                       \\nREVDAT   6   23-APR-79 1CYC    3       ATOM                                     \\nREVDAT   5   01-NOV-77 1CYC    1       AUTHOR REMARK FORMUL                     \\nREVDAT   4   09-SEP-77 1CYC    3       ATOM                                     \\nREVDAT   3   13-JUN-77 1CYC    1       HET                                      \\nREVDAT   2   25-OCT-76 1CYC    2       CONECT                                   \\nREVDAT   1   06-OCT-76 1CYC    0                                                \\nJRNL        AUTH   N.TANAKA,T.YAMANE,T.TSUKIHARA,T.ASHIDA,M.KAKUDO              \\nJRNL        TITL   THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C   \\nJRNL        TITL 2 AT 2.3 A RESOLUTION. II. STRUCTURE AND FUNCTION.             \\nJRNL        REF    J.BIOCHEM.(TOKYO)             V.  77   147 1975              \\nJRNL        REFN                   ISSN 0021-924X                               \\nJRNL        PMID   166072                                                       \\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   T.TSUKIHARA,T.YAMANE,N.TANAKA,T.ASHIDA,M.KAKUDO              \\nREMARK   1  TITL   OXIDATION OF A FERROCYTOCHROME C IN THE CRYSTALLINE          \\nREMARK   1  TITL 2 STATE-STRUCTURAL CHANGE AND ANION BINDING                    \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  73  1163 1973              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 2                                                          \\nREMARK   1  AUTH   T.ASHIDA,N.TANAKA,T.YAMANE,T.TSUKIHARA,M.KAKUDO              \\nREMARK   1  TITL   THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C   \\nREMARK   1  TITL 2 AT 2.3 ANGSTROMS RESOLUTION                                  \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  73   463 1973              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 3                                                          \\nREMARK   1  AUTH   T.ASHIDA,T.UEKI,T.TSUKIHARA,A.SUGIHARA,T.TAKANO,M.KAKUDO     \\nREMARK   1  TITL   THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C   \\nREMARK   1  TITL 2 AT 4 ANGSTROMS RESOLUTION                                    \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  70   913 1971              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 4                                                          \\nREMARK   1  AUTH   T.NAKAYAMA,K.TITANI,K.NARITA                                 \\nREMARK   1  TITL   THE AMINO ACID SEQUENCE OF CYTOCHROME C FROM BONITO          \\nREMARK   1  TITL 2 (KATSUWONUS PELAMIS, LINNAEUS)                               \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  70   311 1971              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 5                                                          \\nREMARK   1  EDIT   R.J.FELDMANN                                                 \\nREMARK   1  REF    ATLAS OF MACROMOLECULAR                120 1976              \\nREMARK   1  REF  2 STRUCTURE ON MICROFICHE                                      \\nREMARK   1  PUBL   TRACOR JITCO INC.,ROCKVILLE,MD.                              \\nREMARK   1  REFN                                                                \\nREMARK   1 REFERENCE 6                                                          \\nREMARK   1  EDIT   M.O.DAYHOFF                                                  \\nREMARK   1  REF    ATLAS OF PROTEIN SEQUENCE     V.   5    39 1976              \\nREMARK   1  REF  2 AND STRUCTURE,SUPPLEMENT 2                                   \\nREMARK   1  PUBL   NATIONAL BIOMEDICAL RESEARCH FOUNDATION, SILVER SPRING,MD.   \\nREMARK   1  REFN                   ISSN 0-912466-05-7                           \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    2.30 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : NULL                                                 \\nREMARK   3   AUTHORS     : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 2.30                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : NULL                           \\nREMARK   3   DATA CUTOFF            (SIGMA(F)) : NULL                           \\nREMARK   3   DATA CUTOFF HIGH         (ABS(F)) : NULL                           \\nREMARK   3   DATA CUTOFF LOW          (ABS(F)) : NULL                           \\nREMARK   3   COMPLETENESS (WORKING+TEST)   (%) : NULL                           \\nREMARK   3   NUMBER OF REFLECTIONS             : NULL                           \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   CROSS-VALIDATION METHOD          : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SELECTION  : NULL                            \\nREMARK   3   R VALUE            (WORKING SET) : NULL                            \\nREMARK   3   FREE R VALUE                     : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : NULL                            \\nREMARK   3   FREE R VALUE TEST SET COUNT      : NULL                            \\nREMARK   3   ESTIMATED ERROR OF FREE R VALUE  : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  FIT IN THE HIGHEST RESOLUTION BIN.                                  \\nREMARK   3   TOTAL NUMBER OF BINS USED           : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE HIGH       (A) : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE LOW        (A) : NULL                         \\nREMARK   3   BIN COMPLETENESS (WORKING+TEST) (%) : NULL                         \\nREMARK   3   REFLECTIONS IN BIN    (WORKING SET) : NULL                         \\nREMARK   3   BIN R VALUE           (WORKING SET) : NULL                         \\nREMARK   3   BIN FREE R VALUE                    : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET SIZE  (%) : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET COUNT     : NULL                         \\nREMARK   3   ESTIMATED ERROR OF BIN FREE R VALUE : NULL                         \\nREMARK   3                                                                      \\nREMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \\nREMARK   3   PROTEIN ATOMS            : 1590                                    \\nREMARK   3   NUCLEIC ACID ATOMS       : 0                                       \\nREMARK   3   HETEROGEN ATOMS          : 86                                      \\nREMARK   3   SOLVENT ATOMS            : 1                                       \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : NULL                           \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : NULL                           \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : NULL                                                 \\nREMARK   3    B22 (A**2) : NULL                                                 \\nREMARK   3    B33 (A**2) : NULL                                                 \\nREMARK   3    B12 (A**2) : NULL                                                 \\nREMARK   3    B13 (A**2) : NULL                                                 \\nREMARK   3    B23 (A**2) : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  ESTIMATED COORDINATE ERROR.                                         \\nREMARK   3   ESD FROM LUZZATI PLOT        (A) : NULL                            \\nREMARK   3   ESD FROM SIGMAA              (A) : NULL                            \\nREMARK   3   LOW RESOLUTION CUTOFF        (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  CROSS-VALIDATED ESTIMATED COORDINATE ERROR.                         \\nREMARK   3   ESD FROM C-V LUZZATI PLOT    (A) : NULL                            \\nREMARK   3   ESD FROM C-V SIGMAA          (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  RMS DEVIATIONS FROM IDEAL VALUES.                                   \\nREMARK   3   BOND LENGTHS                 (A) : NULL                            \\nREMARK   3   BOND ANGLES            (DEGREES) : NULL                            \\nREMARK   3   DIHEDRAL ANGLES        (DEGREES) : NULL                            \\nREMARK   3   IMPROPER ANGLES        (DEGREES) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL MODEL : NULL                                      \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL FACTOR RESTRAINTS.    RMS    SIGMA                \\nREMARK   3   MAIN-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   MAIN-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  NCS MODEL : NULL                                                    \\nREMARK   3                                                                      \\nREMARK   3  NCS RESTRAINTS.                         RMS   SIGMA/WEIGHT          \\nREMARK   3   GROUP  1  POSITIONAL            (A) : NULL  ; NULL                 \\nREMARK   3   GROUP  1  B-FACTOR           (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  PARAMETER FILE  1  : NULL                                           \\nREMARK   3  TOPOLOGY FILE  1   : NULL                                           \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS: NULL                                      \\nREMARK   4                                                                      \\nREMARK   4 1CYC COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY BNL.                                \\nREMARK 100 THE DEPOSITION ID IS D_1000172595.                                   \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : NULL                               \\nREMARK 200  TEMPERATURE           (KELVIN) : NULL                               \\nREMARK 200  PH                             : NULL                               \\nREMARK 200  NUMBER OF CRYSTALS USED        : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : NULL                               \\nREMARK 200  RADIATION SOURCE               : NULL                               \\nREMARK 200  BEAMLINE                       : NULL                               \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : NULL                               \\nREMARK 200  WAVELENGTH OR RANGE        (A) : NULL                               \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : NULL                               \\nREMARK 200  DETECTOR MANUFACTURER          : NULL                               \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : NULL                               \\nREMARK 200  DATA SCALING SOFTWARE          : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : NULL                               \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : NULL                               \\nREMARK 200  RESOLUTION RANGE LOW       (A) : NULL                               \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY                : NULL                               \\nREMARK 200  R MERGE                    (I) : NULL                               \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : NULL                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : NULL                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : NULL                               \\nREMARK 200  R MERGE FOR SHELL          (I) : NULL                               \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: NULL                                           \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: NULL                         \\nREMARK 200 SOFTWARE USED: NULL                                                  \\nREMARK 200 STARTING MODEL: NULL                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: NULL                                                         \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): 39.18                                     \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 2.02                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: NULL                                     \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: P 21 21 21                       \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -X+1/2,-Y,Z+1/2                                         \\nREMARK 290       3555   -X,Y+1/2,-Z+1/2                                         \\nREMARK 290       4555   X+1/2,-Y+1/2,-Z                                         \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -1.000000  0.000000  0.000000       28.84000            \\nREMARK 290   SMTRY2   2  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000       18.91500            \\nREMARK 290   SMTRY1   3 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   3  0.000000  1.000000  0.000000       42.29000            \\nREMARK 290   SMTRY3   3  0.000000  0.000000 -1.000000       18.91500            \\nREMARK 290   SMTRY1   4  1.000000  0.000000  0.000000       28.84000            \\nREMARK 290   SMTRY2   4  0.000000 -1.000000  0.000000       42.29000            \\nREMARK 290   SMTRY3   4  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1, 2                                                    \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: MONOMERIC                         \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 2                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: MONOMERIC                         \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: B                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 470                                                                      \\nREMARK 470 MISSING ATOM                                                         \\nREMARK 470 THE FOLLOWING RESIDUES HAVE MISSING ATOMS (M=MODEL NUMBER;           \\nREMARK 470 RES=RESIDUE NAME; C=CHAIN IDENTIFIER; SSEQ=SEQUENCE NUMBER;          \\nREMARK 470 I=INSERTION CODE):                                                   \\nREMARK 470   M RES CSSEQI  ATOMS                                                \\nREMARK 470     ASN A  22    CB   CG   OD1  ND2                                  \\nREMARK 470     LYS A  39    CE   NZ                                             \\nREMARK 470     ASN B  22    CB   CG   OD1  ND2                                  \\nREMARK 470     LYS B  39    CE   NZ                                             \\nREMARK 620                                                                      \\nREMARK 620 METAL COORDINATION                                                   \\nREMARK 620 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 620 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE):                             \\nREMARK 620                                                                      \\nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \\nREMARK 620                             HEC A 104  FE                            \\nREMARK 620 N RES CSSEQI ATOM                                                    \\nREMARK 620 1 HIS A  18   NE2                                                    \\nREMARK 620 2 HEC A 104   NA   87.0                                              \\nREMARK 620 3 HEC A 104   NB   87.8  90.6                                        \\nREMARK 620 4 HEC A 104   NC   92.0 178.7  90.1                                  \\nREMARK 620 5 HEC A 104   ND   91.1  86.9 177.4  92.3                            \\nREMARK 620 6 MET A  80   SD  170.9 102.1  93.0  79.0  88.6                      \\nREMARK 620 N                    1     2     3     4     5                       \\nREMARK 620                                                                      \\nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \\nREMARK 620                             HEC B 104  FE                            \\nREMARK 620 N RES CSSEQI ATOM                                                    \\nREMARK 620 1 HIS B  18   NE2                                                    \\nREMARK 620 2 HEC B 104   NA   87.0                                              \\nREMARK 620 3 HEC B 104   NB   87.8  90.6                                        \\nREMARK 620 4 HEC B 104   NC   92.0 178.7  90.1                                  \\nREMARK 620 5 HEC B 104   ND   91.1  86.9 177.4  92.3                            \\nREMARK 620 6 MET B  80   SD  170.9 102.1  93.0  79.0  88.6                      \\nREMARK 620 N                    1     2     3     4     5                       \\nREMARK 800                                                                      \\nREMARK 800 SITE                                                                 \\nREMARK 800 SITE_IDENTIFIER: AC1                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE HEC A 104                 \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: AC2                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE HEC B 104                 \\nDBREF  1CYC A    1   103  UNP    P00025   CYC_KATPE        1    103             \\nDBREF  1CYC B    1   103  UNP    P00025   CYC_KATPE        1    103             \\nSEQRES   1 A  103  GLY ASP VAL ALA LYS GLY LYS LYS THR PHE VAL GLN LYS          \\nSEQRES   2 A  103  CYS ALA GLN CYS HIS THR VAL GLU ASN GLY GLY LYS HIS          \\nSEQRES   3 A  103  LYS VAL GLY PRO ASN LEU TRP GLY LEU PHE GLY ARG LYS          \\nSEQRES   4 A  103  THR GLY GLN ALA GLU GLY TYR SER TYR THR ASP ALA ASN          \\nSEQRES   5 A  103  LYS SER LYS GLY ILE VAL TRP ASN GLU ASN THR LEU MET          \\nSEQRES   6 A  103  GLU TYR LEU GLU ASN PRO LYS LYS TYR ILE PRO GLY THR          \\nSEQRES   7 A  103  LYS MET ILE PHE ALA GLY ILE LYS LYS LYS GLY GLU ARG          \\nSEQRES   8 A  103  GLN ASP LEU VAL ALA TYR LEU LYS SER ALA THR SER              \\nSEQRES   1 B  103  GLY ASP VAL ALA LYS GLY LYS LYS THR PHE VAL GLN LYS          \\nSEQRES   2 B  103  CYS ALA GLN CYS HIS THR VAL GLU ASN GLY GLY LYS HIS          \\nSEQRES   3 B  103  LYS VAL GLY PRO ASN LEU TRP GLY LEU PHE GLY ARG LYS          \\nSEQRES   4 B  103  THR GLY GLN ALA GLU GLY TYR SER TYR THR ASP ALA ASN          \\nSEQRES   5 B  103  LYS SER LYS GLY ILE VAL TRP ASN GLU ASN THR LEU MET          \\nSEQRES   6 B  103  GLU TYR LEU GLU ASN PRO LYS LYS TYR ILE PRO GLY THR          \\nSEQRES   7 B  103  LYS MET ILE PHE ALA GLY ILE LYS LYS LYS GLY GLU ARG          \\nSEQRES   8 B  103  GLN ASP LEU VAL ALA TYR LEU LYS SER ALA THR SER              \\nHET    HEC  A 104      43                                                       \\nHET    HEC  B 104      43                                                       \\nHETNAM     HEC HEME C                                                           \\nFORMUL   3  HEC    2(C34 H34 FE N4 O4)                                          \\nFORMUL   5  HOH   *(H2 O)                                                       \\nHELIX    1  H1 GLY A    1  VAL A   11  1                                  11    \\nHELIX    2  H4 CYS A   14  CYS A   17  514 AND 17 BOUND TO HEME GROUP      4    \\nHELIX    3  H5 THR A   49  LYS A   55  5LOOSE FROM 49-53,3/10 53-55        7    \\nHELIX    4  H2 ASN A   60  GLU A   69  1                                  10    \\nHELIX    5  H3 GLU A   90  SER A  103  1                                  14    \\nHELIX    6  H6 GLY B    1  VAL B   11  1                                  11    \\nHELIX    7  H7 CYS B   14  CYS B   17  514 AND 17 BOUND TO HEME GROUP      4    \\nHELIX    8  H8 THR B   49  LYS B   55  5LOOSE FROM 49-53,3/10 53-55        7    \\nHELIX    9  H9 ASN B   60  GLU B   69  1                                  10    \\nHELIX   10 H10 GLU B   90  SER B  103  1                                  14    \\nLINK         SG  CYS A  14                 CAB HEC A 104     1555   1555  1.48  \\nLINK         SG  CYS A  17                 CAC HEC A 104     1555   1555  1.86  \\nLINK         CE2 TYR A  48                 O2D HEC A 104     1555   1555  2.02  \\nLINK         SG  CYS B  14                 CAB HEC B 104     1555   1555  1.48  \\nLINK         SG  CYS B  17                 CAC HEC B 104     1555   1555  1.86  \\nLINK         CE2 TYR B  48                 O2D HEC B 104     1555   1555  2.02  \\nLINK         NE2 HIS A  18                FE   HEC A 104     1555   1555  2.60  \\nLINK         SD  MET A  80                FE   HEC A 104     1555   1555  2.49  \\nLINK         NE2 HIS B  18                FE   HEC B 104     1555   1555  2.60  \\nLINK         SD  MET B  80                FE   HEC B 104     1555   1555  2.49  \\nSITE     1 AC1 17 LYS A  13  CYS A  14  CYS A  17  HIS A  18                    \\nSITE     2 AC1 17 LEU A  32  LEU A  35  ARG A  38  THR A  40                    \\nSITE     3 AC1 17 TYR A  46  TYR A  48  ASN A  52  TRP A  59                    \\nSITE     4 AC1 17 THR A  78  LYS A  79  MET A  80  PHE A  82                    \\nSITE     5 AC1 17 HOH A 105                                                     \\nSITE     1 AC2 16 LYS B  13  CYS B  14  CYS B  17  HIS B  18                    \\nSITE     2 AC2 16 LEU B  32  LEU B  35  ARG B  38  THR B  40                    \\nSITE     3 AC2 16 TYR B  46  TYR B  48  ASN B  52  TRP B  59                    \\nSITE     4 AC2 16 THR B  78  LYS B  79  MET B  80  PHE B  82                    \\nCRYST1   57.680   84.580   37.830  90.00  90.00  90.00 P 21 21 21    8          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.017337  0.000000  0.000000        0.00000                         \\nSCALE2      0.000000  0.011823  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.026434        0.00000                         \\nMTRIX1   1  1.000000  0.000000  0.000000       28.84000    1                    \\nMTRIX2   1  0.000000  1.000000  0.000000        0.00000    1                    \\nMTRIX3   1  0.000000  0.000000  1.000000        0.00000    1                    \\nATOM      1  N   GLY A   1     -21.138  13.774  -7.711  1.00  0.00           N  \\nATOM      2  CA  GLY A   1     -19.878  14.593  -7.963  1.00  0.00           C  \\nATOM      3  C   GLY A   1     -18.996  14.530  -6.703  1.00  0.00           C  \\nATOM      4  O   GLY A   1     -18.114  13.711  -6.640  1.00  0.00           O  \\nATOM      5  N   ASP A   2     -18.996  15.412  -5.821  1.00  0.00           N  \\nATOM      6  CA  ASP A   2     -17.610  15.097  -5.191  1.00  0.00           C  \\nATOM      7  C   ASP A   2     -17.862  14.152  -3.931  1.00  0.00           C  \\nATOM      8  O   ASP A   2     -17.106  13.333  -3.553  1.00  0.00           O  \\nATOM      9  CB  ASP A   2     -17.169  16.546  -4.435  1.00  0.00           C  \\nATOM     10  CG  ASP A   2     -17.988  17.365  -3.427  1.00  0.00           C  \\nATOM     11  OD1 ASP A   2     -19.500  17.365  -3.994  1.00  0.00           O  \\nATOM     12  OD2 ASP A   2     -17.484  18.373  -2.293  1.00  0.00           O  \\nATOM     13  N   VAL A   3     -19.059  14.089  -3.679  1.00  0.00           N  \\nATOM     14  CA  VAL A   3     -19.815  12.955  -2.860  1.00  0.00           C  \\nATOM     15  C   VAL A   3     -19.374  11.506  -3.805  1.00  0.00           C  \\nATOM     16  O   VAL A   3     -18.870  10.624  -3.427  1.00  0.00           O  \\nATOM     17  CB  VAL A   3     -22.587  13.270  -3.175  1.00  0.00           C  \\nATOM     18  CG1 VAL A   3     -21.957  11.947  -3.427  1.00  0.00           C  \\nATOM     19  CG2 VAL A   3     -21.705  13.963  -2.167  1.00  0.00           C  \\nATOM     20  N   ALA A   4     -19.437  11.380  -5.002  1.00  0.00           N  \\nATOM     21  CA  ALA A   4     -18.681  10.120  -5.569  1.00  0.00           C  \\nATOM     22  C   ALA A   4     -17.358   9.805  -4.939  1.00  0.00           C  \\nATOM     23  O   ALA A   4     -17.295   8.923  -4.183  1.00  0.00           O  \\nATOM     24  CB  ALA A   4     -18.429  10.435  -7.144  1.00  0.00           C  \\nATOM     25  N   LYS A   5     -16.476  10.687  -5.191  1.00  0.00           N  \\nATOM     26  CA  LYS A   5     -15.216  10.498  -4.372  1.00  0.00           C  \\nATOM     27  C   LYS A   5     -15.279  10.246  -2.797  1.00  0.00           C  \\nATOM     28  O   LYS A   5     -14.838   9.238  -2.293  1.00  0.00           O  \\nATOM     29  CB  LYS A   5     -14.208  11.695  -4.246  1.00  0.00           C  \\nATOM     30  CG  LYS A   5     -12.948  11.758  -3.553  1.00  0.00           C  \\nATOM     31  CD  LYS A   5     -11.814  13.081  -3.679  1.00  0.00           C  \\nATOM     32  CE  LYS A   5     -10.806  13.144  -2.545  1.00  0.00           C  \\nATOM     33  NZ  LYS A   5      -9.735  14.404  -2.293  1.00  0.00           N  \\nATOM     34  N   GLY A   6     -16.161  10.750  -1.915  1.00  0.00           N  \\nATOM     35  CA  GLY A   6     -16.350  10.435  -0.403  1.00  0.00           C  \\nATOM     36  C   GLY A   6     -16.539   8.923  -0.655  1.00  0.00           C  \\nATOM     37  O   GLY A   6     -15.909   8.041  -0.025  1.00  0.00           O  \\nATOM     38  N   LYS A   7     -17.421   8.419  -1.411  1.00  0.00           N  \\nATOM     39  CA  LYS A   7     -17.799   6.844  -1.033  1.00  0.00           C  \\nATOM     40  C   LYS A   7     -16.476   6.151  -1.474  1.00  0.00           C  \\nATOM     41  O   LYS A   7     -15.972   4.954  -0.970  1.00  0.00           O  \\nATOM     42  CB  LYS A   7     -18.870   6.403  -2.293  1.00  0.00           C  \\nATOM     43  CG  LYS A   7     -18.870   5.017  -2.734  1.00  0.00           C  \\nATOM     44  CD  LYS A   7     -20.193   4.765  -3.427  1.00  0.00           C  \\nATOM     45  CE  LYS A   7     -20.193   3.190  -4.057  1.00  0.00           C  \\nATOM     46  NZ  LYS A   7     -21.516   2.623  -4.309  1.00  0.00           N  \\nATOM     47  N   LYS A   8     -15.846   6.403  -2.545  1.00  0.00           N  \\nATOM     48  CA  LYS A   8     -14.460   5.899  -2.923  1.00  0.00           C  \\nATOM     49  C   LYS A   8     -13.452   5.899  -1.663  1.00  0.00           C  \\nATOM     50  O   LYS A   8     -13.011   5.017  -1.348  1.00  0.00           O  \\nATOM     51  CB  LYS A   8     -13.704   6.844  -3.931  1.00  0.00           C  \\nATOM     52  CG  LYS A   8     -12.318   6.529  -4.183  1.00  0.00           C  \\nATOM     53  CD  LYS A   8     -11.562   7.537  -5.065  1.00  0.00           C  \\nATOM     54  CE  LYS A   8     -10.176   7.222  -5.821  1.00  0.00           C  \\nATOM     55  NZ  LYS A   8      -9.357   8.419  -6.325  1.00  0.00           N  \\nATOM     56  N   THR A   9     -13.452   6.970  -1.159  1.00  0.00           N  \\nATOM     57  CA  THR A   9     -12.633   7.033   0.227  1.00  0.00           C  \\nATOM     58  C   THR A   9     -13.515   6.277   1.298  1.00  0.00           C  \\nATOM     59  O   THR A   9     -12.948   5.773   2.369  1.00  0.00           O  \\nATOM     60  CB  THR A   9     -12.381   8.419   0.920  1.00  0.00           C  \\nATOM     61  OG1 THR A   9     -11.058   8.923   0.542  1.00  0.00           O  \\nATOM     62  CG2 THR A   9     -12.444   8.734   2.558  1.00  0.00           C  \\nATOM     63  N   PHE A  10     -14.775   6.403   1.235  1.00  0.00           N  \\nATOM     64  CA  PHE A  10     -15.531   5.647   2.117  1.00  0.00           C  \\nATOM     65  C   PHE A  10     -15.216   4.135   1.739  1.00  0.00           C  \\nATOM     66  O   PHE A  10     -14.460   3.757   2.495  1.00  0.00           O  \\nATOM     67  CB  PHE A  10     -17.106   5.647   1.991  1.00  0.00           C  \\nATOM     68  CG  PHE A  10     -17.736   4.891   3.251  1.00  0.00           C  \\nATOM     69  CD1 PHE A  10     -18.492   3.757   2.747  1.00  0.00           C  \\nATOM     70  CD2 PHE A  10     -17.862   5.143   4.133  1.00  0.00           C  \\nATOM     71  CE1 PHE A  10     -18.996   3.001   3.818  1.00  0.00           C  \\nATOM     72  CE2 PHE A  10     -18.366   4.513   4.889  1.00  0.00           C  \\nATOM     73  CZ  PHE A  10     -18.996   3.379   5.015  1.00  0.00           C  \\nATOM     74  N   VAL A  11     -15.216   3.379   0.479  1.00  0.00           N  \\nATOM     75  CA  VAL A  11     -14.964   2.056   0.605  1.00  0.00           C  \\nATOM     76  C   VAL A  11     -13.326   1.867   1.109  1.00  0.00           C  \\nATOM     77  O   VAL A  11     -12.948   0.985   1.865  1.00  0.00           O  \\nATOM     78  CB  VAL A  11     -15.153   1.237  -0.529  1.00  0.00           C  \\nATOM     79  CG1 VAL A  11     -16.098   1.930  -1.537  1.00  0.00           C  \\nATOM     80  CG2 VAL A  11     -13.956   1.111  -1.285  1.00  0.00           C  \\nATOM     81  N   GLN A  12     -12.570   2.749   0.794  1.00  0.00           N  \\nATOM     82  CA  GLN A  12     -10.995   2.749   1.235  1.00  0.00           C  \\nATOM     83  C   GLN A  12     -10.680   2.938   2.873  1.00  0.00           C  \\nATOM     84  O   GLN A  12     -10.176   1.930   3.377  1.00  0.00           O  \\nATOM     85  CB  GLN A  12     -10.428   3.820   0.479  1.00  0.00           C  \\nATOM     86  CG  GLN A  12      -9.861   3.253  -0.718  1.00  0.00           C  \\nATOM     87  CD  GLN A  12      -9.168   4.135  -2.293  1.00  0.00           C  \\nATOM     88  OE1 GLN A  12      -7.593   3.820  -2.041  1.00  0.00           O  \\nATOM     89  NE2 GLN A  12      -9.672   5.143  -2.671  1.00  0.00           N  \\nATOM     90  N   LYS A  13     -10.995   3.757   3.629  1.00  0.00           N  \\nATOM     91  CA  LYS A  13     -10.869   3.757   5.015  1.00  0.00           C  \\nATOM     92  C   LYS A  13     -12.066   3.694   5.960  1.00  0.00           C  \\nATOM     93  O   LYS A  13     -12.003   3.127   6.779  1.00  0.00           O  \\nATOM     94  CB  LYS A  13     -10.239   5.017   5.141  1.00  0.00           C  \\nATOM     95  CG  LYS A  13      -9.672   5.521   3.881  1.00  0.00           C  \\nATOM     96  CD  LYS A  13      -8.286   6.277   4.007  1.00  0.00           C  \\nATOM     97  CE  LYS A  13      -8.160   6.907   2.621  1.00  0.00           C  \\nATOM     98  NZ  LYS A  13      -6.837   7.789   2.243  1.00  0.00           N  \\nATOM     99  N   CYS A  14     -13.137   4.072   5.519  1.00  0.00           N  \\nATOM    100  CA  CYS A  14     -14.460   4.072   6.464  1.00  0.00           C  \\nATOM    101  C   CYS A  14     -15.216   2.812   5.897  1.00  0.00           C  \\nATOM    102  O   CYS A  14     -15.342   1.804   6.716  1.00  0.00           O  \\nATOM    103  CB  CYS A  14     -15.468   5.332   5.897  1.00  0.00           C  \\nATOM    104  SG  CYS A  14     -14.712   6.844   6.464  1.00  0.00           S  \\nATOM    105  N   ALA A  15     -15.720   2.623   4.637  1.00  0.00           N  \\nATOM    106  CA  ALA A  15     -16.161   1.237   4.322  1.00  0.00           C  \\nATOM    107  C   ALA A  15     -16.035  -0.086   5.015  1.00  0.00           C  \\nATOM    108  O   ALA A  15     -17.043  -0.590   5.393  1.00  0.00           O  \\nATOM    109  CB  ALA A  15     -15.405   0.985   3.062  1.00  0.00           C  \\nATOM    110  N   GLN A  16     -14.964  -0.653   4.889  1.00  0.00           N  \\nATOM    111  CA  GLN A  16     -14.901  -2.039   5.897  1.00  0.00           C  \\nATOM    112  C   GLN A  16     -15.090  -1.787   7.283  1.00  0.00           C  \\nATOM    113  O   GLN A  16     -14.838  -2.669   8.291  1.00  0.00           O  \\nATOM    114  CB  GLN A  16     -13.578  -2.039   5.519  1.00  0.00           C  \\nATOM    115  CG  GLN A  16     -12.318  -1.724   6.275  1.00  0.00           C  \\nATOM    116  CD  GLN A  16     -11.625  -0.842   5.015  1.00  0.00           C  \\nATOM    117  OE1 GLN A  16     -10.365  -0.149   5.330  1.00  0.00           O  \\nATOM    118  NE2 GLN A  16     -12.444  -0.464   4.385  1.00  0.00           N  \\nATOM    119  N   CYS A  17     -15.216  -0.653   7.787  1.00  0.00           N  \\nATOM    120  CA  CYS A  17     -15.405  -0.905   9.299  1.00  0.00           C  \\nATOM    121  C   CYS A  17     -16.917  -0.401   9.173  1.00  0.00           C  \\nATOM    122  O   CYS A  17     -17.736  -1.283   9.173  1.00  0.00           O  \\nATOM    123  CB  CYS A  17     -14.460   0.355  10.055  1.00  0.00           C  \\nATOM    124  SG  CYS A  17     -13.326  -0.338  11.441  1.00  0.00           S  \\nATOM    125  N   HIS A  18     -17.232   0.733   9.551  1.00  0.00           N  \\nATOM    126  CA  HIS A  18     -18.870   0.985   9.803  1.00  0.00           C  \\nATOM    127  C   HIS A  18     -19.878   0.229   8.795  1.00  0.00           C  \\nATOM    128  O   HIS A  18     -19.626  -0.023   7.472  1.00  0.00           O  \\nATOM    129  CB  HIS A  18     -19.185   2.245   9.173  1.00  0.00           C  \\nATOM    130  CG  HIS A  18     -18.681   3.127  10.181  1.00  0.00           C  \\nATOM    131  ND1 HIS A  18     -18.618   3.253  11.567  1.00  0.00           N  \\nATOM    132  CD2 HIS A  18     -17.736   4.072  10.559  1.00  0.00           C  \\nATOM    133  CE1 HIS A  18     -18.051   3.694  12.827  1.00  0.00           C  \\nATOM    134  NE2 HIS A  18     -17.232   4.387  11.819  1.00  0.00           N  \\nATOM    135  N   THR A  19     -21.012   0.103   9.299  1.00  0.00           N  \\nATOM    136  CA  THR A  19     -22.272  -0.401   8.417  1.00  0.00           C  \\nATOM    137  C   THR A  19     -22.902   0.922   7.661  1.00  0.00           C  \\nATOM    138  O   THR A  19     -22.272   1.867   7.157  1.00  0.00           O  \\nATOM    139  CB  THR A  19     -23.280  -1.283   8.795  1.00  0.00           C  \\nATOM    140  OG1 THR A  19     -23.532  -0.905   9.740  1.00  0.00           O  \\nATOM    141  CG2 THR A  19     -22.776  -2.669   8.543  1.00  0.00           C  \\nATOM    142  N   VAL A  20     -24.036   1.111   7.472  1.00  0.00           N  \\nATOM    143  CA  VAL A  20     -24.414   2.371   6.779  1.00  0.00           C  \\nATOM    144  C   VAL A  20     -26.052   2.434   6.401  1.00  0.00           C  \\nATOM    145  O   VAL A  20     -26.556   3.442   5.771  1.00  0.00           O  \\nATOM    146  CB  VAL A  20     -23.784   2.560   5.519  1.00  0.00           C  \\nATOM    147  CG1 VAL A  20     -24.540   2.119   4.637  1.00  0.00           C  \\nATOM    148  CG2 VAL A  20     -23.658   4.009   5.267  1.00  0.00           C  \\nATOM    149  N   GLU A  21     -26.682   1.363   6.527  1.00  0.00           N  \\nATOM    150  CA  GLU A  21     -28.194   1.363   5.897  1.00  0.00           C  \\nATOM    151  C   GLU A  21     -29.013   1.741   7.283  1.00  0.00           C  \\nATOM    152  O   GLU A  21     -28.635   2.623   7.913  1.00  0.00           O  \\nATOM    153  CB  GLU A  21     -28.572  -0.086   5.645  1.00  0.00           C  \\nATOM    154  CG  GLU A  21     -28.698  -0.779   4.133  1.00  0.00           C  \\nATOM    155  CD  GLU A  21     -27.690  -1.913   4.007  1.00  0.00           C  \\nATOM    156  OE1 GLU A  21     -26.682  -2.039   5.141  1.00  0.00           O  \\nATOM    157  OE2 GLU A  21     -27.753  -2.921   2.873  1.00  0.00           O  \\nATOM    158  N   ASN A  22     -30.210   1.363   7.283  1.00  0.00           N  \\nATOM    159  CA  ASN A  22     -30.966   1.867   8.417  1.00  0.00           C  \\nATOM    160  C   ASN A  22     -30.714   1.237   9.677  1.00  0.00           C  \\nATOM    161  O   ASN A  22     -31.218   0.103  10.055  1.00  0.00           O  \\nATOM    162  N   GLY A  23     -29.958   1.552  10.433  1.00  0.00           N  \\nATOM    163  CA  GLY A  23     -29.517   0.859  11.819  1.00  0.00           C  \\nATOM    164  C   GLY A  23     -28.572  -0.338  11.252  1.00  0.00           C  \\nATOM    165  O   GLY A  23     -28.194  -1.283  12.071  1.00  0.00           O  \\nATOM    166  N   GLY A  24     -28.068  -0.149   9.929  1.00  0.00           N  \\nATOM    167  CA  GLY A  24     -27.060  -1.409   9.047  1.00  0.00           C  \\nATOM    168  C   GLY A  24     -26.871  -2.480  10.118  1.00  0.00           C  \\nATOM    169  O   GLY A  24     -27.186  -3.803   9.929  1.00  0.00           O  \\nATOM    170  N   LYS A  25     -26.304  -2.165  11.441  1.00  0.00           N  \\nATOM    171  CA  LYS A  25     -26.241  -3.047  12.701  1.00  0.00           C  \\nATOM    172  C   LYS A  25     -24.792  -2.795  13.079  1.00  0.00           C  \\nATOM    173  O   LYS A  25     -23.910  -2.417  12.323  1.00  0.00           O  \\nATOM    174  CB  LYS A  25     -26.241  -4.433  11.819  1.00  0.00           C  \\nATOM    175  CG  LYS A  25     -25.296  -4.496  10.433  1.00  0.00           C  \\nATOM    176  CD  LYS A  25     -24.918  -5.819   9.803  1.00  0.00           C  \\nATOM    177  CE  LYS A  25     -23.784  -5.819   8.543  1.00  0.00           C  \\nATOM    178  NZ  LYS A  25     -23.658  -6.827   7.661  1.00  0.00           N  \\nATOM    179  N   HIS A  26     -24.414  -2.921  14.465  1.00  0.00           N  \\nATOM    180  CA  HIS A  26     -23.028  -2.417  14.654  1.00  0.00           C  \\nATOM    181  C   HIS A  26     -21.894  -3.425  14.087  1.00  0.00           C  \\nATOM    182  O   HIS A  26     -22.083  -3.803  12.827  1.00  0.00           O  \\nATOM    183  CB  HIS A  26     -23.028  -2.543  16.103  1.00  0.00           C  \\nATOM    184  CG  HIS A  26     -24.036  -1.094  16.355  1.00  0.00           C  \\nATOM    185  ND1 HIS A  26     -24.540  -0.338  15.473  1.00  0.00           N  \\nATOM    186  CD2 HIS A  26     -24.666  -0.905  17.363  1.00  0.00           C  \\nATOM    187  CE1 HIS A  26     -25.422   0.355  16.103  1.00  0.00           C  \\nATOM    188  NE2 HIS A  26     -25.422   0.103  17.237  1.00  0.00           N  \\nATOM    189  N   LYS A  27     -20.886  -3.551  14.717  1.00  0.00           N  \\nATOM    190  CA  LYS A  27     -19.752  -4.433  14.213  1.00  0.00           C  \\nATOM    191  C   LYS A  27     -18.366  -3.677  14.276  1.00  0.00           C  \\nATOM    192  O   LYS A  27     -17.484  -3.677  13.331  1.00  0.00           O  \\nATOM    193  CB  LYS A  27     -20.004  -4.559  12.575  1.00  0.00           C  \\nATOM    194  CG  LYS A  27     -20.382  -3.425  11.882  1.00  0.00           C  \\nATOM    195  CD  LYS A  27     -19.500  -3.047  11.063  1.00  0.00           C  \\nATOM    196  CE  LYS A  27     -19.752  -3.677   9.551  1.00  0.00           C  \\nATOM    197  NZ  LYS A  27     -18.618  -3.677   8.858  1.00  0.00           N  \\nATOM    198  N   VAL A  28     -18.177  -3.047  15.410  1.00  0.00           N  \\nATOM    199  CA  VAL A  28     -16.854  -2.228  15.410  1.00  0.00           C  \\nATOM    200  C   VAL A  28     -17.358  -0.905  15.473  1.00  0.00           C  \\nATOM    201  O   VAL A  28     -17.610  -0.275  16.481  1.00  0.00           O  \\nATOM    202  CB  VAL A  28     -15.846  -2.291  14.780  1.00  0.00           C  \\nATOM    203  CG1 VAL A  28     -15.594  -1.094  14.087  1.00  0.00           C  \\nATOM    204  CG2 VAL A  28     -14.586  -2.858  15.095  1.00  0.00           C  \\nATOM    205  N   GLY A  29     -17.610  -0.023  14.591  1.00  0.00           N  \\nATOM    206  CA  GLY A  29     -18.240   1.111  14.843  1.00  0.00           C  \\nATOM    207  C   GLY A  29     -19.815   1.048  14.843  1.00  0.00           C  \\nATOM    208  O   GLY A  29     -20.193   0.103  13.961  1.00  0.00           O  \\nATOM    209  N   PRO A  30     -20.634   1.615  15.977  1.00  0.00           N  \\nATOM    210  CA  PRO A  30     -22.020   1.489  15.725  1.00  0.00           C  \\nATOM    211  C   PRO A  30     -22.398   1.804  14.339  1.00  0.00           C  \\nATOM    212  O   PRO A  30     -22.020   2.371  13.457  1.00  0.00           O  \\nATOM    213  CB  PRO A  30     -22.524   2.371  16.481  1.00  0.00           C  \\nATOM    214  CG  PRO A  30     -21.642   1.993  16.859  1.00  0.00           C  \\nATOM    215  CD  PRO A  30     -20.508   2.497  16.607  1.00  0.00           C  \\nATOM    216  N   ASN A  31     -23.910   1.552  13.961  1.00  0.00           N  \\nATOM    217  CA  ASN A  31     -24.414   2.119  12.701  1.00  0.00           C  \\nATOM    218  C   ASN A  31     -24.036   3.631  12.764  1.00  0.00           C  \\nATOM    219  O   ASN A  31     -24.036   4.387  13.835  1.00  0.00           O  \\nATOM    220  CB  ASN A  31     -25.926   1.993  12.701  1.00  0.00           C  \\nATOM    221  CG  ASN A  31     -26.052   2.056  11.063  1.00  0.00           C  \\nATOM    222  OD1 ASN A  31     -27.186   2.623  10.307  1.00  0.00           O  \\nATOM    223  ND2 ASN A  31     -25.359   1.363  10.181  1.00  0.00           N  \\nATOM    224  N   LEU A  32     -23.658   4.261  11.819  1.00  0.00           N  \\nATOM    225  CA  LEU A  32     -23.280   5.899  11.819  1.00  0.00           C  \\nATOM    226  C   LEU A  32     -24.666   6.403  11.315  1.00  0.00           C  \\nATOM    227  O   LEU A  32     -24.666   6.655   9.929  1.00  0.00           O  \\nATOM    228  CB  LEU A  32     -22.146   6.466  11.315  1.00  0.00           C  \\nATOM    229  CG  LEU A  32     -20.886   6.529  12.323  1.00  0.00           C  \\nATOM    230  CD1 LEU A  32     -20.382   7.978  12.323  1.00  0.00           C  \\nATOM    231  CD2 LEU A  32     -21.327   6.277  13.457  1.00  0.00           C  \\nATOM    232  N   TRP A  33     -25.611   6.151  11.693  1.00  0.00           N  \\nATOM    233  CA  TRP A  33     -27.186   6.529  11.189  1.00  0.00           C  \\nATOM    234  C   TRP A  33     -27.312   7.915  10.433  1.00  0.00           C  \\nATOM    235  O   TRP A  33     -27.312   7.852   9.299  1.00  0.00           O  \\nATOM    236  CB  TRP A  33     -28.446   6.403  12.071  1.00  0.00           C  \\nATOM    237  CG  TRP A  33     -28.572   5.773  13.205  1.00  0.00           C  \\nATOM    238  CD1 TRP A  33     -27.564   4.891  13.583  1.00  0.00           C  \\nATOM    239  CD2 TRP A  33     -29.328   5.647  14.213  1.00  0.00           C  \\nATOM    240  NE1 TRP A  33     -27.816   4.387  14.843  1.00  0.00           N  \\nATOM    241  CE2 TRP A  33     -29.013   4.828  15.158  1.00  0.00           C  \\nATOM    242  CE3 TRP A  33     -30.462   6.277  14.213  1.00  0.00           C  \\nATOM    243  CZ2 TRP A  33     -29.706   4.639  15.851  1.00  0.00           C  \\nATOM    244  CZ3 TRP A  33     -31.092   5.962  15.284  1.00  0.00           C  \\nATOM    245  CH2 TRP A  33     -30.714   5.143  15.914  1.00  0.00           C  \\nATOM    246  N   GLY A  34     -27.564   8.797  11.315  1.00  0.00           N  \\nATOM    247  CA  GLY A  34     -27.690  10.120  10.685  1.00  0.00           C  \\nATOM    248  C   GLY A  34     -26.934  11.191  11.315  1.00  0.00           C  \\nATOM    249  O   GLY A  34     -27.312  12.199  12.134  1.00  0.00           O  \\nATOM    250  N   LEU A  35     -25.548  11.191  11.252  1.00  0.00           N  \\nATOM    251  CA  LEU A  35     -24.666  12.199  11.945  1.00  0.00           C  \\nATOM    252  C   LEU A  35     -25.548  13.459  12.386  1.00  0.00           C  \\nATOM    253  O   LEU A  35     -25.800  13.963  13.457  1.00  0.00           O  \\nATOM    254  CB  LEU A  35     -23.154  12.640  11.567  1.00  0.00           C  \\nATOM    255  CG  LEU A  35     -22.335  11.506  11.189  1.00  0.00           C  \\nATOM    256  CD1 LEU A  35     -20.886  11.695  11.189  1.00  0.00           C  \\nATOM    257  CD2 LEU A  35     -22.650  10.183  11.819  1.00  0.00           C  \\nATOM    258  N   PHE A  36     -26.052  13.963  11.441  1.00  0.00           N  \\nATOM    259  CA  PHE A  36     -26.934  15.223  11.693  1.00  0.00           C  \\nATOM    260  C   PHE A  36     -28.131  15.223  12.701  1.00  0.00           C  \\nATOM    261  O   PHE A  36     -29.328  15.223  12.449  1.00  0.00           O  \\nATOM    262  CB  PHE A  36     -27.186  15.601  10.055  1.00  0.00           C  \\nATOM    263  CG  PHE A  36     -25.926  15.601   9.425  1.00  0.00           C  \\nATOM    264  CD1 PHE A  36     -25.674  15.727   8.102  1.00  0.00           C  \\nATOM    265  CD2 PHE A  36     -24.792  15.097   9.803  1.00  0.00           C  \\nATOM    266  CE1 PHE A  36     -24.540  15.475   7.409  1.00  0.00           C  \\nATOM    267  CE2 PHE A  36     -23.658  15.097   9.551  1.00  0.00           C  \\nATOM    268  CZ  PHE A  36     -23.406  15.097   8.291  1.00  0.00           C  \\nATOM    269  N   GLY A  37     -27.879  15.538  13.961  1.00  0.00           N  \\nATOM    270  CA  GLY A  37     -28.509  15.664  15.347  1.00  0.00           C  \\nATOM    271  C   GLY A  37     -28.005  14.971  16.355  1.00  0.00           C  \\nATOM    272  O   GLY A  37     -28.572  14.215  17.300  1.00  0.00           O  \\nATOM    273  N   ARG A  38     -26.682  15.097  16.859  1.00  0.00           N  \\nATOM    274  CA  ARG A  38     -26.556  14.341  18.119  1.00  0.00           C  \\nATOM    275  C   ARG A  38     -25.170  14.845  18.371  1.00  0.00           C  \\nATOM    276  O   ARG A  38     -24.414  15.349  17.741  1.00  0.00           O  \\nATOM    277  CB  ARG A  38     -26.178  13.081  17.741  1.00  0.00           C  \\nATOM    278  CG  ARG A  38     -25.863  12.703  16.355  1.00  0.00           C  \\nATOM    279  CD  ARG A  38     -25.422  11.191  16.040  1.00  0.00           C  \\nATOM    280  NE  ARG A  38     -24.666  10.561  16.985  1.00  0.00           N  \\nATOM    281  CZ  ARG A  38     -24.792   9.238  16.859  1.00  0.00           C  \\nATOM    282  NH1 ARG A  38     -24.351   8.545  17.489  1.00  0.00           N  \\nATOM    283  NH2 ARG A  38     -25.485   8.797  15.851  1.00  0.00           N  \\nATOM    284  N   LYS A  39     -25.044  14.593  19.757  1.00  0.00           N  \\nATOM    285  CA  LYS A  39     -24.036  15.160  20.639  1.00  0.00           C  \\nATOM    286  C   LYS A  39     -23.028  14.089  20.891  1.00  0.00           C  \\nATOM    287  O   LYS A  39     -23.406  13.207  20.009  1.00  0.00           O  \\nATOM    288  CB  LYS A  39     -24.540  15.601  22.151  1.00  0.00           C  \\nATOM    289  CG  LYS A  39     -24.414  15.475  23.663  1.00  0.00           C  \\nATOM    290  CD  LYS A  39     -25.170  16.231  24.923  1.00  0.00           C  \\nATOM    291  N   THR A  40     -22.020  13.837  21.521  1.00  0.00           N  \\nATOM    292  CA  THR A  40     -21.390  12.388  21.269  1.00  0.00           C  \\nATOM    293  C   THR A  40     -21.642  11.191  22.277  1.00  0.00           C  \\nATOM    294  O   THR A  40     -21.516  11.695  23.411  1.00  0.00           O  \\nATOM    295  CB  THR A  40     -19.752  12.577  21.269  1.00  0.00           C  \\nATOM    296  OG1 THR A  40     -19.374  11.884  19.883  1.00  0.00           O  \\nATOM    297  CG2 THR A  40     -19.626  13.837  20.765  1.00  0.00           C  \\nATOM    298  N   GLY A  41     -22.020  10.309  21.773  1.00  0.00           N  \\nATOM    299  CA  GLY A  41     -22.650   9.553  22.907  1.00  0.00           C  \\nATOM    300  C   GLY A  41     -23.910   8.797  22.403  1.00  0.00           C  \\nATOM    301  O   GLY A  41     -24.162   8.545  21.395  1.00  0.00           O  \\nATOM    302  N   GLN A  42     -24.918   8.419  23.033  1.00  0.00           N  \\nATOM    303  CA  GLN A  42     -26.241   7.915  23.033  1.00  0.00           C  \\nATOM    304  C   GLN A  42     -26.304   7.537  21.647  1.00  0.00           C  \\nATOM    305  O   GLN A  42     -26.556   8.545  20.765  1.00  0.00           O  \\nATOM    306  CB  GLN A  42     -27.438   8.545  23.537  1.00  0.00           C  \\nATOM    307  CG  GLN A  42     -28.068   7.789  24.419  1.00  0.00           C  \\nATOM    308  CD  GLN A  42     -29.076   7.096  24.230  1.00  0.00           C  \\nATOM    309  OE1 GLN A  42     -29.706   5.899  24.797  1.00  0.00           O  \\nATOM    310  NE2 GLN A  42     -29.706   7.411  22.781  1.00  0.00           N  \\nATOM    311  N   ALA A  43     -26.304   6.277  21.143  1.00  0.00           N  \\nATOM    312  CA  ALA A  43     -26.682   6.025  19.505  1.00  0.00           C  \\nATOM    313  C   ALA A  43     -27.186   4.639  19.442  1.00  0.00           C  \\nATOM    314  O   ALA A  43     -27.060   4.009  18.497  1.00  0.00           O  \\nATOM    315  CB  ALA A  43     -25.548   6.025  18.497  1.00  0.00           C  \\nATOM    316  N   GLU A  44     -27.438   3.883  20.450  1.00  0.00           N  \\nATOM    317  CA  GLU A  44     -28.131   2.497  20.261  1.00  0.00           C  \\nATOM    318  C   GLU A  44     -27.060   1.615  20.891  1.00  0.00           C  \\nATOM    319  O   GLU A  44     -26.556   0.481  20.324  1.00  0.00           O  \\nATOM    320  CB  GLU A  44     -28.635   2.371  18.749  1.00  0.00           C  \\nATOM    321  CG  GLU A  44     -30.210   0.993  18.749  1.00  0.00           C  \\nATOM    322  CD  GLU A  44     -30.840   2.875  19.379  1.00  0.00           C  \\nATOM    323  OE1 GLU A  44     -32.100   2.875  18.875  1.00  0.00           O  \\nATOM    324  OE2 GLU A  44     -30.084   4.135  19.505  1.00  0.00           O  \\nATOM    325  N   GLY A  45     -26.808   1.615  22.277  1.00  0.00           N  \\nATOM    326  CA  GLY A  45     -25.674   0.859  22.781  1.00  0.00           C  \\nATOM    327  C   GLY A  45     -24.162   1.363  22.907  1.00  0.00           C  \\nATOM    328  O   GLY A  45     -23.532   1.363  21.521  1.00  0.00           O  \\nATOM    329  N   TYR A  46     -23.532   1.993  23.789  1.00  0.00           N  \\nATOM    330  CA  TYR A  46     -22.146   2.875  23.537  1.00  0.00           C  \\nATOM    331  C   TYR A  46     -21.327   1.993  24.356  1.00  0.00           C  \\nATOM    332  O   TYR A  46     -21.327   0.733  24.167  1.00  0.00           O  \\nATOM    333  CB  TYR A  46     -22.524   4.198  23.159  1.00  0.00           C  \\nATOM    334  CG  TYR A  46     -22.146   5.143  22.151  1.00  0.00           C  \\nATOM    335  CD1 TYR A  46     -22.776   5.269  21.521  1.00  0.00           C  \\nATOM    336  CD2 TYR A  46     -20.886   5.647  21.647  1.00  0.00           C  \\nATOM    337  CE1 TYR A  46     -22.524   5.899  20.387  1.00  0.00           C  \\nATOM    338  CE2 TYR A  46     -20.382   6.277  20.513  1.00  0.00           C  \\nATOM    339  CZ  TYR A  46     -21.264   6.277  20.009  1.00  0.00           C  \\nATOM    340  OH  TYR A  46     -21.138   6.655  19.001  1.00  0.00           O  \\nATOM    341  N   SER A  47     -20.634   2.560  25.049  1.00  0.00           N  \\nATOM    342  CA  SER A  47     -19.374   1.741  25.679  1.00  0.00           C  \\nATOM    343  C   SER A  47     -18.114   2.623  25.679  1.00  0.00           C  \\nATOM    344  O   SER A  47     -18.114   3.379  24.797  1.00  0.00           O  \\nATOM    345  CB  SER A  47     -19.248   0.355  25.427  1.00  0.00           C  \\nATOM    346  OG  SER A  47     -18.870   0.481  23.789  1.00  0.00           O  \\nATOM    347  N   TYR A  48     -17.484   2.749  26.939  1.00  0.00           N  \\nATOM    348  CA  TYR A  48     -16.413   3.757  26.939  1.00  0.00           C  \\nATOM    349  C   TYR A  48     -16.602   4.891  27.695  1.00  0.00           C  \\nATOM    350  O   TYR A  48     -17.610   5.269  27.821  1.00  0.00           O  \\nATOM    351  CB  TYR A  48     -16.224   3.757  25.301  1.00  0.00           C  \\nATOM    352  CG  TYR A  48     -15.216   4.639  24.545  1.00  0.00           C  \\nATOM    353  CD1 TYR A  48     -14.082   5.143  24.797  1.00  0.00           C  \\nATOM    354  CD2 TYR A  48     -15.342   4.765  23.411  1.00  0.00           C  \\nATOM    355  CE1 TYR A  48     -13.326   6.025  24.167  1.00  0.00           C  \\nATOM    356  CE2 TYR A  48     -14.712   5.521  22.529  1.00  0.00           C  \\nATOM    357  CZ  TYR A  48     -13.704   6.088  23.033  1.00  0.00           C  \\nATOM    358  OH  TYR A  48     -13.074   6.907  22.151  1.00  0.00           O  \\nATOM    359  N   THR A  49     -15.594   5.647  28.199  1.00  0.00           N  \\nATOM    360  CA  THR A  49     -15.846   6.907  29.207  1.00  0.00           C  \\nATOM    361  C   THR A  49     -15.594   8.041  28.829  1.00  0.00           C  \\nATOM    362  O   THR A  49     -16.350   8.671  27.947  1.00  0.00           O  \\nATOM    363  CB  THR A  49     -15.531   6.403  30.845  1.00  0.00           C  \\nATOM    364  OG1 THR A  49     -14.208   5.899  30.845  1.00  0.00           O  \\nATOM    365  CG2 THR A  49     -16.224   5.395  31.223  1.00  0.00           C  \\nATOM    366  N   ASP A  50     -14.334   8.482  28.703  1.00  0.00           N  \\nATOM    367  CA  ASP A  50     -14.082   9.805  28.199  1.00  0.00           C  \\nATOM    368  C   ASP A  50     -13.704   9.679  26.813  1.00  0.00           C  \\nATOM    369  O   ASP A  50     -13.074   8.545  26.183  1.00  0.00           O  \\nATOM    370  CB  ASP A  50     -13.074  10.435  28.829  1.00  0.00           C  \\nATOM    371  CG  ASP A  50     -12.948  10.183  30.089  1.00  0.00           C  \\nATOM    372  OD1 ASP A  50     -12.066   9.049  30.593  1.00  0.00           O  \\nATOM    373  OD2 ASP A  50     -13.956  10.309  30.971  1.00  0.00           O  \\nATOM    374  N   ALA A  51     -13.830  10.309  25.805  1.00  0.00           N  \\nATOM    375  CA  ALA A  51     -13.704   9.931  24.419  1.00  0.00           C  \\nATOM    376  C   ALA A  51     -15.216   9.805  23.915  1.00  0.00           C  \\nATOM    377  O   ALA A  51     -16.098   9.301  24.167  1.00  0.00           O  \\nATOM    378  CB  ALA A  51     -13.200   8.671  24.041  1.00  0.00           C  \\nATOM    379  N   ASN A  52     -15.342  10.561  22.907  1.00  0.00           N  \\nATOM    380  CA  ASN A  52     -16.539  10.561  22.277  1.00  0.00           C  \\nATOM    381  C   ASN A  52     -17.232  11.380  22.907  1.00  0.00           C  \\nATOM    382  O   ASN A  52     -17.232  12.766  22.781  1.00  0.00           O  \\nATOM    383  CB  ASN A  52     -17.610   9.553  21.899  1.00  0.00           C  \\nATOM    384  CG  ASN A  52     -17.484   8.671  20.639  1.00  0.00           C  \\nATOM    385  OD1 ASN A  52     -16.980   7.789  20.135  1.00  0.00           O  \\nATOM    386  ND2 ASN A  52     -18.240   9.427  19.631  1.00  0.00           N  \\nATOM    387  N   LYS A  53     -17.862  10.939  24.041  1.00  0.00           N  \\nATOM    388  CA  LYS A  53     -18.240  11.947  25.301  1.00  0.00           C  \\nATOM    389  C   LYS A  53     -17.106  12.703  25.805  1.00  0.00           C  \\nATOM    390  O   LYS A  53     -16.476  13.711  25.175  1.00  0.00           O  \\nATOM    391  CB  LYS A  53     -18.870  11.317  26.561  1.00  0.00           C  \\nATOM    392  CG  LYS A  53     -20.130  10.435  26.057  1.00  0.00           C  \\nATOM    393  CD  LYS A  53     -20.634   9.301  27.065  1.00  0.00           C  \\nATOM    394  CE  LYS A  53     -22.146   8.671  26.372  1.00  0.00           C  \\nATOM    395  NZ  LYS A  53     -22.524   7.663  27.191  1.00  0.00           N  \\nATOM    396  N   SER A  54     -16.602  12.325  27.065  1.00  0.00           N  \\nATOM    397  CA  SER A  54     -15.594  13.081  27.821  1.00  0.00           C  \\nATOM    398  C   SER A  54     -15.594  14.593  27.443  1.00  0.00           C  \\nATOM    399  O   SER A  54     -16.476  15.223  27.695  1.00  0.00           O  \\nATOM    400  CB  SER A  54     -14.460  12.703  27.569  1.00  0.00           C  \\nATOM    401  OG  SER A  54     -13.452  12.955  28.703  1.00  0.00           O  \\nATOM    402  N   LYS A  55     -14.838  15.223  26.813  1.00  0.00           N  \\nATOM    403  CA  LYS A  55     -14.964  16.420  26.057  1.00  0.00           C  \\nATOM    404  C   LYS A  55     -16.602  16.987  25.931  1.00  0.00           C  \\nATOM    405  O   LYS A  55     -16.854  18.121  26.183  1.00  0.00           O  \\nATOM    406  CB  LYS A  55     -14.208  16.609  24.734  1.00  0.00           C  \\nATOM    407  CG  LYS A  55     -14.712  17.743  23.600  1.00  0.00           C  \\nATOM    408  CD  LYS A  55     -13.452  17.743  22.844  1.00  0.00           C  \\nATOM    409  CE  LYS A  55     -13.578  17.995  21.143  1.00  0.00           C  \\nATOM    410  NZ  LYS A  55     -12.444  17.743  20.135  1.00  0.00           N  \\nATOM    411  N   GLY A  56     -17.232  16.042  25.679  1.00  0.00           N  \\nATOM    412  CA  GLY A  56     -18.870  16.042  25.301  1.00  0.00           C  \\nATOM    413  C   GLY A  56     -19.122  17.239  24.293  1.00  0.00           C  \\nATOM    414  O   GLY A  56     -18.492  18.247  23.852  1.00  0.00           O  \\nATOM    415  N   ILE A  57     -20.130  17.113  23.285  1.00  0.00           N  \\nATOM    416  CA  ILE A  57     -20.004  17.869  22.025  1.00  0.00           C  \\nATOM    417  C   ILE A  57     -21.264  18.121  21.143  1.00  0.00           C  \\nATOM    418  O   ILE A  57     -22.398  18.499  21.395  1.00  0.00           O  \\nATOM    419  CB  ILE A  57     -18.996  17.491  21.395  1.00  0.00           C  \\nATOM    420  CG1 ILE A  57     -19.374  16.357  20.513  1.00  0.00           C  \\nATOM    421  CG2 ILE A  57     -17.862  17.050  22.025  1.00  0.00           C  \\nATOM    422  CD1 ILE A  57     -19.122  16.609  19.127  1.00  0.00           C  \\nATOM    423  N   VAL A  58     -21.264  17.743  19.631  1.00  0.00           N  \\nATOM    424  CA  VAL A  58     -22.524  17.995  19.001  1.00  0.00           C  \\nATOM    425  C   VAL A  58     -22.020  17.239  17.615  1.00  0.00           C  \\nATOM    426  O   VAL A  58     -21.642  18.121  16.985  1.00  0.00           O  \\nATOM    427  CB  VAL A  58     -23.280  19.129  19.127  1.00  0.00           C  \\nATOM    428  CG1 VAL A  58     -22.398  20.263  18.749  1.00  0.00           C  \\nATOM    429  CG2 VAL A  58     -24.540  19.192  18.371  1.00  0.00           C  \\nATOM    430  N   TRP A  59     -22.398  16.231  16.859  1.00  0.00           N  \\nATOM    431  CA  TRP A  59     -22.020  16.105  15.221  1.00  0.00           C  \\nATOM    432  C   TRP A  59     -22.524  17.239  14.213  1.00  0.00           C  \\nATOM    433  O   TRP A  59     -23.784  17.365  14.339  1.00  0.00           O  \\nATOM    434  CB  TRP A  59     -22.146  14.845  14.717  1.00  0.00           C  \\nATOM    435  CG  TRP A  59     -21.264  13.963  15.347  1.00  0.00           C  \\nATOM    436  CD1 TRP A  59     -21.390  12.703  15.851  1.00  0.00           C  \\nATOM    437  CD2 TRP A  59     -19.878  14.152  15.536  1.00  0.00           C  \\nATOM    438  NE1 TRP A  59     -20.382  12.451  16.607  1.00  0.00           N  \\nATOM    439  CE2 TRP A  59     -19.500  13.207  15.977  1.00  0.00           C  \\nATOM    440  CE3 TRP A  59     -19.248  15.160  15.095  1.00  0.00           C  \\nATOM    441  CZ2 TRP A  59     -18.114  13.459  16.607  1.00  0.00           C  \\nATOM    442  CZ3 TRP A  59     -18.051  15.223  15.473  1.00  0.00           C  \\nATOM    443  CH2 TRP A  59     -17.484  14.467  16.229  1.00  0.00           C  \\nATOM    444  N   ASN A  60     -21.768  17.869  13.772  1.00  0.00           N  \\nATOM    445  CA  ASN A  60     -22.398  19.129  12.953  1.00  0.00           C  \\nATOM    446  C   ASN A  60     -21.642  19.255  11.441  1.00  0.00           C  \\nATOM    447  O   ASN A  60     -20.634  18.751  11.189  1.00  0.00           O  \\nATOM    448  CB  ASN A  60     -22.524  20.515  13.583  1.00  0.00           C  \\nATOM    449  CG  ASN A  60     -22.776  21.649  12.701  1.00  0.00           C  \\nATOM    450  OD1 ASN A  60     -24.036  21.712  11.945  1.00  0.00           O  \\nATOM    451  ND2 ASN A  60     -21.894  22.783  12.701  1.00  0.00           N  \\nATOM    452  N   GLU A  61     -22.020  20.137  10.811  1.00  0.00           N  \\nATOM    453  CA  GLU A  61     -21.138  20.263   9.425  1.00  0.00           C  \\nATOM    454  C   GLU A  61     -19.626  20.389  10.181  1.00  0.00           C  \\nATOM    455  O   GLU A  61     -18.744  19.507   9.803  1.00  0.00           O  \\nATOM    456  CB  GLU A  61     -21.516  21.523   8.669  1.00  0.00           C  \\nATOM    457  CG  GLU A  61     -22.020  21.523   7.409  1.00  0.00           C  \\nATOM    458  CD  GLU A  61     -23.154  20.767   7.283  1.00  0.00           C  \\nATOM    459  OE1 GLU A  61     -24.540  20.767   7.535  1.00  0.00           O  \\nATOM    460  OE2 GLU A  61     -22.902  19.381   6.653  1.00  0.00           O  \\nATOM    461  N   ASN A  62     -19.248  21.397  10.685  1.00  0.00           N  \\nATOM    462  CA  ASN A  62     -17.736  21.586  11.441  1.00  0.00           C  \\nATOM    463  C   ASN A  62     -17.106  20.137  11.945  1.00  0.00           C  \\nATOM    464  O   ASN A  62     -15.972  20.515  12.071  1.00  0.00           O  \\nATOM    465  CB  ASN A  62     -17.736  22.783  11.945  1.00  0.00           C  \\nATOM    466  CG  ASN A  62     -17.610  24.043  11.441  1.00  0.00           C  \\nATOM    467  OD1 ASN A  62     -18.744  24.547  10.559  1.00  0.00           O  \\nATOM    468  ND2 ASN A  62     -16.539  24.799  11.063  1.00  0.00           N  \\nATOM    469  N   THR A  63     -17.610  20.263  13.268  1.00  0.00           N  \\nATOM    470  CA  THR A  63     -17.043  19.255  14.087  1.00  0.00           C  \\nATOM    471  C   THR A  63     -16.602  17.869  13.331  1.00  0.00           C  \\nATOM    472  O   THR A  63     -15.720  17.239  13.331  1.00  0.00           O  \\nATOM    473  CB  THR A  63     -18.051  18.751  15.095  1.00  0.00           C  \\nATOM    474  OG1 THR A  63     -17.358  18.499  15.851  1.00  0.00           O  \\nATOM    475  CG2 THR A  63     -18.492  17.365  14.528  1.00  0.00           C  \\nATOM    476  N   LEU A  64     -17.358  17.617  12.197  1.00  0.00           N  \\nATOM    477  CA  LEU A  64     -16.854  16.357  11.441  1.00  0.00           C  \\nATOM    478  C   LEU A  64     -15.468  16.861  10.685  1.00  0.00           C  \\nATOM    479  O   LEU A  64     -14.838  16.231  10.433  1.00  0.00           O  \\nATOM    480  CB  LEU A  64     -18.114  15.979  10.055  1.00  0.00           C  \\nATOM    481  CG  LEU A  64     -18.744  14.719  10.433  1.00  0.00           C  \\nATOM    482  CD1 LEU A  64     -19.122  14.719  11.567  1.00  0.00           C  \\nATOM    483  CD2 LEU A  64     -19.878  14.593   9.803  1.00  0.00           C  \\nATOM    484  N   MET A  65     -15.468  17.995  10.181  1.00  0.00           N  \\nATOM    485  CA  MET A  65     -13.956  18.499   9.803  1.00  0.00           C  \\nATOM    486  C   MET A  65     -13.200  18.373  11.189  1.00  0.00           C  \\nATOM    487  O   MET A  65     -12.192  17.869  11.189  1.00  0.00           O  \\nATOM    488  CB  MET A  65     -13.956  19.885   9.425  1.00  0.00           C  \\nATOM    489  CG  MET A  65     -13.452  20.137   7.913  1.00  0.00           C  \\nATOM    490  SD  MET A  65     -13.704  18.625   7.283  1.00  0.00           S  \\nATOM    491  CE  MET A  65     -13.452  19.192   5.519  1.00  0.00           C  \\nATOM    492  N   GLU A  66     -13.578  18.751  12.197  1.00  0.00           N  \\nATOM    493  CA  GLU A  66     -13.074  18.751  13.457  1.00  0.00           C  \\nATOM    494  C   GLU A  66     -12.444  17.239  13.646  1.00  0.00           C  \\nATOM    495  O   GLU A  66     -11.310  17.113  13.835  1.00  0.00           O  \\nATOM    496  CB  GLU A  66     -13.830  19.381  14.717  1.00  0.00           C  \\nATOM    497  CG  GLU A  66     -13.578  20.767  14.969  1.00  0.00           C  \\nATOM    498  CD  GLU A  66     -12.318  21.271  15.851  1.00  0.00           C  \\nATOM    499  OE1 GLU A  66     -12.192  22.657  16.103  1.00  0.00           O  \\nATOM    500  OE2 GLU A  66     -11.310  20.641  16.355  1.00  0.00           O  \\nATOM    501  N   TYR A  67     -13.263  16.483  13.457  1.00  0.00           N  \\nATOM    502  CA  TYR A  67     -12.822  14.971  13.457  1.00  0.00           C  \\nATOM    503  C   TYR A  67     -12.192  14.467  12.323  1.00  0.00           C  \\nATOM    504  O   TYR A  67     -11.184  14.089  11.945  1.00  0.00           O  \\nATOM    505  CB  TYR A  67     -14.082  14.341  13.835  1.00  0.00           C  \\nATOM    506  CG  TYR A  67     -14.082  12.955  14.465  1.00  0.00           C  \\nATOM    507  CD1 TYR A  67     -14.838  11.821  14.465  1.00  0.00           C  \\nATOM    508  CD2 TYR A  67     -13.326  13.081  15.599  1.00  0.00           C  \\nATOM    509  CE1 TYR A  67     -14.649  10.876  15.221  1.00  0.00           C  \\nATOM    510  CE2 TYR A  67     -13.200  11.947  16.103  1.00  0.00           C  \\nATOM    511  CZ  TYR A  67     -13.830  10.876  15.977  1.00  0.00           C  \\nATOM    512  OH  TYR A  67     -13.704   9.805  16.607  1.00  0.00           O  \\nATOM    513  N   LEU A  68     -12.822  14.719  11.315  1.00  0.00           N  \\nATOM    514  CA  LEU A  68     -12.192  14.341  10.055  1.00  0.00           C  \\nATOM    515  C   LEU A  68     -10.806  14.719  10.307  1.00  0.00           C  \\nATOM    516  O   LEU A  68      -9.798  13.963  10.307  1.00  0.00           O  \\nATOM    517  CB  LEU A  68     -13.074  14.845   9.173  1.00  0.00           C  \\nATOM    518  CG  LEU A  68     -14.334  13.837   8.795  1.00  0.00           C  \\nATOM    519  CD1 LEU A  68     -15.216  14.467   7.787  1.00  0.00           C  \\nATOM    520  CD2 LEU A  68     -13.704  12.829   7.787  1.00  0.00           C  \\nATOM    521  N   GLU A  69     -10.428  15.853  10.685  1.00  0.00           N  \\nATOM    522  CA  GLU A  69      -8.916  16.231  10.937  1.00  0.00           C  \\nATOM    523  C   GLU A  69      -8.097  14.971  11.441  1.00  0.00           C  \\nATOM    524  O   GLU A  69      -7.278  14.593  10.685  1.00  0.00           O  \\nATOM    525  CB  GLU A  69      -9.042  17.365  11.693  1.00  0.00           C  \\nATOM    526  CG  GLU A  69      -8.034  18.373  11.819  1.00  0.00           C  \\nATOM    527  CD  GLU A  69      -6.900  18.373  12.953  1.00  0.00           C  \\nATOM    528  OE1 GLU A  69      -5.388  18.499  12.827  1.00  0.00           O  \\nATOM    529  OE2 GLU A  69      -7.278  17.869  14.087  1.00  0.00           O  \\nATOM    530  N   ASN A  70      -8.097  14.845  12.575  1.00  0.00           N  \\nATOM    531  CA  ASN A  70      -7.278  13.585  12.827  1.00  0.00           C  \\nATOM    532  C   ASN A  70      -7.404  13.459  14.339  1.00  0.00           C  \\nATOM    533  O   ASN A  70      -7.152  14.215  15.221  1.00  0.00           O  \\nATOM    534  CB  ASN A  70      -5.892  13.837  12.449  1.00  0.00           C  \\nATOM    535  CG  ASN A  70      -4.758  14.089  13.457  1.00  0.00           C  \\nATOM    536  OD1 ASN A  70      -3.624  15.097  13.457  1.00  0.00           O  \\nATOM    537  ND2 ASN A  70      -4.758  13.333  14.717  1.00  0.00           N  \\nATOM    538  N   PRO A  71      -7.908  12.199  14.780  1.00  0.00           N  \\nATOM    539  CA  PRO A  71      -8.475  11.758  15.725  1.00  0.00           C  \\nATOM    540  C   PRO A  71      -7.593  11.758  16.859  1.00  0.00           C  \\nATOM    541  O   PRO A  71      -7.782  11.317  17.993  1.00  0.00           O  \\nATOM    542  CB  PRO A  71      -9.042  10.561  15.536  1.00  0.00           C  \\nATOM    543  CG  PRO A  71      -8.916  10.309  14.213  1.00  0.00           C  \\nATOM    544  CD  PRO A  71      -8.160  11.191  13.583  1.00  0.00           C  \\nATOM    545  N   LYS A  72      -6.522  12.325  16.733  1.00  0.00           N  \\nATOM    546  CA  LYS A  72      -5.388  12.703  17.615  1.00  0.00           C  \\nATOM    547  C   LYS A  72      -5.451  14.215  17.993  1.00  0.00           C  \\nATOM    548  O   LYS A  72      -5.262  14.971  17.111  1.00  0.00           O  \\nATOM    549  CB  LYS A  72      -3.876  12.451  17.111  1.00  0.00           C  \\nATOM    550  CG  LYS A  72      -3.624  11.065  17.363  1.00  0.00           C  \\nATOM    551  CD  LYS A  72      -2.112  10.435  17.237  1.00  0.00           C  \\nATOM    552  CE  LYS A  72      -1.860   8.923  17.741  1.00  0.00           C  \\nATOM    553  NZ  LYS A  72      -0.222   8.545  17.741  1.00  0.00           N  \\nATOM    554  N   LYS A  73      -6.081  14.719  19.127  1.00  0.00           N  \\nATOM    555  CA  LYS A  73      -6.396  16.231  19.127  1.00  0.00           C  \\nATOM    556  C   LYS A  73      -7.782  16.357  18.623  1.00  0.00           C  \\nATOM    557  O   LYS A  73      -8.160  16.231  17.363  1.00  0.00           O  \\nATOM    558  CB  LYS A  73      -5.640  17.113  17.993  1.00  0.00           C  \\nATOM    559  CG  LYS A  73      -4.758  18.247  18.623  1.00  0.00           C  \\nATOM    560  CD  LYS A  73      -3.876  19.003  17.993  1.00  0.00           C  \\nATOM    561  CE  LYS A  73      -2.616  20.389  18.812  1.00  0.00           C  \\nATOM    562  NZ  LYS A  73      -1.986  21.145  18.056  1.00  0.00           N  \\nATOM    563  N   TYR A  74      -8.664  16.231  19.631  1.00  0.00           N  \\nATOM    564  CA  TYR A  74     -10.050  16.105  19.379  1.00  0.00           C  \\nATOM    565  C   TYR A  74     -10.806  14.971  19.883  1.00  0.00           C  \\nATOM    566  O   TYR A  74     -10.554  13.963  19.379  1.00  0.00           O  \\nATOM    567  CB  TYR A  74     -10.554  16.105  17.741  1.00  0.00           C  \\nATOM    568  CG  TYR A  74     -11.814  16.735  17.363  1.00  0.00           C  \\nATOM    569  CD1 TYR A  74     -12.948  16.105  17.237  1.00  0.00           C  \\nATOM    570  CD2 TYR A  74     -11.940  17.995  16.985  1.00  0.00           C  \\nATOM    571  CE1 TYR A  74     -14.145  16.609  16.859  1.00  0.00           C  \\nATOM    572  CE2 TYR A  74     -13.074  18.625  16.607  1.00  0.00           C  \\nATOM    573  CZ  TYR A  74     -14.082  17.869  16.607  1.00  0.00           C  \\nATOM    574  OH  TYR A  74     -15.216  18.373  16.355  1.00  0.00           O  \\nATOM    575  N   ILE A  75     -11.310  15.097  21.017  1.00  0.00           N  \\nATOM    576  CA  ILE A  75     -11.940  14.089  21.647  1.00  0.00           C  \\nATOM    577  C   ILE A  75     -10.806  13.081  21.773  1.00  0.00           C  \\nATOM    578  O   ILE A  75     -10.932  11.821  21.521  1.00  0.00           O  \\nATOM    579  CB  ILE A  75     -13.200  13.585  21.143  1.00  0.00           C  \\nATOM    580  CG1 ILE A  75     -13.641  14.089  19.631  1.00  0.00           C  \\nATOM    581  CG2 ILE A  75     -14.271  13.585  22.025  1.00  0.00           C  \\nATOM    582  CD1 ILE A  75     -14.712  13.459  19.253  1.00  0.00           C  \\nATOM    583  N   PRO A  76      -9.420  13.585  21.899  1.00  0.00           N  \\nATOM    584  CA  PRO A  76      -8.160  12.955  21.647  1.00  0.00           C  \\nATOM    585  C   PRO A  76      -8.160  11.695  22.529  1.00  0.00           C  \\nATOM    586  O   PRO A  76      -8.538  11.695  23.789  1.00  0.00           O  \\nATOM    587  CB  PRO A  76      -6.900  13.837  21.836  1.00  0.00           C  \\nATOM    588  CG  PRO A  76      -7.152  14.845  22.277  1.00  0.00           C  \\nATOM    589  CD  PRO A  76      -8.538  14.845  22.151  1.00  0.00           C  \\nATOM    590  N   GLY A  77      -8.097  10.435  22.088  1.00  0.00           N  \\nATOM    591  CA  GLY A  77      -8.286   9.175  22.907  1.00  0.00           C  \\nATOM    592  C   GLY A  77      -9.546   8.167  22.529  1.00  0.00           C  \\nATOM    593  O   GLY A  77      -9.798   7.159  22.781  1.00  0.00           O  \\nATOM    594  N   THR A  78     -10.050   8.545  21.143  1.00  0.00           N  \\nATOM    595  CA  THR A  78     -11.310   7.789  20.639  1.00  0.00           C  \\nATOM    596  C   THR A  78     -10.995   6.529  20.513  1.00  0.00           C  \\nATOM    597  O   THR A  78      -9.798   6.151  20.009  1.00  0.00           O  \\nATOM    598  CB  THR A  78     -11.625   8.167  19.253  1.00  0.00           C  \\nATOM    599  OG1 THR A  78     -13.074   8.167  19.127  1.00  0.00           O  \\nATOM    600  CG2 THR A  78     -11.184   9.553  18.875  1.00  0.00           C  \\nATOM    601  N   LYS A  79     -11.562   5.521  20.765  1.00  0.00           N  \\nATOM    602  CA  LYS A  79     -11.058   4.072  20.639  1.00  0.00           C  \\nATOM    603  C   LYS A  79     -11.058   3.631  19.001  1.00  0.00           C  \\nATOM    604  O   LYS A  79     -10.932   2.497  18.371  1.00  0.00           O  \\nATOM    605  CB  LYS A  79     -11.562   3.064  21.395  1.00  0.00           C  \\nATOM    606  CG  LYS A  79     -12.570   2.560  20.513  1.00  0.00           C  \\nATOM    607  CD  LYS A  79     -13.704   1.615  20.891  1.00  0.00           C  \\nATOM    608  CE  LYS A  79     -14.838   1.363  20.135  1.00  0.00           C  \\nATOM    609  NZ  LYS A  79     -15.972   0.859  20.513  1.00  0.00           N  \\nATOM    610  N   MET A  80     -11.310   4.639  18.119  1.00  0.00           N  \\nATOM    611  CA  MET A  80     -11.436   4.135  16.607  1.00  0.00           C  \\nATOM    612  C   MET A  80     -10.050   4.198  16.103  1.00  0.00           C  \\nATOM    613  O   MET A  80      -9.294   5.017  16.103  1.00  0.00           O  \\nATOM    614  CB  MET A  80     -12.444   5.017  15.851  1.00  0.00           C  \\nATOM    615  CG  MET A  80     -11.940   6.088  14.969  1.00  0.00           C  \\nATOM    616  SD  MET A  80     -13.074   6.781  13.457  1.00  0.00           S  \\nATOM    617  CE  MET A  80     -12.192   7.852  13.079  1.00  0.00           C  \\nATOM    618  N   ILE A  81      -9.672   3.127  15.725  1.00  0.00           N  \\nATOM    619  CA  ILE A  81      -8.160   3.127  15.095  1.00  0.00           C  \\nATOM    620  C   ILE A  81      -7.908   4.009  13.835  1.00  0.00           C  \\nATOM    621  O   ILE A  81      -8.034   3.505  12.323  1.00  0.00           O  \\nATOM    622  CB  ILE A  81      -7.656   1.867  15.284  1.00  0.00           C  \\nATOM    623  CG1 ILE A  81      -6.522   1.867  16.103  1.00  0.00           C  \\nATOM    624  CG2 ILE A  81      -7.026   1.237  13.835  1.00  0.00           C  \\nATOM    625  CD1 ILE A  81      -6.900   2.245  17.615  1.00  0.00           C  \\nATOM    626  N   PHE A  82      -7.530   5.143  13.961  1.00  0.00           N  \\nATOM    627  CA  PHE A  82      -7.404   5.899  12.449  1.00  0.00           C  \\nATOM    628  C   PHE A  82      -6.144   6.529  12.197  1.00  0.00           C  \\nATOM    629  O   PHE A  82      -5.766   7.411  12.953  1.00  0.00           O  \\nATOM    630  CB  PHE A  82      -8.538   6.655  12.449  1.00  0.00           C  \\nATOM    631  CG  PHE A  82      -8.916   7.159  10.937  1.00  0.00           C  \\nATOM    632  CD1 PHE A  82      -8.916   8.293  10.433  1.00  0.00           C  \\nATOM    633  CD2 PHE A  82      -9.042   6.529  10.055  1.00  0.00           C  \\nATOM    634  CE1 PHE A  82      -9.042   8.671   9.173  1.00  0.00           C  \\nATOM    635  CE2 PHE A  82      -9.042   6.781   8.669  1.00  0.00           C  \\nATOM    636  CZ  PHE A  82      -9.042   7.852   8.291  1.00  0.00           C  \\nATOM    637  N   ALA A  83      -5.640   6.529  11.189  1.00  0.00           N  \\nATOM    638  CA  ALA A  83      -4.506   7.285  10.937  1.00  0.00           C  \\nATOM    639  C   ALA A  83      -4.758   8.797  10.811  1.00  0.00           C  \\nATOM    640  O   ALA A  83      -4.884   9.679  11.441  1.00  0.00           O  \\nATOM    641  CB  ALA A  83      -3.876   6.907   9.173  1.00  0.00           C  \\nATOM    642  N   GLY A  84      -5.136   9.301   9.677  1.00  0.00           N  \\nATOM    643  CA  GLY A  84      -5.325  10.687   9.299  1.00  0.00           C  \\nATOM    644  C   GLY A  84      -6.396  11.317   8.543  1.00  0.00           C  \\nATOM    645  O   GLY A  84      -7.467  11.569   8.921  1.00  0.00           O  \\nATOM    646  N   ILE A  85      -6.207  11.569   7.283  1.00  0.00           N  \\nATOM    647  CA  ILE A  85      -7.152  12.451   6.653  1.00  0.00           C  \\nATOM    648  C   ILE A  85      -6.774  13.963   6.905  1.00  0.00           C  \\nATOM    649  O   ILE A  85      -7.656  14.719   7.220  1.00  0.00           O  \\nATOM    650  CB  ILE A  85      -8.664  12.199   7.031  1.00  0.00           C  \\nATOM    651  CG1 ILE A  85      -9.168  10.813   6.149  1.00  0.00           C  \\nATOM    652  CG2 ILE A  85      -9.420  13.333   6.401  1.00  0.00           C  \\nATOM    653  CD1 ILE A  85     -10.176  11.191   5.204  1.00  0.00           C  \\nATOM    654  N   LYS A  86      -5.640  14.467   6.527  1.00  0.00           N  \\nATOM    655  CA  LYS A  86      -5.325  15.727   7.157  1.00  0.00           C  \\nATOM    656  C   LYS A  86      -5.136  16.546   5.771  1.00  0.00           C  \\nATOM    657  O   LYS A  86      -4.254  17.365   5.267  1.00  0.00           O  \\nATOM    658  CB  LYS A  86      -3.876  15.727   7.913  1.00  0.00           C  \\nATOM    659  CG  LYS A  86      -3.120  14.467   7.535  1.00  0.00           C  \\nATOM    660  CD  LYS A  86      -1.356  14.215   7.787  1.00  0.00           C  \\nATOM    661  CE  LYS A  86      -0.852  12.955   6.653  1.00  0.00           C  \\nATOM    662  NZ  LYS A  86       0.660  12.577   7.157  1.00  0.00           N  \\nATOM    663  N   LYS A  87      -5.892  16.357   5.015  1.00  0.00           N  \\nATOM    664  CA  LYS A  87      -5.892  17.239   3.755  1.00  0.00           C  \\nATOM    665  C   LYS A  87      -7.278  17.869   3.377  1.00  0.00           C  \\nATOM    666  O   LYS A  87      -8.160  17.239   2.873  1.00  0.00           O  \\nATOM    667  CB  LYS A  87      -5.514  16.294   2.558  1.00  0.00           C  \\nATOM    668  CG  LYS A  87      -4.632  16.735   1.235  1.00  0.00           C  \\nATOM    669  CD  LYS A  87      -4.254  15.727   0.353  1.00  0.00           C  \\nATOM    670  CE  LYS A  87      -3.120  16.231  -0.403  1.00  0.00           C  \\nATOM    671  NZ  LYS A  87      -2.868  15.727  -1.915  1.00  0.00           N  \\nATOM    672  N   LYS A  88      -7.278  19.129   3.251  1.00  0.00           N  \\nATOM    673  CA  LYS A  88      -8.412  19.885   2.621  1.00  0.00           C  \\nATOM    674  C   LYS A  88      -9.168  18.877   1.613  1.00  0.00           C  \\nATOM    675  O   LYS A  88     -10.428  18.499   1.613  1.00  0.00           O  \\nATOM    676  CB  LYS A  88      -7.782  21.019   1.991  1.00  0.00           C  \\nATOM    677  CG  LYS A  88      -6.270  20.767   1.865  1.00  0.00           C  \\nATOM    678  CD  LYS A  88      -5.514  21.271   0.605  1.00  0.00           C  \\nATOM    679  CE  LYS A  88      -5.766  20.767  -0.403  1.00  0.00           C  \\nATOM    680  NZ  LYS A  88      -5.388  21.523  -1.663  1.00  0.00           N  \\nATOM    681  N   GLY A  89      -8.538  18.499   0.353  1.00  0.00           N  \\nATOM    682  CA  GLY A  89      -9.168  17.491  -0.781  1.00  0.00           C  \\nATOM    683  C   GLY A  89     -10.050  16.357  -0.025  1.00  0.00           C  \\nATOM    684  O   GLY A  89     -11.184  16.357  -0.025  1.00  0.00           O  \\nATOM    685  N   GLU A  90      -9.546  15.475   0.479  1.00  0.00           N  \\nATOM    686  CA  GLU A  90     -10.050  14.089   1.109  1.00  0.00           C  \\nATOM    687  C   GLU A  90     -10.932  14.593   2.369  1.00  0.00           C  \\nATOM    688  O   GLU A  90     -12.192  14.341   2.621  1.00  0.00           O  \\nATOM    689  CB  GLU A  90      -9.042  13.207   1.613  1.00  0.00           C  \\nATOM    690  CG  GLU A  90      -8.286  12.577   0.605  1.00  0.00           C  \\nATOM    691  CD  GLU A  90      -6.900  11.821   0.857  1.00  0.00           C  \\nATOM    692  OE1 GLU A  90      -6.018  11.443  -0.277  1.00  0.00           O  \\nATOM    693  OE2 GLU A  90      -6.270  11.569   2.243  1.00  0.00           O  \\nATOM    694  N   ARG A  91     -10.554  15.475   2.999  1.00  0.00           N  \\nATOM    695  CA  ARG A  91     -11.184  15.916   4.511  1.00  0.00           C  \\nATOM    696  C   ARG A  91     -12.570  16.672   3.881  1.00  0.00           C  \\nATOM    697  O   ARG A  91     -13.578  16.357   4.133  1.00  0.00           O  \\nATOM    698  CB  ARG A  91     -10.302  16.987   5.141  1.00  0.00           C  \\nATOM    699  CG  ARG A  91     -10.554  17.113   6.527  1.00  0.00           C  \\nATOM    700  CD  ARG A  91      -9.420  18.121   7.283  1.00  0.00           C  \\nATOM    701  NE  ARG A  91      -9.294  19.381   6.527  1.00  0.00           N  \\nATOM    702  CZ  ARG A  91      -7.908  20.137   6.464  1.00  0.00           C  \\nATOM    703  NH1 ARG A  91      -6.900  19.507   7.535  1.00  0.00           N  \\nATOM    704  NH2 ARG A  91      -7.530  21.145   5.519  1.00  0.00           N  \\nATOM    705  N   GLN A  92     -12.444  17.743   3.251  1.00  0.00           N  \\nATOM    706  CA  GLN A  92     -13.830  18.373   2.369  1.00  0.00           C  \\nATOM    707  C   GLN A  92     -14.586  17.113   1.991  1.00  0.00           C  \\nATOM    708  O   GLN A  92     -15.720  16.735   2.243  1.00  0.00           O  \\nATOM    709  CB  GLN A  92     -13.704  19.633   1.739  1.00  0.00           C  \\nATOM    710  CG  GLN A  92     -13.830  19.255   0.164  1.00  0.00           C  \\nATOM    711  CD  GLN A  92     -13.200  20.326  -0.907  1.00  0.00           C  \\nATOM    712  OE1 GLN A  92     -11.688  19.885  -0.403  1.00  0.00           O  \\nATOM    713  NE2 GLN A  92     -13.452  21.460  -1.411  1.00  0.00           N  \\nATOM    714  N   ASP A  93     -13.956  16.357   0.983  1.00  0.00           N  \\nATOM    715  CA  ASP A  93     -14.838  15.223   0.605  1.00  0.00           C  \\nATOM    716  C   ASP A  93     -15.342  14.341   1.613  1.00  0.00           C  \\nATOM    717  O   ASP A  93     -16.602  13.837   1.487  1.00  0.00           O  \\nATOM    718  CB  ASP A  93     -13.830  14.593  -0.529  1.00  0.00           C  \\nATOM    719  CG  ASP A  93     -13.704  14.971  -1.915  1.00  0.00           C  \\nATOM    720  OD1 ASP A  93     -12.444  15.727  -2.419  1.00  0.00           O  \\nATOM    721  OD2 ASP A  93     -14.838  15.223  -2.797  1.00  0.00           O  \\nATOM    722  N   LEU A  94     -14.838  13.963   2.495  1.00  0.00           N  \\nATOM    723  CA  LEU A  94     -15.531  13.207   3.503  1.00  0.00           C  \\nATOM    724  C   LEU A  94     -16.728  13.963   3.881  1.00  0.00           C  \\nATOM    725  O   LEU A  94     -17.925  13.459   3.881  1.00  0.00           O  \\nATOM    726  CB  LEU A  94     -14.586  13.018   4.763  1.00  0.00           C  \\nATOM    727  CG  LEU A  94     -14.586  11.443   5.267  1.00  0.00           C  \\nATOM    728  CD1 LEU A  94     -15.783  10.813   4.637  1.00  0.00           C  \\nATOM    729  CD2 LEU A  94     -13.578  10.561   4.889  1.00  0.00           C  \\nATOM    730  N   VAL A  95     -16.539  15.097   4.637  1.00  0.00           N  \\nATOM    731  CA  VAL A  95     -17.736  15.979   5.141  1.00  0.00           C  \\nATOM    732  C   VAL A  95     -18.744  15.853   3.881  1.00  0.00           C  \\nATOM    733  O   VAL A  95     -20.004  15.601   4.322  1.00  0.00           O  \\nATOM    734  CB  VAL A  95     -17.358  17.365   5.393  1.00  0.00           C  \\nATOM    735  CG1 VAL A  95     -18.492  18.184   5.519  1.00  0.00           C  \\nATOM    736  CG2 VAL A  95     -16.476  17.491   6.401  1.00  0.00           C  \\nATOM    737  N   ALA A  96     -18.492  15.979   2.621  1.00  0.00           N  \\nATOM    738  CA  ALA A  96     -19.752  15.601   1.739  1.00  0.00           C  \\nATOM    739  C   ALA A  96     -20.130  14.215   1.802  1.00  0.00           C  \\nATOM    740  O   ALA A  96     -21.390  13.837   1.802  1.00  0.00           O  \\nATOM    741  CB  ALA A  96     -19.500  16.105   0.353  1.00  0.00           C  \\nATOM    742  N   TYR A  97     -19.248  13.333   1.865  1.00  0.00           N  \\nATOM    743  CA  TYR A  97     -19.626  11.884   1.865  1.00  0.00           C  \\nATOM    744  C   TYR A  97     -20.886  11.632   2.873  1.00  0.00           C  \\nATOM    745  O   TYR A  97     -22.146  11.317   2.621  1.00  0.00           O  \\nATOM    746  CB  TYR A  97     -18.744  10.813   2.243  1.00  0.00           C  \\nATOM    747  CG  TYR A  97     -19.374   9.427   1.739  1.00  0.00           C  \\nATOM    748  CD1 TYR A  97     -19.626   8.419   2.621  1.00  0.00           C  \\nATOM    749  CD2 TYR A  97     -19.626   9.175   0.353  1.00  0.00           C  \\nATOM    750  CE1 TYR A  97     -20.256   7.348   2.117  1.00  0.00           C  \\nATOM    751  CE2 TYR A  97     -20.256   8.041  -0.403  1.00  0.00           C  \\nATOM    752  CZ  TYR A  97     -20.508   7.285   0.857  1.00  0.00           C  \\nATOM    753  OH  TYR A  97     -21.264   6.403   0.353  1.00  0.00           O  \\nATOM    754  N   LEU A  98     -20.760  11.947   4.007  1.00  0.00           N  \\nATOM    755  CA  LEU A  98     -21.768  11.695   5.393  1.00  0.00           C  \\nATOM    756  C   LEU A  98     -23.154  12.703   5.015  1.00  0.00           C  \\nATOM    757  O   LEU A  98     -24.288  12.199   4.889  1.00  0.00           O  \\nATOM    758  CB  LEU A  98     -21.012  11.695   6.527  1.00  0.00           C  \\nATOM    759  CG  LEU A  98     -20.004  10.939   6.401  1.00  0.00           C  \\nATOM    760  CD1 LEU A  98     -19.374  10.687   7.661  1.00  0.00           C  \\nATOM    761  CD2 LEU A  98     -20.508   9.553   5.771  1.00  0.00           C  \\nATOM    762  N   LYS A  99     -23.028  13.963   4.889  1.00  0.00           N  \\nATOM    763  CA  LYS A  99     -24.288  14.593   4.385  1.00  0.00           C  \\nATOM    764  C   LYS A  99     -25.044  13.963   2.999  1.00  0.00           C  \\nATOM    765  O   LYS A  99     -26.304  13.837   3.125  1.00  0.00           O  \\nATOM    766  CB  LYS A  99     -23.910  16.105   4.259  1.00  0.00           C  \\nATOM    767  CG  LYS A  99     -24.918  17.113   4.259  1.00  0.00           C  \\nATOM    768  CD  LYS A  99     -24.666  18.625   3.881  1.00  0.00           C  \\nATOM    769  CE  LYS A  99     -25.674  19.633   4.133  1.00  0.00           C  \\nATOM    770  NZ  LYS A  99     -25.674  21.082   3.755  1.00  0.00           N  \\nATOM    771  N   SER A 100     -24.540  13.459   1.991  1.00  0.00           N  \\nATOM    772  CA  SER A 100     -25.296  12.577   0.857  1.00  0.00           C  \\nATOM    773  C   SER A 100     -25.548  11.191   1.361  1.00  0.00           C  \\nATOM    774  O   SER A 100     -26.934  10.813   1.361  1.00  0.00           O  \\nATOM    775  CB  SER A 100     -24.540  12.577  -0.655  1.00  0.00           C  \\nATOM    776  OG  SER A 100     -25.296  11.695  -1.663  1.00  0.00           O  \\nATOM    777  N   ALA A 101     -24.666  10.435   1.865  1.00  0.00           N  \\nATOM    778  CA  ALA A 101     -25.044   8.797   2.243  1.00  0.00           C  \\nATOM    779  C   ALA A 101     -25.800   8.923   3.629  1.00  0.00           C  \\nATOM    780  O   ALA A 101     -26.808   8.293   3.755  1.00  0.00           O  \\nATOM    781  CB  ALA A 101     -23.910   8.167   2.369  1.00  0.00           C  \\nATOM    782  N   THR A 102     -25.422   9.427   4.637  1.00  0.00           N  \\nATOM    783  CA  THR A 102     -26.052   9.301   6.149  1.00  0.00           C  \\nATOM    784  C   THR A 102     -27.690   9.805   5.897  1.00  0.00           C  \\nATOM    785  O   THR A 102     -28.572   9.049   6.149  1.00  0.00           O  \\nATOM    786  CB  THR A 102     -25.548  10.246   7.220  1.00  0.00           C  \\nATOM    787  OG1 THR A 102     -24.666   9.427   7.913  1.00  0.00           O  \\nATOM    788  CG2 THR A 102     -26.430  10.876   8.039  1.00  0.00           C  \\nATOM    789  N   SER A 103     -27.942  10.939   5.645  1.00  0.00           N  \\nATOM    790  CA  SER A 103     -29.454  11.317   5.519  1.00  0.00           C  \\nATOM    791  C   SER A 103     -30.084  10.813   4.385  1.00  0.00           C  \\nATOM    792  O   SER A 103     -29.454   9.931   3.377  1.00  0.00           O  \\nATOM    793  CB  SER A 103     -29.580  12.892   5.897  1.00  0.00           C  \\nATOM    794  OG  SER A 103     -28.320  13.585   5.393  1.00  0.00           O  \\nATOM    795  OXT SER A 103     -31.344  11.317   4.133  1.00  0.00           O  \\nTER     796      SER A 103                                                      \\nHETATM 1593 FE   HEC A 104     -15.216   5.773  12.701  1.00  0.00          FE  \\nHETATM 1594  CHA HEC A 104     -16.980   6.466  15.347  1.00  0.00           C  \\nHETATM 1595  CHB HEC A 104     -16.476   8.293  11.189  1.00  0.00           C  \\nHETATM 1596  CHC HEC A 104     -13.578   5.017   9.929  1.00  0.00           C  \\nHETATM 1597  CHD HEC A 104     -13.956   3.127  14.339  1.00  0.00           C  \\nHETATM 1598  NA  HEC A 104     -16.476   7.096  13.205  1.00  0.00           N  \\nHETATM 1599  C1A HEC A 104     -17.106   7.159  14.465  1.00  0.00           C  \\nHETATM 1600  C2A HEC A 104     -17.862   8.167  14.591  1.00  0.00           C  \\nHETATM 1601  C3A HEC A 104     -17.736   8.797  13.205  1.00  0.00           C  \\nHETATM 1602  C4A HEC A 104     -17.114   8.167  12.323  1.00  0.00           C  \\nHETATM 1603  CMA HEC A 104     -18.303  10.057  12.701  1.00  0.00           C  \\nHETATM 1604  CAA HEC A 104     -18.618   8.167  15.599  1.00  0.00           C  \\nHETATM 1605  CBA HEC A 104     -20.067   8.734  15.473  1.00  0.00           C  \\nHETATM 1606  CGA HEC A 104     -20.823   8.734  16.859  1.00  0.00           C  \\nHETATM 1607  O1A HEC A 104     -21.705   8.041  17.111  1.00  0.00           O  \\nHETATM 1608  O2A HEC A 104     -20.508   9.490  17.741  1.00  0.00           O  \\nHETATM 1609  NB  HEC A 104     -15.090   6.466  11.126  1.00  0.00           N  \\nHETATM 1610  C1B HEC A 104     -15.594   7.348  10.559  1.00  0.00           C  \\nHETATM 1611  C2B HEC A 104     -15.216   7.663   9.299  1.00  0.00           C  \\nHETATM 1612  C3B HEC A 104     -14.334   6.592   8.921  1.00  0.00           C  \\nHETATM 1613  C4B HEC A 104     -14.334   6.025   9.929  1.00  0.00           C  \\nHETATM 1614  CMB HEC A 104     -15.657   8.797   8.543  1.00  0.00           C  \\nHETATM 1615  CAB HEC A 104     -13.578   6.781   7.409  1.00  0.00           C  \\nHETATM 1616  CBB HEC A 104     -12.570   7.663   7.661  1.00  0.00           C  \\nHETATM 1617  NC  HEC A 104     -13.956   4.387  12.197  1.00  0.00           N  \\nHETATM 1618  C1C HEC A 104     -13.452   4.261  10.811  1.00  0.00           C  \\nHETATM 1619  C2C HEC A 104     -12.759   3.001  10.811  1.00  0.00           C  \\nHETATM 1620  C3C HEC A 104     -12.948   2.623  11.945  1.00  0.00           C  \\nHETATM 1621  C4C HEC A 104     -13.704   3.379  12.953  1.00  0.00           C  \\nHETATM 1622  CMC HEC A 104     -11.688   2.497   9.803  1.00  0.00           C  \\nHETATM 1623  CAC HEC A 104     -12.444   1.111  12.197  1.00  0.00           C  \\nHETATM 1624  CBC HEC A 104     -11.184   0.859  12.008  1.00  0.00           C  \\nHETATM 1625  ND  HEC A 104     -15.405   5.143  14.213  1.00  0.00           N  \\nHETATM 1626  C1D HEC A 104     -14.838   4.009  14.843  1.00  0.00           C  \\nHETATM 1627  C2D HEC A 104     -15.216   3.757  15.977  1.00  0.00           C  \\nHETATM 1628  C3D HEC A 104     -16.098   4.765  16.355  1.00  0.00           C  \\nHETATM 1629  C4D HEC A 104     -16.161   5.395  15.347  1.00  0.00           C  \\nHETATM 1630  CMD HEC A 104     -14.838   2.749  16.733  1.00  0.00           C  \\nHETATM 1631  CAD HEC A 104     -16.854   4.513  17.615  1.00  0.00           C  \\nHETATM 1632  CBD HEC A 104     -16.098   4.135  18.623  1.00  0.00           C  \\nHETATM 1633  CGD HEC A 104     -15.216   5.017  19.505  1.00  0.00           C  \\nHETATM 1634  O1D HEC A 104     -14.082   5.773  18.560  1.00  0.00           O  \\nHETATM 1635  O2D HEC A 104     -15.405   5.647  20.639  1.00  0.00           O  \\nHETATM 1679  O   HOH A 105     -16.496   9.304  17.402  1.00  0.00           O  \\nCONECT  104 1615                                                                \\nCONECT  124 1623                                                                \\nCONECT  134 1593                                                                \\nCONECT  356 1635                                                                \\nCONECT  616 1593                                                                \\nCONECT  900 1658                                                                \\nCONECT  920 1666                                                                \\nCONECT  930 1636                                                                \\nCONECT 1152 1678                                                                \\nCONECT 1412 1636                                                                \\nCONECT 1593  134  616 1598 1609                                                 \\nCONECT 1593 1617 1625                                                           \\nCONECT 1594 1599 1629                                                           \\nCONECT 1595 1602 1610                                                           \\nCONECT 1596 1613 1618                                                           \\nCONECT 1597 1621 1626                                                           \\nCONECT 1598 1593 1599 1602                                                      \\nCONECT 1599 1594 1598 1600                                                      \\nCONECT 1600 1599 1601 1604                                                      \\nCONECT 1601 1600 1602 1603                                                      \\nCONECT 1602 1595 1598 1601                                                      \\nCONECT 1603 1601                                                                \\nCONECT 1604 1600 1605                                                           \\nCONECT 1605 1604 1606                                                           \\nCONECT 1606 1605 1607 1608                                                      \\nCONECT 1607 1606                                                                \\nCONECT 1608 1606                                                                \\nCONECT 1609 1593 1610 1613                                                      \\nCONECT 1610 1595 1609 1611                                                      \\nCONECT 1611 1610 1612 1614                                                      \\nCONECT 1612 1611 1613 1615                                                      \\nCONECT 1613 1596 1609 1612                                                      \\nCONECT 1614 1611                                                                \\nCONECT 1615  104 1612 1616                                                      \\nCONECT 1616 1615                                                                \\nCONECT 1617 1593 1618 1621                                                      \\nCONECT 1618 1596 1617 1619                                                      \\nCONECT 1619 1618 1620 1622                                                      \\nCONECT 1620 1619 1621 1623                                                      \\nCONECT 1621 1597 1617 1620                                                      \\nCONECT 1622 1619                                                                \\nCONECT 1623  124 1620 1624                                                      \\nCONECT 1624 1623                                                                \\nCONECT 1625 1593 1626 1629                                                      \\nCONECT 1626 1597 1625 1627                                                      \\nCONECT 1627 1626 1628 1630                                                      \\nCONECT 1628 1627 1629 1631                                                      \\nCONECT 1629 1594 1625 1628                                                      \\nCONECT 1630 1627                                                                \\nCONECT 1631 1628 1632                                                           \\nCONECT 1632 1631 1633                                                           \\nCONECT 1633 1632 1634 1635                                                      \\nCONECT 1634 1633                                                                \\nCONECT 1635  356 1633                                                           \\nCONECT 1636  930 1412 1641 1652                                                 \\nCONECT 1636 1660 1668                                                           \\nCONECT 1637 1642 1672                                                           \\nCONECT 1638 1645 1653                                                           \\nCONECT 1639 1656 1661                                                           \\nCONECT 1640 1664 1669                                                           \\nCONECT 1641 1636 1642 1645                                                      \\nCONECT 1642 1637 1641 1643                                                      \\nCONECT 1643 1642 1644 1647                                                      \\nCONECT 1644 1643 1645 1646                                                      \\nCONECT 1645 1638 1641 1644                                                      \\nCONECT 1646 1644                                                                \\nCONECT 1647 1643 1648                                                           \\nCONECT 1648 1647 1649                                                           \\nCONECT 1649 1648 1650 1651                                                      \\nCONECT 1650 1649                                                                \\nCONECT 1651 1649                                                                \\nCONECT 1652 1636 1653 1656                                                      \\nCONECT 1653 1638 1652 1654                                                      \\nCONECT 1654 1653 1655 1657                                                      \\nCONECT 1655 1654 1656 1658                                                      \\nCONECT 1656 1639 1652 1655                                                      \\nCONECT 1657 1654                                                                \\nCONECT 1658  900 1655 1659                                                      \\nCONECT 1659 1658                                                                \\nCONECT 1660 1636 1661 1664                                                      \\nCONECT 1661 1639 1660 1662                                                      \\nCONECT 1662 1661 1663 1665                                                      \\nCONECT 1663 1662 1664 1666                                                      \\nCONECT 1664 1640 1660 1663                                                      \\nCONECT 1665 1662                                                                \\nCONECT 1666  920 1663 1667                                                      \\nCONECT 1667 1666                                                                \\nCONECT 1668 1636 1669 1672                                                      \\nCONECT 1669 1640 1668 1670                                                      \\nCONECT 1670 1669 1671 1673                                                      \\nCONECT 1671 1670 1672 1674                                                      \\nCONECT 1672 1637 1668 1671                                                      \\nCONECT 1673 1670                                                                \\nCONECT 1674 1671 1675                                                           \\nCONECT 1675 1674 1676                                                           \\nCONECT 1676 1675 1677 1678                                                      \\nCONECT 1677 1676                                                                \\nCONECT 1678 1152 1676                                                           \\nMASTER      287    0    2   10    0    0    9    9 1677    2   98   16          \\nEND                                                                             \\n\",\"pdb\");\n\tviewer_16742557823931737.setBackgroundColor(\"white\");\n\tviewer_16742557823931737.setStyle({\"cartoon\": {\"color\": \"white\"}});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"1.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"5.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"2.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"5.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"2.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"6.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"2.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"88.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"2.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"91.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"2.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"92.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"2.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"99.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"4.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"7.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"5.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"8.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"5.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"12.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"5.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"88.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"5.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"91.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"5.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"92.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"5.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"6.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"13.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"6.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"7.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"11.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"7.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"12.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"7.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"15.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"7.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"92.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"8.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"11.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"8.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"13.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"8.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"15.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"9.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"90.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"9.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"11.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"15.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"13.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"16.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"13.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"81.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"13.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"82.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"13.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"84.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"13.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"89.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"13.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"14.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"19.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"22.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"19.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"30.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"21.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"30.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"22.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"25.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"22.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"29.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"22.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"30.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"22.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"31.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"22.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"42.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"23.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"26.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"23.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"30.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"24.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"30.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"25.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"30.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"26.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"29.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"26.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"30.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"27.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"80.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"29.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"46.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"30.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"33.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"30.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"42.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"30.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"43.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"37.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"42.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"38.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"55.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"42.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"45.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"50.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"51.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"52.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"53.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"54.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"55.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"56.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"57.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"62.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"66.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"74.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"77.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"40.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"50.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"40.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"55.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"46.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"51.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"47.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"50.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"49.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"54.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"49.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"55.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"49.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"75.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"50.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"53.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"50.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"54.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"50.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"55.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"50.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"70.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"50.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"74.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"50.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"75.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"50.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"76.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"50.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"77.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"50.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"78.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"50.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"79.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"51.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"55.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"52.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"55.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"54.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"75.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"55.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"74.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"61.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"65.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"61.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"66.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"62.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"66.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"63.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"90.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"63.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"64.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"65.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"69.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"66.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"69.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"66.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"70.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"66.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"73.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"66.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"74.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"66.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"81.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"66.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"90.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"66.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"68.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"69.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"72.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"69.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"73.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"69.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"75.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"69.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"81.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"69.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"82.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"69.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"83.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"69.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"84.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"69.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"90.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"70.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"73.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"70.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"74.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"70.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"75.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"70.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"76.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"70.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"77.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"70.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"78.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"70.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"79.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"70.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"80.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"70.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"81.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"70.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"82.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"70.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"83.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"70.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"84.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"71.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"75.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"71.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"81.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"71.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"82.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"77.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"80.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"77.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"81.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"81.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"84.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"83.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"90.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"84.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"90.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"84.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"86.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"90.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"86.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"87.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"90.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"88.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"89.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"90.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"3.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"95.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"3.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"96.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"3.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"97.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"3.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"3.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"99.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"9.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"14.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"9.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"96.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"14.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"15.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"16.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"18.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"20.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"96.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"97.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"11.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"14.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"12.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"16.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"14.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"18.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"14.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"20.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"14.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"96.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"14.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"97.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"15.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"20.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"18.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"28.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"20.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"23.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"20.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"24.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"20.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"32.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"20.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"33.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"20.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"97.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"20.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"101.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"25.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"28.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"25.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"45.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"28.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"45.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"28.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"78.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"28.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"79.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"32.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"35.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"32.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"36.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"33.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"36.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"33.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"58.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"33.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"35.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"58.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"35.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"59.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"35.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"60.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"35.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"97.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"35.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"35.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"101.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"35.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"102.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"36.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"57.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"36.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"58.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"36.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"59.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"37.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"58.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"40.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"58.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"41.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"44.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"41.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"45.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"58.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"61.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"58.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"62.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"58.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"65.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"58.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"66.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"58.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"97.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"58.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"101.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"59.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"64.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"62.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"68.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"64.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"67.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"64.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"68.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"64.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"65.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"68.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"65.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"72.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"67.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"85.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"67.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"67.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"96.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"67.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"97.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"68.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"72.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"68.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"83.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"68.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"85.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"68.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"86.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"85.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"89.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"94.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"97.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"94.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"95.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"95.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"99.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"96.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"99.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"96.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"101.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"98.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"101.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742557823931737.zoomTo();\nviewer_16742557823931737.render();\n});\n</script>",
      "text/html": [
       "<div id=\"3dmolviewer_16742557823931737\"  style=\"position: relative; width: 640px; height: 480px\">\n",
       "        <p id=\"3dmolwarning_16742557823931737\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n",
       "        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n",
       "        </div>\n",
       "<script>\n",
       "\n",
       "var loadScriptAsync = function(uri){\n",
       "  return new Promise((resolve, reject) => {\n",
       "    //this is to ignore the existence of requirejs amd\n",
       "    var savedexports, savedmodule;\n",
       "    if (typeof exports !== 'undefined') savedexports = exports;\n",
       "    else exports = {}\n",
       "    if (typeof module !== 'undefined') savedmodule = module;\n",
       "    else module = {}\n",
       "\n",
       "    var tag = document.createElement('script');\n",
       "    tag.src = uri;\n",
       "    tag.async = true;\n",
       "    tag.onload = () => {\n",
       "        exports = savedexports;\n",
       "        module = savedmodule;\n",
       "        resolve();\n",
       "    };\n",
       "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
       "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
       "});\n",
       "};\n",
       "\n",
       "if(typeof $3Dmolpromise === 'undefined') {\n",
       "$3Dmolpromise = null;\n",
       "  $3Dmolpromise = loadScriptAsync('https://3dmol.org/build/3Dmol.js');\n",
       "}\n",
       "\n",
       "var viewer_16742557823931737 = null;\n",
       "var warn = document.getElementById(\"3dmolwarning_16742557823931737\");\n",
       "if(warn) {\n",
       "    warn.parentNode.removeChild(warn);\n",
       "}\n",
       "$3Dmolpromise.then(function() {\n",
       "viewer_16742557823931737 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_16742557823931737\"),{backgroundColor:\"white\"});\n",
       "viewer_16742557823931737.zoomTo();\n",
       "\tviewer_16742557823931737.addModel(\"HEADER    ELECTRON TRANSPORT                      01-AUG-76   1CYC              \\nTITLE     THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C AT 2.3     \\nTITLE    2 ANGSTROMS RESOLUTION. II. STRUCTURE AND FUNCTION                     \\nCAVEAT     1CYC    THR A 40 HAS WRONG CHIRALITY AT ATOM CB THR A 78 HAS WRONG   \\nCAVEAT   2 1CYC    CHIRALITY AT ATOM CB THR B 40 HAS WRONG CHIRALITY AT ATOM    \\nCAVEAT   3 1CYC    CB THR B 78 HAS WRONG CHIRALITY AT ATOM CB                   \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: FERROCYTOCHROME C;                                         \\nCOMPND   3 CHAIN: A, B;                                                         \\nCOMPND   4 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 ORGANISM_SCIENTIFIC: KATSUWONUS PELAMIS;                             \\nSOURCE   3 ORGANISM_COMMON: SKIPJACK TUNA;                                      \\nSOURCE   4 ORGANISM_TAXID: 8226                                                 \\nKEYWDS    ELECTRON TRANSPORT                                                    \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    N.TANAKA,T.YAMANE,T.TSUKIHARA,T.ASHIDA,M.KAKUDO                       \\nREVDAT  17   03-MAR-21 1CYC    1       CAVEAT COMPND REMARK HET                 \\nREVDAT  17 2                   1       HETNAM HETSYN FORMUL LINK                \\nREVDAT  17 3                   1       ATOM                                     \\nREVDAT  16   29-NOV-17 1CYC    1       HELIX                                    \\nREVDAT  15   24-FEB-09 1CYC    1       VERSN                                    \\nREVDAT  14   01-APR-03 1CYC    1       JRNL                                     \\nREVDAT  13   27-JAN-84 1CYC    1       REMARK                                   \\nREVDAT  12   30-SEP-83 1CYC    1       REVDAT                                   \\nREVDAT  11   19-APR-83 1CYC    1       COMPND SOURCE REMARK                     \\nREVDAT  10   15-JAN-82 1CYC    1       REMARK                                   \\nREVDAT   9   06-JAN-82 1CYC    3       SCALE                                    \\nREVDAT   8   31-DEC-80 1CYC    1       REMARK                                   \\nREVDAT   7   07-APR-80 1CYC    3       SOURCE SEQRES ATOM                       \\nREVDAT   6   23-APR-79 1CYC    3       ATOM                                     \\nREVDAT   5   01-NOV-77 1CYC    1       AUTHOR REMARK FORMUL                     \\nREVDAT   4   09-SEP-77 1CYC    3       ATOM                                     \\nREVDAT   3   13-JUN-77 1CYC    1       HET                                      \\nREVDAT   2   25-OCT-76 1CYC    2       CONECT                                   \\nREVDAT   1   06-OCT-76 1CYC    0                                                \\nJRNL        AUTH   N.TANAKA,T.YAMANE,T.TSUKIHARA,T.ASHIDA,M.KAKUDO              \\nJRNL        TITL   THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C   \\nJRNL        TITL 2 AT 2.3 A RESOLUTION. II. STRUCTURE AND FUNCTION.             \\nJRNL        REF    J.BIOCHEM.(TOKYO)             V.  77   147 1975              \\nJRNL        REFN                   ISSN 0021-924X                               \\nJRNL        PMID   166072                                                       \\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   T.TSUKIHARA,T.YAMANE,N.TANAKA,T.ASHIDA,M.KAKUDO              \\nREMARK   1  TITL   OXIDATION OF A FERROCYTOCHROME C IN THE CRYSTALLINE          \\nREMARK   1  TITL 2 STATE-STRUCTURAL CHANGE AND ANION BINDING                    \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  73  1163 1973              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 2                                                          \\nREMARK   1  AUTH   T.ASHIDA,N.TANAKA,T.YAMANE,T.TSUKIHARA,M.KAKUDO              \\nREMARK   1  TITL   THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C   \\nREMARK   1  TITL 2 AT 2.3 ANGSTROMS RESOLUTION                                  \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  73   463 1973              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 3                                                          \\nREMARK   1  AUTH   T.ASHIDA,T.UEKI,T.TSUKIHARA,A.SUGIHARA,T.TAKANO,M.KAKUDO     \\nREMARK   1  TITL   THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C   \\nREMARK   1  TITL 2 AT 4 ANGSTROMS RESOLUTION                                    \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  70   913 1971              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 4                                                          \\nREMARK   1  AUTH   T.NAKAYAMA,K.TITANI,K.NARITA                                 \\nREMARK   1  TITL   THE AMINO ACID SEQUENCE OF CYTOCHROME C FROM BONITO          \\nREMARK   1  TITL 2 (KATSUWONUS PELAMIS, LINNAEUS)                               \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  70   311 1971              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 5                                                          \\nREMARK   1  EDIT   R.J.FELDMANN                                                 \\nREMARK   1  REF    ATLAS OF MACROMOLECULAR                120 1976              \\nREMARK   1  REF  2 STRUCTURE ON MICROFICHE                                      \\nREMARK   1  PUBL   TRACOR JITCO INC.,ROCKVILLE,MD.                              \\nREMARK   1  REFN                                                                \\nREMARK   1 REFERENCE 6                                                          \\nREMARK   1  EDIT   M.O.DAYHOFF                                                  \\nREMARK   1  REF    ATLAS OF PROTEIN SEQUENCE     V.   5    39 1976              \\nREMARK   1  REF  2 AND STRUCTURE,SUPPLEMENT 2                                   \\nREMARK   1  PUBL   NATIONAL BIOMEDICAL RESEARCH FOUNDATION, SILVER SPRING,MD.   \\nREMARK   1  REFN                   ISSN 0-912466-05-7                           \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    2.30 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : NULL                                                 \\nREMARK   3   AUTHORS     : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 2.30                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : NULL                           \\nREMARK   3   DATA CUTOFF            (SIGMA(F)) : NULL                           \\nREMARK   3   DATA CUTOFF HIGH         (ABS(F)) : NULL                           \\nREMARK   3   DATA CUTOFF LOW          (ABS(F)) : NULL                           \\nREMARK   3   COMPLETENESS (WORKING+TEST)   (%) : NULL                           \\nREMARK   3   NUMBER OF REFLECTIONS             : NULL                           \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   CROSS-VALIDATION METHOD          : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SELECTION  : NULL                            \\nREMARK   3   R VALUE            (WORKING SET) : NULL                            \\nREMARK   3   FREE R VALUE                     : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : NULL                            \\nREMARK   3   FREE R VALUE TEST SET COUNT      : NULL                            \\nREMARK   3   ESTIMATED ERROR OF FREE R VALUE  : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  FIT IN THE HIGHEST RESOLUTION BIN.                                  \\nREMARK   3   TOTAL NUMBER OF BINS USED           : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE HIGH       (A) : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE LOW        (A) : NULL                         \\nREMARK   3   BIN COMPLETENESS (WORKING+TEST) (%) : NULL                         \\nREMARK   3   REFLECTIONS IN BIN    (WORKING SET) : NULL                         \\nREMARK   3   BIN R VALUE           (WORKING SET) : NULL                         \\nREMARK   3   BIN FREE R VALUE                    : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET SIZE  (%) : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET COUNT     : NULL                         \\nREMARK   3   ESTIMATED ERROR OF BIN FREE R VALUE : NULL                         \\nREMARK   3                                                                      \\nREMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \\nREMARK   3   PROTEIN ATOMS            : 1590                                    \\nREMARK   3   NUCLEIC ACID ATOMS       : 0                                       \\nREMARK   3   HETEROGEN ATOMS          : 86                                      \\nREMARK   3   SOLVENT ATOMS            : 1                                       \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : NULL                           \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : NULL                           \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : NULL                                                 \\nREMARK   3    B22 (A**2) : NULL                                                 \\nREMARK   3    B33 (A**2) : NULL                                                 \\nREMARK   3    B12 (A**2) : NULL                                                 \\nREMARK   3    B13 (A**2) : NULL                                                 \\nREMARK   3    B23 (A**2) : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  ESTIMATED COORDINATE ERROR.                                         \\nREMARK   3   ESD FROM LUZZATI PLOT        (A) : NULL                            \\nREMARK   3   ESD FROM SIGMAA              (A) : NULL                            \\nREMARK   3   LOW RESOLUTION CUTOFF        (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  CROSS-VALIDATED ESTIMATED COORDINATE ERROR.                         \\nREMARK   3   ESD FROM C-V LUZZATI PLOT    (A) : NULL                            \\nREMARK   3   ESD FROM C-V SIGMAA          (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  RMS DEVIATIONS FROM IDEAL VALUES.                                   \\nREMARK   3   BOND LENGTHS                 (A) : NULL                            \\nREMARK   3   BOND ANGLES            (DEGREES) : NULL                            \\nREMARK   3   DIHEDRAL ANGLES        (DEGREES) : NULL                            \\nREMARK   3   IMPROPER ANGLES        (DEGREES) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL MODEL : NULL                                      \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL FACTOR RESTRAINTS.    RMS    SIGMA                \\nREMARK   3   MAIN-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   MAIN-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  NCS MODEL : NULL                                                    \\nREMARK   3                                                                      \\nREMARK   3  NCS RESTRAINTS.                         RMS   SIGMA/WEIGHT          \\nREMARK   3   GROUP  1  POSITIONAL            (A) : NULL  ; NULL                 \\nREMARK   3   GROUP  1  B-FACTOR           (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  PARAMETER FILE  1  : NULL                                           \\nREMARK   3  TOPOLOGY FILE  1   : NULL                                           \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS: NULL                                      \\nREMARK   4                                                                      \\nREMARK   4 1CYC COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY BNL.                                \\nREMARK 100 THE DEPOSITION ID IS D_1000172595.                                   \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : NULL                               \\nREMARK 200  TEMPERATURE           (KELVIN) : NULL                               \\nREMARK 200  PH                             : NULL                               \\nREMARK 200  NUMBER OF CRYSTALS USED        : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : NULL                               \\nREMARK 200  RADIATION SOURCE               : NULL                               \\nREMARK 200  BEAMLINE                       : NULL                               \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : NULL                               \\nREMARK 200  WAVELENGTH OR RANGE        (A) : NULL                               \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : NULL                               \\nREMARK 200  DETECTOR MANUFACTURER          : NULL                               \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : NULL                               \\nREMARK 200  DATA SCALING SOFTWARE          : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : NULL                               \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : NULL                               \\nREMARK 200  RESOLUTION RANGE LOW       (A) : NULL                               \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY                : NULL                               \\nREMARK 200  R MERGE                    (I) : NULL                               \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : NULL                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : NULL                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : NULL                               \\nREMARK 200  R MERGE FOR SHELL          (I) : NULL                               \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: NULL                                           \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: NULL                         \\nREMARK 200 SOFTWARE USED: NULL                                                  \\nREMARK 200 STARTING MODEL: NULL                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: NULL                                                         \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): 39.18                                     \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 2.02                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: NULL                                     \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: P 21 21 21                       \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -X+1/2,-Y,Z+1/2                                         \\nREMARK 290       3555   -X,Y+1/2,-Z+1/2                                         \\nREMARK 290       4555   X+1/2,-Y+1/2,-Z                                         \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -1.000000  0.000000  0.000000       28.84000            \\nREMARK 290   SMTRY2   2  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000       18.91500            \\nREMARK 290   SMTRY1   3 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   3  0.000000  1.000000  0.000000       42.29000            \\nREMARK 290   SMTRY3   3  0.000000  0.000000 -1.000000       18.91500            \\nREMARK 290   SMTRY1   4  1.000000  0.000000  0.000000       28.84000            \\nREMARK 290   SMTRY2   4  0.000000 -1.000000  0.000000       42.29000            \\nREMARK 290   SMTRY3   4  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1, 2                                                    \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: MONOMERIC                         \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 2                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: MONOMERIC                         \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: B                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 470                                                                      \\nREMARK 470 MISSING ATOM                                                         \\nREMARK 470 THE FOLLOWING RESIDUES HAVE MISSING ATOMS (M=MODEL NUMBER;           \\nREMARK 470 RES=RESIDUE NAME; C=CHAIN IDENTIFIER; SSEQ=SEQUENCE NUMBER;          \\nREMARK 470 I=INSERTION CODE):                                                   \\nREMARK 470   M RES CSSEQI  ATOMS                                                \\nREMARK 470     ASN A  22    CB   CG   OD1  ND2                                  \\nREMARK 470     LYS A  39    CE   NZ                                             \\nREMARK 470     ASN B  22    CB   CG   OD1  ND2                                  \\nREMARK 470     LYS B  39    CE   NZ                                             \\nREMARK 620                                                                      \\nREMARK 620 METAL COORDINATION                                                   \\nREMARK 620 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 620 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE):                             \\nREMARK 620                                                                      \\nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \\nREMARK 620                             HEC A 104  FE                            \\nREMARK 620 N RES CSSEQI ATOM                                                    \\nREMARK 620 1 HIS A  18   NE2                                                    \\nREMARK 620 2 HEC A 104   NA   87.0                                              \\nREMARK 620 3 HEC A 104   NB   87.8  90.6                                        \\nREMARK 620 4 HEC A 104   NC   92.0 178.7  90.1                                  \\nREMARK 620 5 HEC A 104   ND   91.1  86.9 177.4  92.3                            \\nREMARK 620 6 MET A  80   SD  170.9 102.1  93.0  79.0  88.6                      \\nREMARK 620 N                    1     2     3     4     5                       \\nREMARK 620                                                                      \\nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \\nREMARK 620                             HEC B 104  FE                            \\nREMARK 620 N RES CSSEQI ATOM                                                    \\nREMARK 620 1 HIS B  18   NE2                                                    \\nREMARK 620 2 HEC B 104   NA   87.0                                              \\nREMARK 620 3 HEC B 104   NB   87.8  90.6                                        \\nREMARK 620 4 HEC B 104   NC   92.0 178.7  90.1                                  \\nREMARK 620 5 HEC B 104   ND   91.1  86.9 177.4  92.3                            \\nREMARK 620 6 MET B  80   SD  170.9 102.1  93.0  79.0  88.6                      \\nREMARK 620 N                    1     2     3     4     5                       \\nREMARK 800                                                                      \\nREMARK 800 SITE                                                                 \\nREMARK 800 SITE_IDENTIFIER: AC1                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE HEC A 104                 \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: AC2                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE HEC B 104                 \\nDBREF  1CYC A    1   103  UNP    P00025   CYC_KATPE        1    103             \\nDBREF  1CYC B    1   103  UNP    P00025   CYC_KATPE        1    103             \\nSEQRES   1 A  103  GLY ASP VAL ALA LYS GLY LYS LYS THR PHE VAL GLN LYS          \\nSEQRES   2 A  103  CYS ALA GLN CYS HIS THR VAL GLU ASN GLY GLY LYS HIS          \\nSEQRES   3 A  103  LYS VAL GLY PRO ASN LEU TRP GLY LEU PHE GLY ARG LYS          \\nSEQRES   4 A  103  THR GLY GLN ALA GLU GLY TYR SER TYR THR ASP ALA ASN          \\nSEQRES   5 A  103  LYS SER LYS GLY ILE VAL TRP ASN GLU ASN THR LEU MET          \\nSEQRES   6 A  103  GLU TYR LEU GLU ASN PRO LYS LYS TYR ILE PRO GLY THR          \\nSEQRES   7 A  103  LYS MET ILE PHE ALA GLY ILE LYS LYS LYS GLY GLU ARG          \\nSEQRES   8 A  103  GLN ASP LEU VAL ALA TYR LEU LYS SER ALA THR SER              \\nSEQRES   1 B  103  GLY ASP VAL ALA LYS GLY LYS LYS THR PHE VAL GLN LYS          \\nSEQRES   2 B  103  CYS ALA GLN CYS HIS THR VAL GLU ASN GLY GLY LYS HIS          \\nSEQRES   3 B  103  LYS VAL GLY PRO ASN LEU TRP GLY LEU PHE GLY ARG LYS          \\nSEQRES   4 B  103  THR GLY GLN ALA GLU GLY TYR SER TYR THR ASP ALA ASN          \\nSEQRES   5 B  103  LYS SER LYS GLY ILE VAL TRP ASN GLU ASN THR LEU MET          \\nSEQRES   6 B  103  GLU TYR LEU GLU ASN PRO LYS LYS TYR ILE PRO GLY THR          \\nSEQRES   7 B  103  LYS MET ILE PHE ALA GLY ILE LYS LYS LYS GLY GLU ARG          \\nSEQRES   8 B  103  GLN ASP LEU VAL ALA TYR LEU LYS SER ALA THR SER              \\nHET    HEC  A 104      43                                                       \\nHET    HEC  B 104      43                                                       \\nHETNAM     HEC HEME C                                                           \\nFORMUL   3  HEC    2(C34 H34 FE N4 O4)                                          \\nFORMUL   5  HOH   *(H2 O)                                                       \\nHELIX    1  H1 GLY A    1  VAL A   11  1                                  11    \\nHELIX    2  H4 CYS A   14  CYS A   17  514 AND 17 BOUND TO HEME GROUP      4    \\nHELIX    3  H5 THR A   49  LYS A   55  5LOOSE FROM 49-53,3/10 53-55        7    \\nHELIX    4  H2 ASN A   60  GLU A   69  1                                  10    \\nHELIX    5  H3 GLU A   90  SER A  103  1                                  14    \\nHELIX    6  H6 GLY B    1  VAL B   11  1                                  11    \\nHELIX    7  H7 CYS B   14  CYS B   17  514 AND 17 BOUND TO HEME GROUP      4    \\nHELIX    8  H8 THR B   49  LYS B   55  5LOOSE FROM 49-53,3/10 53-55        7    \\nHELIX    9  H9 ASN B   60  GLU B   69  1                                  10    \\nHELIX   10 H10 GLU B   90  SER B  103  1                                  14    \\nLINK         SG  CYS A  14                 CAB HEC A 104     1555   1555  1.48  \\nLINK         SG  CYS A  17                 CAC HEC A 104     1555   1555  1.86  \\nLINK         CE2 TYR A  48                 O2D HEC A 104     1555   1555  2.02  \\nLINK         SG  CYS B  14                 CAB HEC B 104     1555   1555  1.48  \\nLINK         SG  CYS B  17                 CAC HEC B 104     1555   1555  1.86  \\nLINK         CE2 TYR B  48                 O2D HEC B 104     1555   1555  2.02  \\nLINK         NE2 HIS A  18                FE   HEC A 104     1555   1555  2.60  \\nLINK         SD  MET A  80                FE   HEC A 104     1555   1555  2.49  \\nLINK         NE2 HIS B  18                FE   HEC B 104     1555   1555  2.60  \\nLINK         SD  MET B  80                FE   HEC B 104     1555   1555  2.49  \\nSITE     1 AC1 17 LYS A  13  CYS A  14  CYS A  17  HIS A  18                    \\nSITE     2 AC1 17 LEU A  32  LEU A  35  ARG A  38  THR A  40                    \\nSITE     3 AC1 17 TYR A  46  TYR A  48  ASN A  52  TRP A  59                    \\nSITE     4 AC1 17 THR A  78  LYS A  79  MET A  80  PHE A  82                    \\nSITE     5 AC1 17 HOH A 105                                                     \\nSITE     1 AC2 16 LYS B  13  CYS B  14  CYS B  17  HIS B  18                    \\nSITE     2 AC2 16 LEU B  32  LEU B  35  ARG B  38  THR B  40                    \\nSITE     3 AC2 16 TYR B  46  TYR B  48  ASN B  52  TRP B  59                    \\nSITE     4 AC2 16 THR B  78  LYS B  79  MET B  80  PHE B  82                    \\nCRYST1   57.680   84.580   37.830  90.00  90.00  90.00 P 21 21 21    8          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.017337  0.000000  0.000000        0.00000                         \\nSCALE2      0.000000  0.011823  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.026434        0.00000                         \\nMTRIX1   1  1.000000  0.000000  0.000000       28.84000    1                    \\nMTRIX2   1  0.000000  1.000000  0.000000        0.00000    1                    \\nMTRIX3   1  0.000000  0.000000  1.000000        0.00000    1                    \\nATOM      1  N   GLY A   1     -21.138  13.774  -7.711  1.00  0.00           N  \\nATOM      2  CA  GLY A   1     -19.878  14.593  -7.963  1.00  0.00           C  \\nATOM      3  C   GLY A   1     -18.996  14.530  -6.703  1.00  0.00           C  \\nATOM      4  O   GLY A   1     -18.114  13.711  -6.640  1.00  0.00           O  \\nATOM      5  N   ASP A   2     -18.996  15.412  -5.821  1.00  0.00           N  \\nATOM      6  CA  ASP A   2     -17.610  15.097  -5.191  1.00  0.00           C  \\nATOM      7  C   ASP A   2     -17.862  14.152  -3.931  1.00  0.00           C  \\nATOM      8  O   ASP A   2     -17.106  13.333  -3.553  1.00  0.00           O  \\nATOM      9  CB  ASP A   2     -17.169  16.546  -4.435  1.00  0.00           C  \\nATOM     10  CG  ASP A   2     -17.988  17.365  -3.427  1.00  0.00           C  \\nATOM     11  OD1 ASP A   2     -19.500  17.365  -3.994  1.00  0.00           O  \\nATOM     12  OD2 ASP A   2     -17.484  18.373  -2.293  1.00  0.00           O  \\nATOM     13  N   VAL A   3     -19.059  14.089  -3.679  1.00  0.00           N  \\nATOM     14  CA  VAL A   3     -19.815  12.955  -2.860  1.00  0.00           C  \\nATOM     15  C   VAL A   3     -19.374  11.506  -3.805  1.00  0.00           C  \\nATOM     16  O   VAL A   3     -18.870  10.624  -3.427  1.00  0.00           O  \\nATOM     17  CB  VAL A   3     -22.587  13.270  -3.175  1.00  0.00           C  \\nATOM     18  CG1 VAL A   3     -21.957  11.947  -3.427  1.00  0.00           C  \\nATOM     19  CG2 VAL A   3     -21.705  13.963  -2.167  1.00  0.00           C  \\nATOM     20  N   ALA A   4     -19.437  11.380  -5.002  1.00  0.00           N  \\nATOM     21  CA  ALA A   4     -18.681  10.120  -5.569  1.00  0.00           C  \\nATOM     22  C   ALA A   4     -17.358   9.805  -4.939  1.00  0.00           C  \\nATOM     23  O   ALA A   4     -17.295   8.923  -4.183  1.00  0.00           O  \\nATOM     24  CB  ALA A   4     -18.429  10.435  -7.144  1.00  0.00           C  \\nATOM     25  N   LYS A   5     -16.476  10.687  -5.191  1.00  0.00           N  \\nATOM     26  CA  LYS A   5     -15.216  10.498  -4.372  1.00  0.00           C  \\nATOM     27  C   LYS A   5     -15.279  10.246  -2.797  1.00  0.00           C  \\nATOM     28  O   LYS A   5     -14.838   9.238  -2.293  1.00  0.00           O  \\nATOM     29  CB  LYS A   5     -14.208  11.695  -4.246  1.00  0.00           C  \\nATOM     30  CG  LYS A   5     -12.948  11.758  -3.553  1.00  0.00           C  \\nATOM     31  CD  LYS A   5     -11.814  13.081  -3.679  1.00  0.00           C  \\nATOM     32  CE  LYS A   5     -10.806  13.144  -2.545  1.00  0.00           C  \\nATOM     33  NZ  LYS A   5      -9.735  14.404  -2.293  1.00  0.00           N  \\nATOM     34  N   GLY A   6     -16.161  10.750  -1.915  1.00  0.00           N  \\nATOM     35  CA  GLY A   6     -16.350  10.435  -0.403  1.00  0.00           C  \\nATOM     36  C   GLY A   6     -16.539   8.923  -0.655  1.00  0.00           C  \\nATOM     37  O   GLY A   6     -15.909   8.041  -0.025  1.00  0.00           O  \\nATOM     38  N   LYS A   7     -17.421   8.419  -1.411  1.00  0.00           N  \\nATOM     39  CA  LYS A   7     -17.799   6.844  -1.033  1.00  0.00           C  \\nATOM     40  C   LYS A   7     -16.476   6.151  -1.474  1.00  0.00           C  \\nATOM     41  O   LYS A   7     -15.972   4.954  -0.970  1.00  0.00           O  \\nATOM     42  CB  LYS A   7     -18.870   6.403  -2.293  1.00  0.00           C  \\nATOM     43  CG  LYS A   7     -18.870   5.017  -2.734  1.00  0.00           C  \\nATOM     44  CD  LYS A   7     -20.193   4.765  -3.427  1.00  0.00           C  \\nATOM     45  CE  LYS A   7     -20.193   3.190  -4.057  1.00  0.00           C  \\nATOM     46  NZ  LYS A   7     -21.516   2.623  -4.309  1.00  0.00           N  \\nATOM     47  N   LYS A   8     -15.846   6.403  -2.545  1.00  0.00           N  \\nATOM     48  CA  LYS A   8     -14.460   5.899  -2.923  1.00  0.00           C  \\nATOM     49  C   LYS A   8     -13.452   5.899  -1.663  1.00  0.00           C  \\nATOM     50  O   LYS A   8     -13.011   5.017  -1.348  1.00  0.00           O  \\nATOM     51  CB  LYS A   8     -13.704   6.844  -3.931  1.00  0.00           C  \\nATOM     52  CG  LYS A   8     -12.318   6.529  -4.183  1.00  0.00           C  \\nATOM     53  CD  LYS A   8     -11.562   7.537  -5.065  1.00  0.00           C  \\nATOM     54  CE  LYS A   8     -10.176   7.222  -5.821  1.00  0.00           C  \\nATOM     55  NZ  LYS A   8      -9.357   8.419  -6.325  1.00  0.00           N  \\nATOM     56  N   THR A   9     -13.452   6.970  -1.159  1.00  0.00           N  \\nATOM     57  CA  THR A   9     -12.633   7.033   0.227  1.00  0.00           C  \\nATOM     58  C   THR A   9     -13.515   6.277   1.298  1.00  0.00           C  \\nATOM     59  O   THR A   9     -12.948   5.773   2.369  1.00  0.00           O  \\nATOM     60  CB  THR A   9     -12.381   8.419   0.920  1.00  0.00           C  \\nATOM     61  OG1 THR A   9     -11.058   8.923   0.542  1.00  0.00           O  \\nATOM     62  CG2 THR A   9     -12.444   8.734   2.558  1.00  0.00           C  \\nATOM     63  N   PHE A  10     -14.775   6.403   1.235  1.00  0.00           N  \\nATOM     64  CA  PHE A  10     -15.531   5.647   2.117  1.00  0.00           C  \\nATOM     65  C   PHE A  10     -15.216   4.135   1.739  1.00  0.00           C  \\nATOM     66  O   PHE A  10     -14.460   3.757   2.495  1.00  0.00           O  \\nATOM     67  CB  PHE A  10     -17.106   5.647   1.991  1.00  0.00           C  \\nATOM     68  CG  PHE A  10     -17.736   4.891   3.251  1.00  0.00           C  \\nATOM     69  CD1 PHE A  10     -18.492   3.757   2.747  1.00  0.00           C  \\nATOM     70  CD2 PHE A  10     -17.862   5.143   4.133  1.00  0.00           C  \\nATOM     71  CE1 PHE A  10     -18.996   3.001   3.818  1.00  0.00           C  \\nATOM     72  CE2 PHE A  10     -18.366   4.513   4.889  1.00  0.00           C  \\nATOM     73  CZ  PHE A  10     -18.996   3.379   5.015  1.00  0.00           C  \\nATOM     74  N   VAL A  11     -15.216   3.379   0.479  1.00  0.00           N  \\nATOM     75  CA  VAL A  11     -14.964   2.056   0.605  1.00  0.00           C  \\nATOM     76  C   VAL A  11     -13.326   1.867   1.109  1.00  0.00           C  \\nATOM     77  O   VAL A  11     -12.948   0.985   1.865  1.00  0.00           O  \\nATOM     78  CB  VAL A  11     -15.153   1.237  -0.529  1.00  0.00           C  \\nATOM     79  CG1 VAL A  11     -16.098   1.930  -1.537  1.00  0.00           C  \\nATOM     80  CG2 VAL A  11     -13.956   1.111  -1.285  1.00  0.00           C  \\nATOM     81  N   GLN A  12     -12.570   2.749   0.794  1.00  0.00           N  \\nATOM     82  CA  GLN A  12     -10.995   2.749   1.235  1.00  0.00           C  \\nATOM     83  C   GLN A  12     -10.680   2.938   2.873  1.00  0.00           C  \\nATOM     84  O   GLN A  12     -10.176   1.930   3.377  1.00  0.00           O  \\nATOM     85  CB  GLN A  12     -10.428   3.820   0.479  1.00  0.00           C  \\nATOM     86  CG  GLN A  12      -9.861   3.253  -0.718  1.00  0.00           C  \\nATOM     87  CD  GLN A  12      -9.168   4.135  -2.293  1.00  0.00           C  \\nATOM     88  OE1 GLN A  12      -7.593   3.820  -2.041  1.00  0.00           O  \\nATOM     89  NE2 GLN A  12      -9.672   5.143  -2.671  1.00  0.00           N  \\nATOM     90  N   LYS A  13     -10.995   3.757   3.629  1.00  0.00           N  \\nATOM     91  CA  LYS A  13     -10.869   3.757   5.015  1.00  0.00           C  \\nATOM     92  C   LYS A  13     -12.066   3.694   5.960  1.00  0.00           C  \\nATOM     93  O   LYS A  13     -12.003   3.127   6.779  1.00  0.00           O  \\nATOM     94  CB  LYS A  13     -10.239   5.017   5.141  1.00  0.00           C  \\nATOM     95  CG  LYS A  13      -9.672   5.521   3.881  1.00  0.00           C  \\nATOM     96  CD  LYS A  13      -8.286   6.277   4.007  1.00  0.00           C  \\nATOM     97  CE  LYS A  13      -8.160   6.907   2.621  1.00  0.00           C  \\nATOM     98  NZ  LYS A  13      -6.837   7.789   2.243  1.00  0.00           N  \\nATOM     99  N   CYS A  14     -13.137   4.072   5.519  1.00  0.00           N  \\nATOM    100  CA  CYS A  14     -14.460   4.072   6.464  1.00  0.00           C  \\nATOM    101  C   CYS A  14     -15.216   2.812   5.897  1.00  0.00           C  \\nATOM    102  O   CYS A  14     -15.342   1.804   6.716  1.00  0.00           O  \\nATOM    103  CB  CYS A  14     -15.468   5.332   5.897  1.00  0.00           C  \\nATOM    104  SG  CYS A  14     -14.712   6.844   6.464  1.00  0.00           S  \\nATOM    105  N   ALA A  15     -15.720   2.623   4.637  1.00  0.00           N  \\nATOM    106  CA  ALA A  15     -16.161   1.237   4.322  1.00  0.00           C  \\nATOM    107  C   ALA A  15     -16.035  -0.086   5.015  1.00  0.00           C  \\nATOM    108  O   ALA A  15     -17.043  -0.590   5.393  1.00  0.00           O  \\nATOM    109  CB  ALA A  15     -15.405   0.985   3.062  1.00  0.00           C  \\nATOM    110  N   GLN A  16     -14.964  -0.653   4.889  1.00  0.00           N  \\nATOM    111  CA  GLN A  16     -14.901  -2.039   5.897  1.00  0.00           C  \\nATOM    112  C   GLN A  16     -15.090  -1.787   7.283  1.00  0.00           C  \\nATOM    113  O   GLN A  16     -14.838  -2.669   8.291  1.00  0.00           O  \\nATOM    114  CB  GLN A  16     -13.578  -2.039   5.519  1.00  0.00           C  \\nATOM    115  CG  GLN A  16     -12.318  -1.724   6.275  1.00  0.00           C  \\nATOM    116  CD  GLN A  16     -11.625  -0.842   5.015  1.00  0.00           C  \\nATOM    117  OE1 GLN A  16     -10.365  -0.149   5.330  1.00  0.00           O  \\nATOM    118  NE2 GLN A  16     -12.444  -0.464   4.385  1.00  0.00           N  \\nATOM    119  N   CYS A  17     -15.216  -0.653   7.787  1.00  0.00           N  \\nATOM    120  CA  CYS A  17     -15.405  -0.905   9.299  1.00  0.00           C  \\nATOM    121  C   CYS A  17     -16.917  -0.401   9.173  1.00  0.00           C  \\nATOM    122  O   CYS A  17     -17.736  -1.283   9.173  1.00  0.00           O  \\nATOM    123  CB  CYS A  17     -14.460   0.355  10.055  1.00  0.00           C  \\nATOM    124  SG  CYS A  17     -13.326  -0.338  11.441  1.00  0.00           S  \\nATOM    125  N   HIS A  18     -17.232   0.733   9.551  1.00  0.00           N  \\nATOM    126  CA  HIS A  18     -18.870   0.985   9.803  1.00  0.00           C  \\nATOM    127  C   HIS A  18     -19.878   0.229   8.795  1.00  0.00           C  \\nATOM    128  O   HIS A  18     -19.626  -0.023   7.472  1.00  0.00           O  \\nATOM    129  CB  HIS A  18     -19.185   2.245   9.173  1.00  0.00           C  \\nATOM    130  CG  HIS A  18     -18.681   3.127  10.181  1.00  0.00           C  \\nATOM    131  ND1 HIS A  18     -18.618   3.253  11.567  1.00  0.00           N  \\nATOM    132  CD2 HIS A  18     -17.736   4.072  10.559  1.00  0.00           C  \\nATOM    133  CE1 HIS A  18     -18.051   3.694  12.827  1.00  0.00           C  \\nATOM    134  NE2 HIS A  18     -17.232   4.387  11.819  1.00  0.00           N  \\nATOM    135  N   THR A  19     -21.012   0.103   9.299  1.00  0.00           N  \\nATOM    136  CA  THR A  19     -22.272  -0.401   8.417  1.00  0.00           C  \\nATOM    137  C   THR A  19     -22.902   0.922   7.661  1.00  0.00           C  \\nATOM    138  O   THR A  19     -22.272   1.867   7.157  1.00  0.00           O  \\nATOM    139  CB  THR A  19     -23.280  -1.283   8.795  1.00  0.00           C  \\nATOM    140  OG1 THR A  19     -23.532  -0.905   9.740  1.00  0.00           O  \\nATOM    141  CG2 THR A  19     -22.776  -2.669   8.543  1.00  0.00           C  \\nATOM    142  N   VAL A  20     -24.036   1.111   7.472  1.00  0.00           N  \\nATOM    143  CA  VAL A  20     -24.414   2.371   6.779  1.00  0.00           C  \\nATOM    144  C   VAL A  20     -26.052   2.434   6.401  1.00  0.00           C  \\nATOM    145  O   VAL A  20     -26.556   3.442   5.771  1.00  0.00           O  \\nATOM    146  CB  VAL A  20     -23.784   2.560   5.519  1.00  0.00           C  \\nATOM    147  CG1 VAL A  20     -24.540   2.119   4.637  1.00  0.00           C  \\nATOM    148  CG2 VAL A  20     -23.658   4.009   5.267  1.00  0.00           C  \\nATOM    149  N   GLU A  21     -26.682   1.363   6.527  1.00  0.00           N  \\nATOM    150  CA  GLU A  21     -28.194   1.363   5.897  1.00  0.00           C  \\nATOM    151  C   GLU A  21     -29.013   1.741   7.283  1.00  0.00           C  \\nATOM    152  O   GLU A  21     -28.635   2.623   7.913  1.00  0.00           O  \\nATOM    153  CB  GLU A  21     -28.572  -0.086   5.645  1.00  0.00           C  \\nATOM    154  CG  GLU A  21     -28.698  -0.779   4.133  1.00  0.00           C  \\nATOM    155  CD  GLU A  21     -27.690  -1.913   4.007  1.00  0.00           C  \\nATOM    156  OE1 GLU A  21     -26.682  -2.039   5.141  1.00  0.00           O  \\nATOM    157  OE2 GLU A  21     -27.753  -2.921   2.873  1.00  0.00           O  \\nATOM    158  N   ASN A  22     -30.210   1.363   7.283  1.00  0.00           N  \\nATOM    159  CA  ASN A  22     -30.966   1.867   8.417  1.00  0.00           C  \\nATOM    160  C   ASN A  22     -30.714   1.237   9.677  1.00  0.00           C  \\nATOM    161  O   ASN A  22     -31.218   0.103  10.055  1.00  0.00           O  \\nATOM    162  N   GLY A  23     -29.958   1.552  10.433  1.00  0.00           N  \\nATOM    163  CA  GLY A  23     -29.517   0.859  11.819  1.00  0.00           C  \\nATOM    164  C   GLY A  23     -28.572  -0.338  11.252  1.00  0.00           C  \\nATOM    165  O   GLY A  23     -28.194  -1.283  12.071  1.00  0.00           O  \\nATOM    166  N   GLY A  24     -28.068  -0.149   9.929  1.00  0.00           N  \\nATOM    167  CA  GLY A  24     -27.060  -1.409   9.047  1.00  0.00           C  \\nATOM    168  C   GLY A  24     -26.871  -2.480  10.118  1.00  0.00           C  \\nATOM    169  O   GLY A  24     -27.186  -3.803   9.929  1.00  0.00           O  \\nATOM    170  N   LYS A  25     -26.304  -2.165  11.441  1.00  0.00           N  \\nATOM    171  CA  LYS A  25     -26.241  -3.047  12.701  1.00  0.00           C  \\nATOM    172  C   LYS A  25     -24.792  -2.795  13.079  1.00  0.00           C  \\nATOM    173  O   LYS A  25     -23.910  -2.417  12.323  1.00  0.00           O  \\nATOM    174  CB  LYS A  25     -26.241  -4.433  11.819  1.00  0.00           C  \\nATOM    175  CG  LYS A  25     -25.296  -4.496  10.433  1.00  0.00           C  \\nATOM    176  CD  LYS A  25     -24.918  -5.819   9.803  1.00  0.00           C  \\nATOM    177  CE  LYS A  25     -23.784  -5.819   8.543  1.00  0.00           C  \\nATOM    178  NZ  LYS A  25     -23.658  -6.827   7.661  1.00  0.00           N  \\nATOM    179  N   HIS A  26     -24.414  -2.921  14.465  1.00  0.00           N  \\nATOM    180  CA  HIS A  26     -23.028  -2.417  14.654  1.00  0.00           C  \\nATOM    181  C   HIS A  26     -21.894  -3.425  14.087  1.00  0.00           C  \\nATOM    182  O   HIS A  26     -22.083  -3.803  12.827  1.00  0.00           O  \\nATOM    183  CB  HIS A  26     -23.028  -2.543  16.103  1.00  0.00           C  \\nATOM    184  CG  HIS A  26     -24.036  -1.094  16.355  1.00  0.00           C  \\nATOM    185  ND1 HIS A  26     -24.540  -0.338  15.473  1.00  0.00           N  \\nATOM    186  CD2 HIS A  26     -24.666  -0.905  17.363  1.00  0.00           C  \\nATOM    187  CE1 HIS A  26     -25.422   0.355  16.103  1.00  0.00           C  \\nATOM    188  NE2 HIS A  26     -25.422   0.103  17.237  1.00  0.00           N  \\nATOM    189  N   LYS A  27     -20.886  -3.551  14.717  1.00  0.00           N  \\nATOM    190  CA  LYS A  27     -19.752  -4.433  14.213  1.00  0.00           C  \\nATOM    191  C   LYS A  27     -18.366  -3.677  14.276  1.00  0.00           C  \\nATOM    192  O   LYS A  27     -17.484  -3.677  13.331  1.00  0.00           O  \\nATOM    193  CB  LYS A  27     -20.004  -4.559  12.575  1.00  0.00           C  \\nATOM    194  CG  LYS A  27     -20.382  -3.425  11.882  1.00  0.00           C  \\nATOM    195  CD  LYS A  27     -19.500  -3.047  11.063  1.00  0.00           C  \\nATOM    196  CE  LYS A  27     -19.752  -3.677   9.551  1.00  0.00           C  \\nATOM    197  NZ  LYS A  27     -18.618  -3.677   8.858  1.00  0.00           N  \\nATOM    198  N   VAL A  28     -18.177  -3.047  15.410  1.00  0.00           N  \\nATOM    199  CA  VAL A  28     -16.854  -2.228  15.410  1.00  0.00           C  \\nATOM    200  C   VAL A  28     -17.358  -0.905  15.473  1.00  0.00           C  \\nATOM    201  O   VAL A  28     -17.610  -0.275  16.481  1.00  0.00           O  \\nATOM    202  CB  VAL A  28     -15.846  -2.291  14.780  1.00  0.00           C  \\nATOM    203  CG1 VAL A  28     -15.594  -1.094  14.087  1.00  0.00           C  \\nATOM    204  CG2 VAL A  28     -14.586  -2.858  15.095  1.00  0.00           C  \\nATOM    205  N   GLY A  29     -17.610  -0.023  14.591  1.00  0.00           N  \\nATOM    206  CA  GLY A  29     -18.240   1.111  14.843  1.00  0.00           C  \\nATOM    207  C   GLY A  29     -19.815   1.048  14.843  1.00  0.00           C  \\nATOM    208  O   GLY A  29     -20.193   0.103  13.961  1.00  0.00           O  \\nATOM    209  N   PRO A  30     -20.634   1.615  15.977  1.00  0.00           N  \\nATOM    210  CA  PRO A  30     -22.020   1.489  15.725  1.00  0.00           C  \\nATOM    211  C   PRO A  30     -22.398   1.804  14.339  1.00  0.00           C  \\nATOM    212  O   PRO A  30     -22.020   2.371  13.457  1.00  0.00           O  \\nATOM    213  CB  PRO A  30     -22.524   2.371  16.481  1.00  0.00           C  \\nATOM    214  CG  PRO A  30     -21.642   1.993  16.859  1.00  0.00           C  \\nATOM    215  CD  PRO A  30     -20.508   2.497  16.607  1.00  0.00           C  \\nATOM    216  N   ASN A  31     -23.910   1.552  13.961  1.00  0.00           N  \\nATOM    217  CA  ASN A  31     -24.414   2.119  12.701  1.00  0.00           C  \\nATOM    218  C   ASN A  31     -24.036   3.631  12.764  1.00  0.00           C  \\nATOM    219  O   ASN A  31     -24.036   4.387  13.835  1.00  0.00           O  \\nATOM    220  CB  ASN A  31     -25.926   1.993  12.701  1.00  0.00           C  \\nATOM    221  CG  ASN A  31     -26.052   2.056  11.063  1.00  0.00           C  \\nATOM    222  OD1 ASN A  31     -27.186   2.623  10.307  1.00  0.00           O  \\nATOM    223  ND2 ASN A  31     -25.359   1.363  10.181  1.00  0.00           N  \\nATOM    224  N   LEU A  32     -23.658   4.261  11.819  1.00  0.00           N  \\nATOM    225  CA  LEU A  32     -23.280   5.899  11.819  1.00  0.00           C  \\nATOM    226  C   LEU A  32     -24.666   6.403  11.315  1.00  0.00           C  \\nATOM    227  O   LEU A  32     -24.666   6.655   9.929  1.00  0.00           O  \\nATOM    228  CB  LEU A  32     -22.146   6.466  11.315  1.00  0.00           C  \\nATOM    229  CG  LEU A  32     -20.886   6.529  12.323  1.00  0.00           C  \\nATOM    230  CD1 LEU A  32     -20.382   7.978  12.323  1.00  0.00           C  \\nATOM    231  CD2 LEU A  32     -21.327   6.277  13.457  1.00  0.00           C  \\nATOM    232  N   TRP A  33     -25.611   6.151  11.693  1.00  0.00           N  \\nATOM    233  CA  TRP A  33     -27.186   6.529  11.189  1.00  0.00           C  \\nATOM    234  C   TRP A  33     -27.312   7.915  10.433  1.00  0.00           C  \\nATOM    235  O   TRP A  33     -27.312   7.852   9.299  1.00  0.00           O  \\nATOM    236  CB  TRP A  33     -28.446   6.403  12.071  1.00  0.00           C  \\nATOM    237  CG  TRP A  33     -28.572   5.773  13.205  1.00  0.00           C  \\nATOM    238  CD1 TRP A  33     -27.564   4.891  13.583  1.00  0.00           C  \\nATOM    239  CD2 TRP A  33     -29.328   5.647  14.213  1.00  0.00           C  \\nATOM    240  NE1 TRP A  33     -27.816   4.387  14.843  1.00  0.00           N  \\nATOM    241  CE2 TRP A  33     -29.013   4.828  15.158  1.00  0.00           C  \\nATOM    242  CE3 TRP A  33     -30.462   6.277  14.213  1.00  0.00           C  \\nATOM    243  CZ2 TRP A  33     -29.706   4.639  15.851  1.00  0.00           C  \\nATOM    244  CZ3 TRP A  33     -31.092   5.962  15.284  1.00  0.00           C  \\nATOM    245  CH2 TRP A  33     -30.714   5.143  15.914  1.00  0.00           C  \\nATOM    246  N   GLY A  34     -27.564   8.797  11.315  1.00  0.00           N  \\nATOM    247  CA  GLY A  34     -27.690  10.120  10.685  1.00  0.00           C  \\nATOM    248  C   GLY A  34     -26.934  11.191  11.315  1.00  0.00           C  \\nATOM    249  O   GLY A  34     -27.312  12.199  12.134  1.00  0.00           O  \\nATOM    250  N   LEU A  35     -25.548  11.191  11.252  1.00  0.00           N  \\nATOM    251  CA  LEU A  35     -24.666  12.199  11.945  1.00  0.00           C  \\nATOM    252  C   LEU A  35     -25.548  13.459  12.386  1.00  0.00           C  \\nATOM    253  O   LEU A  35     -25.800  13.963  13.457  1.00  0.00           O  \\nATOM    254  CB  LEU A  35     -23.154  12.640  11.567  1.00  0.00           C  \\nATOM    255  CG  LEU A  35     -22.335  11.506  11.189  1.00  0.00           C  \\nATOM    256  CD1 LEU A  35     -20.886  11.695  11.189  1.00  0.00           C  \\nATOM    257  CD2 LEU A  35     -22.650  10.183  11.819  1.00  0.00           C  \\nATOM    258  N   PHE A  36     -26.052  13.963  11.441  1.00  0.00           N  \\nATOM    259  CA  PHE A  36     -26.934  15.223  11.693  1.00  0.00           C  \\nATOM    260  C   PHE A  36     -28.131  15.223  12.701  1.00  0.00           C  \\nATOM    261  O   PHE A  36     -29.328  15.223  12.449  1.00  0.00           O  \\nATOM    262  CB  PHE A  36     -27.186  15.601  10.055  1.00  0.00           C  \\nATOM    263  CG  PHE A  36     -25.926  15.601   9.425  1.00  0.00           C  \\nATOM    264  CD1 PHE A  36     -25.674  15.727   8.102  1.00  0.00           C  \\nATOM    265  CD2 PHE A  36     -24.792  15.097   9.803  1.00  0.00           C  \\nATOM    266  CE1 PHE A  36     -24.540  15.475   7.409  1.00  0.00           C  \\nATOM    267  CE2 PHE A  36     -23.658  15.097   9.551  1.00  0.00           C  \\nATOM    268  CZ  PHE A  36     -23.406  15.097   8.291  1.00  0.00           C  \\nATOM    269  N   GLY A  37     -27.879  15.538  13.961  1.00  0.00           N  \\nATOM    270  CA  GLY A  37     -28.509  15.664  15.347  1.00  0.00           C  \\nATOM    271  C   GLY A  37     -28.005  14.971  16.355  1.00  0.00           C  \\nATOM    272  O   GLY A  37     -28.572  14.215  17.300  1.00  0.00           O  \\nATOM    273  N   ARG A  38     -26.682  15.097  16.859  1.00  0.00           N  \\nATOM    274  CA  ARG A  38     -26.556  14.341  18.119  1.00  0.00           C  \\nATOM    275  C   ARG A  38     -25.170  14.845  18.371  1.00  0.00           C  \\nATOM    276  O   ARG A  38     -24.414  15.349  17.741  1.00  0.00           O  \\nATOM    277  CB  ARG A  38     -26.178  13.081  17.741  1.00  0.00           C  \\nATOM    278  CG  ARG A  38     -25.863  12.703  16.355  1.00  0.00           C  \\nATOM    279  CD  ARG A  38     -25.422  11.191  16.040  1.00  0.00           C  \\nATOM    280  NE  ARG A  38     -24.666  10.561  16.985  1.00  0.00           N  \\nATOM    281  CZ  ARG A  38     -24.792   9.238  16.859  1.00  0.00           C  \\nATOM    282  NH1 ARG A  38     -24.351   8.545  17.489  1.00  0.00           N  \\nATOM    283  NH2 ARG A  38     -25.485   8.797  15.851  1.00  0.00           N  \\nATOM    284  N   LYS A  39     -25.044  14.593  19.757  1.00  0.00           N  \\nATOM    285  CA  LYS A  39     -24.036  15.160  20.639  1.00  0.00           C  \\nATOM    286  C   LYS A  39     -23.028  14.089  20.891  1.00  0.00           C  \\nATOM    287  O   LYS A  39     -23.406  13.207  20.009  1.00  0.00           O  \\nATOM    288  CB  LYS A  39     -24.540  15.601  22.151  1.00  0.00           C  \\nATOM    289  CG  LYS A  39     -24.414  15.475  23.663  1.00  0.00           C  \\nATOM    290  CD  LYS A  39     -25.170  16.231  24.923  1.00  0.00           C  \\nATOM    291  N   THR A  40     -22.020  13.837  21.521  1.00  0.00           N  \\nATOM    292  CA  THR A  40     -21.390  12.388  21.269  1.00  0.00           C  \\nATOM    293  C   THR A  40     -21.642  11.191  22.277  1.00  0.00           C  \\nATOM    294  O   THR A  40     -21.516  11.695  23.411  1.00  0.00           O  \\nATOM    295  CB  THR A  40     -19.752  12.577  21.269  1.00  0.00           C  \\nATOM    296  OG1 THR A  40     -19.374  11.884  19.883  1.00  0.00           O  \\nATOM    297  CG2 THR A  40     -19.626  13.837  20.765  1.00  0.00           C  \\nATOM    298  N   GLY A  41     -22.020  10.309  21.773  1.00  0.00           N  \\nATOM    299  CA  GLY A  41     -22.650   9.553  22.907  1.00  0.00           C  \\nATOM    300  C   GLY A  41     -23.910   8.797  22.403  1.00  0.00           C  \\nATOM    301  O   GLY A  41     -24.162   8.545  21.395  1.00  0.00           O  \\nATOM    302  N   GLN A  42     -24.918   8.419  23.033  1.00  0.00           N  \\nATOM    303  CA  GLN A  42     -26.241   7.915  23.033  1.00  0.00           C  \\nATOM    304  C   GLN A  42     -26.304   7.537  21.647  1.00  0.00           C  \\nATOM    305  O   GLN A  42     -26.556   8.545  20.765  1.00  0.00           O  \\nATOM    306  CB  GLN A  42     -27.438   8.545  23.537  1.00  0.00           C  \\nATOM    307  CG  GLN A  42     -28.068   7.789  24.419  1.00  0.00           C  \\nATOM    308  CD  GLN A  42     -29.076   7.096  24.230  1.00  0.00           C  \\nATOM    309  OE1 GLN A  42     -29.706   5.899  24.797  1.00  0.00           O  \\nATOM    310  NE2 GLN A  42     -29.706   7.411  22.781  1.00  0.00           N  \\nATOM    311  N   ALA A  43     -26.304   6.277  21.143  1.00  0.00           N  \\nATOM    312  CA  ALA A  43     -26.682   6.025  19.505  1.00  0.00           C  \\nATOM    313  C   ALA A  43     -27.186   4.639  19.442  1.00  0.00           C  \\nATOM    314  O   ALA A  43     -27.060   4.009  18.497  1.00  0.00           O  \\nATOM    315  CB  ALA A  43     -25.548   6.025  18.497  1.00  0.00           C  \\nATOM    316  N   GLU A  44     -27.438   3.883  20.450  1.00  0.00           N  \\nATOM    317  CA  GLU A  44     -28.131   2.497  20.261  1.00  0.00           C  \\nATOM    318  C   GLU A  44     -27.060   1.615  20.891  1.00  0.00           C  \\nATOM    319  O   GLU A  44     -26.556   0.481  20.324  1.00  0.00           O  \\nATOM    320  CB  GLU A  44     -28.635   2.371  18.749  1.00  0.00           C  \\nATOM    321  CG  GLU A  44     -30.210   0.993  18.749  1.00  0.00           C  \\nATOM    322  CD  GLU A  44     -30.840   2.875  19.379  1.00  0.00           C  \\nATOM    323  OE1 GLU A  44     -32.100   2.875  18.875  1.00  0.00           O  \\nATOM    324  OE2 GLU A  44     -30.084   4.135  19.505  1.00  0.00           O  \\nATOM    325  N   GLY A  45     -26.808   1.615  22.277  1.00  0.00           N  \\nATOM    326  CA  GLY A  45     -25.674   0.859  22.781  1.00  0.00           C  \\nATOM    327  C   GLY A  45     -24.162   1.363  22.907  1.00  0.00           C  \\nATOM    328  O   GLY A  45     -23.532   1.363  21.521  1.00  0.00           O  \\nATOM    329  N   TYR A  46     -23.532   1.993  23.789  1.00  0.00           N  \\nATOM    330  CA  TYR A  46     -22.146   2.875  23.537  1.00  0.00           C  \\nATOM    331  C   TYR A  46     -21.327   1.993  24.356  1.00  0.00           C  \\nATOM    332  O   TYR A  46     -21.327   0.733  24.167  1.00  0.00           O  \\nATOM    333  CB  TYR A  46     -22.524   4.198  23.159  1.00  0.00           C  \\nATOM    334  CG  TYR A  46     -22.146   5.143  22.151  1.00  0.00           C  \\nATOM    335  CD1 TYR A  46     -22.776   5.269  21.521  1.00  0.00           C  \\nATOM    336  CD2 TYR A  46     -20.886   5.647  21.647  1.00  0.00           C  \\nATOM    337  CE1 TYR A  46     -22.524   5.899  20.387  1.00  0.00           C  \\nATOM    338  CE2 TYR A  46     -20.382   6.277  20.513  1.00  0.00           C  \\nATOM    339  CZ  TYR A  46     -21.264   6.277  20.009  1.00  0.00           C  \\nATOM    340  OH  TYR A  46     -21.138   6.655  19.001  1.00  0.00           O  \\nATOM    341  N   SER A  47     -20.634   2.560  25.049  1.00  0.00           N  \\nATOM    342  CA  SER A  47     -19.374   1.741  25.679  1.00  0.00           C  \\nATOM    343  C   SER A  47     -18.114   2.623  25.679  1.00  0.00           C  \\nATOM    344  O   SER A  47     -18.114   3.379  24.797  1.00  0.00           O  \\nATOM    345  CB  SER A  47     -19.248   0.355  25.427  1.00  0.00           C  \\nATOM    346  OG  SER A  47     -18.870   0.481  23.789  1.00  0.00           O  \\nATOM    347  N   TYR A  48     -17.484   2.749  26.939  1.00  0.00           N  \\nATOM    348  CA  TYR A  48     -16.413   3.757  26.939  1.00  0.00           C  \\nATOM    349  C   TYR A  48     -16.602   4.891  27.695  1.00  0.00           C  \\nATOM    350  O   TYR A  48     -17.610   5.269  27.821  1.00  0.00           O  \\nATOM    351  CB  TYR A  48     -16.224   3.757  25.301  1.00  0.00           C  \\nATOM    352  CG  TYR A  48     -15.216   4.639  24.545  1.00  0.00           C  \\nATOM    353  CD1 TYR A  48     -14.082   5.143  24.797  1.00  0.00           C  \\nATOM    354  CD2 TYR A  48     -15.342   4.765  23.411  1.00  0.00           C  \\nATOM    355  CE1 TYR A  48     -13.326   6.025  24.167  1.00  0.00           C  \\nATOM    356  CE2 TYR A  48     -14.712   5.521  22.529  1.00  0.00           C  \\nATOM    357  CZ  TYR A  48     -13.704   6.088  23.033  1.00  0.00           C  \\nATOM    358  OH  TYR A  48     -13.074   6.907  22.151  1.00  0.00           O  \\nATOM    359  N   THR A  49     -15.594   5.647  28.199  1.00  0.00           N  \\nATOM    360  CA  THR A  49     -15.846   6.907  29.207  1.00  0.00           C  \\nATOM    361  C   THR A  49     -15.594   8.041  28.829  1.00  0.00           C  \\nATOM    362  O   THR A  49     -16.350   8.671  27.947  1.00  0.00           O  \\nATOM    363  CB  THR A  49     -15.531   6.403  30.845  1.00  0.00           C  \\nATOM    364  OG1 THR A  49     -14.208   5.899  30.845  1.00  0.00           O  \\nATOM    365  CG2 THR A  49     -16.224   5.395  31.223  1.00  0.00           C  \\nATOM    366  N   ASP A  50     -14.334   8.482  28.703  1.00  0.00           N  \\nATOM    367  CA  ASP A  50     -14.082   9.805  28.199  1.00  0.00           C  \\nATOM    368  C   ASP A  50     -13.704   9.679  26.813  1.00  0.00           C  \\nATOM    369  O   ASP A  50     -13.074   8.545  26.183  1.00  0.00           O  \\nATOM    370  CB  ASP A  50     -13.074  10.435  28.829  1.00  0.00           C  \\nATOM    371  CG  ASP A  50     -12.948  10.183  30.089  1.00  0.00           C  \\nATOM    372  OD1 ASP A  50     -12.066   9.049  30.593  1.00  0.00           O  \\nATOM    373  OD2 ASP A  50     -13.956  10.309  30.971  1.00  0.00           O  \\nATOM    374  N   ALA A  51     -13.830  10.309  25.805  1.00  0.00           N  \\nATOM    375  CA  ALA A  51     -13.704   9.931  24.419  1.00  0.00           C  \\nATOM    376  C   ALA A  51     -15.216   9.805  23.915  1.00  0.00           C  \\nATOM    377  O   ALA A  51     -16.098   9.301  24.167  1.00  0.00           O  \\nATOM    378  CB  ALA A  51     -13.200   8.671  24.041  1.00  0.00           C  \\nATOM    379  N   ASN A  52     -15.342  10.561  22.907  1.00  0.00           N  \\nATOM    380  CA  ASN A  52     -16.539  10.561  22.277  1.00  0.00           C  \\nATOM    381  C   ASN A  52     -17.232  11.380  22.907  1.00  0.00           C  \\nATOM    382  O   ASN A  52     -17.232  12.766  22.781  1.00  0.00           O  \\nATOM    383  CB  ASN A  52     -17.610   9.553  21.899  1.00  0.00           C  \\nATOM    384  CG  ASN A  52     -17.484   8.671  20.639  1.00  0.00           C  \\nATOM    385  OD1 ASN A  52     -16.980   7.789  20.135  1.00  0.00           O  \\nATOM    386  ND2 ASN A  52     -18.240   9.427  19.631  1.00  0.00           N  \\nATOM    387  N   LYS A  53     -17.862  10.939  24.041  1.00  0.00           N  \\nATOM    388  CA  LYS A  53     -18.240  11.947  25.301  1.00  0.00           C  \\nATOM    389  C   LYS A  53     -17.106  12.703  25.805  1.00  0.00           C  \\nATOM    390  O   LYS A  53     -16.476  13.711  25.175  1.00  0.00           O  \\nATOM    391  CB  LYS A  53     -18.870  11.317  26.561  1.00  0.00           C  \\nATOM    392  CG  LYS A  53     -20.130  10.435  26.057  1.00  0.00           C  \\nATOM    393  CD  LYS A  53     -20.634   9.301  27.065  1.00  0.00           C  \\nATOM    394  CE  LYS A  53     -22.146   8.671  26.372  1.00  0.00           C  \\nATOM    395  NZ  LYS A  53     -22.524   7.663  27.191  1.00  0.00           N  \\nATOM    396  N   SER A  54     -16.602  12.325  27.065  1.00  0.00           N  \\nATOM    397  CA  SER A  54     -15.594  13.081  27.821  1.00  0.00           C  \\nATOM    398  C   SER A  54     -15.594  14.593  27.443  1.00  0.00           C  \\nATOM    399  O   SER A  54     -16.476  15.223  27.695  1.00  0.00           O  \\nATOM    400  CB  SER A  54     -14.460  12.703  27.569  1.00  0.00           C  \\nATOM    401  OG  SER A  54     -13.452  12.955  28.703  1.00  0.00           O  \\nATOM    402  N   LYS A  55     -14.838  15.223  26.813  1.00  0.00           N  \\nATOM    403  CA  LYS A  55     -14.964  16.420  26.057  1.00  0.00           C  \\nATOM    404  C   LYS A  55     -16.602  16.987  25.931  1.00  0.00           C  \\nATOM    405  O   LYS A  55     -16.854  18.121  26.183  1.00  0.00           O  \\nATOM    406  CB  LYS A  55     -14.208  16.609  24.734  1.00  0.00           C  \\nATOM    407  CG  LYS A  55     -14.712  17.743  23.600  1.00  0.00           C  \\nATOM    408  CD  LYS A  55     -13.452  17.743  22.844  1.00  0.00           C  \\nATOM    409  CE  LYS A  55     -13.578  17.995  21.143  1.00  0.00           C  \\nATOM    410  NZ  LYS A  55     -12.444  17.743  20.135  1.00  0.00           N  \\nATOM    411  N   GLY A  56     -17.232  16.042  25.679  1.00  0.00           N  \\nATOM    412  CA  GLY A  56     -18.870  16.042  25.301  1.00  0.00           C  \\nATOM    413  C   GLY A  56     -19.122  17.239  24.293  1.00  0.00           C  \\nATOM    414  O   GLY A  56     -18.492  18.247  23.852  1.00  0.00           O  \\nATOM    415  N   ILE A  57     -20.130  17.113  23.285  1.00  0.00           N  \\nATOM    416  CA  ILE A  57     -20.004  17.869  22.025  1.00  0.00           C  \\nATOM    417  C   ILE A  57     -21.264  18.121  21.143  1.00  0.00           C  \\nATOM    418  O   ILE A  57     -22.398  18.499  21.395  1.00  0.00           O  \\nATOM    419  CB  ILE A  57     -18.996  17.491  21.395  1.00  0.00           C  \\nATOM    420  CG1 ILE A  57     -19.374  16.357  20.513  1.00  0.00           C  \\nATOM    421  CG2 ILE A  57     -17.862  17.050  22.025  1.00  0.00           C  \\nATOM    422  CD1 ILE A  57     -19.122  16.609  19.127  1.00  0.00           C  \\nATOM    423  N   VAL A  58     -21.264  17.743  19.631  1.00  0.00           N  \\nATOM    424  CA  VAL A  58     -22.524  17.995  19.001  1.00  0.00           C  \\nATOM    425  C   VAL A  58     -22.020  17.239  17.615  1.00  0.00           C  \\nATOM    426  O   VAL A  58     -21.642  18.121  16.985  1.00  0.00           O  \\nATOM    427  CB  VAL A  58     -23.280  19.129  19.127  1.00  0.00           C  \\nATOM    428  CG1 VAL A  58     -22.398  20.263  18.749  1.00  0.00           C  \\nATOM    429  CG2 VAL A  58     -24.540  19.192  18.371  1.00  0.00           C  \\nATOM    430  N   TRP A  59     -22.398  16.231  16.859  1.00  0.00           N  \\nATOM    431  CA  TRP A  59     -22.020  16.105  15.221  1.00  0.00           C  \\nATOM    432  C   TRP A  59     -22.524  17.239  14.213  1.00  0.00           C  \\nATOM    433  O   TRP A  59     -23.784  17.365  14.339  1.00  0.00           O  \\nATOM    434  CB  TRP A  59     -22.146  14.845  14.717  1.00  0.00           C  \\nATOM    435  CG  TRP A  59     -21.264  13.963  15.347  1.00  0.00           C  \\nATOM    436  CD1 TRP A  59     -21.390  12.703  15.851  1.00  0.00           C  \\nATOM    437  CD2 TRP A  59     -19.878  14.152  15.536  1.00  0.00           C  \\nATOM    438  NE1 TRP A  59     -20.382  12.451  16.607  1.00  0.00           N  \\nATOM    439  CE2 TRP A  59     -19.500  13.207  15.977  1.00  0.00           C  \\nATOM    440  CE3 TRP A  59     -19.248  15.160  15.095  1.00  0.00           C  \\nATOM    441  CZ2 TRP A  59     -18.114  13.459  16.607  1.00  0.00           C  \\nATOM    442  CZ3 TRP A  59     -18.051  15.223  15.473  1.00  0.00           C  \\nATOM    443  CH2 TRP A  59     -17.484  14.467  16.229  1.00  0.00           C  \\nATOM    444  N   ASN A  60     -21.768  17.869  13.772  1.00  0.00           N  \\nATOM    445  CA  ASN A  60     -22.398  19.129  12.953  1.00  0.00           C  \\nATOM    446  C   ASN A  60     -21.642  19.255  11.441  1.00  0.00           C  \\nATOM    447  O   ASN A  60     -20.634  18.751  11.189  1.00  0.00           O  \\nATOM    448  CB  ASN A  60     -22.524  20.515  13.583  1.00  0.00           C  \\nATOM    449  CG  ASN A  60     -22.776  21.649  12.701  1.00  0.00           C  \\nATOM    450  OD1 ASN A  60     -24.036  21.712  11.945  1.00  0.00           O  \\nATOM    451  ND2 ASN A  60     -21.894  22.783  12.701  1.00  0.00           N  \\nATOM    452  N   GLU A  61     -22.020  20.137  10.811  1.00  0.00           N  \\nATOM    453  CA  GLU A  61     -21.138  20.263   9.425  1.00  0.00           C  \\nATOM    454  C   GLU A  61     -19.626  20.389  10.181  1.00  0.00           C  \\nATOM    455  O   GLU A  61     -18.744  19.507   9.803  1.00  0.00           O  \\nATOM    456  CB  GLU A  61     -21.516  21.523   8.669  1.00  0.00           C  \\nATOM    457  CG  GLU A  61     -22.020  21.523   7.409  1.00  0.00           C  \\nATOM    458  CD  GLU A  61     -23.154  20.767   7.283  1.00  0.00           C  \\nATOM    459  OE1 GLU A  61     -24.540  20.767   7.535  1.00  0.00           O  \\nATOM    460  OE2 GLU A  61     -22.902  19.381   6.653  1.00  0.00           O  \\nATOM    461  N   ASN A  62     -19.248  21.397  10.685  1.00  0.00           N  \\nATOM    462  CA  ASN A  62     -17.736  21.586  11.441  1.00  0.00           C  \\nATOM    463  C   ASN A  62     -17.106  20.137  11.945  1.00  0.00           C  \\nATOM    464  O   ASN A  62     -15.972  20.515  12.071  1.00  0.00           O  \\nATOM    465  CB  ASN A  62     -17.736  22.783  11.945  1.00  0.00           C  \\nATOM    466  CG  ASN A  62     -17.610  24.043  11.441  1.00  0.00           C  \\nATOM    467  OD1 ASN A  62     -18.744  24.547  10.559  1.00  0.00           O  \\nATOM    468  ND2 ASN A  62     -16.539  24.799  11.063  1.00  0.00           N  \\nATOM    469  N   THR A  63     -17.610  20.263  13.268  1.00  0.00           N  \\nATOM    470  CA  THR A  63     -17.043  19.255  14.087  1.00  0.00           C  \\nATOM    471  C   THR A  63     -16.602  17.869  13.331  1.00  0.00           C  \\nATOM    472  O   THR A  63     -15.720  17.239  13.331  1.00  0.00           O  \\nATOM    473  CB  THR A  63     -18.051  18.751  15.095  1.00  0.00           C  \\nATOM    474  OG1 THR A  63     -17.358  18.499  15.851  1.00  0.00           O  \\nATOM    475  CG2 THR A  63     -18.492  17.365  14.528  1.00  0.00           C  \\nATOM    476  N   LEU A  64     -17.358  17.617  12.197  1.00  0.00           N  \\nATOM    477  CA  LEU A  64     -16.854  16.357  11.441  1.00  0.00           C  \\nATOM    478  C   LEU A  64     -15.468  16.861  10.685  1.00  0.00           C  \\nATOM    479  O   LEU A  64     -14.838  16.231  10.433  1.00  0.00           O  \\nATOM    480  CB  LEU A  64     -18.114  15.979  10.055  1.00  0.00           C  \\nATOM    481  CG  LEU A  64     -18.744  14.719  10.433  1.00  0.00           C  \\nATOM    482  CD1 LEU A  64     -19.122  14.719  11.567  1.00  0.00           C  \\nATOM    483  CD2 LEU A  64     -19.878  14.593   9.803  1.00  0.00           C  \\nATOM    484  N   MET A  65     -15.468  17.995  10.181  1.00  0.00           N  \\nATOM    485  CA  MET A  65     -13.956  18.499   9.803  1.00  0.00           C  \\nATOM    486  C   MET A  65     -13.200  18.373  11.189  1.00  0.00           C  \\nATOM    487  O   MET A  65     -12.192  17.869  11.189  1.00  0.00           O  \\nATOM    488  CB  MET A  65     -13.956  19.885   9.425  1.00  0.00           C  \\nATOM    489  CG  MET A  65     -13.452  20.137   7.913  1.00  0.00           C  \\nATOM    490  SD  MET A  65     -13.704  18.625   7.283  1.00  0.00           S  \\nATOM    491  CE  MET A  65     -13.452  19.192   5.519  1.00  0.00           C  \\nATOM    492  N   GLU A  66     -13.578  18.751  12.197  1.00  0.00           N  \\nATOM    493  CA  GLU A  66     -13.074  18.751  13.457  1.00  0.00           C  \\nATOM    494  C   GLU A  66     -12.444  17.239  13.646  1.00  0.00           C  \\nATOM    495  O   GLU A  66     -11.310  17.113  13.835  1.00  0.00           O  \\nATOM    496  CB  GLU A  66     -13.830  19.381  14.717  1.00  0.00           C  \\nATOM    497  CG  GLU A  66     -13.578  20.767  14.969  1.00  0.00           C  \\nATOM    498  CD  GLU A  66     -12.318  21.271  15.851  1.00  0.00           C  \\nATOM    499  OE1 GLU A  66     -12.192  22.657  16.103  1.00  0.00           O  \\nATOM    500  OE2 GLU A  66     -11.310  20.641  16.355  1.00  0.00           O  \\nATOM    501  N   TYR A  67     -13.263  16.483  13.457  1.00  0.00           N  \\nATOM    502  CA  TYR A  67     -12.822  14.971  13.457  1.00  0.00           C  \\nATOM    503  C   TYR A  67     -12.192  14.467  12.323  1.00  0.00           C  \\nATOM    504  O   TYR A  67     -11.184  14.089  11.945  1.00  0.00           O  \\nATOM    505  CB  TYR A  67     -14.082  14.341  13.835  1.00  0.00           C  \\nATOM    506  CG  TYR A  67     -14.082  12.955  14.465  1.00  0.00           C  \\nATOM    507  CD1 TYR A  67     -14.838  11.821  14.465  1.00  0.00           C  \\nATOM    508  CD2 TYR A  67     -13.326  13.081  15.599  1.00  0.00           C  \\nATOM    509  CE1 TYR A  67     -14.649  10.876  15.221  1.00  0.00           C  \\nATOM    510  CE2 TYR A  67     -13.200  11.947  16.103  1.00  0.00           C  \\nATOM    511  CZ  TYR A  67     -13.830  10.876  15.977  1.00  0.00           C  \\nATOM    512  OH  TYR A  67     -13.704   9.805  16.607  1.00  0.00           O  \\nATOM    513  N   LEU A  68     -12.822  14.719  11.315  1.00  0.00           N  \\nATOM    514  CA  LEU A  68     -12.192  14.341  10.055  1.00  0.00           C  \\nATOM    515  C   LEU A  68     -10.806  14.719  10.307  1.00  0.00           C  \\nATOM    516  O   LEU A  68      -9.798  13.963  10.307  1.00  0.00           O  \\nATOM    517  CB  LEU A  68     -13.074  14.845   9.173  1.00  0.00           C  \\nATOM    518  CG  LEU A  68     -14.334  13.837   8.795  1.00  0.00           C  \\nATOM    519  CD1 LEU A  68     -15.216  14.467   7.787  1.00  0.00           C  \\nATOM    520  CD2 LEU A  68     -13.704  12.829   7.787  1.00  0.00           C  \\nATOM    521  N   GLU A  69     -10.428  15.853  10.685  1.00  0.00           N  \\nATOM    522  CA  GLU A  69      -8.916  16.231  10.937  1.00  0.00           C  \\nATOM    523  C   GLU A  69      -8.097  14.971  11.441  1.00  0.00           C  \\nATOM    524  O   GLU A  69      -7.278  14.593  10.685  1.00  0.00           O  \\nATOM    525  CB  GLU A  69      -9.042  17.365  11.693  1.00  0.00           C  \\nATOM    526  CG  GLU A  69      -8.034  18.373  11.819  1.00  0.00           C  \\nATOM    527  CD  GLU A  69      -6.900  18.373  12.953  1.00  0.00           C  \\nATOM    528  OE1 GLU A  69      -5.388  18.499  12.827  1.00  0.00           O  \\nATOM    529  OE2 GLU A  69      -7.278  17.869  14.087  1.00  0.00           O  \\nATOM    530  N   ASN A  70      -8.097  14.845  12.575  1.00  0.00           N  \\nATOM    531  CA  ASN A  70      -7.278  13.585  12.827  1.00  0.00           C  \\nATOM    532  C   ASN A  70      -7.404  13.459  14.339  1.00  0.00           C  \\nATOM    533  O   ASN A  70      -7.152  14.215  15.221  1.00  0.00           O  \\nATOM    534  CB  ASN A  70      -5.892  13.837  12.449  1.00  0.00           C  \\nATOM    535  CG  ASN A  70      -4.758  14.089  13.457  1.00  0.00           C  \\nATOM    536  OD1 ASN A  70      -3.624  15.097  13.457  1.00  0.00           O  \\nATOM    537  ND2 ASN A  70      -4.758  13.333  14.717  1.00  0.00           N  \\nATOM    538  N   PRO A  71      -7.908  12.199  14.780  1.00  0.00           N  \\nATOM    539  CA  PRO A  71      -8.475  11.758  15.725  1.00  0.00           C  \\nATOM    540  C   PRO A  71      -7.593  11.758  16.859  1.00  0.00           C  \\nATOM    541  O   PRO A  71      -7.782  11.317  17.993  1.00  0.00           O  \\nATOM    542  CB  PRO A  71      -9.042  10.561  15.536  1.00  0.00           C  \\nATOM    543  CG  PRO A  71      -8.916  10.309  14.213  1.00  0.00           C  \\nATOM    544  CD  PRO A  71      -8.160  11.191  13.583  1.00  0.00           C  \\nATOM    545  N   LYS A  72      -6.522  12.325  16.733  1.00  0.00           N  \\nATOM    546  CA  LYS A  72      -5.388  12.703  17.615  1.00  0.00           C  \\nATOM    547  C   LYS A  72      -5.451  14.215  17.993  1.00  0.00           C  \\nATOM    548  O   LYS A  72      -5.262  14.971  17.111  1.00  0.00           O  \\nATOM    549  CB  LYS A  72      -3.876  12.451  17.111  1.00  0.00           C  \\nATOM    550  CG  LYS A  72      -3.624  11.065  17.363  1.00  0.00           C  \\nATOM    551  CD  LYS A  72      -2.112  10.435  17.237  1.00  0.00           C  \\nATOM    552  CE  LYS A  72      -1.860   8.923  17.741  1.00  0.00           C  \\nATOM    553  NZ  LYS A  72      -0.222   8.545  17.741  1.00  0.00           N  \\nATOM    554  N   LYS A  73      -6.081  14.719  19.127  1.00  0.00           N  \\nATOM    555  CA  LYS A  73      -6.396  16.231  19.127  1.00  0.00           C  \\nATOM    556  C   LYS A  73      -7.782  16.357  18.623  1.00  0.00           C  \\nATOM    557  O   LYS A  73      -8.160  16.231  17.363  1.00  0.00           O  \\nATOM    558  CB  LYS A  73      -5.640  17.113  17.993  1.00  0.00           C  \\nATOM    559  CG  LYS A  73      -4.758  18.247  18.623  1.00  0.00           C  \\nATOM    560  CD  LYS A  73      -3.876  19.003  17.993  1.00  0.00           C  \\nATOM    561  CE  LYS A  73      -2.616  20.389  18.812  1.00  0.00           C  \\nATOM    562  NZ  LYS A  73      -1.986  21.145  18.056  1.00  0.00           N  \\nATOM    563  N   TYR A  74      -8.664  16.231  19.631  1.00  0.00           N  \\nATOM    564  CA  TYR A  74     -10.050  16.105  19.379  1.00  0.00           C  \\nATOM    565  C   TYR A  74     -10.806  14.971  19.883  1.00  0.00           C  \\nATOM    566  O   TYR A  74     -10.554  13.963  19.379  1.00  0.00           O  \\nATOM    567  CB  TYR A  74     -10.554  16.105  17.741  1.00  0.00           C  \\nATOM    568  CG  TYR A  74     -11.814  16.735  17.363  1.00  0.00           C  \\nATOM    569  CD1 TYR A  74     -12.948  16.105  17.237  1.00  0.00           C  \\nATOM    570  CD2 TYR A  74     -11.940  17.995  16.985  1.00  0.00           C  \\nATOM    571  CE1 TYR A  74     -14.145  16.609  16.859  1.00  0.00           C  \\nATOM    572  CE2 TYR A  74     -13.074  18.625  16.607  1.00  0.00           C  \\nATOM    573  CZ  TYR A  74     -14.082  17.869  16.607  1.00  0.00           C  \\nATOM    574  OH  TYR A  74     -15.216  18.373  16.355  1.00  0.00           O  \\nATOM    575  N   ILE A  75     -11.310  15.097  21.017  1.00  0.00           N  \\nATOM    576  CA  ILE A  75     -11.940  14.089  21.647  1.00  0.00           C  \\nATOM    577  C   ILE A  75     -10.806  13.081  21.773  1.00  0.00           C  \\nATOM    578  O   ILE A  75     -10.932  11.821  21.521  1.00  0.00           O  \\nATOM    579  CB  ILE A  75     -13.200  13.585  21.143  1.00  0.00           C  \\nATOM    580  CG1 ILE A  75     -13.641  14.089  19.631  1.00  0.00           C  \\nATOM    581  CG2 ILE A  75     -14.271  13.585  22.025  1.00  0.00           C  \\nATOM    582  CD1 ILE A  75     -14.712  13.459  19.253  1.00  0.00           C  \\nATOM    583  N   PRO A  76      -9.420  13.585  21.899  1.00  0.00           N  \\nATOM    584  CA  PRO A  76      -8.160  12.955  21.647  1.00  0.00           C  \\nATOM    585  C   PRO A  76      -8.160  11.695  22.529  1.00  0.00           C  \\nATOM    586  O   PRO A  76      -8.538  11.695  23.789  1.00  0.00           O  \\nATOM    587  CB  PRO A  76      -6.900  13.837  21.836  1.00  0.00           C  \\nATOM    588  CG  PRO A  76      -7.152  14.845  22.277  1.00  0.00           C  \\nATOM    589  CD  PRO A  76      -8.538  14.845  22.151  1.00  0.00           C  \\nATOM    590  N   GLY A  77      -8.097  10.435  22.088  1.00  0.00           N  \\nATOM    591  CA  GLY A  77      -8.286   9.175  22.907  1.00  0.00           C  \\nATOM    592  C   GLY A  77      -9.546   8.167  22.529  1.00  0.00           C  \\nATOM    593  O   GLY A  77      -9.798   7.159  22.781  1.00  0.00           O  \\nATOM    594  N   THR A  78     -10.050   8.545  21.143  1.00  0.00           N  \\nATOM    595  CA  THR A  78     -11.310   7.789  20.639  1.00  0.00           C  \\nATOM    596  C   THR A  78     -10.995   6.529  20.513  1.00  0.00           C  \\nATOM    597  O   THR A  78      -9.798   6.151  20.009  1.00  0.00           O  \\nATOM    598  CB  THR A  78     -11.625   8.167  19.253  1.00  0.00           C  \\nATOM    599  OG1 THR A  78     -13.074   8.167  19.127  1.00  0.00           O  \\nATOM    600  CG2 THR A  78     -11.184   9.553  18.875  1.00  0.00           C  \\nATOM    601  N   LYS A  79     -11.562   5.521  20.765  1.00  0.00           N  \\nATOM    602  CA  LYS A  79     -11.058   4.072  20.639  1.00  0.00           C  \\nATOM    603  C   LYS A  79     -11.058   3.631  19.001  1.00  0.00           C  \\nATOM    604  O   LYS A  79     -10.932   2.497  18.371  1.00  0.00           O  \\nATOM    605  CB  LYS A  79     -11.562   3.064  21.395  1.00  0.00           C  \\nATOM    606  CG  LYS A  79     -12.570   2.560  20.513  1.00  0.00           C  \\nATOM    607  CD  LYS A  79     -13.704   1.615  20.891  1.00  0.00           C  \\nATOM    608  CE  LYS A  79     -14.838   1.363  20.135  1.00  0.00           C  \\nATOM    609  NZ  LYS A  79     -15.972   0.859  20.513  1.00  0.00           N  \\nATOM    610  N   MET A  80     -11.310   4.639  18.119  1.00  0.00           N  \\nATOM    611  CA  MET A  80     -11.436   4.135  16.607  1.00  0.00           C  \\nATOM    612  C   MET A  80     -10.050   4.198  16.103  1.00  0.00           C  \\nATOM    613  O   MET A  80      -9.294   5.017  16.103  1.00  0.00           O  \\nATOM    614  CB  MET A  80     -12.444   5.017  15.851  1.00  0.00           C  \\nATOM    615  CG  MET A  80     -11.940   6.088  14.969  1.00  0.00           C  \\nATOM    616  SD  MET A  80     -13.074   6.781  13.457  1.00  0.00           S  \\nATOM    617  CE  MET A  80     -12.192   7.852  13.079  1.00  0.00           C  \\nATOM    618  N   ILE A  81      -9.672   3.127  15.725  1.00  0.00           N  \\nATOM    619  CA  ILE A  81      -8.160   3.127  15.095  1.00  0.00           C  \\nATOM    620  C   ILE A  81      -7.908   4.009  13.835  1.00  0.00           C  \\nATOM    621  O   ILE A  81      -8.034   3.505  12.323  1.00  0.00           O  \\nATOM    622  CB  ILE A  81      -7.656   1.867  15.284  1.00  0.00           C  \\nATOM    623  CG1 ILE A  81      -6.522   1.867  16.103  1.00  0.00           C  \\nATOM    624  CG2 ILE A  81      -7.026   1.237  13.835  1.00  0.00           C  \\nATOM    625  CD1 ILE A  81      -6.900   2.245  17.615  1.00  0.00           C  \\nATOM    626  N   PHE A  82      -7.530   5.143  13.961  1.00  0.00           N  \\nATOM    627  CA  PHE A  82      -7.404   5.899  12.449  1.00  0.00           C  \\nATOM    628  C   PHE A  82      -6.144   6.529  12.197  1.00  0.00           C  \\nATOM    629  O   PHE A  82      -5.766   7.411  12.953  1.00  0.00           O  \\nATOM    630  CB  PHE A  82      -8.538   6.655  12.449  1.00  0.00           C  \\nATOM    631  CG  PHE A  82      -8.916   7.159  10.937  1.00  0.00           C  \\nATOM    632  CD1 PHE A  82      -8.916   8.293  10.433  1.00  0.00           C  \\nATOM    633  CD2 PHE A  82      -9.042   6.529  10.055  1.00  0.00           C  \\nATOM    634  CE1 PHE A  82      -9.042   8.671   9.173  1.00  0.00           C  \\nATOM    635  CE2 PHE A  82      -9.042   6.781   8.669  1.00  0.00           C  \\nATOM    636  CZ  PHE A  82      -9.042   7.852   8.291  1.00  0.00           C  \\nATOM    637  N   ALA A  83      -5.640   6.529  11.189  1.00  0.00           N  \\nATOM    638  CA  ALA A  83      -4.506   7.285  10.937  1.00  0.00           C  \\nATOM    639  C   ALA A  83      -4.758   8.797  10.811  1.00  0.00           C  \\nATOM    640  O   ALA A  83      -4.884   9.679  11.441  1.00  0.00           O  \\nATOM    641  CB  ALA A  83      -3.876   6.907   9.173  1.00  0.00           C  \\nATOM    642  N   GLY A  84      -5.136   9.301   9.677  1.00  0.00           N  \\nATOM    643  CA  GLY A  84      -5.325  10.687   9.299  1.00  0.00           C  \\nATOM    644  C   GLY A  84      -6.396  11.317   8.543  1.00  0.00           C  \\nATOM    645  O   GLY A  84      -7.467  11.569   8.921  1.00  0.00           O  \\nATOM    646  N   ILE A  85      -6.207  11.569   7.283  1.00  0.00           N  \\nATOM    647  CA  ILE A  85      -7.152  12.451   6.653  1.00  0.00           C  \\nATOM    648  C   ILE A  85      -6.774  13.963   6.905  1.00  0.00           C  \\nATOM    649  O   ILE A  85      -7.656  14.719   7.220  1.00  0.00           O  \\nATOM    650  CB  ILE A  85      -8.664  12.199   7.031  1.00  0.00           C  \\nATOM    651  CG1 ILE A  85      -9.168  10.813   6.149  1.00  0.00           C  \\nATOM    652  CG2 ILE A  85      -9.420  13.333   6.401  1.00  0.00           C  \\nATOM    653  CD1 ILE A  85     -10.176  11.191   5.204  1.00  0.00           C  \\nATOM    654  N   LYS A  86      -5.640  14.467   6.527  1.00  0.00           N  \\nATOM    655  CA  LYS A  86      -5.325  15.727   7.157  1.00  0.00           C  \\nATOM    656  C   LYS A  86      -5.136  16.546   5.771  1.00  0.00           C  \\nATOM    657  O   LYS A  86      -4.254  17.365   5.267  1.00  0.00           O  \\nATOM    658  CB  LYS A  86      -3.876  15.727   7.913  1.00  0.00           C  \\nATOM    659  CG  LYS A  86      -3.120  14.467   7.535  1.00  0.00           C  \\nATOM    660  CD  LYS A  86      -1.356  14.215   7.787  1.00  0.00           C  \\nATOM    661  CE  LYS A  86      -0.852  12.955   6.653  1.00  0.00           C  \\nATOM    662  NZ  LYS A  86       0.660  12.577   7.157  1.00  0.00           N  \\nATOM    663  N   LYS A  87      -5.892  16.357   5.015  1.00  0.00           N  \\nATOM    664  CA  LYS A  87      -5.892  17.239   3.755  1.00  0.00           C  \\nATOM    665  C   LYS A  87      -7.278  17.869   3.377  1.00  0.00           C  \\nATOM    666  O   LYS A  87      -8.160  17.239   2.873  1.00  0.00           O  \\nATOM    667  CB  LYS A  87      -5.514  16.294   2.558  1.00  0.00           C  \\nATOM    668  CG  LYS A  87      -4.632  16.735   1.235  1.00  0.00           C  \\nATOM    669  CD  LYS A  87      -4.254  15.727   0.353  1.00  0.00           C  \\nATOM    670  CE  LYS A  87      -3.120  16.231  -0.403  1.00  0.00           C  \\nATOM    671  NZ  LYS A  87      -2.868  15.727  -1.915  1.00  0.00           N  \\nATOM    672  N   LYS A  88      -7.278  19.129   3.251  1.00  0.00           N  \\nATOM    673  CA  LYS A  88      -8.412  19.885   2.621  1.00  0.00           C  \\nATOM    674  C   LYS A  88      -9.168  18.877   1.613  1.00  0.00           C  \\nATOM    675  O   LYS A  88     -10.428  18.499   1.613  1.00  0.00           O  \\nATOM    676  CB  LYS A  88      -7.782  21.019   1.991  1.00  0.00           C  \\nATOM    677  CG  LYS A  88      -6.270  20.767   1.865  1.00  0.00           C  \\nATOM    678  CD  LYS A  88      -5.514  21.271   0.605  1.00  0.00           C  \\nATOM    679  CE  LYS A  88      -5.766  20.767  -0.403  1.00  0.00           C  \\nATOM    680  NZ  LYS A  88      -5.388  21.523  -1.663  1.00  0.00           N  \\nATOM    681  N   GLY A  89      -8.538  18.499   0.353  1.00  0.00           N  \\nATOM    682  CA  GLY A  89      -9.168  17.491  -0.781  1.00  0.00           C  \\nATOM    683  C   GLY A  89     -10.050  16.357  -0.025  1.00  0.00           C  \\nATOM    684  O   GLY A  89     -11.184  16.357  -0.025  1.00  0.00           O  \\nATOM    685  N   GLU A  90      -9.546  15.475   0.479  1.00  0.00           N  \\nATOM    686  CA  GLU A  90     -10.050  14.089   1.109  1.00  0.00           C  \\nATOM    687  C   GLU A  90     -10.932  14.593   2.369  1.00  0.00           C  \\nATOM    688  O   GLU A  90     -12.192  14.341   2.621  1.00  0.00           O  \\nATOM    689  CB  GLU A  90      -9.042  13.207   1.613  1.00  0.00           C  \\nATOM    690  CG  GLU A  90      -8.286  12.577   0.605  1.00  0.00           C  \\nATOM    691  CD  GLU A  90      -6.900  11.821   0.857  1.00  0.00           C  \\nATOM    692  OE1 GLU A  90      -6.018  11.443  -0.277  1.00  0.00           O  \\nATOM    693  OE2 GLU A  90      -6.270  11.569   2.243  1.00  0.00           O  \\nATOM    694  N   ARG A  91     -10.554  15.475   2.999  1.00  0.00           N  \\nATOM    695  CA  ARG A  91     -11.184  15.916   4.511  1.00  0.00           C  \\nATOM    696  C   ARG A  91     -12.570  16.672   3.881  1.00  0.00           C  \\nATOM    697  O   ARG A  91     -13.578  16.357   4.133  1.00  0.00           O  \\nATOM    698  CB  ARG A  91     -10.302  16.987   5.141  1.00  0.00           C  \\nATOM    699  CG  ARG A  91     -10.554  17.113   6.527  1.00  0.00           C  \\nATOM    700  CD  ARG A  91      -9.420  18.121   7.283  1.00  0.00           C  \\nATOM    701  NE  ARG A  91      -9.294  19.381   6.527  1.00  0.00           N  \\nATOM    702  CZ  ARG A  91      -7.908  20.137   6.464  1.00  0.00           C  \\nATOM    703  NH1 ARG A  91      -6.900  19.507   7.535  1.00  0.00           N  \\nATOM    704  NH2 ARG A  91      -7.530  21.145   5.519  1.00  0.00           N  \\nATOM    705  N   GLN A  92     -12.444  17.743   3.251  1.00  0.00           N  \\nATOM    706  CA  GLN A  92     -13.830  18.373   2.369  1.00  0.00           C  \\nATOM    707  C   GLN A  92     -14.586  17.113   1.991  1.00  0.00           C  \\nATOM    708  O   GLN A  92     -15.720  16.735   2.243  1.00  0.00           O  \\nATOM    709  CB  GLN A  92     -13.704  19.633   1.739  1.00  0.00           C  \\nATOM    710  CG  GLN A  92     -13.830  19.255   0.164  1.00  0.00           C  \\nATOM    711  CD  GLN A  92     -13.200  20.326  -0.907  1.00  0.00           C  \\nATOM    712  OE1 GLN A  92     -11.688  19.885  -0.403  1.00  0.00           O  \\nATOM    713  NE2 GLN A  92     -13.452  21.460  -1.411  1.00  0.00           N  \\nATOM    714  N   ASP A  93     -13.956  16.357   0.983  1.00  0.00           N  \\nATOM    715  CA  ASP A  93     -14.838  15.223   0.605  1.00  0.00           C  \\nATOM    716  C   ASP A  93     -15.342  14.341   1.613  1.00  0.00           C  \\nATOM    717  O   ASP A  93     -16.602  13.837   1.487  1.00  0.00           O  \\nATOM    718  CB  ASP A  93     -13.830  14.593  -0.529  1.00  0.00           C  \\nATOM    719  CG  ASP A  93     -13.704  14.971  -1.915  1.00  0.00           C  \\nATOM    720  OD1 ASP A  93     -12.444  15.727  -2.419  1.00  0.00           O  \\nATOM    721  OD2 ASP A  93     -14.838  15.223  -2.797  1.00  0.00           O  \\nATOM    722  N   LEU A  94     -14.838  13.963   2.495  1.00  0.00           N  \\nATOM    723  CA  LEU A  94     -15.531  13.207   3.503  1.00  0.00           C  \\nATOM    724  C   LEU A  94     -16.728  13.963   3.881  1.00  0.00           C  \\nATOM    725  O   LEU A  94     -17.925  13.459   3.881  1.00  0.00           O  \\nATOM    726  CB  LEU A  94     -14.586  13.018   4.763  1.00  0.00           C  \\nATOM    727  CG  LEU A  94     -14.586  11.443   5.267  1.00  0.00           C  \\nATOM    728  CD1 LEU A  94     -15.783  10.813   4.637  1.00  0.00           C  \\nATOM    729  CD2 LEU A  94     -13.578  10.561   4.889  1.00  0.00           C  \\nATOM    730  N   VAL A  95     -16.539  15.097   4.637  1.00  0.00           N  \\nATOM    731  CA  VAL A  95     -17.736  15.979   5.141  1.00  0.00           C  \\nATOM    732  C   VAL A  95     -18.744  15.853   3.881  1.00  0.00           C  \\nATOM    733  O   VAL A  95     -20.004  15.601   4.322  1.00  0.00           O  \\nATOM    734  CB  VAL A  95     -17.358  17.365   5.393  1.00  0.00           C  \\nATOM    735  CG1 VAL A  95     -18.492  18.184   5.519  1.00  0.00           C  \\nATOM    736  CG2 VAL A  95     -16.476  17.491   6.401  1.00  0.00           C  \\nATOM    737  N   ALA A  96     -18.492  15.979   2.621  1.00  0.00           N  \\nATOM    738  CA  ALA A  96     -19.752  15.601   1.739  1.00  0.00           C  \\nATOM    739  C   ALA A  96     -20.130  14.215   1.802  1.00  0.00           C  \\nATOM    740  O   ALA A  96     -21.390  13.837   1.802  1.00  0.00           O  \\nATOM    741  CB  ALA A  96     -19.500  16.105   0.353  1.00  0.00           C  \\nATOM    742  N   TYR A  97     -19.248  13.333   1.865  1.00  0.00           N  \\nATOM    743  CA  TYR A  97     -19.626  11.884   1.865  1.00  0.00           C  \\nATOM    744  C   TYR A  97     -20.886  11.632   2.873  1.00  0.00           C  \\nATOM    745  O   TYR A  97     -22.146  11.317   2.621  1.00  0.00           O  \\nATOM    746  CB  TYR A  97     -18.744  10.813   2.243  1.00  0.00           C  \\nATOM    747  CG  TYR A  97     -19.374   9.427   1.739  1.00  0.00           C  \\nATOM    748  CD1 TYR A  97     -19.626   8.419   2.621  1.00  0.00           C  \\nATOM    749  CD2 TYR A  97     -19.626   9.175   0.353  1.00  0.00           C  \\nATOM    750  CE1 TYR A  97     -20.256   7.348   2.117  1.00  0.00           C  \\nATOM    751  CE2 TYR A  97     -20.256   8.041  -0.403  1.00  0.00           C  \\nATOM    752  CZ  TYR A  97     -20.508   7.285   0.857  1.00  0.00           C  \\nATOM    753  OH  TYR A  97     -21.264   6.403   0.353  1.00  0.00           O  \\nATOM    754  N   LEU A  98     -20.760  11.947   4.007  1.00  0.00           N  \\nATOM    755  CA  LEU A  98     -21.768  11.695   5.393  1.00  0.00           C  \\nATOM    756  C   LEU A  98     -23.154  12.703   5.015  1.00  0.00           C  \\nATOM    757  O   LEU A  98     -24.288  12.199   4.889  1.00  0.00           O  \\nATOM    758  CB  LEU A  98     -21.012  11.695   6.527  1.00  0.00           C  \\nATOM    759  CG  LEU A  98     -20.004  10.939   6.401  1.00  0.00           C  \\nATOM    760  CD1 LEU A  98     -19.374  10.687   7.661  1.00  0.00           C  \\nATOM    761  CD2 LEU A  98     -20.508   9.553   5.771  1.00  0.00           C  \\nATOM    762  N   LYS A  99     -23.028  13.963   4.889  1.00  0.00           N  \\nATOM    763  CA  LYS A  99     -24.288  14.593   4.385  1.00  0.00           C  \\nATOM    764  C   LYS A  99     -25.044  13.963   2.999  1.00  0.00           C  \\nATOM    765  O   LYS A  99     -26.304  13.837   3.125  1.00  0.00           O  \\nATOM    766  CB  LYS A  99     -23.910  16.105   4.259  1.00  0.00           C  \\nATOM    767  CG  LYS A  99     -24.918  17.113   4.259  1.00  0.00           C  \\nATOM    768  CD  LYS A  99     -24.666  18.625   3.881  1.00  0.00           C  \\nATOM    769  CE  LYS A  99     -25.674  19.633   4.133  1.00  0.00           C  \\nATOM    770  NZ  LYS A  99     -25.674  21.082   3.755  1.00  0.00           N  \\nATOM    771  N   SER A 100     -24.540  13.459   1.991  1.00  0.00           N  \\nATOM    772  CA  SER A 100     -25.296  12.577   0.857  1.00  0.00           C  \\nATOM    773  C   SER A 100     -25.548  11.191   1.361  1.00  0.00           C  \\nATOM    774  O   SER A 100     -26.934  10.813   1.361  1.00  0.00           O  \\nATOM    775  CB  SER A 100     -24.540  12.577  -0.655  1.00  0.00           C  \\nATOM    776  OG  SER A 100     -25.296  11.695  -1.663  1.00  0.00           O  \\nATOM    777  N   ALA A 101     -24.666  10.435   1.865  1.00  0.00           N  \\nATOM    778  CA  ALA A 101     -25.044   8.797   2.243  1.00  0.00           C  \\nATOM    779  C   ALA A 101     -25.800   8.923   3.629  1.00  0.00           C  \\nATOM    780  O   ALA A 101     -26.808   8.293   3.755  1.00  0.00           O  \\nATOM    781  CB  ALA A 101     -23.910   8.167   2.369  1.00  0.00           C  \\nATOM    782  N   THR A 102     -25.422   9.427   4.637  1.00  0.00           N  \\nATOM    783  CA  THR A 102     -26.052   9.301   6.149  1.00  0.00           C  \\nATOM    784  C   THR A 102     -27.690   9.805   5.897  1.00  0.00           C  \\nATOM    785  O   THR A 102     -28.572   9.049   6.149  1.00  0.00           O  \\nATOM    786  CB  THR A 102     -25.548  10.246   7.220  1.00  0.00           C  \\nATOM    787  OG1 THR A 102     -24.666   9.427   7.913  1.00  0.00           O  \\nATOM    788  CG2 THR A 102     -26.430  10.876   8.039  1.00  0.00           C  \\nATOM    789  N   SER A 103     -27.942  10.939   5.645  1.00  0.00           N  \\nATOM    790  CA  SER A 103     -29.454  11.317   5.519  1.00  0.00           C  \\nATOM    791  C   SER A 103     -30.084  10.813   4.385  1.00  0.00           C  \\nATOM    792  O   SER A 103     -29.454   9.931   3.377  1.00  0.00           O  \\nATOM    793  CB  SER A 103     -29.580  12.892   5.897  1.00  0.00           C  \\nATOM    794  OG  SER A 103     -28.320  13.585   5.393  1.00  0.00           O  \\nATOM    795  OXT SER A 103     -31.344  11.317   4.133  1.00  0.00           O  \\nTER     796      SER A 103                                                      \\nHETATM 1593 FE   HEC A 104     -15.216   5.773  12.701  1.00  0.00          FE  \\nHETATM 1594  CHA HEC A 104     -16.980   6.466  15.347  1.00  0.00           C  \\nHETATM 1595  CHB HEC A 104     -16.476   8.293  11.189  1.00  0.00           C  \\nHETATM 1596  CHC HEC A 104     -13.578   5.017   9.929  1.00  0.00           C  \\nHETATM 1597  CHD HEC A 104     -13.956   3.127  14.339  1.00  0.00           C  \\nHETATM 1598  NA  HEC A 104     -16.476   7.096  13.205  1.00  0.00           N  \\nHETATM 1599  C1A HEC A 104     -17.106   7.159  14.465  1.00  0.00           C  \\nHETATM 1600  C2A HEC A 104     -17.862   8.167  14.591  1.00  0.00           C  \\nHETATM 1601  C3A HEC A 104     -17.736   8.797  13.205  1.00  0.00           C  \\nHETATM 1602  C4A HEC A 104     -17.114   8.167  12.323  1.00  0.00           C  \\nHETATM 1603  CMA HEC A 104     -18.303  10.057  12.701  1.00  0.00           C  \\nHETATM 1604  CAA HEC A 104     -18.618   8.167  15.599  1.00  0.00           C  \\nHETATM 1605  CBA HEC A 104     -20.067   8.734  15.473  1.00  0.00           C  \\nHETATM 1606  CGA HEC A 104     -20.823   8.734  16.859  1.00  0.00           C  \\nHETATM 1607  O1A HEC A 104     -21.705   8.041  17.111  1.00  0.00           O  \\nHETATM 1608  O2A HEC A 104     -20.508   9.490  17.741  1.00  0.00           O  \\nHETATM 1609  NB  HEC A 104     -15.090   6.466  11.126  1.00  0.00           N  \\nHETATM 1610  C1B HEC A 104     -15.594   7.348  10.559  1.00  0.00           C  \\nHETATM 1611  C2B HEC A 104     -15.216   7.663   9.299  1.00  0.00           C  \\nHETATM 1612  C3B HEC A 104     -14.334   6.592   8.921  1.00  0.00           C  \\nHETATM 1613  C4B HEC A 104     -14.334   6.025   9.929  1.00  0.00           C  \\nHETATM 1614  CMB HEC A 104     -15.657   8.797   8.543  1.00  0.00           C  \\nHETATM 1615  CAB HEC A 104     -13.578   6.781   7.409  1.00  0.00           C  \\nHETATM 1616  CBB HEC A 104     -12.570   7.663   7.661  1.00  0.00           C  \\nHETATM 1617  NC  HEC A 104     -13.956   4.387  12.197  1.00  0.00           N  \\nHETATM 1618  C1C HEC A 104     -13.452   4.261  10.811  1.00  0.00           C  \\nHETATM 1619  C2C HEC A 104     -12.759   3.001  10.811  1.00  0.00           C  \\nHETATM 1620  C3C HEC A 104     -12.948   2.623  11.945  1.00  0.00           C  \\nHETATM 1621  C4C HEC A 104     -13.704   3.379  12.953  1.00  0.00           C  \\nHETATM 1622  CMC HEC A 104     -11.688   2.497   9.803  1.00  0.00           C  \\nHETATM 1623  CAC HEC A 104     -12.444   1.111  12.197  1.00  0.00           C  \\nHETATM 1624  CBC HEC A 104     -11.184   0.859  12.008  1.00  0.00           C  \\nHETATM 1625  ND  HEC A 104     -15.405   5.143  14.213  1.00  0.00           N  \\nHETATM 1626  C1D HEC A 104     -14.838   4.009  14.843  1.00  0.00           C  \\nHETATM 1627  C2D HEC A 104     -15.216   3.757  15.977  1.00  0.00           C  \\nHETATM 1628  C3D HEC A 104     -16.098   4.765  16.355  1.00  0.00           C  \\nHETATM 1629  C4D HEC A 104     -16.161   5.395  15.347  1.00  0.00           C  \\nHETATM 1630  CMD HEC A 104     -14.838   2.749  16.733  1.00  0.00           C  \\nHETATM 1631  CAD HEC A 104     -16.854   4.513  17.615  1.00  0.00           C  \\nHETATM 1632  CBD HEC A 104     -16.098   4.135  18.623  1.00  0.00           C  \\nHETATM 1633  CGD HEC A 104     -15.216   5.017  19.505  1.00  0.00           C  \\nHETATM 1634  O1D HEC A 104     -14.082   5.773  18.560  1.00  0.00           O  \\nHETATM 1635  O2D HEC A 104     -15.405   5.647  20.639  1.00  0.00           O  \\nHETATM 1679  O   HOH A 105     -16.496   9.304  17.402  1.00  0.00           O  \\nCONECT  104 1615                                                                \\nCONECT  124 1623                                                                \\nCONECT  134 1593                                                                \\nCONECT  356 1635                                                                \\nCONECT  616 1593                                                                \\nCONECT  900 1658                                                                \\nCONECT  920 1666                                                                \\nCONECT  930 1636                                                                \\nCONECT 1152 1678                                                                \\nCONECT 1412 1636                                                                \\nCONECT 1593  134  616 1598 1609                                                 \\nCONECT 1593 1617 1625                                                           \\nCONECT 1594 1599 1629                                                           \\nCONECT 1595 1602 1610                                                           \\nCONECT 1596 1613 1618                                                           \\nCONECT 1597 1621 1626                                                           \\nCONECT 1598 1593 1599 1602                                                      \\nCONECT 1599 1594 1598 1600                                                      \\nCONECT 1600 1599 1601 1604                                                      \\nCONECT 1601 1600 1602 1603                                                      \\nCONECT 1602 1595 1598 1601                                                      \\nCONECT 1603 1601                                                                \\nCONECT 1604 1600 1605                                                           \\nCONECT 1605 1604 1606                                                           \\nCONECT 1606 1605 1607 1608                                                      \\nCONECT 1607 1606                                                                \\nCONECT 1608 1606                                                                \\nCONECT 1609 1593 1610 1613                                                      \\nCONECT 1610 1595 1609 1611                                                      \\nCONECT 1611 1610 1612 1614                                                      \\nCONECT 1612 1611 1613 1615                                                      \\nCONECT 1613 1596 1609 1612                                                      \\nCONECT 1614 1611                                                                \\nCONECT 1615  104 1612 1616                                                      \\nCONECT 1616 1615                                                                \\nCONECT 1617 1593 1618 1621                                                      \\nCONECT 1618 1596 1617 1619                                                      \\nCONECT 1619 1618 1620 1622                                                      \\nCONECT 1620 1619 1621 1623                                                      \\nCONECT 1621 1597 1617 1620                                                      \\nCONECT 1622 1619                                                                \\nCONECT 1623  124 1620 1624                                                      \\nCONECT 1624 1623                                                                \\nCONECT 1625 1593 1626 1629                                                      \\nCONECT 1626 1597 1625 1627                                                      \\nCONECT 1627 1626 1628 1630                                                      \\nCONECT 1628 1627 1629 1631                                                      \\nCONECT 1629 1594 1625 1628                                                      \\nCONECT 1630 1627                                                                \\nCONECT 1631 1628 1632                                                           \\nCONECT 1632 1631 1633                                                           \\nCONECT 1633 1632 1634 1635                                                      \\nCONECT 1634 1633                                                                \\nCONECT 1635  356 1633                                                           \\nCONECT 1636  930 1412 1641 1652                                                 \\nCONECT 1636 1660 1668                                                           \\nCONECT 1637 1642 1672                                                           \\nCONECT 1638 1645 1653                                                           \\nCONECT 1639 1656 1661                                                           \\nCONECT 1640 1664 1669                                                           \\nCONECT 1641 1636 1642 1645                                                      \\nCONECT 1642 1637 1641 1643                                                      \\nCONECT 1643 1642 1644 1647                                                      \\nCONECT 1644 1643 1645 1646                                                      \\nCONECT 1645 1638 1641 1644                                                      \\nCONECT 1646 1644                                                                \\nCONECT 1647 1643 1648                                                           \\nCONECT 1648 1647 1649                                                           \\nCONECT 1649 1648 1650 1651                                                      \\nCONECT 1650 1649                                                                \\nCONECT 1651 1649                                                                \\nCONECT 1652 1636 1653 1656                                                      \\nCONECT 1653 1638 1652 1654                                                      \\nCONECT 1654 1653 1655 1657                                                      \\nCONECT 1655 1654 1656 1658                                                      \\nCONECT 1656 1639 1652 1655                                                      \\nCONECT 1657 1654                                                                \\nCONECT 1658  900 1655 1659                                                      \\nCONECT 1659 1658                                                                \\nCONECT 1660 1636 1661 1664                                                      \\nCONECT 1661 1639 1660 1662                                                      \\nCONECT 1662 1661 1663 1665                                                      \\nCONECT 1663 1662 1664 1666                                                      \\nCONECT 1664 1640 1660 1663                                                      \\nCONECT 1665 1662                                                                \\nCONECT 1666  920 1663 1667                                                      \\nCONECT 1667 1666                                                                \\nCONECT 1668 1636 1669 1672                                                      \\nCONECT 1669 1640 1668 1670                                                      \\nCONECT 1670 1669 1671 1673                                                      \\nCONECT 1671 1670 1672 1674                                                      \\nCONECT 1672 1637 1668 1671                                                      \\nCONECT 1673 1670                                                                \\nCONECT 1674 1671 1675                                                           \\nCONECT 1675 1674 1676                                                           \\nCONECT 1676 1675 1677 1678                                                      \\nCONECT 1677 1676                                                                \\nCONECT 1678 1152 1676                                                           \\nMASTER      287    0    2   10    0    0    9    9 1677    2   98   16          \\nEND                                                                             \\n\",\"pdb\");\n",
       "\tviewer_16742557823931737.setBackgroundColor(\"white\");\n",
       "\tviewer_16742557823931737.setStyle({\"cartoon\": {\"color\": \"white\"}});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"1.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"5.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"2.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"5.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"2.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"6.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"2.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"88.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"2.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"91.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"2.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"92.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"2.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"99.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"4.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"7.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"5.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"8.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"5.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"12.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"5.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"88.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"5.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"91.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"5.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"92.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"5.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"6.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"13.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"6.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"7.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"11.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"7.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"12.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"7.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"15.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"7.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"92.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"8.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"11.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"8.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"13.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"8.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"15.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"9.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"90.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"9.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"11.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"15.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"13.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"16.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"13.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"81.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"13.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"82.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"13.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"84.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"13.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"89.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"13.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"14.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"19.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"22.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"19.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"30.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"21.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"30.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"22.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"25.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"22.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"29.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"22.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"30.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"22.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"31.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"22.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"42.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"23.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"26.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"23.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"30.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"24.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"30.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"25.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"30.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"26.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"29.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"26.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"30.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"27.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"80.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"29.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"46.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"30.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"33.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"30.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"42.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"30.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"43.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"37.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"42.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"38.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"55.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"42.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"45.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"50.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"51.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"52.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"53.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"54.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"55.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"56.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"57.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"62.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"66.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"74.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"77.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"40.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"50.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"40.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"55.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"46.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"51.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"47.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"50.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"49.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"54.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"49.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"55.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"49.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"75.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"50.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"53.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"50.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"54.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"50.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"55.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"50.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"70.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"50.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"74.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"50.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"75.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"50.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"76.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"50.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"77.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"50.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"78.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"50.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"79.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"51.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"55.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"52.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"55.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"54.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"75.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"55.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"74.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"61.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"65.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"61.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"66.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"62.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"66.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"63.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"90.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"63.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"64.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"65.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"69.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"66.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"69.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"66.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"70.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"66.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"73.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"66.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"74.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"66.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"81.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"66.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"90.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"66.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"68.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"69.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"72.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"69.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"73.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"69.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"75.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"69.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"81.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"69.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"82.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"69.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"83.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"69.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"84.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"69.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"90.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"70.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"73.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"70.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"74.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"70.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"75.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"70.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"76.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"70.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"77.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"70.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"78.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"70.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"79.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"70.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"80.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"70.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"81.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"70.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"82.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"70.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"83.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"70.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"84.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"71.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"75.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"71.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"81.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"71.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"82.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"77.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"80.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"77.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"81.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"81.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"84.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"83.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"90.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"84.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"90.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"84.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"86.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"90.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"86.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"87.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"90.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"88.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"89.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"90.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93.0\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"3.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"95.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"3.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"96.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"3.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"97.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"3.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"3.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"99.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"9.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"14.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"9.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"96.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"14.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"15.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"16.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"18.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"20.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"96.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"97.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"11.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"14.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"12.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"16.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"14.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"18.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"14.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"20.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"14.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"96.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"14.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"97.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"15.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"20.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"18.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"28.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"20.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"23.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"20.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"24.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"20.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"32.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"20.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"33.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"20.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"97.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"20.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"101.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"25.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"28.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"25.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"45.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"28.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"45.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"28.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"78.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"28.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"79.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"32.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"35.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"32.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"36.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"33.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"36.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"33.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"58.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"33.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"35.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"58.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"35.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"59.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"35.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"60.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"35.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"97.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"35.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"35.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"101.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"35.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"102.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"36.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"57.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"36.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"58.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"36.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"59.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"37.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"58.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"40.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"58.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"41.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"44.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"41.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"45.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"58.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"61.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"58.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"62.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"58.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"65.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"58.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"66.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"58.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"97.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"58.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"101.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"59.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"64.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"62.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"68.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"64.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"67.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"64.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"68.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"64.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"65.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"68.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"65.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"72.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"67.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"85.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"67.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"67.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"96.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"67.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"97.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"68.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"72.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"68.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"83.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"68.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"85.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"68.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"86.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"85.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"89.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"94.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"97.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"94.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"95.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"95.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"99.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"96.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"99.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"96.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"101.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"98.0\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"101.0\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742557823931737.zoomTo();\n",
       "viewer_16742557823931737.render();\n",
       "});\n",
       "</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "<py3Dmol.view at 0x7f572233f610>"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "view_frustration_(pdb_file_chain,model_all,\n",
    "                  contact_map=np.triu(model_all.mask[:len(model_all.sequence),:len(model_all.sequence)]))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "ename": "NameError",
     "evalue": "name 'r1' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "Cell \u001b[0;32mIn[23], line 6\u001b[0m\n\u001b[1;32m      3\u001b[0m residues\u001b[38;5;241m=\u001b[39mnp\u001b[38;5;241m.\u001b[39marange(\u001b[38;5;28mlen\u001b[39m(model_all\u001b[38;5;241m.\u001b[39msequence))\n\u001b[1;32m      4\u001b[0m pair_frustration\u001b[38;5;241m=\u001b[39mmodel_all\u001b[38;5;241m.\u001b[39mfrustration(\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mmutational\u001b[39m\u001b[38;5;124m'\u001b[39m)\u001b[38;5;241m*\u001b[39mcontact_map\n\u001b[0;32m----> 6\u001b[0m all_contacts\u001b[38;5;241m=\u001b[39mnp\u001b[38;5;241m.\u001b[39marray([\u001b[43mr1\u001b[49m\u001b[38;5;241m.\u001b[39mravel(), r2\u001b[38;5;241m.\u001b[39mravel(), contact_map\u001b[38;5;241m.\u001b[39mravel()])\u001b[38;5;241m.\u001b[39mT\n\u001b[1;32m      7\u001b[0m all_contacts\u001b[38;5;241m=\u001b[39mall_contacts[all_contacts[:,\u001b[38;5;241m2\u001b[39m]\u001b[38;5;241m==\u001b[39m\u001b[38;5;241m1\u001b[39m]\n\u001b[1;32m      8\u001b[0m \u001b[38;5;28mlen\u001b[39m(all_contacts)\n",
      "\u001b[0;31mNameError\u001b[0m: name 'r1' is not defined"
     ]
    }
   ],
   "source": [
    "#sns.heatmap(model_all.mask)\n",
    "\n",
    "residues=np.arange(len(model_all.sequence))\n",
    "pair_frustration=model_all.frustration('mutational')*contact_map\n",
    "\n",
    "all_contacts=np.array([r1.ravel(), r2.ravel(), contact_map.ravel()]).T\n",
    "all_contacts=all_contacts[all_contacts[:,2]==1]\n",
    "len(all_contacts)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [
    {
     "ename": "NameError",
     "evalue": "name 'all_contacts' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "Cell \u001b[0;32mIn[24], line 1\u001b[0m\n\u001b[0;32m----> 1\u001b[0m frustra_table\u001b[38;5;241m=\u001b[39mpd\u001b[38;5;241m.\u001b[39mDataFrame(\u001b[43mall_contacts\u001b[49m)\n\u001b[1;32m      2\u001b[0m frustra_table\n",
      "\u001b[0;31mNameError\u001b[0m: name 'all_contacts' is not defined"
     ]
    }
   ],
   "source": [
    "frustra_table=pd.DataFrame(all_contacts)\n",
    "frustra_table"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "ename": "NameError",
     "evalue": "name 'residues' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "Cell \u001b[0;32mIn[22], line 1\u001b[0m\n\u001b[0;32m----> 1\u001b[0m r1, r2 \u001b[38;5;241m=\u001b[39m np\u001b[38;5;241m.\u001b[39mmeshgrid(\u001b[43mresidues\u001b[49m, residues, indexing\u001b[38;5;241m=\u001b[39m\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mij\u001b[39m\u001b[38;5;124m'\u001b[39m)\n\u001b[1;32m      2\u001b[0m sel_frustration \u001b[38;5;241m=\u001b[39m np\u001b[38;5;241m.\u001b[39marray([r1\u001b[38;5;241m.\u001b[39mravel(), r2\u001b[38;5;241m.\u001b[39mravel(), pair_frustration\u001b[38;5;241m.\u001b[39mravel()])\u001b[38;5;241m.\u001b[39mT\n\u001b[1;32m      3\u001b[0m minimally_frustrated \u001b[38;5;241m=\u001b[39m sel_frustration[sel_frustration[:, \u001b[38;5;241m-\u001b[39m\u001b[38;5;241m1\u001b[39m] \u001b[38;5;241m<\u001b[39m \u001b[38;5;241m-\u001b[39m\u001b[38;5;241m0.78\u001b[39m]\n",
      "\u001b[0;31mNameError\u001b[0m: name 'residues' is not defined"
     ]
    }
   ],
   "source": [
    "r1, r2 = np.meshgrid(residues, residues, indexing='ij')\n",
    "sel_frustration = np.array([r1.ravel(), r2.ravel(), pair_frustration.ravel()]).T\n",
    "minimally_frustrated = sel_frustration[sel_frustration[:, -1] < -0.78]\n",
    "\n",
    "s = np.argsort(minimally_frustrated[:, -1])\n",
    "print(minimally_frustrated[s].shape)\n",
    "minimally_frustrated = minimally_frustrated[s]\n",
    "\n",
    "frustrated = sel_frustration[sel_frustration[:, -1] > 1]\n",
    "#s = np.argsort(frustrated[:, -1])[::-1]\n",
    "print(frustrated[s].shape)\n",
    "\n",
    "frustrated = frustrated[s]\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 173 atoms and 1 coordinate set(s) were parsed in 0.00s.\n"
     ]
    }
   ],
   "source": [
    "path_bs=path_f+'pdb_files_'+bs_name+'/'\n",
    "model_exon=dca_frustratometer.AWSEMFrustratometer(path_bs+'exon_0.pdb',\n",
    "                                                  chain='A',sequence_cutoff=3,  distance_cutoff=10)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "ename": "IndexError",
     "evalue": "index 22 is out of bounds for axis 0 with size 22",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mIndexError\u001b[0m                                Traceback (most recent call last)",
      "Cell \u001b[0;32mIn[18], line 2\u001b[0m\n\u001b[1;32m      1\u001b[0m contact_map\u001b[38;5;241m=\u001b[39mnp\u001b[38;5;241m.\u001b[39mtriu(model_exon\u001b[38;5;241m.\u001b[39mmask)\n\u001b[0;32m----> 2\u001b[0m pair_frustration\u001b[38;5;241m=\u001b[39m\u001b[43mmodel_exon\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mfrustration\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[38;5;124;43mmutational\u001b[39;49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[43m)\u001b[49m\u001b[38;5;241m*\u001b[39mcontact_map\n\u001b[1;32m      3\u001b[0m sns\u001b[38;5;241m.\u001b[39mheatmap(pair_frustration,cmap\u001b[38;5;241m=\u001b[39m\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mvlag\u001b[39m\u001b[38;5;124m'\u001b[39m,center\u001b[38;5;241m=\u001b[39m\u001b[38;5;241m0\u001b[39m)\n",
      "File \u001b[0;32m~/Downloads/DCA_Frustratometer-629aa7ca25e37a3818fc968d45950160697b2cd6/dca_frustratometer/classes/DCA.py:400\u001b[0m, in \u001b[0;36mPottsModel.frustration\u001b[0;34m(self, kind, aa_freq, correction)\u001b[0m\n\u001b[1;32m    399\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21mfrustration\u001b[39m(\u001b[38;5;28mself\u001b[39m, kind\u001b[38;5;241m=\u001b[39m\u001b[38;5;124m'\u001b[39m\u001b[38;5;124msingleresidue\u001b[39m\u001b[38;5;124m'\u001b[39m, aa_freq\u001b[38;5;241m=\u001b[39m\u001b[38;5;28;01mNone\u001b[39;00m, correction\u001b[38;5;241m=\u001b[39m\u001b[38;5;241m0\u001b[39m):\n\u001b[0;32m--> 400\u001b[0m     decoy_fluctuation \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mdecoy_fluctuation\u001b[49m\u001b[43m(\u001b[49m\u001b[43mkind\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    401\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m kind \u001b[38;5;241m==\u001b[39m \u001b[38;5;124m'\u001b[39m\u001b[38;5;124msingleresidue\u001b[39m\u001b[38;5;124m'\u001b[39m:\n\u001b[1;32m    402\u001b[0m         \u001b[38;5;28;01mif\u001b[39;00m aa_freq \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m:\n",
      "File \u001b[0;32m~/Downloads/DCA_Frustratometer-629aa7ca25e37a3818fc968d45950160697b2cd6/dca_frustratometer/classes/DCA.py:382\u001b[0m, in \u001b[0;36mPottsModel.decoy_fluctuation\u001b[0;34m(self, kind)\u001b[0m\n\u001b[1;32m    380\u001b[0m     fluctuation \u001b[38;5;241m=\u001b[39m frustration\u001b[38;5;241m.\u001b[39mcompute_singleresidue_decoy_energy_fluctuation(\u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39msequence, \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mpotts_model, \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mmask)\n\u001b[1;32m    381\u001b[0m \u001b[38;5;28;01melif\u001b[39;00m kind \u001b[38;5;241m==\u001b[39m \u001b[38;5;124m'\u001b[39m\u001b[38;5;124mmutational\u001b[39m\u001b[38;5;124m'\u001b[39m:\n\u001b[0;32m--> 382\u001b[0m     fluctuation \u001b[38;5;241m=\u001b[39m \u001b[43mfrustration\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mcompute_mutational_decoy_energy_fluctuation\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43msequence\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mpotts_model\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mmask\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    383\u001b[0m \u001b[38;5;28;01melif\u001b[39;00m kind \u001b[38;5;241m==\u001b[39m \u001b[38;5;124m'\u001b[39m\u001b[38;5;124mconfigurational\u001b[39m\u001b[38;5;124m'\u001b[39m:\n\u001b[1;32m    384\u001b[0m     fluctuation \u001b[38;5;241m=\u001b[39m frustration\u001b[38;5;241m.\u001b[39mcompute_configurational_decoy_energy_fluctuation(\u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39msequence, \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mpotts_model, \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mmask)\n",
      "File \u001b[0;32m~/Downloads/DCA_Frustratometer-629aa7ca25e37a3818fc968d45950160697b2cd6/dca_frustratometer/frustration/frustratometer.py:146\u001b[0m, in \u001b[0;36mcompute_mutational_decoy_energy_fluctuation\u001b[0;34m(seq, potts_model, mask)\u001b[0m\n\u001b[1;32m    142\u001b[0m pos1, pos2, aa1, aa2 \u001b[38;5;241m=\u001b[39m np\u001b[38;5;241m.\u001b[39mmeshgrid(np\u001b[38;5;241m.\u001b[39marange(seq_len), np\u001b[38;5;241m.\u001b[39marange(seq_len), np\u001b[38;5;241m.\u001b[39marange(\u001b[38;5;241m21\u001b[39m), np\u001b[38;5;241m.\u001b[39marange(\u001b[38;5;241m21\u001b[39m),\n\u001b[1;32m    143\u001b[0m                                    indexing\u001b[38;5;241m=\u001b[39m\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mij\u001b[39m\u001b[38;5;124m'\u001b[39m, sparse\u001b[38;5;241m=\u001b[39m\u001b[38;5;28;01mTrue\u001b[39;00m)\n\u001b[1;32m    145\u001b[0m decoy_energy \u001b[38;5;241m=\u001b[39m np\u001b[38;5;241m.\u001b[39mzeros([seq_len, seq_len, \u001b[38;5;241m21\u001b[39m, \u001b[38;5;241m21\u001b[39m])\n\u001b[0;32m--> 146\u001b[0m decoy_energy \u001b[38;5;241m-\u001b[39m\u001b[38;5;241m=\u001b[39m (\u001b[43mpotts_model\u001b[49m\u001b[43m[\u001b[49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[38;5;124;43mh\u001b[39;49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[43m]\u001b[49m\u001b[43m[\u001b[49m\u001b[43mpos1\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43maa1\u001b[49m\u001b[43m]\u001b[49m \u001b[38;5;241m-\u001b[39m potts_model[\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mh\u001b[39m\u001b[38;5;124m'\u001b[39m][pos1, seq_index[pos1]])  \u001b[38;5;66;03m# h correction aa1\u001b[39;00m\n\u001b[1;32m    147\u001b[0m decoy_energy \u001b[38;5;241m-\u001b[39m\u001b[38;5;241m=\u001b[39m (potts_model[\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mh\u001b[39m\u001b[38;5;124m'\u001b[39m][pos2, aa2] \u001b[38;5;241m-\u001b[39m potts_model[\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mh\u001b[39m\u001b[38;5;124m'\u001b[39m][pos2, seq_index[pos2]])  \u001b[38;5;66;03m# h correction aa2\u001b[39;00m\n\u001b[1;32m    149\u001b[0m j_correction \u001b[38;5;241m=\u001b[39m np\u001b[38;5;241m.\u001b[39mzeros([seq_len, seq_len, \u001b[38;5;241m21\u001b[39m, \u001b[38;5;241m21\u001b[39m])\n",
      "\u001b[0;31mIndexError\u001b[0m: index 22 is out of bounds for axis 0 with size 22"
     ]
    }
   ],
   "source": [
    "contact_map=np.triu(model_exon.mask)\n",
    "pair_frustration=model_exon.frustration('mutational')*contact_map\n",
    "sns.heatmap(pair_frustration,cmap='vlag',center=0)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "23"
      ]
     },
     "execution_count": 38,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(model_exon.sequence)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# using the frustra server"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [],
   "source": [
    "def make_contact_col(frustra_all):\n",
    "    frustra_all['contact']=frustra_all['#Res1'].astype(str)+frustra_all.AA1+'-'+frustra_all.Res2.astype(str)+frustra_all.AA2\n",
    "    return frustra_all"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {},
   "outputs": [],
   "source": [
    "fr_index_dict={'highly':2,'neutral':1,'minimally':0}\n",
    "path_bs=path_f+'pdb_files_'+bs_name+'/'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "metadata": {},
   "outputs": [],
   "source": [
    "frustra_all=pd.read_csv(path_bs+pdb_code+'_'+chain+'.pdb_mutational',sep=' ')\n",
    "frustra_all=make_contact_col(frustra_all)\n",
    "j=0\n",
    "frustra_ex=pd.read_csv(path_bs+'exon_'+str(j)+'.pdb_mutational',sep=' ')\n",
    "frustra_ex=make_contact_col(frustra_ex)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "94"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sum(frustra_all.FrstState=='highly')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {},
   "outputs": [
    {
     "ename": "SyntaxError",
     "evalue": "invalid syntax (76895863.py, line 1)",
     "output_type": "error",
     "traceback": [
      "\u001b[0;36m  Cell \u001b[0;32mIn[44], line 1\u001b[0;36m\u001b[0m\n\u001b[0;31m    frustra_all.\u001b[0m\n\u001b[0m                ^\u001b[0m\n\u001b[0;31mSyntaxError\u001b[0m\u001b[0;31m:\u001b[0m invalid syntax\n"
     ]
    }
   ],
   "source": [
    "frustra_all."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "fr_dif=frustra_ex.merge(frustra_all,on='contact')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {},
   "outputs": [],
   "source": [
    "fr_dif['fr_exon']=fr_dif.FrstState_x.map(fr_index_dict)\n",
    "fr_dif['fr_all']=fr_dif.FrstState_y.map(fr_index_dict)\n",
    "\n",
    "fr_dif['fr_dif']=-fr_dif['fr_all']+fr_dif['fr_exon']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "fr_dif  FrstState_x  FrstState_y\n",
       "-1      minimally    neutral        0.040541\n",
       "        neutral      highly         0.013514\n",
       " 0      highly       highly         0.108108\n",
       "        minimally    minimally      0.202703\n",
       "        neutral      neutral        0.364865\n",
       " 1      highly       neutral        0.162162\n",
       "        neutral      minimally      0.108108\n",
       "Name: Res2_x, dtype: float64"
      ]
     },
     "execution_count": 47,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "fr_dif.groupby(['fr_dif','FrstState_x','FrstState_y']).Res2_x.count()/len(fr_dif)\n",
    "# estos son solo los contactos internos del exon\n",
    "# faltaria por otro lado los exon-resto"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "def view_frustration(pdb_filename,frustra_table):\n",
    "    view = py3Dmol.view(js='https://3dmol.org/build/3Dmol.js')\n",
    "    view.addModel(open(pdb_filename,'r').read(),'pdb')\n",
    "\n",
    "    view.setBackgroundColor('white')\n",
    "    view.setStyle({'cartoon':{'color':'white'}})\n",
    "    \n",
    "    fr_index_dict={'highly':2,'neutral':1,'minimally':0}\n",
    "    frustra_color=['green','blue','red']\n",
    "    frustra_num=frustra_table.FrstState.map(fr_index_dict)\n",
    "\n",
    "    for x in range(len(frustra_table)):\n",
    "        if frustra_num[x]!=1:\n",
    "            view.addLine({'start':{'chain':'A','resi':[str(frustra_table['#Res1'][x])]},\n",
    "                          'end':{'chain':'A','resi':[str(frustra_table['Res2'][x])]},\n",
    "              'color':frustra_color[frustra_num[x]], 'dashed':False,'linewidth':3})\n",
    "  \n",
    "    view.zoomTo(viewer=(0,0))\n",
    "\n",
    "    return view"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_16742555222774558\"  style=\"position: relative; width: 640px; height: 480px\">\n        <p id=\"3dmolwarning_16742555222774558\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    //this is to ignore the existence of requirejs amd\n    var savedexports, savedmodule;\n    if (typeof exports !== 'undefined') savedexports = exports;\n    else exports = {}\n    if (typeof module !== 'undefined') savedmodule = module;\n    else module = {}\n\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n        exports = savedexports;\n        module = savedmodule;\n        resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://3dmol.org/build/3Dmol.js');\n}\n\nvar viewer_16742555222774558 = null;\nvar warn = document.getElementById(\"3dmolwarning_16742555222774558\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_16742555222774558 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_16742555222774558\"),{backgroundColor:\"white\"});\nviewer_16742555222774558.zoomTo();\n\tviewer_16742555222774558.addModel(\"HEADER    ELECTRON TRANSPORT                      01-AUG-76   1CYC              \\nTITLE     THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C AT 2.3     \\nTITLE    2 ANGSTROMS RESOLUTION. II. STRUCTURE AND FUNCTION                     \\nCAVEAT     1CYC    THR A 40 HAS WRONG CHIRALITY AT ATOM CB THR A 78 HAS WRONG   \\nCAVEAT   2 1CYC    CHIRALITY AT ATOM CB THR B 40 HAS WRONG CHIRALITY AT ATOM    \\nCAVEAT   3 1CYC    CB THR B 78 HAS WRONG CHIRALITY AT ATOM CB                   \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: FERROCYTOCHROME C;                                         \\nCOMPND   3 CHAIN: A, B;                                                         \\nCOMPND   4 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 ORGANISM_SCIENTIFIC: KATSUWONUS PELAMIS;                             \\nSOURCE   3 ORGANISM_COMMON: SKIPJACK TUNA;                                      \\nSOURCE   4 ORGANISM_TAXID: 8226                                                 \\nKEYWDS    ELECTRON TRANSPORT                                                    \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    N.TANAKA,T.YAMANE,T.TSUKIHARA,T.ASHIDA,M.KAKUDO                       \\nREVDAT  17   03-MAR-21 1CYC    1       CAVEAT COMPND REMARK HET                 \\nREVDAT  17 2                   1       HETNAM HETSYN FORMUL LINK                \\nREVDAT  17 3                   1       ATOM                                     \\nREVDAT  16   29-NOV-17 1CYC    1       HELIX                                    \\nREVDAT  15   24-FEB-09 1CYC    1       VERSN                                    \\nREVDAT  14   01-APR-03 1CYC    1       JRNL                                     \\nREVDAT  13   27-JAN-84 1CYC    1       REMARK                                   \\nREVDAT  12   30-SEP-83 1CYC    1       REVDAT                                   \\nREVDAT  11   19-APR-83 1CYC    1       COMPND SOURCE REMARK                     \\nREVDAT  10   15-JAN-82 1CYC    1       REMARK                                   \\nREVDAT   9   06-JAN-82 1CYC    3       SCALE                                    \\nREVDAT   8   31-DEC-80 1CYC    1       REMARK                                   \\nREVDAT   7   07-APR-80 1CYC    3       SOURCE SEQRES ATOM                       \\nREVDAT   6   23-APR-79 1CYC    3       ATOM                                     \\nREVDAT   5   01-NOV-77 1CYC    1       AUTHOR REMARK FORMUL                     \\nREVDAT   4   09-SEP-77 1CYC    3       ATOM                                     \\nREVDAT   3   13-JUN-77 1CYC    1       HET                                      \\nREVDAT   2   25-OCT-76 1CYC    2       CONECT                                   \\nREVDAT   1   06-OCT-76 1CYC    0                                                \\nJRNL        AUTH   N.TANAKA,T.YAMANE,T.TSUKIHARA,T.ASHIDA,M.KAKUDO              \\nJRNL        TITL   THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C   \\nJRNL        TITL 2 AT 2.3 A RESOLUTION. II. STRUCTURE AND FUNCTION.             \\nJRNL        REF    J.BIOCHEM.(TOKYO)             V.  77   147 1975              \\nJRNL        REFN                   ISSN 0021-924X                               \\nJRNL        PMID   166072                                                       \\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   T.TSUKIHARA,T.YAMANE,N.TANAKA,T.ASHIDA,M.KAKUDO              \\nREMARK   1  TITL   OXIDATION OF A FERROCYTOCHROME C IN THE CRYSTALLINE          \\nREMARK   1  TITL 2 STATE-STRUCTURAL CHANGE AND ANION BINDING                    \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  73  1163 1973              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 2                                                          \\nREMARK   1  AUTH   T.ASHIDA,N.TANAKA,T.YAMANE,T.TSUKIHARA,M.KAKUDO              \\nREMARK   1  TITL   THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C   \\nREMARK   1  TITL 2 AT 2.3 ANGSTROMS RESOLUTION                                  \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  73   463 1973              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 3                                                          \\nREMARK   1  AUTH   T.ASHIDA,T.UEKI,T.TSUKIHARA,A.SUGIHARA,T.TAKANO,M.KAKUDO     \\nREMARK   1  TITL   THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C   \\nREMARK   1  TITL 2 AT 4 ANGSTROMS RESOLUTION                                    \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  70   913 1971              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 4                                                          \\nREMARK   1  AUTH   T.NAKAYAMA,K.TITANI,K.NARITA                                 \\nREMARK   1  TITL   THE AMINO ACID SEQUENCE OF CYTOCHROME C FROM BONITO          \\nREMARK   1  TITL 2 (KATSUWONUS PELAMIS, LINNAEUS)                               \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  70   311 1971              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 5                                                          \\nREMARK   1  EDIT   R.J.FELDMANN                                                 \\nREMARK   1  REF    ATLAS OF MACROMOLECULAR                120 1976              \\nREMARK   1  REF  2 STRUCTURE ON MICROFICHE                                      \\nREMARK   1  PUBL   TRACOR JITCO INC.,ROCKVILLE,MD.                              \\nREMARK   1  REFN                                                                \\nREMARK   1 REFERENCE 6                                                          \\nREMARK   1  EDIT   M.O.DAYHOFF                                                  \\nREMARK   1  REF    ATLAS OF PROTEIN SEQUENCE     V.   5    39 1976              \\nREMARK   1  REF  2 AND STRUCTURE,SUPPLEMENT 2                                   \\nREMARK   1  PUBL   NATIONAL BIOMEDICAL RESEARCH FOUNDATION, SILVER SPRING,MD.   \\nREMARK   1  REFN                   ISSN 0-912466-05-7                           \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    2.30 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : NULL                                                 \\nREMARK   3   AUTHORS     : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 2.30                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : NULL                           \\nREMARK   3   DATA CUTOFF            (SIGMA(F)) : NULL                           \\nREMARK   3   DATA CUTOFF HIGH         (ABS(F)) : NULL                           \\nREMARK   3   DATA CUTOFF LOW          (ABS(F)) : NULL                           \\nREMARK   3   COMPLETENESS (WORKING+TEST)   (%) : NULL                           \\nREMARK   3   NUMBER OF REFLECTIONS             : NULL                           \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   CROSS-VALIDATION METHOD          : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SELECTION  : NULL                            \\nREMARK   3   R VALUE            (WORKING SET) : NULL                            \\nREMARK   3   FREE R VALUE                     : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : NULL                            \\nREMARK   3   FREE R VALUE TEST SET COUNT      : NULL                            \\nREMARK   3   ESTIMATED ERROR OF FREE R VALUE  : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  FIT IN THE HIGHEST RESOLUTION BIN.                                  \\nREMARK   3   TOTAL NUMBER OF BINS USED           : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE HIGH       (A) : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE LOW        (A) : NULL                         \\nREMARK   3   BIN COMPLETENESS (WORKING+TEST) (%) : NULL                         \\nREMARK   3   REFLECTIONS IN BIN    (WORKING SET) : NULL                         \\nREMARK   3   BIN R VALUE           (WORKING SET) : NULL                         \\nREMARK   3   BIN FREE R VALUE                    : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET SIZE  (%) : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET COUNT     : NULL                         \\nREMARK   3   ESTIMATED ERROR OF BIN FREE R VALUE : NULL                         \\nREMARK   3                                                                      \\nREMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \\nREMARK   3   PROTEIN ATOMS            : 1590                                    \\nREMARK   3   NUCLEIC ACID ATOMS       : 0                                       \\nREMARK   3   HETEROGEN ATOMS          : 86                                      \\nREMARK   3   SOLVENT ATOMS            : 1                                       \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : NULL                           \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : NULL                           \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : NULL                                                 \\nREMARK   3    B22 (A**2) : NULL                                                 \\nREMARK   3    B33 (A**2) : NULL                                                 \\nREMARK   3    B12 (A**2) : NULL                                                 \\nREMARK   3    B13 (A**2) : NULL                                                 \\nREMARK   3    B23 (A**2) : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  ESTIMATED COORDINATE ERROR.                                         \\nREMARK   3   ESD FROM LUZZATI PLOT        (A) : NULL                            \\nREMARK   3   ESD FROM SIGMAA              (A) : NULL                            \\nREMARK   3   LOW RESOLUTION CUTOFF        (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  CROSS-VALIDATED ESTIMATED COORDINATE ERROR.                         \\nREMARK   3   ESD FROM C-V LUZZATI PLOT    (A) : NULL                            \\nREMARK   3   ESD FROM C-V SIGMAA          (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  RMS DEVIATIONS FROM IDEAL VALUES.                                   \\nREMARK   3   BOND LENGTHS                 (A) : NULL                            \\nREMARK   3   BOND ANGLES            (DEGREES) : NULL                            \\nREMARK   3   DIHEDRAL ANGLES        (DEGREES) : NULL                            \\nREMARK   3   IMPROPER ANGLES        (DEGREES) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL MODEL : NULL                                      \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL FACTOR RESTRAINTS.    RMS    SIGMA                \\nREMARK   3   MAIN-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   MAIN-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  NCS MODEL : NULL                                                    \\nREMARK   3                                                                      \\nREMARK   3  NCS RESTRAINTS.                         RMS   SIGMA/WEIGHT          \\nREMARK   3   GROUP  1  POSITIONAL            (A) : NULL  ; NULL                 \\nREMARK   3   GROUP  1  B-FACTOR           (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  PARAMETER FILE  1  : NULL                                           \\nREMARK   3  TOPOLOGY FILE  1   : NULL                                           \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS: NULL                                      \\nREMARK   4                                                                      \\nREMARK   4 1CYC COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY BNL.                                \\nREMARK 100 THE DEPOSITION ID IS D_1000172595.                                   \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : NULL                               \\nREMARK 200  TEMPERATURE           (KELVIN) : NULL                               \\nREMARK 200  PH                             : NULL                               \\nREMARK 200  NUMBER OF CRYSTALS USED        : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : NULL                               \\nREMARK 200  RADIATION SOURCE               : NULL                               \\nREMARK 200  BEAMLINE                       : NULL                               \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : NULL                               \\nREMARK 200  WAVELENGTH OR RANGE        (A) : NULL                               \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : NULL                               \\nREMARK 200  DETECTOR MANUFACTURER          : NULL                               \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : NULL                               \\nREMARK 200  DATA SCALING SOFTWARE          : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : NULL                               \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : NULL                               \\nREMARK 200  RESOLUTION RANGE LOW       (A) : NULL                               \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY                : NULL                               \\nREMARK 200  R MERGE                    (I) : NULL                               \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : NULL                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : NULL                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : NULL                               \\nREMARK 200  R MERGE FOR SHELL          (I) : NULL                               \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: NULL                                           \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: NULL                         \\nREMARK 200 SOFTWARE USED: NULL                                                  \\nREMARK 200 STARTING MODEL: NULL                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: NULL                                                         \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): 39.18                                     \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 2.02                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: NULL                                     \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: P 21 21 21                       \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -X+1/2,-Y,Z+1/2                                         \\nREMARK 290       3555   -X,Y+1/2,-Z+1/2                                         \\nREMARK 290       4555   X+1/2,-Y+1/2,-Z                                         \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -1.000000  0.000000  0.000000       28.84000            \\nREMARK 290   SMTRY2   2  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000       18.91500            \\nREMARK 290   SMTRY1   3 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   3  0.000000  1.000000  0.000000       42.29000            \\nREMARK 290   SMTRY3   3  0.000000  0.000000 -1.000000       18.91500            \\nREMARK 290   SMTRY1   4  1.000000  0.000000  0.000000       28.84000            \\nREMARK 290   SMTRY2   4  0.000000 -1.000000  0.000000       42.29000            \\nREMARK 290   SMTRY3   4  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1, 2                                                    \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: MONOMERIC                         \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 2                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: MONOMERIC                         \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: B                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 470                                                                      \\nREMARK 470 MISSING ATOM                                                         \\nREMARK 470 THE FOLLOWING RESIDUES HAVE MISSING ATOMS (M=MODEL NUMBER;           \\nREMARK 470 RES=RESIDUE NAME; C=CHAIN IDENTIFIER; SSEQ=SEQUENCE NUMBER;          \\nREMARK 470 I=INSERTION CODE):                                                   \\nREMARK 470   M RES CSSEQI  ATOMS                                                \\nREMARK 470     ASN A  22    CB   CG   OD1  ND2                                  \\nREMARK 470     LYS A  39    CE   NZ                                             \\nREMARK 470     ASN B  22    CB   CG   OD1  ND2                                  \\nREMARK 470     LYS B  39    CE   NZ                                             \\nREMARK 620                                                                      \\nREMARK 620 METAL COORDINATION                                                   \\nREMARK 620 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 620 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE):                             \\nREMARK 620                                                                      \\nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \\nREMARK 620                             HEC A 104  FE                            \\nREMARK 620 N RES CSSEQI ATOM                                                    \\nREMARK 620 1 HIS A  18   NE2                                                    \\nREMARK 620 2 HEC A 104   NA   87.0                                              \\nREMARK 620 3 HEC A 104   NB   87.8  90.6                                        \\nREMARK 620 4 HEC A 104   NC   92.0 178.7  90.1                                  \\nREMARK 620 5 HEC A 104   ND   91.1  86.9 177.4  92.3                            \\nREMARK 620 6 MET A  80   SD  170.9 102.1  93.0  79.0  88.6                      \\nREMARK 620 N                    1     2     3     4     5                       \\nREMARK 620                                                                      \\nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \\nREMARK 620                             HEC B 104  FE                            \\nREMARK 620 N RES CSSEQI ATOM                                                    \\nREMARK 620 1 HIS B  18   NE2                                                    \\nREMARK 620 2 HEC B 104   NA   87.0                                              \\nREMARK 620 3 HEC B 104   NB   87.8  90.6                                        \\nREMARK 620 4 HEC B 104   NC   92.0 178.7  90.1                                  \\nREMARK 620 5 HEC B 104   ND   91.1  86.9 177.4  92.3                            \\nREMARK 620 6 MET B  80   SD  170.9 102.1  93.0  79.0  88.6                      \\nREMARK 620 N                    1     2     3     4     5                       \\nREMARK 800                                                                      \\nREMARK 800 SITE                                                                 \\nREMARK 800 SITE_IDENTIFIER: AC1                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE HEC A 104                 \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: AC2                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE HEC B 104                 \\nDBREF  1CYC A    1   103  UNP    P00025   CYC_KATPE        1    103             \\nDBREF  1CYC B    1   103  UNP    P00025   CYC_KATPE        1    103             \\nSEQRES   1 A  103  GLY ASP VAL ALA LYS GLY LYS LYS THR PHE VAL GLN LYS          \\nSEQRES   2 A  103  CYS ALA GLN CYS HIS THR VAL GLU ASN GLY GLY LYS HIS          \\nSEQRES   3 A  103  LYS VAL GLY PRO ASN LEU TRP GLY LEU PHE GLY ARG LYS          \\nSEQRES   4 A  103  THR GLY GLN ALA GLU GLY TYR SER TYR THR ASP ALA ASN          \\nSEQRES   5 A  103  LYS SER LYS GLY ILE VAL TRP ASN GLU ASN THR LEU MET          \\nSEQRES   6 A  103  GLU TYR LEU GLU ASN PRO LYS LYS TYR ILE PRO GLY THR          \\nSEQRES   7 A  103  LYS MET ILE PHE ALA GLY ILE LYS LYS LYS GLY GLU ARG          \\nSEQRES   8 A  103  GLN ASP LEU VAL ALA TYR LEU LYS SER ALA THR SER              \\nSEQRES   1 B  103  GLY ASP VAL ALA LYS GLY LYS LYS THR PHE VAL GLN LYS          \\nSEQRES   2 B  103  CYS ALA GLN CYS HIS THR VAL GLU ASN GLY GLY LYS HIS          \\nSEQRES   3 B  103  LYS VAL GLY PRO ASN LEU TRP GLY LEU PHE GLY ARG LYS          \\nSEQRES   4 B  103  THR GLY GLN ALA GLU GLY TYR SER TYR THR ASP ALA ASN          \\nSEQRES   5 B  103  LYS SER LYS GLY ILE VAL TRP ASN GLU ASN THR LEU MET          \\nSEQRES   6 B  103  GLU TYR LEU GLU ASN PRO LYS LYS TYR ILE PRO GLY THR          \\nSEQRES   7 B  103  LYS MET ILE PHE ALA GLY ILE LYS LYS LYS GLY GLU ARG          \\nSEQRES   8 B  103  GLN ASP LEU VAL ALA TYR LEU LYS SER ALA THR SER              \\nHET    HEC  A 104      43                                                       \\nHET    HEC  B 104      43                                                       \\nHETNAM     HEC HEME C                                                           \\nFORMUL   3  HEC    2(C34 H34 FE N4 O4)                                          \\nFORMUL   5  HOH   *(H2 O)                                                       \\nHELIX    1  H1 GLY A    1  VAL A   11  1                                  11    \\nHELIX    2  H4 CYS A   14  CYS A   17  514 AND 17 BOUND TO HEME GROUP      4    \\nHELIX    3  H5 THR A   49  LYS A   55  5LOOSE FROM 49-53,3/10 53-55        7    \\nHELIX    4  H2 ASN A   60  GLU A   69  1                                  10    \\nHELIX    5  H3 GLU A   90  SER A  103  1                                  14    \\nHELIX    6  H6 GLY B    1  VAL B   11  1                                  11    \\nHELIX    7  H7 CYS B   14  CYS B   17  514 AND 17 BOUND TO HEME GROUP      4    \\nHELIX    8  H8 THR B   49  LYS B   55  5LOOSE FROM 49-53,3/10 53-55        7    \\nHELIX    9  H9 ASN B   60  GLU B   69  1                                  10    \\nHELIX   10 H10 GLU B   90  SER B  103  1                                  14    \\nLINK         SG  CYS A  14                 CAB HEC A 104     1555   1555  1.48  \\nLINK         SG  CYS A  17                 CAC HEC A 104     1555   1555  1.86  \\nLINK         CE2 TYR A  48                 O2D HEC A 104     1555   1555  2.02  \\nLINK         SG  CYS B  14                 CAB HEC B 104     1555   1555  1.48  \\nLINK         SG  CYS B  17                 CAC HEC B 104     1555   1555  1.86  \\nLINK         CE2 TYR B  48                 O2D HEC B 104     1555   1555  2.02  \\nLINK         NE2 HIS A  18                FE   HEC A 104     1555   1555  2.60  \\nLINK         SD  MET A  80                FE   HEC A 104     1555   1555  2.49  \\nLINK         NE2 HIS B  18                FE   HEC B 104     1555   1555  2.60  \\nLINK         SD  MET B  80                FE   HEC B 104     1555   1555  2.49  \\nSITE     1 AC1 17 LYS A  13  CYS A  14  CYS A  17  HIS A  18                    \\nSITE     2 AC1 17 LEU A  32  LEU A  35  ARG A  38  THR A  40                    \\nSITE     3 AC1 17 TYR A  46  TYR A  48  ASN A  52  TRP A  59                    \\nSITE     4 AC1 17 THR A  78  LYS A  79  MET A  80  PHE A  82                    \\nSITE     5 AC1 17 HOH A 105                                                     \\nSITE     1 AC2 16 LYS B  13  CYS B  14  CYS B  17  HIS B  18                    \\nSITE     2 AC2 16 LEU B  32  LEU B  35  ARG B  38  THR B  40                    \\nSITE     3 AC2 16 TYR B  46  TYR B  48  ASN B  52  TRP B  59                    \\nSITE     4 AC2 16 THR B  78  LYS B  79  MET B  80  PHE B  82                    \\nCRYST1   57.680   84.580   37.830  90.00  90.00  90.00 P 21 21 21    8          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.017337  0.000000  0.000000        0.00000                         \\nSCALE2      0.000000  0.011823  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.026434        0.00000                         \\nMTRIX1   1  1.000000  0.000000  0.000000       28.84000    1                    \\nMTRIX2   1  0.000000  1.000000  0.000000        0.00000    1                    \\nMTRIX3   1  0.000000  0.000000  1.000000        0.00000    1                    \\nATOM      1  N   GLY A   1     -21.138  13.774  -7.711  1.00  0.00           N  \\nATOM      2  CA  GLY A   1     -19.878  14.593  -7.963  1.00  0.00           C  \\nATOM      3  C   GLY A   1     -18.996  14.530  -6.703  1.00  0.00           C  \\nATOM      4  O   GLY A   1     -18.114  13.711  -6.640  1.00  0.00           O  \\nATOM      5  N   ASP A   2     -18.996  15.412  -5.821  1.00  0.00           N  \\nATOM      6  CA  ASP A   2     -17.610  15.097  -5.191  1.00  0.00           C  \\nATOM      7  C   ASP A   2     -17.862  14.152  -3.931  1.00  0.00           C  \\nATOM      8  O   ASP A   2     -17.106  13.333  -3.553  1.00  0.00           O  \\nATOM      9  CB  ASP A   2     -17.169  16.546  -4.435  1.00  0.00           C  \\nATOM     10  CG  ASP A   2     -17.988  17.365  -3.427  1.00  0.00           C  \\nATOM     11  OD1 ASP A   2     -19.500  17.365  -3.994  1.00  0.00           O  \\nATOM     12  OD2 ASP A   2     -17.484  18.373  -2.293  1.00  0.00           O  \\nATOM     13  N   VAL A   3     -19.059  14.089  -3.679  1.00  0.00           N  \\nATOM     14  CA  VAL A   3     -19.815  12.955  -2.860  1.00  0.00           C  \\nATOM     15  C   VAL A   3     -19.374  11.506  -3.805  1.00  0.00           C  \\nATOM     16  O   VAL A   3     -18.870  10.624  -3.427  1.00  0.00           O  \\nATOM     17  CB  VAL A   3     -22.587  13.270  -3.175  1.00  0.00           C  \\nATOM     18  CG1 VAL A   3     -21.957  11.947  -3.427  1.00  0.00           C  \\nATOM     19  CG2 VAL A   3     -21.705  13.963  -2.167  1.00  0.00           C  \\nATOM     20  N   ALA A   4     -19.437  11.380  -5.002  1.00  0.00           N  \\nATOM     21  CA  ALA A   4     -18.681  10.120  -5.569  1.00  0.00           C  \\nATOM     22  C   ALA A   4     -17.358   9.805  -4.939  1.00  0.00           C  \\nATOM     23  O   ALA A   4     -17.295   8.923  -4.183  1.00  0.00           O  \\nATOM     24  CB  ALA A   4     -18.429  10.435  -7.144  1.00  0.00           C  \\nATOM     25  N   LYS A   5     -16.476  10.687  -5.191  1.00  0.00           N  \\nATOM     26  CA  LYS A   5     -15.216  10.498  -4.372  1.00  0.00           C  \\nATOM     27  C   LYS A   5     -15.279  10.246  -2.797  1.00  0.00           C  \\nATOM     28  O   LYS A   5     -14.838   9.238  -2.293  1.00  0.00           O  \\nATOM     29  CB  LYS A   5     -14.208  11.695  -4.246  1.00  0.00           C  \\nATOM     30  CG  LYS A   5     -12.948  11.758  -3.553  1.00  0.00           C  \\nATOM     31  CD  LYS A   5     -11.814  13.081  -3.679  1.00  0.00           C  \\nATOM     32  CE  LYS A   5     -10.806  13.144  -2.545  1.00  0.00           C  \\nATOM     33  NZ  LYS A   5      -9.735  14.404  -2.293  1.00  0.00           N  \\nATOM     34  N   GLY A   6     -16.161  10.750  -1.915  1.00  0.00           N  \\nATOM     35  CA  GLY A   6     -16.350  10.435  -0.403  1.00  0.00           C  \\nATOM     36  C   GLY A   6     -16.539   8.923  -0.655  1.00  0.00           C  \\nATOM     37  O   GLY A   6     -15.909   8.041  -0.025  1.00  0.00           O  \\nATOM     38  N   LYS A   7     -17.421   8.419  -1.411  1.00  0.00           N  \\nATOM     39  CA  LYS A   7     -17.799   6.844  -1.033  1.00  0.00           C  \\nATOM     40  C   LYS A   7     -16.476   6.151  -1.474  1.00  0.00           C  \\nATOM     41  O   LYS A   7     -15.972   4.954  -0.970  1.00  0.00           O  \\nATOM     42  CB  LYS A   7     -18.870   6.403  -2.293  1.00  0.00           C  \\nATOM     43  CG  LYS A   7     -18.870   5.017  -2.734  1.00  0.00           C  \\nATOM     44  CD  LYS A   7     -20.193   4.765  -3.427  1.00  0.00           C  \\nATOM     45  CE  LYS A   7     -20.193   3.190  -4.057  1.00  0.00           C  \\nATOM     46  NZ  LYS A   7     -21.516   2.623  -4.309  1.00  0.00           N  \\nATOM     47  N   LYS A   8     -15.846   6.403  -2.545  1.00  0.00           N  \\nATOM     48  CA  LYS A   8     -14.460   5.899  -2.923  1.00  0.00           C  \\nATOM     49  C   LYS A   8     -13.452   5.899  -1.663  1.00  0.00           C  \\nATOM     50  O   LYS A   8     -13.011   5.017  -1.348  1.00  0.00           O  \\nATOM     51  CB  LYS A   8     -13.704   6.844  -3.931  1.00  0.00           C  \\nATOM     52  CG  LYS A   8     -12.318   6.529  -4.183  1.00  0.00           C  \\nATOM     53  CD  LYS A   8     -11.562   7.537  -5.065  1.00  0.00           C  \\nATOM     54  CE  LYS A   8     -10.176   7.222  -5.821  1.00  0.00           C  \\nATOM     55  NZ  LYS A   8      -9.357   8.419  -6.325  1.00  0.00           N  \\nATOM     56  N   THR A   9     -13.452   6.970  -1.159  1.00  0.00           N  \\nATOM     57  CA  THR A   9     -12.633   7.033   0.227  1.00  0.00           C  \\nATOM     58  C   THR A   9     -13.515   6.277   1.298  1.00  0.00           C  \\nATOM     59  O   THR A   9     -12.948   5.773   2.369  1.00  0.00           O  \\nATOM     60  CB  THR A   9     -12.381   8.419   0.920  1.00  0.00           C  \\nATOM     61  OG1 THR A   9     -11.058   8.923   0.542  1.00  0.00           O  \\nATOM     62  CG2 THR A   9     -12.444   8.734   2.558  1.00  0.00           C  \\nATOM     63  N   PHE A  10     -14.775   6.403   1.235  1.00  0.00           N  \\nATOM     64  CA  PHE A  10     -15.531   5.647   2.117  1.00  0.00           C  \\nATOM     65  C   PHE A  10     -15.216   4.135   1.739  1.00  0.00           C  \\nATOM     66  O   PHE A  10     -14.460   3.757   2.495  1.00  0.00           O  \\nATOM     67  CB  PHE A  10     -17.106   5.647   1.991  1.00  0.00           C  \\nATOM     68  CG  PHE A  10     -17.736   4.891   3.251  1.00  0.00           C  \\nATOM     69  CD1 PHE A  10     -18.492   3.757   2.747  1.00  0.00           C  \\nATOM     70  CD2 PHE A  10     -17.862   5.143   4.133  1.00  0.00           C  \\nATOM     71  CE1 PHE A  10     -18.996   3.001   3.818  1.00  0.00           C  \\nATOM     72  CE2 PHE A  10     -18.366   4.513   4.889  1.00  0.00           C  \\nATOM     73  CZ  PHE A  10     -18.996   3.379   5.015  1.00  0.00           C  \\nATOM     74  N   VAL A  11     -15.216   3.379   0.479  1.00  0.00           N  \\nATOM     75  CA  VAL A  11     -14.964   2.056   0.605  1.00  0.00           C  \\nATOM     76  C   VAL A  11     -13.326   1.867   1.109  1.00  0.00           C  \\nATOM     77  O   VAL A  11     -12.948   0.985   1.865  1.00  0.00           O  \\nATOM     78  CB  VAL A  11     -15.153   1.237  -0.529  1.00  0.00           C  \\nATOM     79  CG1 VAL A  11     -16.098   1.930  -1.537  1.00  0.00           C  \\nATOM     80  CG2 VAL A  11     -13.956   1.111  -1.285  1.00  0.00           C  \\nATOM     81  N   GLN A  12     -12.570   2.749   0.794  1.00  0.00           N  \\nATOM     82  CA  GLN A  12     -10.995   2.749   1.235  1.00  0.00           C  \\nATOM     83  C   GLN A  12     -10.680   2.938   2.873  1.00  0.00           C  \\nATOM     84  O   GLN A  12     -10.176   1.930   3.377  1.00  0.00           O  \\nATOM     85  CB  GLN A  12     -10.428   3.820   0.479  1.00  0.00           C  \\nATOM     86  CG  GLN A  12      -9.861   3.253  -0.718  1.00  0.00           C  \\nATOM     87  CD  GLN A  12      -9.168   4.135  -2.293  1.00  0.00           C  \\nATOM     88  OE1 GLN A  12      -7.593   3.820  -2.041  1.00  0.00           O  \\nATOM     89  NE2 GLN A  12      -9.672   5.143  -2.671  1.00  0.00           N  \\nATOM     90  N   LYS A  13     -10.995   3.757   3.629  1.00  0.00           N  \\nATOM     91  CA  LYS A  13     -10.869   3.757   5.015  1.00  0.00           C  \\nATOM     92  C   LYS A  13     -12.066   3.694   5.960  1.00  0.00           C  \\nATOM     93  O   LYS A  13     -12.003   3.127   6.779  1.00  0.00           O  \\nATOM     94  CB  LYS A  13     -10.239   5.017   5.141  1.00  0.00           C  \\nATOM     95  CG  LYS A  13      -9.672   5.521   3.881  1.00  0.00           C  \\nATOM     96  CD  LYS A  13      -8.286   6.277   4.007  1.00  0.00           C  \\nATOM     97  CE  LYS A  13      -8.160   6.907   2.621  1.00  0.00           C  \\nATOM     98  NZ  LYS A  13      -6.837   7.789   2.243  1.00  0.00           N  \\nATOM     99  N   CYS A  14     -13.137   4.072   5.519  1.00  0.00           N  \\nATOM    100  CA  CYS A  14     -14.460   4.072   6.464  1.00  0.00           C  \\nATOM    101  C   CYS A  14     -15.216   2.812   5.897  1.00  0.00           C  \\nATOM    102  O   CYS A  14     -15.342   1.804   6.716  1.00  0.00           O  \\nATOM    103  CB  CYS A  14     -15.468   5.332   5.897  1.00  0.00           C  \\nATOM    104  SG  CYS A  14     -14.712   6.844   6.464  1.00  0.00           S  \\nATOM    105  N   ALA A  15     -15.720   2.623   4.637  1.00  0.00           N  \\nATOM    106  CA  ALA A  15     -16.161   1.237   4.322  1.00  0.00           C  \\nATOM    107  C   ALA A  15     -16.035  -0.086   5.015  1.00  0.00           C  \\nATOM    108  O   ALA A  15     -17.043  -0.590   5.393  1.00  0.00           O  \\nATOM    109  CB  ALA A  15     -15.405   0.985   3.062  1.00  0.00           C  \\nATOM    110  N   GLN A  16     -14.964  -0.653   4.889  1.00  0.00           N  \\nATOM    111  CA  GLN A  16     -14.901  -2.039   5.897  1.00  0.00           C  \\nATOM    112  C   GLN A  16     -15.090  -1.787   7.283  1.00  0.00           C  \\nATOM    113  O   GLN A  16     -14.838  -2.669   8.291  1.00  0.00           O  \\nATOM    114  CB  GLN A  16     -13.578  -2.039   5.519  1.00  0.00           C  \\nATOM    115  CG  GLN A  16     -12.318  -1.724   6.275  1.00  0.00           C  \\nATOM    116  CD  GLN A  16     -11.625  -0.842   5.015  1.00  0.00           C  \\nATOM    117  OE1 GLN A  16     -10.365  -0.149   5.330  1.00  0.00           O  \\nATOM    118  NE2 GLN A  16     -12.444  -0.464   4.385  1.00  0.00           N  \\nATOM    119  N   CYS A  17     -15.216  -0.653   7.787  1.00  0.00           N  \\nATOM    120  CA  CYS A  17     -15.405  -0.905   9.299  1.00  0.00           C  \\nATOM    121  C   CYS A  17     -16.917  -0.401   9.173  1.00  0.00           C  \\nATOM    122  O   CYS A  17     -17.736  -1.283   9.173  1.00  0.00           O  \\nATOM    123  CB  CYS A  17     -14.460   0.355  10.055  1.00  0.00           C  \\nATOM    124  SG  CYS A  17     -13.326  -0.338  11.441  1.00  0.00           S  \\nATOM    125  N   HIS A  18     -17.232   0.733   9.551  1.00  0.00           N  \\nATOM    126  CA  HIS A  18     -18.870   0.985   9.803  1.00  0.00           C  \\nATOM    127  C   HIS A  18     -19.878   0.229   8.795  1.00  0.00           C  \\nATOM    128  O   HIS A  18     -19.626  -0.023   7.472  1.00  0.00           O  \\nATOM    129  CB  HIS A  18     -19.185   2.245   9.173  1.00  0.00           C  \\nATOM    130  CG  HIS A  18     -18.681   3.127  10.181  1.00  0.00           C  \\nATOM    131  ND1 HIS A  18     -18.618   3.253  11.567  1.00  0.00           N  \\nATOM    132  CD2 HIS A  18     -17.736   4.072  10.559  1.00  0.00           C  \\nATOM    133  CE1 HIS A  18     -18.051   3.694  12.827  1.00  0.00           C  \\nATOM    134  NE2 HIS A  18     -17.232   4.387  11.819  1.00  0.00           N  \\nATOM    135  N   THR A  19     -21.012   0.103   9.299  1.00  0.00           N  \\nATOM    136  CA  THR A  19     -22.272  -0.401   8.417  1.00  0.00           C  \\nATOM    137  C   THR A  19     -22.902   0.922   7.661  1.00  0.00           C  \\nATOM    138  O   THR A  19     -22.272   1.867   7.157  1.00  0.00           O  \\nATOM    139  CB  THR A  19     -23.280  -1.283   8.795  1.00  0.00           C  \\nATOM    140  OG1 THR A  19     -23.532  -0.905   9.740  1.00  0.00           O  \\nATOM    141  CG2 THR A  19     -22.776  -2.669   8.543  1.00  0.00           C  \\nATOM    142  N   VAL A  20     -24.036   1.111   7.472  1.00  0.00           N  \\nATOM    143  CA  VAL A  20     -24.414   2.371   6.779  1.00  0.00           C  \\nATOM    144  C   VAL A  20     -26.052   2.434   6.401  1.00  0.00           C  \\nATOM    145  O   VAL A  20     -26.556   3.442   5.771  1.00  0.00           O  \\nATOM    146  CB  VAL A  20     -23.784   2.560   5.519  1.00  0.00           C  \\nATOM    147  CG1 VAL A  20     -24.540   2.119   4.637  1.00  0.00           C  \\nATOM    148  CG2 VAL A  20     -23.658   4.009   5.267  1.00  0.00           C  \\nATOM    149  N   GLU A  21     -26.682   1.363   6.527  1.00  0.00           N  \\nATOM    150  CA  GLU A  21     -28.194   1.363   5.897  1.00  0.00           C  \\nATOM    151  C   GLU A  21     -29.013   1.741   7.283  1.00  0.00           C  \\nATOM    152  O   GLU A  21     -28.635   2.623   7.913  1.00  0.00           O  \\nATOM    153  CB  GLU A  21     -28.572  -0.086   5.645  1.00  0.00           C  \\nATOM    154  CG  GLU A  21     -28.698  -0.779   4.133  1.00  0.00           C  \\nATOM    155  CD  GLU A  21     -27.690  -1.913   4.007  1.00  0.00           C  \\nATOM    156  OE1 GLU A  21     -26.682  -2.039   5.141  1.00  0.00           O  \\nATOM    157  OE2 GLU A  21     -27.753  -2.921   2.873  1.00  0.00           O  \\nATOM    158  N   ASN A  22     -30.210   1.363   7.283  1.00  0.00           N  \\nATOM    159  CA  ASN A  22     -30.966   1.867   8.417  1.00  0.00           C  \\nATOM    160  C   ASN A  22     -30.714   1.237   9.677  1.00  0.00           C  \\nATOM    161  O   ASN A  22     -31.218   0.103  10.055  1.00  0.00           O  \\nATOM    162  N   GLY A  23     -29.958   1.552  10.433  1.00  0.00           N  \\nATOM    163  CA  GLY A  23     -29.517   0.859  11.819  1.00  0.00           C  \\nATOM    164  C   GLY A  23     -28.572  -0.338  11.252  1.00  0.00           C  \\nATOM    165  O   GLY A  23     -28.194  -1.283  12.071  1.00  0.00           O  \\nATOM    166  N   GLY A  24     -28.068  -0.149   9.929  1.00  0.00           N  \\nATOM    167  CA  GLY A  24     -27.060  -1.409   9.047  1.00  0.00           C  \\nATOM    168  C   GLY A  24     -26.871  -2.480  10.118  1.00  0.00           C  \\nATOM    169  O   GLY A  24     -27.186  -3.803   9.929  1.00  0.00           O  \\nATOM    170  N   LYS A  25     -26.304  -2.165  11.441  1.00  0.00           N  \\nATOM    171  CA  LYS A  25     -26.241  -3.047  12.701  1.00  0.00           C  \\nATOM    172  C   LYS A  25     -24.792  -2.795  13.079  1.00  0.00           C  \\nATOM    173  O   LYS A  25     -23.910  -2.417  12.323  1.00  0.00           O  \\nATOM    174  CB  LYS A  25     -26.241  -4.433  11.819  1.00  0.00           C  \\nATOM    175  CG  LYS A  25     -25.296  -4.496  10.433  1.00  0.00           C  \\nATOM    176  CD  LYS A  25     -24.918  -5.819   9.803  1.00  0.00           C  \\nATOM    177  CE  LYS A  25     -23.784  -5.819   8.543  1.00  0.00           C  \\nATOM    178  NZ  LYS A  25     -23.658  -6.827   7.661  1.00  0.00           N  \\nATOM    179  N   HIS A  26     -24.414  -2.921  14.465  1.00  0.00           N  \\nATOM    180  CA  HIS A  26     -23.028  -2.417  14.654  1.00  0.00           C  \\nATOM    181  C   HIS A  26     -21.894  -3.425  14.087  1.00  0.00           C  \\nATOM    182  O   HIS A  26     -22.083  -3.803  12.827  1.00  0.00           O  \\nATOM    183  CB  HIS A  26     -23.028  -2.543  16.103  1.00  0.00           C  \\nATOM    184  CG  HIS A  26     -24.036  -1.094  16.355  1.00  0.00           C  \\nATOM    185  ND1 HIS A  26     -24.540  -0.338  15.473  1.00  0.00           N  \\nATOM    186  CD2 HIS A  26     -24.666  -0.905  17.363  1.00  0.00           C  \\nATOM    187  CE1 HIS A  26     -25.422   0.355  16.103  1.00  0.00           C  \\nATOM    188  NE2 HIS A  26     -25.422   0.103  17.237  1.00  0.00           N  \\nATOM    189  N   LYS A  27     -20.886  -3.551  14.717  1.00  0.00           N  \\nATOM    190  CA  LYS A  27     -19.752  -4.433  14.213  1.00  0.00           C  \\nATOM    191  C   LYS A  27     -18.366  -3.677  14.276  1.00  0.00           C  \\nATOM    192  O   LYS A  27     -17.484  -3.677  13.331  1.00  0.00           O  \\nATOM    193  CB  LYS A  27     -20.004  -4.559  12.575  1.00  0.00           C  \\nATOM    194  CG  LYS A  27     -20.382  -3.425  11.882  1.00  0.00           C  \\nATOM    195  CD  LYS A  27     -19.500  -3.047  11.063  1.00  0.00           C  \\nATOM    196  CE  LYS A  27     -19.752  -3.677   9.551  1.00  0.00           C  \\nATOM    197  NZ  LYS A  27     -18.618  -3.677   8.858  1.00  0.00           N  \\nATOM    198  N   VAL A  28     -18.177  -3.047  15.410  1.00  0.00           N  \\nATOM    199  CA  VAL A  28     -16.854  -2.228  15.410  1.00  0.00           C  \\nATOM    200  C   VAL A  28     -17.358  -0.905  15.473  1.00  0.00           C  \\nATOM    201  O   VAL A  28     -17.610  -0.275  16.481  1.00  0.00           O  \\nATOM    202  CB  VAL A  28     -15.846  -2.291  14.780  1.00  0.00           C  \\nATOM    203  CG1 VAL A  28     -15.594  -1.094  14.087  1.00  0.00           C  \\nATOM    204  CG2 VAL A  28     -14.586  -2.858  15.095  1.00  0.00           C  \\nATOM    205  N   GLY A  29     -17.610  -0.023  14.591  1.00  0.00           N  \\nATOM    206  CA  GLY A  29     -18.240   1.111  14.843  1.00  0.00           C  \\nATOM    207  C   GLY A  29     -19.815   1.048  14.843  1.00  0.00           C  \\nATOM    208  O   GLY A  29     -20.193   0.103  13.961  1.00  0.00           O  \\nATOM    209  N   PRO A  30     -20.634   1.615  15.977  1.00  0.00           N  \\nATOM    210  CA  PRO A  30     -22.020   1.489  15.725  1.00  0.00           C  \\nATOM    211  C   PRO A  30     -22.398   1.804  14.339  1.00  0.00           C  \\nATOM    212  O   PRO A  30     -22.020   2.371  13.457  1.00  0.00           O  \\nATOM    213  CB  PRO A  30     -22.524   2.371  16.481  1.00  0.00           C  \\nATOM    214  CG  PRO A  30     -21.642   1.993  16.859  1.00  0.00           C  \\nATOM    215  CD  PRO A  30     -20.508   2.497  16.607  1.00  0.00           C  \\nATOM    216  N   ASN A  31     -23.910   1.552  13.961  1.00  0.00           N  \\nATOM    217  CA  ASN A  31     -24.414   2.119  12.701  1.00  0.00           C  \\nATOM    218  C   ASN A  31     -24.036   3.631  12.764  1.00  0.00           C  \\nATOM    219  O   ASN A  31     -24.036   4.387  13.835  1.00  0.00           O  \\nATOM    220  CB  ASN A  31     -25.926   1.993  12.701  1.00  0.00           C  \\nATOM    221  CG  ASN A  31     -26.052   2.056  11.063  1.00  0.00           C  \\nATOM    222  OD1 ASN A  31     -27.186   2.623  10.307  1.00  0.00           O  \\nATOM    223  ND2 ASN A  31     -25.359   1.363  10.181  1.00  0.00           N  \\nATOM    224  N   LEU A  32     -23.658   4.261  11.819  1.00  0.00           N  \\nATOM    225  CA  LEU A  32     -23.280   5.899  11.819  1.00  0.00           C  \\nATOM    226  C   LEU A  32     -24.666   6.403  11.315  1.00  0.00           C  \\nATOM    227  O   LEU A  32     -24.666   6.655   9.929  1.00  0.00           O  \\nATOM    228  CB  LEU A  32     -22.146   6.466  11.315  1.00  0.00           C  \\nATOM    229  CG  LEU A  32     -20.886   6.529  12.323  1.00  0.00           C  \\nATOM    230  CD1 LEU A  32     -20.382   7.978  12.323  1.00  0.00           C  \\nATOM    231  CD2 LEU A  32     -21.327   6.277  13.457  1.00  0.00           C  \\nATOM    232  N   TRP A  33     -25.611   6.151  11.693  1.00  0.00           N  \\nATOM    233  CA  TRP A  33     -27.186   6.529  11.189  1.00  0.00           C  \\nATOM    234  C   TRP A  33     -27.312   7.915  10.433  1.00  0.00           C  \\nATOM    235  O   TRP A  33     -27.312   7.852   9.299  1.00  0.00           O  \\nATOM    236  CB  TRP A  33     -28.446   6.403  12.071  1.00  0.00           C  \\nATOM    237  CG  TRP A  33     -28.572   5.773  13.205  1.00  0.00           C  \\nATOM    238  CD1 TRP A  33     -27.564   4.891  13.583  1.00  0.00           C  \\nATOM    239  CD2 TRP A  33     -29.328   5.647  14.213  1.00  0.00           C  \\nATOM    240  NE1 TRP A  33     -27.816   4.387  14.843  1.00  0.00           N  \\nATOM    241  CE2 TRP A  33     -29.013   4.828  15.158  1.00  0.00           C  \\nATOM    242  CE3 TRP A  33     -30.462   6.277  14.213  1.00  0.00           C  \\nATOM    243  CZ2 TRP A  33     -29.706   4.639  15.851  1.00  0.00           C  \\nATOM    244  CZ3 TRP A  33     -31.092   5.962  15.284  1.00  0.00           C  \\nATOM    245  CH2 TRP A  33     -30.714   5.143  15.914  1.00  0.00           C  \\nATOM    246  N   GLY A  34     -27.564   8.797  11.315  1.00  0.00           N  \\nATOM    247  CA  GLY A  34     -27.690  10.120  10.685  1.00  0.00           C  \\nATOM    248  C   GLY A  34     -26.934  11.191  11.315  1.00  0.00           C  \\nATOM    249  O   GLY A  34     -27.312  12.199  12.134  1.00  0.00           O  \\nATOM    250  N   LEU A  35     -25.548  11.191  11.252  1.00  0.00           N  \\nATOM    251  CA  LEU A  35     -24.666  12.199  11.945  1.00  0.00           C  \\nATOM    252  C   LEU A  35     -25.548  13.459  12.386  1.00  0.00           C  \\nATOM    253  O   LEU A  35     -25.800  13.963  13.457  1.00  0.00           O  \\nATOM    254  CB  LEU A  35     -23.154  12.640  11.567  1.00  0.00           C  \\nATOM    255  CG  LEU A  35     -22.335  11.506  11.189  1.00  0.00           C  \\nATOM    256  CD1 LEU A  35     -20.886  11.695  11.189  1.00  0.00           C  \\nATOM    257  CD2 LEU A  35     -22.650  10.183  11.819  1.00  0.00           C  \\nATOM    258  N   PHE A  36     -26.052  13.963  11.441  1.00  0.00           N  \\nATOM    259  CA  PHE A  36     -26.934  15.223  11.693  1.00  0.00           C  \\nATOM    260  C   PHE A  36     -28.131  15.223  12.701  1.00  0.00           C  \\nATOM    261  O   PHE A  36     -29.328  15.223  12.449  1.00  0.00           O  \\nATOM    262  CB  PHE A  36     -27.186  15.601  10.055  1.00  0.00           C  \\nATOM    263  CG  PHE A  36     -25.926  15.601   9.425  1.00  0.00           C  \\nATOM    264  CD1 PHE A  36     -25.674  15.727   8.102  1.00  0.00           C  \\nATOM    265  CD2 PHE A  36     -24.792  15.097   9.803  1.00  0.00           C  \\nATOM    266  CE1 PHE A  36     -24.540  15.475   7.409  1.00  0.00           C  \\nATOM    267  CE2 PHE A  36     -23.658  15.097   9.551  1.00  0.00           C  \\nATOM    268  CZ  PHE A  36     -23.406  15.097   8.291  1.00  0.00           C  \\nATOM    269  N   GLY A  37     -27.879  15.538  13.961  1.00  0.00           N  \\nATOM    270  CA  GLY A  37     -28.509  15.664  15.347  1.00  0.00           C  \\nATOM    271  C   GLY A  37     -28.005  14.971  16.355  1.00  0.00           C  \\nATOM    272  O   GLY A  37     -28.572  14.215  17.300  1.00  0.00           O  \\nATOM    273  N   ARG A  38     -26.682  15.097  16.859  1.00  0.00           N  \\nATOM    274  CA  ARG A  38     -26.556  14.341  18.119  1.00  0.00           C  \\nATOM    275  C   ARG A  38     -25.170  14.845  18.371  1.00  0.00           C  \\nATOM    276  O   ARG A  38     -24.414  15.349  17.741  1.00  0.00           O  \\nATOM    277  CB  ARG A  38     -26.178  13.081  17.741  1.00  0.00           C  \\nATOM    278  CG  ARG A  38     -25.863  12.703  16.355  1.00  0.00           C  \\nATOM    279  CD  ARG A  38     -25.422  11.191  16.040  1.00  0.00           C  \\nATOM    280  NE  ARG A  38     -24.666  10.561  16.985  1.00  0.00           N  \\nATOM    281  CZ  ARG A  38     -24.792   9.238  16.859  1.00  0.00           C  \\nATOM    282  NH1 ARG A  38     -24.351   8.545  17.489  1.00  0.00           N  \\nATOM    283  NH2 ARG A  38     -25.485   8.797  15.851  1.00  0.00           N  \\nATOM    284  N   LYS A  39     -25.044  14.593  19.757  1.00  0.00           N  \\nATOM    285  CA  LYS A  39     -24.036  15.160  20.639  1.00  0.00           C  \\nATOM    286  C   LYS A  39     -23.028  14.089  20.891  1.00  0.00           C  \\nATOM    287  O   LYS A  39     -23.406  13.207  20.009  1.00  0.00           O  \\nATOM    288  CB  LYS A  39     -24.540  15.601  22.151  1.00  0.00           C  \\nATOM    289  CG  LYS A  39     -24.414  15.475  23.663  1.00  0.00           C  \\nATOM    290  CD  LYS A  39     -25.170  16.231  24.923  1.00  0.00           C  \\nATOM    291  N   THR A  40     -22.020  13.837  21.521  1.00  0.00           N  \\nATOM    292  CA  THR A  40     -21.390  12.388  21.269  1.00  0.00           C  \\nATOM    293  C   THR A  40     -21.642  11.191  22.277  1.00  0.00           C  \\nATOM    294  O   THR A  40     -21.516  11.695  23.411  1.00  0.00           O  \\nATOM    295  CB  THR A  40     -19.752  12.577  21.269  1.00  0.00           C  \\nATOM    296  OG1 THR A  40     -19.374  11.884  19.883  1.00  0.00           O  \\nATOM    297  CG2 THR A  40     -19.626  13.837  20.765  1.00  0.00           C  \\nATOM    298  N   GLY A  41     -22.020  10.309  21.773  1.00  0.00           N  \\nATOM    299  CA  GLY A  41     -22.650   9.553  22.907  1.00  0.00           C  \\nATOM    300  C   GLY A  41     -23.910   8.797  22.403  1.00  0.00           C  \\nATOM    301  O   GLY A  41     -24.162   8.545  21.395  1.00  0.00           O  \\nATOM    302  N   GLN A  42     -24.918   8.419  23.033  1.00  0.00           N  \\nATOM    303  CA  GLN A  42     -26.241   7.915  23.033  1.00  0.00           C  \\nATOM    304  C   GLN A  42     -26.304   7.537  21.647  1.00  0.00           C  \\nATOM    305  O   GLN A  42     -26.556   8.545  20.765  1.00  0.00           O  \\nATOM    306  CB  GLN A  42     -27.438   8.545  23.537  1.00  0.00           C  \\nATOM    307  CG  GLN A  42     -28.068   7.789  24.419  1.00  0.00           C  \\nATOM    308  CD  GLN A  42     -29.076   7.096  24.230  1.00  0.00           C  \\nATOM    309  OE1 GLN A  42     -29.706   5.899  24.797  1.00  0.00           O  \\nATOM    310  NE2 GLN A  42     -29.706   7.411  22.781  1.00  0.00           N  \\nATOM    311  N   ALA A  43     -26.304   6.277  21.143  1.00  0.00           N  \\nATOM    312  CA  ALA A  43     -26.682   6.025  19.505  1.00  0.00           C  \\nATOM    313  C   ALA A  43     -27.186   4.639  19.442  1.00  0.00           C  \\nATOM    314  O   ALA A  43     -27.060   4.009  18.497  1.00  0.00           O  \\nATOM    315  CB  ALA A  43     -25.548   6.025  18.497  1.00  0.00           C  \\nATOM    316  N   GLU A  44     -27.438   3.883  20.450  1.00  0.00           N  \\nATOM    317  CA  GLU A  44     -28.131   2.497  20.261  1.00  0.00           C  \\nATOM    318  C   GLU A  44     -27.060   1.615  20.891  1.00  0.00           C  \\nATOM    319  O   GLU A  44     -26.556   0.481  20.324  1.00  0.00           O  \\nATOM    320  CB  GLU A  44     -28.635   2.371  18.749  1.00  0.00           C  \\nATOM    321  CG  GLU A  44     -30.210   0.993  18.749  1.00  0.00           C  \\nATOM    322  CD  GLU A  44     -30.840   2.875  19.379  1.00  0.00           C  \\nATOM    323  OE1 GLU A  44     -32.100   2.875  18.875  1.00  0.00           O  \\nATOM    324  OE2 GLU A  44     -30.084   4.135  19.505  1.00  0.00           O  \\nATOM    325  N   GLY A  45     -26.808   1.615  22.277  1.00  0.00           N  \\nATOM    326  CA  GLY A  45     -25.674   0.859  22.781  1.00  0.00           C  \\nATOM    327  C   GLY A  45     -24.162   1.363  22.907  1.00  0.00           C  \\nATOM    328  O   GLY A  45     -23.532   1.363  21.521  1.00  0.00           O  \\nATOM    329  N   TYR A  46     -23.532   1.993  23.789  1.00  0.00           N  \\nATOM    330  CA  TYR A  46     -22.146   2.875  23.537  1.00  0.00           C  \\nATOM    331  C   TYR A  46     -21.327   1.993  24.356  1.00  0.00           C  \\nATOM    332  O   TYR A  46     -21.327   0.733  24.167  1.00  0.00           O  \\nATOM    333  CB  TYR A  46     -22.524   4.198  23.159  1.00  0.00           C  \\nATOM    334  CG  TYR A  46     -22.146   5.143  22.151  1.00  0.00           C  \\nATOM    335  CD1 TYR A  46     -22.776   5.269  21.521  1.00  0.00           C  \\nATOM    336  CD2 TYR A  46     -20.886   5.647  21.647  1.00  0.00           C  \\nATOM    337  CE1 TYR A  46     -22.524   5.899  20.387  1.00  0.00           C  \\nATOM    338  CE2 TYR A  46     -20.382   6.277  20.513  1.00  0.00           C  \\nATOM    339  CZ  TYR A  46     -21.264   6.277  20.009  1.00  0.00           C  \\nATOM    340  OH  TYR A  46     -21.138   6.655  19.001  1.00  0.00           O  \\nATOM    341  N   SER A  47     -20.634   2.560  25.049  1.00  0.00           N  \\nATOM    342  CA  SER A  47     -19.374   1.741  25.679  1.00  0.00           C  \\nATOM    343  C   SER A  47     -18.114   2.623  25.679  1.00  0.00           C  \\nATOM    344  O   SER A  47     -18.114   3.379  24.797  1.00  0.00           O  \\nATOM    345  CB  SER A  47     -19.248   0.355  25.427  1.00  0.00           C  \\nATOM    346  OG  SER A  47     -18.870   0.481  23.789  1.00  0.00           O  \\nATOM    347  N   TYR A  48     -17.484   2.749  26.939  1.00  0.00           N  \\nATOM    348  CA  TYR A  48     -16.413   3.757  26.939  1.00  0.00           C  \\nATOM    349  C   TYR A  48     -16.602   4.891  27.695  1.00  0.00           C  \\nATOM    350  O   TYR A  48     -17.610   5.269  27.821  1.00  0.00           O  \\nATOM    351  CB  TYR A  48     -16.224   3.757  25.301  1.00  0.00           C  \\nATOM    352  CG  TYR A  48     -15.216   4.639  24.545  1.00  0.00           C  \\nATOM    353  CD1 TYR A  48     -14.082   5.143  24.797  1.00  0.00           C  \\nATOM    354  CD2 TYR A  48     -15.342   4.765  23.411  1.00  0.00           C  \\nATOM    355  CE1 TYR A  48     -13.326   6.025  24.167  1.00  0.00           C  \\nATOM    356  CE2 TYR A  48     -14.712   5.521  22.529  1.00  0.00           C  \\nATOM    357  CZ  TYR A  48     -13.704   6.088  23.033  1.00  0.00           C  \\nATOM    358  OH  TYR A  48     -13.074   6.907  22.151  1.00  0.00           O  \\nATOM    359  N   THR A  49     -15.594   5.647  28.199  1.00  0.00           N  \\nATOM    360  CA  THR A  49     -15.846   6.907  29.207  1.00  0.00           C  \\nATOM    361  C   THR A  49     -15.594   8.041  28.829  1.00  0.00           C  \\nATOM    362  O   THR A  49     -16.350   8.671  27.947  1.00  0.00           O  \\nATOM    363  CB  THR A  49     -15.531   6.403  30.845  1.00  0.00           C  \\nATOM    364  OG1 THR A  49     -14.208   5.899  30.845  1.00  0.00           O  \\nATOM    365  CG2 THR A  49     -16.224   5.395  31.223  1.00  0.00           C  \\nATOM    366  N   ASP A  50     -14.334   8.482  28.703  1.00  0.00           N  \\nATOM    367  CA  ASP A  50     -14.082   9.805  28.199  1.00  0.00           C  \\nATOM    368  C   ASP A  50     -13.704   9.679  26.813  1.00  0.00           C  \\nATOM    369  O   ASP A  50     -13.074   8.545  26.183  1.00  0.00           O  \\nATOM    370  CB  ASP A  50     -13.074  10.435  28.829  1.00  0.00           C  \\nATOM    371  CG  ASP A  50     -12.948  10.183  30.089  1.00  0.00           C  \\nATOM    372  OD1 ASP A  50     -12.066   9.049  30.593  1.00  0.00           O  \\nATOM    373  OD2 ASP A  50     -13.956  10.309  30.971  1.00  0.00           O  \\nATOM    374  N   ALA A  51     -13.830  10.309  25.805  1.00  0.00           N  \\nATOM    375  CA  ALA A  51     -13.704   9.931  24.419  1.00  0.00           C  \\nATOM    376  C   ALA A  51     -15.216   9.805  23.915  1.00  0.00           C  \\nATOM    377  O   ALA A  51     -16.098   9.301  24.167  1.00  0.00           O  \\nATOM    378  CB  ALA A  51     -13.200   8.671  24.041  1.00  0.00           C  \\nATOM    379  N   ASN A  52     -15.342  10.561  22.907  1.00  0.00           N  \\nATOM    380  CA  ASN A  52     -16.539  10.561  22.277  1.00  0.00           C  \\nATOM    381  C   ASN A  52     -17.232  11.380  22.907  1.00  0.00           C  \\nATOM    382  O   ASN A  52     -17.232  12.766  22.781  1.00  0.00           O  \\nATOM    383  CB  ASN A  52     -17.610   9.553  21.899  1.00  0.00           C  \\nATOM    384  CG  ASN A  52     -17.484   8.671  20.639  1.00  0.00           C  \\nATOM    385  OD1 ASN A  52     -16.980   7.789  20.135  1.00  0.00           O  \\nATOM    386  ND2 ASN A  52     -18.240   9.427  19.631  1.00  0.00           N  \\nATOM    387  N   LYS A  53     -17.862  10.939  24.041  1.00  0.00           N  \\nATOM    388  CA  LYS A  53     -18.240  11.947  25.301  1.00  0.00           C  \\nATOM    389  C   LYS A  53     -17.106  12.703  25.805  1.00  0.00           C  \\nATOM    390  O   LYS A  53     -16.476  13.711  25.175  1.00  0.00           O  \\nATOM    391  CB  LYS A  53     -18.870  11.317  26.561  1.00  0.00           C  \\nATOM    392  CG  LYS A  53     -20.130  10.435  26.057  1.00  0.00           C  \\nATOM    393  CD  LYS A  53     -20.634   9.301  27.065  1.00  0.00           C  \\nATOM    394  CE  LYS A  53     -22.146   8.671  26.372  1.00  0.00           C  \\nATOM    395  NZ  LYS A  53     -22.524   7.663  27.191  1.00  0.00           N  \\nATOM    396  N   SER A  54     -16.602  12.325  27.065  1.00  0.00           N  \\nATOM    397  CA  SER A  54     -15.594  13.081  27.821  1.00  0.00           C  \\nATOM    398  C   SER A  54     -15.594  14.593  27.443  1.00  0.00           C  \\nATOM    399  O   SER A  54     -16.476  15.223  27.695  1.00  0.00           O  \\nATOM    400  CB  SER A  54     -14.460  12.703  27.569  1.00  0.00           C  \\nATOM    401  OG  SER A  54     -13.452  12.955  28.703  1.00  0.00           O  \\nATOM    402  N   LYS A  55     -14.838  15.223  26.813  1.00  0.00           N  \\nATOM    403  CA  LYS A  55     -14.964  16.420  26.057  1.00  0.00           C  \\nATOM    404  C   LYS A  55     -16.602  16.987  25.931  1.00  0.00           C  \\nATOM    405  O   LYS A  55     -16.854  18.121  26.183  1.00  0.00           O  \\nATOM    406  CB  LYS A  55     -14.208  16.609  24.734  1.00  0.00           C  \\nATOM    407  CG  LYS A  55     -14.712  17.743  23.600  1.00  0.00           C  \\nATOM    408  CD  LYS A  55     -13.452  17.743  22.844  1.00  0.00           C  \\nATOM    409  CE  LYS A  55     -13.578  17.995  21.143  1.00  0.00           C  \\nATOM    410  NZ  LYS A  55     -12.444  17.743  20.135  1.00  0.00           N  \\nATOM    411  N   GLY A  56     -17.232  16.042  25.679  1.00  0.00           N  \\nATOM    412  CA  GLY A  56     -18.870  16.042  25.301  1.00  0.00           C  \\nATOM    413  C   GLY A  56     -19.122  17.239  24.293  1.00  0.00           C  \\nATOM    414  O   GLY A  56     -18.492  18.247  23.852  1.00  0.00           O  \\nATOM    415  N   ILE A  57     -20.130  17.113  23.285  1.00  0.00           N  \\nATOM    416  CA  ILE A  57     -20.004  17.869  22.025  1.00  0.00           C  \\nATOM    417  C   ILE A  57     -21.264  18.121  21.143  1.00  0.00           C  \\nATOM    418  O   ILE A  57     -22.398  18.499  21.395  1.00  0.00           O  \\nATOM    419  CB  ILE A  57     -18.996  17.491  21.395  1.00  0.00           C  \\nATOM    420  CG1 ILE A  57     -19.374  16.357  20.513  1.00  0.00           C  \\nATOM    421  CG2 ILE A  57     -17.862  17.050  22.025  1.00  0.00           C  \\nATOM    422  CD1 ILE A  57     -19.122  16.609  19.127  1.00  0.00           C  \\nATOM    423  N   VAL A  58     -21.264  17.743  19.631  1.00  0.00           N  \\nATOM    424  CA  VAL A  58     -22.524  17.995  19.001  1.00  0.00           C  \\nATOM    425  C   VAL A  58     -22.020  17.239  17.615  1.00  0.00           C  \\nATOM    426  O   VAL A  58     -21.642  18.121  16.985  1.00  0.00           O  \\nATOM    427  CB  VAL A  58     -23.280  19.129  19.127  1.00  0.00           C  \\nATOM    428  CG1 VAL A  58     -22.398  20.263  18.749  1.00  0.00           C  \\nATOM    429  CG2 VAL A  58     -24.540  19.192  18.371  1.00  0.00           C  \\nATOM    430  N   TRP A  59     -22.398  16.231  16.859  1.00  0.00           N  \\nATOM    431  CA  TRP A  59     -22.020  16.105  15.221  1.00  0.00           C  \\nATOM    432  C   TRP A  59     -22.524  17.239  14.213  1.00  0.00           C  \\nATOM    433  O   TRP A  59     -23.784  17.365  14.339  1.00  0.00           O  \\nATOM    434  CB  TRP A  59     -22.146  14.845  14.717  1.00  0.00           C  \\nATOM    435  CG  TRP A  59     -21.264  13.963  15.347  1.00  0.00           C  \\nATOM    436  CD1 TRP A  59     -21.390  12.703  15.851  1.00  0.00           C  \\nATOM    437  CD2 TRP A  59     -19.878  14.152  15.536  1.00  0.00           C  \\nATOM    438  NE1 TRP A  59     -20.382  12.451  16.607  1.00  0.00           N  \\nATOM    439  CE2 TRP A  59     -19.500  13.207  15.977  1.00  0.00           C  \\nATOM    440  CE3 TRP A  59     -19.248  15.160  15.095  1.00  0.00           C  \\nATOM    441  CZ2 TRP A  59     -18.114  13.459  16.607  1.00  0.00           C  \\nATOM    442  CZ3 TRP A  59     -18.051  15.223  15.473  1.00  0.00           C  \\nATOM    443  CH2 TRP A  59     -17.484  14.467  16.229  1.00  0.00           C  \\nATOM    444  N   ASN A  60     -21.768  17.869  13.772  1.00  0.00           N  \\nATOM    445  CA  ASN A  60     -22.398  19.129  12.953  1.00  0.00           C  \\nATOM    446  C   ASN A  60     -21.642  19.255  11.441  1.00  0.00           C  \\nATOM    447  O   ASN A  60     -20.634  18.751  11.189  1.00  0.00           O  \\nATOM    448  CB  ASN A  60     -22.524  20.515  13.583  1.00  0.00           C  \\nATOM    449  CG  ASN A  60     -22.776  21.649  12.701  1.00  0.00           C  \\nATOM    450  OD1 ASN A  60     -24.036  21.712  11.945  1.00  0.00           O  \\nATOM    451  ND2 ASN A  60     -21.894  22.783  12.701  1.00  0.00           N  \\nATOM    452  N   GLU A  61     -22.020  20.137  10.811  1.00  0.00           N  \\nATOM    453  CA  GLU A  61     -21.138  20.263   9.425  1.00  0.00           C  \\nATOM    454  C   GLU A  61     -19.626  20.389  10.181  1.00  0.00           C  \\nATOM    455  O   GLU A  61     -18.744  19.507   9.803  1.00  0.00           O  \\nATOM    456  CB  GLU A  61     -21.516  21.523   8.669  1.00  0.00           C  \\nATOM    457  CG  GLU A  61     -22.020  21.523   7.409  1.00  0.00           C  \\nATOM    458  CD  GLU A  61     -23.154  20.767   7.283  1.00  0.00           C  \\nATOM    459  OE1 GLU A  61     -24.540  20.767   7.535  1.00  0.00           O  \\nATOM    460  OE2 GLU A  61     -22.902  19.381   6.653  1.00  0.00           O  \\nATOM    461  N   ASN A  62     -19.248  21.397  10.685  1.00  0.00           N  \\nATOM    462  CA  ASN A  62     -17.736  21.586  11.441  1.00  0.00           C  \\nATOM    463  C   ASN A  62     -17.106  20.137  11.945  1.00  0.00           C  \\nATOM    464  O   ASN A  62     -15.972  20.515  12.071  1.00  0.00           O  \\nATOM    465  CB  ASN A  62     -17.736  22.783  11.945  1.00  0.00           C  \\nATOM    466  CG  ASN A  62     -17.610  24.043  11.441  1.00  0.00           C  \\nATOM    467  OD1 ASN A  62     -18.744  24.547  10.559  1.00  0.00           O  \\nATOM    468  ND2 ASN A  62     -16.539  24.799  11.063  1.00  0.00           N  \\nATOM    469  N   THR A  63     -17.610  20.263  13.268  1.00  0.00           N  \\nATOM    470  CA  THR A  63     -17.043  19.255  14.087  1.00  0.00           C  \\nATOM    471  C   THR A  63     -16.602  17.869  13.331  1.00  0.00           C  \\nATOM    472  O   THR A  63     -15.720  17.239  13.331  1.00  0.00           O  \\nATOM    473  CB  THR A  63     -18.051  18.751  15.095  1.00  0.00           C  \\nATOM    474  OG1 THR A  63     -17.358  18.499  15.851  1.00  0.00           O  \\nATOM    475  CG2 THR A  63     -18.492  17.365  14.528  1.00  0.00           C  \\nATOM    476  N   LEU A  64     -17.358  17.617  12.197  1.00  0.00           N  \\nATOM    477  CA  LEU A  64     -16.854  16.357  11.441  1.00  0.00           C  \\nATOM    478  C   LEU A  64     -15.468  16.861  10.685  1.00  0.00           C  \\nATOM    479  O   LEU A  64     -14.838  16.231  10.433  1.00  0.00           O  \\nATOM    480  CB  LEU A  64     -18.114  15.979  10.055  1.00  0.00           C  \\nATOM    481  CG  LEU A  64     -18.744  14.719  10.433  1.00  0.00           C  \\nATOM    482  CD1 LEU A  64     -19.122  14.719  11.567  1.00  0.00           C  \\nATOM    483  CD2 LEU A  64     -19.878  14.593   9.803  1.00  0.00           C  \\nATOM    484  N   MET A  65     -15.468  17.995  10.181  1.00  0.00           N  \\nATOM    485  CA  MET A  65     -13.956  18.499   9.803  1.00  0.00           C  \\nATOM    486  C   MET A  65     -13.200  18.373  11.189  1.00  0.00           C  \\nATOM    487  O   MET A  65     -12.192  17.869  11.189  1.00  0.00           O  \\nATOM    488  CB  MET A  65     -13.956  19.885   9.425  1.00  0.00           C  \\nATOM    489  CG  MET A  65     -13.452  20.137   7.913  1.00  0.00           C  \\nATOM    490  SD  MET A  65     -13.704  18.625   7.283  1.00  0.00           S  \\nATOM    491  CE  MET A  65     -13.452  19.192   5.519  1.00  0.00           C  \\nATOM    492  N   GLU A  66     -13.578  18.751  12.197  1.00  0.00           N  \\nATOM    493  CA  GLU A  66     -13.074  18.751  13.457  1.00  0.00           C  \\nATOM    494  C   GLU A  66     -12.444  17.239  13.646  1.00  0.00           C  \\nATOM    495  O   GLU A  66     -11.310  17.113  13.835  1.00  0.00           O  \\nATOM    496  CB  GLU A  66     -13.830  19.381  14.717  1.00  0.00           C  \\nATOM    497  CG  GLU A  66     -13.578  20.767  14.969  1.00  0.00           C  \\nATOM    498  CD  GLU A  66     -12.318  21.271  15.851  1.00  0.00           C  \\nATOM    499  OE1 GLU A  66     -12.192  22.657  16.103  1.00  0.00           O  \\nATOM    500  OE2 GLU A  66     -11.310  20.641  16.355  1.00  0.00           O  \\nATOM    501  N   TYR A  67     -13.263  16.483  13.457  1.00  0.00           N  \\nATOM    502  CA  TYR A  67     -12.822  14.971  13.457  1.00  0.00           C  \\nATOM    503  C   TYR A  67     -12.192  14.467  12.323  1.00  0.00           C  \\nATOM    504  O   TYR A  67     -11.184  14.089  11.945  1.00  0.00           O  \\nATOM    505  CB  TYR A  67     -14.082  14.341  13.835  1.00  0.00           C  \\nATOM    506  CG  TYR A  67     -14.082  12.955  14.465  1.00  0.00           C  \\nATOM    507  CD1 TYR A  67     -14.838  11.821  14.465  1.00  0.00           C  \\nATOM    508  CD2 TYR A  67     -13.326  13.081  15.599  1.00  0.00           C  \\nATOM    509  CE1 TYR A  67     -14.649  10.876  15.221  1.00  0.00           C  \\nATOM    510  CE2 TYR A  67     -13.200  11.947  16.103  1.00  0.00           C  \\nATOM    511  CZ  TYR A  67     -13.830  10.876  15.977  1.00  0.00           C  \\nATOM    512  OH  TYR A  67     -13.704   9.805  16.607  1.00  0.00           O  \\nATOM    513  N   LEU A  68     -12.822  14.719  11.315  1.00  0.00           N  \\nATOM    514  CA  LEU A  68     -12.192  14.341  10.055  1.00  0.00           C  \\nATOM    515  C   LEU A  68     -10.806  14.719  10.307  1.00  0.00           C  \\nATOM    516  O   LEU A  68      -9.798  13.963  10.307  1.00  0.00           O  \\nATOM    517  CB  LEU A  68     -13.074  14.845   9.173  1.00  0.00           C  \\nATOM    518  CG  LEU A  68     -14.334  13.837   8.795  1.00  0.00           C  \\nATOM    519  CD1 LEU A  68     -15.216  14.467   7.787  1.00  0.00           C  \\nATOM    520  CD2 LEU A  68     -13.704  12.829   7.787  1.00  0.00           C  \\nATOM    521  N   GLU A  69     -10.428  15.853  10.685  1.00  0.00           N  \\nATOM    522  CA  GLU A  69      -8.916  16.231  10.937  1.00  0.00           C  \\nATOM    523  C   GLU A  69      -8.097  14.971  11.441  1.00  0.00           C  \\nATOM    524  O   GLU A  69      -7.278  14.593  10.685  1.00  0.00           O  \\nATOM    525  CB  GLU A  69      -9.042  17.365  11.693  1.00  0.00           C  \\nATOM    526  CG  GLU A  69      -8.034  18.373  11.819  1.00  0.00           C  \\nATOM    527  CD  GLU A  69      -6.900  18.373  12.953  1.00  0.00           C  \\nATOM    528  OE1 GLU A  69      -5.388  18.499  12.827  1.00  0.00           O  \\nATOM    529  OE2 GLU A  69      -7.278  17.869  14.087  1.00  0.00           O  \\nATOM    530  N   ASN A  70      -8.097  14.845  12.575  1.00  0.00           N  \\nATOM    531  CA  ASN A  70      -7.278  13.585  12.827  1.00  0.00           C  \\nATOM    532  C   ASN A  70      -7.404  13.459  14.339  1.00  0.00           C  \\nATOM    533  O   ASN A  70      -7.152  14.215  15.221  1.00  0.00           O  \\nATOM    534  CB  ASN A  70      -5.892  13.837  12.449  1.00  0.00           C  \\nATOM    535  CG  ASN A  70      -4.758  14.089  13.457  1.00  0.00           C  \\nATOM    536  OD1 ASN A  70      -3.624  15.097  13.457  1.00  0.00           O  \\nATOM    537  ND2 ASN A  70      -4.758  13.333  14.717  1.00  0.00           N  \\nATOM    538  N   PRO A  71      -7.908  12.199  14.780  1.00  0.00           N  \\nATOM    539  CA  PRO A  71      -8.475  11.758  15.725  1.00  0.00           C  \\nATOM    540  C   PRO A  71      -7.593  11.758  16.859  1.00  0.00           C  \\nATOM    541  O   PRO A  71      -7.782  11.317  17.993  1.00  0.00           O  \\nATOM    542  CB  PRO A  71      -9.042  10.561  15.536  1.00  0.00           C  \\nATOM    543  CG  PRO A  71      -8.916  10.309  14.213  1.00  0.00           C  \\nATOM    544  CD  PRO A  71      -8.160  11.191  13.583  1.00  0.00           C  \\nATOM    545  N   LYS A  72      -6.522  12.325  16.733  1.00  0.00           N  \\nATOM    546  CA  LYS A  72      -5.388  12.703  17.615  1.00  0.00           C  \\nATOM    547  C   LYS A  72      -5.451  14.215  17.993  1.00  0.00           C  \\nATOM    548  O   LYS A  72      -5.262  14.971  17.111  1.00  0.00           O  \\nATOM    549  CB  LYS A  72      -3.876  12.451  17.111  1.00  0.00           C  \\nATOM    550  CG  LYS A  72      -3.624  11.065  17.363  1.00  0.00           C  \\nATOM    551  CD  LYS A  72      -2.112  10.435  17.237  1.00  0.00           C  \\nATOM    552  CE  LYS A  72      -1.860   8.923  17.741  1.00  0.00           C  \\nATOM    553  NZ  LYS A  72      -0.222   8.545  17.741  1.00  0.00           N  \\nATOM    554  N   LYS A  73      -6.081  14.719  19.127  1.00  0.00           N  \\nATOM    555  CA  LYS A  73      -6.396  16.231  19.127  1.00  0.00           C  \\nATOM    556  C   LYS A  73      -7.782  16.357  18.623  1.00  0.00           C  \\nATOM    557  O   LYS A  73      -8.160  16.231  17.363  1.00  0.00           O  \\nATOM    558  CB  LYS A  73      -5.640  17.113  17.993  1.00  0.00           C  \\nATOM    559  CG  LYS A  73      -4.758  18.247  18.623  1.00  0.00           C  \\nATOM    560  CD  LYS A  73      -3.876  19.003  17.993  1.00  0.00           C  \\nATOM    561  CE  LYS A  73      -2.616  20.389  18.812  1.00  0.00           C  \\nATOM    562  NZ  LYS A  73      -1.986  21.145  18.056  1.00  0.00           N  \\nATOM    563  N   TYR A  74      -8.664  16.231  19.631  1.00  0.00           N  \\nATOM    564  CA  TYR A  74     -10.050  16.105  19.379  1.00  0.00           C  \\nATOM    565  C   TYR A  74     -10.806  14.971  19.883  1.00  0.00           C  \\nATOM    566  O   TYR A  74     -10.554  13.963  19.379  1.00  0.00           O  \\nATOM    567  CB  TYR A  74     -10.554  16.105  17.741  1.00  0.00           C  \\nATOM    568  CG  TYR A  74     -11.814  16.735  17.363  1.00  0.00           C  \\nATOM    569  CD1 TYR A  74     -12.948  16.105  17.237  1.00  0.00           C  \\nATOM    570  CD2 TYR A  74     -11.940  17.995  16.985  1.00  0.00           C  \\nATOM    571  CE1 TYR A  74     -14.145  16.609  16.859  1.00  0.00           C  \\nATOM    572  CE2 TYR A  74     -13.074  18.625  16.607  1.00  0.00           C  \\nATOM    573  CZ  TYR A  74     -14.082  17.869  16.607  1.00  0.00           C  \\nATOM    574  OH  TYR A  74     -15.216  18.373  16.355  1.00  0.00           O  \\nATOM    575  N   ILE A  75     -11.310  15.097  21.017  1.00  0.00           N  \\nATOM    576  CA  ILE A  75     -11.940  14.089  21.647  1.00  0.00           C  \\nATOM    577  C   ILE A  75     -10.806  13.081  21.773  1.00  0.00           C  \\nATOM    578  O   ILE A  75     -10.932  11.821  21.521  1.00  0.00           O  \\nATOM    579  CB  ILE A  75     -13.200  13.585  21.143  1.00  0.00           C  \\nATOM    580  CG1 ILE A  75     -13.641  14.089  19.631  1.00  0.00           C  \\nATOM    581  CG2 ILE A  75     -14.271  13.585  22.025  1.00  0.00           C  \\nATOM    582  CD1 ILE A  75     -14.712  13.459  19.253  1.00  0.00           C  \\nATOM    583  N   PRO A  76      -9.420  13.585  21.899  1.00  0.00           N  \\nATOM    584  CA  PRO A  76      -8.160  12.955  21.647  1.00  0.00           C  \\nATOM    585  C   PRO A  76      -8.160  11.695  22.529  1.00  0.00           C  \\nATOM    586  O   PRO A  76      -8.538  11.695  23.789  1.00  0.00           O  \\nATOM    587  CB  PRO A  76      -6.900  13.837  21.836  1.00  0.00           C  \\nATOM    588  CG  PRO A  76      -7.152  14.845  22.277  1.00  0.00           C  \\nATOM    589  CD  PRO A  76      -8.538  14.845  22.151  1.00  0.00           C  \\nATOM    590  N   GLY A  77      -8.097  10.435  22.088  1.00  0.00           N  \\nATOM    591  CA  GLY A  77      -8.286   9.175  22.907  1.00  0.00           C  \\nATOM    592  C   GLY A  77      -9.546   8.167  22.529  1.00  0.00           C  \\nATOM    593  O   GLY A  77      -9.798   7.159  22.781  1.00  0.00           O  \\nATOM    594  N   THR A  78     -10.050   8.545  21.143  1.00  0.00           N  \\nATOM    595  CA  THR A  78     -11.310   7.789  20.639  1.00  0.00           C  \\nATOM    596  C   THR A  78     -10.995   6.529  20.513  1.00  0.00           C  \\nATOM    597  O   THR A  78      -9.798   6.151  20.009  1.00  0.00           O  \\nATOM    598  CB  THR A  78     -11.625   8.167  19.253  1.00  0.00           C  \\nATOM    599  OG1 THR A  78     -13.074   8.167  19.127  1.00  0.00           O  \\nATOM    600  CG2 THR A  78     -11.184   9.553  18.875  1.00  0.00           C  \\nATOM    601  N   LYS A  79     -11.562   5.521  20.765  1.00  0.00           N  \\nATOM    602  CA  LYS A  79     -11.058   4.072  20.639  1.00  0.00           C  \\nATOM    603  C   LYS A  79     -11.058   3.631  19.001  1.00  0.00           C  \\nATOM    604  O   LYS A  79     -10.932   2.497  18.371  1.00  0.00           O  \\nATOM    605  CB  LYS A  79     -11.562   3.064  21.395  1.00  0.00           C  \\nATOM    606  CG  LYS A  79     -12.570   2.560  20.513  1.00  0.00           C  \\nATOM    607  CD  LYS A  79     -13.704   1.615  20.891  1.00  0.00           C  \\nATOM    608  CE  LYS A  79     -14.838   1.363  20.135  1.00  0.00           C  \\nATOM    609  NZ  LYS A  79     -15.972   0.859  20.513  1.00  0.00           N  \\nATOM    610  N   MET A  80     -11.310   4.639  18.119  1.00  0.00           N  \\nATOM    611  CA  MET A  80     -11.436   4.135  16.607  1.00  0.00           C  \\nATOM    612  C   MET A  80     -10.050   4.198  16.103  1.00  0.00           C  \\nATOM    613  O   MET A  80      -9.294   5.017  16.103  1.00  0.00           O  \\nATOM    614  CB  MET A  80     -12.444   5.017  15.851  1.00  0.00           C  \\nATOM    615  CG  MET A  80     -11.940   6.088  14.969  1.00  0.00           C  \\nATOM    616  SD  MET A  80     -13.074   6.781  13.457  1.00  0.00           S  \\nATOM    617  CE  MET A  80     -12.192   7.852  13.079  1.00  0.00           C  \\nATOM    618  N   ILE A  81      -9.672   3.127  15.725  1.00  0.00           N  \\nATOM    619  CA  ILE A  81      -8.160   3.127  15.095  1.00  0.00           C  \\nATOM    620  C   ILE A  81      -7.908   4.009  13.835  1.00  0.00           C  \\nATOM    621  O   ILE A  81      -8.034   3.505  12.323  1.00  0.00           O  \\nATOM    622  CB  ILE A  81      -7.656   1.867  15.284  1.00  0.00           C  \\nATOM    623  CG1 ILE A  81      -6.522   1.867  16.103  1.00  0.00           C  \\nATOM    624  CG2 ILE A  81      -7.026   1.237  13.835  1.00  0.00           C  \\nATOM    625  CD1 ILE A  81      -6.900   2.245  17.615  1.00  0.00           C  \\nATOM    626  N   PHE A  82      -7.530   5.143  13.961  1.00  0.00           N  \\nATOM    627  CA  PHE A  82      -7.404   5.899  12.449  1.00  0.00           C  \\nATOM    628  C   PHE A  82      -6.144   6.529  12.197  1.00  0.00           C  \\nATOM    629  O   PHE A  82      -5.766   7.411  12.953  1.00  0.00           O  \\nATOM    630  CB  PHE A  82      -8.538   6.655  12.449  1.00  0.00           C  \\nATOM    631  CG  PHE A  82      -8.916   7.159  10.937  1.00  0.00           C  \\nATOM    632  CD1 PHE A  82      -8.916   8.293  10.433  1.00  0.00           C  \\nATOM    633  CD2 PHE A  82      -9.042   6.529  10.055  1.00  0.00           C  \\nATOM    634  CE1 PHE A  82      -9.042   8.671   9.173  1.00  0.00           C  \\nATOM    635  CE2 PHE A  82      -9.042   6.781   8.669  1.00  0.00           C  \\nATOM    636  CZ  PHE A  82      -9.042   7.852   8.291  1.00  0.00           C  \\nATOM    637  N   ALA A  83      -5.640   6.529  11.189  1.00  0.00           N  \\nATOM    638  CA  ALA A  83      -4.506   7.285  10.937  1.00  0.00           C  \\nATOM    639  C   ALA A  83      -4.758   8.797  10.811  1.00  0.00           C  \\nATOM    640  O   ALA A  83      -4.884   9.679  11.441  1.00  0.00           O  \\nATOM    641  CB  ALA A  83      -3.876   6.907   9.173  1.00  0.00           C  \\nATOM    642  N   GLY A  84      -5.136   9.301   9.677  1.00  0.00           N  \\nATOM    643  CA  GLY A  84      -5.325  10.687   9.299  1.00  0.00           C  \\nATOM    644  C   GLY A  84      -6.396  11.317   8.543  1.00  0.00           C  \\nATOM    645  O   GLY A  84      -7.467  11.569   8.921  1.00  0.00           O  \\nATOM    646  N   ILE A  85      -6.207  11.569   7.283  1.00  0.00           N  \\nATOM    647  CA  ILE A  85      -7.152  12.451   6.653  1.00  0.00           C  \\nATOM    648  C   ILE A  85      -6.774  13.963   6.905  1.00  0.00           C  \\nATOM    649  O   ILE A  85      -7.656  14.719   7.220  1.00  0.00           O  \\nATOM    650  CB  ILE A  85      -8.664  12.199   7.031  1.00  0.00           C  \\nATOM    651  CG1 ILE A  85      -9.168  10.813   6.149  1.00  0.00           C  \\nATOM    652  CG2 ILE A  85      -9.420  13.333   6.401  1.00  0.00           C  \\nATOM    653  CD1 ILE A  85     -10.176  11.191   5.204  1.00  0.00           C  \\nATOM    654  N   LYS A  86      -5.640  14.467   6.527  1.00  0.00           N  \\nATOM    655  CA  LYS A  86      -5.325  15.727   7.157  1.00  0.00           C  \\nATOM    656  C   LYS A  86      -5.136  16.546   5.771  1.00  0.00           C  \\nATOM    657  O   LYS A  86      -4.254  17.365   5.267  1.00  0.00           O  \\nATOM    658  CB  LYS A  86      -3.876  15.727   7.913  1.00  0.00           C  \\nATOM    659  CG  LYS A  86      -3.120  14.467   7.535  1.00  0.00           C  \\nATOM    660  CD  LYS A  86      -1.356  14.215   7.787  1.00  0.00           C  \\nATOM    661  CE  LYS A  86      -0.852  12.955   6.653  1.00  0.00           C  \\nATOM    662  NZ  LYS A  86       0.660  12.577   7.157  1.00  0.00           N  \\nATOM    663  N   LYS A  87      -5.892  16.357   5.015  1.00  0.00           N  \\nATOM    664  CA  LYS A  87      -5.892  17.239   3.755  1.00  0.00           C  \\nATOM    665  C   LYS A  87      -7.278  17.869   3.377  1.00  0.00           C  \\nATOM    666  O   LYS A  87      -8.160  17.239   2.873  1.00  0.00           O  \\nATOM    667  CB  LYS A  87      -5.514  16.294   2.558  1.00  0.00           C  \\nATOM    668  CG  LYS A  87      -4.632  16.735   1.235  1.00  0.00           C  \\nATOM    669  CD  LYS A  87      -4.254  15.727   0.353  1.00  0.00           C  \\nATOM    670  CE  LYS A  87      -3.120  16.231  -0.403  1.00  0.00           C  \\nATOM    671  NZ  LYS A  87      -2.868  15.727  -1.915  1.00  0.00           N  \\nATOM    672  N   LYS A  88      -7.278  19.129   3.251  1.00  0.00           N  \\nATOM    673  CA  LYS A  88      -8.412  19.885   2.621  1.00  0.00           C  \\nATOM    674  C   LYS A  88      -9.168  18.877   1.613  1.00  0.00           C  \\nATOM    675  O   LYS A  88     -10.428  18.499   1.613  1.00  0.00           O  \\nATOM    676  CB  LYS A  88      -7.782  21.019   1.991  1.00  0.00           C  \\nATOM    677  CG  LYS A  88      -6.270  20.767   1.865  1.00  0.00           C  \\nATOM    678  CD  LYS A  88      -5.514  21.271   0.605  1.00  0.00           C  \\nATOM    679  CE  LYS A  88      -5.766  20.767  -0.403  1.00  0.00           C  \\nATOM    680  NZ  LYS A  88      -5.388  21.523  -1.663  1.00  0.00           N  \\nATOM    681  N   GLY A  89      -8.538  18.499   0.353  1.00  0.00           N  \\nATOM    682  CA  GLY A  89      -9.168  17.491  -0.781  1.00  0.00           C  \\nATOM    683  C   GLY A  89     -10.050  16.357  -0.025  1.00  0.00           C  \\nATOM    684  O   GLY A  89     -11.184  16.357  -0.025  1.00  0.00           O  \\nATOM    685  N   GLU A  90      -9.546  15.475   0.479  1.00  0.00           N  \\nATOM    686  CA  GLU A  90     -10.050  14.089   1.109  1.00  0.00           C  \\nATOM    687  C   GLU A  90     -10.932  14.593   2.369  1.00  0.00           C  \\nATOM    688  O   GLU A  90     -12.192  14.341   2.621  1.00  0.00           O  \\nATOM    689  CB  GLU A  90      -9.042  13.207   1.613  1.00  0.00           C  \\nATOM    690  CG  GLU A  90      -8.286  12.577   0.605  1.00  0.00           C  \\nATOM    691  CD  GLU A  90      -6.900  11.821   0.857  1.00  0.00           C  \\nATOM    692  OE1 GLU A  90      -6.018  11.443  -0.277  1.00  0.00           O  \\nATOM    693  OE2 GLU A  90      -6.270  11.569   2.243  1.00  0.00           O  \\nATOM    694  N   ARG A  91     -10.554  15.475   2.999  1.00  0.00           N  \\nATOM    695  CA  ARG A  91     -11.184  15.916   4.511  1.00  0.00           C  \\nATOM    696  C   ARG A  91     -12.570  16.672   3.881  1.00  0.00           C  \\nATOM    697  O   ARG A  91     -13.578  16.357   4.133  1.00  0.00           O  \\nATOM    698  CB  ARG A  91     -10.302  16.987   5.141  1.00  0.00           C  \\nATOM    699  CG  ARG A  91     -10.554  17.113   6.527  1.00  0.00           C  \\nATOM    700  CD  ARG A  91      -9.420  18.121   7.283  1.00  0.00           C  \\nATOM    701  NE  ARG A  91      -9.294  19.381   6.527  1.00  0.00           N  \\nATOM    702  CZ  ARG A  91      -7.908  20.137   6.464  1.00  0.00           C  \\nATOM    703  NH1 ARG A  91      -6.900  19.507   7.535  1.00  0.00           N  \\nATOM    704  NH2 ARG A  91      -7.530  21.145   5.519  1.00  0.00           N  \\nATOM    705  N   GLN A  92     -12.444  17.743   3.251  1.00  0.00           N  \\nATOM    706  CA  GLN A  92     -13.830  18.373   2.369  1.00  0.00           C  \\nATOM    707  C   GLN A  92     -14.586  17.113   1.991  1.00  0.00           C  \\nATOM    708  O   GLN A  92     -15.720  16.735   2.243  1.00  0.00           O  \\nATOM    709  CB  GLN A  92     -13.704  19.633   1.739  1.00  0.00           C  \\nATOM    710  CG  GLN A  92     -13.830  19.255   0.164  1.00  0.00           C  \\nATOM    711  CD  GLN A  92     -13.200  20.326  -0.907  1.00  0.00           C  \\nATOM    712  OE1 GLN A  92     -11.688  19.885  -0.403  1.00  0.00           O  \\nATOM    713  NE2 GLN A  92     -13.452  21.460  -1.411  1.00  0.00           N  \\nATOM    714  N   ASP A  93     -13.956  16.357   0.983  1.00  0.00           N  \\nATOM    715  CA  ASP A  93     -14.838  15.223   0.605  1.00  0.00           C  \\nATOM    716  C   ASP A  93     -15.342  14.341   1.613  1.00  0.00           C  \\nATOM    717  O   ASP A  93     -16.602  13.837   1.487  1.00  0.00           O  \\nATOM    718  CB  ASP A  93     -13.830  14.593  -0.529  1.00  0.00           C  \\nATOM    719  CG  ASP A  93     -13.704  14.971  -1.915  1.00  0.00           C  \\nATOM    720  OD1 ASP A  93     -12.444  15.727  -2.419  1.00  0.00           O  \\nATOM    721  OD2 ASP A  93     -14.838  15.223  -2.797  1.00  0.00           O  \\nATOM    722  N   LEU A  94     -14.838  13.963   2.495  1.00  0.00           N  \\nATOM    723  CA  LEU A  94     -15.531  13.207   3.503  1.00  0.00           C  \\nATOM    724  C   LEU A  94     -16.728  13.963   3.881  1.00  0.00           C  \\nATOM    725  O   LEU A  94     -17.925  13.459   3.881  1.00  0.00           O  \\nATOM    726  CB  LEU A  94     -14.586  13.018   4.763  1.00  0.00           C  \\nATOM    727  CG  LEU A  94     -14.586  11.443   5.267  1.00  0.00           C  \\nATOM    728  CD1 LEU A  94     -15.783  10.813   4.637  1.00  0.00           C  \\nATOM    729  CD2 LEU A  94     -13.578  10.561   4.889  1.00  0.00           C  \\nATOM    730  N   VAL A  95     -16.539  15.097   4.637  1.00  0.00           N  \\nATOM    731  CA  VAL A  95     -17.736  15.979   5.141  1.00  0.00           C  \\nATOM    732  C   VAL A  95     -18.744  15.853   3.881  1.00  0.00           C  \\nATOM    733  O   VAL A  95     -20.004  15.601   4.322  1.00  0.00           O  \\nATOM    734  CB  VAL A  95     -17.358  17.365   5.393  1.00  0.00           C  \\nATOM    735  CG1 VAL A  95     -18.492  18.184   5.519  1.00  0.00           C  \\nATOM    736  CG2 VAL A  95     -16.476  17.491   6.401  1.00  0.00           C  \\nATOM    737  N   ALA A  96     -18.492  15.979   2.621  1.00  0.00           N  \\nATOM    738  CA  ALA A  96     -19.752  15.601   1.739  1.00  0.00           C  \\nATOM    739  C   ALA A  96     -20.130  14.215   1.802  1.00  0.00           C  \\nATOM    740  O   ALA A  96     -21.390  13.837   1.802  1.00  0.00           O  \\nATOM    741  CB  ALA A  96     -19.500  16.105   0.353  1.00  0.00           C  \\nATOM    742  N   TYR A  97     -19.248  13.333   1.865  1.00  0.00           N  \\nATOM    743  CA  TYR A  97     -19.626  11.884   1.865  1.00  0.00           C  \\nATOM    744  C   TYR A  97     -20.886  11.632   2.873  1.00  0.00           C  \\nATOM    745  O   TYR A  97     -22.146  11.317   2.621  1.00  0.00           O  \\nATOM    746  CB  TYR A  97     -18.744  10.813   2.243  1.00  0.00           C  \\nATOM    747  CG  TYR A  97     -19.374   9.427   1.739  1.00  0.00           C  \\nATOM    748  CD1 TYR A  97     -19.626   8.419   2.621  1.00  0.00           C  \\nATOM    749  CD2 TYR A  97     -19.626   9.175   0.353  1.00  0.00           C  \\nATOM    750  CE1 TYR A  97     -20.256   7.348   2.117  1.00  0.00           C  \\nATOM    751  CE2 TYR A  97     -20.256   8.041  -0.403  1.00  0.00           C  \\nATOM    752  CZ  TYR A  97     -20.508   7.285   0.857  1.00  0.00           C  \\nATOM    753  OH  TYR A  97     -21.264   6.403   0.353  1.00  0.00           O  \\nATOM    754  N   LEU A  98     -20.760  11.947   4.007  1.00  0.00           N  \\nATOM    755  CA  LEU A  98     -21.768  11.695   5.393  1.00  0.00           C  \\nATOM    756  C   LEU A  98     -23.154  12.703   5.015  1.00  0.00           C  \\nATOM    757  O   LEU A  98     -24.288  12.199   4.889  1.00  0.00           O  \\nATOM    758  CB  LEU A  98     -21.012  11.695   6.527  1.00  0.00           C  \\nATOM    759  CG  LEU A  98     -20.004  10.939   6.401  1.00  0.00           C  \\nATOM    760  CD1 LEU A  98     -19.374  10.687   7.661  1.00  0.00           C  \\nATOM    761  CD2 LEU A  98     -20.508   9.553   5.771  1.00  0.00           C  \\nATOM    762  N   LYS A  99     -23.028  13.963   4.889  1.00  0.00           N  \\nATOM    763  CA  LYS A  99     -24.288  14.593   4.385  1.00  0.00           C  \\nATOM    764  C   LYS A  99     -25.044  13.963   2.999  1.00  0.00           C  \\nATOM    765  O   LYS A  99     -26.304  13.837   3.125  1.00  0.00           O  \\nATOM    766  CB  LYS A  99     -23.910  16.105   4.259  1.00  0.00           C  \\nATOM    767  CG  LYS A  99     -24.918  17.113   4.259  1.00  0.00           C  \\nATOM    768  CD  LYS A  99     -24.666  18.625   3.881  1.00  0.00           C  \\nATOM    769  CE  LYS A  99     -25.674  19.633   4.133  1.00  0.00           C  \\nATOM    770  NZ  LYS A  99     -25.674  21.082   3.755  1.00  0.00           N  \\nATOM    771  N   SER A 100     -24.540  13.459   1.991  1.00  0.00           N  \\nATOM    772  CA  SER A 100     -25.296  12.577   0.857  1.00  0.00           C  \\nATOM    773  C   SER A 100     -25.548  11.191   1.361  1.00  0.00           C  \\nATOM    774  O   SER A 100     -26.934  10.813   1.361  1.00  0.00           O  \\nATOM    775  CB  SER A 100     -24.540  12.577  -0.655  1.00  0.00           C  \\nATOM    776  OG  SER A 100     -25.296  11.695  -1.663  1.00  0.00           O  \\nATOM    777  N   ALA A 101     -24.666  10.435   1.865  1.00  0.00           N  \\nATOM    778  CA  ALA A 101     -25.044   8.797   2.243  1.00  0.00           C  \\nATOM    779  C   ALA A 101     -25.800   8.923   3.629  1.00  0.00           C  \\nATOM    780  O   ALA A 101     -26.808   8.293   3.755  1.00  0.00           O  \\nATOM    781  CB  ALA A 101     -23.910   8.167   2.369  1.00  0.00           C  \\nATOM    782  N   THR A 102     -25.422   9.427   4.637  1.00  0.00           N  \\nATOM    783  CA  THR A 102     -26.052   9.301   6.149  1.00  0.00           C  \\nATOM    784  C   THR A 102     -27.690   9.805   5.897  1.00  0.00           C  \\nATOM    785  O   THR A 102     -28.572   9.049   6.149  1.00  0.00           O  \\nATOM    786  CB  THR A 102     -25.548  10.246   7.220  1.00  0.00           C  \\nATOM    787  OG1 THR A 102     -24.666   9.427   7.913  1.00  0.00           O  \\nATOM    788  CG2 THR A 102     -26.430  10.876   8.039  1.00  0.00           C  \\nATOM    789  N   SER A 103     -27.942  10.939   5.645  1.00  0.00           N  \\nATOM    790  CA  SER A 103     -29.454  11.317   5.519  1.00  0.00           C  \\nATOM    791  C   SER A 103     -30.084  10.813   4.385  1.00  0.00           C  \\nATOM    792  O   SER A 103     -29.454   9.931   3.377  1.00  0.00           O  \\nATOM    793  CB  SER A 103     -29.580  12.892   5.897  1.00  0.00           C  \\nATOM    794  OG  SER A 103     -28.320  13.585   5.393  1.00  0.00           O  \\nATOM    795  OXT SER A 103     -31.344  11.317   4.133  1.00  0.00           O  \\nTER     796      SER A 103                                                      \\nHETATM 1593 FE   HEC A 104     -15.216   5.773  12.701  1.00  0.00          FE  \\nHETATM 1594  CHA HEC A 104     -16.980   6.466  15.347  1.00  0.00           C  \\nHETATM 1595  CHB HEC A 104     -16.476   8.293  11.189  1.00  0.00           C  \\nHETATM 1596  CHC HEC A 104     -13.578   5.017   9.929  1.00  0.00           C  \\nHETATM 1597  CHD HEC A 104     -13.956   3.127  14.339  1.00  0.00           C  \\nHETATM 1598  NA  HEC A 104     -16.476   7.096  13.205  1.00  0.00           N  \\nHETATM 1599  C1A HEC A 104     -17.106   7.159  14.465  1.00  0.00           C  \\nHETATM 1600  C2A HEC A 104     -17.862   8.167  14.591  1.00  0.00           C  \\nHETATM 1601  C3A HEC A 104     -17.736   8.797  13.205  1.00  0.00           C  \\nHETATM 1602  C4A HEC A 104     -17.114   8.167  12.323  1.00  0.00           C  \\nHETATM 1603  CMA HEC A 104     -18.303  10.057  12.701  1.00  0.00           C  \\nHETATM 1604  CAA HEC A 104     -18.618   8.167  15.599  1.00  0.00           C  \\nHETATM 1605  CBA HEC A 104     -20.067   8.734  15.473  1.00  0.00           C  \\nHETATM 1606  CGA HEC A 104     -20.823   8.734  16.859  1.00  0.00           C  \\nHETATM 1607  O1A HEC A 104     -21.705   8.041  17.111  1.00  0.00           O  \\nHETATM 1608  O2A HEC A 104     -20.508   9.490  17.741  1.00  0.00           O  \\nHETATM 1609  NB  HEC A 104     -15.090   6.466  11.126  1.00  0.00           N  \\nHETATM 1610  C1B HEC A 104     -15.594   7.348  10.559  1.00  0.00           C  \\nHETATM 1611  C2B HEC A 104     -15.216   7.663   9.299  1.00  0.00           C  \\nHETATM 1612  C3B HEC A 104     -14.334   6.592   8.921  1.00  0.00           C  \\nHETATM 1613  C4B HEC A 104     -14.334   6.025   9.929  1.00  0.00           C  \\nHETATM 1614  CMB HEC A 104     -15.657   8.797   8.543  1.00  0.00           C  \\nHETATM 1615  CAB HEC A 104     -13.578   6.781   7.409  1.00  0.00           C  \\nHETATM 1616  CBB HEC A 104     -12.570   7.663   7.661  1.00  0.00           C  \\nHETATM 1617  NC  HEC A 104     -13.956   4.387  12.197  1.00  0.00           N  \\nHETATM 1618  C1C HEC A 104     -13.452   4.261  10.811  1.00  0.00           C  \\nHETATM 1619  C2C HEC A 104     -12.759   3.001  10.811  1.00  0.00           C  \\nHETATM 1620  C3C HEC A 104     -12.948   2.623  11.945  1.00  0.00           C  \\nHETATM 1621  C4C HEC A 104     -13.704   3.379  12.953  1.00  0.00           C  \\nHETATM 1622  CMC HEC A 104     -11.688   2.497   9.803  1.00  0.00           C  \\nHETATM 1623  CAC HEC A 104     -12.444   1.111  12.197  1.00  0.00           C  \\nHETATM 1624  CBC HEC A 104     -11.184   0.859  12.008  1.00  0.00           C  \\nHETATM 1625  ND  HEC A 104     -15.405   5.143  14.213  1.00  0.00           N  \\nHETATM 1626  C1D HEC A 104     -14.838   4.009  14.843  1.00  0.00           C  \\nHETATM 1627  C2D HEC A 104     -15.216   3.757  15.977  1.00  0.00           C  \\nHETATM 1628  C3D HEC A 104     -16.098   4.765  16.355  1.00  0.00           C  \\nHETATM 1629  C4D HEC A 104     -16.161   5.395  15.347  1.00  0.00           C  \\nHETATM 1630  CMD HEC A 104     -14.838   2.749  16.733  1.00  0.00           C  \\nHETATM 1631  CAD HEC A 104     -16.854   4.513  17.615  1.00  0.00           C  \\nHETATM 1632  CBD HEC A 104     -16.098   4.135  18.623  1.00  0.00           C  \\nHETATM 1633  CGD HEC A 104     -15.216   5.017  19.505  1.00  0.00           C  \\nHETATM 1634  O1D HEC A 104     -14.082   5.773  18.560  1.00  0.00           O  \\nHETATM 1635  O2D HEC A 104     -15.405   5.647  20.639  1.00  0.00           O  \\nHETATM 1679  O   HOH A 105     -16.496   9.304  17.402  1.00  0.00           O  \\nCONECT  104 1615                                                                \\nCONECT  124 1623                                                                \\nCONECT  134 1593                                                                \\nCONECT  356 1635                                                                \\nCONECT  616 1593                                                                \\nCONECT  900 1658                                                                \\nCONECT  920 1666                                                                \\nCONECT  930 1636                                                                \\nCONECT 1152 1678                                                                \\nCONECT 1412 1636                                                                \\nCONECT 1593  134  616 1598 1609                                                 \\nCONECT 1593 1617 1625                                                           \\nCONECT 1594 1599 1629                                                           \\nCONECT 1595 1602 1610                                                           \\nCONECT 1596 1613 1618                                                           \\nCONECT 1597 1621 1626                                                           \\nCONECT 1598 1593 1599 1602                                                      \\nCONECT 1599 1594 1598 1600                                                      \\nCONECT 1600 1599 1601 1604                                                      \\nCONECT 1601 1600 1602 1603                                                      \\nCONECT 1602 1595 1598 1601                                                      \\nCONECT 1603 1601                                                                \\nCONECT 1604 1600 1605                                                           \\nCONECT 1605 1604 1606                                                           \\nCONECT 1606 1605 1607 1608                                                      \\nCONECT 1607 1606                                                                \\nCONECT 1608 1606                                                                \\nCONECT 1609 1593 1610 1613                                                      \\nCONECT 1610 1595 1609 1611                                                      \\nCONECT 1611 1610 1612 1614                                                      \\nCONECT 1612 1611 1613 1615                                                      \\nCONECT 1613 1596 1609 1612                                                      \\nCONECT 1614 1611                                                                \\nCONECT 1615  104 1612 1616                                                      \\nCONECT 1616 1615                                                                \\nCONECT 1617 1593 1618 1621                                                      \\nCONECT 1618 1596 1617 1619                                                      \\nCONECT 1619 1618 1620 1622                                                      \\nCONECT 1620 1619 1621 1623                                                      \\nCONECT 1621 1597 1617 1620                                                      \\nCONECT 1622 1619                                                                \\nCONECT 1623  124 1620 1624                                                      \\nCONECT 1624 1623                                                                \\nCONECT 1625 1593 1626 1629                                                      \\nCONECT 1626 1597 1625 1627                                                      \\nCONECT 1627 1626 1628 1630                                                      \\nCONECT 1628 1627 1629 1631                                                      \\nCONECT 1629 1594 1625 1628                                                      \\nCONECT 1630 1627                                                                \\nCONECT 1631 1628 1632                                                           \\nCONECT 1632 1631 1633                                                           \\nCONECT 1633 1632 1634 1635                                                      \\nCONECT 1634 1633                                                                \\nCONECT 1635  356 1633                                                           \\nCONECT 1636  930 1412 1641 1652                                                 \\nCONECT 1636 1660 1668                                                           \\nCONECT 1637 1642 1672                                                           \\nCONECT 1638 1645 1653                                                           \\nCONECT 1639 1656 1661                                                           \\nCONECT 1640 1664 1669                                                           \\nCONECT 1641 1636 1642 1645                                                      \\nCONECT 1642 1637 1641 1643                                                      \\nCONECT 1643 1642 1644 1647                                                      \\nCONECT 1644 1643 1645 1646                                                      \\nCONECT 1645 1638 1641 1644                                                      \\nCONECT 1646 1644                                                                \\nCONECT 1647 1643 1648                                                           \\nCONECT 1648 1647 1649                                                           \\nCONECT 1649 1648 1650 1651                                                      \\nCONECT 1650 1649                                                                \\nCONECT 1651 1649                                                                \\nCONECT 1652 1636 1653 1656                                                      \\nCONECT 1653 1638 1652 1654                                                      \\nCONECT 1654 1653 1655 1657                                                      \\nCONECT 1655 1654 1656 1658                                                      \\nCONECT 1656 1639 1652 1655                                                      \\nCONECT 1657 1654                                                                \\nCONECT 1658  900 1655 1659                                                      \\nCONECT 1659 1658                                                                \\nCONECT 1660 1636 1661 1664                                                      \\nCONECT 1661 1639 1660 1662                                                      \\nCONECT 1662 1661 1663 1665                                                      \\nCONECT 1663 1662 1664 1666                                                      \\nCONECT 1664 1640 1660 1663                                                      \\nCONECT 1665 1662                                                                \\nCONECT 1666  920 1663 1667                                                      \\nCONECT 1667 1666                                                                \\nCONECT 1668 1636 1669 1672                                                      \\nCONECT 1669 1640 1668 1670                                                      \\nCONECT 1670 1669 1671 1673                                                      \\nCONECT 1671 1670 1672 1674                                                      \\nCONECT 1672 1637 1668 1671                                                      \\nCONECT 1673 1670                                                                \\nCONECT 1674 1671 1675                                                           \\nCONECT 1675 1674 1676                                                           \\nCONECT 1676 1675 1677 1678                                                      \\nCONECT 1677 1676                                                                \\nCONECT 1678 1152 1676                                                           \\nMASTER      287    0    2   10    0    0    9    9 1677    2   98   16          \\nEND                                                                             \\n\",\"pdb\");\n\tviewer_16742555222774558.setBackgroundColor(\"white\");\n\tviewer_16742555222774558.setStyle({\"cartoon\": {\"color\": \"white\"}});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"2\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"5\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"2\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"6\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"2\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"3\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"96\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"3\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"101\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"4\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"8\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"5\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"7\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"5\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"8\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"5\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"5\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"6\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"8\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"6\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"6\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"6\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"95\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"6\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"7\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"11\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"7\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"97\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"7\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"100\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"8\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"8\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"11\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"8\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"12\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"8\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"15\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"8\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"8\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"97\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"9\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"9\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"85\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"9\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"11\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"12\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"13\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"13\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"83\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"13\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"16\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"18\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"32\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"15\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"15\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"18\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"15\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"19\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"27\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"28\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"29\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"80\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"81\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"82\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"18\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"18\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"24\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"18\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"26\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"18\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"28\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"18\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"32\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"19\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"26\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"19\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"32\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"22\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"23\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"24\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"32\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"101\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"21\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"33\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"22\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"31\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"22\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"33\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"23\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"32\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"23\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"33\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"24\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"26\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"25\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"27\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"25\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"30\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"25\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"31\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"26\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"28\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"27\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"30\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"27\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"31\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"28\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"81\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"29\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"32\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"30\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"32\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"30\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"33\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"30\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"43\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"30\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"44\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"30\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"45\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"30\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"46\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"31\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"33\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"31\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"43\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"31\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"44\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"32\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"34\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"32\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"35\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"32\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"43\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"32\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"59\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"32\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"32\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"101\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"32\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"102\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"33\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"38\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"33\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"43\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"33\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"44\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"33\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"103\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"34\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"35\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"37\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"35\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"38\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"35\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"59\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"35\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"61\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"35\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"64\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"35\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"35\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"103\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"36\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"64\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"36\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"37\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"39\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"38\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"57\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"41\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"42\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"52\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"53\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"56\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"59\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"40\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"46\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"41\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"46\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"41\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"48\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"41\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"57\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"42\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"46\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"42\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"53\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"43\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"45\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"43\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"46\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"43\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"52\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"44\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"46\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"46\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"48\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"46\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"52\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"46\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"53\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"47\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"79\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"48\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"50\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"48\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"52\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"48\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"53\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"48\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"54\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"48\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"78\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"48\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"79\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"49\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"51\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"49\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"53\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"50\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"75\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"51\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"56\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"51\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"75\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"51\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"77\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"51\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"80\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"52\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"55\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"52\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"79\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"53\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"55\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"54\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"57\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"54\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"75\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"55\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"74\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"55\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"76\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"56\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"75\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"57\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"59\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"57\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"75\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"59\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"64\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"59\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"66\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"59\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"60\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"62\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"60\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"64\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"60\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"66\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"61\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"64\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"61\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"95\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"61\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"99\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"62\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"64\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"62\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"66\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"62\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"95\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"63\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"68\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"64\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"66\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"64\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"67\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"64\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"68\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"64\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"69\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"64\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"91\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"64\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"64\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"95\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"64\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"97\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"64\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"64\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"99\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"65\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"68\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"65\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"65\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"95\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"66\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"73\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"66\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"74\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"67\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"68\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"70\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"68\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"71\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"68\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"74\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"68\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"84\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"68\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"85\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"68\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"86\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"68\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"90\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"68\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"91\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"68\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"92\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"68\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"68\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"95\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"68\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"69\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"72\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"69\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"85\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"70\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"72\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"70\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"76\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"70\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"85\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"71\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"74\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"71\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"76\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"72\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"76\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"72\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"84\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"73\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"75\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"74\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"76\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"75\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"77\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"78\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"81\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"79\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"81\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"80\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"82\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"81\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"83\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"82\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"85\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"83\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"85\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"84\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"90\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"85\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"87\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"85\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"89\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"85\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"90\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"85\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"91\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"85\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"86\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"90\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"87\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"89\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"87\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"92\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"87\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"88\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"92\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"88\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"89\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"89\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"90\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"91\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"91\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"95\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"92\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"92\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"95\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"93\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"97\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"96\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"97\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"95\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"96\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"96\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"101\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"100\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"101\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"102\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"103\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555222774558.zoomTo();\nviewer_16742555222774558.render();\n});\n</script>",
      "text/html": [
       "<div id=\"3dmolviewer_16742555222774558\"  style=\"position: relative; width: 640px; height: 480px\">\n",
       "        <p id=\"3dmolwarning_16742555222774558\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n",
       "        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n",
       "        </div>\n",
       "<script>\n",
       "\n",
       "var loadScriptAsync = function(uri){\n",
       "  return new Promise((resolve, reject) => {\n",
       "    //this is to ignore the existence of requirejs amd\n",
       "    var savedexports, savedmodule;\n",
       "    if (typeof exports !== 'undefined') savedexports = exports;\n",
       "    else exports = {}\n",
       "    if (typeof module !== 'undefined') savedmodule = module;\n",
       "    else module = {}\n",
       "\n",
       "    var tag = document.createElement('script');\n",
       "    tag.src = uri;\n",
       "    tag.async = true;\n",
       "    tag.onload = () => {\n",
       "        exports = savedexports;\n",
       "        module = savedmodule;\n",
       "        resolve();\n",
       "    };\n",
       "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
       "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
       "});\n",
       "};\n",
       "\n",
       "if(typeof $3Dmolpromise === 'undefined') {\n",
       "$3Dmolpromise = null;\n",
       "  $3Dmolpromise = loadScriptAsync('https://3dmol.org/build/3Dmol.js');\n",
       "}\n",
       "\n",
       "var viewer_16742555222774558 = null;\n",
       "var warn = document.getElementById(\"3dmolwarning_16742555222774558\");\n",
       "if(warn) {\n",
       "    warn.parentNode.removeChild(warn);\n",
       "}\n",
       "$3Dmolpromise.then(function() {\n",
       "viewer_16742555222774558 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_16742555222774558\"),{backgroundColor:\"white\"});\n",
       "viewer_16742555222774558.zoomTo();\n",
       "\tviewer_16742555222774558.addModel(\"HEADER    ELECTRON TRANSPORT                      01-AUG-76   1CYC              \\nTITLE     THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C AT 2.3     \\nTITLE    2 ANGSTROMS RESOLUTION. II. STRUCTURE AND FUNCTION                     \\nCAVEAT     1CYC    THR A 40 HAS WRONG CHIRALITY AT ATOM CB THR A 78 HAS WRONG   \\nCAVEAT   2 1CYC    CHIRALITY AT ATOM CB THR B 40 HAS WRONG CHIRALITY AT ATOM    \\nCAVEAT   3 1CYC    CB THR B 78 HAS WRONG CHIRALITY AT ATOM CB                   \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: FERROCYTOCHROME C;                                         \\nCOMPND   3 CHAIN: A, B;                                                         \\nCOMPND   4 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 ORGANISM_SCIENTIFIC: KATSUWONUS PELAMIS;                             \\nSOURCE   3 ORGANISM_COMMON: SKIPJACK TUNA;                                      \\nSOURCE   4 ORGANISM_TAXID: 8226                                                 \\nKEYWDS    ELECTRON TRANSPORT                                                    \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    N.TANAKA,T.YAMANE,T.TSUKIHARA,T.ASHIDA,M.KAKUDO                       \\nREVDAT  17   03-MAR-21 1CYC    1       CAVEAT COMPND REMARK HET                 \\nREVDAT  17 2                   1       HETNAM HETSYN FORMUL LINK                \\nREVDAT  17 3                   1       ATOM                                     \\nREVDAT  16   29-NOV-17 1CYC    1       HELIX                                    \\nREVDAT  15   24-FEB-09 1CYC    1       VERSN                                    \\nREVDAT  14   01-APR-03 1CYC    1       JRNL                                     \\nREVDAT  13   27-JAN-84 1CYC    1       REMARK                                   \\nREVDAT  12   30-SEP-83 1CYC    1       REVDAT                                   \\nREVDAT  11   19-APR-83 1CYC    1       COMPND SOURCE REMARK                     \\nREVDAT  10   15-JAN-82 1CYC    1       REMARK                                   \\nREVDAT   9   06-JAN-82 1CYC    3       SCALE                                    \\nREVDAT   8   31-DEC-80 1CYC    1       REMARK                                   \\nREVDAT   7   07-APR-80 1CYC    3       SOURCE SEQRES ATOM                       \\nREVDAT   6   23-APR-79 1CYC    3       ATOM                                     \\nREVDAT   5   01-NOV-77 1CYC    1       AUTHOR REMARK FORMUL                     \\nREVDAT   4   09-SEP-77 1CYC    3       ATOM                                     \\nREVDAT   3   13-JUN-77 1CYC    1       HET                                      \\nREVDAT   2   25-OCT-76 1CYC    2       CONECT                                   \\nREVDAT   1   06-OCT-76 1CYC    0                                                \\nJRNL        AUTH   N.TANAKA,T.YAMANE,T.TSUKIHARA,T.ASHIDA,M.KAKUDO              \\nJRNL        TITL   THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C   \\nJRNL        TITL 2 AT 2.3 A RESOLUTION. II. STRUCTURE AND FUNCTION.             \\nJRNL        REF    J.BIOCHEM.(TOKYO)             V.  77   147 1975              \\nJRNL        REFN                   ISSN 0021-924X                               \\nJRNL        PMID   166072                                                       \\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   T.TSUKIHARA,T.YAMANE,N.TANAKA,T.ASHIDA,M.KAKUDO              \\nREMARK   1  TITL   OXIDATION OF A FERROCYTOCHROME C IN THE CRYSTALLINE          \\nREMARK   1  TITL 2 STATE-STRUCTURAL CHANGE AND ANION BINDING                    \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  73  1163 1973              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 2                                                          \\nREMARK   1  AUTH   T.ASHIDA,N.TANAKA,T.YAMANE,T.TSUKIHARA,M.KAKUDO              \\nREMARK   1  TITL   THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C   \\nREMARK   1  TITL 2 AT 2.3 ANGSTROMS RESOLUTION                                  \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  73   463 1973              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 3                                                          \\nREMARK   1  AUTH   T.ASHIDA,T.UEKI,T.TSUKIHARA,A.SUGIHARA,T.TAKANO,M.KAKUDO     \\nREMARK   1  TITL   THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C   \\nREMARK   1  TITL 2 AT 4 ANGSTROMS RESOLUTION                                    \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  70   913 1971              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 4                                                          \\nREMARK   1  AUTH   T.NAKAYAMA,K.TITANI,K.NARITA                                 \\nREMARK   1  TITL   THE AMINO ACID SEQUENCE OF CYTOCHROME C FROM BONITO          \\nREMARK   1  TITL 2 (KATSUWONUS PELAMIS, LINNAEUS)                               \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  70   311 1971              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 5                                                          \\nREMARK   1  EDIT   R.J.FELDMANN                                                 \\nREMARK   1  REF    ATLAS OF MACROMOLECULAR                120 1976              \\nREMARK   1  REF  2 STRUCTURE ON MICROFICHE                                      \\nREMARK   1  PUBL   TRACOR JITCO INC.,ROCKVILLE,MD.                              \\nREMARK   1  REFN                                                                \\nREMARK   1 REFERENCE 6                                                          \\nREMARK   1  EDIT   M.O.DAYHOFF                                                  \\nREMARK   1  REF    ATLAS OF PROTEIN SEQUENCE     V.   5    39 1976              \\nREMARK   1  REF  2 AND STRUCTURE,SUPPLEMENT 2                                   \\nREMARK   1  PUBL   NATIONAL BIOMEDICAL RESEARCH FOUNDATION, SILVER SPRING,MD.   \\nREMARK   1  REFN                   ISSN 0-912466-05-7                           \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    2.30 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : NULL                                                 \\nREMARK   3   AUTHORS     : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 2.30                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : NULL                           \\nREMARK   3   DATA CUTOFF            (SIGMA(F)) : NULL                           \\nREMARK   3   DATA CUTOFF HIGH         (ABS(F)) : NULL                           \\nREMARK   3   DATA CUTOFF LOW          (ABS(F)) : NULL                           \\nREMARK   3   COMPLETENESS (WORKING+TEST)   (%) : NULL                           \\nREMARK   3   NUMBER OF REFLECTIONS             : NULL                           \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   CROSS-VALIDATION METHOD          : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SELECTION  : NULL                            \\nREMARK   3   R VALUE            (WORKING SET) : NULL                            \\nREMARK   3   FREE R VALUE                     : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : NULL                            \\nREMARK   3   FREE R VALUE TEST SET COUNT      : NULL                            \\nREMARK   3   ESTIMATED ERROR OF FREE R VALUE  : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  FIT IN THE HIGHEST RESOLUTION BIN.                                  \\nREMARK   3   TOTAL NUMBER OF BINS USED           : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE HIGH       (A) : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE LOW        (A) : NULL                         \\nREMARK   3   BIN COMPLETENESS (WORKING+TEST) (%) : NULL                         \\nREMARK   3   REFLECTIONS IN BIN    (WORKING SET) : NULL                         \\nREMARK   3   BIN R VALUE           (WORKING SET) : NULL                         \\nREMARK   3   BIN FREE R VALUE                    : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET SIZE  (%) : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET COUNT     : NULL                         \\nREMARK   3   ESTIMATED ERROR OF BIN FREE R VALUE : NULL                         \\nREMARK   3                                                                      \\nREMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \\nREMARK   3   PROTEIN ATOMS            : 1590                                    \\nREMARK   3   NUCLEIC ACID ATOMS       : 0                                       \\nREMARK   3   HETEROGEN ATOMS          : 86                                      \\nREMARK   3   SOLVENT ATOMS            : 1                                       \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : NULL                           \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : NULL                           \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : NULL                                                 \\nREMARK   3    B22 (A**2) : NULL                                                 \\nREMARK   3    B33 (A**2) : NULL                                                 \\nREMARK   3    B12 (A**2) : NULL                                                 \\nREMARK   3    B13 (A**2) : NULL                                                 \\nREMARK   3    B23 (A**2) : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  ESTIMATED COORDINATE ERROR.                                         \\nREMARK   3   ESD FROM LUZZATI PLOT        (A) : NULL                            \\nREMARK   3   ESD FROM SIGMAA              (A) : NULL                            \\nREMARK   3   LOW RESOLUTION CUTOFF        (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  CROSS-VALIDATED ESTIMATED COORDINATE ERROR.                         \\nREMARK   3   ESD FROM C-V LUZZATI PLOT    (A) : NULL                            \\nREMARK   3   ESD FROM C-V SIGMAA          (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  RMS DEVIATIONS FROM IDEAL VALUES.                                   \\nREMARK   3   BOND LENGTHS                 (A) : NULL                            \\nREMARK   3   BOND ANGLES            (DEGREES) : NULL                            \\nREMARK   3   DIHEDRAL ANGLES        (DEGREES) : NULL                            \\nREMARK   3   IMPROPER ANGLES        (DEGREES) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL MODEL : NULL                                      \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL FACTOR RESTRAINTS.    RMS    SIGMA                \\nREMARK   3   MAIN-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   MAIN-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  NCS MODEL : NULL                                                    \\nREMARK   3                                                                      \\nREMARK   3  NCS RESTRAINTS.                         RMS   SIGMA/WEIGHT          \\nREMARK   3   GROUP  1  POSITIONAL            (A) : NULL  ; NULL                 \\nREMARK   3   GROUP  1  B-FACTOR           (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  PARAMETER FILE  1  : NULL                                           \\nREMARK   3  TOPOLOGY FILE  1   : NULL                                           \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS: NULL                                      \\nREMARK   4                                                                      \\nREMARK   4 1CYC COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY BNL.                                \\nREMARK 100 THE DEPOSITION ID IS D_1000172595.                                   \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : NULL                               \\nREMARK 200  TEMPERATURE           (KELVIN) : NULL                               \\nREMARK 200  PH                             : NULL                               \\nREMARK 200  NUMBER OF CRYSTALS USED        : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : NULL                               \\nREMARK 200  RADIATION SOURCE               : NULL                               \\nREMARK 200  BEAMLINE                       : NULL                               \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : NULL                               \\nREMARK 200  WAVELENGTH OR RANGE        (A) : NULL                               \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : NULL                               \\nREMARK 200  DETECTOR MANUFACTURER          : NULL                               \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : NULL                               \\nREMARK 200  DATA SCALING SOFTWARE          : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : NULL                               \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : NULL                               \\nREMARK 200  RESOLUTION RANGE LOW       (A) : NULL                               \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY                : NULL                               \\nREMARK 200  R MERGE                    (I) : NULL                               \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : NULL                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : NULL                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : NULL                               \\nREMARK 200  R MERGE FOR SHELL          (I) : NULL                               \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: NULL                                           \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: NULL                         \\nREMARK 200 SOFTWARE USED: NULL                                                  \\nREMARK 200 STARTING MODEL: NULL                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: NULL                                                         \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): 39.18                                     \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 2.02                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: NULL                                     \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: P 21 21 21                       \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -X+1/2,-Y,Z+1/2                                         \\nREMARK 290       3555   -X,Y+1/2,-Z+1/2                                         \\nREMARK 290       4555   X+1/2,-Y+1/2,-Z                                         \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -1.000000  0.000000  0.000000       28.84000            \\nREMARK 290   SMTRY2   2  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000       18.91500            \\nREMARK 290   SMTRY1   3 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   3  0.000000  1.000000  0.000000       42.29000            \\nREMARK 290   SMTRY3   3  0.000000  0.000000 -1.000000       18.91500            \\nREMARK 290   SMTRY1   4  1.000000  0.000000  0.000000       28.84000            \\nREMARK 290   SMTRY2   4  0.000000 -1.000000  0.000000       42.29000            \\nREMARK 290   SMTRY3   4  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1, 2                                                    \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: MONOMERIC                         \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 2                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: MONOMERIC                         \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: B                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 470                                                                      \\nREMARK 470 MISSING ATOM                                                         \\nREMARK 470 THE FOLLOWING RESIDUES HAVE MISSING ATOMS (M=MODEL NUMBER;           \\nREMARK 470 RES=RESIDUE NAME; C=CHAIN IDENTIFIER; SSEQ=SEQUENCE NUMBER;          \\nREMARK 470 I=INSERTION CODE):                                                   \\nREMARK 470   M RES CSSEQI  ATOMS                                                \\nREMARK 470     ASN A  22    CB   CG   OD1  ND2                                  \\nREMARK 470     LYS A  39    CE   NZ                                             \\nREMARK 470     ASN B  22    CB   CG   OD1  ND2                                  \\nREMARK 470     LYS B  39    CE   NZ                                             \\nREMARK 620                                                                      \\nREMARK 620 METAL COORDINATION                                                   \\nREMARK 620 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 620 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE):                             \\nREMARK 620                                                                      \\nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \\nREMARK 620                             HEC A 104  FE                            \\nREMARK 620 N RES CSSEQI ATOM                                                    \\nREMARK 620 1 HIS A  18   NE2                                                    \\nREMARK 620 2 HEC A 104   NA   87.0                                              \\nREMARK 620 3 HEC A 104   NB   87.8  90.6                                        \\nREMARK 620 4 HEC A 104   NC   92.0 178.7  90.1                                  \\nREMARK 620 5 HEC A 104   ND   91.1  86.9 177.4  92.3                            \\nREMARK 620 6 MET A  80   SD  170.9 102.1  93.0  79.0  88.6                      \\nREMARK 620 N                    1     2     3     4     5                       \\nREMARK 620                                                                      \\nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \\nREMARK 620                             HEC B 104  FE                            \\nREMARK 620 N RES CSSEQI ATOM                                                    \\nREMARK 620 1 HIS B  18   NE2                                                    \\nREMARK 620 2 HEC B 104   NA   87.0                                              \\nREMARK 620 3 HEC B 104   NB   87.8  90.6                                        \\nREMARK 620 4 HEC B 104   NC   92.0 178.7  90.1                                  \\nREMARK 620 5 HEC B 104   ND   91.1  86.9 177.4  92.3                            \\nREMARK 620 6 MET B  80   SD  170.9 102.1  93.0  79.0  88.6                      \\nREMARK 620 N                    1     2     3     4     5                       \\nREMARK 800                                                                      \\nREMARK 800 SITE                                                                 \\nREMARK 800 SITE_IDENTIFIER: AC1                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE HEC A 104                 \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: AC2                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE HEC B 104                 \\nDBREF  1CYC A    1   103  UNP    P00025   CYC_KATPE        1    103             \\nDBREF  1CYC B    1   103  UNP    P00025   CYC_KATPE        1    103             \\nSEQRES   1 A  103  GLY ASP VAL ALA LYS GLY LYS LYS THR PHE VAL GLN LYS          \\nSEQRES   2 A  103  CYS ALA GLN CYS HIS THR VAL GLU ASN GLY GLY LYS HIS          \\nSEQRES   3 A  103  LYS VAL GLY PRO ASN LEU TRP GLY LEU PHE GLY ARG LYS          \\nSEQRES   4 A  103  THR GLY GLN ALA GLU GLY TYR SER TYR THR ASP ALA ASN          \\nSEQRES   5 A  103  LYS SER LYS GLY ILE VAL TRP ASN GLU ASN THR LEU MET          \\nSEQRES   6 A  103  GLU TYR LEU GLU ASN PRO LYS LYS TYR ILE PRO GLY THR          \\nSEQRES   7 A  103  LYS MET ILE PHE ALA GLY ILE LYS LYS LYS GLY GLU ARG          \\nSEQRES   8 A  103  GLN ASP LEU VAL ALA TYR LEU LYS SER ALA THR SER              \\nSEQRES   1 B  103  GLY ASP VAL ALA LYS GLY LYS LYS THR PHE VAL GLN LYS          \\nSEQRES   2 B  103  CYS ALA GLN CYS HIS THR VAL GLU ASN GLY GLY LYS HIS          \\nSEQRES   3 B  103  LYS VAL GLY PRO ASN LEU TRP GLY LEU PHE GLY ARG LYS          \\nSEQRES   4 B  103  THR GLY GLN ALA GLU GLY TYR SER TYR THR ASP ALA ASN          \\nSEQRES   5 B  103  LYS SER LYS GLY ILE VAL TRP ASN GLU ASN THR LEU MET          \\nSEQRES   6 B  103  GLU TYR LEU GLU ASN PRO LYS LYS TYR ILE PRO GLY THR          \\nSEQRES   7 B  103  LYS MET ILE PHE ALA GLY ILE LYS LYS LYS GLY GLU ARG          \\nSEQRES   8 B  103  GLN ASP LEU VAL ALA TYR LEU LYS SER ALA THR SER              \\nHET    HEC  A 104      43                                                       \\nHET    HEC  B 104      43                                                       \\nHETNAM     HEC HEME C                                                           \\nFORMUL   3  HEC    2(C34 H34 FE N4 O4)                                          \\nFORMUL   5  HOH   *(H2 O)                                                       \\nHELIX    1  H1 GLY A    1  VAL A   11  1                                  11    \\nHELIX    2  H4 CYS A   14  CYS A   17  514 AND 17 BOUND TO HEME GROUP      4    \\nHELIX    3  H5 THR A   49  LYS A   55  5LOOSE FROM 49-53,3/10 53-55        7    \\nHELIX    4  H2 ASN A   60  GLU A   69  1                                  10    \\nHELIX    5  H3 GLU A   90  SER A  103  1                                  14    \\nHELIX    6  H6 GLY B    1  VAL B   11  1                                  11    \\nHELIX    7  H7 CYS B   14  CYS B   17  514 AND 17 BOUND TO HEME GROUP      4    \\nHELIX    8  H8 THR B   49  LYS B   55  5LOOSE FROM 49-53,3/10 53-55        7    \\nHELIX    9  H9 ASN B   60  GLU B   69  1                                  10    \\nHELIX   10 H10 GLU B   90  SER B  103  1                                  14    \\nLINK         SG  CYS A  14                 CAB HEC A 104     1555   1555  1.48  \\nLINK         SG  CYS A  17                 CAC HEC A 104     1555   1555  1.86  \\nLINK         CE2 TYR A  48                 O2D HEC A 104     1555   1555  2.02  \\nLINK         SG  CYS B  14                 CAB HEC B 104     1555   1555  1.48  \\nLINK         SG  CYS B  17                 CAC HEC B 104     1555   1555  1.86  \\nLINK         CE2 TYR B  48                 O2D HEC B 104     1555   1555  2.02  \\nLINK         NE2 HIS A  18                FE   HEC A 104     1555   1555  2.60  \\nLINK         SD  MET A  80                FE   HEC A 104     1555   1555  2.49  \\nLINK         NE2 HIS B  18                FE   HEC B 104     1555   1555  2.60  \\nLINK         SD  MET B  80                FE   HEC B 104     1555   1555  2.49  \\nSITE     1 AC1 17 LYS A  13  CYS A  14  CYS A  17  HIS A  18                    \\nSITE     2 AC1 17 LEU A  32  LEU A  35  ARG A  38  THR A  40                    \\nSITE     3 AC1 17 TYR A  46  TYR A  48  ASN A  52  TRP A  59                    \\nSITE     4 AC1 17 THR A  78  LYS A  79  MET A  80  PHE A  82                    \\nSITE     5 AC1 17 HOH A 105                                                     \\nSITE     1 AC2 16 LYS B  13  CYS B  14  CYS B  17  HIS B  18                    \\nSITE     2 AC2 16 LEU B  32  LEU B  35  ARG B  38  THR B  40                    \\nSITE     3 AC2 16 TYR B  46  TYR B  48  ASN B  52  TRP B  59                    \\nSITE     4 AC2 16 THR B  78  LYS B  79  MET B  80  PHE B  82                    \\nCRYST1   57.680   84.580   37.830  90.00  90.00  90.00 P 21 21 21    8          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.017337  0.000000  0.000000        0.00000                         \\nSCALE2      0.000000  0.011823  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.026434        0.00000                         \\nMTRIX1   1  1.000000  0.000000  0.000000       28.84000    1                    \\nMTRIX2   1  0.000000  1.000000  0.000000        0.00000    1                    \\nMTRIX3   1  0.000000  0.000000  1.000000        0.00000    1                    \\nATOM      1  N   GLY A   1     -21.138  13.774  -7.711  1.00  0.00           N  \\nATOM      2  CA  GLY A   1     -19.878  14.593  -7.963  1.00  0.00           C  \\nATOM      3  C   GLY A   1     -18.996  14.530  -6.703  1.00  0.00           C  \\nATOM      4  O   GLY A   1     -18.114  13.711  -6.640  1.00  0.00           O  \\nATOM      5  N   ASP A   2     -18.996  15.412  -5.821  1.00  0.00           N  \\nATOM      6  CA  ASP A   2     -17.610  15.097  -5.191  1.00  0.00           C  \\nATOM      7  C   ASP A   2     -17.862  14.152  -3.931  1.00  0.00           C  \\nATOM      8  O   ASP A   2     -17.106  13.333  -3.553  1.00  0.00           O  \\nATOM      9  CB  ASP A   2     -17.169  16.546  -4.435  1.00  0.00           C  \\nATOM     10  CG  ASP A   2     -17.988  17.365  -3.427  1.00  0.00           C  \\nATOM     11  OD1 ASP A   2     -19.500  17.365  -3.994  1.00  0.00           O  \\nATOM     12  OD2 ASP A   2     -17.484  18.373  -2.293  1.00  0.00           O  \\nATOM     13  N   VAL A   3     -19.059  14.089  -3.679  1.00  0.00           N  \\nATOM     14  CA  VAL A   3     -19.815  12.955  -2.860  1.00  0.00           C  \\nATOM     15  C   VAL A   3     -19.374  11.506  -3.805  1.00  0.00           C  \\nATOM     16  O   VAL A   3     -18.870  10.624  -3.427  1.00  0.00           O  \\nATOM     17  CB  VAL A   3     -22.587  13.270  -3.175  1.00  0.00           C  \\nATOM     18  CG1 VAL A   3     -21.957  11.947  -3.427  1.00  0.00           C  \\nATOM     19  CG2 VAL A   3     -21.705  13.963  -2.167  1.00  0.00           C  \\nATOM     20  N   ALA A   4     -19.437  11.380  -5.002  1.00  0.00           N  \\nATOM     21  CA  ALA A   4     -18.681  10.120  -5.569  1.00  0.00           C  \\nATOM     22  C   ALA A   4     -17.358   9.805  -4.939  1.00  0.00           C  \\nATOM     23  O   ALA A   4     -17.295   8.923  -4.183  1.00  0.00           O  \\nATOM     24  CB  ALA A   4     -18.429  10.435  -7.144  1.00  0.00           C  \\nATOM     25  N   LYS A   5     -16.476  10.687  -5.191  1.00  0.00           N  \\nATOM     26  CA  LYS A   5     -15.216  10.498  -4.372  1.00  0.00           C  \\nATOM     27  C   LYS A   5     -15.279  10.246  -2.797  1.00  0.00           C  \\nATOM     28  O   LYS A   5     -14.838   9.238  -2.293  1.00  0.00           O  \\nATOM     29  CB  LYS A   5     -14.208  11.695  -4.246  1.00  0.00           C  \\nATOM     30  CG  LYS A   5     -12.948  11.758  -3.553  1.00  0.00           C  \\nATOM     31  CD  LYS A   5     -11.814  13.081  -3.679  1.00  0.00           C  \\nATOM     32  CE  LYS A   5     -10.806  13.144  -2.545  1.00  0.00           C  \\nATOM     33  NZ  LYS A   5      -9.735  14.404  -2.293  1.00  0.00           N  \\nATOM     34  N   GLY A   6     -16.161  10.750  -1.915  1.00  0.00           N  \\nATOM     35  CA  GLY A   6     -16.350  10.435  -0.403  1.00  0.00           C  \\nATOM     36  C   GLY A   6     -16.539   8.923  -0.655  1.00  0.00           C  \\nATOM     37  O   GLY A   6     -15.909   8.041  -0.025  1.00  0.00           O  \\nATOM     38  N   LYS A   7     -17.421   8.419  -1.411  1.00  0.00           N  \\nATOM     39  CA  LYS A   7     -17.799   6.844  -1.033  1.00  0.00           C  \\nATOM     40  C   LYS A   7     -16.476   6.151  -1.474  1.00  0.00           C  \\nATOM     41  O   LYS A   7     -15.972   4.954  -0.970  1.00  0.00           O  \\nATOM     42  CB  LYS A   7     -18.870   6.403  -2.293  1.00  0.00           C  \\nATOM     43  CG  LYS A   7     -18.870   5.017  -2.734  1.00  0.00           C  \\nATOM     44  CD  LYS A   7     -20.193   4.765  -3.427  1.00  0.00           C  \\nATOM     45  CE  LYS A   7     -20.193   3.190  -4.057  1.00  0.00           C  \\nATOM     46  NZ  LYS A   7     -21.516   2.623  -4.309  1.00  0.00           N  \\nATOM     47  N   LYS A   8     -15.846   6.403  -2.545  1.00  0.00           N  \\nATOM     48  CA  LYS A   8     -14.460   5.899  -2.923  1.00  0.00           C  \\nATOM     49  C   LYS A   8     -13.452   5.899  -1.663  1.00  0.00           C  \\nATOM     50  O   LYS A   8     -13.011   5.017  -1.348  1.00  0.00           O  \\nATOM     51  CB  LYS A   8     -13.704   6.844  -3.931  1.00  0.00           C  \\nATOM     52  CG  LYS A   8     -12.318   6.529  -4.183  1.00  0.00           C  \\nATOM     53  CD  LYS A   8     -11.562   7.537  -5.065  1.00  0.00           C  \\nATOM     54  CE  LYS A   8     -10.176   7.222  -5.821  1.00  0.00           C  \\nATOM     55  NZ  LYS A   8      -9.357   8.419  -6.325  1.00  0.00           N  \\nATOM     56  N   THR A   9     -13.452   6.970  -1.159  1.00  0.00           N  \\nATOM     57  CA  THR A   9     -12.633   7.033   0.227  1.00  0.00           C  \\nATOM     58  C   THR A   9     -13.515   6.277   1.298  1.00  0.00           C  \\nATOM     59  O   THR A   9     -12.948   5.773   2.369  1.00  0.00           O  \\nATOM     60  CB  THR A   9     -12.381   8.419   0.920  1.00  0.00           C  \\nATOM     61  OG1 THR A   9     -11.058   8.923   0.542  1.00  0.00           O  \\nATOM     62  CG2 THR A   9     -12.444   8.734   2.558  1.00  0.00           C  \\nATOM     63  N   PHE A  10     -14.775   6.403   1.235  1.00  0.00           N  \\nATOM     64  CA  PHE A  10     -15.531   5.647   2.117  1.00  0.00           C  \\nATOM     65  C   PHE A  10     -15.216   4.135   1.739  1.00  0.00           C  \\nATOM     66  O   PHE A  10     -14.460   3.757   2.495  1.00  0.00           O  \\nATOM     67  CB  PHE A  10     -17.106   5.647   1.991  1.00  0.00           C  \\nATOM     68  CG  PHE A  10     -17.736   4.891   3.251  1.00  0.00           C  \\nATOM     69  CD1 PHE A  10     -18.492   3.757   2.747  1.00  0.00           C  \\nATOM     70  CD2 PHE A  10     -17.862   5.143   4.133  1.00  0.00           C  \\nATOM     71  CE1 PHE A  10     -18.996   3.001   3.818  1.00  0.00           C  \\nATOM     72  CE2 PHE A  10     -18.366   4.513   4.889  1.00  0.00           C  \\nATOM     73  CZ  PHE A  10     -18.996   3.379   5.015  1.00  0.00           C  \\nATOM     74  N   VAL A  11     -15.216   3.379   0.479  1.00  0.00           N  \\nATOM     75  CA  VAL A  11     -14.964   2.056   0.605  1.00  0.00           C  \\nATOM     76  C   VAL A  11     -13.326   1.867   1.109  1.00  0.00           C  \\nATOM     77  O   VAL A  11     -12.948   0.985   1.865  1.00  0.00           O  \\nATOM     78  CB  VAL A  11     -15.153   1.237  -0.529  1.00  0.00           C  \\nATOM     79  CG1 VAL A  11     -16.098   1.930  -1.537  1.00  0.00           C  \\nATOM     80  CG2 VAL A  11     -13.956   1.111  -1.285  1.00  0.00           C  \\nATOM     81  N   GLN A  12     -12.570   2.749   0.794  1.00  0.00           N  \\nATOM     82  CA  GLN A  12     -10.995   2.749   1.235  1.00  0.00           C  \\nATOM     83  C   GLN A  12     -10.680   2.938   2.873  1.00  0.00           C  \\nATOM     84  O   GLN A  12     -10.176   1.930   3.377  1.00  0.00           O  \\nATOM     85  CB  GLN A  12     -10.428   3.820   0.479  1.00  0.00           C  \\nATOM     86  CG  GLN A  12      -9.861   3.253  -0.718  1.00  0.00           C  \\nATOM     87  CD  GLN A  12      -9.168   4.135  -2.293  1.00  0.00           C  \\nATOM     88  OE1 GLN A  12      -7.593   3.820  -2.041  1.00  0.00           O  \\nATOM     89  NE2 GLN A  12      -9.672   5.143  -2.671  1.00  0.00           N  \\nATOM     90  N   LYS A  13     -10.995   3.757   3.629  1.00  0.00           N  \\nATOM     91  CA  LYS A  13     -10.869   3.757   5.015  1.00  0.00           C  \\nATOM     92  C   LYS A  13     -12.066   3.694   5.960  1.00  0.00           C  \\nATOM     93  O   LYS A  13     -12.003   3.127   6.779  1.00  0.00           O  \\nATOM     94  CB  LYS A  13     -10.239   5.017   5.141  1.00  0.00           C  \\nATOM     95  CG  LYS A  13      -9.672   5.521   3.881  1.00  0.00           C  \\nATOM     96  CD  LYS A  13      -8.286   6.277   4.007  1.00  0.00           C  \\nATOM     97  CE  LYS A  13      -8.160   6.907   2.621  1.00  0.00           C  \\nATOM     98  NZ  LYS A  13      -6.837   7.789   2.243  1.00  0.00           N  \\nATOM     99  N   CYS A  14     -13.137   4.072   5.519  1.00  0.00           N  \\nATOM    100  CA  CYS A  14     -14.460   4.072   6.464  1.00  0.00           C  \\nATOM    101  C   CYS A  14     -15.216   2.812   5.897  1.00  0.00           C  \\nATOM    102  O   CYS A  14     -15.342   1.804   6.716  1.00  0.00           O  \\nATOM    103  CB  CYS A  14     -15.468   5.332   5.897  1.00  0.00           C  \\nATOM    104  SG  CYS A  14     -14.712   6.844   6.464  1.00  0.00           S  \\nATOM    105  N   ALA A  15     -15.720   2.623   4.637  1.00  0.00           N  \\nATOM    106  CA  ALA A  15     -16.161   1.237   4.322  1.00  0.00           C  \\nATOM    107  C   ALA A  15     -16.035  -0.086   5.015  1.00  0.00           C  \\nATOM    108  O   ALA A  15     -17.043  -0.590   5.393  1.00  0.00           O  \\nATOM    109  CB  ALA A  15     -15.405   0.985   3.062  1.00  0.00           C  \\nATOM    110  N   GLN A  16     -14.964  -0.653   4.889  1.00  0.00           N  \\nATOM    111  CA  GLN A  16     -14.901  -2.039   5.897  1.00  0.00           C  \\nATOM    112  C   GLN A  16     -15.090  -1.787   7.283  1.00  0.00           C  \\nATOM    113  O   GLN A  16     -14.838  -2.669   8.291  1.00  0.00           O  \\nATOM    114  CB  GLN A  16     -13.578  -2.039   5.519  1.00  0.00           C  \\nATOM    115  CG  GLN A  16     -12.318  -1.724   6.275  1.00  0.00           C  \\nATOM    116  CD  GLN A  16     -11.625  -0.842   5.015  1.00  0.00           C  \\nATOM    117  OE1 GLN A  16     -10.365  -0.149   5.330  1.00  0.00           O  \\nATOM    118  NE2 GLN A  16     -12.444  -0.464   4.385  1.00  0.00           N  \\nATOM    119  N   CYS A  17     -15.216  -0.653   7.787  1.00  0.00           N  \\nATOM    120  CA  CYS A  17     -15.405  -0.905   9.299  1.00  0.00           C  \\nATOM    121  C   CYS A  17     -16.917  -0.401   9.173  1.00  0.00           C  \\nATOM    122  O   CYS A  17     -17.736  -1.283   9.173  1.00  0.00           O  \\nATOM    123  CB  CYS A  17     -14.460   0.355  10.055  1.00  0.00           C  \\nATOM    124  SG  CYS A  17     -13.326  -0.338  11.441  1.00  0.00           S  \\nATOM    125  N   HIS A  18     -17.232   0.733   9.551  1.00  0.00           N  \\nATOM    126  CA  HIS A  18     -18.870   0.985   9.803  1.00  0.00           C  \\nATOM    127  C   HIS A  18     -19.878   0.229   8.795  1.00  0.00           C  \\nATOM    128  O   HIS A  18     -19.626  -0.023   7.472  1.00  0.00           O  \\nATOM    129  CB  HIS A  18     -19.185   2.245   9.173  1.00  0.00           C  \\nATOM    130  CG  HIS A  18     -18.681   3.127  10.181  1.00  0.00           C  \\nATOM    131  ND1 HIS A  18     -18.618   3.253  11.567  1.00  0.00           N  \\nATOM    132  CD2 HIS A  18     -17.736   4.072  10.559  1.00  0.00           C  \\nATOM    133  CE1 HIS A  18     -18.051   3.694  12.827  1.00  0.00           C  \\nATOM    134  NE2 HIS A  18     -17.232   4.387  11.819  1.00  0.00           N  \\nATOM    135  N   THR A  19     -21.012   0.103   9.299  1.00  0.00           N  \\nATOM    136  CA  THR A  19     -22.272  -0.401   8.417  1.00  0.00           C  \\nATOM    137  C   THR A  19     -22.902   0.922   7.661  1.00  0.00           C  \\nATOM    138  O   THR A  19     -22.272   1.867   7.157  1.00  0.00           O  \\nATOM    139  CB  THR A  19     -23.280  -1.283   8.795  1.00  0.00           C  \\nATOM    140  OG1 THR A  19     -23.532  -0.905   9.740  1.00  0.00           O  \\nATOM    141  CG2 THR A  19     -22.776  -2.669   8.543  1.00  0.00           C  \\nATOM    142  N   VAL A  20     -24.036   1.111   7.472  1.00  0.00           N  \\nATOM    143  CA  VAL A  20     -24.414   2.371   6.779  1.00  0.00           C  \\nATOM    144  C   VAL A  20     -26.052   2.434   6.401  1.00  0.00           C  \\nATOM    145  O   VAL A  20     -26.556   3.442   5.771  1.00  0.00           O  \\nATOM    146  CB  VAL A  20     -23.784   2.560   5.519  1.00  0.00           C  \\nATOM    147  CG1 VAL A  20     -24.540   2.119   4.637  1.00  0.00           C  \\nATOM    148  CG2 VAL A  20     -23.658   4.009   5.267  1.00  0.00           C  \\nATOM    149  N   GLU A  21     -26.682   1.363   6.527  1.00  0.00           N  \\nATOM    150  CA  GLU A  21     -28.194   1.363   5.897  1.00  0.00           C  \\nATOM    151  C   GLU A  21     -29.013   1.741   7.283  1.00  0.00           C  \\nATOM    152  O   GLU A  21     -28.635   2.623   7.913  1.00  0.00           O  \\nATOM    153  CB  GLU A  21     -28.572  -0.086   5.645  1.00  0.00           C  \\nATOM    154  CG  GLU A  21     -28.698  -0.779   4.133  1.00  0.00           C  \\nATOM    155  CD  GLU A  21     -27.690  -1.913   4.007  1.00  0.00           C  \\nATOM    156  OE1 GLU A  21     -26.682  -2.039   5.141  1.00  0.00           O  \\nATOM    157  OE2 GLU A  21     -27.753  -2.921   2.873  1.00  0.00           O  \\nATOM    158  N   ASN A  22     -30.210   1.363   7.283  1.00  0.00           N  \\nATOM    159  CA  ASN A  22     -30.966   1.867   8.417  1.00  0.00           C  \\nATOM    160  C   ASN A  22     -30.714   1.237   9.677  1.00  0.00           C  \\nATOM    161  O   ASN A  22     -31.218   0.103  10.055  1.00  0.00           O  \\nATOM    162  N   GLY A  23     -29.958   1.552  10.433  1.00  0.00           N  \\nATOM    163  CA  GLY A  23     -29.517   0.859  11.819  1.00  0.00           C  \\nATOM    164  C   GLY A  23     -28.572  -0.338  11.252  1.00  0.00           C  \\nATOM    165  O   GLY A  23     -28.194  -1.283  12.071  1.00  0.00           O  \\nATOM    166  N   GLY A  24     -28.068  -0.149   9.929  1.00  0.00           N  \\nATOM    167  CA  GLY A  24     -27.060  -1.409   9.047  1.00  0.00           C  \\nATOM    168  C   GLY A  24     -26.871  -2.480  10.118  1.00  0.00           C  \\nATOM    169  O   GLY A  24     -27.186  -3.803   9.929  1.00  0.00           O  \\nATOM    170  N   LYS A  25     -26.304  -2.165  11.441  1.00  0.00           N  \\nATOM    171  CA  LYS A  25     -26.241  -3.047  12.701  1.00  0.00           C  \\nATOM    172  C   LYS A  25     -24.792  -2.795  13.079  1.00  0.00           C  \\nATOM    173  O   LYS A  25     -23.910  -2.417  12.323  1.00  0.00           O  \\nATOM    174  CB  LYS A  25     -26.241  -4.433  11.819  1.00  0.00           C  \\nATOM    175  CG  LYS A  25     -25.296  -4.496  10.433  1.00  0.00           C  \\nATOM    176  CD  LYS A  25     -24.918  -5.819   9.803  1.00  0.00           C  \\nATOM    177  CE  LYS A  25     -23.784  -5.819   8.543  1.00  0.00           C  \\nATOM    178  NZ  LYS A  25     -23.658  -6.827   7.661  1.00  0.00           N  \\nATOM    179  N   HIS A  26     -24.414  -2.921  14.465  1.00  0.00           N  \\nATOM    180  CA  HIS A  26     -23.028  -2.417  14.654  1.00  0.00           C  \\nATOM    181  C   HIS A  26     -21.894  -3.425  14.087  1.00  0.00           C  \\nATOM    182  O   HIS A  26     -22.083  -3.803  12.827  1.00  0.00           O  \\nATOM    183  CB  HIS A  26     -23.028  -2.543  16.103  1.00  0.00           C  \\nATOM    184  CG  HIS A  26     -24.036  -1.094  16.355  1.00  0.00           C  \\nATOM    185  ND1 HIS A  26     -24.540  -0.338  15.473  1.00  0.00           N  \\nATOM    186  CD2 HIS A  26     -24.666  -0.905  17.363  1.00  0.00           C  \\nATOM    187  CE1 HIS A  26     -25.422   0.355  16.103  1.00  0.00           C  \\nATOM    188  NE2 HIS A  26     -25.422   0.103  17.237  1.00  0.00           N  \\nATOM    189  N   LYS A  27     -20.886  -3.551  14.717  1.00  0.00           N  \\nATOM    190  CA  LYS A  27     -19.752  -4.433  14.213  1.00  0.00           C  \\nATOM    191  C   LYS A  27     -18.366  -3.677  14.276  1.00  0.00           C  \\nATOM    192  O   LYS A  27     -17.484  -3.677  13.331  1.00  0.00           O  \\nATOM    193  CB  LYS A  27     -20.004  -4.559  12.575  1.00  0.00           C  \\nATOM    194  CG  LYS A  27     -20.382  -3.425  11.882  1.00  0.00           C  \\nATOM    195  CD  LYS A  27     -19.500  -3.047  11.063  1.00  0.00           C  \\nATOM    196  CE  LYS A  27     -19.752  -3.677   9.551  1.00  0.00           C  \\nATOM    197  NZ  LYS A  27     -18.618  -3.677   8.858  1.00  0.00           N  \\nATOM    198  N   VAL A  28     -18.177  -3.047  15.410  1.00  0.00           N  \\nATOM    199  CA  VAL A  28     -16.854  -2.228  15.410  1.00  0.00           C  \\nATOM    200  C   VAL A  28     -17.358  -0.905  15.473  1.00  0.00           C  \\nATOM    201  O   VAL A  28     -17.610  -0.275  16.481  1.00  0.00           O  \\nATOM    202  CB  VAL A  28     -15.846  -2.291  14.780  1.00  0.00           C  \\nATOM    203  CG1 VAL A  28     -15.594  -1.094  14.087  1.00  0.00           C  \\nATOM    204  CG2 VAL A  28     -14.586  -2.858  15.095  1.00  0.00           C  \\nATOM    205  N   GLY A  29     -17.610  -0.023  14.591  1.00  0.00           N  \\nATOM    206  CA  GLY A  29     -18.240   1.111  14.843  1.00  0.00           C  \\nATOM    207  C   GLY A  29     -19.815   1.048  14.843  1.00  0.00           C  \\nATOM    208  O   GLY A  29     -20.193   0.103  13.961  1.00  0.00           O  \\nATOM    209  N   PRO A  30     -20.634   1.615  15.977  1.00  0.00           N  \\nATOM    210  CA  PRO A  30     -22.020   1.489  15.725  1.00  0.00           C  \\nATOM    211  C   PRO A  30     -22.398   1.804  14.339  1.00  0.00           C  \\nATOM    212  O   PRO A  30     -22.020   2.371  13.457  1.00  0.00           O  \\nATOM    213  CB  PRO A  30     -22.524   2.371  16.481  1.00  0.00           C  \\nATOM    214  CG  PRO A  30     -21.642   1.993  16.859  1.00  0.00           C  \\nATOM    215  CD  PRO A  30     -20.508   2.497  16.607  1.00  0.00           C  \\nATOM    216  N   ASN A  31     -23.910   1.552  13.961  1.00  0.00           N  \\nATOM    217  CA  ASN A  31     -24.414   2.119  12.701  1.00  0.00           C  \\nATOM    218  C   ASN A  31     -24.036   3.631  12.764  1.00  0.00           C  \\nATOM    219  O   ASN A  31     -24.036   4.387  13.835  1.00  0.00           O  \\nATOM    220  CB  ASN A  31     -25.926   1.993  12.701  1.00  0.00           C  \\nATOM    221  CG  ASN A  31     -26.052   2.056  11.063  1.00  0.00           C  \\nATOM    222  OD1 ASN A  31     -27.186   2.623  10.307  1.00  0.00           O  \\nATOM    223  ND2 ASN A  31     -25.359   1.363  10.181  1.00  0.00           N  \\nATOM    224  N   LEU A  32     -23.658   4.261  11.819  1.00  0.00           N  \\nATOM    225  CA  LEU A  32     -23.280   5.899  11.819  1.00  0.00           C  \\nATOM    226  C   LEU A  32     -24.666   6.403  11.315  1.00  0.00           C  \\nATOM    227  O   LEU A  32     -24.666   6.655   9.929  1.00  0.00           O  \\nATOM    228  CB  LEU A  32     -22.146   6.466  11.315  1.00  0.00           C  \\nATOM    229  CG  LEU A  32     -20.886   6.529  12.323  1.00  0.00           C  \\nATOM    230  CD1 LEU A  32     -20.382   7.978  12.323  1.00  0.00           C  \\nATOM    231  CD2 LEU A  32     -21.327   6.277  13.457  1.00  0.00           C  \\nATOM    232  N   TRP A  33     -25.611   6.151  11.693  1.00  0.00           N  \\nATOM    233  CA  TRP A  33     -27.186   6.529  11.189  1.00  0.00           C  \\nATOM    234  C   TRP A  33     -27.312   7.915  10.433  1.00  0.00           C  \\nATOM    235  O   TRP A  33     -27.312   7.852   9.299  1.00  0.00           O  \\nATOM    236  CB  TRP A  33     -28.446   6.403  12.071  1.00  0.00           C  \\nATOM    237  CG  TRP A  33     -28.572   5.773  13.205  1.00  0.00           C  \\nATOM    238  CD1 TRP A  33     -27.564   4.891  13.583  1.00  0.00           C  \\nATOM    239  CD2 TRP A  33     -29.328   5.647  14.213  1.00  0.00           C  \\nATOM    240  NE1 TRP A  33     -27.816   4.387  14.843  1.00  0.00           N  \\nATOM    241  CE2 TRP A  33     -29.013   4.828  15.158  1.00  0.00           C  \\nATOM    242  CE3 TRP A  33     -30.462   6.277  14.213  1.00  0.00           C  \\nATOM    243  CZ2 TRP A  33     -29.706   4.639  15.851  1.00  0.00           C  \\nATOM    244  CZ3 TRP A  33     -31.092   5.962  15.284  1.00  0.00           C  \\nATOM    245  CH2 TRP A  33     -30.714   5.143  15.914  1.00  0.00           C  \\nATOM    246  N   GLY A  34     -27.564   8.797  11.315  1.00  0.00           N  \\nATOM    247  CA  GLY A  34     -27.690  10.120  10.685  1.00  0.00           C  \\nATOM    248  C   GLY A  34     -26.934  11.191  11.315  1.00  0.00           C  \\nATOM    249  O   GLY A  34     -27.312  12.199  12.134  1.00  0.00           O  \\nATOM    250  N   LEU A  35     -25.548  11.191  11.252  1.00  0.00           N  \\nATOM    251  CA  LEU A  35     -24.666  12.199  11.945  1.00  0.00           C  \\nATOM    252  C   LEU A  35     -25.548  13.459  12.386  1.00  0.00           C  \\nATOM    253  O   LEU A  35     -25.800  13.963  13.457  1.00  0.00           O  \\nATOM    254  CB  LEU A  35     -23.154  12.640  11.567  1.00  0.00           C  \\nATOM    255  CG  LEU A  35     -22.335  11.506  11.189  1.00  0.00           C  \\nATOM    256  CD1 LEU A  35     -20.886  11.695  11.189  1.00  0.00           C  \\nATOM    257  CD2 LEU A  35     -22.650  10.183  11.819  1.00  0.00           C  \\nATOM    258  N   PHE A  36     -26.052  13.963  11.441  1.00  0.00           N  \\nATOM    259  CA  PHE A  36     -26.934  15.223  11.693  1.00  0.00           C  \\nATOM    260  C   PHE A  36     -28.131  15.223  12.701  1.00  0.00           C  \\nATOM    261  O   PHE A  36     -29.328  15.223  12.449  1.00  0.00           O  \\nATOM    262  CB  PHE A  36     -27.186  15.601  10.055  1.00  0.00           C  \\nATOM    263  CG  PHE A  36     -25.926  15.601   9.425  1.00  0.00           C  \\nATOM    264  CD1 PHE A  36     -25.674  15.727   8.102  1.00  0.00           C  \\nATOM    265  CD2 PHE A  36     -24.792  15.097   9.803  1.00  0.00           C  \\nATOM    266  CE1 PHE A  36     -24.540  15.475   7.409  1.00  0.00           C  \\nATOM    267  CE2 PHE A  36     -23.658  15.097   9.551  1.00  0.00           C  \\nATOM    268  CZ  PHE A  36     -23.406  15.097   8.291  1.00  0.00           C  \\nATOM    269  N   GLY A  37     -27.879  15.538  13.961  1.00  0.00           N  \\nATOM    270  CA  GLY A  37     -28.509  15.664  15.347  1.00  0.00           C  \\nATOM    271  C   GLY A  37     -28.005  14.971  16.355  1.00  0.00           C  \\nATOM    272  O   GLY A  37     -28.572  14.215  17.300  1.00  0.00           O  \\nATOM    273  N   ARG A  38     -26.682  15.097  16.859  1.00  0.00           N  \\nATOM    274  CA  ARG A  38     -26.556  14.341  18.119  1.00  0.00           C  \\nATOM    275  C   ARG A  38     -25.170  14.845  18.371  1.00  0.00           C  \\nATOM    276  O   ARG A  38     -24.414  15.349  17.741  1.00  0.00           O  \\nATOM    277  CB  ARG A  38     -26.178  13.081  17.741  1.00  0.00           C  \\nATOM    278  CG  ARG A  38     -25.863  12.703  16.355  1.00  0.00           C  \\nATOM    279  CD  ARG A  38     -25.422  11.191  16.040  1.00  0.00           C  \\nATOM    280  NE  ARG A  38     -24.666  10.561  16.985  1.00  0.00           N  \\nATOM    281  CZ  ARG A  38     -24.792   9.238  16.859  1.00  0.00           C  \\nATOM    282  NH1 ARG A  38     -24.351   8.545  17.489  1.00  0.00           N  \\nATOM    283  NH2 ARG A  38     -25.485   8.797  15.851  1.00  0.00           N  \\nATOM    284  N   LYS A  39     -25.044  14.593  19.757  1.00  0.00           N  \\nATOM    285  CA  LYS A  39     -24.036  15.160  20.639  1.00  0.00           C  \\nATOM    286  C   LYS A  39     -23.028  14.089  20.891  1.00  0.00           C  \\nATOM    287  O   LYS A  39     -23.406  13.207  20.009  1.00  0.00           O  \\nATOM    288  CB  LYS A  39     -24.540  15.601  22.151  1.00  0.00           C  \\nATOM    289  CG  LYS A  39     -24.414  15.475  23.663  1.00  0.00           C  \\nATOM    290  CD  LYS A  39     -25.170  16.231  24.923  1.00  0.00           C  \\nATOM    291  N   THR A  40     -22.020  13.837  21.521  1.00  0.00           N  \\nATOM    292  CA  THR A  40     -21.390  12.388  21.269  1.00  0.00           C  \\nATOM    293  C   THR A  40     -21.642  11.191  22.277  1.00  0.00           C  \\nATOM    294  O   THR A  40     -21.516  11.695  23.411  1.00  0.00           O  \\nATOM    295  CB  THR A  40     -19.752  12.577  21.269  1.00  0.00           C  \\nATOM    296  OG1 THR A  40     -19.374  11.884  19.883  1.00  0.00           O  \\nATOM    297  CG2 THR A  40     -19.626  13.837  20.765  1.00  0.00           C  \\nATOM    298  N   GLY A  41     -22.020  10.309  21.773  1.00  0.00           N  \\nATOM    299  CA  GLY A  41     -22.650   9.553  22.907  1.00  0.00           C  \\nATOM    300  C   GLY A  41     -23.910   8.797  22.403  1.00  0.00           C  \\nATOM    301  O   GLY A  41     -24.162   8.545  21.395  1.00  0.00           O  \\nATOM    302  N   GLN A  42     -24.918   8.419  23.033  1.00  0.00           N  \\nATOM    303  CA  GLN A  42     -26.241   7.915  23.033  1.00  0.00           C  \\nATOM    304  C   GLN A  42     -26.304   7.537  21.647  1.00  0.00           C  \\nATOM    305  O   GLN A  42     -26.556   8.545  20.765  1.00  0.00           O  \\nATOM    306  CB  GLN A  42     -27.438   8.545  23.537  1.00  0.00           C  \\nATOM    307  CG  GLN A  42     -28.068   7.789  24.419  1.00  0.00           C  \\nATOM    308  CD  GLN A  42     -29.076   7.096  24.230  1.00  0.00           C  \\nATOM    309  OE1 GLN A  42     -29.706   5.899  24.797  1.00  0.00           O  \\nATOM    310  NE2 GLN A  42     -29.706   7.411  22.781  1.00  0.00           N  \\nATOM    311  N   ALA A  43     -26.304   6.277  21.143  1.00  0.00           N  \\nATOM    312  CA  ALA A  43     -26.682   6.025  19.505  1.00  0.00           C  \\nATOM    313  C   ALA A  43     -27.186   4.639  19.442  1.00  0.00           C  \\nATOM    314  O   ALA A  43     -27.060   4.009  18.497  1.00  0.00           O  \\nATOM    315  CB  ALA A  43     -25.548   6.025  18.497  1.00  0.00           C  \\nATOM    316  N   GLU A  44     -27.438   3.883  20.450  1.00  0.00           N  \\nATOM    317  CA  GLU A  44     -28.131   2.497  20.261  1.00  0.00           C  \\nATOM    318  C   GLU A  44     -27.060   1.615  20.891  1.00  0.00           C  \\nATOM    319  O   GLU A  44     -26.556   0.481  20.324  1.00  0.00           O  \\nATOM    320  CB  GLU A  44     -28.635   2.371  18.749  1.00  0.00           C  \\nATOM    321  CG  GLU A  44     -30.210   0.993  18.749  1.00  0.00           C  \\nATOM    322  CD  GLU A  44     -30.840   2.875  19.379  1.00  0.00           C  \\nATOM    323  OE1 GLU A  44     -32.100   2.875  18.875  1.00  0.00           O  \\nATOM    324  OE2 GLU A  44     -30.084   4.135  19.505  1.00  0.00           O  \\nATOM    325  N   GLY A  45     -26.808   1.615  22.277  1.00  0.00           N  \\nATOM    326  CA  GLY A  45     -25.674   0.859  22.781  1.00  0.00           C  \\nATOM    327  C   GLY A  45     -24.162   1.363  22.907  1.00  0.00           C  \\nATOM    328  O   GLY A  45     -23.532   1.363  21.521  1.00  0.00           O  \\nATOM    329  N   TYR A  46     -23.532   1.993  23.789  1.00  0.00           N  \\nATOM    330  CA  TYR A  46     -22.146   2.875  23.537  1.00  0.00           C  \\nATOM    331  C   TYR A  46     -21.327   1.993  24.356  1.00  0.00           C  \\nATOM    332  O   TYR A  46     -21.327   0.733  24.167  1.00  0.00           O  \\nATOM    333  CB  TYR A  46     -22.524   4.198  23.159  1.00  0.00           C  \\nATOM    334  CG  TYR A  46     -22.146   5.143  22.151  1.00  0.00           C  \\nATOM    335  CD1 TYR A  46     -22.776   5.269  21.521  1.00  0.00           C  \\nATOM    336  CD2 TYR A  46     -20.886   5.647  21.647  1.00  0.00           C  \\nATOM    337  CE1 TYR A  46     -22.524   5.899  20.387  1.00  0.00           C  \\nATOM    338  CE2 TYR A  46     -20.382   6.277  20.513  1.00  0.00           C  \\nATOM    339  CZ  TYR A  46     -21.264   6.277  20.009  1.00  0.00           C  \\nATOM    340  OH  TYR A  46     -21.138   6.655  19.001  1.00  0.00           O  \\nATOM    341  N   SER A  47     -20.634   2.560  25.049  1.00  0.00           N  \\nATOM    342  CA  SER A  47     -19.374   1.741  25.679  1.00  0.00           C  \\nATOM    343  C   SER A  47     -18.114   2.623  25.679  1.00  0.00           C  \\nATOM    344  O   SER A  47     -18.114   3.379  24.797  1.00  0.00           O  \\nATOM    345  CB  SER A  47     -19.248   0.355  25.427  1.00  0.00           C  \\nATOM    346  OG  SER A  47     -18.870   0.481  23.789  1.00  0.00           O  \\nATOM    347  N   TYR A  48     -17.484   2.749  26.939  1.00  0.00           N  \\nATOM    348  CA  TYR A  48     -16.413   3.757  26.939  1.00  0.00           C  \\nATOM    349  C   TYR A  48     -16.602   4.891  27.695  1.00  0.00           C  \\nATOM    350  O   TYR A  48     -17.610   5.269  27.821  1.00  0.00           O  \\nATOM    351  CB  TYR A  48     -16.224   3.757  25.301  1.00  0.00           C  \\nATOM    352  CG  TYR A  48     -15.216   4.639  24.545  1.00  0.00           C  \\nATOM    353  CD1 TYR A  48     -14.082   5.143  24.797  1.00  0.00           C  \\nATOM    354  CD2 TYR A  48     -15.342   4.765  23.411  1.00  0.00           C  \\nATOM    355  CE1 TYR A  48     -13.326   6.025  24.167  1.00  0.00           C  \\nATOM    356  CE2 TYR A  48     -14.712   5.521  22.529  1.00  0.00           C  \\nATOM    357  CZ  TYR A  48     -13.704   6.088  23.033  1.00  0.00           C  \\nATOM    358  OH  TYR A  48     -13.074   6.907  22.151  1.00  0.00           O  \\nATOM    359  N   THR A  49     -15.594   5.647  28.199  1.00  0.00           N  \\nATOM    360  CA  THR A  49     -15.846   6.907  29.207  1.00  0.00           C  \\nATOM    361  C   THR A  49     -15.594   8.041  28.829  1.00  0.00           C  \\nATOM    362  O   THR A  49     -16.350   8.671  27.947  1.00  0.00           O  \\nATOM    363  CB  THR A  49     -15.531   6.403  30.845  1.00  0.00           C  \\nATOM    364  OG1 THR A  49     -14.208   5.899  30.845  1.00  0.00           O  \\nATOM    365  CG2 THR A  49     -16.224   5.395  31.223  1.00  0.00           C  \\nATOM    366  N   ASP A  50     -14.334   8.482  28.703  1.00  0.00           N  \\nATOM    367  CA  ASP A  50     -14.082   9.805  28.199  1.00  0.00           C  \\nATOM    368  C   ASP A  50     -13.704   9.679  26.813  1.00  0.00           C  \\nATOM    369  O   ASP A  50     -13.074   8.545  26.183  1.00  0.00           O  \\nATOM    370  CB  ASP A  50     -13.074  10.435  28.829  1.00  0.00           C  \\nATOM    371  CG  ASP A  50     -12.948  10.183  30.089  1.00  0.00           C  \\nATOM    372  OD1 ASP A  50     -12.066   9.049  30.593  1.00  0.00           O  \\nATOM    373  OD2 ASP A  50     -13.956  10.309  30.971  1.00  0.00           O  \\nATOM    374  N   ALA A  51     -13.830  10.309  25.805  1.00  0.00           N  \\nATOM    375  CA  ALA A  51     -13.704   9.931  24.419  1.00  0.00           C  \\nATOM    376  C   ALA A  51     -15.216   9.805  23.915  1.00  0.00           C  \\nATOM    377  O   ALA A  51     -16.098   9.301  24.167  1.00  0.00           O  \\nATOM    378  CB  ALA A  51     -13.200   8.671  24.041  1.00  0.00           C  \\nATOM    379  N   ASN A  52     -15.342  10.561  22.907  1.00  0.00           N  \\nATOM    380  CA  ASN A  52     -16.539  10.561  22.277  1.00  0.00           C  \\nATOM    381  C   ASN A  52     -17.232  11.380  22.907  1.00  0.00           C  \\nATOM    382  O   ASN A  52     -17.232  12.766  22.781  1.00  0.00           O  \\nATOM    383  CB  ASN A  52     -17.610   9.553  21.899  1.00  0.00           C  \\nATOM    384  CG  ASN A  52     -17.484   8.671  20.639  1.00  0.00           C  \\nATOM    385  OD1 ASN A  52     -16.980   7.789  20.135  1.00  0.00           O  \\nATOM    386  ND2 ASN A  52     -18.240   9.427  19.631  1.00  0.00           N  \\nATOM    387  N   LYS A  53     -17.862  10.939  24.041  1.00  0.00           N  \\nATOM    388  CA  LYS A  53     -18.240  11.947  25.301  1.00  0.00           C  \\nATOM    389  C   LYS A  53     -17.106  12.703  25.805  1.00  0.00           C  \\nATOM    390  O   LYS A  53     -16.476  13.711  25.175  1.00  0.00           O  \\nATOM    391  CB  LYS A  53     -18.870  11.317  26.561  1.00  0.00           C  \\nATOM    392  CG  LYS A  53     -20.130  10.435  26.057  1.00  0.00           C  \\nATOM    393  CD  LYS A  53     -20.634   9.301  27.065  1.00  0.00           C  \\nATOM    394  CE  LYS A  53     -22.146   8.671  26.372  1.00  0.00           C  \\nATOM    395  NZ  LYS A  53     -22.524   7.663  27.191  1.00  0.00           N  \\nATOM    396  N   SER A  54     -16.602  12.325  27.065  1.00  0.00           N  \\nATOM    397  CA  SER A  54     -15.594  13.081  27.821  1.00  0.00           C  \\nATOM    398  C   SER A  54     -15.594  14.593  27.443  1.00  0.00           C  \\nATOM    399  O   SER A  54     -16.476  15.223  27.695  1.00  0.00           O  \\nATOM    400  CB  SER A  54     -14.460  12.703  27.569  1.00  0.00           C  \\nATOM    401  OG  SER A  54     -13.452  12.955  28.703  1.00  0.00           O  \\nATOM    402  N   LYS A  55     -14.838  15.223  26.813  1.00  0.00           N  \\nATOM    403  CA  LYS A  55     -14.964  16.420  26.057  1.00  0.00           C  \\nATOM    404  C   LYS A  55     -16.602  16.987  25.931  1.00  0.00           C  \\nATOM    405  O   LYS A  55     -16.854  18.121  26.183  1.00  0.00           O  \\nATOM    406  CB  LYS A  55     -14.208  16.609  24.734  1.00  0.00           C  \\nATOM    407  CG  LYS A  55     -14.712  17.743  23.600  1.00  0.00           C  \\nATOM    408  CD  LYS A  55     -13.452  17.743  22.844  1.00  0.00           C  \\nATOM    409  CE  LYS A  55     -13.578  17.995  21.143  1.00  0.00           C  \\nATOM    410  NZ  LYS A  55     -12.444  17.743  20.135  1.00  0.00           N  \\nATOM    411  N   GLY A  56     -17.232  16.042  25.679  1.00  0.00           N  \\nATOM    412  CA  GLY A  56     -18.870  16.042  25.301  1.00  0.00           C  \\nATOM    413  C   GLY A  56     -19.122  17.239  24.293  1.00  0.00           C  \\nATOM    414  O   GLY A  56     -18.492  18.247  23.852  1.00  0.00           O  \\nATOM    415  N   ILE A  57     -20.130  17.113  23.285  1.00  0.00           N  \\nATOM    416  CA  ILE A  57     -20.004  17.869  22.025  1.00  0.00           C  \\nATOM    417  C   ILE A  57     -21.264  18.121  21.143  1.00  0.00           C  \\nATOM    418  O   ILE A  57     -22.398  18.499  21.395  1.00  0.00           O  \\nATOM    419  CB  ILE A  57     -18.996  17.491  21.395  1.00  0.00           C  \\nATOM    420  CG1 ILE A  57     -19.374  16.357  20.513  1.00  0.00           C  \\nATOM    421  CG2 ILE A  57     -17.862  17.050  22.025  1.00  0.00           C  \\nATOM    422  CD1 ILE A  57     -19.122  16.609  19.127  1.00  0.00           C  \\nATOM    423  N   VAL A  58     -21.264  17.743  19.631  1.00  0.00           N  \\nATOM    424  CA  VAL A  58     -22.524  17.995  19.001  1.00  0.00           C  \\nATOM    425  C   VAL A  58     -22.020  17.239  17.615  1.00  0.00           C  \\nATOM    426  O   VAL A  58     -21.642  18.121  16.985  1.00  0.00           O  \\nATOM    427  CB  VAL A  58     -23.280  19.129  19.127  1.00  0.00           C  \\nATOM    428  CG1 VAL A  58     -22.398  20.263  18.749  1.00  0.00           C  \\nATOM    429  CG2 VAL A  58     -24.540  19.192  18.371  1.00  0.00           C  \\nATOM    430  N   TRP A  59     -22.398  16.231  16.859  1.00  0.00           N  \\nATOM    431  CA  TRP A  59     -22.020  16.105  15.221  1.00  0.00           C  \\nATOM    432  C   TRP A  59     -22.524  17.239  14.213  1.00  0.00           C  \\nATOM    433  O   TRP A  59     -23.784  17.365  14.339  1.00  0.00           O  \\nATOM    434  CB  TRP A  59     -22.146  14.845  14.717  1.00  0.00           C  \\nATOM    435  CG  TRP A  59     -21.264  13.963  15.347  1.00  0.00           C  \\nATOM    436  CD1 TRP A  59     -21.390  12.703  15.851  1.00  0.00           C  \\nATOM    437  CD2 TRP A  59     -19.878  14.152  15.536  1.00  0.00           C  \\nATOM    438  NE1 TRP A  59     -20.382  12.451  16.607  1.00  0.00           N  \\nATOM    439  CE2 TRP A  59     -19.500  13.207  15.977  1.00  0.00           C  \\nATOM    440  CE3 TRP A  59     -19.248  15.160  15.095  1.00  0.00           C  \\nATOM    441  CZ2 TRP A  59     -18.114  13.459  16.607  1.00  0.00           C  \\nATOM    442  CZ3 TRP A  59     -18.051  15.223  15.473  1.00  0.00           C  \\nATOM    443  CH2 TRP A  59     -17.484  14.467  16.229  1.00  0.00           C  \\nATOM    444  N   ASN A  60     -21.768  17.869  13.772  1.00  0.00           N  \\nATOM    445  CA  ASN A  60     -22.398  19.129  12.953  1.00  0.00           C  \\nATOM    446  C   ASN A  60     -21.642  19.255  11.441  1.00  0.00           C  \\nATOM    447  O   ASN A  60     -20.634  18.751  11.189  1.00  0.00           O  \\nATOM    448  CB  ASN A  60     -22.524  20.515  13.583  1.00  0.00           C  \\nATOM    449  CG  ASN A  60     -22.776  21.649  12.701  1.00  0.00           C  \\nATOM    450  OD1 ASN A  60     -24.036  21.712  11.945  1.00  0.00           O  \\nATOM    451  ND2 ASN A  60     -21.894  22.783  12.701  1.00  0.00           N  \\nATOM    452  N   GLU A  61     -22.020  20.137  10.811  1.00  0.00           N  \\nATOM    453  CA  GLU A  61     -21.138  20.263   9.425  1.00  0.00           C  \\nATOM    454  C   GLU A  61     -19.626  20.389  10.181  1.00  0.00           C  \\nATOM    455  O   GLU A  61     -18.744  19.507   9.803  1.00  0.00           O  \\nATOM    456  CB  GLU A  61     -21.516  21.523   8.669  1.00  0.00           C  \\nATOM    457  CG  GLU A  61     -22.020  21.523   7.409  1.00  0.00           C  \\nATOM    458  CD  GLU A  61     -23.154  20.767   7.283  1.00  0.00           C  \\nATOM    459  OE1 GLU A  61     -24.540  20.767   7.535  1.00  0.00           O  \\nATOM    460  OE2 GLU A  61     -22.902  19.381   6.653  1.00  0.00           O  \\nATOM    461  N   ASN A  62     -19.248  21.397  10.685  1.00  0.00           N  \\nATOM    462  CA  ASN A  62     -17.736  21.586  11.441  1.00  0.00           C  \\nATOM    463  C   ASN A  62     -17.106  20.137  11.945  1.00  0.00           C  \\nATOM    464  O   ASN A  62     -15.972  20.515  12.071  1.00  0.00           O  \\nATOM    465  CB  ASN A  62     -17.736  22.783  11.945  1.00  0.00           C  \\nATOM    466  CG  ASN A  62     -17.610  24.043  11.441  1.00  0.00           C  \\nATOM    467  OD1 ASN A  62     -18.744  24.547  10.559  1.00  0.00           O  \\nATOM    468  ND2 ASN A  62     -16.539  24.799  11.063  1.00  0.00           N  \\nATOM    469  N   THR A  63     -17.610  20.263  13.268  1.00  0.00           N  \\nATOM    470  CA  THR A  63     -17.043  19.255  14.087  1.00  0.00           C  \\nATOM    471  C   THR A  63     -16.602  17.869  13.331  1.00  0.00           C  \\nATOM    472  O   THR A  63     -15.720  17.239  13.331  1.00  0.00           O  \\nATOM    473  CB  THR A  63     -18.051  18.751  15.095  1.00  0.00           C  \\nATOM    474  OG1 THR A  63     -17.358  18.499  15.851  1.00  0.00           O  \\nATOM    475  CG2 THR A  63     -18.492  17.365  14.528  1.00  0.00           C  \\nATOM    476  N   LEU A  64     -17.358  17.617  12.197  1.00  0.00           N  \\nATOM    477  CA  LEU A  64     -16.854  16.357  11.441  1.00  0.00           C  \\nATOM    478  C   LEU A  64     -15.468  16.861  10.685  1.00  0.00           C  \\nATOM    479  O   LEU A  64     -14.838  16.231  10.433  1.00  0.00           O  \\nATOM    480  CB  LEU A  64     -18.114  15.979  10.055  1.00  0.00           C  \\nATOM    481  CG  LEU A  64     -18.744  14.719  10.433  1.00  0.00           C  \\nATOM    482  CD1 LEU A  64     -19.122  14.719  11.567  1.00  0.00           C  \\nATOM    483  CD2 LEU A  64     -19.878  14.593   9.803  1.00  0.00           C  \\nATOM    484  N   MET A  65     -15.468  17.995  10.181  1.00  0.00           N  \\nATOM    485  CA  MET A  65     -13.956  18.499   9.803  1.00  0.00           C  \\nATOM    486  C   MET A  65     -13.200  18.373  11.189  1.00  0.00           C  \\nATOM    487  O   MET A  65     -12.192  17.869  11.189  1.00  0.00           O  \\nATOM    488  CB  MET A  65     -13.956  19.885   9.425  1.00  0.00           C  \\nATOM    489  CG  MET A  65     -13.452  20.137   7.913  1.00  0.00           C  \\nATOM    490  SD  MET A  65     -13.704  18.625   7.283  1.00  0.00           S  \\nATOM    491  CE  MET A  65     -13.452  19.192   5.519  1.00  0.00           C  \\nATOM    492  N   GLU A  66     -13.578  18.751  12.197  1.00  0.00           N  \\nATOM    493  CA  GLU A  66     -13.074  18.751  13.457  1.00  0.00           C  \\nATOM    494  C   GLU A  66     -12.444  17.239  13.646  1.00  0.00           C  \\nATOM    495  O   GLU A  66     -11.310  17.113  13.835  1.00  0.00           O  \\nATOM    496  CB  GLU A  66     -13.830  19.381  14.717  1.00  0.00           C  \\nATOM    497  CG  GLU A  66     -13.578  20.767  14.969  1.00  0.00           C  \\nATOM    498  CD  GLU A  66     -12.318  21.271  15.851  1.00  0.00           C  \\nATOM    499  OE1 GLU A  66     -12.192  22.657  16.103  1.00  0.00           O  \\nATOM    500  OE2 GLU A  66     -11.310  20.641  16.355  1.00  0.00           O  \\nATOM    501  N   TYR A  67     -13.263  16.483  13.457  1.00  0.00           N  \\nATOM    502  CA  TYR A  67     -12.822  14.971  13.457  1.00  0.00           C  \\nATOM    503  C   TYR A  67     -12.192  14.467  12.323  1.00  0.00           C  \\nATOM    504  O   TYR A  67     -11.184  14.089  11.945  1.00  0.00           O  \\nATOM    505  CB  TYR A  67     -14.082  14.341  13.835  1.00  0.00           C  \\nATOM    506  CG  TYR A  67     -14.082  12.955  14.465  1.00  0.00           C  \\nATOM    507  CD1 TYR A  67     -14.838  11.821  14.465  1.00  0.00           C  \\nATOM    508  CD2 TYR A  67     -13.326  13.081  15.599  1.00  0.00           C  \\nATOM    509  CE1 TYR A  67     -14.649  10.876  15.221  1.00  0.00           C  \\nATOM    510  CE2 TYR A  67     -13.200  11.947  16.103  1.00  0.00           C  \\nATOM    511  CZ  TYR A  67     -13.830  10.876  15.977  1.00  0.00           C  \\nATOM    512  OH  TYR A  67     -13.704   9.805  16.607  1.00  0.00           O  \\nATOM    513  N   LEU A  68     -12.822  14.719  11.315  1.00  0.00           N  \\nATOM    514  CA  LEU A  68     -12.192  14.341  10.055  1.00  0.00           C  \\nATOM    515  C   LEU A  68     -10.806  14.719  10.307  1.00  0.00           C  \\nATOM    516  O   LEU A  68      -9.798  13.963  10.307  1.00  0.00           O  \\nATOM    517  CB  LEU A  68     -13.074  14.845   9.173  1.00  0.00           C  \\nATOM    518  CG  LEU A  68     -14.334  13.837   8.795  1.00  0.00           C  \\nATOM    519  CD1 LEU A  68     -15.216  14.467   7.787  1.00  0.00           C  \\nATOM    520  CD2 LEU A  68     -13.704  12.829   7.787  1.00  0.00           C  \\nATOM    521  N   GLU A  69     -10.428  15.853  10.685  1.00  0.00           N  \\nATOM    522  CA  GLU A  69      -8.916  16.231  10.937  1.00  0.00           C  \\nATOM    523  C   GLU A  69      -8.097  14.971  11.441  1.00  0.00           C  \\nATOM    524  O   GLU A  69      -7.278  14.593  10.685  1.00  0.00           O  \\nATOM    525  CB  GLU A  69      -9.042  17.365  11.693  1.00  0.00           C  \\nATOM    526  CG  GLU A  69      -8.034  18.373  11.819  1.00  0.00           C  \\nATOM    527  CD  GLU A  69      -6.900  18.373  12.953  1.00  0.00           C  \\nATOM    528  OE1 GLU A  69      -5.388  18.499  12.827  1.00  0.00           O  \\nATOM    529  OE2 GLU A  69      -7.278  17.869  14.087  1.00  0.00           O  \\nATOM    530  N   ASN A  70      -8.097  14.845  12.575  1.00  0.00           N  \\nATOM    531  CA  ASN A  70      -7.278  13.585  12.827  1.00  0.00           C  \\nATOM    532  C   ASN A  70      -7.404  13.459  14.339  1.00  0.00           C  \\nATOM    533  O   ASN A  70      -7.152  14.215  15.221  1.00  0.00           O  \\nATOM    534  CB  ASN A  70      -5.892  13.837  12.449  1.00  0.00           C  \\nATOM    535  CG  ASN A  70      -4.758  14.089  13.457  1.00  0.00           C  \\nATOM    536  OD1 ASN A  70      -3.624  15.097  13.457  1.00  0.00           O  \\nATOM    537  ND2 ASN A  70      -4.758  13.333  14.717  1.00  0.00           N  \\nATOM    538  N   PRO A  71      -7.908  12.199  14.780  1.00  0.00           N  \\nATOM    539  CA  PRO A  71      -8.475  11.758  15.725  1.00  0.00           C  \\nATOM    540  C   PRO A  71      -7.593  11.758  16.859  1.00  0.00           C  \\nATOM    541  O   PRO A  71      -7.782  11.317  17.993  1.00  0.00           O  \\nATOM    542  CB  PRO A  71      -9.042  10.561  15.536  1.00  0.00           C  \\nATOM    543  CG  PRO A  71      -8.916  10.309  14.213  1.00  0.00           C  \\nATOM    544  CD  PRO A  71      -8.160  11.191  13.583  1.00  0.00           C  \\nATOM    545  N   LYS A  72      -6.522  12.325  16.733  1.00  0.00           N  \\nATOM    546  CA  LYS A  72      -5.388  12.703  17.615  1.00  0.00           C  \\nATOM    547  C   LYS A  72      -5.451  14.215  17.993  1.00  0.00           C  \\nATOM    548  O   LYS A  72      -5.262  14.971  17.111  1.00  0.00           O  \\nATOM    549  CB  LYS A  72      -3.876  12.451  17.111  1.00  0.00           C  \\nATOM    550  CG  LYS A  72      -3.624  11.065  17.363  1.00  0.00           C  \\nATOM    551  CD  LYS A  72      -2.112  10.435  17.237  1.00  0.00           C  \\nATOM    552  CE  LYS A  72      -1.860   8.923  17.741  1.00  0.00           C  \\nATOM    553  NZ  LYS A  72      -0.222   8.545  17.741  1.00  0.00           N  \\nATOM    554  N   LYS A  73      -6.081  14.719  19.127  1.00  0.00           N  \\nATOM    555  CA  LYS A  73      -6.396  16.231  19.127  1.00  0.00           C  \\nATOM    556  C   LYS A  73      -7.782  16.357  18.623  1.00  0.00           C  \\nATOM    557  O   LYS A  73      -8.160  16.231  17.363  1.00  0.00           O  \\nATOM    558  CB  LYS A  73      -5.640  17.113  17.993  1.00  0.00           C  \\nATOM    559  CG  LYS A  73      -4.758  18.247  18.623  1.00  0.00           C  \\nATOM    560  CD  LYS A  73      -3.876  19.003  17.993  1.00  0.00           C  \\nATOM    561  CE  LYS A  73      -2.616  20.389  18.812  1.00  0.00           C  \\nATOM    562  NZ  LYS A  73      -1.986  21.145  18.056  1.00  0.00           N  \\nATOM    563  N   TYR A  74      -8.664  16.231  19.631  1.00  0.00           N  \\nATOM    564  CA  TYR A  74     -10.050  16.105  19.379  1.00  0.00           C  \\nATOM    565  C   TYR A  74     -10.806  14.971  19.883  1.00  0.00           C  \\nATOM    566  O   TYR A  74     -10.554  13.963  19.379  1.00  0.00           O  \\nATOM    567  CB  TYR A  74     -10.554  16.105  17.741  1.00  0.00           C  \\nATOM    568  CG  TYR A  74     -11.814  16.735  17.363  1.00  0.00           C  \\nATOM    569  CD1 TYR A  74     -12.948  16.105  17.237  1.00  0.00           C  \\nATOM    570  CD2 TYR A  74     -11.940  17.995  16.985  1.00  0.00           C  \\nATOM    571  CE1 TYR A  74     -14.145  16.609  16.859  1.00  0.00           C  \\nATOM    572  CE2 TYR A  74     -13.074  18.625  16.607  1.00  0.00           C  \\nATOM    573  CZ  TYR A  74     -14.082  17.869  16.607  1.00  0.00           C  \\nATOM    574  OH  TYR A  74     -15.216  18.373  16.355  1.00  0.00           O  \\nATOM    575  N   ILE A  75     -11.310  15.097  21.017  1.00  0.00           N  \\nATOM    576  CA  ILE A  75     -11.940  14.089  21.647  1.00  0.00           C  \\nATOM    577  C   ILE A  75     -10.806  13.081  21.773  1.00  0.00           C  \\nATOM    578  O   ILE A  75     -10.932  11.821  21.521  1.00  0.00           O  \\nATOM    579  CB  ILE A  75     -13.200  13.585  21.143  1.00  0.00           C  \\nATOM    580  CG1 ILE A  75     -13.641  14.089  19.631  1.00  0.00           C  \\nATOM    581  CG2 ILE A  75     -14.271  13.585  22.025  1.00  0.00           C  \\nATOM    582  CD1 ILE A  75     -14.712  13.459  19.253  1.00  0.00           C  \\nATOM    583  N   PRO A  76      -9.420  13.585  21.899  1.00  0.00           N  \\nATOM    584  CA  PRO A  76      -8.160  12.955  21.647  1.00  0.00           C  \\nATOM    585  C   PRO A  76      -8.160  11.695  22.529  1.00  0.00           C  \\nATOM    586  O   PRO A  76      -8.538  11.695  23.789  1.00  0.00           O  \\nATOM    587  CB  PRO A  76      -6.900  13.837  21.836  1.00  0.00           C  \\nATOM    588  CG  PRO A  76      -7.152  14.845  22.277  1.00  0.00           C  \\nATOM    589  CD  PRO A  76      -8.538  14.845  22.151  1.00  0.00           C  \\nATOM    590  N   GLY A  77      -8.097  10.435  22.088  1.00  0.00           N  \\nATOM    591  CA  GLY A  77      -8.286   9.175  22.907  1.00  0.00           C  \\nATOM    592  C   GLY A  77      -9.546   8.167  22.529  1.00  0.00           C  \\nATOM    593  O   GLY A  77      -9.798   7.159  22.781  1.00  0.00           O  \\nATOM    594  N   THR A  78     -10.050   8.545  21.143  1.00  0.00           N  \\nATOM    595  CA  THR A  78     -11.310   7.789  20.639  1.00  0.00           C  \\nATOM    596  C   THR A  78     -10.995   6.529  20.513  1.00  0.00           C  \\nATOM    597  O   THR A  78      -9.798   6.151  20.009  1.00  0.00           O  \\nATOM    598  CB  THR A  78     -11.625   8.167  19.253  1.00  0.00           C  \\nATOM    599  OG1 THR A  78     -13.074   8.167  19.127  1.00  0.00           O  \\nATOM    600  CG2 THR A  78     -11.184   9.553  18.875  1.00  0.00           C  \\nATOM    601  N   LYS A  79     -11.562   5.521  20.765  1.00  0.00           N  \\nATOM    602  CA  LYS A  79     -11.058   4.072  20.639  1.00  0.00           C  \\nATOM    603  C   LYS A  79     -11.058   3.631  19.001  1.00  0.00           C  \\nATOM    604  O   LYS A  79     -10.932   2.497  18.371  1.00  0.00           O  \\nATOM    605  CB  LYS A  79     -11.562   3.064  21.395  1.00  0.00           C  \\nATOM    606  CG  LYS A  79     -12.570   2.560  20.513  1.00  0.00           C  \\nATOM    607  CD  LYS A  79     -13.704   1.615  20.891  1.00  0.00           C  \\nATOM    608  CE  LYS A  79     -14.838   1.363  20.135  1.00  0.00           C  \\nATOM    609  NZ  LYS A  79     -15.972   0.859  20.513  1.00  0.00           N  \\nATOM    610  N   MET A  80     -11.310   4.639  18.119  1.00  0.00           N  \\nATOM    611  CA  MET A  80     -11.436   4.135  16.607  1.00  0.00           C  \\nATOM    612  C   MET A  80     -10.050   4.198  16.103  1.00  0.00           C  \\nATOM    613  O   MET A  80      -9.294   5.017  16.103  1.00  0.00           O  \\nATOM    614  CB  MET A  80     -12.444   5.017  15.851  1.00  0.00           C  \\nATOM    615  CG  MET A  80     -11.940   6.088  14.969  1.00  0.00           C  \\nATOM    616  SD  MET A  80     -13.074   6.781  13.457  1.00  0.00           S  \\nATOM    617  CE  MET A  80     -12.192   7.852  13.079  1.00  0.00           C  \\nATOM    618  N   ILE A  81      -9.672   3.127  15.725  1.00  0.00           N  \\nATOM    619  CA  ILE A  81      -8.160   3.127  15.095  1.00  0.00           C  \\nATOM    620  C   ILE A  81      -7.908   4.009  13.835  1.00  0.00           C  \\nATOM    621  O   ILE A  81      -8.034   3.505  12.323  1.00  0.00           O  \\nATOM    622  CB  ILE A  81      -7.656   1.867  15.284  1.00  0.00           C  \\nATOM    623  CG1 ILE A  81      -6.522   1.867  16.103  1.00  0.00           C  \\nATOM    624  CG2 ILE A  81      -7.026   1.237  13.835  1.00  0.00           C  \\nATOM    625  CD1 ILE A  81      -6.900   2.245  17.615  1.00  0.00           C  \\nATOM    626  N   PHE A  82      -7.530   5.143  13.961  1.00  0.00           N  \\nATOM    627  CA  PHE A  82      -7.404   5.899  12.449  1.00  0.00           C  \\nATOM    628  C   PHE A  82      -6.144   6.529  12.197  1.00  0.00           C  \\nATOM    629  O   PHE A  82      -5.766   7.411  12.953  1.00  0.00           O  \\nATOM    630  CB  PHE A  82      -8.538   6.655  12.449  1.00  0.00           C  \\nATOM    631  CG  PHE A  82      -8.916   7.159  10.937  1.00  0.00           C  \\nATOM    632  CD1 PHE A  82      -8.916   8.293  10.433  1.00  0.00           C  \\nATOM    633  CD2 PHE A  82      -9.042   6.529  10.055  1.00  0.00           C  \\nATOM    634  CE1 PHE A  82      -9.042   8.671   9.173  1.00  0.00           C  \\nATOM    635  CE2 PHE A  82      -9.042   6.781   8.669  1.00  0.00           C  \\nATOM    636  CZ  PHE A  82      -9.042   7.852   8.291  1.00  0.00           C  \\nATOM    637  N   ALA A  83      -5.640   6.529  11.189  1.00  0.00           N  \\nATOM    638  CA  ALA A  83      -4.506   7.285  10.937  1.00  0.00           C  \\nATOM    639  C   ALA A  83      -4.758   8.797  10.811  1.00  0.00           C  \\nATOM    640  O   ALA A  83      -4.884   9.679  11.441  1.00  0.00           O  \\nATOM    641  CB  ALA A  83      -3.876   6.907   9.173  1.00  0.00           C  \\nATOM    642  N   GLY A  84      -5.136   9.301   9.677  1.00  0.00           N  \\nATOM    643  CA  GLY A  84      -5.325  10.687   9.299  1.00  0.00           C  \\nATOM    644  C   GLY A  84      -6.396  11.317   8.543  1.00  0.00           C  \\nATOM    645  O   GLY A  84      -7.467  11.569   8.921  1.00  0.00           O  \\nATOM    646  N   ILE A  85      -6.207  11.569   7.283  1.00  0.00           N  \\nATOM    647  CA  ILE A  85      -7.152  12.451   6.653  1.00  0.00           C  \\nATOM    648  C   ILE A  85      -6.774  13.963   6.905  1.00  0.00           C  \\nATOM    649  O   ILE A  85      -7.656  14.719   7.220  1.00  0.00           O  \\nATOM    650  CB  ILE A  85      -8.664  12.199   7.031  1.00  0.00           C  \\nATOM    651  CG1 ILE A  85      -9.168  10.813   6.149  1.00  0.00           C  \\nATOM    652  CG2 ILE A  85      -9.420  13.333   6.401  1.00  0.00           C  \\nATOM    653  CD1 ILE A  85     -10.176  11.191   5.204  1.00  0.00           C  \\nATOM    654  N   LYS A  86      -5.640  14.467   6.527  1.00  0.00           N  \\nATOM    655  CA  LYS A  86      -5.325  15.727   7.157  1.00  0.00           C  \\nATOM    656  C   LYS A  86      -5.136  16.546   5.771  1.00  0.00           C  \\nATOM    657  O   LYS A  86      -4.254  17.365   5.267  1.00  0.00           O  \\nATOM    658  CB  LYS A  86      -3.876  15.727   7.913  1.00  0.00           C  \\nATOM    659  CG  LYS A  86      -3.120  14.467   7.535  1.00  0.00           C  \\nATOM    660  CD  LYS A  86      -1.356  14.215   7.787  1.00  0.00           C  \\nATOM    661  CE  LYS A  86      -0.852  12.955   6.653  1.00  0.00           C  \\nATOM    662  NZ  LYS A  86       0.660  12.577   7.157  1.00  0.00           N  \\nATOM    663  N   LYS A  87      -5.892  16.357   5.015  1.00  0.00           N  \\nATOM    664  CA  LYS A  87      -5.892  17.239   3.755  1.00  0.00           C  \\nATOM    665  C   LYS A  87      -7.278  17.869   3.377  1.00  0.00           C  \\nATOM    666  O   LYS A  87      -8.160  17.239   2.873  1.00  0.00           O  \\nATOM    667  CB  LYS A  87      -5.514  16.294   2.558  1.00  0.00           C  \\nATOM    668  CG  LYS A  87      -4.632  16.735   1.235  1.00  0.00           C  \\nATOM    669  CD  LYS A  87      -4.254  15.727   0.353  1.00  0.00           C  \\nATOM    670  CE  LYS A  87      -3.120  16.231  -0.403  1.00  0.00           C  \\nATOM    671  NZ  LYS A  87      -2.868  15.727  -1.915  1.00  0.00           N  \\nATOM    672  N   LYS A  88      -7.278  19.129   3.251  1.00  0.00           N  \\nATOM    673  CA  LYS A  88      -8.412  19.885   2.621  1.00  0.00           C  \\nATOM    674  C   LYS A  88      -9.168  18.877   1.613  1.00  0.00           C  \\nATOM    675  O   LYS A  88     -10.428  18.499   1.613  1.00  0.00           O  \\nATOM    676  CB  LYS A  88      -7.782  21.019   1.991  1.00  0.00           C  \\nATOM    677  CG  LYS A  88      -6.270  20.767   1.865  1.00  0.00           C  \\nATOM    678  CD  LYS A  88      -5.514  21.271   0.605  1.00  0.00           C  \\nATOM    679  CE  LYS A  88      -5.766  20.767  -0.403  1.00  0.00           C  \\nATOM    680  NZ  LYS A  88      -5.388  21.523  -1.663  1.00  0.00           N  \\nATOM    681  N   GLY A  89      -8.538  18.499   0.353  1.00  0.00           N  \\nATOM    682  CA  GLY A  89      -9.168  17.491  -0.781  1.00  0.00           C  \\nATOM    683  C   GLY A  89     -10.050  16.357  -0.025  1.00  0.00           C  \\nATOM    684  O   GLY A  89     -11.184  16.357  -0.025  1.00  0.00           O  \\nATOM    685  N   GLU A  90      -9.546  15.475   0.479  1.00  0.00           N  \\nATOM    686  CA  GLU A  90     -10.050  14.089   1.109  1.00  0.00           C  \\nATOM    687  C   GLU A  90     -10.932  14.593   2.369  1.00  0.00           C  \\nATOM    688  O   GLU A  90     -12.192  14.341   2.621  1.00  0.00           O  \\nATOM    689  CB  GLU A  90      -9.042  13.207   1.613  1.00  0.00           C  \\nATOM    690  CG  GLU A  90      -8.286  12.577   0.605  1.00  0.00           C  \\nATOM    691  CD  GLU A  90      -6.900  11.821   0.857  1.00  0.00           C  \\nATOM    692  OE1 GLU A  90      -6.018  11.443  -0.277  1.00  0.00           O  \\nATOM    693  OE2 GLU A  90      -6.270  11.569   2.243  1.00  0.00           O  \\nATOM    694  N   ARG A  91     -10.554  15.475   2.999  1.00  0.00           N  \\nATOM    695  CA  ARG A  91     -11.184  15.916   4.511  1.00  0.00           C  \\nATOM    696  C   ARG A  91     -12.570  16.672   3.881  1.00  0.00           C  \\nATOM    697  O   ARG A  91     -13.578  16.357   4.133  1.00  0.00           O  \\nATOM    698  CB  ARG A  91     -10.302  16.987   5.141  1.00  0.00           C  \\nATOM    699  CG  ARG A  91     -10.554  17.113   6.527  1.00  0.00           C  \\nATOM    700  CD  ARG A  91      -9.420  18.121   7.283  1.00  0.00           C  \\nATOM    701  NE  ARG A  91      -9.294  19.381   6.527  1.00  0.00           N  \\nATOM    702  CZ  ARG A  91      -7.908  20.137   6.464  1.00  0.00           C  \\nATOM    703  NH1 ARG A  91      -6.900  19.507   7.535  1.00  0.00           N  \\nATOM    704  NH2 ARG A  91      -7.530  21.145   5.519  1.00  0.00           N  \\nATOM    705  N   GLN A  92     -12.444  17.743   3.251  1.00  0.00           N  \\nATOM    706  CA  GLN A  92     -13.830  18.373   2.369  1.00  0.00           C  \\nATOM    707  C   GLN A  92     -14.586  17.113   1.991  1.00  0.00           C  \\nATOM    708  O   GLN A  92     -15.720  16.735   2.243  1.00  0.00           O  \\nATOM    709  CB  GLN A  92     -13.704  19.633   1.739  1.00  0.00           C  \\nATOM    710  CG  GLN A  92     -13.830  19.255   0.164  1.00  0.00           C  \\nATOM    711  CD  GLN A  92     -13.200  20.326  -0.907  1.00  0.00           C  \\nATOM    712  OE1 GLN A  92     -11.688  19.885  -0.403  1.00  0.00           O  \\nATOM    713  NE2 GLN A  92     -13.452  21.460  -1.411  1.00  0.00           N  \\nATOM    714  N   ASP A  93     -13.956  16.357   0.983  1.00  0.00           N  \\nATOM    715  CA  ASP A  93     -14.838  15.223   0.605  1.00  0.00           C  \\nATOM    716  C   ASP A  93     -15.342  14.341   1.613  1.00  0.00           C  \\nATOM    717  O   ASP A  93     -16.602  13.837   1.487  1.00  0.00           O  \\nATOM    718  CB  ASP A  93     -13.830  14.593  -0.529  1.00  0.00           C  \\nATOM    719  CG  ASP A  93     -13.704  14.971  -1.915  1.00  0.00           C  \\nATOM    720  OD1 ASP A  93     -12.444  15.727  -2.419  1.00  0.00           O  \\nATOM    721  OD2 ASP A  93     -14.838  15.223  -2.797  1.00  0.00           O  \\nATOM    722  N   LEU A  94     -14.838  13.963   2.495  1.00  0.00           N  \\nATOM    723  CA  LEU A  94     -15.531  13.207   3.503  1.00  0.00           C  \\nATOM    724  C   LEU A  94     -16.728  13.963   3.881  1.00  0.00           C  \\nATOM    725  O   LEU A  94     -17.925  13.459   3.881  1.00  0.00           O  \\nATOM    726  CB  LEU A  94     -14.586  13.018   4.763  1.00  0.00           C  \\nATOM    727  CG  LEU A  94     -14.586  11.443   5.267  1.00  0.00           C  \\nATOM    728  CD1 LEU A  94     -15.783  10.813   4.637  1.00  0.00           C  \\nATOM    729  CD2 LEU A  94     -13.578  10.561   4.889  1.00  0.00           C  \\nATOM    730  N   VAL A  95     -16.539  15.097   4.637  1.00  0.00           N  \\nATOM    731  CA  VAL A  95     -17.736  15.979   5.141  1.00  0.00           C  \\nATOM    732  C   VAL A  95     -18.744  15.853   3.881  1.00  0.00           C  \\nATOM    733  O   VAL A  95     -20.004  15.601   4.322  1.00  0.00           O  \\nATOM    734  CB  VAL A  95     -17.358  17.365   5.393  1.00  0.00           C  \\nATOM    735  CG1 VAL A  95     -18.492  18.184   5.519  1.00  0.00           C  \\nATOM    736  CG2 VAL A  95     -16.476  17.491   6.401  1.00  0.00           C  \\nATOM    737  N   ALA A  96     -18.492  15.979   2.621  1.00  0.00           N  \\nATOM    738  CA  ALA A  96     -19.752  15.601   1.739  1.00  0.00           C  \\nATOM    739  C   ALA A  96     -20.130  14.215   1.802  1.00  0.00           C  \\nATOM    740  O   ALA A  96     -21.390  13.837   1.802  1.00  0.00           O  \\nATOM    741  CB  ALA A  96     -19.500  16.105   0.353  1.00  0.00           C  \\nATOM    742  N   TYR A  97     -19.248  13.333   1.865  1.00  0.00           N  \\nATOM    743  CA  TYR A  97     -19.626  11.884   1.865  1.00  0.00           C  \\nATOM    744  C   TYR A  97     -20.886  11.632   2.873  1.00  0.00           C  \\nATOM    745  O   TYR A  97     -22.146  11.317   2.621  1.00  0.00           O  \\nATOM    746  CB  TYR A  97     -18.744  10.813   2.243  1.00  0.00           C  \\nATOM    747  CG  TYR A  97     -19.374   9.427   1.739  1.00  0.00           C  \\nATOM    748  CD1 TYR A  97     -19.626   8.419   2.621  1.00  0.00           C  \\nATOM    749  CD2 TYR A  97     -19.626   9.175   0.353  1.00  0.00           C  \\nATOM    750  CE1 TYR A  97     -20.256   7.348   2.117  1.00  0.00           C  \\nATOM    751  CE2 TYR A  97     -20.256   8.041  -0.403  1.00  0.00           C  \\nATOM    752  CZ  TYR A  97     -20.508   7.285   0.857  1.00  0.00           C  \\nATOM    753  OH  TYR A  97     -21.264   6.403   0.353  1.00  0.00           O  \\nATOM    754  N   LEU A  98     -20.760  11.947   4.007  1.00  0.00           N  \\nATOM    755  CA  LEU A  98     -21.768  11.695   5.393  1.00  0.00           C  \\nATOM    756  C   LEU A  98     -23.154  12.703   5.015  1.00  0.00           C  \\nATOM    757  O   LEU A  98     -24.288  12.199   4.889  1.00  0.00           O  \\nATOM    758  CB  LEU A  98     -21.012  11.695   6.527  1.00  0.00           C  \\nATOM    759  CG  LEU A  98     -20.004  10.939   6.401  1.00  0.00           C  \\nATOM    760  CD1 LEU A  98     -19.374  10.687   7.661  1.00  0.00           C  \\nATOM    761  CD2 LEU A  98     -20.508   9.553   5.771  1.00  0.00           C  \\nATOM    762  N   LYS A  99     -23.028  13.963   4.889  1.00  0.00           N  \\nATOM    763  CA  LYS A  99     -24.288  14.593   4.385  1.00  0.00           C  \\nATOM    764  C   LYS A  99     -25.044  13.963   2.999  1.00  0.00           C  \\nATOM    765  O   LYS A  99     -26.304  13.837   3.125  1.00  0.00           O  \\nATOM    766  CB  LYS A  99     -23.910  16.105   4.259  1.00  0.00           C  \\nATOM    767  CG  LYS A  99     -24.918  17.113   4.259  1.00  0.00           C  \\nATOM    768  CD  LYS A  99     -24.666  18.625   3.881  1.00  0.00           C  \\nATOM    769  CE  LYS A  99     -25.674  19.633   4.133  1.00  0.00           C  \\nATOM    770  NZ  LYS A  99     -25.674  21.082   3.755  1.00  0.00           N  \\nATOM    771  N   SER A 100     -24.540  13.459   1.991  1.00  0.00           N  \\nATOM    772  CA  SER A 100     -25.296  12.577   0.857  1.00  0.00           C  \\nATOM    773  C   SER A 100     -25.548  11.191   1.361  1.00  0.00           C  \\nATOM    774  O   SER A 100     -26.934  10.813   1.361  1.00  0.00           O  \\nATOM    775  CB  SER A 100     -24.540  12.577  -0.655  1.00  0.00           C  \\nATOM    776  OG  SER A 100     -25.296  11.695  -1.663  1.00  0.00           O  \\nATOM    777  N   ALA A 101     -24.666  10.435   1.865  1.00  0.00           N  \\nATOM    778  CA  ALA A 101     -25.044   8.797   2.243  1.00  0.00           C  \\nATOM    779  C   ALA A 101     -25.800   8.923   3.629  1.00  0.00           C  \\nATOM    780  O   ALA A 101     -26.808   8.293   3.755  1.00  0.00           O  \\nATOM    781  CB  ALA A 101     -23.910   8.167   2.369  1.00  0.00           C  \\nATOM    782  N   THR A 102     -25.422   9.427   4.637  1.00  0.00           N  \\nATOM    783  CA  THR A 102     -26.052   9.301   6.149  1.00  0.00           C  \\nATOM    784  C   THR A 102     -27.690   9.805   5.897  1.00  0.00           C  \\nATOM    785  O   THR A 102     -28.572   9.049   6.149  1.00  0.00           O  \\nATOM    786  CB  THR A 102     -25.548  10.246   7.220  1.00  0.00           C  \\nATOM    787  OG1 THR A 102     -24.666   9.427   7.913  1.00  0.00           O  \\nATOM    788  CG2 THR A 102     -26.430  10.876   8.039  1.00  0.00           C  \\nATOM    789  N   SER A 103     -27.942  10.939   5.645  1.00  0.00           N  \\nATOM    790  CA  SER A 103     -29.454  11.317   5.519  1.00  0.00           C  \\nATOM    791  C   SER A 103     -30.084  10.813   4.385  1.00  0.00           C  \\nATOM    792  O   SER A 103     -29.454   9.931   3.377  1.00  0.00           O  \\nATOM    793  CB  SER A 103     -29.580  12.892   5.897  1.00  0.00           C  \\nATOM    794  OG  SER A 103     -28.320  13.585   5.393  1.00  0.00           O  \\nATOM    795  OXT SER A 103     -31.344  11.317   4.133  1.00  0.00           O  \\nTER     796      SER A 103                                                      \\nHETATM 1593 FE   HEC A 104     -15.216   5.773  12.701  1.00  0.00          FE  \\nHETATM 1594  CHA HEC A 104     -16.980   6.466  15.347  1.00  0.00           C  \\nHETATM 1595  CHB HEC A 104     -16.476   8.293  11.189  1.00  0.00           C  \\nHETATM 1596  CHC HEC A 104     -13.578   5.017   9.929  1.00  0.00           C  \\nHETATM 1597  CHD HEC A 104     -13.956   3.127  14.339  1.00  0.00           C  \\nHETATM 1598  NA  HEC A 104     -16.476   7.096  13.205  1.00  0.00           N  \\nHETATM 1599  C1A HEC A 104     -17.106   7.159  14.465  1.00  0.00           C  \\nHETATM 1600  C2A HEC A 104     -17.862   8.167  14.591  1.00  0.00           C  \\nHETATM 1601  C3A HEC A 104     -17.736   8.797  13.205  1.00  0.00           C  \\nHETATM 1602  C4A HEC A 104     -17.114   8.167  12.323  1.00  0.00           C  \\nHETATM 1603  CMA HEC A 104     -18.303  10.057  12.701  1.00  0.00           C  \\nHETATM 1604  CAA HEC A 104     -18.618   8.167  15.599  1.00  0.00           C  \\nHETATM 1605  CBA HEC A 104     -20.067   8.734  15.473  1.00  0.00           C  \\nHETATM 1606  CGA HEC A 104     -20.823   8.734  16.859  1.00  0.00           C  \\nHETATM 1607  O1A HEC A 104     -21.705   8.041  17.111  1.00  0.00           O  \\nHETATM 1608  O2A HEC A 104     -20.508   9.490  17.741  1.00  0.00           O  \\nHETATM 1609  NB  HEC A 104     -15.090   6.466  11.126  1.00  0.00           N  \\nHETATM 1610  C1B HEC A 104     -15.594   7.348  10.559  1.00  0.00           C  \\nHETATM 1611  C2B HEC A 104     -15.216   7.663   9.299  1.00  0.00           C  \\nHETATM 1612  C3B HEC A 104     -14.334   6.592   8.921  1.00  0.00           C  \\nHETATM 1613  C4B HEC A 104     -14.334   6.025   9.929  1.00  0.00           C  \\nHETATM 1614  CMB HEC A 104     -15.657   8.797   8.543  1.00  0.00           C  \\nHETATM 1615  CAB HEC A 104     -13.578   6.781   7.409  1.00  0.00           C  \\nHETATM 1616  CBB HEC A 104     -12.570   7.663   7.661  1.00  0.00           C  \\nHETATM 1617  NC  HEC A 104     -13.956   4.387  12.197  1.00  0.00           N  \\nHETATM 1618  C1C HEC A 104     -13.452   4.261  10.811  1.00  0.00           C  \\nHETATM 1619  C2C HEC A 104     -12.759   3.001  10.811  1.00  0.00           C  \\nHETATM 1620  C3C HEC A 104     -12.948   2.623  11.945  1.00  0.00           C  \\nHETATM 1621  C4C HEC A 104     -13.704   3.379  12.953  1.00  0.00           C  \\nHETATM 1622  CMC HEC A 104     -11.688   2.497   9.803  1.00  0.00           C  \\nHETATM 1623  CAC HEC A 104     -12.444   1.111  12.197  1.00  0.00           C  \\nHETATM 1624  CBC HEC A 104     -11.184   0.859  12.008  1.00  0.00           C  \\nHETATM 1625  ND  HEC A 104     -15.405   5.143  14.213  1.00  0.00           N  \\nHETATM 1626  C1D HEC A 104     -14.838   4.009  14.843  1.00  0.00           C  \\nHETATM 1627  C2D HEC A 104     -15.216   3.757  15.977  1.00  0.00           C  \\nHETATM 1628  C3D HEC A 104     -16.098   4.765  16.355  1.00  0.00           C  \\nHETATM 1629  C4D HEC A 104     -16.161   5.395  15.347  1.00  0.00           C  \\nHETATM 1630  CMD HEC A 104     -14.838   2.749  16.733  1.00  0.00           C  \\nHETATM 1631  CAD HEC A 104     -16.854   4.513  17.615  1.00  0.00           C  \\nHETATM 1632  CBD HEC A 104     -16.098   4.135  18.623  1.00  0.00           C  \\nHETATM 1633  CGD HEC A 104     -15.216   5.017  19.505  1.00  0.00           C  \\nHETATM 1634  O1D HEC A 104     -14.082   5.773  18.560  1.00  0.00           O  \\nHETATM 1635  O2D HEC A 104     -15.405   5.647  20.639  1.00  0.00           O  \\nHETATM 1679  O   HOH A 105     -16.496   9.304  17.402  1.00  0.00           O  \\nCONECT  104 1615                                                                \\nCONECT  124 1623                                                                \\nCONECT  134 1593                                                                \\nCONECT  356 1635                                                                \\nCONECT  616 1593                                                                \\nCONECT  900 1658                                                                \\nCONECT  920 1666                                                                \\nCONECT  930 1636                                                                \\nCONECT 1152 1678                                                                \\nCONECT 1412 1636                                                                \\nCONECT 1593  134  616 1598 1609                                                 \\nCONECT 1593 1617 1625                                                           \\nCONECT 1594 1599 1629                                                           \\nCONECT 1595 1602 1610                                                           \\nCONECT 1596 1613 1618                                                           \\nCONECT 1597 1621 1626                                                           \\nCONECT 1598 1593 1599 1602                                                      \\nCONECT 1599 1594 1598 1600                                                      \\nCONECT 1600 1599 1601 1604                                                      \\nCONECT 1601 1600 1602 1603                                                      \\nCONECT 1602 1595 1598 1601                                                      \\nCONECT 1603 1601                                                                \\nCONECT 1604 1600 1605                                                           \\nCONECT 1605 1604 1606                                                           \\nCONECT 1606 1605 1607 1608                                                      \\nCONECT 1607 1606                                                                \\nCONECT 1608 1606                                                                \\nCONECT 1609 1593 1610 1613                                                      \\nCONECT 1610 1595 1609 1611                                                      \\nCONECT 1611 1610 1612 1614                                                      \\nCONECT 1612 1611 1613 1615                                                      \\nCONECT 1613 1596 1609 1612                                                      \\nCONECT 1614 1611                                                                \\nCONECT 1615  104 1612 1616                                                      \\nCONECT 1616 1615                                                                \\nCONECT 1617 1593 1618 1621                                                      \\nCONECT 1618 1596 1617 1619                                                      \\nCONECT 1619 1618 1620 1622                                                      \\nCONECT 1620 1619 1621 1623                                                      \\nCONECT 1621 1597 1617 1620                                                      \\nCONECT 1622 1619                                                                \\nCONECT 1623  124 1620 1624                                                      \\nCONECT 1624 1623                                                                \\nCONECT 1625 1593 1626 1629                                                      \\nCONECT 1626 1597 1625 1627                                                      \\nCONECT 1627 1626 1628 1630                                                      \\nCONECT 1628 1627 1629 1631                                                      \\nCONECT 1629 1594 1625 1628                                                      \\nCONECT 1630 1627                                                                \\nCONECT 1631 1628 1632                                                           \\nCONECT 1632 1631 1633                                                           \\nCONECT 1633 1632 1634 1635                                                      \\nCONECT 1634 1633                                                                \\nCONECT 1635  356 1633                                                           \\nCONECT 1636  930 1412 1641 1652                                                 \\nCONECT 1636 1660 1668                                                           \\nCONECT 1637 1642 1672                                                           \\nCONECT 1638 1645 1653                                                           \\nCONECT 1639 1656 1661                                                           \\nCONECT 1640 1664 1669                                                           \\nCONECT 1641 1636 1642 1645                                                      \\nCONECT 1642 1637 1641 1643                                                      \\nCONECT 1643 1642 1644 1647                                                      \\nCONECT 1644 1643 1645 1646                                                      \\nCONECT 1645 1638 1641 1644                                                      \\nCONECT 1646 1644                                                                \\nCONECT 1647 1643 1648                                                           \\nCONECT 1648 1647 1649                                                           \\nCONECT 1649 1648 1650 1651                                                      \\nCONECT 1650 1649                                                                \\nCONECT 1651 1649                                                                \\nCONECT 1652 1636 1653 1656                                                      \\nCONECT 1653 1638 1652 1654                                                      \\nCONECT 1654 1653 1655 1657                                                      \\nCONECT 1655 1654 1656 1658                                                      \\nCONECT 1656 1639 1652 1655                                                      \\nCONECT 1657 1654                                                                \\nCONECT 1658  900 1655 1659                                                      \\nCONECT 1659 1658                                                                \\nCONECT 1660 1636 1661 1664                                                      \\nCONECT 1661 1639 1660 1662                                                      \\nCONECT 1662 1661 1663 1665                                                      \\nCONECT 1663 1662 1664 1666                                                      \\nCONECT 1664 1640 1660 1663                                                      \\nCONECT 1665 1662                                                                \\nCONECT 1666  920 1663 1667                                                      \\nCONECT 1667 1666                                                                \\nCONECT 1668 1636 1669 1672                                                      \\nCONECT 1669 1640 1668 1670                                                      \\nCONECT 1670 1669 1671 1673                                                      \\nCONECT 1671 1670 1672 1674                                                      \\nCONECT 1672 1637 1668 1671                                                      \\nCONECT 1673 1670                                                                \\nCONECT 1674 1671 1675                                                           \\nCONECT 1675 1674 1676                                                           \\nCONECT 1676 1675 1677 1678                                                      \\nCONECT 1677 1676                                                                \\nCONECT 1678 1152 1676                                                           \\nMASTER      287    0    2   10    0    0    9    9 1677    2   98   16          \\nEND                                                                             \\n\",\"pdb\");\n",
       "\tviewer_16742555222774558.setBackgroundColor(\"white\");\n",
       "\tviewer_16742555222774558.setStyle({\"cartoon\": {\"color\": \"white\"}});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"2\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"5\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"2\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"6\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"2\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"3\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"96\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"3\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"101\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"4\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"8\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"5\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"7\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"5\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"8\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"5\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"5\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"6\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"8\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"6\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"6\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"6\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"95\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"6\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"7\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"11\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"7\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"97\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"7\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"100\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"8\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"8\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"11\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"8\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"12\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"8\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"15\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"8\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"8\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"97\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"9\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"9\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"85\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"9\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"11\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"12\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"13\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"13\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"83\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"13\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"16\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"18\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"32\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"15\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"15\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"18\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"15\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"19\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"27\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"28\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"29\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"80\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"81\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"82\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"18\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"18\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"24\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"18\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"26\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"18\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"28\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"18\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"32\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"19\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"26\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"19\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"32\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"22\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"23\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"24\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"32\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"101\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"21\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"33\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"22\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"31\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"22\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"33\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"23\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"32\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"23\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"33\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"24\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"26\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"25\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"27\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"25\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"30\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"25\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"31\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"26\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"28\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"27\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"30\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"27\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"31\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"28\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"81\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"29\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"32\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"30\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"32\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"30\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"33\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"30\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"43\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"30\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"44\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"30\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"45\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"30\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"46\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"31\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"33\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"31\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"43\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"31\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"44\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"32\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"34\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"32\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"35\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"32\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"43\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"32\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"59\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"32\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"32\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"101\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"32\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"102\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"33\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"38\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"33\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"43\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"33\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"44\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"33\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"103\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"34\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"35\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"37\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"35\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"38\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"35\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"59\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"35\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"61\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"35\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"64\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"35\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"35\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"103\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"36\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"64\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"36\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"37\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"39\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"38\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"57\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"41\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"42\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"52\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"53\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"56\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"39\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"59\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"40\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"46\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"41\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"46\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"41\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"48\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"41\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"57\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"42\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"46\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"42\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"53\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"43\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"45\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"43\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"46\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"43\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"52\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"44\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"46\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"46\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"48\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"46\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"52\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"46\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"53\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"47\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"79\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"48\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"50\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"48\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"52\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"48\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"53\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"48\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"54\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"48\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"78\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"48\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"79\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"49\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"51\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"49\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"53\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"50\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"75\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"51\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"56\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"51\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"75\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"51\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"77\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"51\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"80\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"52\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"55\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"52\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"79\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"53\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"55\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"54\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"57\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"54\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"75\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"55\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"74\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"55\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"76\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"56\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"75\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"57\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"59\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"57\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"75\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"59\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"64\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"59\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"66\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"59\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"60\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"62\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"60\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"64\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"60\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"66\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"61\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"64\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"61\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"95\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"61\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"99\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"62\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"64\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"62\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"66\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"62\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"95\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"63\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"68\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"64\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"66\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"64\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"67\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"64\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"68\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"64\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"69\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"64\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"91\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"64\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"64\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"95\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"64\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"97\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"64\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"64\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"99\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"65\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"68\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"65\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"65\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"95\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"66\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"73\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"66\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"74\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"67\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"68\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"70\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"68\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"71\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"68\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"74\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"68\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"84\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"68\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"85\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"68\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"86\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"68\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"90\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"68\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"91\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"68\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"92\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"68\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"68\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"95\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"68\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"69\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"72\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"69\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"85\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"70\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"72\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"70\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"76\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"70\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"85\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"71\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"74\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"71\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"76\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"72\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"76\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"72\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"84\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"73\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"75\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"74\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"76\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"75\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"77\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"78\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"81\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"79\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"81\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"80\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"82\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"81\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"83\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"82\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"85\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"83\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"85\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"84\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"90\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"85\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"87\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"85\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"89\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"85\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"90\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"85\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"91\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"85\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"86\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"90\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"87\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"89\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"87\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"92\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"87\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"88\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"92\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"88\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"89\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"89\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"90\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"91\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"91\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"95\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"92\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"92\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"95\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"93\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"97\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"96\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"97\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"95\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"96\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"96\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"101\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"100\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"101\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"102\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"103\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555222774558.zoomTo();\n",
       "viewer_16742555222774558.render();\n",
       "});\n",
       "</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "<py3Dmol.view at 0x7f127a2bd4f0>"
      ]
     },
     "execution_count": 49,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "view_frustration(pdb_file_chain,frustra_all)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_16742555228337302\"  style=\"position: relative; width: 640px; height: 480px\">\n        <p id=\"3dmolwarning_16742555228337302\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    //this is to ignore the existence of requirejs amd\n    var savedexports, savedmodule;\n    if (typeof exports !== 'undefined') savedexports = exports;\n    else exports = {}\n    if (typeof module !== 'undefined') savedmodule = module;\n    else module = {}\n\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n        exports = savedexports;\n        module = savedmodule;\n        resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://3dmol.org/build/3Dmol.js');\n}\n\nvar viewer_16742555228337302 = null;\nvar warn = document.getElementById(\"3dmolwarning_16742555228337302\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_16742555228337302 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_16742555228337302\"),{backgroundColor:\"white\"});\nviewer_16742555228337302.zoomTo();\n\tviewer_16742555228337302.addModel(\"HEADER    ELECTRON TRANSPORT                      01-AUG-76   1CYC              \\nTITLE     THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C AT 2.3     \\nTITLE    2 ANGSTROMS RESOLUTION. II. STRUCTURE AND FUNCTION                     \\nCAVEAT     1CYC    THR A 40 HAS WRONG CHIRALITY AT ATOM CB THR A 78 HAS WRONG   \\nCAVEAT   2 1CYC    CHIRALITY AT ATOM CB THR B 40 HAS WRONG CHIRALITY AT ATOM    \\nCAVEAT   3 1CYC    CB THR B 78 HAS WRONG CHIRALITY AT ATOM CB                   \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: FERROCYTOCHROME C;                                         \\nCOMPND   3 CHAIN: A, B;                                                         \\nCOMPND   4 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 ORGANISM_SCIENTIFIC: KATSUWONUS PELAMIS;                             \\nSOURCE   3 ORGANISM_COMMON: SKIPJACK TUNA;                                      \\nSOURCE   4 ORGANISM_TAXID: 8226                                                 \\nKEYWDS    ELECTRON TRANSPORT                                                    \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    N.TANAKA,T.YAMANE,T.TSUKIHARA,T.ASHIDA,M.KAKUDO                       \\nREVDAT  17   03-MAR-21 1CYC    1       CAVEAT COMPND REMARK HET                 \\nREVDAT  17 2                   1       HETNAM HETSYN FORMUL LINK                \\nREVDAT  17 3                   1       ATOM                                     \\nREVDAT  16   29-NOV-17 1CYC    1       HELIX                                    \\nREVDAT  15   24-FEB-09 1CYC    1       VERSN                                    \\nREVDAT  14   01-APR-03 1CYC    1       JRNL                                     \\nREVDAT  13   27-JAN-84 1CYC    1       REMARK                                   \\nREVDAT  12   30-SEP-83 1CYC    1       REVDAT                                   \\nREVDAT  11   19-APR-83 1CYC    1       COMPND SOURCE REMARK                     \\nREVDAT  10   15-JAN-82 1CYC    1       REMARK                                   \\nREVDAT   9   06-JAN-82 1CYC    3       SCALE                                    \\nREVDAT   8   31-DEC-80 1CYC    1       REMARK                                   \\nREVDAT   7   07-APR-80 1CYC    3       SOURCE SEQRES ATOM                       \\nREVDAT   6   23-APR-79 1CYC    3       ATOM                                     \\nREVDAT   5   01-NOV-77 1CYC    1       AUTHOR REMARK FORMUL                     \\nREVDAT   4   09-SEP-77 1CYC    3       ATOM                                     \\nREVDAT   3   13-JUN-77 1CYC    1       HET                                      \\nREVDAT   2   25-OCT-76 1CYC    2       CONECT                                   \\nREVDAT   1   06-OCT-76 1CYC    0                                                \\nJRNL        AUTH   N.TANAKA,T.YAMANE,T.TSUKIHARA,T.ASHIDA,M.KAKUDO              \\nJRNL        TITL   THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C   \\nJRNL        TITL 2 AT 2.3 A RESOLUTION. II. STRUCTURE AND FUNCTION.             \\nJRNL        REF    J.BIOCHEM.(TOKYO)             V.  77   147 1975              \\nJRNL        REFN                   ISSN 0021-924X                               \\nJRNL        PMID   166072                                                       \\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   T.TSUKIHARA,T.YAMANE,N.TANAKA,T.ASHIDA,M.KAKUDO              \\nREMARK   1  TITL   OXIDATION OF A FERROCYTOCHROME C IN THE CRYSTALLINE          \\nREMARK   1  TITL 2 STATE-STRUCTURAL CHANGE AND ANION BINDING                    \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  73  1163 1973              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 2                                                          \\nREMARK   1  AUTH   T.ASHIDA,N.TANAKA,T.YAMANE,T.TSUKIHARA,M.KAKUDO              \\nREMARK   1  TITL   THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C   \\nREMARK   1  TITL 2 AT 2.3 ANGSTROMS RESOLUTION                                  \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  73   463 1973              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 3                                                          \\nREMARK   1  AUTH   T.ASHIDA,T.UEKI,T.TSUKIHARA,A.SUGIHARA,T.TAKANO,M.KAKUDO     \\nREMARK   1  TITL   THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C   \\nREMARK   1  TITL 2 AT 4 ANGSTROMS RESOLUTION                                    \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  70   913 1971              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 4                                                          \\nREMARK   1  AUTH   T.NAKAYAMA,K.TITANI,K.NARITA                                 \\nREMARK   1  TITL   THE AMINO ACID SEQUENCE OF CYTOCHROME C FROM BONITO          \\nREMARK   1  TITL 2 (KATSUWONUS PELAMIS, LINNAEUS)                               \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  70   311 1971              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 5                                                          \\nREMARK   1  EDIT   R.J.FELDMANN                                                 \\nREMARK   1  REF    ATLAS OF MACROMOLECULAR                120 1976              \\nREMARK   1  REF  2 STRUCTURE ON MICROFICHE                                      \\nREMARK   1  PUBL   TRACOR JITCO INC.,ROCKVILLE,MD.                              \\nREMARK   1  REFN                                                                \\nREMARK   1 REFERENCE 6                                                          \\nREMARK   1  EDIT   M.O.DAYHOFF                                                  \\nREMARK   1  REF    ATLAS OF PROTEIN SEQUENCE     V.   5    39 1976              \\nREMARK   1  REF  2 AND STRUCTURE,SUPPLEMENT 2                                   \\nREMARK   1  PUBL   NATIONAL BIOMEDICAL RESEARCH FOUNDATION, SILVER SPRING,MD.   \\nREMARK   1  REFN                   ISSN 0-912466-05-7                           \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    2.30 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : NULL                                                 \\nREMARK   3   AUTHORS     : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 2.30                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : NULL                           \\nREMARK   3   DATA CUTOFF            (SIGMA(F)) : NULL                           \\nREMARK   3   DATA CUTOFF HIGH         (ABS(F)) : NULL                           \\nREMARK   3   DATA CUTOFF LOW          (ABS(F)) : NULL                           \\nREMARK   3   COMPLETENESS (WORKING+TEST)   (%) : NULL                           \\nREMARK   3   NUMBER OF REFLECTIONS             : NULL                           \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   CROSS-VALIDATION METHOD          : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SELECTION  : NULL                            \\nREMARK   3   R VALUE            (WORKING SET) : NULL                            \\nREMARK   3   FREE R VALUE                     : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : NULL                            \\nREMARK   3   FREE R VALUE TEST SET COUNT      : NULL                            \\nREMARK   3   ESTIMATED ERROR OF FREE R VALUE  : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  FIT IN THE HIGHEST RESOLUTION BIN.                                  \\nREMARK   3   TOTAL NUMBER OF BINS USED           : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE HIGH       (A) : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE LOW        (A) : NULL                         \\nREMARK   3   BIN COMPLETENESS (WORKING+TEST) (%) : NULL                         \\nREMARK   3   REFLECTIONS IN BIN    (WORKING SET) : NULL                         \\nREMARK   3   BIN R VALUE           (WORKING SET) : NULL                         \\nREMARK   3   BIN FREE R VALUE                    : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET SIZE  (%) : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET COUNT     : NULL                         \\nREMARK   3   ESTIMATED ERROR OF BIN FREE R VALUE : NULL                         \\nREMARK   3                                                                      \\nREMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \\nREMARK   3   PROTEIN ATOMS            : 1590                                    \\nREMARK   3   NUCLEIC ACID ATOMS       : 0                                       \\nREMARK   3   HETEROGEN ATOMS          : 86                                      \\nREMARK   3   SOLVENT ATOMS            : 1                                       \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : NULL                           \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : NULL                           \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : NULL                                                 \\nREMARK   3    B22 (A**2) : NULL                                                 \\nREMARK   3    B33 (A**2) : NULL                                                 \\nREMARK   3    B12 (A**2) : NULL                                                 \\nREMARK   3    B13 (A**2) : NULL                                                 \\nREMARK   3    B23 (A**2) : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  ESTIMATED COORDINATE ERROR.                                         \\nREMARK   3   ESD FROM LUZZATI PLOT        (A) : NULL                            \\nREMARK   3   ESD FROM SIGMAA              (A) : NULL                            \\nREMARK   3   LOW RESOLUTION CUTOFF        (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  CROSS-VALIDATED ESTIMATED COORDINATE ERROR.                         \\nREMARK   3   ESD FROM C-V LUZZATI PLOT    (A) : NULL                            \\nREMARK   3   ESD FROM C-V SIGMAA          (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  RMS DEVIATIONS FROM IDEAL VALUES.                                   \\nREMARK   3   BOND LENGTHS                 (A) : NULL                            \\nREMARK   3   BOND ANGLES            (DEGREES) : NULL                            \\nREMARK   3   DIHEDRAL ANGLES        (DEGREES) : NULL                            \\nREMARK   3   IMPROPER ANGLES        (DEGREES) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL MODEL : NULL                                      \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL FACTOR RESTRAINTS.    RMS    SIGMA                \\nREMARK   3   MAIN-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   MAIN-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  NCS MODEL : NULL                                                    \\nREMARK   3                                                                      \\nREMARK   3  NCS RESTRAINTS.                         RMS   SIGMA/WEIGHT          \\nREMARK   3   GROUP  1  POSITIONAL            (A) : NULL  ; NULL                 \\nREMARK   3   GROUP  1  B-FACTOR           (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  PARAMETER FILE  1  : NULL                                           \\nREMARK   3  TOPOLOGY FILE  1   : NULL                                           \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS: NULL                                      \\nREMARK   4                                                                      \\nREMARK   4 1CYC COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY BNL.                                \\nREMARK 100 THE DEPOSITION ID IS D_1000172595.                                   \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : NULL                               \\nREMARK 200  TEMPERATURE           (KELVIN) : NULL                               \\nREMARK 200  PH                             : NULL                               \\nREMARK 200  NUMBER OF CRYSTALS USED        : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : NULL                               \\nREMARK 200  RADIATION SOURCE               : NULL                               \\nREMARK 200  BEAMLINE                       : NULL                               \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : NULL                               \\nREMARK 200  WAVELENGTH OR RANGE        (A) : NULL                               \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : NULL                               \\nREMARK 200  DETECTOR MANUFACTURER          : NULL                               \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : NULL                               \\nREMARK 200  DATA SCALING SOFTWARE          : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : NULL                               \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : NULL                               \\nREMARK 200  RESOLUTION RANGE LOW       (A) : NULL                               \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY                : NULL                               \\nREMARK 200  R MERGE                    (I) : NULL                               \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : NULL                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : NULL                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : NULL                               \\nREMARK 200  R MERGE FOR SHELL          (I) : NULL                               \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: NULL                                           \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: NULL                         \\nREMARK 200 SOFTWARE USED: NULL                                                  \\nREMARK 200 STARTING MODEL: NULL                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: NULL                                                         \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): 39.18                                     \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 2.02                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: NULL                                     \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: P 21 21 21                       \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -X+1/2,-Y,Z+1/2                                         \\nREMARK 290       3555   -X,Y+1/2,-Z+1/2                                         \\nREMARK 290       4555   X+1/2,-Y+1/2,-Z                                         \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -1.000000  0.000000  0.000000       28.84000            \\nREMARK 290   SMTRY2   2  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000       18.91500            \\nREMARK 290   SMTRY1   3 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   3  0.000000  1.000000  0.000000       42.29000            \\nREMARK 290   SMTRY3   3  0.000000  0.000000 -1.000000       18.91500            \\nREMARK 290   SMTRY1   4  1.000000  0.000000  0.000000       28.84000            \\nREMARK 290   SMTRY2   4  0.000000 -1.000000  0.000000       42.29000            \\nREMARK 290   SMTRY3   4  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1, 2                                                    \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: MONOMERIC                         \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 2                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: MONOMERIC                         \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: B                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 470                                                                      \\nREMARK 470 MISSING ATOM                                                         \\nREMARK 470 THE FOLLOWING RESIDUES HAVE MISSING ATOMS (M=MODEL NUMBER;           \\nREMARK 470 RES=RESIDUE NAME; C=CHAIN IDENTIFIER; SSEQ=SEQUENCE NUMBER;          \\nREMARK 470 I=INSERTION CODE):                                                   \\nREMARK 470   M RES CSSEQI  ATOMS                                                \\nREMARK 470     ASN A  22    CB   CG   OD1  ND2                                  \\nREMARK 470     LYS A  39    CE   NZ                                             \\nREMARK 470     ASN B  22    CB   CG   OD1  ND2                                  \\nREMARK 470     LYS B  39    CE   NZ                                             \\nREMARK 620                                                                      \\nREMARK 620 METAL COORDINATION                                                   \\nREMARK 620 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 620 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE):                             \\nREMARK 620                                                                      \\nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \\nREMARK 620                             HEC A 104  FE                            \\nREMARK 620 N RES CSSEQI ATOM                                                    \\nREMARK 620 1 HIS A  18   NE2                                                    \\nREMARK 620 2 HEC A 104   NA   87.0                                              \\nREMARK 620 3 HEC A 104   NB   87.8  90.6                                        \\nREMARK 620 4 HEC A 104   NC   92.0 178.7  90.1                                  \\nREMARK 620 5 HEC A 104   ND   91.1  86.9 177.4  92.3                            \\nREMARK 620 6 MET A  80   SD  170.9 102.1  93.0  79.0  88.6                      \\nREMARK 620 N                    1     2     3     4     5                       \\nREMARK 620                                                                      \\nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \\nREMARK 620                             HEC B 104  FE                            \\nREMARK 620 N RES CSSEQI ATOM                                                    \\nREMARK 620 1 HIS B  18   NE2                                                    \\nREMARK 620 2 HEC B 104   NA   87.0                                              \\nREMARK 620 3 HEC B 104   NB   87.8  90.6                                        \\nREMARK 620 4 HEC B 104   NC   92.0 178.7  90.1                                  \\nREMARK 620 5 HEC B 104   ND   91.1  86.9 177.4  92.3                            \\nREMARK 620 6 MET B  80   SD  170.9 102.1  93.0  79.0  88.6                      \\nREMARK 620 N                    1     2     3     4     5                       \\nREMARK 800                                                                      \\nREMARK 800 SITE                                                                 \\nREMARK 800 SITE_IDENTIFIER: AC1                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE HEC A 104                 \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: AC2                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE HEC B 104                 \\nDBREF  1CYC A    1   103  UNP    P00025   CYC_KATPE        1    103             \\nDBREF  1CYC B    1   103  UNP    P00025   CYC_KATPE        1    103             \\nSEQRES   1 A  103  GLY ASP VAL ALA LYS GLY LYS LYS THR PHE VAL GLN LYS          \\nSEQRES   2 A  103  CYS ALA GLN CYS HIS THR VAL GLU ASN GLY GLY LYS HIS          \\nSEQRES   3 A  103  LYS VAL GLY PRO ASN LEU TRP GLY LEU PHE GLY ARG LYS          \\nSEQRES   4 A  103  THR GLY GLN ALA GLU GLY TYR SER TYR THR ASP ALA ASN          \\nSEQRES   5 A  103  LYS SER LYS GLY ILE VAL TRP ASN GLU ASN THR LEU MET          \\nSEQRES   6 A  103  GLU TYR LEU GLU ASN PRO LYS LYS TYR ILE PRO GLY THR          \\nSEQRES   7 A  103  LYS MET ILE PHE ALA GLY ILE LYS LYS LYS GLY GLU ARG          \\nSEQRES   8 A  103  GLN ASP LEU VAL ALA TYR LEU LYS SER ALA THR SER              \\nSEQRES   1 B  103  GLY ASP VAL ALA LYS GLY LYS LYS THR PHE VAL GLN LYS          \\nSEQRES   2 B  103  CYS ALA GLN CYS HIS THR VAL GLU ASN GLY GLY LYS HIS          \\nSEQRES   3 B  103  LYS VAL GLY PRO ASN LEU TRP GLY LEU PHE GLY ARG LYS          \\nSEQRES   4 B  103  THR GLY GLN ALA GLU GLY TYR SER TYR THR ASP ALA ASN          \\nSEQRES   5 B  103  LYS SER LYS GLY ILE VAL TRP ASN GLU ASN THR LEU MET          \\nSEQRES   6 B  103  GLU TYR LEU GLU ASN PRO LYS LYS TYR ILE PRO GLY THR          \\nSEQRES   7 B  103  LYS MET ILE PHE ALA GLY ILE LYS LYS LYS GLY GLU ARG          \\nSEQRES   8 B  103  GLN ASP LEU VAL ALA TYR LEU LYS SER ALA THR SER              \\nHET    HEC  A 104      43                                                       \\nHET    HEC  B 104      43                                                       \\nHETNAM     HEC HEME C                                                           \\nFORMUL   3  HEC    2(C34 H34 FE N4 O4)                                          \\nFORMUL   5  HOH   *(H2 O)                                                       \\nHELIX    1  H1 GLY A    1  VAL A   11  1                                  11    \\nHELIX    2  H4 CYS A   14  CYS A   17  514 AND 17 BOUND TO HEME GROUP      4    \\nHELIX    3  H5 THR A   49  LYS A   55  5LOOSE FROM 49-53,3/10 53-55        7    \\nHELIX    4  H2 ASN A   60  GLU A   69  1                                  10    \\nHELIX    5  H3 GLU A   90  SER A  103  1                                  14    \\nHELIX    6  H6 GLY B    1  VAL B   11  1                                  11    \\nHELIX    7  H7 CYS B   14  CYS B   17  514 AND 17 BOUND TO HEME GROUP      4    \\nHELIX    8  H8 THR B   49  LYS B   55  5LOOSE FROM 49-53,3/10 53-55        7    \\nHELIX    9  H9 ASN B   60  GLU B   69  1                                  10    \\nHELIX   10 H10 GLU B   90  SER B  103  1                                  14    \\nLINK         SG  CYS A  14                 CAB HEC A 104     1555   1555  1.48  \\nLINK         SG  CYS A  17                 CAC HEC A 104     1555   1555  1.86  \\nLINK         CE2 TYR A  48                 O2D HEC A 104     1555   1555  2.02  \\nLINK         SG  CYS B  14                 CAB HEC B 104     1555   1555  1.48  \\nLINK         SG  CYS B  17                 CAC HEC B 104     1555   1555  1.86  \\nLINK         CE2 TYR B  48                 O2D HEC B 104     1555   1555  2.02  \\nLINK         NE2 HIS A  18                FE   HEC A 104     1555   1555  2.60  \\nLINK         SD  MET A  80                FE   HEC A 104     1555   1555  2.49  \\nLINK         NE2 HIS B  18                FE   HEC B 104     1555   1555  2.60  \\nLINK         SD  MET B  80                FE   HEC B 104     1555   1555  2.49  \\nSITE     1 AC1 17 LYS A  13  CYS A  14  CYS A  17  HIS A  18                    \\nSITE     2 AC1 17 LEU A  32  LEU A  35  ARG A  38  THR A  40                    \\nSITE     3 AC1 17 TYR A  46  TYR A  48  ASN A  52  TRP A  59                    \\nSITE     4 AC1 17 THR A  78  LYS A  79  MET A  80  PHE A  82                    \\nSITE     5 AC1 17 HOH A 105                                                     \\nSITE     1 AC2 16 LYS B  13  CYS B  14  CYS B  17  HIS B  18                    \\nSITE     2 AC2 16 LEU B  32  LEU B  35  ARG B  38  THR B  40                    \\nSITE     3 AC2 16 TYR B  46  TYR B  48  ASN B  52  TRP B  59                    \\nSITE     4 AC2 16 THR B  78  LYS B  79  MET B  80  PHE B  82                    \\nCRYST1   57.680   84.580   37.830  90.00  90.00  90.00 P 21 21 21    8          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.017337  0.000000  0.000000        0.00000                         \\nSCALE2      0.000000  0.011823  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.026434        0.00000                         \\nMTRIX1   1  1.000000  0.000000  0.000000       28.84000    1                    \\nMTRIX2   1  0.000000  1.000000  0.000000        0.00000    1                    \\nMTRIX3   1  0.000000  0.000000  1.000000        0.00000    1                    \\nATOM     25  N   LYS A   5     -16.476  10.687  -5.191  1.00  0.00           N  \\nATOM     26  CA  LYS A   5     -15.216  10.498  -4.372  1.00  0.00           C  \\nATOM     27  C   LYS A   5     -15.279  10.246  -2.797  1.00  0.00           C  \\nATOM     28  O   LYS A   5     -14.838   9.238  -2.293  1.00  0.00           O  \\nATOM     29  CB  LYS A   5     -14.208  11.695  -4.246  1.00  0.00           C  \\nATOM     30  CG  LYS A   5     -12.948  11.758  -3.553  1.00  0.00           C  \\nATOM     31  CD  LYS A   5     -11.814  13.081  -3.679  1.00  0.00           C  \\nATOM     32  CE  LYS A   5     -10.806  13.144  -2.545  1.00  0.00           C  \\nATOM     33  NZ  LYS A   5      -9.735  14.404  -2.293  1.00  0.00           N  \\nATOM     34  N   GLY A   6     -16.161  10.750  -1.915  1.00  0.00           N  \\nATOM     35  CA  GLY A   6     -16.350  10.435  -0.403  1.00  0.00           C  \\nATOM     36  C   GLY A   6     -16.539   8.923  -0.655  1.00  0.00           C  \\nATOM     37  O   GLY A   6     -15.909   8.041  -0.025  1.00  0.00           O  \\nATOM     38  N   LYS A   7     -17.421   8.419  -1.411  1.00  0.00           N  \\nATOM     39  CA  LYS A   7     -17.799   6.844  -1.033  1.00  0.00           C  \\nATOM     40  C   LYS A   7     -16.476   6.151  -1.474  1.00  0.00           C  \\nATOM     41  O   LYS A   7     -15.972   4.954  -0.970  1.00  0.00           O  \\nATOM     42  CB  LYS A   7     -18.870   6.403  -2.293  1.00  0.00           C  \\nATOM     43  CG  LYS A   7     -18.870   5.017  -2.734  1.00  0.00           C  \\nATOM     44  CD  LYS A   7     -20.193   4.765  -3.427  1.00  0.00           C  \\nATOM     45  CE  LYS A   7     -20.193   3.190  -4.057  1.00  0.00           C  \\nATOM     46  NZ  LYS A   7     -21.516   2.623  -4.309  1.00  0.00           N  \\nATOM     47  N   LYS A   8     -15.846   6.403  -2.545  1.00  0.00           N  \\nATOM     48  CA  LYS A   8     -14.460   5.899  -2.923  1.00  0.00           C  \\nATOM     49  C   LYS A   8     -13.452   5.899  -1.663  1.00  0.00           C  \\nATOM     50  O   LYS A   8     -13.011   5.017  -1.348  1.00  0.00           O  \\nATOM     51  CB  LYS A   8     -13.704   6.844  -3.931  1.00  0.00           C  \\nATOM     52  CG  LYS A   8     -12.318   6.529  -4.183  1.00  0.00           C  \\nATOM     53  CD  LYS A   8     -11.562   7.537  -5.065  1.00  0.00           C  \\nATOM     54  CE  LYS A   8     -10.176   7.222  -5.821  1.00  0.00           C  \\nATOM     55  NZ  LYS A   8      -9.357   8.419  -6.325  1.00  0.00           N  \\nATOM     56  N   THR A   9     -13.452   6.970  -1.159  1.00  0.00           N  \\nATOM     57  CA  THR A   9     -12.633   7.033   0.227  1.00  0.00           C  \\nATOM     58  C   THR A   9     -13.515   6.277   1.298  1.00  0.00           C  \\nATOM     59  O   THR A   9     -12.948   5.773   2.369  1.00  0.00           O  \\nATOM     60  CB  THR A   9     -12.381   8.419   0.920  1.00  0.00           C  \\nATOM     61  OG1 THR A   9     -11.058   8.923   0.542  1.00  0.00           O  \\nATOM     62  CG2 THR A   9     -12.444   8.734   2.558  1.00  0.00           C  \\nATOM     63  N   PHE A  10     -14.775   6.403   1.235  1.00  0.00           N  \\nATOM     64  CA  PHE A  10     -15.531   5.647   2.117  1.00  0.00           C  \\nATOM     65  C   PHE A  10     -15.216   4.135   1.739  1.00  0.00           C  \\nATOM     66  O   PHE A  10     -14.460   3.757   2.495  1.00  0.00           O  \\nATOM     67  CB  PHE A  10     -17.106   5.647   1.991  1.00  0.00           C  \\nATOM     68  CG  PHE A  10     -17.736   4.891   3.251  1.00  0.00           C  \\nATOM     69  CD1 PHE A  10     -18.492   3.757   2.747  1.00  0.00           C  \\nATOM     70  CD2 PHE A  10     -17.862   5.143   4.133  1.00  0.00           C  \\nATOM     71  CE1 PHE A  10     -18.996   3.001   3.818  1.00  0.00           C  \\nATOM     72  CE2 PHE A  10     -18.366   4.513   4.889  1.00  0.00           C  \\nATOM     73  CZ  PHE A  10     -18.996   3.379   5.015  1.00  0.00           C  \\nATOM     74  N   VAL A  11     -15.216   3.379   0.479  1.00  0.00           N  \\nATOM     75  CA  VAL A  11     -14.964   2.056   0.605  1.00  0.00           C  \\nATOM     76  C   VAL A  11     -13.326   1.867   1.109  1.00  0.00           C  \\nATOM     77  O   VAL A  11     -12.948   0.985   1.865  1.00  0.00           O  \\nATOM     78  CB  VAL A  11     -15.153   1.237  -0.529  1.00  0.00           C  \\nATOM     79  CG1 VAL A  11     -16.098   1.930  -1.537  1.00  0.00           C  \\nATOM     80  CG2 VAL A  11     -13.956   1.111  -1.285  1.00  0.00           C  \\nATOM     81  N   GLN A  12     -12.570   2.749   0.794  1.00  0.00           N  \\nATOM     82  CA  GLN A  12     -10.995   2.749   1.235  1.00  0.00           C  \\nATOM     83  C   GLN A  12     -10.680   2.938   2.873  1.00  0.00           C  \\nATOM     84  O   GLN A  12     -10.176   1.930   3.377  1.00  0.00           O  \\nATOM     85  CB  GLN A  12     -10.428   3.820   0.479  1.00  0.00           C  \\nATOM     86  CG  GLN A  12      -9.861   3.253  -0.718  1.00  0.00           C  \\nATOM     87  CD  GLN A  12      -9.168   4.135  -2.293  1.00  0.00           C  \\nATOM     88  OE1 GLN A  12      -7.593   3.820  -2.041  1.00  0.00           O  \\nATOM     89  NE2 GLN A  12      -9.672   5.143  -2.671  1.00  0.00           N  \\nATOM     90  N   LYS A  13     -10.995   3.757   3.629  1.00  0.00           N  \\nATOM     91  CA  LYS A  13     -10.869   3.757   5.015  1.00  0.00           C  \\nATOM     92  C   LYS A  13     -12.066   3.694   5.960  1.00  0.00           C  \\nATOM     93  O   LYS A  13     -12.003   3.127   6.779  1.00  0.00           O  \\nATOM     94  CB  LYS A  13     -10.239   5.017   5.141  1.00  0.00           C  \\nATOM     95  CG  LYS A  13      -9.672   5.521   3.881  1.00  0.00           C  \\nATOM     96  CD  LYS A  13      -8.286   6.277   4.007  1.00  0.00           C  \\nATOM     97  CE  LYS A  13      -8.160   6.907   2.621  1.00  0.00           C  \\nATOM     98  NZ  LYS A  13      -6.837   7.789   2.243  1.00  0.00           N  \\nATOM     99  N   CYS A  14     -13.137   4.072   5.519  1.00  0.00           N  \\nATOM    100  CA  CYS A  14     -14.460   4.072   6.464  1.00  0.00           C  \\nATOM    101  C   CYS A  14     -15.216   2.812   5.897  1.00  0.00           C  \\nATOM    102  O   CYS A  14     -15.342   1.804   6.716  1.00  0.00           O  \\nATOM    103  CB  CYS A  14     -15.468   5.332   5.897  1.00  0.00           C  \\nATOM    104  SG  CYS A  14     -14.712   6.844   6.464  1.00  0.00           S  \\nATOM    105  N   ALA A  15     -15.720   2.623   4.637  1.00  0.00           N  \\nATOM    106  CA  ALA A  15     -16.161   1.237   4.322  1.00  0.00           C  \\nATOM    107  C   ALA A  15     -16.035  -0.086   5.015  1.00  0.00           C  \\nATOM    108  O   ALA A  15     -17.043  -0.590   5.393  1.00  0.00           O  \\nATOM    109  CB  ALA A  15     -15.405   0.985   3.062  1.00  0.00           C  \\nATOM    110  N   GLN A  16     -14.964  -0.653   4.889  1.00  0.00           N  \\nATOM    111  CA  GLN A  16     -14.901  -2.039   5.897  1.00  0.00           C  \\nATOM    112  C   GLN A  16     -15.090  -1.787   7.283  1.00  0.00           C  \\nATOM    113  O   GLN A  16     -14.838  -2.669   8.291  1.00  0.00           O  \\nATOM    114  CB  GLN A  16     -13.578  -2.039   5.519  1.00  0.00           C  \\nATOM    115  CG  GLN A  16     -12.318  -1.724   6.275  1.00  0.00           C  \\nATOM    116  CD  GLN A  16     -11.625  -0.842   5.015  1.00  0.00           C  \\nATOM    117  OE1 GLN A  16     -10.365  -0.149   5.330  1.00  0.00           O  \\nATOM    118  NE2 GLN A  16     -12.444  -0.464   4.385  1.00  0.00           N  \\nATOM    119  N   CYS A  17     -15.216  -0.653   7.787  1.00  0.00           N  \\nATOM    120  CA  CYS A  17     -15.405  -0.905   9.299  1.00  0.00           C  \\nATOM    121  C   CYS A  17     -16.917  -0.401   9.173  1.00  0.00           C  \\nATOM    122  O   CYS A  17     -17.736  -1.283   9.173  1.00  0.00           O  \\nATOM    123  CB  CYS A  17     -14.460   0.355  10.055  1.00  0.00           C  \\nATOM    124  SG  CYS A  17     -13.326  -0.338  11.441  1.00  0.00           S  \\nATOM    125  N   HIS A  18     -17.232   0.733   9.551  1.00  0.00           N  \\nATOM    126  CA  HIS A  18     -18.870   0.985   9.803  1.00  0.00           C  \\nATOM    127  C   HIS A  18     -19.878   0.229   8.795  1.00  0.00           C  \\nATOM    128  O   HIS A  18     -19.626  -0.023   7.472  1.00  0.00           O  \\nATOM    129  CB  HIS A  18     -19.185   2.245   9.173  1.00  0.00           C  \\nATOM    130  CG  HIS A  18     -18.681   3.127  10.181  1.00  0.00           C  \\nATOM    131  ND1 HIS A  18     -18.618   3.253  11.567  1.00  0.00           N  \\nATOM    132  CD2 HIS A  18     -17.736   4.072  10.559  1.00  0.00           C  \\nATOM    133  CE1 HIS A  18     -18.051   3.694  12.827  1.00  0.00           C  \\nATOM    134  NE2 HIS A  18     -17.232   4.387  11.819  1.00  0.00           N  \\nATOM    135  N   THR A  19     -21.012   0.103   9.299  1.00  0.00           N  \\nATOM    136  CA  THR A  19     -22.272  -0.401   8.417  1.00  0.00           C  \\nATOM    137  C   THR A  19     -22.902   0.922   7.661  1.00  0.00           C  \\nATOM    138  O   THR A  19     -22.272   1.867   7.157  1.00  0.00           O  \\nATOM    139  CB  THR A  19     -23.280  -1.283   8.795  1.00  0.00           C  \\nATOM    140  OG1 THR A  19     -23.532  -0.905   9.740  1.00  0.00           O  \\nATOM    141  CG2 THR A  19     -22.776  -2.669   8.543  1.00  0.00           C  \\nATOM    142  N   VAL A  20     -24.036   1.111   7.472  1.00  0.00           N  \\nATOM    143  CA  VAL A  20     -24.414   2.371   6.779  1.00  0.00           C  \\nATOM    144  C   VAL A  20     -26.052   2.434   6.401  1.00  0.00           C  \\nATOM    145  O   VAL A  20     -26.556   3.442   5.771  1.00  0.00           O  \\nATOM    146  CB  VAL A  20     -23.784   2.560   5.519  1.00  0.00           C  \\nATOM    147  CG1 VAL A  20     -24.540   2.119   4.637  1.00  0.00           C  \\nATOM    148  CG2 VAL A  20     -23.658   4.009   5.267  1.00  0.00           C  \\nATOM    149  N   GLU A  21     -26.682   1.363   6.527  1.00  0.00           N  \\nATOM    150  CA  GLU A  21     -28.194   1.363   5.897  1.00  0.00           C  \\nATOM    151  C   GLU A  21     -29.013   1.741   7.283  1.00  0.00           C  \\nATOM    152  O   GLU A  21     -28.635   2.623   7.913  1.00  0.00           O  \\nATOM    153  CB  GLU A  21     -28.572  -0.086   5.645  1.00  0.00           C  \\nATOM    154  CG  GLU A  21     -28.698  -0.779   4.133  1.00  0.00           C  \\nATOM    155  CD  GLU A  21     -27.690  -1.913   4.007  1.00  0.00           C  \\nATOM    156  OE1 GLU A  21     -26.682  -2.039   5.141  1.00  0.00           O  \\nATOM    157  OE2 GLU A  21     -27.753  -2.921   2.873  1.00  0.00           O  \\nATOM    158  N   ASN A  22     -30.210   1.363   7.283  1.00  0.00           N  \\nATOM    159  CA  ASN A  22     -30.966   1.867   8.417  1.00  0.00           C  \\nATOM    160  C   ASN A  22     -30.714   1.237   9.677  1.00  0.00           C  \\nATOM    161  O   ASN A  22     -31.218   0.103  10.055  1.00  0.00           O  \\nATOM    162  N   GLY A  23     -29.958   1.552  10.433  1.00  0.00           N  \\nATOM    163  CA  GLY A  23     -29.517   0.859  11.819  1.00  0.00           C  \\nATOM    164  C   GLY A  23     -28.572  -0.338  11.252  1.00  0.00           C  \\nATOM    165  O   GLY A  23     -28.194  -1.283  12.071  1.00  0.00           O  \\nATOM    166  N   GLY A  24     -28.068  -0.149   9.929  1.00  0.00           N  \\nATOM    167  CA  GLY A  24     -27.060  -1.409   9.047  1.00  0.00           C  \\nATOM    168  C   GLY A  24     -26.871  -2.480  10.118  1.00  0.00           C  \\nATOM    169  O   GLY A  24     -27.186  -3.803   9.929  1.00  0.00           O  \\nATOM    170  N   LYS A  25     -26.304  -2.165  11.441  1.00  0.00           N  \\nATOM    171  CA  LYS A  25     -26.241  -3.047  12.701  1.00  0.00           C  \\nATOM    172  C   LYS A  25     -24.792  -2.795  13.079  1.00  0.00           C  \\nATOM    173  O   LYS A  25     -23.910  -2.417  12.323  1.00  0.00           O  \\nATOM    174  CB  LYS A  25     -26.241  -4.433  11.819  1.00  0.00           C  \\nATOM    175  CG  LYS A  25     -25.296  -4.496  10.433  1.00  0.00           C  \\nATOM    176  CD  LYS A  25     -24.918  -5.819   9.803  1.00  0.00           C  \\nATOM    177  CE  LYS A  25     -23.784  -5.819   8.543  1.00  0.00           C  \\nATOM    178  NZ  LYS A  25     -23.658  -6.827   7.661  1.00  0.00           N  \\nATOM    179  N   HIS A  26     -24.414  -2.921  14.465  1.00  0.00           N  \\nATOM    180  CA  HIS A  26     -23.028  -2.417  14.654  1.00  0.00           C  \\nATOM    181  C   HIS A  26     -21.894  -3.425  14.087  1.00  0.00           C  \\nATOM    182  O   HIS A  26     -22.083  -3.803  12.827  1.00  0.00           O  \\nATOM    183  CB  HIS A  26     -23.028  -2.543  16.103  1.00  0.00           C  \\nATOM    184  CG  HIS A  26     -24.036  -1.094  16.355  1.00  0.00           C  \\nATOM    185  ND1 HIS A  26     -24.540  -0.338  15.473  1.00  0.00           N  \\nATOM    186  CD2 HIS A  26     -24.666  -0.905  17.363  1.00  0.00           C  \\nATOM    187  CE1 HIS A  26     -25.422   0.355  16.103  1.00  0.00           C  \\nATOM    188  NE2 HIS A  26     -25.422   0.103  17.237  1.00  0.00           N  \\nATOM    189  N   LYS A  27     -20.886  -3.551  14.717  1.00  0.00           N  \\nATOM    190  CA  LYS A  27     -19.752  -4.433  14.213  1.00  0.00           C  \\nATOM    191  C   LYS A  27     -18.366  -3.677  14.276  1.00  0.00           C  \\nATOM    192  O   LYS A  27     -17.484  -3.677  13.331  1.00  0.00           O  \\nATOM    193  CB  LYS A  27     -20.004  -4.559  12.575  1.00  0.00           C  \\nATOM    194  CG  LYS A  27     -20.382  -3.425  11.882  1.00  0.00           C  \\nATOM    195  CD  LYS A  27     -19.500  -3.047  11.063  1.00  0.00           C  \\nATOM    196  CE  LYS A  27     -19.752  -3.677   9.551  1.00  0.00           C  \\nATOM    197  NZ  LYS A  27     -18.618  -3.677   8.858  1.00  0.00           N  \\nCONECT  104 1615                                                                \\nCONECT  124 1623                                                                \\nCONECT  134 1593                                                                \\nCONECT  356 1635                                                                \\nCONECT  616 1593                                                                \\nCONECT  900 1658                                                                \\nCONECT  920 1666                                                                \\nCONECT  930 1636                                                                \\nCONECT 1152 1678                                                                \\nCONECT 1412 1636                                                                \\nCONECT 1593  134  616 1598 1609                                                 \\nCONECT 1593 1617 1625                                                           \\nCONECT 1594 1599 1629                                                           \\nCONECT 1595 1602 1610                                                           \\nCONECT 1596 1613 1618                                                           \\nCONECT 1597 1621 1626                                                           \\nCONECT 1598 1593 1599 1602                                                      \\nCONECT 1599 1594 1598 1600                                                      \\nCONECT 1600 1599 1601 1604                                                      \\nCONECT 1601 1600 1602 1603                                                      \\nCONECT 1602 1595 1598 1601                                                      \\nCONECT 1603 1601                                                                \\nCONECT 1604 1600 1605                                                           \\nCONECT 1605 1604 1606                                                           \\nCONECT 1606 1605 1607 1608                                                      \\nCONECT 1607 1606                                                                \\nCONECT 1608 1606                                                                \\nCONECT 1609 1593 1610 1613                                                      \\nCONECT 1610 1595 1609 1611                                                      \\nCONECT 1611 1610 1612 1614                                                      \\nCONECT 1612 1611 1613 1615                                                      \\nCONECT 1613 1596 1609 1612                                                      \\nCONECT 1614 1611                                                                \\nCONECT 1615  104 1612 1616                                                      \\nCONECT 1616 1615                                                                \\nCONECT 1617 1593 1618 1621                                                      \\nCONECT 1618 1596 1617 1619                                                      \\nCONECT 1619 1618 1620 1622                                                      \\nCONECT 1620 1619 1621 1623                                                      \\nCONECT 1621 1597 1617 1620                                                      \\nCONECT 1622 1619                                                                \\nCONECT 1623  124 1620 1624                                                      \\nCONECT 1624 1623                                                                \\nCONECT 1625 1593 1626 1629                                                      \\nCONECT 1626 1597 1625 1627                                                      \\nCONECT 1627 1626 1628 1630                                                      \\nCONECT 1628 1627 1629 1631                                                      \\nCONECT 1629 1594 1625 1628                                                      \\nCONECT 1630 1627                                                                \\nCONECT 1631 1628 1632                                                           \\nCONECT 1632 1631 1633                                                           \\nCONECT 1633 1632 1634 1635                                                      \\nCONECT 1634 1633                                                                \\nCONECT 1635  356 1633                                                           \\nCONECT 1636  930 1412 1641 1652                                                 \\nCONECT 1636 1660 1668                                                           \\nCONECT 1637 1642 1672                                                           \\nCONECT 1638 1645 1653                                                           \\nCONECT 1639 1656 1661                                                           \\nCONECT 1640 1664 1669                                                           \\nCONECT 1641 1636 1642 1645                                                      \\nCONECT 1642 1637 1641 1643                                                      \\nCONECT 1643 1642 1644 1647                                                      \\nCONECT 1644 1643 1645 1646                                                      \\nCONECT 1645 1638 1641 1644                                                      \\nCONECT 1646 1644                                                                \\nCONECT 1647 1643 1648                                                           \\nCONECT 1648 1647 1649                                                           \\nCONECT 1649 1648 1650 1651                                                      \\nCONECT 1650 1649                                                                \\nCONECT 1651 1649                                                                \\nCONECT 1652 1636 1653 1656                                                      \\nCONECT 1653 1638 1652 1654                                                      \\nCONECT 1654 1653 1655 1657                                                      \\nCONECT 1655 1654 1656 1658                                                      \\nCONECT 1656 1639 1652 1655                                                      \\nCONECT 1657 1654                                                                \\nCONECT 1658  900 1655 1659                                                      \\nCONECT 1659 1658                                                                \\nCONECT 1660 1636 1661 1664                                                      \\nCONECT 1661 1639 1660 1662                                                      \\nCONECT 1662 1661 1663 1665                                                      \\nCONECT 1663 1662 1664 1666                                                      \\nCONECT 1664 1640 1660 1663                                                      \\nCONECT 1665 1662                                                                \\nCONECT 1666  920 1663 1667                                                      \\nCONECT 1667 1666                                                                \\nCONECT 1668 1636 1669 1672                                                      \\nCONECT 1669 1640 1668 1670                                                      \\nCONECT 1670 1669 1671 1673                                                      \\nCONECT 1671 1670 1672 1674                                                      \\nCONECT 1672 1637 1668 1671                                                      \\nCONECT 1673 1670                                                                \\nCONECT 1674 1671 1675                                                           \\nCONECT 1675 1674 1676                                                           \\nCONECT 1676 1675 1677 1678                                                      \\nCONECT 1677 1676                                                                \\nCONECT 1678 1152 1676                                                           \\nMASTER      287    0    2   10    0    0    9    9 1677    2   98   16          \\nEND                                                                             \\n\",\"pdb\");\n\tviewer_16742555228337302.setBackgroundColor(\"white\");\n\tviewer_16742555228337302.setStyle({\"cartoon\": {\"color\": \"white\"}});\n\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"5\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"7\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"5\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"8\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"5\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"6\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"8\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"6\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"6\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"7\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"7\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"11\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"7\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"15\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"8\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"8\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"11\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"8\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"15\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"9\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"13\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"15\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"16\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"11\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"13\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"11\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"12\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"13\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"15\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"13\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"16\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"16\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"18\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"15\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"18\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"16\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"18\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"19\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"27\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"18\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"18\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"24\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"18\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"26\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"19\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"26\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"21\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"25\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"23\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"25\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"23\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"26\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"24\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"26\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"25\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"27\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228337302.zoomTo();\nviewer_16742555228337302.render();\n});\n</script>",
      "text/html": [
       "<div id=\"3dmolviewer_16742555228337302\"  style=\"position: relative; width: 640px; height: 480px\">\n",
       "        <p id=\"3dmolwarning_16742555228337302\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n",
       "        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n",
       "        </div>\n",
       "<script>\n",
       "\n",
       "var loadScriptAsync = function(uri){\n",
       "  return new Promise((resolve, reject) => {\n",
       "    //this is to ignore the existence of requirejs amd\n",
       "    var savedexports, savedmodule;\n",
       "    if (typeof exports !== 'undefined') savedexports = exports;\n",
       "    else exports = {}\n",
       "    if (typeof module !== 'undefined') savedmodule = module;\n",
       "    else module = {}\n",
       "\n",
       "    var tag = document.createElement('script');\n",
       "    tag.src = uri;\n",
       "    tag.async = true;\n",
       "    tag.onload = () => {\n",
       "        exports = savedexports;\n",
       "        module = savedmodule;\n",
       "        resolve();\n",
       "    };\n",
       "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
       "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
       "});\n",
       "};\n",
       "\n",
       "if(typeof $3Dmolpromise === 'undefined') {\n",
       "$3Dmolpromise = null;\n",
       "  $3Dmolpromise = loadScriptAsync('https://3dmol.org/build/3Dmol.js');\n",
       "}\n",
       "\n",
       "var viewer_16742555228337302 = null;\n",
       "var warn = document.getElementById(\"3dmolwarning_16742555228337302\");\n",
       "if(warn) {\n",
       "    warn.parentNode.removeChild(warn);\n",
       "}\n",
       "$3Dmolpromise.then(function() {\n",
       "viewer_16742555228337302 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_16742555228337302\"),{backgroundColor:\"white\"});\n",
       "viewer_16742555228337302.zoomTo();\n",
       "\tviewer_16742555228337302.addModel(\"HEADER    ELECTRON TRANSPORT                      01-AUG-76   1CYC              \\nTITLE     THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C AT 2.3     \\nTITLE    2 ANGSTROMS RESOLUTION. II. STRUCTURE AND FUNCTION                     \\nCAVEAT     1CYC    THR A 40 HAS WRONG CHIRALITY AT ATOM CB THR A 78 HAS WRONG   \\nCAVEAT   2 1CYC    CHIRALITY AT ATOM CB THR B 40 HAS WRONG CHIRALITY AT ATOM    \\nCAVEAT   3 1CYC    CB THR B 78 HAS WRONG CHIRALITY AT ATOM CB                   \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: FERROCYTOCHROME C;                                         \\nCOMPND   3 CHAIN: A, B;                                                         \\nCOMPND   4 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 ORGANISM_SCIENTIFIC: KATSUWONUS PELAMIS;                             \\nSOURCE   3 ORGANISM_COMMON: SKIPJACK TUNA;                                      \\nSOURCE   4 ORGANISM_TAXID: 8226                                                 \\nKEYWDS    ELECTRON TRANSPORT                                                    \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    N.TANAKA,T.YAMANE,T.TSUKIHARA,T.ASHIDA,M.KAKUDO                       \\nREVDAT  17   03-MAR-21 1CYC    1       CAVEAT COMPND REMARK HET                 \\nREVDAT  17 2                   1       HETNAM HETSYN FORMUL LINK                \\nREVDAT  17 3                   1       ATOM                                     \\nREVDAT  16   29-NOV-17 1CYC    1       HELIX                                    \\nREVDAT  15   24-FEB-09 1CYC    1       VERSN                                    \\nREVDAT  14   01-APR-03 1CYC    1       JRNL                                     \\nREVDAT  13   27-JAN-84 1CYC    1       REMARK                                   \\nREVDAT  12   30-SEP-83 1CYC    1       REVDAT                                   \\nREVDAT  11   19-APR-83 1CYC    1       COMPND SOURCE REMARK                     \\nREVDAT  10   15-JAN-82 1CYC    1       REMARK                                   \\nREVDAT   9   06-JAN-82 1CYC    3       SCALE                                    \\nREVDAT   8   31-DEC-80 1CYC    1       REMARK                                   \\nREVDAT   7   07-APR-80 1CYC    3       SOURCE SEQRES ATOM                       \\nREVDAT   6   23-APR-79 1CYC    3       ATOM                                     \\nREVDAT   5   01-NOV-77 1CYC    1       AUTHOR REMARK FORMUL                     \\nREVDAT   4   09-SEP-77 1CYC    3       ATOM                                     \\nREVDAT   3   13-JUN-77 1CYC    1       HET                                      \\nREVDAT   2   25-OCT-76 1CYC    2       CONECT                                   \\nREVDAT   1   06-OCT-76 1CYC    0                                                \\nJRNL        AUTH   N.TANAKA,T.YAMANE,T.TSUKIHARA,T.ASHIDA,M.KAKUDO              \\nJRNL        TITL   THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C   \\nJRNL        TITL 2 AT 2.3 A RESOLUTION. II. STRUCTURE AND FUNCTION.             \\nJRNL        REF    J.BIOCHEM.(TOKYO)             V.  77   147 1975              \\nJRNL        REFN                   ISSN 0021-924X                               \\nJRNL        PMID   166072                                                       \\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   T.TSUKIHARA,T.YAMANE,N.TANAKA,T.ASHIDA,M.KAKUDO              \\nREMARK   1  TITL   OXIDATION OF A FERROCYTOCHROME C IN THE CRYSTALLINE          \\nREMARK   1  TITL 2 STATE-STRUCTURAL CHANGE AND ANION BINDING                    \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  73  1163 1973              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 2                                                          \\nREMARK   1  AUTH   T.ASHIDA,N.TANAKA,T.YAMANE,T.TSUKIHARA,M.KAKUDO              \\nREMARK   1  TITL   THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C   \\nREMARK   1  TITL 2 AT 2.3 ANGSTROMS RESOLUTION                                  \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  73   463 1973              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 3                                                          \\nREMARK   1  AUTH   T.ASHIDA,T.UEKI,T.TSUKIHARA,A.SUGIHARA,T.TAKANO,M.KAKUDO     \\nREMARK   1  TITL   THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C   \\nREMARK   1  TITL 2 AT 4 ANGSTROMS RESOLUTION                                    \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  70   913 1971              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 4                                                          \\nREMARK   1  AUTH   T.NAKAYAMA,K.TITANI,K.NARITA                                 \\nREMARK   1  TITL   THE AMINO ACID SEQUENCE OF CYTOCHROME C FROM BONITO          \\nREMARK   1  TITL 2 (KATSUWONUS PELAMIS, LINNAEUS)                               \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  70   311 1971              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 5                                                          \\nREMARK   1  EDIT   R.J.FELDMANN                                                 \\nREMARK   1  REF    ATLAS OF MACROMOLECULAR                120 1976              \\nREMARK   1  REF  2 STRUCTURE ON MICROFICHE                                      \\nREMARK   1  PUBL   TRACOR JITCO INC.,ROCKVILLE,MD.                              \\nREMARK   1  REFN                                                                \\nREMARK   1 REFERENCE 6                                                          \\nREMARK   1  EDIT   M.O.DAYHOFF                                                  \\nREMARK   1  REF    ATLAS OF PROTEIN SEQUENCE     V.   5    39 1976              \\nREMARK   1  REF  2 AND STRUCTURE,SUPPLEMENT 2                                   \\nREMARK   1  PUBL   NATIONAL BIOMEDICAL RESEARCH FOUNDATION, SILVER SPRING,MD.   \\nREMARK   1  REFN                   ISSN 0-912466-05-7                           \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    2.30 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : NULL                                                 \\nREMARK   3   AUTHORS     : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 2.30                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : NULL                           \\nREMARK   3   DATA CUTOFF            (SIGMA(F)) : NULL                           \\nREMARK   3   DATA CUTOFF HIGH         (ABS(F)) : NULL                           \\nREMARK   3   DATA CUTOFF LOW          (ABS(F)) : NULL                           \\nREMARK   3   COMPLETENESS (WORKING+TEST)   (%) : NULL                           \\nREMARK   3   NUMBER OF REFLECTIONS             : NULL                           \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   CROSS-VALIDATION METHOD          : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SELECTION  : NULL                            \\nREMARK   3   R VALUE            (WORKING SET) : NULL                            \\nREMARK   3   FREE R VALUE                     : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : NULL                            \\nREMARK   3   FREE R VALUE TEST SET COUNT      : NULL                            \\nREMARK   3   ESTIMATED ERROR OF FREE R VALUE  : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  FIT IN THE HIGHEST RESOLUTION BIN.                                  \\nREMARK   3   TOTAL NUMBER OF BINS USED           : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE HIGH       (A) : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE LOW        (A) : NULL                         \\nREMARK   3   BIN COMPLETENESS (WORKING+TEST) (%) : NULL                         \\nREMARK   3   REFLECTIONS IN BIN    (WORKING SET) : NULL                         \\nREMARK   3   BIN R VALUE           (WORKING SET) : NULL                         \\nREMARK   3   BIN FREE R VALUE                    : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET SIZE  (%) : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET COUNT     : NULL                         \\nREMARK   3   ESTIMATED ERROR OF BIN FREE R VALUE : NULL                         \\nREMARK   3                                                                      \\nREMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \\nREMARK   3   PROTEIN ATOMS            : 1590                                    \\nREMARK   3   NUCLEIC ACID ATOMS       : 0                                       \\nREMARK   3   HETEROGEN ATOMS          : 86                                      \\nREMARK   3   SOLVENT ATOMS            : 1                                       \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : NULL                           \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : NULL                           \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : NULL                                                 \\nREMARK   3    B22 (A**2) : NULL                                                 \\nREMARK   3    B33 (A**2) : NULL                                                 \\nREMARK   3    B12 (A**2) : NULL                                                 \\nREMARK   3    B13 (A**2) : NULL                                                 \\nREMARK   3    B23 (A**2) : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  ESTIMATED COORDINATE ERROR.                                         \\nREMARK   3   ESD FROM LUZZATI PLOT        (A) : NULL                            \\nREMARK   3   ESD FROM SIGMAA              (A) : NULL                            \\nREMARK   3   LOW RESOLUTION CUTOFF        (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  CROSS-VALIDATED ESTIMATED COORDINATE ERROR.                         \\nREMARK   3   ESD FROM C-V LUZZATI PLOT    (A) : NULL                            \\nREMARK   3   ESD FROM C-V SIGMAA          (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  RMS DEVIATIONS FROM IDEAL VALUES.                                   \\nREMARK   3   BOND LENGTHS                 (A) : NULL                            \\nREMARK   3   BOND ANGLES            (DEGREES) : NULL                            \\nREMARK   3   DIHEDRAL ANGLES        (DEGREES) : NULL                            \\nREMARK   3   IMPROPER ANGLES        (DEGREES) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL MODEL : NULL                                      \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL FACTOR RESTRAINTS.    RMS    SIGMA                \\nREMARK   3   MAIN-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   MAIN-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  NCS MODEL : NULL                                                    \\nREMARK   3                                                                      \\nREMARK   3  NCS RESTRAINTS.                         RMS   SIGMA/WEIGHT          \\nREMARK   3   GROUP  1  POSITIONAL            (A) : NULL  ; NULL                 \\nREMARK   3   GROUP  1  B-FACTOR           (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  PARAMETER FILE  1  : NULL                                           \\nREMARK   3  TOPOLOGY FILE  1   : NULL                                           \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS: NULL                                      \\nREMARK   4                                                                      \\nREMARK   4 1CYC COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY BNL.                                \\nREMARK 100 THE DEPOSITION ID IS D_1000172595.                                   \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : NULL                               \\nREMARK 200  TEMPERATURE           (KELVIN) : NULL                               \\nREMARK 200  PH                             : NULL                               \\nREMARK 200  NUMBER OF CRYSTALS USED        : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : NULL                               \\nREMARK 200  RADIATION SOURCE               : NULL                               \\nREMARK 200  BEAMLINE                       : NULL                               \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : NULL                               \\nREMARK 200  WAVELENGTH OR RANGE        (A) : NULL                               \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : NULL                               \\nREMARK 200  DETECTOR MANUFACTURER          : NULL                               \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : NULL                               \\nREMARK 200  DATA SCALING SOFTWARE          : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : NULL                               \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : NULL                               \\nREMARK 200  RESOLUTION RANGE LOW       (A) : NULL                               \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY                : NULL                               \\nREMARK 200  R MERGE                    (I) : NULL                               \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : NULL                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : NULL                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : NULL                               \\nREMARK 200  R MERGE FOR SHELL          (I) : NULL                               \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: NULL                                           \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: NULL                         \\nREMARK 200 SOFTWARE USED: NULL                                                  \\nREMARK 200 STARTING MODEL: NULL                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: NULL                                                         \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): 39.18                                     \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 2.02                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: NULL                                     \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: P 21 21 21                       \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -X+1/2,-Y,Z+1/2                                         \\nREMARK 290       3555   -X,Y+1/2,-Z+1/2                                         \\nREMARK 290       4555   X+1/2,-Y+1/2,-Z                                         \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -1.000000  0.000000  0.000000       28.84000            \\nREMARK 290   SMTRY2   2  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000       18.91500            \\nREMARK 290   SMTRY1   3 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   3  0.000000  1.000000  0.000000       42.29000            \\nREMARK 290   SMTRY3   3  0.000000  0.000000 -1.000000       18.91500            \\nREMARK 290   SMTRY1   4  1.000000  0.000000  0.000000       28.84000            \\nREMARK 290   SMTRY2   4  0.000000 -1.000000  0.000000       42.29000            \\nREMARK 290   SMTRY3   4  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1, 2                                                    \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: MONOMERIC                         \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 2                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: MONOMERIC                         \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: B                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 470                                                                      \\nREMARK 470 MISSING ATOM                                                         \\nREMARK 470 THE FOLLOWING RESIDUES HAVE MISSING ATOMS (M=MODEL NUMBER;           \\nREMARK 470 RES=RESIDUE NAME; C=CHAIN IDENTIFIER; SSEQ=SEQUENCE NUMBER;          \\nREMARK 470 I=INSERTION CODE):                                                   \\nREMARK 470   M RES CSSEQI  ATOMS                                                \\nREMARK 470     ASN A  22    CB   CG   OD1  ND2                                  \\nREMARK 470     LYS A  39    CE   NZ                                             \\nREMARK 470     ASN B  22    CB   CG   OD1  ND2                                  \\nREMARK 470     LYS B  39    CE   NZ                                             \\nREMARK 620                                                                      \\nREMARK 620 METAL COORDINATION                                                   \\nREMARK 620 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 620 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE):                             \\nREMARK 620                                                                      \\nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \\nREMARK 620                             HEC A 104  FE                            \\nREMARK 620 N RES CSSEQI ATOM                                                    \\nREMARK 620 1 HIS A  18   NE2                                                    \\nREMARK 620 2 HEC A 104   NA   87.0                                              \\nREMARK 620 3 HEC A 104   NB   87.8  90.6                                        \\nREMARK 620 4 HEC A 104   NC   92.0 178.7  90.1                                  \\nREMARK 620 5 HEC A 104   ND   91.1  86.9 177.4  92.3                            \\nREMARK 620 6 MET A  80   SD  170.9 102.1  93.0  79.0  88.6                      \\nREMARK 620 N                    1     2     3     4     5                       \\nREMARK 620                                                                      \\nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \\nREMARK 620                             HEC B 104  FE                            \\nREMARK 620 N RES CSSEQI ATOM                                                    \\nREMARK 620 1 HIS B  18   NE2                                                    \\nREMARK 620 2 HEC B 104   NA   87.0                                              \\nREMARK 620 3 HEC B 104   NB   87.8  90.6                                        \\nREMARK 620 4 HEC B 104   NC   92.0 178.7  90.1                                  \\nREMARK 620 5 HEC B 104   ND   91.1  86.9 177.4  92.3                            \\nREMARK 620 6 MET B  80   SD  170.9 102.1  93.0  79.0  88.6                      \\nREMARK 620 N                    1     2     3     4     5                       \\nREMARK 800                                                                      \\nREMARK 800 SITE                                                                 \\nREMARK 800 SITE_IDENTIFIER: AC1                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE HEC A 104                 \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: AC2                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE HEC B 104                 \\nDBREF  1CYC A    1   103  UNP    P00025   CYC_KATPE        1    103             \\nDBREF  1CYC B    1   103  UNP    P00025   CYC_KATPE        1    103             \\nSEQRES   1 A  103  GLY ASP VAL ALA LYS GLY LYS LYS THR PHE VAL GLN LYS          \\nSEQRES   2 A  103  CYS ALA GLN CYS HIS THR VAL GLU ASN GLY GLY LYS HIS          \\nSEQRES   3 A  103  LYS VAL GLY PRO ASN LEU TRP GLY LEU PHE GLY ARG LYS          \\nSEQRES   4 A  103  THR GLY GLN ALA GLU GLY TYR SER TYR THR ASP ALA ASN          \\nSEQRES   5 A  103  LYS SER LYS GLY ILE VAL TRP ASN GLU ASN THR LEU MET          \\nSEQRES   6 A  103  GLU TYR LEU GLU ASN PRO LYS LYS TYR ILE PRO GLY THR          \\nSEQRES   7 A  103  LYS MET ILE PHE ALA GLY ILE LYS LYS LYS GLY GLU ARG          \\nSEQRES   8 A  103  GLN ASP LEU VAL ALA TYR LEU LYS SER ALA THR SER              \\nSEQRES   1 B  103  GLY ASP VAL ALA LYS GLY LYS LYS THR PHE VAL GLN LYS          \\nSEQRES   2 B  103  CYS ALA GLN CYS HIS THR VAL GLU ASN GLY GLY LYS HIS          \\nSEQRES   3 B  103  LYS VAL GLY PRO ASN LEU TRP GLY LEU PHE GLY ARG LYS          \\nSEQRES   4 B  103  THR GLY GLN ALA GLU GLY TYR SER TYR THR ASP ALA ASN          \\nSEQRES   5 B  103  LYS SER LYS GLY ILE VAL TRP ASN GLU ASN THR LEU MET          \\nSEQRES   6 B  103  GLU TYR LEU GLU ASN PRO LYS LYS TYR ILE PRO GLY THR          \\nSEQRES   7 B  103  LYS MET ILE PHE ALA GLY ILE LYS LYS LYS GLY GLU ARG          \\nSEQRES   8 B  103  GLN ASP LEU VAL ALA TYR LEU LYS SER ALA THR SER              \\nHET    HEC  A 104      43                                                       \\nHET    HEC  B 104      43                                                       \\nHETNAM     HEC HEME C                                                           \\nFORMUL   3  HEC    2(C34 H34 FE N4 O4)                                          \\nFORMUL   5  HOH   *(H2 O)                                                       \\nHELIX    1  H1 GLY A    1  VAL A   11  1                                  11    \\nHELIX    2  H4 CYS A   14  CYS A   17  514 AND 17 BOUND TO HEME GROUP      4    \\nHELIX    3  H5 THR A   49  LYS A   55  5LOOSE FROM 49-53,3/10 53-55        7    \\nHELIX    4  H2 ASN A   60  GLU A   69  1                                  10    \\nHELIX    5  H3 GLU A   90  SER A  103  1                                  14    \\nHELIX    6  H6 GLY B    1  VAL B   11  1                                  11    \\nHELIX    7  H7 CYS B   14  CYS B   17  514 AND 17 BOUND TO HEME GROUP      4    \\nHELIX    8  H8 THR B   49  LYS B   55  5LOOSE FROM 49-53,3/10 53-55        7    \\nHELIX    9  H9 ASN B   60  GLU B   69  1                                  10    \\nHELIX   10 H10 GLU B   90  SER B  103  1                                  14    \\nLINK         SG  CYS A  14                 CAB HEC A 104     1555   1555  1.48  \\nLINK         SG  CYS A  17                 CAC HEC A 104     1555   1555  1.86  \\nLINK         CE2 TYR A  48                 O2D HEC A 104     1555   1555  2.02  \\nLINK         SG  CYS B  14                 CAB HEC B 104     1555   1555  1.48  \\nLINK         SG  CYS B  17                 CAC HEC B 104     1555   1555  1.86  \\nLINK         CE2 TYR B  48                 O2D HEC B 104     1555   1555  2.02  \\nLINK         NE2 HIS A  18                FE   HEC A 104     1555   1555  2.60  \\nLINK         SD  MET A  80                FE   HEC A 104     1555   1555  2.49  \\nLINK         NE2 HIS B  18                FE   HEC B 104     1555   1555  2.60  \\nLINK         SD  MET B  80                FE   HEC B 104     1555   1555  2.49  \\nSITE     1 AC1 17 LYS A  13  CYS A  14  CYS A  17  HIS A  18                    \\nSITE     2 AC1 17 LEU A  32  LEU A  35  ARG A  38  THR A  40                    \\nSITE     3 AC1 17 TYR A  46  TYR A  48  ASN A  52  TRP A  59                    \\nSITE     4 AC1 17 THR A  78  LYS A  79  MET A  80  PHE A  82                    \\nSITE     5 AC1 17 HOH A 105                                                     \\nSITE     1 AC2 16 LYS B  13  CYS B  14  CYS B  17  HIS B  18                    \\nSITE     2 AC2 16 LEU B  32  LEU B  35  ARG B  38  THR B  40                    \\nSITE     3 AC2 16 TYR B  46  TYR B  48  ASN B  52  TRP B  59                    \\nSITE     4 AC2 16 THR B  78  LYS B  79  MET B  80  PHE B  82                    \\nCRYST1   57.680   84.580   37.830  90.00  90.00  90.00 P 21 21 21    8          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.017337  0.000000  0.000000        0.00000                         \\nSCALE2      0.000000  0.011823  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.026434        0.00000                         \\nMTRIX1   1  1.000000  0.000000  0.000000       28.84000    1                    \\nMTRIX2   1  0.000000  1.000000  0.000000        0.00000    1                    \\nMTRIX3   1  0.000000  0.000000  1.000000        0.00000    1                    \\nATOM     25  N   LYS A   5     -16.476  10.687  -5.191  1.00  0.00           N  \\nATOM     26  CA  LYS A   5     -15.216  10.498  -4.372  1.00  0.00           C  \\nATOM     27  C   LYS A   5     -15.279  10.246  -2.797  1.00  0.00           C  \\nATOM     28  O   LYS A   5     -14.838   9.238  -2.293  1.00  0.00           O  \\nATOM     29  CB  LYS A   5     -14.208  11.695  -4.246  1.00  0.00           C  \\nATOM     30  CG  LYS A   5     -12.948  11.758  -3.553  1.00  0.00           C  \\nATOM     31  CD  LYS A   5     -11.814  13.081  -3.679  1.00  0.00           C  \\nATOM     32  CE  LYS A   5     -10.806  13.144  -2.545  1.00  0.00           C  \\nATOM     33  NZ  LYS A   5      -9.735  14.404  -2.293  1.00  0.00           N  \\nATOM     34  N   GLY A   6     -16.161  10.750  -1.915  1.00  0.00           N  \\nATOM     35  CA  GLY A   6     -16.350  10.435  -0.403  1.00  0.00           C  \\nATOM     36  C   GLY A   6     -16.539   8.923  -0.655  1.00  0.00           C  \\nATOM     37  O   GLY A   6     -15.909   8.041  -0.025  1.00  0.00           O  \\nATOM     38  N   LYS A   7     -17.421   8.419  -1.411  1.00  0.00           N  \\nATOM     39  CA  LYS A   7     -17.799   6.844  -1.033  1.00  0.00           C  \\nATOM     40  C   LYS A   7     -16.476   6.151  -1.474  1.00  0.00           C  \\nATOM     41  O   LYS A   7     -15.972   4.954  -0.970  1.00  0.00           O  \\nATOM     42  CB  LYS A   7     -18.870   6.403  -2.293  1.00  0.00           C  \\nATOM     43  CG  LYS A   7     -18.870   5.017  -2.734  1.00  0.00           C  \\nATOM     44  CD  LYS A   7     -20.193   4.765  -3.427  1.00  0.00           C  \\nATOM     45  CE  LYS A   7     -20.193   3.190  -4.057  1.00  0.00           C  \\nATOM     46  NZ  LYS A   7     -21.516   2.623  -4.309  1.00  0.00           N  \\nATOM     47  N   LYS A   8     -15.846   6.403  -2.545  1.00  0.00           N  \\nATOM     48  CA  LYS A   8     -14.460   5.899  -2.923  1.00  0.00           C  \\nATOM     49  C   LYS A   8     -13.452   5.899  -1.663  1.00  0.00           C  \\nATOM     50  O   LYS A   8     -13.011   5.017  -1.348  1.00  0.00           O  \\nATOM     51  CB  LYS A   8     -13.704   6.844  -3.931  1.00  0.00           C  \\nATOM     52  CG  LYS A   8     -12.318   6.529  -4.183  1.00  0.00           C  \\nATOM     53  CD  LYS A   8     -11.562   7.537  -5.065  1.00  0.00           C  \\nATOM     54  CE  LYS A   8     -10.176   7.222  -5.821  1.00  0.00           C  \\nATOM     55  NZ  LYS A   8      -9.357   8.419  -6.325  1.00  0.00           N  \\nATOM     56  N   THR A   9     -13.452   6.970  -1.159  1.00  0.00           N  \\nATOM     57  CA  THR A   9     -12.633   7.033   0.227  1.00  0.00           C  \\nATOM     58  C   THR A   9     -13.515   6.277   1.298  1.00  0.00           C  \\nATOM     59  O   THR A   9     -12.948   5.773   2.369  1.00  0.00           O  \\nATOM     60  CB  THR A   9     -12.381   8.419   0.920  1.00  0.00           C  \\nATOM     61  OG1 THR A   9     -11.058   8.923   0.542  1.00  0.00           O  \\nATOM     62  CG2 THR A   9     -12.444   8.734   2.558  1.00  0.00           C  \\nATOM     63  N   PHE A  10     -14.775   6.403   1.235  1.00  0.00           N  \\nATOM     64  CA  PHE A  10     -15.531   5.647   2.117  1.00  0.00           C  \\nATOM     65  C   PHE A  10     -15.216   4.135   1.739  1.00  0.00           C  \\nATOM     66  O   PHE A  10     -14.460   3.757   2.495  1.00  0.00           O  \\nATOM     67  CB  PHE A  10     -17.106   5.647   1.991  1.00  0.00           C  \\nATOM     68  CG  PHE A  10     -17.736   4.891   3.251  1.00  0.00           C  \\nATOM     69  CD1 PHE A  10     -18.492   3.757   2.747  1.00  0.00           C  \\nATOM     70  CD2 PHE A  10     -17.862   5.143   4.133  1.00  0.00           C  \\nATOM     71  CE1 PHE A  10     -18.996   3.001   3.818  1.00  0.00           C  \\nATOM     72  CE2 PHE A  10     -18.366   4.513   4.889  1.00  0.00           C  \\nATOM     73  CZ  PHE A  10     -18.996   3.379   5.015  1.00  0.00           C  \\nATOM     74  N   VAL A  11     -15.216   3.379   0.479  1.00  0.00           N  \\nATOM     75  CA  VAL A  11     -14.964   2.056   0.605  1.00  0.00           C  \\nATOM     76  C   VAL A  11     -13.326   1.867   1.109  1.00  0.00           C  \\nATOM     77  O   VAL A  11     -12.948   0.985   1.865  1.00  0.00           O  \\nATOM     78  CB  VAL A  11     -15.153   1.237  -0.529  1.00  0.00           C  \\nATOM     79  CG1 VAL A  11     -16.098   1.930  -1.537  1.00  0.00           C  \\nATOM     80  CG2 VAL A  11     -13.956   1.111  -1.285  1.00  0.00           C  \\nATOM     81  N   GLN A  12     -12.570   2.749   0.794  1.00  0.00           N  \\nATOM     82  CA  GLN A  12     -10.995   2.749   1.235  1.00  0.00           C  \\nATOM     83  C   GLN A  12     -10.680   2.938   2.873  1.00  0.00           C  \\nATOM     84  O   GLN A  12     -10.176   1.930   3.377  1.00  0.00           O  \\nATOM     85  CB  GLN A  12     -10.428   3.820   0.479  1.00  0.00           C  \\nATOM     86  CG  GLN A  12      -9.861   3.253  -0.718  1.00  0.00           C  \\nATOM     87  CD  GLN A  12      -9.168   4.135  -2.293  1.00  0.00           C  \\nATOM     88  OE1 GLN A  12      -7.593   3.820  -2.041  1.00  0.00           O  \\nATOM     89  NE2 GLN A  12      -9.672   5.143  -2.671  1.00  0.00           N  \\nATOM     90  N   LYS A  13     -10.995   3.757   3.629  1.00  0.00           N  \\nATOM     91  CA  LYS A  13     -10.869   3.757   5.015  1.00  0.00           C  \\nATOM     92  C   LYS A  13     -12.066   3.694   5.960  1.00  0.00           C  \\nATOM     93  O   LYS A  13     -12.003   3.127   6.779  1.00  0.00           O  \\nATOM     94  CB  LYS A  13     -10.239   5.017   5.141  1.00  0.00           C  \\nATOM     95  CG  LYS A  13      -9.672   5.521   3.881  1.00  0.00           C  \\nATOM     96  CD  LYS A  13      -8.286   6.277   4.007  1.00  0.00           C  \\nATOM     97  CE  LYS A  13      -8.160   6.907   2.621  1.00  0.00           C  \\nATOM     98  NZ  LYS A  13      -6.837   7.789   2.243  1.00  0.00           N  \\nATOM     99  N   CYS A  14     -13.137   4.072   5.519  1.00  0.00           N  \\nATOM    100  CA  CYS A  14     -14.460   4.072   6.464  1.00  0.00           C  \\nATOM    101  C   CYS A  14     -15.216   2.812   5.897  1.00  0.00           C  \\nATOM    102  O   CYS A  14     -15.342   1.804   6.716  1.00  0.00           O  \\nATOM    103  CB  CYS A  14     -15.468   5.332   5.897  1.00  0.00           C  \\nATOM    104  SG  CYS A  14     -14.712   6.844   6.464  1.00  0.00           S  \\nATOM    105  N   ALA A  15     -15.720   2.623   4.637  1.00  0.00           N  \\nATOM    106  CA  ALA A  15     -16.161   1.237   4.322  1.00  0.00           C  \\nATOM    107  C   ALA A  15     -16.035  -0.086   5.015  1.00  0.00           C  \\nATOM    108  O   ALA A  15     -17.043  -0.590   5.393  1.00  0.00           O  \\nATOM    109  CB  ALA A  15     -15.405   0.985   3.062  1.00  0.00           C  \\nATOM    110  N   GLN A  16     -14.964  -0.653   4.889  1.00  0.00           N  \\nATOM    111  CA  GLN A  16     -14.901  -2.039   5.897  1.00  0.00           C  \\nATOM    112  C   GLN A  16     -15.090  -1.787   7.283  1.00  0.00           C  \\nATOM    113  O   GLN A  16     -14.838  -2.669   8.291  1.00  0.00           O  \\nATOM    114  CB  GLN A  16     -13.578  -2.039   5.519  1.00  0.00           C  \\nATOM    115  CG  GLN A  16     -12.318  -1.724   6.275  1.00  0.00           C  \\nATOM    116  CD  GLN A  16     -11.625  -0.842   5.015  1.00  0.00           C  \\nATOM    117  OE1 GLN A  16     -10.365  -0.149   5.330  1.00  0.00           O  \\nATOM    118  NE2 GLN A  16     -12.444  -0.464   4.385  1.00  0.00           N  \\nATOM    119  N   CYS A  17     -15.216  -0.653   7.787  1.00  0.00           N  \\nATOM    120  CA  CYS A  17     -15.405  -0.905   9.299  1.00  0.00           C  \\nATOM    121  C   CYS A  17     -16.917  -0.401   9.173  1.00  0.00           C  \\nATOM    122  O   CYS A  17     -17.736  -1.283   9.173  1.00  0.00           O  \\nATOM    123  CB  CYS A  17     -14.460   0.355  10.055  1.00  0.00           C  \\nATOM    124  SG  CYS A  17     -13.326  -0.338  11.441  1.00  0.00           S  \\nATOM    125  N   HIS A  18     -17.232   0.733   9.551  1.00  0.00           N  \\nATOM    126  CA  HIS A  18     -18.870   0.985   9.803  1.00  0.00           C  \\nATOM    127  C   HIS A  18     -19.878   0.229   8.795  1.00  0.00           C  \\nATOM    128  O   HIS A  18     -19.626  -0.023   7.472  1.00  0.00           O  \\nATOM    129  CB  HIS A  18     -19.185   2.245   9.173  1.00  0.00           C  \\nATOM    130  CG  HIS A  18     -18.681   3.127  10.181  1.00  0.00           C  \\nATOM    131  ND1 HIS A  18     -18.618   3.253  11.567  1.00  0.00           N  \\nATOM    132  CD2 HIS A  18     -17.736   4.072  10.559  1.00  0.00           C  \\nATOM    133  CE1 HIS A  18     -18.051   3.694  12.827  1.00  0.00           C  \\nATOM    134  NE2 HIS A  18     -17.232   4.387  11.819  1.00  0.00           N  \\nATOM    135  N   THR A  19     -21.012   0.103   9.299  1.00  0.00           N  \\nATOM    136  CA  THR A  19     -22.272  -0.401   8.417  1.00  0.00           C  \\nATOM    137  C   THR A  19     -22.902   0.922   7.661  1.00  0.00           C  \\nATOM    138  O   THR A  19     -22.272   1.867   7.157  1.00  0.00           O  \\nATOM    139  CB  THR A  19     -23.280  -1.283   8.795  1.00  0.00           C  \\nATOM    140  OG1 THR A  19     -23.532  -0.905   9.740  1.00  0.00           O  \\nATOM    141  CG2 THR A  19     -22.776  -2.669   8.543  1.00  0.00           C  \\nATOM    142  N   VAL A  20     -24.036   1.111   7.472  1.00  0.00           N  \\nATOM    143  CA  VAL A  20     -24.414   2.371   6.779  1.00  0.00           C  \\nATOM    144  C   VAL A  20     -26.052   2.434   6.401  1.00  0.00           C  \\nATOM    145  O   VAL A  20     -26.556   3.442   5.771  1.00  0.00           O  \\nATOM    146  CB  VAL A  20     -23.784   2.560   5.519  1.00  0.00           C  \\nATOM    147  CG1 VAL A  20     -24.540   2.119   4.637  1.00  0.00           C  \\nATOM    148  CG2 VAL A  20     -23.658   4.009   5.267  1.00  0.00           C  \\nATOM    149  N   GLU A  21     -26.682   1.363   6.527  1.00  0.00           N  \\nATOM    150  CA  GLU A  21     -28.194   1.363   5.897  1.00  0.00           C  \\nATOM    151  C   GLU A  21     -29.013   1.741   7.283  1.00  0.00           C  \\nATOM    152  O   GLU A  21     -28.635   2.623   7.913  1.00  0.00           O  \\nATOM    153  CB  GLU A  21     -28.572  -0.086   5.645  1.00  0.00           C  \\nATOM    154  CG  GLU A  21     -28.698  -0.779   4.133  1.00  0.00           C  \\nATOM    155  CD  GLU A  21     -27.690  -1.913   4.007  1.00  0.00           C  \\nATOM    156  OE1 GLU A  21     -26.682  -2.039   5.141  1.00  0.00           O  \\nATOM    157  OE2 GLU A  21     -27.753  -2.921   2.873  1.00  0.00           O  \\nATOM    158  N   ASN A  22     -30.210   1.363   7.283  1.00  0.00           N  \\nATOM    159  CA  ASN A  22     -30.966   1.867   8.417  1.00  0.00           C  \\nATOM    160  C   ASN A  22     -30.714   1.237   9.677  1.00  0.00           C  \\nATOM    161  O   ASN A  22     -31.218   0.103  10.055  1.00  0.00           O  \\nATOM    162  N   GLY A  23     -29.958   1.552  10.433  1.00  0.00           N  \\nATOM    163  CA  GLY A  23     -29.517   0.859  11.819  1.00  0.00           C  \\nATOM    164  C   GLY A  23     -28.572  -0.338  11.252  1.00  0.00           C  \\nATOM    165  O   GLY A  23     -28.194  -1.283  12.071  1.00  0.00           O  \\nATOM    166  N   GLY A  24     -28.068  -0.149   9.929  1.00  0.00           N  \\nATOM    167  CA  GLY A  24     -27.060  -1.409   9.047  1.00  0.00           C  \\nATOM    168  C   GLY A  24     -26.871  -2.480  10.118  1.00  0.00           C  \\nATOM    169  O   GLY A  24     -27.186  -3.803   9.929  1.00  0.00           O  \\nATOM    170  N   LYS A  25     -26.304  -2.165  11.441  1.00  0.00           N  \\nATOM    171  CA  LYS A  25     -26.241  -3.047  12.701  1.00  0.00           C  \\nATOM    172  C   LYS A  25     -24.792  -2.795  13.079  1.00  0.00           C  \\nATOM    173  O   LYS A  25     -23.910  -2.417  12.323  1.00  0.00           O  \\nATOM    174  CB  LYS A  25     -26.241  -4.433  11.819  1.00  0.00           C  \\nATOM    175  CG  LYS A  25     -25.296  -4.496  10.433  1.00  0.00           C  \\nATOM    176  CD  LYS A  25     -24.918  -5.819   9.803  1.00  0.00           C  \\nATOM    177  CE  LYS A  25     -23.784  -5.819   8.543  1.00  0.00           C  \\nATOM    178  NZ  LYS A  25     -23.658  -6.827   7.661  1.00  0.00           N  \\nATOM    179  N   HIS A  26     -24.414  -2.921  14.465  1.00  0.00           N  \\nATOM    180  CA  HIS A  26     -23.028  -2.417  14.654  1.00  0.00           C  \\nATOM    181  C   HIS A  26     -21.894  -3.425  14.087  1.00  0.00           C  \\nATOM    182  O   HIS A  26     -22.083  -3.803  12.827  1.00  0.00           O  \\nATOM    183  CB  HIS A  26     -23.028  -2.543  16.103  1.00  0.00           C  \\nATOM    184  CG  HIS A  26     -24.036  -1.094  16.355  1.00  0.00           C  \\nATOM    185  ND1 HIS A  26     -24.540  -0.338  15.473  1.00  0.00           N  \\nATOM    186  CD2 HIS A  26     -24.666  -0.905  17.363  1.00  0.00           C  \\nATOM    187  CE1 HIS A  26     -25.422   0.355  16.103  1.00  0.00           C  \\nATOM    188  NE2 HIS A  26     -25.422   0.103  17.237  1.00  0.00           N  \\nATOM    189  N   LYS A  27     -20.886  -3.551  14.717  1.00  0.00           N  \\nATOM    190  CA  LYS A  27     -19.752  -4.433  14.213  1.00  0.00           C  \\nATOM    191  C   LYS A  27     -18.366  -3.677  14.276  1.00  0.00           C  \\nATOM    192  O   LYS A  27     -17.484  -3.677  13.331  1.00  0.00           O  \\nATOM    193  CB  LYS A  27     -20.004  -4.559  12.575  1.00  0.00           C  \\nATOM    194  CG  LYS A  27     -20.382  -3.425  11.882  1.00  0.00           C  \\nATOM    195  CD  LYS A  27     -19.500  -3.047  11.063  1.00  0.00           C  \\nATOM    196  CE  LYS A  27     -19.752  -3.677   9.551  1.00  0.00           C  \\nATOM    197  NZ  LYS A  27     -18.618  -3.677   8.858  1.00  0.00           N  \\nCONECT  104 1615                                                                \\nCONECT  124 1623                                                                \\nCONECT  134 1593                                                                \\nCONECT  356 1635                                                                \\nCONECT  616 1593                                                                \\nCONECT  900 1658                                                                \\nCONECT  920 1666                                                                \\nCONECT  930 1636                                                                \\nCONECT 1152 1678                                                                \\nCONECT 1412 1636                                                                \\nCONECT 1593  134  616 1598 1609                                                 \\nCONECT 1593 1617 1625                                                           \\nCONECT 1594 1599 1629                                                           \\nCONECT 1595 1602 1610                                                           \\nCONECT 1596 1613 1618                                                           \\nCONECT 1597 1621 1626                                                           \\nCONECT 1598 1593 1599 1602                                                      \\nCONECT 1599 1594 1598 1600                                                      \\nCONECT 1600 1599 1601 1604                                                      \\nCONECT 1601 1600 1602 1603                                                      \\nCONECT 1602 1595 1598 1601                                                      \\nCONECT 1603 1601                                                                \\nCONECT 1604 1600 1605                                                           \\nCONECT 1605 1604 1606                                                           \\nCONECT 1606 1605 1607 1608                                                      \\nCONECT 1607 1606                                                                \\nCONECT 1608 1606                                                                \\nCONECT 1609 1593 1610 1613                                                      \\nCONECT 1610 1595 1609 1611                                                      \\nCONECT 1611 1610 1612 1614                                                      \\nCONECT 1612 1611 1613 1615                                                      \\nCONECT 1613 1596 1609 1612                                                      \\nCONECT 1614 1611                                                                \\nCONECT 1615  104 1612 1616                                                      \\nCONECT 1616 1615                                                                \\nCONECT 1617 1593 1618 1621                                                      \\nCONECT 1618 1596 1617 1619                                                      \\nCONECT 1619 1618 1620 1622                                                      \\nCONECT 1620 1619 1621 1623                                                      \\nCONECT 1621 1597 1617 1620                                                      \\nCONECT 1622 1619                                                                \\nCONECT 1623  124 1620 1624                                                      \\nCONECT 1624 1623                                                                \\nCONECT 1625 1593 1626 1629                                                      \\nCONECT 1626 1597 1625 1627                                                      \\nCONECT 1627 1626 1628 1630                                                      \\nCONECT 1628 1627 1629 1631                                                      \\nCONECT 1629 1594 1625 1628                                                      \\nCONECT 1630 1627                                                                \\nCONECT 1631 1628 1632                                                           \\nCONECT 1632 1631 1633                                                           \\nCONECT 1633 1632 1634 1635                                                      \\nCONECT 1634 1633                                                                \\nCONECT 1635  356 1633                                                           \\nCONECT 1636  930 1412 1641 1652                                                 \\nCONECT 1636 1660 1668                                                           \\nCONECT 1637 1642 1672                                                           \\nCONECT 1638 1645 1653                                                           \\nCONECT 1639 1656 1661                                                           \\nCONECT 1640 1664 1669                                                           \\nCONECT 1641 1636 1642 1645                                                      \\nCONECT 1642 1637 1641 1643                                                      \\nCONECT 1643 1642 1644 1647                                                      \\nCONECT 1644 1643 1645 1646                                                      \\nCONECT 1645 1638 1641 1644                                                      \\nCONECT 1646 1644                                                                \\nCONECT 1647 1643 1648                                                           \\nCONECT 1648 1647 1649                                                           \\nCONECT 1649 1648 1650 1651                                                      \\nCONECT 1650 1649                                                                \\nCONECT 1651 1649                                                                \\nCONECT 1652 1636 1653 1656                                                      \\nCONECT 1653 1638 1652 1654                                                      \\nCONECT 1654 1653 1655 1657                                                      \\nCONECT 1655 1654 1656 1658                                                      \\nCONECT 1656 1639 1652 1655                                                      \\nCONECT 1657 1654                                                                \\nCONECT 1658  900 1655 1659                                                      \\nCONECT 1659 1658                                                                \\nCONECT 1660 1636 1661 1664                                                      \\nCONECT 1661 1639 1660 1662                                                      \\nCONECT 1662 1661 1663 1665                                                      \\nCONECT 1663 1662 1664 1666                                                      \\nCONECT 1664 1640 1660 1663                                                      \\nCONECT 1665 1662                                                                \\nCONECT 1666  920 1663 1667                                                      \\nCONECT 1667 1666                                                                \\nCONECT 1668 1636 1669 1672                                                      \\nCONECT 1669 1640 1668 1670                                                      \\nCONECT 1670 1669 1671 1673                                                      \\nCONECT 1671 1670 1672 1674                                                      \\nCONECT 1672 1637 1668 1671                                                      \\nCONECT 1673 1670                                                                \\nCONECT 1674 1671 1675                                                           \\nCONECT 1675 1674 1676                                                           \\nCONECT 1676 1675 1677 1678                                                      \\nCONECT 1677 1676                                                                \\nCONECT 1678 1152 1676                                                           \\nMASTER      287    0    2   10    0    0    9    9 1677    2   98   16          \\nEND                                                                             \\n\",\"pdb\");\n",
       "\tviewer_16742555228337302.setBackgroundColor(\"white\");\n",
       "\tviewer_16742555228337302.setStyle({\"cartoon\": {\"color\": \"white\"}});\n",
       "\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"5\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"7\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"5\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"8\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"5\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"6\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"8\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"6\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"6\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"7\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"7\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"11\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"7\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"15\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"8\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"8\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"11\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"8\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"15\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"9\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"13\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"15\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"16\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"11\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"13\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"11\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"12\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"13\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"15\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"13\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"16\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"16\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"18\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"15\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"18\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"16\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"18\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"19\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"27\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"18\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"18\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"24\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"18\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"26\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"19\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"26\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"21\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"25\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"23\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"25\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"23\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"26\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"24\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"26\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228337302.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"25\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"27\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228337302.zoomTo();\n",
       "viewer_16742555228337302.render();\n",
       "});\n",
       "</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "<py3Dmol.view at 0x7f127a2bd700>"
      ]
     },
     "execution_count": 50,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "view_frustration(path_bs+'exon_0.pdb',frustra_ex)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "def view_frustration_dif(pdb_filename_all,pdb_filename_exon,frustra_table):\n",
    "    view = py3Dmol.view(js='https://3dmol.org/build/3Dmol.js')\n",
    "    view.addModel(open(pdb_filename_all,'r').read(),'pdb')\n",
    "\n",
    "    view.setBackgroundColor('white')\n",
    "    view.setStyle({'cartoon':{'color':'white'}})\n",
    "    \n",
    "    for res in np.unique(np.hstack([fr_dif['#Res1_x'],fr_dif.Res2_x])):\n",
    "        view.addStyle({'chain':'A','resi':[str(res)]},{'cartoon':{'color':'grey'}})\n",
    "    \n",
    "    #OJO SI HAY UN -2 O +2 LO ROMPO, CORREGIR\n",
    "    frustra_color=['red','blue','green'] # worse, equal, better \n",
    "    frustra_num=frustra_table.fr_dif\n",
    "\n",
    "    for x in range(len(frustra_table)):\n",
    "        if frustra_num[x]!=0:\n",
    "            view.addLine({'start':{'chain':'A','resi':[str(frustra_table['#Res1_x'][x])]},\n",
    "                          'end':{'chain':'A','resi':[str(frustra_table['Res2_x'][x])]},\n",
    "             'color':frustra_color[frustra_num[x]+1], 'dashed':False,'linewidth':3})\n",
    "  \n",
    "    view.zoomTo(viewer=(0,0))\n",
    "\n",
    "    return view"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_16742555228826048\"  style=\"position: relative; width: 640px; height: 480px\">\n        <p id=\"3dmolwarning_16742555228826048\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    //this is to ignore the existence of requirejs amd\n    var savedexports, savedmodule;\n    if (typeof exports !== 'undefined') savedexports = exports;\n    else exports = {}\n    if (typeof module !== 'undefined') savedmodule = module;\n    else module = {}\n\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n        exports = savedexports;\n        module = savedmodule;\n        resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://3dmol.org/build/3Dmol.js');\n}\n\nvar viewer_16742555228826048 = null;\nvar warn = document.getElementById(\"3dmolwarning_16742555228826048\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_16742555228826048 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_16742555228826048\"),{backgroundColor:\"white\"});\nviewer_16742555228826048.zoomTo();\n\tviewer_16742555228826048.addModel(\"HEADER    ELECTRON TRANSPORT                      01-AUG-76   1CYC              \\nTITLE     THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C AT 2.3     \\nTITLE    2 ANGSTROMS RESOLUTION. II. STRUCTURE AND FUNCTION                     \\nCAVEAT     1CYC    THR A 40 HAS WRONG CHIRALITY AT ATOM CB THR A 78 HAS WRONG   \\nCAVEAT   2 1CYC    CHIRALITY AT ATOM CB THR B 40 HAS WRONG CHIRALITY AT ATOM    \\nCAVEAT   3 1CYC    CB THR B 78 HAS WRONG CHIRALITY AT ATOM CB                   \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: FERROCYTOCHROME C;                                         \\nCOMPND   3 CHAIN: A, B;                                                         \\nCOMPND   4 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 ORGANISM_SCIENTIFIC: KATSUWONUS PELAMIS;                             \\nSOURCE   3 ORGANISM_COMMON: SKIPJACK TUNA;                                      \\nSOURCE   4 ORGANISM_TAXID: 8226                                                 \\nKEYWDS    ELECTRON TRANSPORT                                                    \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    N.TANAKA,T.YAMANE,T.TSUKIHARA,T.ASHIDA,M.KAKUDO                       \\nREVDAT  17   03-MAR-21 1CYC    1       CAVEAT COMPND REMARK HET                 \\nREVDAT  17 2                   1       HETNAM HETSYN FORMUL LINK                \\nREVDAT  17 3                   1       ATOM                                     \\nREVDAT  16   29-NOV-17 1CYC    1       HELIX                                    \\nREVDAT  15   24-FEB-09 1CYC    1       VERSN                                    \\nREVDAT  14   01-APR-03 1CYC    1       JRNL                                     \\nREVDAT  13   27-JAN-84 1CYC    1       REMARK                                   \\nREVDAT  12   30-SEP-83 1CYC    1       REVDAT                                   \\nREVDAT  11   19-APR-83 1CYC    1       COMPND SOURCE REMARK                     \\nREVDAT  10   15-JAN-82 1CYC    1       REMARK                                   \\nREVDAT   9   06-JAN-82 1CYC    3       SCALE                                    \\nREVDAT   8   31-DEC-80 1CYC    1       REMARK                                   \\nREVDAT   7   07-APR-80 1CYC    3       SOURCE SEQRES ATOM                       \\nREVDAT   6   23-APR-79 1CYC    3       ATOM                                     \\nREVDAT   5   01-NOV-77 1CYC    1       AUTHOR REMARK FORMUL                     \\nREVDAT   4   09-SEP-77 1CYC    3       ATOM                                     \\nREVDAT   3   13-JUN-77 1CYC    1       HET                                      \\nREVDAT   2   25-OCT-76 1CYC    2       CONECT                                   \\nREVDAT   1   06-OCT-76 1CYC    0                                                \\nJRNL        AUTH   N.TANAKA,T.YAMANE,T.TSUKIHARA,T.ASHIDA,M.KAKUDO              \\nJRNL        TITL   THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C   \\nJRNL        TITL 2 AT 2.3 A RESOLUTION. II. STRUCTURE AND FUNCTION.             \\nJRNL        REF    J.BIOCHEM.(TOKYO)             V.  77   147 1975              \\nJRNL        REFN                   ISSN 0021-924X                               \\nJRNL        PMID   166072                                                       \\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   T.TSUKIHARA,T.YAMANE,N.TANAKA,T.ASHIDA,M.KAKUDO              \\nREMARK   1  TITL   OXIDATION OF A FERROCYTOCHROME C IN THE CRYSTALLINE          \\nREMARK   1  TITL 2 STATE-STRUCTURAL CHANGE AND ANION BINDING                    \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  73  1163 1973              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 2                                                          \\nREMARK   1  AUTH   T.ASHIDA,N.TANAKA,T.YAMANE,T.TSUKIHARA,M.KAKUDO              \\nREMARK   1  TITL   THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C   \\nREMARK   1  TITL 2 AT 2.3 ANGSTROMS RESOLUTION                                  \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  73   463 1973              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 3                                                          \\nREMARK   1  AUTH   T.ASHIDA,T.UEKI,T.TSUKIHARA,A.SUGIHARA,T.TAKANO,M.KAKUDO     \\nREMARK   1  TITL   THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C   \\nREMARK   1  TITL 2 AT 4 ANGSTROMS RESOLUTION                                    \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  70   913 1971              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 4                                                          \\nREMARK   1  AUTH   T.NAKAYAMA,K.TITANI,K.NARITA                                 \\nREMARK   1  TITL   THE AMINO ACID SEQUENCE OF CYTOCHROME C FROM BONITO          \\nREMARK   1  TITL 2 (KATSUWONUS PELAMIS, LINNAEUS)                               \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  70   311 1971              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 5                                                          \\nREMARK   1  EDIT   R.J.FELDMANN                                                 \\nREMARK   1  REF    ATLAS OF MACROMOLECULAR                120 1976              \\nREMARK   1  REF  2 STRUCTURE ON MICROFICHE                                      \\nREMARK   1  PUBL   TRACOR JITCO INC.,ROCKVILLE,MD.                              \\nREMARK   1  REFN                                                                \\nREMARK   1 REFERENCE 6                                                          \\nREMARK   1  EDIT   M.O.DAYHOFF                                                  \\nREMARK   1  REF    ATLAS OF PROTEIN SEQUENCE     V.   5    39 1976              \\nREMARK   1  REF  2 AND STRUCTURE,SUPPLEMENT 2                                   \\nREMARK   1  PUBL   NATIONAL BIOMEDICAL RESEARCH FOUNDATION, SILVER SPRING,MD.   \\nREMARK   1  REFN                   ISSN 0-912466-05-7                           \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    2.30 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : NULL                                                 \\nREMARK   3   AUTHORS     : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 2.30                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : NULL                           \\nREMARK   3   DATA CUTOFF            (SIGMA(F)) : NULL                           \\nREMARK   3   DATA CUTOFF HIGH         (ABS(F)) : NULL                           \\nREMARK   3   DATA CUTOFF LOW          (ABS(F)) : NULL                           \\nREMARK   3   COMPLETENESS (WORKING+TEST)   (%) : NULL                           \\nREMARK   3   NUMBER OF REFLECTIONS             : NULL                           \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   CROSS-VALIDATION METHOD          : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SELECTION  : NULL                            \\nREMARK   3   R VALUE            (WORKING SET) : NULL                            \\nREMARK   3   FREE R VALUE                     : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : NULL                            \\nREMARK   3   FREE R VALUE TEST SET COUNT      : NULL                            \\nREMARK   3   ESTIMATED ERROR OF FREE R VALUE  : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  FIT IN THE HIGHEST RESOLUTION BIN.                                  \\nREMARK   3   TOTAL NUMBER OF BINS USED           : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE HIGH       (A) : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE LOW        (A) : NULL                         \\nREMARK   3   BIN COMPLETENESS (WORKING+TEST) (%) : NULL                         \\nREMARK   3   REFLECTIONS IN BIN    (WORKING SET) : NULL                         \\nREMARK   3   BIN R VALUE           (WORKING SET) : NULL                         \\nREMARK   3   BIN FREE R VALUE                    : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET SIZE  (%) : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET COUNT     : NULL                         \\nREMARK   3   ESTIMATED ERROR OF BIN FREE R VALUE : NULL                         \\nREMARK   3                                                                      \\nREMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \\nREMARK   3   PROTEIN ATOMS            : 1590                                    \\nREMARK   3   NUCLEIC ACID ATOMS       : 0                                       \\nREMARK   3   HETEROGEN ATOMS          : 86                                      \\nREMARK   3   SOLVENT ATOMS            : 1                                       \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : NULL                           \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : NULL                           \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : NULL                                                 \\nREMARK   3    B22 (A**2) : NULL                                                 \\nREMARK   3    B33 (A**2) : NULL                                                 \\nREMARK   3    B12 (A**2) : NULL                                                 \\nREMARK   3    B13 (A**2) : NULL                                                 \\nREMARK   3    B23 (A**2) : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  ESTIMATED COORDINATE ERROR.                                         \\nREMARK   3   ESD FROM LUZZATI PLOT        (A) : NULL                            \\nREMARK   3   ESD FROM SIGMAA              (A) : NULL                            \\nREMARK   3   LOW RESOLUTION CUTOFF        (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  CROSS-VALIDATED ESTIMATED COORDINATE ERROR.                         \\nREMARK   3   ESD FROM C-V LUZZATI PLOT    (A) : NULL                            \\nREMARK   3   ESD FROM C-V SIGMAA          (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  RMS DEVIATIONS FROM IDEAL VALUES.                                   \\nREMARK   3   BOND LENGTHS                 (A) : NULL                            \\nREMARK   3   BOND ANGLES            (DEGREES) : NULL                            \\nREMARK   3   DIHEDRAL ANGLES        (DEGREES) : NULL                            \\nREMARK   3   IMPROPER ANGLES        (DEGREES) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL MODEL : NULL                                      \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL FACTOR RESTRAINTS.    RMS    SIGMA                \\nREMARK   3   MAIN-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   MAIN-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  NCS MODEL : NULL                                                    \\nREMARK   3                                                                      \\nREMARK   3  NCS RESTRAINTS.                         RMS   SIGMA/WEIGHT          \\nREMARK   3   GROUP  1  POSITIONAL            (A) : NULL  ; NULL                 \\nREMARK   3   GROUP  1  B-FACTOR           (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  PARAMETER FILE  1  : NULL                                           \\nREMARK   3  TOPOLOGY FILE  1   : NULL                                           \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS: NULL                                      \\nREMARK   4                                                                      \\nREMARK   4 1CYC COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY BNL.                                \\nREMARK 100 THE DEPOSITION ID IS D_1000172595.                                   \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : NULL                               \\nREMARK 200  TEMPERATURE           (KELVIN) : NULL                               \\nREMARK 200  PH                             : NULL                               \\nREMARK 200  NUMBER OF CRYSTALS USED        : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : NULL                               \\nREMARK 200  RADIATION SOURCE               : NULL                               \\nREMARK 200  BEAMLINE                       : NULL                               \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : NULL                               \\nREMARK 200  WAVELENGTH OR RANGE        (A) : NULL                               \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : NULL                               \\nREMARK 200  DETECTOR MANUFACTURER          : NULL                               \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : NULL                               \\nREMARK 200  DATA SCALING SOFTWARE          : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : NULL                               \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : NULL                               \\nREMARK 200  RESOLUTION RANGE LOW       (A) : NULL                               \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY                : NULL                               \\nREMARK 200  R MERGE                    (I) : NULL                               \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : NULL                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : NULL                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : NULL                               \\nREMARK 200  R MERGE FOR SHELL          (I) : NULL                               \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: NULL                                           \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: NULL                         \\nREMARK 200 SOFTWARE USED: NULL                                                  \\nREMARK 200 STARTING MODEL: NULL                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: NULL                                                         \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): 39.18                                     \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 2.02                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: NULL                                     \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: P 21 21 21                       \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -X+1/2,-Y,Z+1/2                                         \\nREMARK 290       3555   -X,Y+1/2,-Z+1/2                                         \\nREMARK 290       4555   X+1/2,-Y+1/2,-Z                                         \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -1.000000  0.000000  0.000000       28.84000            \\nREMARK 290   SMTRY2   2  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000       18.91500            \\nREMARK 290   SMTRY1   3 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   3  0.000000  1.000000  0.000000       42.29000            \\nREMARK 290   SMTRY3   3  0.000000  0.000000 -1.000000       18.91500            \\nREMARK 290   SMTRY1   4  1.000000  0.000000  0.000000       28.84000            \\nREMARK 290   SMTRY2   4  0.000000 -1.000000  0.000000       42.29000            \\nREMARK 290   SMTRY3   4  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1, 2                                                    \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: MONOMERIC                         \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 2                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: MONOMERIC                         \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: B                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 470                                                                      \\nREMARK 470 MISSING ATOM                                                         \\nREMARK 470 THE FOLLOWING RESIDUES HAVE MISSING ATOMS (M=MODEL NUMBER;           \\nREMARK 470 RES=RESIDUE NAME; C=CHAIN IDENTIFIER; SSEQ=SEQUENCE NUMBER;          \\nREMARK 470 I=INSERTION CODE):                                                   \\nREMARK 470   M RES CSSEQI  ATOMS                                                \\nREMARK 470     ASN A  22    CB   CG   OD1  ND2                                  \\nREMARK 470     LYS A  39    CE   NZ                                             \\nREMARK 470     ASN B  22    CB   CG   OD1  ND2                                  \\nREMARK 470     LYS B  39    CE   NZ                                             \\nREMARK 620                                                                      \\nREMARK 620 METAL COORDINATION                                                   \\nREMARK 620 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 620 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE):                             \\nREMARK 620                                                                      \\nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \\nREMARK 620                             HEC A 104  FE                            \\nREMARK 620 N RES CSSEQI ATOM                                                    \\nREMARK 620 1 HIS A  18   NE2                                                    \\nREMARK 620 2 HEC A 104   NA   87.0                                              \\nREMARK 620 3 HEC A 104   NB   87.8  90.6                                        \\nREMARK 620 4 HEC A 104   NC   92.0 178.7  90.1                                  \\nREMARK 620 5 HEC A 104   ND   91.1  86.9 177.4  92.3                            \\nREMARK 620 6 MET A  80   SD  170.9 102.1  93.0  79.0  88.6                      \\nREMARK 620 N                    1     2     3     4     5                       \\nREMARK 620                                                                      \\nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \\nREMARK 620                             HEC B 104  FE                            \\nREMARK 620 N RES CSSEQI ATOM                                                    \\nREMARK 620 1 HIS B  18   NE2                                                    \\nREMARK 620 2 HEC B 104   NA   87.0                                              \\nREMARK 620 3 HEC B 104   NB   87.8  90.6                                        \\nREMARK 620 4 HEC B 104   NC   92.0 178.7  90.1                                  \\nREMARK 620 5 HEC B 104   ND   91.1  86.9 177.4  92.3                            \\nREMARK 620 6 MET B  80   SD  170.9 102.1  93.0  79.0  88.6                      \\nREMARK 620 N                    1     2     3     4     5                       \\nREMARK 800                                                                      \\nREMARK 800 SITE                                                                 \\nREMARK 800 SITE_IDENTIFIER: AC1                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE HEC A 104                 \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: AC2                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE HEC B 104                 \\nDBREF  1CYC A    1   103  UNP    P00025   CYC_KATPE        1    103             \\nDBREF  1CYC B    1   103  UNP    P00025   CYC_KATPE        1    103             \\nSEQRES   1 A  103  GLY ASP VAL ALA LYS GLY LYS LYS THR PHE VAL GLN LYS          \\nSEQRES   2 A  103  CYS ALA GLN CYS HIS THR VAL GLU ASN GLY GLY LYS HIS          \\nSEQRES   3 A  103  LYS VAL GLY PRO ASN LEU TRP GLY LEU PHE GLY ARG LYS          \\nSEQRES   4 A  103  THR GLY GLN ALA GLU GLY TYR SER TYR THR ASP ALA ASN          \\nSEQRES   5 A  103  LYS SER LYS GLY ILE VAL TRP ASN GLU ASN THR LEU MET          \\nSEQRES   6 A  103  GLU TYR LEU GLU ASN PRO LYS LYS TYR ILE PRO GLY THR          \\nSEQRES   7 A  103  LYS MET ILE PHE ALA GLY ILE LYS LYS LYS GLY GLU ARG          \\nSEQRES   8 A  103  GLN ASP LEU VAL ALA TYR LEU LYS SER ALA THR SER              \\nSEQRES   1 B  103  GLY ASP VAL ALA LYS GLY LYS LYS THR PHE VAL GLN LYS          \\nSEQRES   2 B  103  CYS ALA GLN CYS HIS THR VAL GLU ASN GLY GLY LYS HIS          \\nSEQRES   3 B  103  LYS VAL GLY PRO ASN LEU TRP GLY LEU PHE GLY ARG LYS          \\nSEQRES   4 B  103  THR GLY GLN ALA GLU GLY TYR SER TYR THR ASP ALA ASN          \\nSEQRES   5 B  103  LYS SER LYS GLY ILE VAL TRP ASN GLU ASN THR LEU MET          \\nSEQRES   6 B  103  GLU TYR LEU GLU ASN PRO LYS LYS TYR ILE PRO GLY THR          \\nSEQRES   7 B  103  LYS MET ILE PHE ALA GLY ILE LYS LYS LYS GLY GLU ARG          \\nSEQRES   8 B  103  GLN ASP LEU VAL ALA TYR LEU LYS SER ALA THR SER              \\nHET    HEC  A 104      43                                                       \\nHET    HEC  B 104      43                                                       \\nHETNAM     HEC HEME C                                                           \\nFORMUL   3  HEC    2(C34 H34 FE N4 O4)                                          \\nFORMUL   5  HOH   *(H2 O)                                                       \\nHELIX    1  H1 GLY A    1  VAL A   11  1                                  11    \\nHELIX    2  H4 CYS A   14  CYS A   17  514 AND 17 BOUND TO HEME GROUP      4    \\nHELIX    3  H5 THR A   49  LYS A   55  5LOOSE FROM 49-53,3/10 53-55        7    \\nHELIX    4  H2 ASN A   60  GLU A   69  1                                  10    \\nHELIX    5  H3 GLU A   90  SER A  103  1                                  14    \\nHELIX    6  H6 GLY B    1  VAL B   11  1                                  11    \\nHELIX    7  H7 CYS B   14  CYS B   17  514 AND 17 BOUND TO HEME GROUP      4    \\nHELIX    8  H8 THR B   49  LYS B   55  5LOOSE FROM 49-53,3/10 53-55        7    \\nHELIX    9  H9 ASN B   60  GLU B   69  1                                  10    \\nHELIX   10 H10 GLU B   90  SER B  103  1                                  14    \\nLINK         SG  CYS A  14                 CAB HEC A 104     1555   1555  1.48  \\nLINK         SG  CYS A  17                 CAC HEC A 104     1555   1555  1.86  \\nLINK         CE2 TYR A  48                 O2D HEC A 104     1555   1555  2.02  \\nLINK         SG  CYS B  14                 CAB HEC B 104     1555   1555  1.48  \\nLINK         SG  CYS B  17                 CAC HEC B 104     1555   1555  1.86  \\nLINK         CE2 TYR B  48                 O2D HEC B 104     1555   1555  2.02  \\nLINK         NE2 HIS A  18                FE   HEC A 104     1555   1555  2.60  \\nLINK         SD  MET A  80                FE   HEC A 104     1555   1555  2.49  \\nLINK         NE2 HIS B  18                FE   HEC B 104     1555   1555  2.60  \\nLINK         SD  MET B  80                FE   HEC B 104     1555   1555  2.49  \\nSITE     1 AC1 17 LYS A  13  CYS A  14  CYS A  17  HIS A  18                    \\nSITE     2 AC1 17 LEU A  32  LEU A  35  ARG A  38  THR A  40                    \\nSITE     3 AC1 17 TYR A  46  TYR A  48  ASN A  52  TRP A  59                    \\nSITE     4 AC1 17 THR A  78  LYS A  79  MET A  80  PHE A  82                    \\nSITE     5 AC1 17 HOH A 105                                                     \\nSITE     1 AC2 16 LYS B  13  CYS B  14  CYS B  17  HIS B  18                    \\nSITE     2 AC2 16 LEU B  32  LEU B  35  ARG B  38  THR B  40                    \\nSITE     3 AC2 16 TYR B  46  TYR B  48  ASN B  52  TRP B  59                    \\nSITE     4 AC2 16 THR B  78  LYS B  79  MET B  80  PHE B  82                    \\nCRYST1   57.680   84.580   37.830  90.00  90.00  90.00 P 21 21 21    8          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.017337  0.000000  0.000000        0.00000                         \\nSCALE2      0.000000  0.011823  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.026434        0.00000                         \\nMTRIX1   1  1.000000  0.000000  0.000000       28.84000    1                    \\nMTRIX2   1  0.000000  1.000000  0.000000        0.00000    1                    \\nMTRIX3   1  0.000000  0.000000  1.000000        0.00000    1                    \\nATOM      1  N   GLY A   1     -21.138  13.774  -7.711  1.00  0.00           N  \\nATOM      2  CA  GLY A   1     -19.878  14.593  -7.963  1.00  0.00           C  \\nATOM      3  C   GLY A   1     -18.996  14.530  -6.703  1.00  0.00           C  \\nATOM      4  O   GLY A   1     -18.114  13.711  -6.640  1.00  0.00           O  \\nATOM      5  N   ASP A   2     -18.996  15.412  -5.821  1.00  0.00           N  \\nATOM      6  CA  ASP A   2     -17.610  15.097  -5.191  1.00  0.00           C  \\nATOM      7  C   ASP A   2     -17.862  14.152  -3.931  1.00  0.00           C  \\nATOM      8  O   ASP A   2     -17.106  13.333  -3.553  1.00  0.00           O  \\nATOM      9  CB  ASP A   2     -17.169  16.546  -4.435  1.00  0.00           C  \\nATOM     10  CG  ASP A   2     -17.988  17.365  -3.427  1.00  0.00           C  \\nATOM     11  OD1 ASP A   2     -19.500  17.365  -3.994  1.00  0.00           O  \\nATOM     12  OD2 ASP A   2     -17.484  18.373  -2.293  1.00  0.00           O  \\nATOM     13  N   VAL A   3     -19.059  14.089  -3.679  1.00  0.00           N  \\nATOM     14  CA  VAL A   3     -19.815  12.955  -2.860  1.00  0.00           C  \\nATOM     15  C   VAL A   3     -19.374  11.506  -3.805  1.00  0.00           C  \\nATOM     16  O   VAL A   3     -18.870  10.624  -3.427  1.00  0.00           O  \\nATOM     17  CB  VAL A   3     -22.587  13.270  -3.175  1.00  0.00           C  \\nATOM     18  CG1 VAL A   3     -21.957  11.947  -3.427  1.00  0.00           C  \\nATOM     19  CG2 VAL A   3     -21.705  13.963  -2.167  1.00  0.00           C  \\nATOM     20  N   ALA A   4     -19.437  11.380  -5.002  1.00  0.00           N  \\nATOM     21  CA  ALA A   4     -18.681  10.120  -5.569  1.00  0.00           C  \\nATOM     22  C   ALA A   4     -17.358   9.805  -4.939  1.00  0.00           C  \\nATOM     23  O   ALA A   4     -17.295   8.923  -4.183  1.00  0.00           O  \\nATOM     24  CB  ALA A   4     -18.429  10.435  -7.144  1.00  0.00           C  \\nATOM     25  N   LYS A   5     -16.476  10.687  -5.191  1.00  0.00           N  \\nATOM     26  CA  LYS A   5     -15.216  10.498  -4.372  1.00  0.00           C  \\nATOM     27  C   LYS A   5     -15.279  10.246  -2.797  1.00  0.00           C  \\nATOM     28  O   LYS A   5     -14.838   9.238  -2.293  1.00  0.00           O  \\nATOM     29  CB  LYS A   5     -14.208  11.695  -4.246  1.00  0.00           C  \\nATOM     30  CG  LYS A   5     -12.948  11.758  -3.553  1.00  0.00           C  \\nATOM     31  CD  LYS A   5     -11.814  13.081  -3.679  1.00  0.00           C  \\nATOM     32  CE  LYS A   5     -10.806  13.144  -2.545  1.00  0.00           C  \\nATOM     33  NZ  LYS A   5      -9.735  14.404  -2.293  1.00  0.00           N  \\nATOM     34  N   GLY A   6     -16.161  10.750  -1.915  1.00  0.00           N  \\nATOM     35  CA  GLY A   6     -16.350  10.435  -0.403  1.00  0.00           C  \\nATOM     36  C   GLY A   6     -16.539   8.923  -0.655  1.00  0.00           C  \\nATOM     37  O   GLY A   6     -15.909   8.041  -0.025  1.00  0.00           O  \\nATOM     38  N   LYS A   7     -17.421   8.419  -1.411  1.00  0.00           N  \\nATOM     39  CA  LYS A   7     -17.799   6.844  -1.033  1.00  0.00           C  \\nATOM     40  C   LYS A   7     -16.476   6.151  -1.474  1.00  0.00           C  \\nATOM     41  O   LYS A   7     -15.972   4.954  -0.970  1.00  0.00           O  \\nATOM     42  CB  LYS A   7     -18.870   6.403  -2.293  1.00  0.00           C  \\nATOM     43  CG  LYS A   7     -18.870   5.017  -2.734  1.00  0.00           C  \\nATOM     44  CD  LYS A   7     -20.193   4.765  -3.427  1.00  0.00           C  \\nATOM     45  CE  LYS A   7     -20.193   3.190  -4.057  1.00  0.00           C  \\nATOM     46  NZ  LYS A   7     -21.516   2.623  -4.309  1.00  0.00           N  \\nATOM     47  N   LYS A   8     -15.846   6.403  -2.545  1.00  0.00           N  \\nATOM     48  CA  LYS A   8     -14.460   5.899  -2.923  1.00  0.00           C  \\nATOM     49  C   LYS A   8     -13.452   5.899  -1.663  1.00  0.00           C  \\nATOM     50  O   LYS A   8     -13.011   5.017  -1.348  1.00  0.00           O  \\nATOM     51  CB  LYS A   8     -13.704   6.844  -3.931  1.00  0.00           C  \\nATOM     52  CG  LYS A   8     -12.318   6.529  -4.183  1.00  0.00           C  \\nATOM     53  CD  LYS A   8     -11.562   7.537  -5.065  1.00  0.00           C  \\nATOM     54  CE  LYS A   8     -10.176   7.222  -5.821  1.00  0.00           C  \\nATOM     55  NZ  LYS A   8      -9.357   8.419  -6.325  1.00  0.00           N  \\nATOM     56  N   THR A   9     -13.452   6.970  -1.159  1.00  0.00           N  \\nATOM     57  CA  THR A   9     -12.633   7.033   0.227  1.00  0.00           C  \\nATOM     58  C   THR A   9     -13.515   6.277   1.298  1.00  0.00           C  \\nATOM     59  O   THR A   9     -12.948   5.773   2.369  1.00  0.00           O  \\nATOM     60  CB  THR A   9     -12.381   8.419   0.920  1.00  0.00           C  \\nATOM     61  OG1 THR A   9     -11.058   8.923   0.542  1.00  0.00           O  \\nATOM     62  CG2 THR A   9     -12.444   8.734   2.558  1.00  0.00           C  \\nATOM     63  N   PHE A  10     -14.775   6.403   1.235  1.00  0.00           N  \\nATOM     64  CA  PHE A  10     -15.531   5.647   2.117  1.00  0.00           C  \\nATOM     65  C   PHE A  10     -15.216   4.135   1.739  1.00  0.00           C  \\nATOM     66  O   PHE A  10     -14.460   3.757   2.495  1.00  0.00           O  \\nATOM     67  CB  PHE A  10     -17.106   5.647   1.991  1.00  0.00           C  \\nATOM     68  CG  PHE A  10     -17.736   4.891   3.251  1.00  0.00           C  \\nATOM     69  CD1 PHE A  10     -18.492   3.757   2.747  1.00  0.00           C  \\nATOM     70  CD2 PHE A  10     -17.862   5.143   4.133  1.00  0.00           C  \\nATOM     71  CE1 PHE A  10     -18.996   3.001   3.818  1.00  0.00           C  \\nATOM     72  CE2 PHE A  10     -18.366   4.513   4.889  1.00  0.00           C  \\nATOM     73  CZ  PHE A  10     -18.996   3.379   5.015  1.00  0.00           C  \\nATOM     74  N   VAL A  11     -15.216   3.379   0.479  1.00  0.00           N  \\nATOM     75  CA  VAL A  11     -14.964   2.056   0.605  1.00  0.00           C  \\nATOM     76  C   VAL A  11     -13.326   1.867   1.109  1.00  0.00           C  \\nATOM     77  O   VAL A  11     -12.948   0.985   1.865  1.00  0.00           O  \\nATOM     78  CB  VAL A  11     -15.153   1.237  -0.529  1.00  0.00           C  \\nATOM     79  CG1 VAL A  11     -16.098   1.930  -1.537  1.00  0.00           C  \\nATOM     80  CG2 VAL A  11     -13.956   1.111  -1.285  1.00  0.00           C  \\nATOM     81  N   GLN A  12     -12.570   2.749   0.794  1.00  0.00           N  \\nATOM     82  CA  GLN A  12     -10.995   2.749   1.235  1.00  0.00           C  \\nATOM     83  C   GLN A  12     -10.680   2.938   2.873  1.00  0.00           C  \\nATOM     84  O   GLN A  12     -10.176   1.930   3.377  1.00  0.00           O  \\nATOM     85  CB  GLN A  12     -10.428   3.820   0.479  1.00  0.00           C  \\nATOM     86  CG  GLN A  12      -9.861   3.253  -0.718  1.00  0.00           C  \\nATOM     87  CD  GLN A  12      -9.168   4.135  -2.293  1.00  0.00           C  \\nATOM     88  OE1 GLN A  12      -7.593   3.820  -2.041  1.00  0.00           O  \\nATOM     89  NE2 GLN A  12      -9.672   5.143  -2.671  1.00  0.00           N  \\nATOM     90  N   LYS A  13     -10.995   3.757   3.629  1.00  0.00           N  \\nATOM     91  CA  LYS A  13     -10.869   3.757   5.015  1.00  0.00           C  \\nATOM     92  C   LYS A  13     -12.066   3.694   5.960  1.00  0.00           C  \\nATOM     93  O   LYS A  13     -12.003   3.127   6.779  1.00  0.00           O  \\nATOM     94  CB  LYS A  13     -10.239   5.017   5.141  1.00  0.00           C  \\nATOM     95  CG  LYS A  13      -9.672   5.521   3.881  1.00  0.00           C  \\nATOM     96  CD  LYS A  13      -8.286   6.277   4.007  1.00  0.00           C  \\nATOM     97  CE  LYS A  13      -8.160   6.907   2.621  1.00  0.00           C  \\nATOM     98  NZ  LYS A  13      -6.837   7.789   2.243  1.00  0.00           N  \\nATOM     99  N   CYS A  14     -13.137   4.072   5.519  1.00  0.00           N  \\nATOM    100  CA  CYS A  14     -14.460   4.072   6.464  1.00  0.00           C  \\nATOM    101  C   CYS A  14     -15.216   2.812   5.897  1.00  0.00           C  \\nATOM    102  O   CYS A  14     -15.342   1.804   6.716  1.00  0.00           O  \\nATOM    103  CB  CYS A  14     -15.468   5.332   5.897  1.00  0.00           C  \\nATOM    104  SG  CYS A  14     -14.712   6.844   6.464  1.00  0.00           S  \\nATOM    105  N   ALA A  15     -15.720   2.623   4.637  1.00  0.00           N  \\nATOM    106  CA  ALA A  15     -16.161   1.237   4.322  1.00  0.00           C  \\nATOM    107  C   ALA A  15     -16.035  -0.086   5.015  1.00  0.00           C  \\nATOM    108  O   ALA A  15     -17.043  -0.590   5.393  1.00  0.00           O  \\nATOM    109  CB  ALA A  15     -15.405   0.985   3.062  1.00  0.00           C  \\nATOM    110  N   GLN A  16     -14.964  -0.653   4.889  1.00  0.00           N  \\nATOM    111  CA  GLN A  16     -14.901  -2.039   5.897  1.00  0.00           C  \\nATOM    112  C   GLN A  16     -15.090  -1.787   7.283  1.00  0.00           C  \\nATOM    113  O   GLN A  16     -14.838  -2.669   8.291  1.00  0.00           O  \\nATOM    114  CB  GLN A  16     -13.578  -2.039   5.519  1.00  0.00           C  \\nATOM    115  CG  GLN A  16     -12.318  -1.724   6.275  1.00  0.00           C  \\nATOM    116  CD  GLN A  16     -11.625  -0.842   5.015  1.00  0.00           C  \\nATOM    117  OE1 GLN A  16     -10.365  -0.149   5.330  1.00  0.00           O  \\nATOM    118  NE2 GLN A  16     -12.444  -0.464   4.385  1.00  0.00           N  \\nATOM    119  N   CYS A  17     -15.216  -0.653   7.787  1.00  0.00           N  \\nATOM    120  CA  CYS A  17     -15.405  -0.905   9.299  1.00  0.00           C  \\nATOM    121  C   CYS A  17     -16.917  -0.401   9.173  1.00  0.00           C  \\nATOM    122  O   CYS A  17     -17.736  -1.283   9.173  1.00  0.00           O  \\nATOM    123  CB  CYS A  17     -14.460   0.355  10.055  1.00  0.00           C  \\nATOM    124  SG  CYS A  17     -13.326  -0.338  11.441  1.00  0.00           S  \\nATOM    125  N   HIS A  18     -17.232   0.733   9.551  1.00  0.00           N  \\nATOM    126  CA  HIS A  18     -18.870   0.985   9.803  1.00  0.00           C  \\nATOM    127  C   HIS A  18     -19.878   0.229   8.795  1.00  0.00           C  \\nATOM    128  O   HIS A  18     -19.626  -0.023   7.472  1.00  0.00           O  \\nATOM    129  CB  HIS A  18     -19.185   2.245   9.173  1.00  0.00           C  \\nATOM    130  CG  HIS A  18     -18.681   3.127  10.181  1.00  0.00           C  \\nATOM    131  ND1 HIS A  18     -18.618   3.253  11.567  1.00  0.00           N  \\nATOM    132  CD2 HIS A  18     -17.736   4.072  10.559  1.00  0.00           C  \\nATOM    133  CE1 HIS A  18     -18.051   3.694  12.827  1.00  0.00           C  \\nATOM    134  NE2 HIS A  18     -17.232   4.387  11.819  1.00  0.00           N  \\nATOM    135  N   THR A  19     -21.012   0.103   9.299  1.00  0.00           N  \\nATOM    136  CA  THR A  19     -22.272  -0.401   8.417  1.00  0.00           C  \\nATOM    137  C   THR A  19     -22.902   0.922   7.661  1.00  0.00           C  \\nATOM    138  O   THR A  19     -22.272   1.867   7.157  1.00  0.00           O  \\nATOM    139  CB  THR A  19     -23.280  -1.283   8.795  1.00  0.00           C  \\nATOM    140  OG1 THR A  19     -23.532  -0.905   9.740  1.00  0.00           O  \\nATOM    141  CG2 THR A  19     -22.776  -2.669   8.543  1.00  0.00           C  \\nATOM    142  N   VAL A  20     -24.036   1.111   7.472  1.00  0.00           N  \\nATOM    143  CA  VAL A  20     -24.414   2.371   6.779  1.00  0.00           C  \\nATOM    144  C   VAL A  20     -26.052   2.434   6.401  1.00  0.00           C  \\nATOM    145  O   VAL A  20     -26.556   3.442   5.771  1.00  0.00           O  \\nATOM    146  CB  VAL A  20     -23.784   2.560   5.519  1.00  0.00           C  \\nATOM    147  CG1 VAL A  20     -24.540   2.119   4.637  1.00  0.00           C  \\nATOM    148  CG2 VAL A  20     -23.658   4.009   5.267  1.00  0.00           C  \\nATOM    149  N   GLU A  21     -26.682   1.363   6.527  1.00  0.00           N  \\nATOM    150  CA  GLU A  21     -28.194   1.363   5.897  1.00  0.00           C  \\nATOM    151  C   GLU A  21     -29.013   1.741   7.283  1.00  0.00           C  \\nATOM    152  O   GLU A  21     -28.635   2.623   7.913  1.00  0.00           O  \\nATOM    153  CB  GLU A  21     -28.572  -0.086   5.645  1.00  0.00           C  \\nATOM    154  CG  GLU A  21     -28.698  -0.779   4.133  1.00  0.00           C  \\nATOM    155  CD  GLU A  21     -27.690  -1.913   4.007  1.00  0.00           C  \\nATOM    156  OE1 GLU A  21     -26.682  -2.039   5.141  1.00  0.00           O  \\nATOM    157  OE2 GLU A  21     -27.753  -2.921   2.873  1.00  0.00           O  \\nATOM    158  N   ASN A  22     -30.210   1.363   7.283  1.00  0.00           N  \\nATOM    159  CA  ASN A  22     -30.966   1.867   8.417  1.00  0.00           C  \\nATOM    160  C   ASN A  22     -30.714   1.237   9.677  1.00  0.00           C  \\nATOM    161  O   ASN A  22     -31.218   0.103  10.055  1.00  0.00           O  \\nATOM    162  N   GLY A  23     -29.958   1.552  10.433  1.00  0.00           N  \\nATOM    163  CA  GLY A  23     -29.517   0.859  11.819  1.00  0.00           C  \\nATOM    164  C   GLY A  23     -28.572  -0.338  11.252  1.00  0.00           C  \\nATOM    165  O   GLY A  23     -28.194  -1.283  12.071  1.00  0.00           O  \\nATOM    166  N   GLY A  24     -28.068  -0.149   9.929  1.00  0.00           N  \\nATOM    167  CA  GLY A  24     -27.060  -1.409   9.047  1.00  0.00           C  \\nATOM    168  C   GLY A  24     -26.871  -2.480  10.118  1.00  0.00           C  \\nATOM    169  O   GLY A  24     -27.186  -3.803   9.929  1.00  0.00           O  \\nATOM    170  N   LYS A  25     -26.304  -2.165  11.441  1.00  0.00           N  \\nATOM    171  CA  LYS A  25     -26.241  -3.047  12.701  1.00  0.00           C  \\nATOM    172  C   LYS A  25     -24.792  -2.795  13.079  1.00  0.00           C  \\nATOM    173  O   LYS A  25     -23.910  -2.417  12.323  1.00  0.00           O  \\nATOM    174  CB  LYS A  25     -26.241  -4.433  11.819  1.00  0.00           C  \\nATOM    175  CG  LYS A  25     -25.296  -4.496  10.433  1.00  0.00           C  \\nATOM    176  CD  LYS A  25     -24.918  -5.819   9.803  1.00  0.00           C  \\nATOM    177  CE  LYS A  25     -23.784  -5.819   8.543  1.00  0.00           C  \\nATOM    178  NZ  LYS A  25     -23.658  -6.827   7.661  1.00  0.00           N  \\nATOM    179  N   HIS A  26     -24.414  -2.921  14.465  1.00  0.00           N  \\nATOM    180  CA  HIS A  26     -23.028  -2.417  14.654  1.00  0.00           C  \\nATOM    181  C   HIS A  26     -21.894  -3.425  14.087  1.00  0.00           C  \\nATOM    182  O   HIS A  26     -22.083  -3.803  12.827  1.00  0.00           O  \\nATOM    183  CB  HIS A  26     -23.028  -2.543  16.103  1.00  0.00           C  \\nATOM    184  CG  HIS A  26     -24.036  -1.094  16.355  1.00  0.00           C  \\nATOM    185  ND1 HIS A  26     -24.540  -0.338  15.473  1.00  0.00           N  \\nATOM    186  CD2 HIS A  26     -24.666  -0.905  17.363  1.00  0.00           C  \\nATOM    187  CE1 HIS A  26     -25.422   0.355  16.103  1.00  0.00           C  \\nATOM    188  NE2 HIS A  26     -25.422   0.103  17.237  1.00  0.00           N  \\nATOM    189  N   LYS A  27     -20.886  -3.551  14.717  1.00  0.00           N  \\nATOM    190  CA  LYS A  27     -19.752  -4.433  14.213  1.00  0.00           C  \\nATOM    191  C   LYS A  27     -18.366  -3.677  14.276  1.00  0.00           C  \\nATOM    192  O   LYS A  27     -17.484  -3.677  13.331  1.00  0.00           O  \\nATOM    193  CB  LYS A  27     -20.004  -4.559  12.575  1.00  0.00           C  \\nATOM    194  CG  LYS A  27     -20.382  -3.425  11.882  1.00  0.00           C  \\nATOM    195  CD  LYS A  27     -19.500  -3.047  11.063  1.00  0.00           C  \\nATOM    196  CE  LYS A  27     -19.752  -3.677   9.551  1.00  0.00           C  \\nATOM    197  NZ  LYS A  27     -18.618  -3.677   8.858  1.00  0.00           N  \\nATOM    198  N   VAL A  28     -18.177  -3.047  15.410  1.00  0.00           N  \\nATOM    199  CA  VAL A  28     -16.854  -2.228  15.410  1.00  0.00           C  \\nATOM    200  C   VAL A  28     -17.358  -0.905  15.473  1.00  0.00           C  \\nATOM    201  O   VAL A  28     -17.610  -0.275  16.481  1.00  0.00           O  \\nATOM    202  CB  VAL A  28     -15.846  -2.291  14.780  1.00  0.00           C  \\nATOM    203  CG1 VAL A  28     -15.594  -1.094  14.087  1.00  0.00           C  \\nATOM    204  CG2 VAL A  28     -14.586  -2.858  15.095  1.00  0.00           C  \\nATOM    205  N   GLY A  29     -17.610  -0.023  14.591  1.00  0.00           N  \\nATOM    206  CA  GLY A  29     -18.240   1.111  14.843  1.00  0.00           C  \\nATOM    207  C   GLY A  29     -19.815   1.048  14.843  1.00  0.00           C  \\nATOM    208  O   GLY A  29     -20.193   0.103  13.961  1.00  0.00           O  \\nATOM    209  N   PRO A  30     -20.634   1.615  15.977  1.00  0.00           N  \\nATOM    210  CA  PRO A  30     -22.020   1.489  15.725  1.00  0.00           C  \\nATOM    211  C   PRO A  30     -22.398   1.804  14.339  1.00  0.00           C  \\nATOM    212  O   PRO A  30     -22.020   2.371  13.457  1.00  0.00           O  \\nATOM    213  CB  PRO A  30     -22.524   2.371  16.481  1.00  0.00           C  \\nATOM    214  CG  PRO A  30     -21.642   1.993  16.859  1.00  0.00           C  \\nATOM    215  CD  PRO A  30     -20.508   2.497  16.607  1.00  0.00           C  \\nATOM    216  N   ASN A  31     -23.910   1.552  13.961  1.00  0.00           N  \\nATOM    217  CA  ASN A  31     -24.414   2.119  12.701  1.00  0.00           C  \\nATOM    218  C   ASN A  31     -24.036   3.631  12.764  1.00  0.00           C  \\nATOM    219  O   ASN A  31     -24.036   4.387  13.835  1.00  0.00           O  \\nATOM    220  CB  ASN A  31     -25.926   1.993  12.701  1.00  0.00           C  \\nATOM    221  CG  ASN A  31     -26.052   2.056  11.063  1.00  0.00           C  \\nATOM    222  OD1 ASN A  31     -27.186   2.623  10.307  1.00  0.00           O  \\nATOM    223  ND2 ASN A  31     -25.359   1.363  10.181  1.00  0.00           N  \\nATOM    224  N   LEU A  32     -23.658   4.261  11.819  1.00  0.00           N  \\nATOM    225  CA  LEU A  32     -23.280   5.899  11.819  1.00  0.00           C  \\nATOM    226  C   LEU A  32     -24.666   6.403  11.315  1.00  0.00           C  \\nATOM    227  O   LEU A  32     -24.666   6.655   9.929  1.00  0.00           O  \\nATOM    228  CB  LEU A  32     -22.146   6.466  11.315  1.00  0.00           C  \\nATOM    229  CG  LEU A  32     -20.886   6.529  12.323  1.00  0.00           C  \\nATOM    230  CD1 LEU A  32     -20.382   7.978  12.323  1.00  0.00           C  \\nATOM    231  CD2 LEU A  32     -21.327   6.277  13.457  1.00  0.00           C  \\nATOM    232  N   TRP A  33     -25.611   6.151  11.693  1.00  0.00           N  \\nATOM    233  CA  TRP A  33     -27.186   6.529  11.189  1.00  0.00           C  \\nATOM    234  C   TRP A  33     -27.312   7.915  10.433  1.00  0.00           C  \\nATOM    235  O   TRP A  33     -27.312   7.852   9.299  1.00  0.00           O  \\nATOM    236  CB  TRP A  33     -28.446   6.403  12.071  1.00  0.00           C  \\nATOM    237  CG  TRP A  33     -28.572   5.773  13.205  1.00  0.00           C  \\nATOM    238  CD1 TRP A  33     -27.564   4.891  13.583  1.00  0.00           C  \\nATOM    239  CD2 TRP A  33     -29.328   5.647  14.213  1.00  0.00           C  \\nATOM    240  NE1 TRP A  33     -27.816   4.387  14.843  1.00  0.00           N  \\nATOM    241  CE2 TRP A  33     -29.013   4.828  15.158  1.00  0.00           C  \\nATOM    242  CE3 TRP A  33     -30.462   6.277  14.213  1.00  0.00           C  \\nATOM    243  CZ2 TRP A  33     -29.706   4.639  15.851  1.00  0.00           C  \\nATOM    244  CZ3 TRP A  33     -31.092   5.962  15.284  1.00  0.00           C  \\nATOM    245  CH2 TRP A  33     -30.714   5.143  15.914  1.00  0.00           C  \\nATOM    246  N   GLY A  34     -27.564   8.797  11.315  1.00  0.00           N  \\nATOM    247  CA  GLY A  34     -27.690  10.120  10.685  1.00  0.00           C  \\nATOM    248  C   GLY A  34     -26.934  11.191  11.315  1.00  0.00           C  \\nATOM    249  O   GLY A  34     -27.312  12.199  12.134  1.00  0.00           O  \\nATOM    250  N   LEU A  35     -25.548  11.191  11.252  1.00  0.00           N  \\nATOM    251  CA  LEU A  35     -24.666  12.199  11.945  1.00  0.00           C  \\nATOM    252  C   LEU A  35     -25.548  13.459  12.386  1.00  0.00           C  \\nATOM    253  O   LEU A  35     -25.800  13.963  13.457  1.00  0.00           O  \\nATOM    254  CB  LEU A  35     -23.154  12.640  11.567  1.00  0.00           C  \\nATOM    255  CG  LEU A  35     -22.335  11.506  11.189  1.00  0.00           C  \\nATOM    256  CD1 LEU A  35     -20.886  11.695  11.189  1.00  0.00           C  \\nATOM    257  CD2 LEU A  35     -22.650  10.183  11.819  1.00  0.00           C  \\nATOM    258  N   PHE A  36     -26.052  13.963  11.441  1.00  0.00           N  \\nATOM    259  CA  PHE A  36     -26.934  15.223  11.693  1.00  0.00           C  \\nATOM    260  C   PHE A  36     -28.131  15.223  12.701  1.00  0.00           C  \\nATOM    261  O   PHE A  36     -29.328  15.223  12.449  1.00  0.00           O  \\nATOM    262  CB  PHE A  36     -27.186  15.601  10.055  1.00  0.00           C  \\nATOM    263  CG  PHE A  36     -25.926  15.601   9.425  1.00  0.00           C  \\nATOM    264  CD1 PHE A  36     -25.674  15.727   8.102  1.00  0.00           C  \\nATOM    265  CD2 PHE A  36     -24.792  15.097   9.803  1.00  0.00           C  \\nATOM    266  CE1 PHE A  36     -24.540  15.475   7.409  1.00  0.00           C  \\nATOM    267  CE2 PHE A  36     -23.658  15.097   9.551  1.00  0.00           C  \\nATOM    268  CZ  PHE A  36     -23.406  15.097   8.291  1.00  0.00           C  \\nATOM    269  N   GLY A  37     -27.879  15.538  13.961  1.00  0.00           N  \\nATOM    270  CA  GLY A  37     -28.509  15.664  15.347  1.00  0.00           C  \\nATOM    271  C   GLY A  37     -28.005  14.971  16.355  1.00  0.00           C  \\nATOM    272  O   GLY A  37     -28.572  14.215  17.300  1.00  0.00           O  \\nATOM    273  N   ARG A  38     -26.682  15.097  16.859  1.00  0.00           N  \\nATOM    274  CA  ARG A  38     -26.556  14.341  18.119  1.00  0.00           C  \\nATOM    275  C   ARG A  38     -25.170  14.845  18.371  1.00  0.00           C  \\nATOM    276  O   ARG A  38     -24.414  15.349  17.741  1.00  0.00           O  \\nATOM    277  CB  ARG A  38     -26.178  13.081  17.741  1.00  0.00           C  \\nATOM    278  CG  ARG A  38     -25.863  12.703  16.355  1.00  0.00           C  \\nATOM    279  CD  ARG A  38     -25.422  11.191  16.040  1.00  0.00           C  \\nATOM    280  NE  ARG A  38     -24.666  10.561  16.985  1.00  0.00           N  \\nATOM    281  CZ  ARG A  38     -24.792   9.238  16.859  1.00  0.00           C  \\nATOM    282  NH1 ARG A  38     -24.351   8.545  17.489  1.00  0.00           N  \\nATOM    283  NH2 ARG A  38     -25.485   8.797  15.851  1.00  0.00           N  \\nATOM    284  N   LYS A  39     -25.044  14.593  19.757  1.00  0.00           N  \\nATOM    285  CA  LYS A  39     -24.036  15.160  20.639  1.00  0.00           C  \\nATOM    286  C   LYS A  39     -23.028  14.089  20.891  1.00  0.00           C  \\nATOM    287  O   LYS A  39     -23.406  13.207  20.009  1.00  0.00           O  \\nATOM    288  CB  LYS A  39     -24.540  15.601  22.151  1.00  0.00           C  \\nATOM    289  CG  LYS A  39     -24.414  15.475  23.663  1.00  0.00           C  \\nATOM    290  CD  LYS A  39     -25.170  16.231  24.923  1.00  0.00           C  \\nATOM    291  N   THR A  40     -22.020  13.837  21.521  1.00  0.00           N  \\nATOM    292  CA  THR A  40     -21.390  12.388  21.269  1.00  0.00           C  \\nATOM    293  C   THR A  40     -21.642  11.191  22.277  1.00  0.00           C  \\nATOM    294  O   THR A  40     -21.516  11.695  23.411  1.00  0.00           O  \\nATOM    295  CB  THR A  40     -19.752  12.577  21.269  1.00  0.00           C  \\nATOM    296  OG1 THR A  40     -19.374  11.884  19.883  1.00  0.00           O  \\nATOM    297  CG2 THR A  40     -19.626  13.837  20.765  1.00  0.00           C  \\nATOM    298  N   GLY A  41     -22.020  10.309  21.773  1.00  0.00           N  \\nATOM    299  CA  GLY A  41     -22.650   9.553  22.907  1.00  0.00           C  \\nATOM    300  C   GLY A  41     -23.910   8.797  22.403  1.00  0.00           C  \\nATOM    301  O   GLY A  41     -24.162   8.545  21.395  1.00  0.00           O  \\nATOM    302  N   GLN A  42     -24.918   8.419  23.033  1.00  0.00           N  \\nATOM    303  CA  GLN A  42     -26.241   7.915  23.033  1.00  0.00           C  \\nATOM    304  C   GLN A  42     -26.304   7.537  21.647  1.00  0.00           C  \\nATOM    305  O   GLN A  42     -26.556   8.545  20.765  1.00  0.00           O  \\nATOM    306  CB  GLN A  42     -27.438   8.545  23.537  1.00  0.00           C  \\nATOM    307  CG  GLN A  42     -28.068   7.789  24.419  1.00  0.00           C  \\nATOM    308  CD  GLN A  42     -29.076   7.096  24.230  1.00  0.00           C  \\nATOM    309  OE1 GLN A  42     -29.706   5.899  24.797  1.00  0.00           O  \\nATOM    310  NE2 GLN A  42     -29.706   7.411  22.781  1.00  0.00           N  \\nATOM    311  N   ALA A  43     -26.304   6.277  21.143  1.00  0.00           N  \\nATOM    312  CA  ALA A  43     -26.682   6.025  19.505  1.00  0.00           C  \\nATOM    313  C   ALA A  43     -27.186   4.639  19.442  1.00  0.00           C  \\nATOM    314  O   ALA A  43     -27.060   4.009  18.497  1.00  0.00           O  \\nATOM    315  CB  ALA A  43     -25.548   6.025  18.497  1.00  0.00           C  \\nATOM    316  N   GLU A  44     -27.438   3.883  20.450  1.00  0.00           N  \\nATOM    317  CA  GLU A  44     -28.131   2.497  20.261  1.00  0.00           C  \\nATOM    318  C   GLU A  44     -27.060   1.615  20.891  1.00  0.00           C  \\nATOM    319  O   GLU A  44     -26.556   0.481  20.324  1.00  0.00           O  \\nATOM    320  CB  GLU A  44     -28.635   2.371  18.749  1.00  0.00           C  \\nATOM    321  CG  GLU A  44     -30.210   0.993  18.749  1.00  0.00           C  \\nATOM    322  CD  GLU A  44     -30.840   2.875  19.379  1.00  0.00           C  \\nATOM    323  OE1 GLU A  44     -32.100   2.875  18.875  1.00  0.00           O  \\nATOM    324  OE2 GLU A  44     -30.084   4.135  19.505  1.00  0.00           O  \\nATOM    325  N   GLY A  45     -26.808   1.615  22.277  1.00  0.00           N  \\nATOM    326  CA  GLY A  45     -25.674   0.859  22.781  1.00  0.00           C  \\nATOM    327  C   GLY A  45     -24.162   1.363  22.907  1.00  0.00           C  \\nATOM    328  O   GLY A  45     -23.532   1.363  21.521  1.00  0.00           O  \\nATOM    329  N   TYR A  46     -23.532   1.993  23.789  1.00  0.00           N  \\nATOM    330  CA  TYR A  46     -22.146   2.875  23.537  1.00  0.00           C  \\nATOM    331  C   TYR A  46     -21.327   1.993  24.356  1.00  0.00           C  \\nATOM    332  O   TYR A  46     -21.327   0.733  24.167  1.00  0.00           O  \\nATOM    333  CB  TYR A  46     -22.524   4.198  23.159  1.00  0.00           C  \\nATOM    334  CG  TYR A  46     -22.146   5.143  22.151  1.00  0.00           C  \\nATOM    335  CD1 TYR A  46     -22.776   5.269  21.521  1.00  0.00           C  \\nATOM    336  CD2 TYR A  46     -20.886   5.647  21.647  1.00  0.00           C  \\nATOM    337  CE1 TYR A  46     -22.524   5.899  20.387  1.00  0.00           C  \\nATOM    338  CE2 TYR A  46     -20.382   6.277  20.513  1.00  0.00           C  \\nATOM    339  CZ  TYR A  46     -21.264   6.277  20.009  1.00  0.00           C  \\nATOM    340  OH  TYR A  46     -21.138   6.655  19.001  1.00  0.00           O  \\nATOM    341  N   SER A  47     -20.634   2.560  25.049  1.00  0.00           N  \\nATOM    342  CA  SER A  47     -19.374   1.741  25.679  1.00  0.00           C  \\nATOM    343  C   SER A  47     -18.114   2.623  25.679  1.00  0.00           C  \\nATOM    344  O   SER A  47     -18.114   3.379  24.797  1.00  0.00           O  \\nATOM    345  CB  SER A  47     -19.248   0.355  25.427  1.00  0.00           C  \\nATOM    346  OG  SER A  47     -18.870   0.481  23.789  1.00  0.00           O  \\nATOM    347  N   TYR A  48     -17.484   2.749  26.939  1.00  0.00           N  \\nATOM    348  CA  TYR A  48     -16.413   3.757  26.939  1.00  0.00           C  \\nATOM    349  C   TYR A  48     -16.602   4.891  27.695  1.00  0.00           C  \\nATOM    350  O   TYR A  48     -17.610   5.269  27.821  1.00  0.00           O  \\nATOM    351  CB  TYR A  48     -16.224   3.757  25.301  1.00  0.00           C  \\nATOM    352  CG  TYR A  48     -15.216   4.639  24.545  1.00  0.00           C  \\nATOM    353  CD1 TYR A  48     -14.082   5.143  24.797  1.00  0.00           C  \\nATOM    354  CD2 TYR A  48     -15.342   4.765  23.411  1.00  0.00           C  \\nATOM    355  CE1 TYR A  48     -13.326   6.025  24.167  1.00  0.00           C  \\nATOM    356  CE2 TYR A  48     -14.712   5.521  22.529  1.00  0.00           C  \\nATOM    357  CZ  TYR A  48     -13.704   6.088  23.033  1.00  0.00           C  \\nATOM    358  OH  TYR A  48     -13.074   6.907  22.151  1.00  0.00           O  \\nATOM    359  N   THR A  49     -15.594   5.647  28.199  1.00  0.00           N  \\nATOM    360  CA  THR A  49     -15.846   6.907  29.207  1.00  0.00           C  \\nATOM    361  C   THR A  49     -15.594   8.041  28.829  1.00  0.00           C  \\nATOM    362  O   THR A  49     -16.350   8.671  27.947  1.00  0.00           O  \\nATOM    363  CB  THR A  49     -15.531   6.403  30.845  1.00  0.00           C  \\nATOM    364  OG1 THR A  49     -14.208   5.899  30.845  1.00  0.00           O  \\nATOM    365  CG2 THR A  49     -16.224   5.395  31.223  1.00  0.00           C  \\nATOM    366  N   ASP A  50     -14.334   8.482  28.703  1.00  0.00           N  \\nATOM    367  CA  ASP A  50     -14.082   9.805  28.199  1.00  0.00           C  \\nATOM    368  C   ASP A  50     -13.704   9.679  26.813  1.00  0.00           C  \\nATOM    369  O   ASP A  50     -13.074   8.545  26.183  1.00  0.00           O  \\nATOM    370  CB  ASP A  50     -13.074  10.435  28.829  1.00  0.00           C  \\nATOM    371  CG  ASP A  50     -12.948  10.183  30.089  1.00  0.00           C  \\nATOM    372  OD1 ASP A  50     -12.066   9.049  30.593  1.00  0.00           O  \\nATOM    373  OD2 ASP A  50     -13.956  10.309  30.971  1.00  0.00           O  \\nATOM    374  N   ALA A  51     -13.830  10.309  25.805  1.00  0.00           N  \\nATOM    375  CA  ALA A  51     -13.704   9.931  24.419  1.00  0.00           C  \\nATOM    376  C   ALA A  51     -15.216   9.805  23.915  1.00  0.00           C  \\nATOM    377  O   ALA A  51     -16.098   9.301  24.167  1.00  0.00           O  \\nATOM    378  CB  ALA A  51     -13.200   8.671  24.041  1.00  0.00           C  \\nATOM    379  N   ASN A  52     -15.342  10.561  22.907  1.00  0.00           N  \\nATOM    380  CA  ASN A  52     -16.539  10.561  22.277  1.00  0.00           C  \\nATOM    381  C   ASN A  52     -17.232  11.380  22.907  1.00  0.00           C  \\nATOM    382  O   ASN A  52     -17.232  12.766  22.781  1.00  0.00           O  \\nATOM    383  CB  ASN A  52     -17.610   9.553  21.899  1.00  0.00           C  \\nATOM    384  CG  ASN A  52     -17.484   8.671  20.639  1.00  0.00           C  \\nATOM    385  OD1 ASN A  52     -16.980   7.789  20.135  1.00  0.00           O  \\nATOM    386  ND2 ASN A  52     -18.240   9.427  19.631  1.00  0.00           N  \\nATOM    387  N   LYS A  53     -17.862  10.939  24.041  1.00  0.00           N  \\nATOM    388  CA  LYS A  53     -18.240  11.947  25.301  1.00  0.00           C  \\nATOM    389  C   LYS A  53     -17.106  12.703  25.805  1.00  0.00           C  \\nATOM    390  O   LYS A  53     -16.476  13.711  25.175  1.00  0.00           O  \\nATOM    391  CB  LYS A  53     -18.870  11.317  26.561  1.00  0.00           C  \\nATOM    392  CG  LYS A  53     -20.130  10.435  26.057  1.00  0.00           C  \\nATOM    393  CD  LYS A  53     -20.634   9.301  27.065  1.00  0.00           C  \\nATOM    394  CE  LYS A  53     -22.146   8.671  26.372  1.00  0.00           C  \\nATOM    395  NZ  LYS A  53     -22.524   7.663  27.191  1.00  0.00           N  \\nATOM    396  N   SER A  54     -16.602  12.325  27.065  1.00  0.00           N  \\nATOM    397  CA  SER A  54     -15.594  13.081  27.821  1.00  0.00           C  \\nATOM    398  C   SER A  54     -15.594  14.593  27.443  1.00  0.00           C  \\nATOM    399  O   SER A  54     -16.476  15.223  27.695  1.00  0.00           O  \\nATOM    400  CB  SER A  54     -14.460  12.703  27.569  1.00  0.00           C  \\nATOM    401  OG  SER A  54     -13.452  12.955  28.703  1.00  0.00           O  \\nATOM    402  N   LYS A  55     -14.838  15.223  26.813  1.00  0.00           N  \\nATOM    403  CA  LYS A  55     -14.964  16.420  26.057  1.00  0.00           C  \\nATOM    404  C   LYS A  55     -16.602  16.987  25.931  1.00  0.00           C  \\nATOM    405  O   LYS A  55     -16.854  18.121  26.183  1.00  0.00           O  \\nATOM    406  CB  LYS A  55     -14.208  16.609  24.734  1.00  0.00           C  \\nATOM    407  CG  LYS A  55     -14.712  17.743  23.600  1.00  0.00           C  \\nATOM    408  CD  LYS A  55     -13.452  17.743  22.844  1.00  0.00           C  \\nATOM    409  CE  LYS A  55     -13.578  17.995  21.143  1.00  0.00           C  \\nATOM    410  NZ  LYS A  55     -12.444  17.743  20.135  1.00  0.00           N  \\nATOM    411  N   GLY A  56     -17.232  16.042  25.679  1.00  0.00           N  \\nATOM    412  CA  GLY A  56     -18.870  16.042  25.301  1.00  0.00           C  \\nATOM    413  C   GLY A  56     -19.122  17.239  24.293  1.00  0.00           C  \\nATOM    414  O   GLY A  56     -18.492  18.247  23.852  1.00  0.00           O  \\nATOM    415  N   ILE A  57     -20.130  17.113  23.285  1.00  0.00           N  \\nATOM    416  CA  ILE A  57     -20.004  17.869  22.025  1.00  0.00           C  \\nATOM    417  C   ILE A  57     -21.264  18.121  21.143  1.00  0.00           C  \\nATOM    418  O   ILE A  57     -22.398  18.499  21.395  1.00  0.00           O  \\nATOM    419  CB  ILE A  57     -18.996  17.491  21.395  1.00  0.00           C  \\nATOM    420  CG1 ILE A  57     -19.374  16.357  20.513  1.00  0.00           C  \\nATOM    421  CG2 ILE A  57     -17.862  17.050  22.025  1.00  0.00           C  \\nATOM    422  CD1 ILE A  57     -19.122  16.609  19.127  1.00  0.00           C  \\nATOM    423  N   VAL A  58     -21.264  17.743  19.631  1.00  0.00           N  \\nATOM    424  CA  VAL A  58     -22.524  17.995  19.001  1.00  0.00           C  \\nATOM    425  C   VAL A  58     -22.020  17.239  17.615  1.00  0.00           C  \\nATOM    426  O   VAL A  58     -21.642  18.121  16.985  1.00  0.00           O  \\nATOM    427  CB  VAL A  58     -23.280  19.129  19.127  1.00  0.00           C  \\nATOM    428  CG1 VAL A  58     -22.398  20.263  18.749  1.00  0.00           C  \\nATOM    429  CG2 VAL A  58     -24.540  19.192  18.371  1.00  0.00           C  \\nATOM    430  N   TRP A  59     -22.398  16.231  16.859  1.00  0.00           N  \\nATOM    431  CA  TRP A  59     -22.020  16.105  15.221  1.00  0.00           C  \\nATOM    432  C   TRP A  59     -22.524  17.239  14.213  1.00  0.00           C  \\nATOM    433  O   TRP A  59     -23.784  17.365  14.339  1.00  0.00           O  \\nATOM    434  CB  TRP A  59     -22.146  14.845  14.717  1.00  0.00           C  \\nATOM    435  CG  TRP A  59     -21.264  13.963  15.347  1.00  0.00           C  \\nATOM    436  CD1 TRP A  59     -21.390  12.703  15.851  1.00  0.00           C  \\nATOM    437  CD2 TRP A  59     -19.878  14.152  15.536  1.00  0.00           C  \\nATOM    438  NE1 TRP A  59     -20.382  12.451  16.607  1.00  0.00           N  \\nATOM    439  CE2 TRP A  59     -19.500  13.207  15.977  1.00  0.00           C  \\nATOM    440  CE3 TRP A  59     -19.248  15.160  15.095  1.00  0.00           C  \\nATOM    441  CZ2 TRP A  59     -18.114  13.459  16.607  1.00  0.00           C  \\nATOM    442  CZ3 TRP A  59     -18.051  15.223  15.473  1.00  0.00           C  \\nATOM    443  CH2 TRP A  59     -17.484  14.467  16.229  1.00  0.00           C  \\nATOM    444  N   ASN A  60     -21.768  17.869  13.772  1.00  0.00           N  \\nATOM    445  CA  ASN A  60     -22.398  19.129  12.953  1.00  0.00           C  \\nATOM    446  C   ASN A  60     -21.642  19.255  11.441  1.00  0.00           C  \\nATOM    447  O   ASN A  60     -20.634  18.751  11.189  1.00  0.00           O  \\nATOM    448  CB  ASN A  60     -22.524  20.515  13.583  1.00  0.00           C  \\nATOM    449  CG  ASN A  60     -22.776  21.649  12.701  1.00  0.00           C  \\nATOM    450  OD1 ASN A  60     -24.036  21.712  11.945  1.00  0.00           O  \\nATOM    451  ND2 ASN A  60     -21.894  22.783  12.701  1.00  0.00           N  \\nATOM    452  N   GLU A  61     -22.020  20.137  10.811  1.00  0.00           N  \\nATOM    453  CA  GLU A  61     -21.138  20.263   9.425  1.00  0.00           C  \\nATOM    454  C   GLU A  61     -19.626  20.389  10.181  1.00  0.00           C  \\nATOM    455  O   GLU A  61     -18.744  19.507   9.803  1.00  0.00           O  \\nATOM    456  CB  GLU A  61     -21.516  21.523   8.669  1.00  0.00           C  \\nATOM    457  CG  GLU A  61     -22.020  21.523   7.409  1.00  0.00           C  \\nATOM    458  CD  GLU A  61     -23.154  20.767   7.283  1.00  0.00           C  \\nATOM    459  OE1 GLU A  61     -24.540  20.767   7.535  1.00  0.00           O  \\nATOM    460  OE2 GLU A  61     -22.902  19.381   6.653  1.00  0.00           O  \\nATOM    461  N   ASN A  62     -19.248  21.397  10.685  1.00  0.00           N  \\nATOM    462  CA  ASN A  62     -17.736  21.586  11.441  1.00  0.00           C  \\nATOM    463  C   ASN A  62     -17.106  20.137  11.945  1.00  0.00           C  \\nATOM    464  O   ASN A  62     -15.972  20.515  12.071  1.00  0.00           O  \\nATOM    465  CB  ASN A  62     -17.736  22.783  11.945  1.00  0.00           C  \\nATOM    466  CG  ASN A  62     -17.610  24.043  11.441  1.00  0.00           C  \\nATOM    467  OD1 ASN A  62     -18.744  24.547  10.559  1.00  0.00           O  \\nATOM    468  ND2 ASN A  62     -16.539  24.799  11.063  1.00  0.00           N  \\nATOM    469  N   THR A  63     -17.610  20.263  13.268  1.00  0.00           N  \\nATOM    470  CA  THR A  63     -17.043  19.255  14.087  1.00  0.00           C  \\nATOM    471  C   THR A  63     -16.602  17.869  13.331  1.00  0.00           C  \\nATOM    472  O   THR A  63     -15.720  17.239  13.331  1.00  0.00           O  \\nATOM    473  CB  THR A  63     -18.051  18.751  15.095  1.00  0.00           C  \\nATOM    474  OG1 THR A  63     -17.358  18.499  15.851  1.00  0.00           O  \\nATOM    475  CG2 THR A  63     -18.492  17.365  14.528  1.00  0.00           C  \\nATOM    476  N   LEU A  64     -17.358  17.617  12.197  1.00  0.00           N  \\nATOM    477  CA  LEU A  64     -16.854  16.357  11.441  1.00  0.00           C  \\nATOM    478  C   LEU A  64     -15.468  16.861  10.685  1.00  0.00           C  \\nATOM    479  O   LEU A  64     -14.838  16.231  10.433  1.00  0.00           O  \\nATOM    480  CB  LEU A  64     -18.114  15.979  10.055  1.00  0.00           C  \\nATOM    481  CG  LEU A  64     -18.744  14.719  10.433  1.00  0.00           C  \\nATOM    482  CD1 LEU A  64     -19.122  14.719  11.567  1.00  0.00           C  \\nATOM    483  CD2 LEU A  64     -19.878  14.593   9.803  1.00  0.00           C  \\nATOM    484  N   MET A  65     -15.468  17.995  10.181  1.00  0.00           N  \\nATOM    485  CA  MET A  65     -13.956  18.499   9.803  1.00  0.00           C  \\nATOM    486  C   MET A  65     -13.200  18.373  11.189  1.00  0.00           C  \\nATOM    487  O   MET A  65     -12.192  17.869  11.189  1.00  0.00           O  \\nATOM    488  CB  MET A  65     -13.956  19.885   9.425  1.00  0.00           C  \\nATOM    489  CG  MET A  65     -13.452  20.137   7.913  1.00  0.00           C  \\nATOM    490  SD  MET A  65     -13.704  18.625   7.283  1.00  0.00           S  \\nATOM    491  CE  MET A  65     -13.452  19.192   5.519  1.00  0.00           C  \\nATOM    492  N   GLU A  66     -13.578  18.751  12.197  1.00  0.00           N  \\nATOM    493  CA  GLU A  66     -13.074  18.751  13.457  1.00  0.00           C  \\nATOM    494  C   GLU A  66     -12.444  17.239  13.646  1.00  0.00           C  \\nATOM    495  O   GLU A  66     -11.310  17.113  13.835  1.00  0.00           O  \\nATOM    496  CB  GLU A  66     -13.830  19.381  14.717  1.00  0.00           C  \\nATOM    497  CG  GLU A  66     -13.578  20.767  14.969  1.00  0.00           C  \\nATOM    498  CD  GLU A  66     -12.318  21.271  15.851  1.00  0.00           C  \\nATOM    499  OE1 GLU A  66     -12.192  22.657  16.103  1.00  0.00           O  \\nATOM    500  OE2 GLU A  66     -11.310  20.641  16.355  1.00  0.00           O  \\nATOM    501  N   TYR A  67     -13.263  16.483  13.457  1.00  0.00           N  \\nATOM    502  CA  TYR A  67     -12.822  14.971  13.457  1.00  0.00           C  \\nATOM    503  C   TYR A  67     -12.192  14.467  12.323  1.00  0.00           C  \\nATOM    504  O   TYR A  67     -11.184  14.089  11.945  1.00  0.00           O  \\nATOM    505  CB  TYR A  67     -14.082  14.341  13.835  1.00  0.00           C  \\nATOM    506  CG  TYR A  67     -14.082  12.955  14.465  1.00  0.00           C  \\nATOM    507  CD1 TYR A  67     -14.838  11.821  14.465  1.00  0.00           C  \\nATOM    508  CD2 TYR A  67     -13.326  13.081  15.599  1.00  0.00           C  \\nATOM    509  CE1 TYR A  67     -14.649  10.876  15.221  1.00  0.00           C  \\nATOM    510  CE2 TYR A  67     -13.200  11.947  16.103  1.00  0.00           C  \\nATOM    511  CZ  TYR A  67     -13.830  10.876  15.977  1.00  0.00           C  \\nATOM    512  OH  TYR A  67     -13.704   9.805  16.607  1.00  0.00           O  \\nATOM    513  N   LEU A  68     -12.822  14.719  11.315  1.00  0.00           N  \\nATOM    514  CA  LEU A  68     -12.192  14.341  10.055  1.00  0.00           C  \\nATOM    515  C   LEU A  68     -10.806  14.719  10.307  1.00  0.00           C  \\nATOM    516  O   LEU A  68      -9.798  13.963  10.307  1.00  0.00           O  \\nATOM    517  CB  LEU A  68     -13.074  14.845   9.173  1.00  0.00           C  \\nATOM    518  CG  LEU A  68     -14.334  13.837   8.795  1.00  0.00           C  \\nATOM    519  CD1 LEU A  68     -15.216  14.467   7.787  1.00  0.00           C  \\nATOM    520  CD2 LEU A  68     -13.704  12.829   7.787  1.00  0.00           C  \\nATOM    521  N   GLU A  69     -10.428  15.853  10.685  1.00  0.00           N  \\nATOM    522  CA  GLU A  69      -8.916  16.231  10.937  1.00  0.00           C  \\nATOM    523  C   GLU A  69      -8.097  14.971  11.441  1.00  0.00           C  \\nATOM    524  O   GLU A  69      -7.278  14.593  10.685  1.00  0.00           O  \\nATOM    525  CB  GLU A  69      -9.042  17.365  11.693  1.00  0.00           C  \\nATOM    526  CG  GLU A  69      -8.034  18.373  11.819  1.00  0.00           C  \\nATOM    527  CD  GLU A  69      -6.900  18.373  12.953  1.00  0.00           C  \\nATOM    528  OE1 GLU A  69      -5.388  18.499  12.827  1.00  0.00           O  \\nATOM    529  OE2 GLU A  69      -7.278  17.869  14.087  1.00  0.00           O  \\nATOM    530  N   ASN A  70      -8.097  14.845  12.575  1.00  0.00           N  \\nATOM    531  CA  ASN A  70      -7.278  13.585  12.827  1.00  0.00           C  \\nATOM    532  C   ASN A  70      -7.404  13.459  14.339  1.00  0.00           C  \\nATOM    533  O   ASN A  70      -7.152  14.215  15.221  1.00  0.00           O  \\nATOM    534  CB  ASN A  70      -5.892  13.837  12.449  1.00  0.00           C  \\nATOM    535  CG  ASN A  70      -4.758  14.089  13.457  1.00  0.00           C  \\nATOM    536  OD1 ASN A  70      -3.624  15.097  13.457  1.00  0.00           O  \\nATOM    537  ND2 ASN A  70      -4.758  13.333  14.717  1.00  0.00           N  \\nATOM    538  N   PRO A  71      -7.908  12.199  14.780  1.00  0.00           N  \\nATOM    539  CA  PRO A  71      -8.475  11.758  15.725  1.00  0.00           C  \\nATOM    540  C   PRO A  71      -7.593  11.758  16.859  1.00  0.00           C  \\nATOM    541  O   PRO A  71      -7.782  11.317  17.993  1.00  0.00           O  \\nATOM    542  CB  PRO A  71      -9.042  10.561  15.536  1.00  0.00           C  \\nATOM    543  CG  PRO A  71      -8.916  10.309  14.213  1.00  0.00           C  \\nATOM    544  CD  PRO A  71      -8.160  11.191  13.583  1.00  0.00           C  \\nATOM    545  N   LYS A  72      -6.522  12.325  16.733  1.00  0.00           N  \\nATOM    546  CA  LYS A  72      -5.388  12.703  17.615  1.00  0.00           C  \\nATOM    547  C   LYS A  72      -5.451  14.215  17.993  1.00  0.00           C  \\nATOM    548  O   LYS A  72      -5.262  14.971  17.111  1.00  0.00           O  \\nATOM    549  CB  LYS A  72      -3.876  12.451  17.111  1.00  0.00           C  \\nATOM    550  CG  LYS A  72      -3.624  11.065  17.363  1.00  0.00           C  \\nATOM    551  CD  LYS A  72      -2.112  10.435  17.237  1.00  0.00           C  \\nATOM    552  CE  LYS A  72      -1.860   8.923  17.741  1.00  0.00           C  \\nATOM    553  NZ  LYS A  72      -0.222   8.545  17.741  1.00  0.00           N  \\nATOM    554  N   LYS A  73      -6.081  14.719  19.127  1.00  0.00           N  \\nATOM    555  CA  LYS A  73      -6.396  16.231  19.127  1.00  0.00           C  \\nATOM    556  C   LYS A  73      -7.782  16.357  18.623  1.00  0.00           C  \\nATOM    557  O   LYS A  73      -8.160  16.231  17.363  1.00  0.00           O  \\nATOM    558  CB  LYS A  73      -5.640  17.113  17.993  1.00  0.00           C  \\nATOM    559  CG  LYS A  73      -4.758  18.247  18.623  1.00  0.00           C  \\nATOM    560  CD  LYS A  73      -3.876  19.003  17.993  1.00  0.00           C  \\nATOM    561  CE  LYS A  73      -2.616  20.389  18.812  1.00  0.00           C  \\nATOM    562  NZ  LYS A  73      -1.986  21.145  18.056  1.00  0.00           N  \\nATOM    563  N   TYR A  74      -8.664  16.231  19.631  1.00  0.00           N  \\nATOM    564  CA  TYR A  74     -10.050  16.105  19.379  1.00  0.00           C  \\nATOM    565  C   TYR A  74     -10.806  14.971  19.883  1.00  0.00           C  \\nATOM    566  O   TYR A  74     -10.554  13.963  19.379  1.00  0.00           O  \\nATOM    567  CB  TYR A  74     -10.554  16.105  17.741  1.00  0.00           C  \\nATOM    568  CG  TYR A  74     -11.814  16.735  17.363  1.00  0.00           C  \\nATOM    569  CD1 TYR A  74     -12.948  16.105  17.237  1.00  0.00           C  \\nATOM    570  CD2 TYR A  74     -11.940  17.995  16.985  1.00  0.00           C  \\nATOM    571  CE1 TYR A  74     -14.145  16.609  16.859  1.00  0.00           C  \\nATOM    572  CE2 TYR A  74     -13.074  18.625  16.607  1.00  0.00           C  \\nATOM    573  CZ  TYR A  74     -14.082  17.869  16.607  1.00  0.00           C  \\nATOM    574  OH  TYR A  74     -15.216  18.373  16.355  1.00  0.00           O  \\nATOM    575  N   ILE A  75     -11.310  15.097  21.017  1.00  0.00           N  \\nATOM    576  CA  ILE A  75     -11.940  14.089  21.647  1.00  0.00           C  \\nATOM    577  C   ILE A  75     -10.806  13.081  21.773  1.00  0.00           C  \\nATOM    578  O   ILE A  75     -10.932  11.821  21.521  1.00  0.00           O  \\nATOM    579  CB  ILE A  75     -13.200  13.585  21.143  1.00  0.00           C  \\nATOM    580  CG1 ILE A  75     -13.641  14.089  19.631  1.00  0.00           C  \\nATOM    581  CG2 ILE A  75     -14.271  13.585  22.025  1.00  0.00           C  \\nATOM    582  CD1 ILE A  75     -14.712  13.459  19.253  1.00  0.00           C  \\nATOM    583  N   PRO A  76      -9.420  13.585  21.899  1.00  0.00           N  \\nATOM    584  CA  PRO A  76      -8.160  12.955  21.647  1.00  0.00           C  \\nATOM    585  C   PRO A  76      -8.160  11.695  22.529  1.00  0.00           C  \\nATOM    586  O   PRO A  76      -8.538  11.695  23.789  1.00  0.00           O  \\nATOM    587  CB  PRO A  76      -6.900  13.837  21.836  1.00  0.00           C  \\nATOM    588  CG  PRO A  76      -7.152  14.845  22.277  1.00  0.00           C  \\nATOM    589  CD  PRO A  76      -8.538  14.845  22.151  1.00  0.00           C  \\nATOM    590  N   GLY A  77      -8.097  10.435  22.088  1.00  0.00           N  \\nATOM    591  CA  GLY A  77      -8.286   9.175  22.907  1.00  0.00           C  \\nATOM    592  C   GLY A  77      -9.546   8.167  22.529  1.00  0.00           C  \\nATOM    593  O   GLY A  77      -9.798   7.159  22.781  1.00  0.00           O  \\nATOM    594  N   THR A  78     -10.050   8.545  21.143  1.00  0.00           N  \\nATOM    595  CA  THR A  78     -11.310   7.789  20.639  1.00  0.00           C  \\nATOM    596  C   THR A  78     -10.995   6.529  20.513  1.00  0.00           C  \\nATOM    597  O   THR A  78      -9.798   6.151  20.009  1.00  0.00           O  \\nATOM    598  CB  THR A  78     -11.625   8.167  19.253  1.00  0.00           C  \\nATOM    599  OG1 THR A  78     -13.074   8.167  19.127  1.00  0.00           O  \\nATOM    600  CG2 THR A  78     -11.184   9.553  18.875  1.00  0.00           C  \\nATOM    601  N   LYS A  79     -11.562   5.521  20.765  1.00  0.00           N  \\nATOM    602  CA  LYS A  79     -11.058   4.072  20.639  1.00  0.00           C  \\nATOM    603  C   LYS A  79     -11.058   3.631  19.001  1.00  0.00           C  \\nATOM    604  O   LYS A  79     -10.932   2.497  18.371  1.00  0.00           O  \\nATOM    605  CB  LYS A  79     -11.562   3.064  21.395  1.00  0.00           C  \\nATOM    606  CG  LYS A  79     -12.570   2.560  20.513  1.00  0.00           C  \\nATOM    607  CD  LYS A  79     -13.704   1.615  20.891  1.00  0.00           C  \\nATOM    608  CE  LYS A  79     -14.838   1.363  20.135  1.00  0.00           C  \\nATOM    609  NZ  LYS A  79     -15.972   0.859  20.513  1.00  0.00           N  \\nATOM    610  N   MET A  80     -11.310   4.639  18.119  1.00  0.00           N  \\nATOM    611  CA  MET A  80     -11.436   4.135  16.607  1.00  0.00           C  \\nATOM    612  C   MET A  80     -10.050   4.198  16.103  1.00  0.00           C  \\nATOM    613  O   MET A  80      -9.294   5.017  16.103  1.00  0.00           O  \\nATOM    614  CB  MET A  80     -12.444   5.017  15.851  1.00  0.00           C  \\nATOM    615  CG  MET A  80     -11.940   6.088  14.969  1.00  0.00           C  \\nATOM    616  SD  MET A  80     -13.074   6.781  13.457  1.00  0.00           S  \\nATOM    617  CE  MET A  80     -12.192   7.852  13.079  1.00  0.00           C  \\nATOM    618  N   ILE A  81      -9.672   3.127  15.725  1.00  0.00           N  \\nATOM    619  CA  ILE A  81      -8.160   3.127  15.095  1.00  0.00           C  \\nATOM    620  C   ILE A  81      -7.908   4.009  13.835  1.00  0.00           C  \\nATOM    621  O   ILE A  81      -8.034   3.505  12.323  1.00  0.00           O  \\nATOM    622  CB  ILE A  81      -7.656   1.867  15.284  1.00  0.00           C  \\nATOM    623  CG1 ILE A  81      -6.522   1.867  16.103  1.00  0.00           C  \\nATOM    624  CG2 ILE A  81      -7.026   1.237  13.835  1.00  0.00           C  \\nATOM    625  CD1 ILE A  81      -6.900   2.245  17.615  1.00  0.00           C  \\nATOM    626  N   PHE A  82      -7.530   5.143  13.961  1.00  0.00           N  \\nATOM    627  CA  PHE A  82      -7.404   5.899  12.449  1.00  0.00           C  \\nATOM    628  C   PHE A  82      -6.144   6.529  12.197  1.00  0.00           C  \\nATOM    629  O   PHE A  82      -5.766   7.411  12.953  1.00  0.00           O  \\nATOM    630  CB  PHE A  82      -8.538   6.655  12.449  1.00  0.00           C  \\nATOM    631  CG  PHE A  82      -8.916   7.159  10.937  1.00  0.00           C  \\nATOM    632  CD1 PHE A  82      -8.916   8.293  10.433  1.00  0.00           C  \\nATOM    633  CD2 PHE A  82      -9.042   6.529  10.055  1.00  0.00           C  \\nATOM    634  CE1 PHE A  82      -9.042   8.671   9.173  1.00  0.00           C  \\nATOM    635  CE2 PHE A  82      -9.042   6.781   8.669  1.00  0.00           C  \\nATOM    636  CZ  PHE A  82      -9.042   7.852   8.291  1.00  0.00           C  \\nATOM    637  N   ALA A  83      -5.640   6.529  11.189  1.00  0.00           N  \\nATOM    638  CA  ALA A  83      -4.506   7.285  10.937  1.00  0.00           C  \\nATOM    639  C   ALA A  83      -4.758   8.797  10.811  1.00  0.00           C  \\nATOM    640  O   ALA A  83      -4.884   9.679  11.441  1.00  0.00           O  \\nATOM    641  CB  ALA A  83      -3.876   6.907   9.173  1.00  0.00           C  \\nATOM    642  N   GLY A  84      -5.136   9.301   9.677  1.00  0.00           N  \\nATOM    643  CA  GLY A  84      -5.325  10.687   9.299  1.00  0.00           C  \\nATOM    644  C   GLY A  84      -6.396  11.317   8.543  1.00  0.00           C  \\nATOM    645  O   GLY A  84      -7.467  11.569   8.921  1.00  0.00           O  \\nATOM    646  N   ILE A  85      -6.207  11.569   7.283  1.00  0.00           N  \\nATOM    647  CA  ILE A  85      -7.152  12.451   6.653  1.00  0.00           C  \\nATOM    648  C   ILE A  85      -6.774  13.963   6.905  1.00  0.00           C  \\nATOM    649  O   ILE A  85      -7.656  14.719   7.220  1.00  0.00           O  \\nATOM    650  CB  ILE A  85      -8.664  12.199   7.031  1.00  0.00           C  \\nATOM    651  CG1 ILE A  85      -9.168  10.813   6.149  1.00  0.00           C  \\nATOM    652  CG2 ILE A  85      -9.420  13.333   6.401  1.00  0.00           C  \\nATOM    653  CD1 ILE A  85     -10.176  11.191   5.204  1.00  0.00           C  \\nATOM    654  N   LYS A  86      -5.640  14.467   6.527  1.00  0.00           N  \\nATOM    655  CA  LYS A  86      -5.325  15.727   7.157  1.00  0.00           C  \\nATOM    656  C   LYS A  86      -5.136  16.546   5.771  1.00  0.00           C  \\nATOM    657  O   LYS A  86      -4.254  17.365   5.267  1.00  0.00           O  \\nATOM    658  CB  LYS A  86      -3.876  15.727   7.913  1.00  0.00           C  \\nATOM    659  CG  LYS A  86      -3.120  14.467   7.535  1.00  0.00           C  \\nATOM    660  CD  LYS A  86      -1.356  14.215   7.787  1.00  0.00           C  \\nATOM    661  CE  LYS A  86      -0.852  12.955   6.653  1.00  0.00           C  \\nATOM    662  NZ  LYS A  86       0.660  12.577   7.157  1.00  0.00           N  \\nATOM    663  N   LYS A  87      -5.892  16.357   5.015  1.00  0.00           N  \\nATOM    664  CA  LYS A  87      -5.892  17.239   3.755  1.00  0.00           C  \\nATOM    665  C   LYS A  87      -7.278  17.869   3.377  1.00  0.00           C  \\nATOM    666  O   LYS A  87      -8.160  17.239   2.873  1.00  0.00           O  \\nATOM    667  CB  LYS A  87      -5.514  16.294   2.558  1.00  0.00           C  \\nATOM    668  CG  LYS A  87      -4.632  16.735   1.235  1.00  0.00           C  \\nATOM    669  CD  LYS A  87      -4.254  15.727   0.353  1.00  0.00           C  \\nATOM    670  CE  LYS A  87      -3.120  16.231  -0.403  1.00  0.00           C  \\nATOM    671  NZ  LYS A  87      -2.868  15.727  -1.915  1.00  0.00           N  \\nATOM    672  N   LYS A  88      -7.278  19.129   3.251  1.00  0.00           N  \\nATOM    673  CA  LYS A  88      -8.412  19.885   2.621  1.00  0.00           C  \\nATOM    674  C   LYS A  88      -9.168  18.877   1.613  1.00  0.00           C  \\nATOM    675  O   LYS A  88     -10.428  18.499   1.613  1.00  0.00           O  \\nATOM    676  CB  LYS A  88      -7.782  21.019   1.991  1.00  0.00           C  \\nATOM    677  CG  LYS A  88      -6.270  20.767   1.865  1.00  0.00           C  \\nATOM    678  CD  LYS A  88      -5.514  21.271   0.605  1.00  0.00           C  \\nATOM    679  CE  LYS A  88      -5.766  20.767  -0.403  1.00  0.00           C  \\nATOM    680  NZ  LYS A  88      -5.388  21.523  -1.663  1.00  0.00           N  \\nATOM    681  N   GLY A  89      -8.538  18.499   0.353  1.00  0.00           N  \\nATOM    682  CA  GLY A  89      -9.168  17.491  -0.781  1.00  0.00           C  \\nATOM    683  C   GLY A  89     -10.050  16.357  -0.025  1.00  0.00           C  \\nATOM    684  O   GLY A  89     -11.184  16.357  -0.025  1.00  0.00           O  \\nATOM    685  N   GLU A  90      -9.546  15.475   0.479  1.00  0.00           N  \\nATOM    686  CA  GLU A  90     -10.050  14.089   1.109  1.00  0.00           C  \\nATOM    687  C   GLU A  90     -10.932  14.593   2.369  1.00  0.00           C  \\nATOM    688  O   GLU A  90     -12.192  14.341   2.621  1.00  0.00           O  \\nATOM    689  CB  GLU A  90      -9.042  13.207   1.613  1.00  0.00           C  \\nATOM    690  CG  GLU A  90      -8.286  12.577   0.605  1.00  0.00           C  \\nATOM    691  CD  GLU A  90      -6.900  11.821   0.857  1.00  0.00           C  \\nATOM    692  OE1 GLU A  90      -6.018  11.443  -0.277  1.00  0.00           O  \\nATOM    693  OE2 GLU A  90      -6.270  11.569   2.243  1.00  0.00           O  \\nATOM    694  N   ARG A  91     -10.554  15.475   2.999  1.00  0.00           N  \\nATOM    695  CA  ARG A  91     -11.184  15.916   4.511  1.00  0.00           C  \\nATOM    696  C   ARG A  91     -12.570  16.672   3.881  1.00  0.00           C  \\nATOM    697  O   ARG A  91     -13.578  16.357   4.133  1.00  0.00           O  \\nATOM    698  CB  ARG A  91     -10.302  16.987   5.141  1.00  0.00           C  \\nATOM    699  CG  ARG A  91     -10.554  17.113   6.527  1.00  0.00           C  \\nATOM    700  CD  ARG A  91      -9.420  18.121   7.283  1.00  0.00           C  \\nATOM    701  NE  ARG A  91      -9.294  19.381   6.527  1.00  0.00           N  \\nATOM    702  CZ  ARG A  91      -7.908  20.137   6.464  1.00  0.00           C  \\nATOM    703  NH1 ARG A  91      -6.900  19.507   7.535  1.00  0.00           N  \\nATOM    704  NH2 ARG A  91      -7.530  21.145   5.519  1.00  0.00           N  \\nATOM    705  N   GLN A  92     -12.444  17.743   3.251  1.00  0.00           N  \\nATOM    706  CA  GLN A  92     -13.830  18.373   2.369  1.00  0.00           C  \\nATOM    707  C   GLN A  92     -14.586  17.113   1.991  1.00  0.00           C  \\nATOM    708  O   GLN A  92     -15.720  16.735   2.243  1.00  0.00           O  \\nATOM    709  CB  GLN A  92     -13.704  19.633   1.739  1.00  0.00           C  \\nATOM    710  CG  GLN A  92     -13.830  19.255   0.164  1.00  0.00           C  \\nATOM    711  CD  GLN A  92     -13.200  20.326  -0.907  1.00  0.00           C  \\nATOM    712  OE1 GLN A  92     -11.688  19.885  -0.403  1.00  0.00           O  \\nATOM    713  NE2 GLN A  92     -13.452  21.460  -1.411  1.00  0.00           N  \\nATOM    714  N   ASP A  93     -13.956  16.357   0.983  1.00  0.00           N  \\nATOM    715  CA  ASP A  93     -14.838  15.223   0.605  1.00  0.00           C  \\nATOM    716  C   ASP A  93     -15.342  14.341   1.613  1.00  0.00           C  \\nATOM    717  O   ASP A  93     -16.602  13.837   1.487  1.00  0.00           O  \\nATOM    718  CB  ASP A  93     -13.830  14.593  -0.529  1.00  0.00           C  \\nATOM    719  CG  ASP A  93     -13.704  14.971  -1.915  1.00  0.00           C  \\nATOM    720  OD1 ASP A  93     -12.444  15.727  -2.419  1.00  0.00           O  \\nATOM    721  OD2 ASP A  93     -14.838  15.223  -2.797  1.00  0.00           O  \\nATOM    722  N   LEU A  94     -14.838  13.963   2.495  1.00  0.00           N  \\nATOM    723  CA  LEU A  94     -15.531  13.207   3.503  1.00  0.00           C  \\nATOM    724  C   LEU A  94     -16.728  13.963   3.881  1.00  0.00           C  \\nATOM    725  O   LEU A  94     -17.925  13.459   3.881  1.00  0.00           O  \\nATOM    726  CB  LEU A  94     -14.586  13.018   4.763  1.00  0.00           C  \\nATOM    727  CG  LEU A  94     -14.586  11.443   5.267  1.00  0.00           C  \\nATOM    728  CD1 LEU A  94     -15.783  10.813   4.637  1.00  0.00           C  \\nATOM    729  CD2 LEU A  94     -13.578  10.561   4.889  1.00  0.00           C  \\nATOM    730  N   VAL A  95     -16.539  15.097   4.637  1.00  0.00           N  \\nATOM    731  CA  VAL A  95     -17.736  15.979   5.141  1.00  0.00           C  \\nATOM    732  C   VAL A  95     -18.744  15.853   3.881  1.00  0.00           C  \\nATOM    733  O   VAL A  95     -20.004  15.601   4.322  1.00  0.00           O  \\nATOM    734  CB  VAL A  95     -17.358  17.365   5.393  1.00  0.00           C  \\nATOM    735  CG1 VAL A  95     -18.492  18.184   5.519  1.00  0.00           C  \\nATOM    736  CG2 VAL A  95     -16.476  17.491   6.401  1.00  0.00           C  \\nATOM    737  N   ALA A  96     -18.492  15.979   2.621  1.00  0.00           N  \\nATOM    738  CA  ALA A  96     -19.752  15.601   1.739  1.00  0.00           C  \\nATOM    739  C   ALA A  96     -20.130  14.215   1.802  1.00  0.00           C  \\nATOM    740  O   ALA A  96     -21.390  13.837   1.802  1.00  0.00           O  \\nATOM    741  CB  ALA A  96     -19.500  16.105   0.353  1.00  0.00           C  \\nATOM    742  N   TYR A  97     -19.248  13.333   1.865  1.00  0.00           N  \\nATOM    743  CA  TYR A  97     -19.626  11.884   1.865  1.00  0.00           C  \\nATOM    744  C   TYR A  97     -20.886  11.632   2.873  1.00  0.00           C  \\nATOM    745  O   TYR A  97     -22.146  11.317   2.621  1.00  0.00           O  \\nATOM    746  CB  TYR A  97     -18.744  10.813   2.243  1.00  0.00           C  \\nATOM    747  CG  TYR A  97     -19.374   9.427   1.739  1.00  0.00           C  \\nATOM    748  CD1 TYR A  97     -19.626   8.419   2.621  1.00  0.00           C  \\nATOM    749  CD2 TYR A  97     -19.626   9.175   0.353  1.00  0.00           C  \\nATOM    750  CE1 TYR A  97     -20.256   7.348   2.117  1.00  0.00           C  \\nATOM    751  CE2 TYR A  97     -20.256   8.041  -0.403  1.00  0.00           C  \\nATOM    752  CZ  TYR A  97     -20.508   7.285   0.857  1.00  0.00           C  \\nATOM    753  OH  TYR A  97     -21.264   6.403   0.353  1.00  0.00           O  \\nATOM    754  N   LEU A  98     -20.760  11.947   4.007  1.00  0.00           N  \\nATOM    755  CA  LEU A  98     -21.768  11.695   5.393  1.00  0.00           C  \\nATOM    756  C   LEU A  98     -23.154  12.703   5.015  1.00  0.00           C  \\nATOM    757  O   LEU A  98     -24.288  12.199   4.889  1.00  0.00           O  \\nATOM    758  CB  LEU A  98     -21.012  11.695   6.527  1.00  0.00           C  \\nATOM    759  CG  LEU A  98     -20.004  10.939   6.401  1.00  0.00           C  \\nATOM    760  CD1 LEU A  98     -19.374  10.687   7.661  1.00  0.00           C  \\nATOM    761  CD2 LEU A  98     -20.508   9.553   5.771  1.00  0.00           C  \\nATOM    762  N   LYS A  99     -23.028  13.963   4.889  1.00  0.00           N  \\nATOM    763  CA  LYS A  99     -24.288  14.593   4.385  1.00  0.00           C  \\nATOM    764  C   LYS A  99     -25.044  13.963   2.999  1.00  0.00           C  \\nATOM    765  O   LYS A  99     -26.304  13.837   3.125  1.00  0.00           O  \\nATOM    766  CB  LYS A  99     -23.910  16.105   4.259  1.00  0.00           C  \\nATOM    767  CG  LYS A  99     -24.918  17.113   4.259  1.00  0.00           C  \\nATOM    768  CD  LYS A  99     -24.666  18.625   3.881  1.00  0.00           C  \\nATOM    769  CE  LYS A  99     -25.674  19.633   4.133  1.00  0.00           C  \\nATOM    770  NZ  LYS A  99     -25.674  21.082   3.755  1.00  0.00           N  \\nATOM    771  N   SER A 100     -24.540  13.459   1.991  1.00  0.00           N  \\nATOM    772  CA  SER A 100     -25.296  12.577   0.857  1.00  0.00           C  \\nATOM    773  C   SER A 100     -25.548  11.191   1.361  1.00  0.00           C  \\nATOM    774  O   SER A 100     -26.934  10.813   1.361  1.00  0.00           O  \\nATOM    775  CB  SER A 100     -24.540  12.577  -0.655  1.00  0.00           C  \\nATOM    776  OG  SER A 100     -25.296  11.695  -1.663  1.00  0.00           O  \\nATOM    777  N   ALA A 101     -24.666  10.435   1.865  1.00  0.00           N  \\nATOM    778  CA  ALA A 101     -25.044   8.797   2.243  1.00  0.00           C  \\nATOM    779  C   ALA A 101     -25.800   8.923   3.629  1.00  0.00           C  \\nATOM    780  O   ALA A 101     -26.808   8.293   3.755  1.00  0.00           O  \\nATOM    781  CB  ALA A 101     -23.910   8.167   2.369  1.00  0.00           C  \\nATOM    782  N   THR A 102     -25.422   9.427   4.637  1.00  0.00           N  \\nATOM    783  CA  THR A 102     -26.052   9.301   6.149  1.00  0.00           C  \\nATOM    784  C   THR A 102     -27.690   9.805   5.897  1.00  0.00           C  \\nATOM    785  O   THR A 102     -28.572   9.049   6.149  1.00  0.00           O  \\nATOM    786  CB  THR A 102     -25.548  10.246   7.220  1.00  0.00           C  \\nATOM    787  OG1 THR A 102     -24.666   9.427   7.913  1.00  0.00           O  \\nATOM    788  CG2 THR A 102     -26.430  10.876   8.039  1.00  0.00           C  \\nATOM    789  N   SER A 103     -27.942  10.939   5.645  1.00  0.00           N  \\nATOM    790  CA  SER A 103     -29.454  11.317   5.519  1.00  0.00           C  \\nATOM    791  C   SER A 103     -30.084  10.813   4.385  1.00  0.00           C  \\nATOM    792  O   SER A 103     -29.454   9.931   3.377  1.00  0.00           O  \\nATOM    793  CB  SER A 103     -29.580  12.892   5.897  1.00  0.00           C  \\nATOM    794  OG  SER A 103     -28.320  13.585   5.393  1.00  0.00           O  \\nATOM    795  OXT SER A 103     -31.344  11.317   4.133  1.00  0.00           O  \\nTER     796      SER A 103                                                      \\nHETATM 1593 FE   HEC A 104     -15.216   5.773  12.701  1.00  0.00          FE  \\nHETATM 1594  CHA HEC A 104     -16.980   6.466  15.347  1.00  0.00           C  \\nHETATM 1595  CHB HEC A 104     -16.476   8.293  11.189  1.00  0.00           C  \\nHETATM 1596  CHC HEC A 104     -13.578   5.017   9.929  1.00  0.00           C  \\nHETATM 1597  CHD HEC A 104     -13.956   3.127  14.339  1.00  0.00           C  \\nHETATM 1598  NA  HEC A 104     -16.476   7.096  13.205  1.00  0.00           N  \\nHETATM 1599  C1A HEC A 104     -17.106   7.159  14.465  1.00  0.00           C  \\nHETATM 1600  C2A HEC A 104     -17.862   8.167  14.591  1.00  0.00           C  \\nHETATM 1601  C3A HEC A 104     -17.736   8.797  13.205  1.00  0.00           C  \\nHETATM 1602  C4A HEC A 104     -17.114   8.167  12.323  1.00  0.00           C  \\nHETATM 1603  CMA HEC A 104     -18.303  10.057  12.701  1.00  0.00           C  \\nHETATM 1604  CAA HEC A 104     -18.618   8.167  15.599  1.00  0.00           C  \\nHETATM 1605  CBA HEC A 104     -20.067   8.734  15.473  1.00  0.00           C  \\nHETATM 1606  CGA HEC A 104     -20.823   8.734  16.859  1.00  0.00           C  \\nHETATM 1607  O1A HEC A 104     -21.705   8.041  17.111  1.00  0.00           O  \\nHETATM 1608  O2A HEC A 104     -20.508   9.490  17.741  1.00  0.00           O  \\nHETATM 1609  NB  HEC A 104     -15.090   6.466  11.126  1.00  0.00           N  \\nHETATM 1610  C1B HEC A 104     -15.594   7.348  10.559  1.00  0.00           C  \\nHETATM 1611  C2B HEC A 104     -15.216   7.663   9.299  1.00  0.00           C  \\nHETATM 1612  C3B HEC A 104     -14.334   6.592   8.921  1.00  0.00           C  \\nHETATM 1613  C4B HEC A 104     -14.334   6.025   9.929  1.00  0.00           C  \\nHETATM 1614  CMB HEC A 104     -15.657   8.797   8.543  1.00  0.00           C  \\nHETATM 1615  CAB HEC A 104     -13.578   6.781   7.409  1.00  0.00           C  \\nHETATM 1616  CBB HEC A 104     -12.570   7.663   7.661  1.00  0.00           C  \\nHETATM 1617  NC  HEC A 104     -13.956   4.387  12.197  1.00  0.00           N  \\nHETATM 1618  C1C HEC A 104     -13.452   4.261  10.811  1.00  0.00           C  \\nHETATM 1619  C2C HEC A 104     -12.759   3.001  10.811  1.00  0.00           C  \\nHETATM 1620  C3C HEC A 104     -12.948   2.623  11.945  1.00  0.00           C  \\nHETATM 1621  C4C HEC A 104     -13.704   3.379  12.953  1.00  0.00           C  \\nHETATM 1622  CMC HEC A 104     -11.688   2.497   9.803  1.00  0.00           C  \\nHETATM 1623  CAC HEC A 104     -12.444   1.111  12.197  1.00  0.00           C  \\nHETATM 1624  CBC HEC A 104     -11.184   0.859  12.008  1.00  0.00           C  \\nHETATM 1625  ND  HEC A 104     -15.405   5.143  14.213  1.00  0.00           N  \\nHETATM 1626  C1D HEC A 104     -14.838   4.009  14.843  1.00  0.00           C  \\nHETATM 1627  C2D HEC A 104     -15.216   3.757  15.977  1.00  0.00           C  \\nHETATM 1628  C3D HEC A 104     -16.098   4.765  16.355  1.00  0.00           C  \\nHETATM 1629  C4D HEC A 104     -16.161   5.395  15.347  1.00  0.00           C  \\nHETATM 1630  CMD HEC A 104     -14.838   2.749  16.733  1.00  0.00           C  \\nHETATM 1631  CAD HEC A 104     -16.854   4.513  17.615  1.00  0.00           C  \\nHETATM 1632  CBD HEC A 104     -16.098   4.135  18.623  1.00  0.00           C  \\nHETATM 1633  CGD HEC A 104     -15.216   5.017  19.505  1.00  0.00           C  \\nHETATM 1634  O1D HEC A 104     -14.082   5.773  18.560  1.00  0.00           O  \\nHETATM 1635  O2D HEC A 104     -15.405   5.647  20.639  1.00  0.00           O  \\nHETATM 1679  O   HOH A 105     -16.496   9.304  17.402  1.00  0.00           O  \\nCONECT  104 1615                                                                \\nCONECT  124 1623                                                                \\nCONECT  134 1593                                                                \\nCONECT  356 1635                                                                \\nCONECT  616 1593                                                                \\nCONECT  900 1658                                                                \\nCONECT  920 1666                                                                \\nCONECT  930 1636                                                                \\nCONECT 1152 1678                                                                \\nCONECT 1412 1636                                                                \\nCONECT 1593  134  616 1598 1609                                                 \\nCONECT 1593 1617 1625                                                           \\nCONECT 1594 1599 1629                                                           \\nCONECT 1595 1602 1610                                                           \\nCONECT 1596 1613 1618                                                           \\nCONECT 1597 1621 1626                                                           \\nCONECT 1598 1593 1599 1602                                                      \\nCONECT 1599 1594 1598 1600                                                      \\nCONECT 1600 1599 1601 1604                                                      \\nCONECT 1601 1600 1602 1603                                                      \\nCONECT 1602 1595 1598 1601                                                      \\nCONECT 1603 1601                                                                \\nCONECT 1604 1600 1605                                                           \\nCONECT 1605 1604 1606                                                           \\nCONECT 1606 1605 1607 1608                                                      \\nCONECT 1607 1606                                                                \\nCONECT 1608 1606                                                                \\nCONECT 1609 1593 1610 1613                                                      \\nCONECT 1610 1595 1609 1611                                                      \\nCONECT 1611 1610 1612 1614                                                      \\nCONECT 1612 1611 1613 1615                                                      \\nCONECT 1613 1596 1609 1612                                                      \\nCONECT 1614 1611                                                                \\nCONECT 1615  104 1612 1616                                                      \\nCONECT 1616 1615                                                                \\nCONECT 1617 1593 1618 1621                                                      \\nCONECT 1618 1596 1617 1619                                                      \\nCONECT 1619 1618 1620 1622                                                      \\nCONECT 1620 1619 1621 1623                                                      \\nCONECT 1621 1597 1617 1620                                                      \\nCONECT 1622 1619                                                                \\nCONECT 1623  124 1620 1624                                                      \\nCONECT 1624 1623                                                                \\nCONECT 1625 1593 1626 1629                                                      \\nCONECT 1626 1597 1625 1627                                                      \\nCONECT 1627 1626 1628 1630                                                      \\nCONECT 1628 1627 1629 1631                                                      \\nCONECT 1629 1594 1625 1628                                                      \\nCONECT 1630 1627                                                                \\nCONECT 1631 1628 1632                                                           \\nCONECT 1632 1631 1633                                                           \\nCONECT 1633 1632 1634 1635                                                      \\nCONECT 1634 1633                                                                \\nCONECT 1635  356 1633                                                           \\nCONECT 1636  930 1412 1641 1652                                                 \\nCONECT 1636 1660 1668                                                           \\nCONECT 1637 1642 1672                                                           \\nCONECT 1638 1645 1653                                                           \\nCONECT 1639 1656 1661                                                           \\nCONECT 1640 1664 1669                                                           \\nCONECT 1641 1636 1642 1645                                                      \\nCONECT 1642 1637 1641 1643                                                      \\nCONECT 1643 1642 1644 1647                                                      \\nCONECT 1644 1643 1645 1646                                                      \\nCONECT 1645 1638 1641 1644                                                      \\nCONECT 1646 1644                                                                \\nCONECT 1647 1643 1648                                                           \\nCONECT 1648 1647 1649                                                           \\nCONECT 1649 1648 1650 1651                                                      \\nCONECT 1650 1649                                                                \\nCONECT 1651 1649                                                                \\nCONECT 1652 1636 1653 1656                                                      \\nCONECT 1653 1638 1652 1654                                                      \\nCONECT 1654 1653 1655 1657                                                      \\nCONECT 1655 1654 1656 1658                                                      \\nCONECT 1656 1639 1652 1655                                                      \\nCONECT 1657 1654                                                                \\nCONECT 1658  900 1655 1659                                                      \\nCONECT 1659 1658                                                                \\nCONECT 1660 1636 1661 1664                                                      \\nCONECT 1661 1639 1660 1662                                                      \\nCONECT 1662 1661 1663 1665                                                      \\nCONECT 1663 1662 1664 1666                                                      \\nCONECT 1664 1640 1660 1663                                                      \\nCONECT 1665 1662                                                                \\nCONECT 1666  920 1663 1667                                                      \\nCONECT 1667 1666                                                                \\nCONECT 1668 1636 1669 1672                                                      \\nCONECT 1669 1640 1668 1670                                                      \\nCONECT 1670 1669 1671 1673                                                      \\nCONECT 1671 1670 1672 1674                                                      \\nCONECT 1672 1637 1668 1671                                                      \\nCONECT 1673 1670                                                                \\nCONECT 1674 1671 1675                                                           \\nCONECT 1675 1674 1676                                                           \\nCONECT 1676 1675 1677 1678                                                      \\nCONECT 1677 1676                                                                \\nCONECT 1678 1152 1676                                                           \\nMASTER      287    0    2   10    0    0    9    9 1677    2   98   16          \\nEND                                                                             \\n\",\"pdb\");\n\tviewer_16742555228826048.setBackgroundColor(\"white\");\n\tviewer_16742555228826048.setStyle({\"cartoon\": {\"color\": \"white\"}});\n\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"5\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"6\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"7\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"8\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"9\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"10\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"11\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"12\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"13\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"14\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"15\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"16\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"17\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"18\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"19\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"20\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"21\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"22\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"23\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"24\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"25\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"26\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"27\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"5\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"6\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"6\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"7\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"7\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"15\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"8\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"12\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"13\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"15\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"16\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"11\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"13\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"13\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"15\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"13\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"16\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"13\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"15\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"15\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"16\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"18\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"22\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"23\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"24\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"21\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"25\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"23\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"25\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"23\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"26\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555228826048.zoomTo();\nviewer_16742555228826048.render();\n});\n</script>",
      "text/html": [
       "<div id=\"3dmolviewer_16742555228826048\"  style=\"position: relative; width: 640px; height: 480px\">\n",
       "        <p id=\"3dmolwarning_16742555228826048\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n",
       "        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n",
       "        </div>\n",
       "<script>\n",
       "\n",
       "var loadScriptAsync = function(uri){\n",
       "  return new Promise((resolve, reject) => {\n",
       "    //this is to ignore the existence of requirejs amd\n",
       "    var savedexports, savedmodule;\n",
       "    if (typeof exports !== 'undefined') savedexports = exports;\n",
       "    else exports = {}\n",
       "    if (typeof module !== 'undefined') savedmodule = module;\n",
       "    else module = {}\n",
       "\n",
       "    var tag = document.createElement('script');\n",
       "    tag.src = uri;\n",
       "    tag.async = true;\n",
       "    tag.onload = () => {\n",
       "        exports = savedexports;\n",
       "        module = savedmodule;\n",
       "        resolve();\n",
       "    };\n",
       "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
       "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
       "});\n",
       "};\n",
       "\n",
       "if(typeof $3Dmolpromise === 'undefined') {\n",
       "$3Dmolpromise = null;\n",
       "  $3Dmolpromise = loadScriptAsync('https://3dmol.org/build/3Dmol.js');\n",
       "}\n",
       "\n",
       "var viewer_16742555228826048 = null;\n",
       "var warn = document.getElementById(\"3dmolwarning_16742555228826048\");\n",
       "if(warn) {\n",
       "    warn.parentNode.removeChild(warn);\n",
       "}\n",
       "$3Dmolpromise.then(function() {\n",
       "viewer_16742555228826048 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_16742555228826048\"),{backgroundColor:\"white\"});\n",
       "viewer_16742555228826048.zoomTo();\n",
       "\tviewer_16742555228826048.addModel(\"HEADER    ELECTRON TRANSPORT                      01-AUG-76   1CYC              \\nTITLE     THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C AT 2.3     \\nTITLE    2 ANGSTROMS RESOLUTION. II. STRUCTURE AND FUNCTION                     \\nCAVEAT     1CYC    THR A 40 HAS WRONG CHIRALITY AT ATOM CB THR A 78 HAS WRONG   \\nCAVEAT   2 1CYC    CHIRALITY AT ATOM CB THR B 40 HAS WRONG CHIRALITY AT ATOM    \\nCAVEAT   3 1CYC    CB THR B 78 HAS WRONG CHIRALITY AT ATOM CB                   \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: FERROCYTOCHROME C;                                         \\nCOMPND   3 CHAIN: A, B;                                                         \\nCOMPND   4 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 ORGANISM_SCIENTIFIC: KATSUWONUS PELAMIS;                             \\nSOURCE   3 ORGANISM_COMMON: SKIPJACK TUNA;                                      \\nSOURCE   4 ORGANISM_TAXID: 8226                                                 \\nKEYWDS    ELECTRON TRANSPORT                                                    \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    N.TANAKA,T.YAMANE,T.TSUKIHARA,T.ASHIDA,M.KAKUDO                       \\nREVDAT  17   03-MAR-21 1CYC    1       CAVEAT COMPND REMARK HET                 \\nREVDAT  17 2                   1       HETNAM HETSYN FORMUL LINK                \\nREVDAT  17 3                   1       ATOM                                     \\nREVDAT  16   29-NOV-17 1CYC    1       HELIX                                    \\nREVDAT  15   24-FEB-09 1CYC    1       VERSN                                    \\nREVDAT  14   01-APR-03 1CYC    1       JRNL                                     \\nREVDAT  13   27-JAN-84 1CYC    1       REMARK                                   \\nREVDAT  12   30-SEP-83 1CYC    1       REVDAT                                   \\nREVDAT  11   19-APR-83 1CYC    1       COMPND SOURCE REMARK                     \\nREVDAT  10   15-JAN-82 1CYC    1       REMARK                                   \\nREVDAT   9   06-JAN-82 1CYC    3       SCALE                                    \\nREVDAT   8   31-DEC-80 1CYC    1       REMARK                                   \\nREVDAT   7   07-APR-80 1CYC    3       SOURCE SEQRES ATOM                       \\nREVDAT   6   23-APR-79 1CYC    3       ATOM                                     \\nREVDAT   5   01-NOV-77 1CYC    1       AUTHOR REMARK FORMUL                     \\nREVDAT   4   09-SEP-77 1CYC    3       ATOM                                     \\nREVDAT   3   13-JUN-77 1CYC    1       HET                                      \\nREVDAT   2   25-OCT-76 1CYC    2       CONECT                                   \\nREVDAT   1   06-OCT-76 1CYC    0                                                \\nJRNL        AUTH   N.TANAKA,T.YAMANE,T.TSUKIHARA,T.ASHIDA,M.KAKUDO              \\nJRNL        TITL   THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C   \\nJRNL        TITL 2 AT 2.3 A RESOLUTION. II. STRUCTURE AND FUNCTION.             \\nJRNL        REF    J.BIOCHEM.(TOKYO)             V.  77   147 1975              \\nJRNL        REFN                   ISSN 0021-924X                               \\nJRNL        PMID   166072                                                       \\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   T.TSUKIHARA,T.YAMANE,N.TANAKA,T.ASHIDA,M.KAKUDO              \\nREMARK   1  TITL   OXIDATION OF A FERROCYTOCHROME C IN THE CRYSTALLINE          \\nREMARK   1  TITL 2 STATE-STRUCTURAL CHANGE AND ANION BINDING                    \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  73  1163 1973              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 2                                                          \\nREMARK   1  AUTH   T.ASHIDA,N.TANAKA,T.YAMANE,T.TSUKIHARA,M.KAKUDO              \\nREMARK   1  TITL   THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C   \\nREMARK   1  TITL 2 AT 2.3 ANGSTROMS RESOLUTION                                  \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  73   463 1973              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 3                                                          \\nREMARK   1  AUTH   T.ASHIDA,T.UEKI,T.TSUKIHARA,A.SUGIHARA,T.TAKANO,M.KAKUDO     \\nREMARK   1  TITL   THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C   \\nREMARK   1  TITL 2 AT 4 ANGSTROMS RESOLUTION                                    \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  70   913 1971              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 4                                                          \\nREMARK   1  AUTH   T.NAKAYAMA,K.TITANI,K.NARITA                                 \\nREMARK   1  TITL   THE AMINO ACID SEQUENCE OF CYTOCHROME C FROM BONITO          \\nREMARK   1  TITL 2 (KATSUWONUS PELAMIS, LINNAEUS)                               \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  70   311 1971              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 5                                                          \\nREMARK   1  EDIT   R.J.FELDMANN                                                 \\nREMARK   1  REF    ATLAS OF MACROMOLECULAR                120 1976              \\nREMARK   1  REF  2 STRUCTURE ON MICROFICHE                                      \\nREMARK   1  PUBL   TRACOR JITCO INC.,ROCKVILLE,MD.                              \\nREMARK   1  REFN                                                                \\nREMARK   1 REFERENCE 6                                                          \\nREMARK   1  EDIT   M.O.DAYHOFF                                                  \\nREMARK   1  REF    ATLAS OF PROTEIN SEQUENCE     V.   5    39 1976              \\nREMARK   1  REF  2 AND STRUCTURE,SUPPLEMENT 2                                   \\nREMARK   1  PUBL   NATIONAL BIOMEDICAL RESEARCH FOUNDATION, SILVER SPRING,MD.   \\nREMARK   1  REFN                   ISSN 0-912466-05-7                           \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    2.30 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : NULL                                                 \\nREMARK   3   AUTHORS     : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 2.30                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : NULL                           \\nREMARK   3   DATA CUTOFF            (SIGMA(F)) : NULL                           \\nREMARK   3   DATA CUTOFF HIGH         (ABS(F)) : NULL                           \\nREMARK   3   DATA CUTOFF LOW          (ABS(F)) : NULL                           \\nREMARK   3   COMPLETENESS (WORKING+TEST)   (%) : NULL                           \\nREMARK   3   NUMBER OF REFLECTIONS             : NULL                           \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   CROSS-VALIDATION METHOD          : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SELECTION  : NULL                            \\nREMARK   3   R VALUE            (WORKING SET) : NULL                            \\nREMARK   3   FREE R VALUE                     : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : NULL                            \\nREMARK   3   FREE R VALUE TEST SET COUNT      : NULL                            \\nREMARK   3   ESTIMATED ERROR OF FREE R VALUE  : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  FIT IN THE HIGHEST RESOLUTION BIN.                                  \\nREMARK   3   TOTAL NUMBER OF BINS USED           : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE HIGH       (A) : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE LOW        (A) : NULL                         \\nREMARK   3   BIN COMPLETENESS (WORKING+TEST) (%) : NULL                         \\nREMARK   3   REFLECTIONS IN BIN    (WORKING SET) : NULL                         \\nREMARK   3   BIN R VALUE           (WORKING SET) : NULL                         \\nREMARK   3   BIN FREE R VALUE                    : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET SIZE  (%) : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET COUNT     : NULL                         \\nREMARK   3   ESTIMATED ERROR OF BIN FREE R VALUE : NULL                         \\nREMARK   3                                                                      \\nREMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \\nREMARK   3   PROTEIN ATOMS            : 1590                                    \\nREMARK   3   NUCLEIC ACID ATOMS       : 0                                       \\nREMARK   3   HETEROGEN ATOMS          : 86                                      \\nREMARK   3   SOLVENT ATOMS            : 1                                       \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : NULL                           \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : NULL                           \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : NULL                                                 \\nREMARK   3    B22 (A**2) : NULL                                                 \\nREMARK   3    B33 (A**2) : NULL                                                 \\nREMARK   3    B12 (A**2) : NULL                                                 \\nREMARK   3    B13 (A**2) : NULL                                                 \\nREMARK   3    B23 (A**2) : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  ESTIMATED COORDINATE ERROR.                                         \\nREMARK   3   ESD FROM LUZZATI PLOT        (A) : NULL                            \\nREMARK   3   ESD FROM SIGMAA              (A) : NULL                            \\nREMARK   3   LOW RESOLUTION CUTOFF        (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  CROSS-VALIDATED ESTIMATED COORDINATE ERROR.                         \\nREMARK   3   ESD FROM C-V LUZZATI PLOT    (A) : NULL                            \\nREMARK   3   ESD FROM C-V SIGMAA          (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  RMS DEVIATIONS FROM IDEAL VALUES.                                   \\nREMARK   3   BOND LENGTHS                 (A) : NULL                            \\nREMARK   3   BOND ANGLES            (DEGREES) : NULL                            \\nREMARK   3   DIHEDRAL ANGLES        (DEGREES) : NULL                            \\nREMARK   3   IMPROPER ANGLES        (DEGREES) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL MODEL : NULL                                      \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL FACTOR RESTRAINTS.    RMS    SIGMA                \\nREMARK   3   MAIN-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   MAIN-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  NCS MODEL : NULL                                                    \\nREMARK   3                                                                      \\nREMARK   3  NCS RESTRAINTS.                         RMS   SIGMA/WEIGHT          \\nREMARK   3   GROUP  1  POSITIONAL            (A) : NULL  ; NULL                 \\nREMARK   3   GROUP  1  B-FACTOR           (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  PARAMETER FILE  1  : NULL                                           \\nREMARK   3  TOPOLOGY FILE  1   : NULL                                           \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS: NULL                                      \\nREMARK   4                                                                      \\nREMARK   4 1CYC COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY BNL.                                \\nREMARK 100 THE DEPOSITION ID IS D_1000172595.                                   \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : NULL                               \\nREMARK 200  TEMPERATURE           (KELVIN) : NULL                               \\nREMARK 200  PH                             : NULL                               \\nREMARK 200  NUMBER OF CRYSTALS USED        : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : NULL                               \\nREMARK 200  RADIATION SOURCE               : NULL                               \\nREMARK 200  BEAMLINE                       : NULL                               \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : NULL                               \\nREMARK 200  WAVELENGTH OR RANGE        (A) : NULL                               \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : NULL                               \\nREMARK 200  DETECTOR MANUFACTURER          : NULL                               \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : NULL                               \\nREMARK 200  DATA SCALING SOFTWARE          : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : NULL                               \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : NULL                               \\nREMARK 200  RESOLUTION RANGE LOW       (A) : NULL                               \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY                : NULL                               \\nREMARK 200  R MERGE                    (I) : NULL                               \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : NULL                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : NULL                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : NULL                               \\nREMARK 200  R MERGE FOR SHELL          (I) : NULL                               \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: NULL                                           \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: NULL                         \\nREMARK 200 SOFTWARE USED: NULL                                                  \\nREMARK 200 STARTING MODEL: NULL                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: NULL                                                         \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): 39.18                                     \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 2.02                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: NULL                                     \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: P 21 21 21                       \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -X+1/2,-Y,Z+1/2                                         \\nREMARK 290       3555   -X,Y+1/2,-Z+1/2                                         \\nREMARK 290       4555   X+1/2,-Y+1/2,-Z                                         \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -1.000000  0.000000  0.000000       28.84000            \\nREMARK 290   SMTRY2   2  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000       18.91500            \\nREMARK 290   SMTRY1   3 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   3  0.000000  1.000000  0.000000       42.29000            \\nREMARK 290   SMTRY3   3  0.000000  0.000000 -1.000000       18.91500            \\nREMARK 290   SMTRY1   4  1.000000  0.000000  0.000000       28.84000            \\nREMARK 290   SMTRY2   4  0.000000 -1.000000  0.000000       42.29000            \\nREMARK 290   SMTRY3   4  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1, 2                                                    \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: MONOMERIC                         \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 2                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: MONOMERIC                         \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: B                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 470                                                                      \\nREMARK 470 MISSING ATOM                                                         \\nREMARK 470 THE FOLLOWING RESIDUES HAVE MISSING ATOMS (M=MODEL NUMBER;           \\nREMARK 470 RES=RESIDUE NAME; C=CHAIN IDENTIFIER; SSEQ=SEQUENCE NUMBER;          \\nREMARK 470 I=INSERTION CODE):                                                   \\nREMARK 470   M RES CSSEQI  ATOMS                                                \\nREMARK 470     ASN A  22    CB   CG   OD1  ND2                                  \\nREMARK 470     LYS A  39    CE   NZ                                             \\nREMARK 470     ASN B  22    CB   CG   OD1  ND2                                  \\nREMARK 470     LYS B  39    CE   NZ                                             \\nREMARK 620                                                                      \\nREMARK 620 METAL COORDINATION                                                   \\nREMARK 620 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 620 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE):                             \\nREMARK 620                                                                      \\nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \\nREMARK 620                             HEC A 104  FE                            \\nREMARK 620 N RES CSSEQI ATOM                                                    \\nREMARK 620 1 HIS A  18   NE2                                                    \\nREMARK 620 2 HEC A 104   NA   87.0                                              \\nREMARK 620 3 HEC A 104   NB   87.8  90.6                                        \\nREMARK 620 4 HEC A 104   NC   92.0 178.7  90.1                                  \\nREMARK 620 5 HEC A 104   ND   91.1  86.9 177.4  92.3                            \\nREMARK 620 6 MET A  80   SD  170.9 102.1  93.0  79.0  88.6                      \\nREMARK 620 N                    1     2     3     4     5                       \\nREMARK 620                                                                      \\nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \\nREMARK 620                             HEC B 104  FE                            \\nREMARK 620 N RES CSSEQI ATOM                                                    \\nREMARK 620 1 HIS B  18   NE2                                                    \\nREMARK 620 2 HEC B 104   NA   87.0                                              \\nREMARK 620 3 HEC B 104   NB   87.8  90.6                                        \\nREMARK 620 4 HEC B 104   NC   92.0 178.7  90.1                                  \\nREMARK 620 5 HEC B 104   ND   91.1  86.9 177.4  92.3                            \\nREMARK 620 6 MET B  80   SD  170.9 102.1  93.0  79.0  88.6                      \\nREMARK 620 N                    1     2     3     4     5                       \\nREMARK 800                                                                      \\nREMARK 800 SITE                                                                 \\nREMARK 800 SITE_IDENTIFIER: AC1                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE HEC A 104                 \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: AC2                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE HEC B 104                 \\nDBREF  1CYC A    1   103  UNP    P00025   CYC_KATPE        1    103             \\nDBREF  1CYC B    1   103  UNP    P00025   CYC_KATPE        1    103             \\nSEQRES   1 A  103  GLY ASP VAL ALA LYS GLY LYS LYS THR PHE VAL GLN LYS          \\nSEQRES   2 A  103  CYS ALA GLN CYS HIS THR VAL GLU ASN GLY GLY LYS HIS          \\nSEQRES   3 A  103  LYS VAL GLY PRO ASN LEU TRP GLY LEU PHE GLY ARG LYS          \\nSEQRES   4 A  103  THR GLY GLN ALA GLU GLY TYR SER TYR THR ASP ALA ASN          \\nSEQRES   5 A  103  LYS SER LYS GLY ILE VAL TRP ASN GLU ASN THR LEU MET          \\nSEQRES   6 A  103  GLU TYR LEU GLU ASN PRO LYS LYS TYR ILE PRO GLY THR          \\nSEQRES   7 A  103  LYS MET ILE PHE ALA GLY ILE LYS LYS LYS GLY GLU ARG          \\nSEQRES   8 A  103  GLN ASP LEU VAL ALA TYR LEU LYS SER ALA THR SER              \\nSEQRES   1 B  103  GLY ASP VAL ALA LYS GLY LYS LYS THR PHE VAL GLN LYS          \\nSEQRES   2 B  103  CYS ALA GLN CYS HIS THR VAL GLU ASN GLY GLY LYS HIS          \\nSEQRES   3 B  103  LYS VAL GLY PRO ASN LEU TRP GLY LEU PHE GLY ARG LYS          \\nSEQRES   4 B  103  THR GLY GLN ALA GLU GLY TYR SER TYR THR ASP ALA ASN          \\nSEQRES   5 B  103  LYS SER LYS GLY ILE VAL TRP ASN GLU ASN THR LEU MET          \\nSEQRES   6 B  103  GLU TYR LEU GLU ASN PRO LYS LYS TYR ILE PRO GLY THR          \\nSEQRES   7 B  103  LYS MET ILE PHE ALA GLY ILE LYS LYS LYS GLY GLU ARG          \\nSEQRES   8 B  103  GLN ASP LEU VAL ALA TYR LEU LYS SER ALA THR SER              \\nHET    HEC  A 104      43                                                       \\nHET    HEC  B 104      43                                                       \\nHETNAM     HEC HEME C                                                           \\nFORMUL   3  HEC    2(C34 H34 FE N4 O4)                                          \\nFORMUL   5  HOH   *(H2 O)                                                       \\nHELIX    1  H1 GLY A    1  VAL A   11  1                                  11    \\nHELIX    2  H4 CYS A   14  CYS A   17  514 AND 17 BOUND TO HEME GROUP      4    \\nHELIX    3  H5 THR A   49  LYS A   55  5LOOSE FROM 49-53,3/10 53-55        7    \\nHELIX    4  H2 ASN A   60  GLU A   69  1                                  10    \\nHELIX    5  H3 GLU A   90  SER A  103  1                                  14    \\nHELIX    6  H6 GLY B    1  VAL B   11  1                                  11    \\nHELIX    7  H7 CYS B   14  CYS B   17  514 AND 17 BOUND TO HEME GROUP      4    \\nHELIX    8  H8 THR B   49  LYS B   55  5LOOSE FROM 49-53,3/10 53-55        7    \\nHELIX    9  H9 ASN B   60  GLU B   69  1                                  10    \\nHELIX   10 H10 GLU B   90  SER B  103  1                                  14    \\nLINK         SG  CYS A  14                 CAB HEC A 104     1555   1555  1.48  \\nLINK         SG  CYS A  17                 CAC HEC A 104     1555   1555  1.86  \\nLINK         CE2 TYR A  48                 O2D HEC A 104     1555   1555  2.02  \\nLINK         SG  CYS B  14                 CAB HEC B 104     1555   1555  1.48  \\nLINK         SG  CYS B  17                 CAC HEC B 104     1555   1555  1.86  \\nLINK         CE2 TYR B  48                 O2D HEC B 104     1555   1555  2.02  \\nLINK         NE2 HIS A  18                FE   HEC A 104     1555   1555  2.60  \\nLINK         SD  MET A  80                FE   HEC A 104     1555   1555  2.49  \\nLINK         NE2 HIS B  18                FE   HEC B 104     1555   1555  2.60  \\nLINK         SD  MET B  80                FE   HEC B 104     1555   1555  2.49  \\nSITE     1 AC1 17 LYS A  13  CYS A  14  CYS A  17  HIS A  18                    \\nSITE     2 AC1 17 LEU A  32  LEU A  35  ARG A  38  THR A  40                    \\nSITE     3 AC1 17 TYR A  46  TYR A  48  ASN A  52  TRP A  59                    \\nSITE     4 AC1 17 THR A  78  LYS A  79  MET A  80  PHE A  82                    \\nSITE     5 AC1 17 HOH A 105                                                     \\nSITE     1 AC2 16 LYS B  13  CYS B  14  CYS B  17  HIS B  18                    \\nSITE     2 AC2 16 LEU B  32  LEU B  35  ARG B  38  THR B  40                    \\nSITE     3 AC2 16 TYR B  46  TYR B  48  ASN B  52  TRP B  59                    \\nSITE     4 AC2 16 THR B  78  LYS B  79  MET B  80  PHE B  82                    \\nCRYST1   57.680   84.580   37.830  90.00  90.00  90.00 P 21 21 21    8          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.017337  0.000000  0.000000        0.00000                         \\nSCALE2      0.000000  0.011823  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.026434        0.00000                         \\nMTRIX1   1  1.000000  0.000000  0.000000       28.84000    1                    \\nMTRIX2   1  0.000000  1.000000  0.000000        0.00000    1                    \\nMTRIX3   1  0.000000  0.000000  1.000000        0.00000    1                    \\nATOM      1  N   GLY A   1     -21.138  13.774  -7.711  1.00  0.00           N  \\nATOM      2  CA  GLY A   1     -19.878  14.593  -7.963  1.00  0.00           C  \\nATOM      3  C   GLY A   1     -18.996  14.530  -6.703  1.00  0.00           C  \\nATOM      4  O   GLY A   1     -18.114  13.711  -6.640  1.00  0.00           O  \\nATOM      5  N   ASP A   2     -18.996  15.412  -5.821  1.00  0.00           N  \\nATOM      6  CA  ASP A   2     -17.610  15.097  -5.191  1.00  0.00           C  \\nATOM      7  C   ASP A   2     -17.862  14.152  -3.931  1.00  0.00           C  \\nATOM      8  O   ASP A   2     -17.106  13.333  -3.553  1.00  0.00           O  \\nATOM      9  CB  ASP A   2     -17.169  16.546  -4.435  1.00  0.00           C  \\nATOM     10  CG  ASP A   2     -17.988  17.365  -3.427  1.00  0.00           C  \\nATOM     11  OD1 ASP A   2     -19.500  17.365  -3.994  1.00  0.00           O  \\nATOM     12  OD2 ASP A   2     -17.484  18.373  -2.293  1.00  0.00           O  \\nATOM     13  N   VAL A   3     -19.059  14.089  -3.679  1.00  0.00           N  \\nATOM     14  CA  VAL A   3     -19.815  12.955  -2.860  1.00  0.00           C  \\nATOM     15  C   VAL A   3     -19.374  11.506  -3.805  1.00  0.00           C  \\nATOM     16  O   VAL A   3     -18.870  10.624  -3.427  1.00  0.00           O  \\nATOM     17  CB  VAL A   3     -22.587  13.270  -3.175  1.00  0.00           C  \\nATOM     18  CG1 VAL A   3     -21.957  11.947  -3.427  1.00  0.00           C  \\nATOM     19  CG2 VAL A   3     -21.705  13.963  -2.167  1.00  0.00           C  \\nATOM     20  N   ALA A   4     -19.437  11.380  -5.002  1.00  0.00           N  \\nATOM     21  CA  ALA A   4     -18.681  10.120  -5.569  1.00  0.00           C  \\nATOM     22  C   ALA A   4     -17.358   9.805  -4.939  1.00  0.00           C  \\nATOM     23  O   ALA A   4     -17.295   8.923  -4.183  1.00  0.00           O  \\nATOM     24  CB  ALA A   4     -18.429  10.435  -7.144  1.00  0.00           C  \\nATOM     25  N   LYS A   5     -16.476  10.687  -5.191  1.00  0.00           N  \\nATOM     26  CA  LYS A   5     -15.216  10.498  -4.372  1.00  0.00           C  \\nATOM     27  C   LYS A   5     -15.279  10.246  -2.797  1.00  0.00           C  \\nATOM     28  O   LYS A   5     -14.838   9.238  -2.293  1.00  0.00           O  \\nATOM     29  CB  LYS A   5     -14.208  11.695  -4.246  1.00  0.00           C  \\nATOM     30  CG  LYS A   5     -12.948  11.758  -3.553  1.00  0.00           C  \\nATOM     31  CD  LYS A   5     -11.814  13.081  -3.679  1.00  0.00           C  \\nATOM     32  CE  LYS A   5     -10.806  13.144  -2.545  1.00  0.00           C  \\nATOM     33  NZ  LYS A   5      -9.735  14.404  -2.293  1.00  0.00           N  \\nATOM     34  N   GLY A   6     -16.161  10.750  -1.915  1.00  0.00           N  \\nATOM     35  CA  GLY A   6     -16.350  10.435  -0.403  1.00  0.00           C  \\nATOM     36  C   GLY A   6     -16.539   8.923  -0.655  1.00  0.00           C  \\nATOM     37  O   GLY A   6     -15.909   8.041  -0.025  1.00  0.00           O  \\nATOM     38  N   LYS A   7     -17.421   8.419  -1.411  1.00  0.00           N  \\nATOM     39  CA  LYS A   7     -17.799   6.844  -1.033  1.00  0.00           C  \\nATOM     40  C   LYS A   7     -16.476   6.151  -1.474  1.00  0.00           C  \\nATOM     41  O   LYS A   7     -15.972   4.954  -0.970  1.00  0.00           O  \\nATOM     42  CB  LYS A   7     -18.870   6.403  -2.293  1.00  0.00           C  \\nATOM     43  CG  LYS A   7     -18.870   5.017  -2.734  1.00  0.00           C  \\nATOM     44  CD  LYS A   7     -20.193   4.765  -3.427  1.00  0.00           C  \\nATOM     45  CE  LYS A   7     -20.193   3.190  -4.057  1.00  0.00           C  \\nATOM     46  NZ  LYS A   7     -21.516   2.623  -4.309  1.00  0.00           N  \\nATOM     47  N   LYS A   8     -15.846   6.403  -2.545  1.00  0.00           N  \\nATOM     48  CA  LYS A   8     -14.460   5.899  -2.923  1.00  0.00           C  \\nATOM     49  C   LYS A   8     -13.452   5.899  -1.663  1.00  0.00           C  \\nATOM     50  O   LYS A   8     -13.011   5.017  -1.348  1.00  0.00           O  \\nATOM     51  CB  LYS A   8     -13.704   6.844  -3.931  1.00  0.00           C  \\nATOM     52  CG  LYS A   8     -12.318   6.529  -4.183  1.00  0.00           C  \\nATOM     53  CD  LYS A   8     -11.562   7.537  -5.065  1.00  0.00           C  \\nATOM     54  CE  LYS A   8     -10.176   7.222  -5.821  1.00  0.00           C  \\nATOM     55  NZ  LYS A   8      -9.357   8.419  -6.325  1.00  0.00           N  \\nATOM     56  N   THR A   9     -13.452   6.970  -1.159  1.00  0.00           N  \\nATOM     57  CA  THR A   9     -12.633   7.033   0.227  1.00  0.00           C  \\nATOM     58  C   THR A   9     -13.515   6.277   1.298  1.00  0.00           C  \\nATOM     59  O   THR A   9     -12.948   5.773   2.369  1.00  0.00           O  \\nATOM     60  CB  THR A   9     -12.381   8.419   0.920  1.00  0.00           C  \\nATOM     61  OG1 THR A   9     -11.058   8.923   0.542  1.00  0.00           O  \\nATOM     62  CG2 THR A   9     -12.444   8.734   2.558  1.00  0.00           C  \\nATOM     63  N   PHE A  10     -14.775   6.403   1.235  1.00  0.00           N  \\nATOM     64  CA  PHE A  10     -15.531   5.647   2.117  1.00  0.00           C  \\nATOM     65  C   PHE A  10     -15.216   4.135   1.739  1.00  0.00           C  \\nATOM     66  O   PHE A  10     -14.460   3.757   2.495  1.00  0.00           O  \\nATOM     67  CB  PHE A  10     -17.106   5.647   1.991  1.00  0.00           C  \\nATOM     68  CG  PHE A  10     -17.736   4.891   3.251  1.00  0.00           C  \\nATOM     69  CD1 PHE A  10     -18.492   3.757   2.747  1.00  0.00           C  \\nATOM     70  CD2 PHE A  10     -17.862   5.143   4.133  1.00  0.00           C  \\nATOM     71  CE1 PHE A  10     -18.996   3.001   3.818  1.00  0.00           C  \\nATOM     72  CE2 PHE A  10     -18.366   4.513   4.889  1.00  0.00           C  \\nATOM     73  CZ  PHE A  10     -18.996   3.379   5.015  1.00  0.00           C  \\nATOM     74  N   VAL A  11     -15.216   3.379   0.479  1.00  0.00           N  \\nATOM     75  CA  VAL A  11     -14.964   2.056   0.605  1.00  0.00           C  \\nATOM     76  C   VAL A  11     -13.326   1.867   1.109  1.00  0.00           C  \\nATOM     77  O   VAL A  11     -12.948   0.985   1.865  1.00  0.00           O  \\nATOM     78  CB  VAL A  11     -15.153   1.237  -0.529  1.00  0.00           C  \\nATOM     79  CG1 VAL A  11     -16.098   1.930  -1.537  1.00  0.00           C  \\nATOM     80  CG2 VAL A  11     -13.956   1.111  -1.285  1.00  0.00           C  \\nATOM     81  N   GLN A  12     -12.570   2.749   0.794  1.00  0.00           N  \\nATOM     82  CA  GLN A  12     -10.995   2.749   1.235  1.00  0.00           C  \\nATOM     83  C   GLN A  12     -10.680   2.938   2.873  1.00  0.00           C  \\nATOM     84  O   GLN A  12     -10.176   1.930   3.377  1.00  0.00           O  \\nATOM     85  CB  GLN A  12     -10.428   3.820   0.479  1.00  0.00           C  \\nATOM     86  CG  GLN A  12      -9.861   3.253  -0.718  1.00  0.00           C  \\nATOM     87  CD  GLN A  12      -9.168   4.135  -2.293  1.00  0.00           C  \\nATOM     88  OE1 GLN A  12      -7.593   3.820  -2.041  1.00  0.00           O  \\nATOM     89  NE2 GLN A  12      -9.672   5.143  -2.671  1.00  0.00           N  \\nATOM     90  N   LYS A  13     -10.995   3.757   3.629  1.00  0.00           N  \\nATOM     91  CA  LYS A  13     -10.869   3.757   5.015  1.00  0.00           C  \\nATOM     92  C   LYS A  13     -12.066   3.694   5.960  1.00  0.00           C  \\nATOM     93  O   LYS A  13     -12.003   3.127   6.779  1.00  0.00           O  \\nATOM     94  CB  LYS A  13     -10.239   5.017   5.141  1.00  0.00           C  \\nATOM     95  CG  LYS A  13      -9.672   5.521   3.881  1.00  0.00           C  \\nATOM     96  CD  LYS A  13      -8.286   6.277   4.007  1.00  0.00           C  \\nATOM     97  CE  LYS A  13      -8.160   6.907   2.621  1.00  0.00           C  \\nATOM     98  NZ  LYS A  13      -6.837   7.789   2.243  1.00  0.00           N  \\nATOM     99  N   CYS A  14     -13.137   4.072   5.519  1.00  0.00           N  \\nATOM    100  CA  CYS A  14     -14.460   4.072   6.464  1.00  0.00           C  \\nATOM    101  C   CYS A  14     -15.216   2.812   5.897  1.00  0.00           C  \\nATOM    102  O   CYS A  14     -15.342   1.804   6.716  1.00  0.00           O  \\nATOM    103  CB  CYS A  14     -15.468   5.332   5.897  1.00  0.00           C  \\nATOM    104  SG  CYS A  14     -14.712   6.844   6.464  1.00  0.00           S  \\nATOM    105  N   ALA A  15     -15.720   2.623   4.637  1.00  0.00           N  \\nATOM    106  CA  ALA A  15     -16.161   1.237   4.322  1.00  0.00           C  \\nATOM    107  C   ALA A  15     -16.035  -0.086   5.015  1.00  0.00           C  \\nATOM    108  O   ALA A  15     -17.043  -0.590   5.393  1.00  0.00           O  \\nATOM    109  CB  ALA A  15     -15.405   0.985   3.062  1.00  0.00           C  \\nATOM    110  N   GLN A  16     -14.964  -0.653   4.889  1.00  0.00           N  \\nATOM    111  CA  GLN A  16     -14.901  -2.039   5.897  1.00  0.00           C  \\nATOM    112  C   GLN A  16     -15.090  -1.787   7.283  1.00  0.00           C  \\nATOM    113  O   GLN A  16     -14.838  -2.669   8.291  1.00  0.00           O  \\nATOM    114  CB  GLN A  16     -13.578  -2.039   5.519  1.00  0.00           C  \\nATOM    115  CG  GLN A  16     -12.318  -1.724   6.275  1.00  0.00           C  \\nATOM    116  CD  GLN A  16     -11.625  -0.842   5.015  1.00  0.00           C  \\nATOM    117  OE1 GLN A  16     -10.365  -0.149   5.330  1.00  0.00           O  \\nATOM    118  NE2 GLN A  16     -12.444  -0.464   4.385  1.00  0.00           N  \\nATOM    119  N   CYS A  17     -15.216  -0.653   7.787  1.00  0.00           N  \\nATOM    120  CA  CYS A  17     -15.405  -0.905   9.299  1.00  0.00           C  \\nATOM    121  C   CYS A  17     -16.917  -0.401   9.173  1.00  0.00           C  \\nATOM    122  O   CYS A  17     -17.736  -1.283   9.173  1.00  0.00           O  \\nATOM    123  CB  CYS A  17     -14.460   0.355  10.055  1.00  0.00           C  \\nATOM    124  SG  CYS A  17     -13.326  -0.338  11.441  1.00  0.00           S  \\nATOM    125  N   HIS A  18     -17.232   0.733   9.551  1.00  0.00           N  \\nATOM    126  CA  HIS A  18     -18.870   0.985   9.803  1.00  0.00           C  \\nATOM    127  C   HIS A  18     -19.878   0.229   8.795  1.00  0.00           C  \\nATOM    128  O   HIS A  18     -19.626  -0.023   7.472  1.00  0.00           O  \\nATOM    129  CB  HIS A  18     -19.185   2.245   9.173  1.00  0.00           C  \\nATOM    130  CG  HIS A  18     -18.681   3.127  10.181  1.00  0.00           C  \\nATOM    131  ND1 HIS A  18     -18.618   3.253  11.567  1.00  0.00           N  \\nATOM    132  CD2 HIS A  18     -17.736   4.072  10.559  1.00  0.00           C  \\nATOM    133  CE1 HIS A  18     -18.051   3.694  12.827  1.00  0.00           C  \\nATOM    134  NE2 HIS A  18     -17.232   4.387  11.819  1.00  0.00           N  \\nATOM    135  N   THR A  19     -21.012   0.103   9.299  1.00  0.00           N  \\nATOM    136  CA  THR A  19     -22.272  -0.401   8.417  1.00  0.00           C  \\nATOM    137  C   THR A  19     -22.902   0.922   7.661  1.00  0.00           C  \\nATOM    138  O   THR A  19     -22.272   1.867   7.157  1.00  0.00           O  \\nATOM    139  CB  THR A  19     -23.280  -1.283   8.795  1.00  0.00           C  \\nATOM    140  OG1 THR A  19     -23.532  -0.905   9.740  1.00  0.00           O  \\nATOM    141  CG2 THR A  19     -22.776  -2.669   8.543  1.00  0.00           C  \\nATOM    142  N   VAL A  20     -24.036   1.111   7.472  1.00  0.00           N  \\nATOM    143  CA  VAL A  20     -24.414   2.371   6.779  1.00  0.00           C  \\nATOM    144  C   VAL A  20     -26.052   2.434   6.401  1.00  0.00           C  \\nATOM    145  O   VAL A  20     -26.556   3.442   5.771  1.00  0.00           O  \\nATOM    146  CB  VAL A  20     -23.784   2.560   5.519  1.00  0.00           C  \\nATOM    147  CG1 VAL A  20     -24.540   2.119   4.637  1.00  0.00           C  \\nATOM    148  CG2 VAL A  20     -23.658   4.009   5.267  1.00  0.00           C  \\nATOM    149  N   GLU A  21     -26.682   1.363   6.527  1.00  0.00           N  \\nATOM    150  CA  GLU A  21     -28.194   1.363   5.897  1.00  0.00           C  \\nATOM    151  C   GLU A  21     -29.013   1.741   7.283  1.00  0.00           C  \\nATOM    152  O   GLU A  21     -28.635   2.623   7.913  1.00  0.00           O  \\nATOM    153  CB  GLU A  21     -28.572  -0.086   5.645  1.00  0.00           C  \\nATOM    154  CG  GLU A  21     -28.698  -0.779   4.133  1.00  0.00           C  \\nATOM    155  CD  GLU A  21     -27.690  -1.913   4.007  1.00  0.00           C  \\nATOM    156  OE1 GLU A  21     -26.682  -2.039   5.141  1.00  0.00           O  \\nATOM    157  OE2 GLU A  21     -27.753  -2.921   2.873  1.00  0.00           O  \\nATOM    158  N   ASN A  22     -30.210   1.363   7.283  1.00  0.00           N  \\nATOM    159  CA  ASN A  22     -30.966   1.867   8.417  1.00  0.00           C  \\nATOM    160  C   ASN A  22     -30.714   1.237   9.677  1.00  0.00           C  \\nATOM    161  O   ASN A  22     -31.218   0.103  10.055  1.00  0.00           O  \\nATOM    162  N   GLY A  23     -29.958   1.552  10.433  1.00  0.00           N  \\nATOM    163  CA  GLY A  23     -29.517   0.859  11.819  1.00  0.00           C  \\nATOM    164  C   GLY A  23     -28.572  -0.338  11.252  1.00  0.00           C  \\nATOM    165  O   GLY A  23     -28.194  -1.283  12.071  1.00  0.00           O  \\nATOM    166  N   GLY A  24     -28.068  -0.149   9.929  1.00  0.00           N  \\nATOM    167  CA  GLY A  24     -27.060  -1.409   9.047  1.00  0.00           C  \\nATOM    168  C   GLY A  24     -26.871  -2.480  10.118  1.00  0.00           C  \\nATOM    169  O   GLY A  24     -27.186  -3.803   9.929  1.00  0.00           O  \\nATOM    170  N   LYS A  25     -26.304  -2.165  11.441  1.00  0.00           N  \\nATOM    171  CA  LYS A  25     -26.241  -3.047  12.701  1.00  0.00           C  \\nATOM    172  C   LYS A  25     -24.792  -2.795  13.079  1.00  0.00           C  \\nATOM    173  O   LYS A  25     -23.910  -2.417  12.323  1.00  0.00           O  \\nATOM    174  CB  LYS A  25     -26.241  -4.433  11.819  1.00  0.00           C  \\nATOM    175  CG  LYS A  25     -25.296  -4.496  10.433  1.00  0.00           C  \\nATOM    176  CD  LYS A  25     -24.918  -5.819   9.803  1.00  0.00           C  \\nATOM    177  CE  LYS A  25     -23.784  -5.819   8.543  1.00  0.00           C  \\nATOM    178  NZ  LYS A  25     -23.658  -6.827   7.661  1.00  0.00           N  \\nATOM    179  N   HIS A  26     -24.414  -2.921  14.465  1.00  0.00           N  \\nATOM    180  CA  HIS A  26     -23.028  -2.417  14.654  1.00  0.00           C  \\nATOM    181  C   HIS A  26     -21.894  -3.425  14.087  1.00  0.00           C  \\nATOM    182  O   HIS A  26     -22.083  -3.803  12.827  1.00  0.00           O  \\nATOM    183  CB  HIS A  26     -23.028  -2.543  16.103  1.00  0.00           C  \\nATOM    184  CG  HIS A  26     -24.036  -1.094  16.355  1.00  0.00           C  \\nATOM    185  ND1 HIS A  26     -24.540  -0.338  15.473  1.00  0.00           N  \\nATOM    186  CD2 HIS A  26     -24.666  -0.905  17.363  1.00  0.00           C  \\nATOM    187  CE1 HIS A  26     -25.422   0.355  16.103  1.00  0.00           C  \\nATOM    188  NE2 HIS A  26     -25.422   0.103  17.237  1.00  0.00           N  \\nATOM    189  N   LYS A  27     -20.886  -3.551  14.717  1.00  0.00           N  \\nATOM    190  CA  LYS A  27     -19.752  -4.433  14.213  1.00  0.00           C  \\nATOM    191  C   LYS A  27     -18.366  -3.677  14.276  1.00  0.00           C  \\nATOM    192  O   LYS A  27     -17.484  -3.677  13.331  1.00  0.00           O  \\nATOM    193  CB  LYS A  27     -20.004  -4.559  12.575  1.00  0.00           C  \\nATOM    194  CG  LYS A  27     -20.382  -3.425  11.882  1.00  0.00           C  \\nATOM    195  CD  LYS A  27     -19.500  -3.047  11.063  1.00  0.00           C  \\nATOM    196  CE  LYS A  27     -19.752  -3.677   9.551  1.00  0.00           C  \\nATOM    197  NZ  LYS A  27     -18.618  -3.677   8.858  1.00  0.00           N  \\nATOM    198  N   VAL A  28     -18.177  -3.047  15.410  1.00  0.00           N  \\nATOM    199  CA  VAL A  28     -16.854  -2.228  15.410  1.00  0.00           C  \\nATOM    200  C   VAL A  28     -17.358  -0.905  15.473  1.00  0.00           C  \\nATOM    201  O   VAL A  28     -17.610  -0.275  16.481  1.00  0.00           O  \\nATOM    202  CB  VAL A  28     -15.846  -2.291  14.780  1.00  0.00           C  \\nATOM    203  CG1 VAL A  28     -15.594  -1.094  14.087  1.00  0.00           C  \\nATOM    204  CG2 VAL A  28     -14.586  -2.858  15.095  1.00  0.00           C  \\nATOM    205  N   GLY A  29     -17.610  -0.023  14.591  1.00  0.00           N  \\nATOM    206  CA  GLY A  29     -18.240   1.111  14.843  1.00  0.00           C  \\nATOM    207  C   GLY A  29     -19.815   1.048  14.843  1.00  0.00           C  \\nATOM    208  O   GLY A  29     -20.193   0.103  13.961  1.00  0.00           O  \\nATOM    209  N   PRO A  30     -20.634   1.615  15.977  1.00  0.00           N  \\nATOM    210  CA  PRO A  30     -22.020   1.489  15.725  1.00  0.00           C  \\nATOM    211  C   PRO A  30     -22.398   1.804  14.339  1.00  0.00           C  \\nATOM    212  O   PRO A  30     -22.020   2.371  13.457  1.00  0.00           O  \\nATOM    213  CB  PRO A  30     -22.524   2.371  16.481  1.00  0.00           C  \\nATOM    214  CG  PRO A  30     -21.642   1.993  16.859  1.00  0.00           C  \\nATOM    215  CD  PRO A  30     -20.508   2.497  16.607  1.00  0.00           C  \\nATOM    216  N   ASN A  31     -23.910   1.552  13.961  1.00  0.00           N  \\nATOM    217  CA  ASN A  31     -24.414   2.119  12.701  1.00  0.00           C  \\nATOM    218  C   ASN A  31     -24.036   3.631  12.764  1.00  0.00           C  \\nATOM    219  O   ASN A  31     -24.036   4.387  13.835  1.00  0.00           O  \\nATOM    220  CB  ASN A  31     -25.926   1.993  12.701  1.00  0.00           C  \\nATOM    221  CG  ASN A  31     -26.052   2.056  11.063  1.00  0.00           C  \\nATOM    222  OD1 ASN A  31     -27.186   2.623  10.307  1.00  0.00           O  \\nATOM    223  ND2 ASN A  31     -25.359   1.363  10.181  1.00  0.00           N  \\nATOM    224  N   LEU A  32     -23.658   4.261  11.819  1.00  0.00           N  \\nATOM    225  CA  LEU A  32     -23.280   5.899  11.819  1.00  0.00           C  \\nATOM    226  C   LEU A  32     -24.666   6.403  11.315  1.00  0.00           C  \\nATOM    227  O   LEU A  32     -24.666   6.655   9.929  1.00  0.00           O  \\nATOM    228  CB  LEU A  32     -22.146   6.466  11.315  1.00  0.00           C  \\nATOM    229  CG  LEU A  32     -20.886   6.529  12.323  1.00  0.00           C  \\nATOM    230  CD1 LEU A  32     -20.382   7.978  12.323  1.00  0.00           C  \\nATOM    231  CD2 LEU A  32     -21.327   6.277  13.457  1.00  0.00           C  \\nATOM    232  N   TRP A  33     -25.611   6.151  11.693  1.00  0.00           N  \\nATOM    233  CA  TRP A  33     -27.186   6.529  11.189  1.00  0.00           C  \\nATOM    234  C   TRP A  33     -27.312   7.915  10.433  1.00  0.00           C  \\nATOM    235  O   TRP A  33     -27.312   7.852   9.299  1.00  0.00           O  \\nATOM    236  CB  TRP A  33     -28.446   6.403  12.071  1.00  0.00           C  \\nATOM    237  CG  TRP A  33     -28.572   5.773  13.205  1.00  0.00           C  \\nATOM    238  CD1 TRP A  33     -27.564   4.891  13.583  1.00  0.00           C  \\nATOM    239  CD2 TRP A  33     -29.328   5.647  14.213  1.00  0.00           C  \\nATOM    240  NE1 TRP A  33     -27.816   4.387  14.843  1.00  0.00           N  \\nATOM    241  CE2 TRP A  33     -29.013   4.828  15.158  1.00  0.00           C  \\nATOM    242  CE3 TRP A  33     -30.462   6.277  14.213  1.00  0.00           C  \\nATOM    243  CZ2 TRP A  33     -29.706   4.639  15.851  1.00  0.00           C  \\nATOM    244  CZ3 TRP A  33     -31.092   5.962  15.284  1.00  0.00           C  \\nATOM    245  CH2 TRP A  33     -30.714   5.143  15.914  1.00  0.00           C  \\nATOM    246  N   GLY A  34     -27.564   8.797  11.315  1.00  0.00           N  \\nATOM    247  CA  GLY A  34     -27.690  10.120  10.685  1.00  0.00           C  \\nATOM    248  C   GLY A  34     -26.934  11.191  11.315  1.00  0.00           C  \\nATOM    249  O   GLY A  34     -27.312  12.199  12.134  1.00  0.00           O  \\nATOM    250  N   LEU A  35     -25.548  11.191  11.252  1.00  0.00           N  \\nATOM    251  CA  LEU A  35     -24.666  12.199  11.945  1.00  0.00           C  \\nATOM    252  C   LEU A  35     -25.548  13.459  12.386  1.00  0.00           C  \\nATOM    253  O   LEU A  35     -25.800  13.963  13.457  1.00  0.00           O  \\nATOM    254  CB  LEU A  35     -23.154  12.640  11.567  1.00  0.00           C  \\nATOM    255  CG  LEU A  35     -22.335  11.506  11.189  1.00  0.00           C  \\nATOM    256  CD1 LEU A  35     -20.886  11.695  11.189  1.00  0.00           C  \\nATOM    257  CD2 LEU A  35     -22.650  10.183  11.819  1.00  0.00           C  \\nATOM    258  N   PHE A  36     -26.052  13.963  11.441  1.00  0.00           N  \\nATOM    259  CA  PHE A  36     -26.934  15.223  11.693  1.00  0.00           C  \\nATOM    260  C   PHE A  36     -28.131  15.223  12.701  1.00  0.00           C  \\nATOM    261  O   PHE A  36     -29.328  15.223  12.449  1.00  0.00           O  \\nATOM    262  CB  PHE A  36     -27.186  15.601  10.055  1.00  0.00           C  \\nATOM    263  CG  PHE A  36     -25.926  15.601   9.425  1.00  0.00           C  \\nATOM    264  CD1 PHE A  36     -25.674  15.727   8.102  1.00  0.00           C  \\nATOM    265  CD2 PHE A  36     -24.792  15.097   9.803  1.00  0.00           C  \\nATOM    266  CE1 PHE A  36     -24.540  15.475   7.409  1.00  0.00           C  \\nATOM    267  CE2 PHE A  36     -23.658  15.097   9.551  1.00  0.00           C  \\nATOM    268  CZ  PHE A  36     -23.406  15.097   8.291  1.00  0.00           C  \\nATOM    269  N   GLY A  37     -27.879  15.538  13.961  1.00  0.00           N  \\nATOM    270  CA  GLY A  37     -28.509  15.664  15.347  1.00  0.00           C  \\nATOM    271  C   GLY A  37     -28.005  14.971  16.355  1.00  0.00           C  \\nATOM    272  O   GLY A  37     -28.572  14.215  17.300  1.00  0.00           O  \\nATOM    273  N   ARG A  38     -26.682  15.097  16.859  1.00  0.00           N  \\nATOM    274  CA  ARG A  38     -26.556  14.341  18.119  1.00  0.00           C  \\nATOM    275  C   ARG A  38     -25.170  14.845  18.371  1.00  0.00           C  \\nATOM    276  O   ARG A  38     -24.414  15.349  17.741  1.00  0.00           O  \\nATOM    277  CB  ARG A  38     -26.178  13.081  17.741  1.00  0.00           C  \\nATOM    278  CG  ARG A  38     -25.863  12.703  16.355  1.00  0.00           C  \\nATOM    279  CD  ARG A  38     -25.422  11.191  16.040  1.00  0.00           C  \\nATOM    280  NE  ARG A  38     -24.666  10.561  16.985  1.00  0.00           N  \\nATOM    281  CZ  ARG A  38     -24.792   9.238  16.859  1.00  0.00           C  \\nATOM    282  NH1 ARG A  38     -24.351   8.545  17.489  1.00  0.00           N  \\nATOM    283  NH2 ARG A  38     -25.485   8.797  15.851  1.00  0.00           N  \\nATOM    284  N   LYS A  39     -25.044  14.593  19.757  1.00  0.00           N  \\nATOM    285  CA  LYS A  39     -24.036  15.160  20.639  1.00  0.00           C  \\nATOM    286  C   LYS A  39     -23.028  14.089  20.891  1.00  0.00           C  \\nATOM    287  O   LYS A  39     -23.406  13.207  20.009  1.00  0.00           O  \\nATOM    288  CB  LYS A  39     -24.540  15.601  22.151  1.00  0.00           C  \\nATOM    289  CG  LYS A  39     -24.414  15.475  23.663  1.00  0.00           C  \\nATOM    290  CD  LYS A  39     -25.170  16.231  24.923  1.00  0.00           C  \\nATOM    291  N   THR A  40     -22.020  13.837  21.521  1.00  0.00           N  \\nATOM    292  CA  THR A  40     -21.390  12.388  21.269  1.00  0.00           C  \\nATOM    293  C   THR A  40     -21.642  11.191  22.277  1.00  0.00           C  \\nATOM    294  O   THR A  40     -21.516  11.695  23.411  1.00  0.00           O  \\nATOM    295  CB  THR A  40     -19.752  12.577  21.269  1.00  0.00           C  \\nATOM    296  OG1 THR A  40     -19.374  11.884  19.883  1.00  0.00           O  \\nATOM    297  CG2 THR A  40     -19.626  13.837  20.765  1.00  0.00           C  \\nATOM    298  N   GLY A  41     -22.020  10.309  21.773  1.00  0.00           N  \\nATOM    299  CA  GLY A  41     -22.650   9.553  22.907  1.00  0.00           C  \\nATOM    300  C   GLY A  41     -23.910   8.797  22.403  1.00  0.00           C  \\nATOM    301  O   GLY A  41     -24.162   8.545  21.395  1.00  0.00           O  \\nATOM    302  N   GLN A  42     -24.918   8.419  23.033  1.00  0.00           N  \\nATOM    303  CA  GLN A  42     -26.241   7.915  23.033  1.00  0.00           C  \\nATOM    304  C   GLN A  42     -26.304   7.537  21.647  1.00  0.00           C  \\nATOM    305  O   GLN A  42     -26.556   8.545  20.765  1.00  0.00           O  \\nATOM    306  CB  GLN A  42     -27.438   8.545  23.537  1.00  0.00           C  \\nATOM    307  CG  GLN A  42     -28.068   7.789  24.419  1.00  0.00           C  \\nATOM    308  CD  GLN A  42     -29.076   7.096  24.230  1.00  0.00           C  \\nATOM    309  OE1 GLN A  42     -29.706   5.899  24.797  1.00  0.00           O  \\nATOM    310  NE2 GLN A  42     -29.706   7.411  22.781  1.00  0.00           N  \\nATOM    311  N   ALA A  43     -26.304   6.277  21.143  1.00  0.00           N  \\nATOM    312  CA  ALA A  43     -26.682   6.025  19.505  1.00  0.00           C  \\nATOM    313  C   ALA A  43     -27.186   4.639  19.442  1.00  0.00           C  \\nATOM    314  O   ALA A  43     -27.060   4.009  18.497  1.00  0.00           O  \\nATOM    315  CB  ALA A  43     -25.548   6.025  18.497  1.00  0.00           C  \\nATOM    316  N   GLU A  44     -27.438   3.883  20.450  1.00  0.00           N  \\nATOM    317  CA  GLU A  44     -28.131   2.497  20.261  1.00  0.00           C  \\nATOM    318  C   GLU A  44     -27.060   1.615  20.891  1.00  0.00           C  \\nATOM    319  O   GLU A  44     -26.556   0.481  20.324  1.00  0.00           O  \\nATOM    320  CB  GLU A  44     -28.635   2.371  18.749  1.00  0.00           C  \\nATOM    321  CG  GLU A  44     -30.210   0.993  18.749  1.00  0.00           C  \\nATOM    322  CD  GLU A  44     -30.840   2.875  19.379  1.00  0.00           C  \\nATOM    323  OE1 GLU A  44     -32.100   2.875  18.875  1.00  0.00           O  \\nATOM    324  OE2 GLU A  44     -30.084   4.135  19.505  1.00  0.00           O  \\nATOM    325  N   GLY A  45     -26.808   1.615  22.277  1.00  0.00           N  \\nATOM    326  CA  GLY A  45     -25.674   0.859  22.781  1.00  0.00           C  \\nATOM    327  C   GLY A  45     -24.162   1.363  22.907  1.00  0.00           C  \\nATOM    328  O   GLY A  45     -23.532   1.363  21.521  1.00  0.00           O  \\nATOM    329  N   TYR A  46     -23.532   1.993  23.789  1.00  0.00           N  \\nATOM    330  CA  TYR A  46     -22.146   2.875  23.537  1.00  0.00           C  \\nATOM    331  C   TYR A  46     -21.327   1.993  24.356  1.00  0.00           C  \\nATOM    332  O   TYR A  46     -21.327   0.733  24.167  1.00  0.00           O  \\nATOM    333  CB  TYR A  46     -22.524   4.198  23.159  1.00  0.00           C  \\nATOM    334  CG  TYR A  46     -22.146   5.143  22.151  1.00  0.00           C  \\nATOM    335  CD1 TYR A  46     -22.776   5.269  21.521  1.00  0.00           C  \\nATOM    336  CD2 TYR A  46     -20.886   5.647  21.647  1.00  0.00           C  \\nATOM    337  CE1 TYR A  46     -22.524   5.899  20.387  1.00  0.00           C  \\nATOM    338  CE2 TYR A  46     -20.382   6.277  20.513  1.00  0.00           C  \\nATOM    339  CZ  TYR A  46     -21.264   6.277  20.009  1.00  0.00           C  \\nATOM    340  OH  TYR A  46     -21.138   6.655  19.001  1.00  0.00           O  \\nATOM    341  N   SER A  47     -20.634   2.560  25.049  1.00  0.00           N  \\nATOM    342  CA  SER A  47     -19.374   1.741  25.679  1.00  0.00           C  \\nATOM    343  C   SER A  47     -18.114   2.623  25.679  1.00  0.00           C  \\nATOM    344  O   SER A  47     -18.114   3.379  24.797  1.00  0.00           O  \\nATOM    345  CB  SER A  47     -19.248   0.355  25.427  1.00  0.00           C  \\nATOM    346  OG  SER A  47     -18.870   0.481  23.789  1.00  0.00           O  \\nATOM    347  N   TYR A  48     -17.484   2.749  26.939  1.00  0.00           N  \\nATOM    348  CA  TYR A  48     -16.413   3.757  26.939  1.00  0.00           C  \\nATOM    349  C   TYR A  48     -16.602   4.891  27.695  1.00  0.00           C  \\nATOM    350  O   TYR A  48     -17.610   5.269  27.821  1.00  0.00           O  \\nATOM    351  CB  TYR A  48     -16.224   3.757  25.301  1.00  0.00           C  \\nATOM    352  CG  TYR A  48     -15.216   4.639  24.545  1.00  0.00           C  \\nATOM    353  CD1 TYR A  48     -14.082   5.143  24.797  1.00  0.00           C  \\nATOM    354  CD2 TYR A  48     -15.342   4.765  23.411  1.00  0.00           C  \\nATOM    355  CE1 TYR A  48     -13.326   6.025  24.167  1.00  0.00           C  \\nATOM    356  CE2 TYR A  48     -14.712   5.521  22.529  1.00  0.00           C  \\nATOM    357  CZ  TYR A  48     -13.704   6.088  23.033  1.00  0.00           C  \\nATOM    358  OH  TYR A  48     -13.074   6.907  22.151  1.00  0.00           O  \\nATOM    359  N   THR A  49     -15.594   5.647  28.199  1.00  0.00           N  \\nATOM    360  CA  THR A  49     -15.846   6.907  29.207  1.00  0.00           C  \\nATOM    361  C   THR A  49     -15.594   8.041  28.829  1.00  0.00           C  \\nATOM    362  O   THR A  49     -16.350   8.671  27.947  1.00  0.00           O  \\nATOM    363  CB  THR A  49     -15.531   6.403  30.845  1.00  0.00           C  \\nATOM    364  OG1 THR A  49     -14.208   5.899  30.845  1.00  0.00           O  \\nATOM    365  CG2 THR A  49     -16.224   5.395  31.223  1.00  0.00           C  \\nATOM    366  N   ASP A  50     -14.334   8.482  28.703  1.00  0.00           N  \\nATOM    367  CA  ASP A  50     -14.082   9.805  28.199  1.00  0.00           C  \\nATOM    368  C   ASP A  50     -13.704   9.679  26.813  1.00  0.00           C  \\nATOM    369  O   ASP A  50     -13.074   8.545  26.183  1.00  0.00           O  \\nATOM    370  CB  ASP A  50     -13.074  10.435  28.829  1.00  0.00           C  \\nATOM    371  CG  ASP A  50     -12.948  10.183  30.089  1.00  0.00           C  \\nATOM    372  OD1 ASP A  50     -12.066   9.049  30.593  1.00  0.00           O  \\nATOM    373  OD2 ASP A  50     -13.956  10.309  30.971  1.00  0.00           O  \\nATOM    374  N   ALA A  51     -13.830  10.309  25.805  1.00  0.00           N  \\nATOM    375  CA  ALA A  51     -13.704   9.931  24.419  1.00  0.00           C  \\nATOM    376  C   ALA A  51     -15.216   9.805  23.915  1.00  0.00           C  \\nATOM    377  O   ALA A  51     -16.098   9.301  24.167  1.00  0.00           O  \\nATOM    378  CB  ALA A  51     -13.200   8.671  24.041  1.00  0.00           C  \\nATOM    379  N   ASN A  52     -15.342  10.561  22.907  1.00  0.00           N  \\nATOM    380  CA  ASN A  52     -16.539  10.561  22.277  1.00  0.00           C  \\nATOM    381  C   ASN A  52     -17.232  11.380  22.907  1.00  0.00           C  \\nATOM    382  O   ASN A  52     -17.232  12.766  22.781  1.00  0.00           O  \\nATOM    383  CB  ASN A  52     -17.610   9.553  21.899  1.00  0.00           C  \\nATOM    384  CG  ASN A  52     -17.484   8.671  20.639  1.00  0.00           C  \\nATOM    385  OD1 ASN A  52     -16.980   7.789  20.135  1.00  0.00           O  \\nATOM    386  ND2 ASN A  52     -18.240   9.427  19.631  1.00  0.00           N  \\nATOM    387  N   LYS A  53     -17.862  10.939  24.041  1.00  0.00           N  \\nATOM    388  CA  LYS A  53     -18.240  11.947  25.301  1.00  0.00           C  \\nATOM    389  C   LYS A  53     -17.106  12.703  25.805  1.00  0.00           C  \\nATOM    390  O   LYS A  53     -16.476  13.711  25.175  1.00  0.00           O  \\nATOM    391  CB  LYS A  53     -18.870  11.317  26.561  1.00  0.00           C  \\nATOM    392  CG  LYS A  53     -20.130  10.435  26.057  1.00  0.00           C  \\nATOM    393  CD  LYS A  53     -20.634   9.301  27.065  1.00  0.00           C  \\nATOM    394  CE  LYS A  53     -22.146   8.671  26.372  1.00  0.00           C  \\nATOM    395  NZ  LYS A  53     -22.524   7.663  27.191  1.00  0.00           N  \\nATOM    396  N   SER A  54     -16.602  12.325  27.065  1.00  0.00           N  \\nATOM    397  CA  SER A  54     -15.594  13.081  27.821  1.00  0.00           C  \\nATOM    398  C   SER A  54     -15.594  14.593  27.443  1.00  0.00           C  \\nATOM    399  O   SER A  54     -16.476  15.223  27.695  1.00  0.00           O  \\nATOM    400  CB  SER A  54     -14.460  12.703  27.569  1.00  0.00           C  \\nATOM    401  OG  SER A  54     -13.452  12.955  28.703  1.00  0.00           O  \\nATOM    402  N   LYS A  55     -14.838  15.223  26.813  1.00  0.00           N  \\nATOM    403  CA  LYS A  55     -14.964  16.420  26.057  1.00  0.00           C  \\nATOM    404  C   LYS A  55     -16.602  16.987  25.931  1.00  0.00           C  \\nATOM    405  O   LYS A  55     -16.854  18.121  26.183  1.00  0.00           O  \\nATOM    406  CB  LYS A  55     -14.208  16.609  24.734  1.00  0.00           C  \\nATOM    407  CG  LYS A  55     -14.712  17.743  23.600  1.00  0.00           C  \\nATOM    408  CD  LYS A  55     -13.452  17.743  22.844  1.00  0.00           C  \\nATOM    409  CE  LYS A  55     -13.578  17.995  21.143  1.00  0.00           C  \\nATOM    410  NZ  LYS A  55     -12.444  17.743  20.135  1.00  0.00           N  \\nATOM    411  N   GLY A  56     -17.232  16.042  25.679  1.00  0.00           N  \\nATOM    412  CA  GLY A  56     -18.870  16.042  25.301  1.00  0.00           C  \\nATOM    413  C   GLY A  56     -19.122  17.239  24.293  1.00  0.00           C  \\nATOM    414  O   GLY A  56     -18.492  18.247  23.852  1.00  0.00           O  \\nATOM    415  N   ILE A  57     -20.130  17.113  23.285  1.00  0.00           N  \\nATOM    416  CA  ILE A  57     -20.004  17.869  22.025  1.00  0.00           C  \\nATOM    417  C   ILE A  57     -21.264  18.121  21.143  1.00  0.00           C  \\nATOM    418  O   ILE A  57     -22.398  18.499  21.395  1.00  0.00           O  \\nATOM    419  CB  ILE A  57     -18.996  17.491  21.395  1.00  0.00           C  \\nATOM    420  CG1 ILE A  57     -19.374  16.357  20.513  1.00  0.00           C  \\nATOM    421  CG2 ILE A  57     -17.862  17.050  22.025  1.00  0.00           C  \\nATOM    422  CD1 ILE A  57     -19.122  16.609  19.127  1.00  0.00           C  \\nATOM    423  N   VAL A  58     -21.264  17.743  19.631  1.00  0.00           N  \\nATOM    424  CA  VAL A  58     -22.524  17.995  19.001  1.00  0.00           C  \\nATOM    425  C   VAL A  58     -22.020  17.239  17.615  1.00  0.00           C  \\nATOM    426  O   VAL A  58     -21.642  18.121  16.985  1.00  0.00           O  \\nATOM    427  CB  VAL A  58     -23.280  19.129  19.127  1.00  0.00           C  \\nATOM    428  CG1 VAL A  58     -22.398  20.263  18.749  1.00  0.00           C  \\nATOM    429  CG2 VAL A  58     -24.540  19.192  18.371  1.00  0.00           C  \\nATOM    430  N   TRP A  59     -22.398  16.231  16.859  1.00  0.00           N  \\nATOM    431  CA  TRP A  59     -22.020  16.105  15.221  1.00  0.00           C  \\nATOM    432  C   TRP A  59     -22.524  17.239  14.213  1.00  0.00           C  \\nATOM    433  O   TRP A  59     -23.784  17.365  14.339  1.00  0.00           O  \\nATOM    434  CB  TRP A  59     -22.146  14.845  14.717  1.00  0.00           C  \\nATOM    435  CG  TRP A  59     -21.264  13.963  15.347  1.00  0.00           C  \\nATOM    436  CD1 TRP A  59     -21.390  12.703  15.851  1.00  0.00           C  \\nATOM    437  CD2 TRP A  59     -19.878  14.152  15.536  1.00  0.00           C  \\nATOM    438  NE1 TRP A  59     -20.382  12.451  16.607  1.00  0.00           N  \\nATOM    439  CE2 TRP A  59     -19.500  13.207  15.977  1.00  0.00           C  \\nATOM    440  CE3 TRP A  59     -19.248  15.160  15.095  1.00  0.00           C  \\nATOM    441  CZ2 TRP A  59     -18.114  13.459  16.607  1.00  0.00           C  \\nATOM    442  CZ3 TRP A  59     -18.051  15.223  15.473  1.00  0.00           C  \\nATOM    443  CH2 TRP A  59     -17.484  14.467  16.229  1.00  0.00           C  \\nATOM    444  N   ASN A  60     -21.768  17.869  13.772  1.00  0.00           N  \\nATOM    445  CA  ASN A  60     -22.398  19.129  12.953  1.00  0.00           C  \\nATOM    446  C   ASN A  60     -21.642  19.255  11.441  1.00  0.00           C  \\nATOM    447  O   ASN A  60     -20.634  18.751  11.189  1.00  0.00           O  \\nATOM    448  CB  ASN A  60     -22.524  20.515  13.583  1.00  0.00           C  \\nATOM    449  CG  ASN A  60     -22.776  21.649  12.701  1.00  0.00           C  \\nATOM    450  OD1 ASN A  60     -24.036  21.712  11.945  1.00  0.00           O  \\nATOM    451  ND2 ASN A  60     -21.894  22.783  12.701  1.00  0.00           N  \\nATOM    452  N   GLU A  61     -22.020  20.137  10.811  1.00  0.00           N  \\nATOM    453  CA  GLU A  61     -21.138  20.263   9.425  1.00  0.00           C  \\nATOM    454  C   GLU A  61     -19.626  20.389  10.181  1.00  0.00           C  \\nATOM    455  O   GLU A  61     -18.744  19.507   9.803  1.00  0.00           O  \\nATOM    456  CB  GLU A  61     -21.516  21.523   8.669  1.00  0.00           C  \\nATOM    457  CG  GLU A  61     -22.020  21.523   7.409  1.00  0.00           C  \\nATOM    458  CD  GLU A  61     -23.154  20.767   7.283  1.00  0.00           C  \\nATOM    459  OE1 GLU A  61     -24.540  20.767   7.535  1.00  0.00           O  \\nATOM    460  OE2 GLU A  61     -22.902  19.381   6.653  1.00  0.00           O  \\nATOM    461  N   ASN A  62     -19.248  21.397  10.685  1.00  0.00           N  \\nATOM    462  CA  ASN A  62     -17.736  21.586  11.441  1.00  0.00           C  \\nATOM    463  C   ASN A  62     -17.106  20.137  11.945  1.00  0.00           C  \\nATOM    464  O   ASN A  62     -15.972  20.515  12.071  1.00  0.00           O  \\nATOM    465  CB  ASN A  62     -17.736  22.783  11.945  1.00  0.00           C  \\nATOM    466  CG  ASN A  62     -17.610  24.043  11.441  1.00  0.00           C  \\nATOM    467  OD1 ASN A  62     -18.744  24.547  10.559  1.00  0.00           O  \\nATOM    468  ND2 ASN A  62     -16.539  24.799  11.063  1.00  0.00           N  \\nATOM    469  N   THR A  63     -17.610  20.263  13.268  1.00  0.00           N  \\nATOM    470  CA  THR A  63     -17.043  19.255  14.087  1.00  0.00           C  \\nATOM    471  C   THR A  63     -16.602  17.869  13.331  1.00  0.00           C  \\nATOM    472  O   THR A  63     -15.720  17.239  13.331  1.00  0.00           O  \\nATOM    473  CB  THR A  63     -18.051  18.751  15.095  1.00  0.00           C  \\nATOM    474  OG1 THR A  63     -17.358  18.499  15.851  1.00  0.00           O  \\nATOM    475  CG2 THR A  63     -18.492  17.365  14.528  1.00  0.00           C  \\nATOM    476  N   LEU A  64     -17.358  17.617  12.197  1.00  0.00           N  \\nATOM    477  CA  LEU A  64     -16.854  16.357  11.441  1.00  0.00           C  \\nATOM    478  C   LEU A  64     -15.468  16.861  10.685  1.00  0.00           C  \\nATOM    479  O   LEU A  64     -14.838  16.231  10.433  1.00  0.00           O  \\nATOM    480  CB  LEU A  64     -18.114  15.979  10.055  1.00  0.00           C  \\nATOM    481  CG  LEU A  64     -18.744  14.719  10.433  1.00  0.00           C  \\nATOM    482  CD1 LEU A  64     -19.122  14.719  11.567  1.00  0.00           C  \\nATOM    483  CD2 LEU A  64     -19.878  14.593   9.803  1.00  0.00           C  \\nATOM    484  N   MET A  65     -15.468  17.995  10.181  1.00  0.00           N  \\nATOM    485  CA  MET A  65     -13.956  18.499   9.803  1.00  0.00           C  \\nATOM    486  C   MET A  65     -13.200  18.373  11.189  1.00  0.00           C  \\nATOM    487  O   MET A  65     -12.192  17.869  11.189  1.00  0.00           O  \\nATOM    488  CB  MET A  65     -13.956  19.885   9.425  1.00  0.00           C  \\nATOM    489  CG  MET A  65     -13.452  20.137   7.913  1.00  0.00           C  \\nATOM    490  SD  MET A  65     -13.704  18.625   7.283  1.00  0.00           S  \\nATOM    491  CE  MET A  65     -13.452  19.192   5.519  1.00  0.00           C  \\nATOM    492  N   GLU A  66     -13.578  18.751  12.197  1.00  0.00           N  \\nATOM    493  CA  GLU A  66     -13.074  18.751  13.457  1.00  0.00           C  \\nATOM    494  C   GLU A  66     -12.444  17.239  13.646  1.00  0.00           C  \\nATOM    495  O   GLU A  66     -11.310  17.113  13.835  1.00  0.00           O  \\nATOM    496  CB  GLU A  66     -13.830  19.381  14.717  1.00  0.00           C  \\nATOM    497  CG  GLU A  66     -13.578  20.767  14.969  1.00  0.00           C  \\nATOM    498  CD  GLU A  66     -12.318  21.271  15.851  1.00  0.00           C  \\nATOM    499  OE1 GLU A  66     -12.192  22.657  16.103  1.00  0.00           O  \\nATOM    500  OE2 GLU A  66     -11.310  20.641  16.355  1.00  0.00           O  \\nATOM    501  N   TYR A  67     -13.263  16.483  13.457  1.00  0.00           N  \\nATOM    502  CA  TYR A  67     -12.822  14.971  13.457  1.00  0.00           C  \\nATOM    503  C   TYR A  67     -12.192  14.467  12.323  1.00  0.00           C  \\nATOM    504  O   TYR A  67     -11.184  14.089  11.945  1.00  0.00           O  \\nATOM    505  CB  TYR A  67     -14.082  14.341  13.835  1.00  0.00           C  \\nATOM    506  CG  TYR A  67     -14.082  12.955  14.465  1.00  0.00           C  \\nATOM    507  CD1 TYR A  67     -14.838  11.821  14.465  1.00  0.00           C  \\nATOM    508  CD2 TYR A  67     -13.326  13.081  15.599  1.00  0.00           C  \\nATOM    509  CE1 TYR A  67     -14.649  10.876  15.221  1.00  0.00           C  \\nATOM    510  CE2 TYR A  67     -13.200  11.947  16.103  1.00  0.00           C  \\nATOM    511  CZ  TYR A  67     -13.830  10.876  15.977  1.00  0.00           C  \\nATOM    512  OH  TYR A  67     -13.704   9.805  16.607  1.00  0.00           O  \\nATOM    513  N   LEU A  68     -12.822  14.719  11.315  1.00  0.00           N  \\nATOM    514  CA  LEU A  68     -12.192  14.341  10.055  1.00  0.00           C  \\nATOM    515  C   LEU A  68     -10.806  14.719  10.307  1.00  0.00           C  \\nATOM    516  O   LEU A  68      -9.798  13.963  10.307  1.00  0.00           O  \\nATOM    517  CB  LEU A  68     -13.074  14.845   9.173  1.00  0.00           C  \\nATOM    518  CG  LEU A  68     -14.334  13.837   8.795  1.00  0.00           C  \\nATOM    519  CD1 LEU A  68     -15.216  14.467   7.787  1.00  0.00           C  \\nATOM    520  CD2 LEU A  68     -13.704  12.829   7.787  1.00  0.00           C  \\nATOM    521  N   GLU A  69     -10.428  15.853  10.685  1.00  0.00           N  \\nATOM    522  CA  GLU A  69      -8.916  16.231  10.937  1.00  0.00           C  \\nATOM    523  C   GLU A  69      -8.097  14.971  11.441  1.00  0.00           C  \\nATOM    524  O   GLU A  69      -7.278  14.593  10.685  1.00  0.00           O  \\nATOM    525  CB  GLU A  69      -9.042  17.365  11.693  1.00  0.00           C  \\nATOM    526  CG  GLU A  69      -8.034  18.373  11.819  1.00  0.00           C  \\nATOM    527  CD  GLU A  69      -6.900  18.373  12.953  1.00  0.00           C  \\nATOM    528  OE1 GLU A  69      -5.388  18.499  12.827  1.00  0.00           O  \\nATOM    529  OE2 GLU A  69      -7.278  17.869  14.087  1.00  0.00           O  \\nATOM    530  N   ASN A  70      -8.097  14.845  12.575  1.00  0.00           N  \\nATOM    531  CA  ASN A  70      -7.278  13.585  12.827  1.00  0.00           C  \\nATOM    532  C   ASN A  70      -7.404  13.459  14.339  1.00  0.00           C  \\nATOM    533  O   ASN A  70      -7.152  14.215  15.221  1.00  0.00           O  \\nATOM    534  CB  ASN A  70      -5.892  13.837  12.449  1.00  0.00           C  \\nATOM    535  CG  ASN A  70      -4.758  14.089  13.457  1.00  0.00           C  \\nATOM    536  OD1 ASN A  70      -3.624  15.097  13.457  1.00  0.00           O  \\nATOM    537  ND2 ASN A  70      -4.758  13.333  14.717  1.00  0.00           N  \\nATOM    538  N   PRO A  71      -7.908  12.199  14.780  1.00  0.00           N  \\nATOM    539  CA  PRO A  71      -8.475  11.758  15.725  1.00  0.00           C  \\nATOM    540  C   PRO A  71      -7.593  11.758  16.859  1.00  0.00           C  \\nATOM    541  O   PRO A  71      -7.782  11.317  17.993  1.00  0.00           O  \\nATOM    542  CB  PRO A  71      -9.042  10.561  15.536  1.00  0.00           C  \\nATOM    543  CG  PRO A  71      -8.916  10.309  14.213  1.00  0.00           C  \\nATOM    544  CD  PRO A  71      -8.160  11.191  13.583  1.00  0.00           C  \\nATOM    545  N   LYS A  72      -6.522  12.325  16.733  1.00  0.00           N  \\nATOM    546  CA  LYS A  72      -5.388  12.703  17.615  1.00  0.00           C  \\nATOM    547  C   LYS A  72      -5.451  14.215  17.993  1.00  0.00           C  \\nATOM    548  O   LYS A  72      -5.262  14.971  17.111  1.00  0.00           O  \\nATOM    549  CB  LYS A  72      -3.876  12.451  17.111  1.00  0.00           C  \\nATOM    550  CG  LYS A  72      -3.624  11.065  17.363  1.00  0.00           C  \\nATOM    551  CD  LYS A  72      -2.112  10.435  17.237  1.00  0.00           C  \\nATOM    552  CE  LYS A  72      -1.860   8.923  17.741  1.00  0.00           C  \\nATOM    553  NZ  LYS A  72      -0.222   8.545  17.741  1.00  0.00           N  \\nATOM    554  N   LYS A  73      -6.081  14.719  19.127  1.00  0.00           N  \\nATOM    555  CA  LYS A  73      -6.396  16.231  19.127  1.00  0.00           C  \\nATOM    556  C   LYS A  73      -7.782  16.357  18.623  1.00  0.00           C  \\nATOM    557  O   LYS A  73      -8.160  16.231  17.363  1.00  0.00           O  \\nATOM    558  CB  LYS A  73      -5.640  17.113  17.993  1.00  0.00           C  \\nATOM    559  CG  LYS A  73      -4.758  18.247  18.623  1.00  0.00           C  \\nATOM    560  CD  LYS A  73      -3.876  19.003  17.993  1.00  0.00           C  \\nATOM    561  CE  LYS A  73      -2.616  20.389  18.812  1.00  0.00           C  \\nATOM    562  NZ  LYS A  73      -1.986  21.145  18.056  1.00  0.00           N  \\nATOM    563  N   TYR A  74      -8.664  16.231  19.631  1.00  0.00           N  \\nATOM    564  CA  TYR A  74     -10.050  16.105  19.379  1.00  0.00           C  \\nATOM    565  C   TYR A  74     -10.806  14.971  19.883  1.00  0.00           C  \\nATOM    566  O   TYR A  74     -10.554  13.963  19.379  1.00  0.00           O  \\nATOM    567  CB  TYR A  74     -10.554  16.105  17.741  1.00  0.00           C  \\nATOM    568  CG  TYR A  74     -11.814  16.735  17.363  1.00  0.00           C  \\nATOM    569  CD1 TYR A  74     -12.948  16.105  17.237  1.00  0.00           C  \\nATOM    570  CD2 TYR A  74     -11.940  17.995  16.985  1.00  0.00           C  \\nATOM    571  CE1 TYR A  74     -14.145  16.609  16.859  1.00  0.00           C  \\nATOM    572  CE2 TYR A  74     -13.074  18.625  16.607  1.00  0.00           C  \\nATOM    573  CZ  TYR A  74     -14.082  17.869  16.607  1.00  0.00           C  \\nATOM    574  OH  TYR A  74     -15.216  18.373  16.355  1.00  0.00           O  \\nATOM    575  N   ILE A  75     -11.310  15.097  21.017  1.00  0.00           N  \\nATOM    576  CA  ILE A  75     -11.940  14.089  21.647  1.00  0.00           C  \\nATOM    577  C   ILE A  75     -10.806  13.081  21.773  1.00  0.00           C  \\nATOM    578  O   ILE A  75     -10.932  11.821  21.521  1.00  0.00           O  \\nATOM    579  CB  ILE A  75     -13.200  13.585  21.143  1.00  0.00           C  \\nATOM    580  CG1 ILE A  75     -13.641  14.089  19.631  1.00  0.00           C  \\nATOM    581  CG2 ILE A  75     -14.271  13.585  22.025  1.00  0.00           C  \\nATOM    582  CD1 ILE A  75     -14.712  13.459  19.253  1.00  0.00           C  \\nATOM    583  N   PRO A  76      -9.420  13.585  21.899  1.00  0.00           N  \\nATOM    584  CA  PRO A  76      -8.160  12.955  21.647  1.00  0.00           C  \\nATOM    585  C   PRO A  76      -8.160  11.695  22.529  1.00  0.00           C  \\nATOM    586  O   PRO A  76      -8.538  11.695  23.789  1.00  0.00           O  \\nATOM    587  CB  PRO A  76      -6.900  13.837  21.836  1.00  0.00           C  \\nATOM    588  CG  PRO A  76      -7.152  14.845  22.277  1.00  0.00           C  \\nATOM    589  CD  PRO A  76      -8.538  14.845  22.151  1.00  0.00           C  \\nATOM    590  N   GLY A  77      -8.097  10.435  22.088  1.00  0.00           N  \\nATOM    591  CA  GLY A  77      -8.286   9.175  22.907  1.00  0.00           C  \\nATOM    592  C   GLY A  77      -9.546   8.167  22.529  1.00  0.00           C  \\nATOM    593  O   GLY A  77      -9.798   7.159  22.781  1.00  0.00           O  \\nATOM    594  N   THR A  78     -10.050   8.545  21.143  1.00  0.00           N  \\nATOM    595  CA  THR A  78     -11.310   7.789  20.639  1.00  0.00           C  \\nATOM    596  C   THR A  78     -10.995   6.529  20.513  1.00  0.00           C  \\nATOM    597  O   THR A  78      -9.798   6.151  20.009  1.00  0.00           O  \\nATOM    598  CB  THR A  78     -11.625   8.167  19.253  1.00  0.00           C  \\nATOM    599  OG1 THR A  78     -13.074   8.167  19.127  1.00  0.00           O  \\nATOM    600  CG2 THR A  78     -11.184   9.553  18.875  1.00  0.00           C  \\nATOM    601  N   LYS A  79     -11.562   5.521  20.765  1.00  0.00           N  \\nATOM    602  CA  LYS A  79     -11.058   4.072  20.639  1.00  0.00           C  \\nATOM    603  C   LYS A  79     -11.058   3.631  19.001  1.00  0.00           C  \\nATOM    604  O   LYS A  79     -10.932   2.497  18.371  1.00  0.00           O  \\nATOM    605  CB  LYS A  79     -11.562   3.064  21.395  1.00  0.00           C  \\nATOM    606  CG  LYS A  79     -12.570   2.560  20.513  1.00  0.00           C  \\nATOM    607  CD  LYS A  79     -13.704   1.615  20.891  1.00  0.00           C  \\nATOM    608  CE  LYS A  79     -14.838   1.363  20.135  1.00  0.00           C  \\nATOM    609  NZ  LYS A  79     -15.972   0.859  20.513  1.00  0.00           N  \\nATOM    610  N   MET A  80     -11.310   4.639  18.119  1.00  0.00           N  \\nATOM    611  CA  MET A  80     -11.436   4.135  16.607  1.00  0.00           C  \\nATOM    612  C   MET A  80     -10.050   4.198  16.103  1.00  0.00           C  \\nATOM    613  O   MET A  80      -9.294   5.017  16.103  1.00  0.00           O  \\nATOM    614  CB  MET A  80     -12.444   5.017  15.851  1.00  0.00           C  \\nATOM    615  CG  MET A  80     -11.940   6.088  14.969  1.00  0.00           C  \\nATOM    616  SD  MET A  80     -13.074   6.781  13.457  1.00  0.00           S  \\nATOM    617  CE  MET A  80     -12.192   7.852  13.079  1.00  0.00           C  \\nATOM    618  N   ILE A  81      -9.672   3.127  15.725  1.00  0.00           N  \\nATOM    619  CA  ILE A  81      -8.160   3.127  15.095  1.00  0.00           C  \\nATOM    620  C   ILE A  81      -7.908   4.009  13.835  1.00  0.00           C  \\nATOM    621  O   ILE A  81      -8.034   3.505  12.323  1.00  0.00           O  \\nATOM    622  CB  ILE A  81      -7.656   1.867  15.284  1.00  0.00           C  \\nATOM    623  CG1 ILE A  81      -6.522   1.867  16.103  1.00  0.00           C  \\nATOM    624  CG2 ILE A  81      -7.026   1.237  13.835  1.00  0.00           C  \\nATOM    625  CD1 ILE A  81      -6.900   2.245  17.615  1.00  0.00           C  \\nATOM    626  N   PHE A  82      -7.530   5.143  13.961  1.00  0.00           N  \\nATOM    627  CA  PHE A  82      -7.404   5.899  12.449  1.00  0.00           C  \\nATOM    628  C   PHE A  82      -6.144   6.529  12.197  1.00  0.00           C  \\nATOM    629  O   PHE A  82      -5.766   7.411  12.953  1.00  0.00           O  \\nATOM    630  CB  PHE A  82      -8.538   6.655  12.449  1.00  0.00           C  \\nATOM    631  CG  PHE A  82      -8.916   7.159  10.937  1.00  0.00           C  \\nATOM    632  CD1 PHE A  82      -8.916   8.293  10.433  1.00  0.00           C  \\nATOM    633  CD2 PHE A  82      -9.042   6.529  10.055  1.00  0.00           C  \\nATOM    634  CE1 PHE A  82      -9.042   8.671   9.173  1.00  0.00           C  \\nATOM    635  CE2 PHE A  82      -9.042   6.781   8.669  1.00  0.00           C  \\nATOM    636  CZ  PHE A  82      -9.042   7.852   8.291  1.00  0.00           C  \\nATOM    637  N   ALA A  83      -5.640   6.529  11.189  1.00  0.00           N  \\nATOM    638  CA  ALA A  83      -4.506   7.285  10.937  1.00  0.00           C  \\nATOM    639  C   ALA A  83      -4.758   8.797  10.811  1.00  0.00           C  \\nATOM    640  O   ALA A  83      -4.884   9.679  11.441  1.00  0.00           O  \\nATOM    641  CB  ALA A  83      -3.876   6.907   9.173  1.00  0.00           C  \\nATOM    642  N   GLY A  84      -5.136   9.301   9.677  1.00  0.00           N  \\nATOM    643  CA  GLY A  84      -5.325  10.687   9.299  1.00  0.00           C  \\nATOM    644  C   GLY A  84      -6.396  11.317   8.543  1.00  0.00           C  \\nATOM    645  O   GLY A  84      -7.467  11.569   8.921  1.00  0.00           O  \\nATOM    646  N   ILE A  85      -6.207  11.569   7.283  1.00  0.00           N  \\nATOM    647  CA  ILE A  85      -7.152  12.451   6.653  1.00  0.00           C  \\nATOM    648  C   ILE A  85      -6.774  13.963   6.905  1.00  0.00           C  \\nATOM    649  O   ILE A  85      -7.656  14.719   7.220  1.00  0.00           O  \\nATOM    650  CB  ILE A  85      -8.664  12.199   7.031  1.00  0.00           C  \\nATOM    651  CG1 ILE A  85      -9.168  10.813   6.149  1.00  0.00           C  \\nATOM    652  CG2 ILE A  85      -9.420  13.333   6.401  1.00  0.00           C  \\nATOM    653  CD1 ILE A  85     -10.176  11.191   5.204  1.00  0.00           C  \\nATOM    654  N   LYS A  86      -5.640  14.467   6.527  1.00  0.00           N  \\nATOM    655  CA  LYS A  86      -5.325  15.727   7.157  1.00  0.00           C  \\nATOM    656  C   LYS A  86      -5.136  16.546   5.771  1.00  0.00           C  \\nATOM    657  O   LYS A  86      -4.254  17.365   5.267  1.00  0.00           O  \\nATOM    658  CB  LYS A  86      -3.876  15.727   7.913  1.00  0.00           C  \\nATOM    659  CG  LYS A  86      -3.120  14.467   7.535  1.00  0.00           C  \\nATOM    660  CD  LYS A  86      -1.356  14.215   7.787  1.00  0.00           C  \\nATOM    661  CE  LYS A  86      -0.852  12.955   6.653  1.00  0.00           C  \\nATOM    662  NZ  LYS A  86       0.660  12.577   7.157  1.00  0.00           N  \\nATOM    663  N   LYS A  87      -5.892  16.357   5.015  1.00  0.00           N  \\nATOM    664  CA  LYS A  87      -5.892  17.239   3.755  1.00  0.00           C  \\nATOM    665  C   LYS A  87      -7.278  17.869   3.377  1.00  0.00           C  \\nATOM    666  O   LYS A  87      -8.160  17.239   2.873  1.00  0.00           O  \\nATOM    667  CB  LYS A  87      -5.514  16.294   2.558  1.00  0.00           C  \\nATOM    668  CG  LYS A  87      -4.632  16.735   1.235  1.00  0.00           C  \\nATOM    669  CD  LYS A  87      -4.254  15.727   0.353  1.00  0.00           C  \\nATOM    670  CE  LYS A  87      -3.120  16.231  -0.403  1.00  0.00           C  \\nATOM    671  NZ  LYS A  87      -2.868  15.727  -1.915  1.00  0.00           N  \\nATOM    672  N   LYS A  88      -7.278  19.129   3.251  1.00  0.00           N  \\nATOM    673  CA  LYS A  88      -8.412  19.885   2.621  1.00  0.00           C  \\nATOM    674  C   LYS A  88      -9.168  18.877   1.613  1.00  0.00           C  \\nATOM    675  O   LYS A  88     -10.428  18.499   1.613  1.00  0.00           O  \\nATOM    676  CB  LYS A  88      -7.782  21.019   1.991  1.00  0.00           C  \\nATOM    677  CG  LYS A  88      -6.270  20.767   1.865  1.00  0.00           C  \\nATOM    678  CD  LYS A  88      -5.514  21.271   0.605  1.00  0.00           C  \\nATOM    679  CE  LYS A  88      -5.766  20.767  -0.403  1.00  0.00           C  \\nATOM    680  NZ  LYS A  88      -5.388  21.523  -1.663  1.00  0.00           N  \\nATOM    681  N   GLY A  89      -8.538  18.499   0.353  1.00  0.00           N  \\nATOM    682  CA  GLY A  89      -9.168  17.491  -0.781  1.00  0.00           C  \\nATOM    683  C   GLY A  89     -10.050  16.357  -0.025  1.00  0.00           C  \\nATOM    684  O   GLY A  89     -11.184  16.357  -0.025  1.00  0.00           O  \\nATOM    685  N   GLU A  90      -9.546  15.475   0.479  1.00  0.00           N  \\nATOM    686  CA  GLU A  90     -10.050  14.089   1.109  1.00  0.00           C  \\nATOM    687  C   GLU A  90     -10.932  14.593   2.369  1.00  0.00           C  \\nATOM    688  O   GLU A  90     -12.192  14.341   2.621  1.00  0.00           O  \\nATOM    689  CB  GLU A  90      -9.042  13.207   1.613  1.00  0.00           C  \\nATOM    690  CG  GLU A  90      -8.286  12.577   0.605  1.00  0.00           C  \\nATOM    691  CD  GLU A  90      -6.900  11.821   0.857  1.00  0.00           C  \\nATOM    692  OE1 GLU A  90      -6.018  11.443  -0.277  1.00  0.00           O  \\nATOM    693  OE2 GLU A  90      -6.270  11.569   2.243  1.00  0.00           O  \\nATOM    694  N   ARG A  91     -10.554  15.475   2.999  1.00  0.00           N  \\nATOM    695  CA  ARG A  91     -11.184  15.916   4.511  1.00  0.00           C  \\nATOM    696  C   ARG A  91     -12.570  16.672   3.881  1.00  0.00           C  \\nATOM    697  O   ARG A  91     -13.578  16.357   4.133  1.00  0.00           O  \\nATOM    698  CB  ARG A  91     -10.302  16.987   5.141  1.00  0.00           C  \\nATOM    699  CG  ARG A  91     -10.554  17.113   6.527  1.00  0.00           C  \\nATOM    700  CD  ARG A  91      -9.420  18.121   7.283  1.00  0.00           C  \\nATOM    701  NE  ARG A  91      -9.294  19.381   6.527  1.00  0.00           N  \\nATOM    702  CZ  ARG A  91      -7.908  20.137   6.464  1.00  0.00           C  \\nATOM    703  NH1 ARG A  91      -6.900  19.507   7.535  1.00  0.00           N  \\nATOM    704  NH2 ARG A  91      -7.530  21.145   5.519  1.00  0.00           N  \\nATOM    705  N   GLN A  92     -12.444  17.743   3.251  1.00  0.00           N  \\nATOM    706  CA  GLN A  92     -13.830  18.373   2.369  1.00  0.00           C  \\nATOM    707  C   GLN A  92     -14.586  17.113   1.991  1.00  0.00           C  \\nATOM    708  O   GLN A  92     -15.720  16.735   2.243  1.00  0.00           O  \\nATOM    709  CB  GLN A  92     -13.704  19.633   1.739  1.00  0.00           C  \\nATOM    710  CG  GLN A  92     -13.830  19.255   0.164  1.00  0.00           C  \\nATOM    711  CD  GLN A  92     -13.200  20.326  -0.907  1.00  0.00           C  \\nATOM    712  OE1 GLN A  92     -11.688  19.885  -0.403  1.00  0.00           O  \\nATOM    713  NE2 GLN A  92     -13.452  21.460  -1.411  1.00  0.00           N  \\nATOM    714  N   ASP A  93     -13.956  16.357   0.983  1.00  0.00           N  \\nATOM    715  CA  ASP A  93     -14.838  15.223   0.605  1.00  0.00           C  \\nATOM    716  C   ASP A  93     -15.342  14.341   1.613  1.00  0.00           C  \\nATOM    717  O   ASP A  93     -16.602  13.837   1.487  1.00  0.00           O  \\nATOM    718  CB  ASP A  93     -13.830  14.593  -0.529  1.00  0.00           C  \\nATOM    719  CG  ASP A  93     -13.704  14.971  -1.915  1.00  0.00           C  \\nATOM    720  OD1 ASP A  93     -12.444  15.727  -2.419  1.00  0.00           O  \\nATOM    721  OD2 ASP A  93     -14.838  15.223  -2.797  1.00  0.00           O  \\nATOM    722  N   LEU A  94     -14.838  13.963   2.495  1.00  0.00           N  \\nATOM    723  CA  LEU A  94     -15.531  13.207   3.503  1.00  0.00           C  \\nATOM    724  C   LEU A  94     -16.728  13.963   3.881  1.00  0.00           C  \\nATOM    725  O   LEU A  94     -17.925  13.459   3.881  1.00  0.00           O  \\nATOM    726  CB  LEU A  94     -14.586  13.018   4.763  1.00  0.00           C  \\nATOM    727  CG  LEU A  94     -14.586  11.443   5.267  1.00  0.00           C  \\nATOM    728  CD1 LEU A  94     -15.783  10.813   4.637  1.00  0.00           C  \\nATOM    729  CD2 LEU A  94     -13.578  10.561   4.889  1.00  0.00           C  \\nATOM    730  N   VAL A  95     -16.539  15.097   4.637  1.00  0.00           N  \\nATOM    731  CA  VAL A  95     -17.736  15.979   5.141  1.00  0.00           C  \\nATOM    732  C   VAL A  95     -18.744  15.853   3.881  1.00  0.00           C  \\nATOM    733  O   VAL A  95     -20.004  15.601   4.322  1.00  0.00           O  \\nATOM    734  CB  VAL A  95     -17.358  17.365   5.393  1.00  0.00           C  \\nATOM    735  CG1 VAL A  95     -18.492  18.184   5.519  1.00  0.00           C  \\nATOM    736  CG2 VAL A  95     -16.476  17.491   6.401  1.00  0.00           C  \\nATOM    737  N   ALA A  96     -18.492  15.979   2.621  1.00  0.00           N  \\nATOM    738  CA  ALA A  96     -19.752  15.601   1.739  1.00  0.00           C  \\nATOM    739  C   ALA A  96     -20.130  14.215   1.802  1.00  0.00           C  \\nATOM    740  O   ALA A  96     -21.390  13.837   1.802  1.00  0.00           O  \\nATOM    741  CB  ALA A  96     -19.500  16.105   0.353  1.00  0.00           C  \\nATOM    742  N   TYR A  97     -19.248  13.333   1.865  1.00  0.00           N  \\nATOM    743  CA  TYR A  97     -19.626  11.884   1.865  1.00  0.00           C  \\nATOM    744  C   TYR A  97     -20.886  11.632   2.873  1.00  0.00           C  \\nATOM    745  O   TYR A  97     -22.146  11.317   2.621  1.00  0.00           O  \\nATOM    746  CB  TYR A  97     -18.744  10.813   2.243  1.00  0.00           C  \\nATOM    747  CG  TYR A  97     -19.374   9.427   1.739  1.00  0.00           C  \\nATOM    748  CD1 TYR A  97     -19.626   8.419   2.621  1.00  0.00           C  \\nATOM    749  CD2 TYR A  97     -19.626   9.175   0.353  1.00  0.00           C  \\nATOM    750  CE1 TYR A  97     -20.256   7.348   2.117  1.00  0.00           C  \\nATOM    751  CE2 TYR A  97     -20.256   8.041  -0.403  1.00  0.00           C  \\nATOM    752  CZ  TYR A  97     -20.508   7.285   0.857  1.00  0.00           C  \\nATOM    753  OH  TYR A  97     -21.264   6.403   0.353  1.00  0.00           O  \\nATOM    754  N   LEU A  98     -20.760  11.947   4.007  1.00  0.00           N  \\nATOM    755  CA  LEU A  98     -21.768  11.695   5.393  1.00  0.00           C  \\nATOM    756  C   LEU A  98     -23.154  12.703   5.015  1.00  0.00           C  \\nATOM    757  O   LEU A  98     -24.288  12.199   4.889  1.00  0.00           O  \\nATOM    758  CB  LEU A  98     -21.012  11.695   6.527  1.00  0.00           C  \\nATOM    759  CG  LEU A  98     -20.004  10.939   6.401  1.00  0.00           C  \\nATOM    760  CD1 LEU A  98     -19.374  10.687   7.661  1.00  0.00           C  \\nATOM    761  CD2 LEU A  98     -20.508   9.553   5.771  1.00  0.00           C  \\nATOM    762  N   LYS A  99     -23.028  13.963   4.889  1.00  0.00           N  \\nATOM    763  CA  LYS A  99     -24.288  14.593   4.385  1.00  0.00           C  \\nATOM    764  C   LYS A  99     -25.044  13.963   2.999  1.00  0.00           C  \\nATOM    765  O   LYS A  99     -26.304  13.837   3.125  1.00  0.00           O  \\nATOM    766  CB  LYS A  99     -23.910  16.105   4.259  1.00  0.00           C  \\nATOM    767  CG  LYS A  99     -24.918  17.113   4.259  1.00  0.00           C  \\nATOM    768  CD  LYS A  99     -24.666  18.625   3.881  1.00  0.00           C  \\nATOM    769  CE  LYS A  99     -25.674  19.633   4.133  1.00  0.00           C  \\nATOM    770  NZ  LYS A  99     -25.674  21.082   3.755  1.00  0.00           N  \\nATOM    771  N   SER A 100     -24.540  13.459   1.991  1.00  0.00           N  \\nATOM    772  CA  SER A 100     -25.296  12.577   0.857  1.00  0.00           C  \\nATOM    773  C   SER A 100     -25.548  11.191   1.361  1.00  0.00           C  \\nATOM    774  O   SER A 100     -26.934  10.813   1.361  1.00  0.00           O  \\nATOM    775  CB  SER A 100     -24.540  12.577  -0.655  1.00  0.00           C  \\nATOM    776  OG  SER A 100     -25.296  11.695  -1.663  1.00  0.00           O  \\nATOM    777  N   ALA A 101     -24.666  10.435   1.865  1.00  0.00           N  \\nATOM    778  CA  ALA A 101     -25.044   8.797   2.243  1.00  0.00           C  \\nATOM    779  C   ALA A 101     -25.800   8.923   3.629  1.00  0.00           C  \\nATOM    780  O   ALA A 101     -26.808   8.293   3.755  1.00  0.00           O  \\nATOM    781  CB  ALA A 101     -23.910   8.167   2.369  1.00  0.00           C  \\nATOM    782  N   THR A 102     -25.422   9.427   4.637  1.00  0.00           N  \\nATOM    783  CA  THR A 102     -26.052   9.301   6.149  1.00  0.00           C  \\nATOM    784  C   THR A 102     -27.690   9.805   5.897  1.00  0.00           C  \\nATOM    785  O   THR A 102     -28.572   9.049   6.149  1.00  0.00           O  \\nATOM    786  CB  THR A 102     -25.548  10.246   7.220  1.00  0.00           C  \\nATOM    787  OG1 THR A 102     -24.666   9.427   7.913  1.00  0.00           O  \\nATOM    788  CG2 THR A 102     -26.430  10.876   8.039  1.00  0.00           C  \\nATOM    789  N   SER A 103     -27.942  10.939   5.645  1.00  0.00           N  \\nATOM    790  CA  SER A 103     -29.454  11.317   5.519  1.00  0.00           C  \\nATOM    791  C   SER A 103     -30.084  10.813   4.385  1.00  0.00           C  \\nATOM    792  O   SER A 103     -29.454   9.931   3.377  1.00  0.00           O  \\nATOM    793  CB  SER A 103     -29.580  12.892   5.897  1.00  0.00           C  \\nATOM    794  OG  SER A 103     -28.320  13.585   5.393  1.00  0.00           O  \\nATOM    795  OXT SER A 103     -31.344  11.317   4.133  1.00  0.00           O  \\nTER     796      SER A 103                                                      \\nHETATM 1593 FE   HEC A 104     -15.216   5.773  12.701  1.00  0.00          FE  \\nHETATM 1594  CHA HEC A 104     -16.980   6.466  15.347  1.00  0.00           C  \\nHETATM 1595  CHB HEC A 104     -16.476   8.293  11.189  1.00  0.00           C  \\nHETATM 1596  CHC HEC A 104     -13.578   5.017   9.929  1.00  0.00           C  \\nHETATM 1597  CHD HEC A 104     -13.956   3.127  14.339  1.00  0.00           C  \\nHETATM 1598  NA  HEC A 104     -16.476   7.096  13.205  1.00  0.00           N  \\nHETATM 1599  C1A HEC A 104     -17.106   7.159  14.465  1.00  0.00           C  \\nHETATM 1600  C2A HEC A 104     -17.862   8.167  14.591  1.00  0.00           C  \\nHETATM 1601  C3A HEC A 104     -17.736   8.797  13.205  1.00  0.00           C  \\nHETATM 1602  C4A HEC A 104     -17.114   8.167  12.323  1.00  0.00           C  \\nHETATM 1603  CMA HEC A 104     -18.303  10.057  12.701  1.00  0.00           C  \\nHETATM 1604  CAA HEC A 104     -18.618   8.167  15.599  1.00  0.00           C  \\nHETATM 1605  CBA HEC A 104     -20.067   8.734  15.473  1.00  0.00           C  \\nHETATM 1606  CGA HEC A 104     -20.823   8.734  16.859  1.00  0.00           C  \\nHETATM 1607  O1A HEC A 104     -21.705   8.041  17.111  1.00  0.00           O  \\nHETATM 1608  O2A HEC A 104     -20.508   9.490  17.741  1.00  0.00           O  \\nHETATM 1609  NB  HEC A 104     -15.090   6.466  11.126  1.00  0.00           N  \\nHETATM 1610  C1B HEC A 104     -15.594   7.348  10.559  1.00  0.00           C  \\nHETATM 1611  C2B HEC A 104     -15.216   7.663   9.299  1.00  0.00           C  \\nHETATM 1612  C3B HEC A 104     -14.334   6.592   8.921  1.00  0.00           C  \\nHETATM 1613  C4B HEC A 104     -14.334   6.025   9.929  1.00  0.00           C  \\nHETATM 1614  CMB HEC A 104     -15.657   8.797   8.543  1.00  0.00           C  \\nHETATM 1615  CAB HEC A 104     -13.578   6.781   7.409  1.00  0.00           C  \\nHETATM 1616  CBB HEC A 104     -12.570   7.663   7.661  1.00  0.00           C  \\nHETATM 1617  NC  HEC A 104     -13.956   4.387  12.197  1.00  0.00           N  \\nHETATM 1618  C1C HEC A 104     -13.452   4.261  10.811  1.00  0.00           C  \\nHETATM 1619  C2C HEC A 104     -12.759   3.001  10.811  1.00  0.00           C  \\nHETATM 1620  C3C HEC A 104     -12.948   2.623  11.945  1.00  0.00           C  \\nHETATM 1621  C4C HEC A 104     -13.704   3.379  12.953  1.00  0.00           C  \\nHETATM 1622  CMC HEC A 104     -11.688   2.497   9.803  1.00  0.00           C  \\nHETATM 1623  CAC HEC A 104     -12.444   1.111  12.197  1.00  0.00           C  \\nHETATM 1624  CBC HEC A 104     -11.184   0.859  12.008  1.00  0.00           C  \\nHETATM 1625  ND  HEC A 104     -15.405   5.143  14.213  1.00  0.00           N  \\nHETATM 1626  C1D HEC A 104     -14.838   4.009  14.843  1.00  0.00           C  \\nHETATM 1627  C2D HEC A 104     -15.216   3.757  15.977  1.00  0.00           C  \\nHETATM 1628  C3D HEC A 104     -16.098   4.765  16.355  1.00  0.00           C  \\nHETATM 1629  C4D HEC A 104     -16.161   5.395  15.347  1.00  0.00           C  \\nHETATM 1630  CMD HEC A 104     -14.838   2.749  16.733  1.00  0.00           C  \\nHETATM 1631  CAD HEC A 104     -16.854   4.513  17.615  1.00  0.00           C  \\nHETATM 1632  CBD HEC A 104     -16.098   4.135  18.623  1.00  0.00           C  \\nHETATM 1633  CGD HEC A 104     -15.216   5.017  19.505  1.00  0.00           C  \\nHETATM 1634  O1D HEC A 104     -14.082   5.773  18.560  1.00  0.00           O  \\nHETATM 1635  O2D HEC A 104     -15.405   5.647  20.639  1.00  0.00           O  \\nHETATM 1679  O   HOH A 105     -16.496   9.304  17.402  1.00  0.00           O  \\nCONECT  104 1615                                                                \\nCONECT  124 1623                                                                \\nCONECT  134 1593                                                                \\nCONECT  356 1635                                                                \\nCONECT  616 1593                                                                \\nCONECT  900 1658                                                                \\nCONECT  920 1666                                                                \\nCONECT  930 1636                                                                \\nCONECT 1152 1678                                                                \\nCONECT 1412 1636                                                                \\nCONECT 1593  134  616 1598 1609                                                 \\nCONECT 1593 1617 1625                                                           \\nCONECT 1594 1599 1629                                                           \\nCONECT 1595 1602 1610                                                           \\nCONECT 1596 1613 1618                                                           \\nCONECT 1597 1621 1626                                                           \\nCONECT 1598 1593 1599 1602                                                      \\nCONECT 1599 1594 1598 1600                                                      \\nCONECT 1600 1599 1601 1604                                                      \\nCONECT 1601 1600 1602 1603                                                      \\nCONECT 1602 1595 1598 1601                                                      \\nCONECT 1603 1601                                                                \\nCONECT 1604 1600 1605                                                           \\nCONECT 1605 1604 1606                                                           \\nCONECT 1606 1605 1607 1608                                                      \\nCONECT 1607 1606                                                                \\nCONECT 1608 1606                                                                \\nCONECT 1609 1593 1610 1613                                                      \\nCONECT 1610 1595 1609 1611                                                      \\nCONECT 1611 1610 1612 1614                                                      \\nCONECT 1612 1611 1613 1615                                                      \\nCONECT 1613 1596 1609 1612                                                      \\nCONECT 1614 1611                                                                \\nCONECT 1615  104 1612 1616                                                      \\nCONECT 1616 1615                                                                \\nCONECT 1617 1593 1618 1621                                                      \\nCONECT 1618 1596 1617 1619                                                      \\nCONECT 1619 1618 1620 1622                                                      \\nCONECT 1620 1619 1621 1623                                                      \\nCONECT 1621 1597 1617 1620                                                      \\nCONECT 1622 1619                                                                \\nCONECT 1623  124 1620 1624                                                      \\nCONECT 1624 1623                                                                \\nCONECT 1625 1593 1626 1629                                                      \\nCONECT 1626 1597 1625 1627                                                      \\nCONECT 1627 1626 1628 1630                                                      \\nCONECT 1628 1627 1629 1631                                                      \\nCONECT 1629 1594 1625 1628                                                      \\nCONECT 1630 1627                                                                \\nCONECT 1631 1628 1632                                                           \\nCONECT 1632 1631 1633                                                           \\nCONECT 1633 1632 1634 1635                                                      \\nCONECT 1634 1633                                                                \\nCONECT 1635  356 1633                                                           \\nCONECT 1636  930 1412 1641 1652                                                 \\nCONECT 1636 1660 1668                                                           \\nCONECT 1637 1642 1672                                                           \\nCONECT 1638 1645 1653                                                           \\nCONECT 1639 1656 1661                                                           \\nCONECT 1640 1664 1669                                                           \\nCONECT 1641 1636 1642 1645                                                      \\nCONECT 1642 1637 1641 1643                                                      \\nCONECT 1643 1642 1644 1647                                                      \\nCONECT 1644 1643 1645 1646                                                      \\nCONECT 1645 1638 1641 1644                                                      \\nCONECT 1646 1644                                                                \\nCONECT 1647 1643 1648                                                           \\nCONECT 1648 1647 1649                                                           \\nCONECT 1649 1648 1650 1651                                                      \\nCONECT 1650 1649                                                                \\nCONECT 1651 1649                                                                \\nCONECT 1652 1636 1653 1656                                                      \\nCONECT 1653 1638 1652 1654                                                      \\nCONECT 1654 1653 1655 1657                                                      \\nCONECT 1655 1654 1656 1658                                                      \\nCONECT 1656 1639 1652 1655                                                      \\nCONECT 1657 1654                                                                \\nCONECT 1658  900 1655 1659                                                      \\nCONECT 1659 1658                                                                \\nCONECT 1660 1636 1661 1664                                                      \\nCONECT 1661 1639 1660 1662                                                      \\nCONECT 1662 1661 1663 1665                                                      \\nCONECT 1663 1662 1664 1666                                                      \\nCONECT 1664 1640 1660 1663                                                      \\nCONECT 1665 1662                                                                \\nCONECT 1666  920 1663 1667                                                      \\nCONECT 1667 1666                                                                \\nCONECT 1668 1636 1669 1672                                                      \\nCONECT 1669 1640 1668 1670                                                      \\nCONECT 1670 1669 1671 1673                                                      \\nCONECT 1671 1670 1672 1674                                                      \\nCONECT 1672 1637 1668 1671                                                      \\nCONECT 1673 1670                                                                \\nCONECT 1674 1671 1675                                                           \\nCONECT 1675 1674 1676                                                           \\nCONECT 1676 1675 1677 1678                                                      \\nCONECT 1677 1676                                                                \\nCONECT 1678 1152 1676                                                           \\nMASTER      287    0    2   10    0    0    9    9 1677    2   98   16          \\nEND                                                                             \\n\",\"pdb\");\n",
       "\tviewer_16742555228826048.setBackgroundColor(\"white\");\n",
       "\tviewer_16742555228826048.setStyle({\"cartoon\": {\"color\": \"white\"}});\n",
       "\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"5\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"6\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"7\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"8\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"9\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"10\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"11\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"12\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"13\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"14\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"15\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"16\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"17\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"18\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"19\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"20\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"21\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"22\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"23\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"24\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"25\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"26\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555228826048.addStyle({\"chain\": \"A\", \"resi\": [\"27\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"5\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"6\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"6\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"7\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"7\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"15\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"8\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"12\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"13\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"15\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"16\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"11\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"13\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"13\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"15\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"13\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"16\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"13\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"15\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"15\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"16\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"18\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"22\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"23\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"24\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"21\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"25\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"23\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"25\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228826048.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"23\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"26\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555228826048.zoomTo();\n",
       "viewer_16742555228826048.render();\n",
       "});\n",
       "</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "<py3Dmol.view at 0x7f127a2bdb20>"
      ]
     },
     "execution_count": 52,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "view_frustration_dif(pdb_file_chain,path_bs+'exon_0.pdb',fr_dif)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# considering also contacts exon-other parts"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/tmp/ipykernel_57087/3890593972.py:17: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  fr_dif2.fr_dif[x]=-fr_dif2['fr_all'][x]+fr_dif2['fr_exon'][x]\n"
     ]
    }
   ],
   "source": [
    "exon_res=np.unique(np.hstack([frustra_ex['#Res1'],frustra_ex.Res2]))\n",
    "ix=[]\n",
    "for i in exon_res:\n",
    "    ix.append(frustra_all.index[frustra_all['#Res1']==i])\n",
    "    ix.append(frustra_all.index[frustra_all['#Res1']==j])\n",
    "ix_=np.unique(np.hstack(ix))\n",
    "fr_all_ex=frustra_all.loc[ix_].reset_index()\n",
    "fr_index_dict2={'highly':1,'neutral':0,'minimally':-1}\n",
    "fr_all_ex['fr_dif']=fr_all_ex.FrstState.map(fr_index_dict2) #this way is comparable with the other result\n",
    "\n",
    "fr_dif2=fr_all_ex.merge(frustra_ex,on='contact',how='left')\n",
    "fr_dif2['fr_exon']=fr_dif2.FrstState_y.map(fr_index_dict)\n",
    "fr_dif2['fr_all']=fr_dif2.FrstState_x.map(fr_index_dict)\n",
    "\n",
    "for x in range(len(fr_dif2)):\n",
    "    if ~np.isnan(fr_dif2['fr_exon'][x]):\n",
    "        fr_dif2.fr_dif[x]=-fr_dif2['fr_all'][x]+fr_dif2['fr_exon'][x]\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "fr_dif  FrstState_x  FrstState_y\n",
       "-1      highly       neutral        0.013514\n",
       "        neutral      minimally      0.040541\n",
       " 0      highly       highly         0.108108\n",
       "        minimally    minimally      0.202703\n",
       "        neutral      neutral        0.364865\n",
       " 1      minimally    neutral        0.108108\n",
       "        neutral      highly         0.162162\n",
       "Name: Res2_x, dtype: float64"
      ]
     },
     "execution_count": 54,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "fr_dif2.groupby(['fr_dif','FrstState_x','FrstState_y']).Res2_x.count()/len(fr_dif)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_16742555283712852\"  style=\"position: relative; width: 640px; height: 480px\">\n        <p id=\"3dmolwarning_16742555283712852\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    //this is to ignore the existence of requirejs amd\n    var savedexports, savedmodule;\n    if (typeof exports !== 'undefined') savedexports = exports;\n    else exports = {}\n    if (typeof module !== 'undefined') savedmodule = module;\n    else module = {}\n\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n        exports = savedexports;\n        module = savedmodule;\n        resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://3dmol.org/build/3Dmol.js');\n}\n\nvar viewer_16742555283712852 = null;\nvar warn = document.getElementById(\"3dmolwarning_16742555283712852\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_16742555283712852 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_16742555283712852\"),{backgroundColor:\"white\"});\nviewer_16742555283712852.zoomTo();\n\tviewer_16742555283712852.addModel(\"HEADER    ELECTRON TRANSPORT                      01-AUG-76   1CYC              \\nTITLE     THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C AT 2.3     \\nTITLE    2 ANGSTROMS RESOLUTION. II. STRUCTURE AND FUNCTION                     \\nCAVEAT     1CYC    THR A 40 HAS WRONG CHIRALITY AT ATOM CB THR A 78 HAS WRONG   \\nCAVEAT   2 1CYC    CHIRALITY AT ATOM CB THR B 40 HAS WRONG CHIRALITY AT ATOM    \\nCAVEAT   3 1CYC    CB THR B 78 HAS WRONG CHIRALITY AT ATOM CB                   \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: FERROCYTOCHROME C;                                         \\nCOMPND   3 CHAIN: A, B;                                                         \\nCOMPND   4 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 ORGANISM_SCIENTIFIC: KATSUWONUS PELAMIS;                             \\nSOURCE   3 ORGANISM_COMMON: SKIPJACK TUNA;                                      \\nSOURCE   4 ORGANISM_TAXID: 8226                                                 \\nKEYWDS    ELECTRON TRANSPORT                                                    \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    N.TANAKA,T.YAMANE,T.TSUKIHARA,T.ASHIDA,M.KAKUDO                       \\nREVDAT  17   03-MAR-21 1CYC    1       CAVEAT COMPND REMARK HET                 \\nREVDAT  17 2                   1       HETNAM HETSYN FORMUL LINK                \\nREVDAT  17 3                   1       ATOM                                     \\nREVDAT  16   29-NOV-17 1CYC    1       HELIX                                    \\nREVDAT  15   24-FEB-09 1CYC    1       VERSN                                    \\nREVDAT  14   01-APR-03 1CYC    1       JRNL                                     \\nREVDAT  13   27-JAN-84 1CYC    1       REMARK                                   \\nREVDAT  12   30-SEP-83 1CYC    1       REVDAT                                   \\nREVDAT  11   19-APR-83 1CYC    1       COMPND SOURCE REMARK                     \\nREVDAT  10   15-JAN-82 1CYC    1       REMARK                                   \\nREVDAT   9   06-JAN-82 1CYC    3       SCALE                                    \\nREVDAT   8   31-DEC-80 1CYC    1       REMARK                                   \\nREVDAT   7   07-APR-80 1CYC    3       SOURCE SEQRES ATOM                       \\nREVDAT   6   23-APR-79 1CYC    3       ATOM                                     \\nREVDAT   5   01-NOV-77 1CYC    1       AUTHOR REMARK FORMUL                     \\nREVDAT   4   09-SEP-77 1CYC    3       ATOM                                     \\nREVDAT   3   13-JUN-77 1CYC    1       HET                                      \\nREVDAT   2   25-OCT-76 1CYC    2       CONECT                                   \\nREVDAT   1   06-OCT-76 1CYC    0                                                \\nJRNL        AUTH   N.TANAKA,T.YAMANE,T.TSUKIHARA,T.ASHIDA,M.KAKUDO              \\nJRNL        TITL   THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C   \\nJRNL        TITL 2 AT 2.3 A RESOLUTION. II. STRUCTURE AND FUNCTION.             \\nJRNL        REF    J.BIOCHEM.(TOKYO)             V.  77   147 1975              \\nJRNL        REFN                   ISSN 0021-924X                               \\nJRNL        PMID   166072                                                       \\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   T.TSUKIHARA,T.YAMANE,N.TANAKA,T.ASHIDA,M.KAKUDO              \\nREMARK   1  TITL   OXIDATION OF A FERROCYTOCHROME C IN THE CRYSTALLINE          \\nREMARK   1  TITL 2 STATE-STRUCTURAL CHANGE AND ANION BINDING                    \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  73  1163 1973              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 2                                                          \\nREMARK   1  AUTH   T.ASHIDA,N.TANAKA,T.YAMANE,T.TSUKIHARA,M.KAKUDO              \\nREMARK   1  TITL   THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C   \\nREMARK   1  TITL 2 AT 2.3 ANGSTROMS RESOLUTION                                  \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  73   463 1973              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 3                                                          \\nREMARK   1  AUTH   T.ASHIDA,T.UEKI,T.TSUKIHARA,A.SUGIHARA,T.TAKANO,M.KAKUDO     \\nREMARK   1  TITL   THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C   \\nREMARK   1  TITL 2 AT 4 ANGSTROMS RESOLUTION                                    \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  70   913 1971              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 4                                                          \\nREMARK   1  AUTH   T.NAKAYAMA,K.TITANI,K.NARITA                                 \\nREMARK   1  TITL   THE AMINO ACID SEQUENCE OF CYTOCHROME C FROM BONITO          \\nREMARK   1  TITL 2 (KATSUWONUS PELAMIS, LINNAEUS)                               \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  70   311 1971              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 5                                                          \\nREMARK   1  EDIT   R.J.FELDMANN                                                 \\nREMARK   1  REF    ATLAS OF MACROMOLECULAR                120 1976              \\nREMARK   1  REF  2 STRUCTURE ON MICROFICHE                                      \\nREMARK   1  PUBL   TRACOR JITCO INC.,ROCKVILLE,MD.                              \\nREMARK   1  REFN                                                                \\nREMARK   1 REFERENCE 6                                                          \\nREMARK   1  EDIT   M.O.DAYHOFF                                                  \\nREMARK   1  REF    ATLAS OF PROTEIN SEQUENCE     V.   5    39 1976              \\nREMARK   1  REF  2 AND STRUCTURE,SUPPLEMENT 2                                   \\nREMARK   1  PUBL   NATIONAL BIOMEDICAL RESEARCH FOUNDATION, SILVER SPRING,MD.   \\nREMARK   1  REFN                   ISSN 0-912466-05-7                           \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    2.30 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : NULL                                                 \\nREMARK   3   AUTHORS     : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 2.30                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : NULL                           \\nREMARK   3   DATA CUTOFF            (SIGMA(F)) : NULL                           \\nREMARK   3   DATA CUTOFF HIGH         (ABS(F)) : NULL                           \\nREMARK   3   DATA CUTOFF LOW          (ABS(F)) : NULL                           \\nREMARK   3   COMPLETENESS (WORKING+TEST)   (%) : NULL                           \\nREMARK   3   NUMBER OF REFLECTIONS             : NULL                           \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   CROSS-VALIDATION METHOD          : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SELECTION  : NULL                            \\nREMARK   3   R VALUE            (WORKING SET) : NULL                            \\nREMARK   3   FREE R VALUE                     : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : NULL                            \\nREMARK   3   FREE R VALUE TEST SET COUNT      : NULL                            \\nREMARK   3   ESTIMATED ERROR OF FREE R VALUE  : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  FIT IN THE HIGHEST RESOLUTION BIN.                                  \\nREMARK   3   TOTAL NUMBER OF BINS USED           : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE HIGH       (A) : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE LOW        (A) : NULL                         \\nREMARK   3   BIN COMPLETENESS (WORKING+TEST) (%) : NULL                         \\nREMARK   3   REFLECTIONS IN BIN    (WORKING SET) : NULL                         \\nREMARK   3   BIN R VALUE           (WORKING SET) : NULL                         \\nREMARK   3   BIN FREE R VALUE                    : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET SIZE  (%) : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET COUNT     : NULL                         \\nREMARK   3   ESTIMATED ERROR OF BIN FREE R VALUE : NULL                         \\nREMARK   3                                                                      \\nREMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \\nREMARK   3   PROTEIN ATOMS            : 1590                                    \\nREMARK   3   NUCLEIC ACID ATOMS       : 0                                       \\nREMARK   3   HETEROGEN ATOMS          : 86                                      \\nREMARK   3   SOLVENT ATOMS            : 1                                       \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : NULL                           \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : NULL                           \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : NULL                                                 \\nREMARK   3    B22 (A**2) : NULL                                                 \\nREMARK   3    B33 (A**2) : NULL                                                 \\nREMARK   3    B12 (A**2) : NULL                                                 \\nREMARK   3    B13 (A**2) : NULL                                                 \\nREMARK   3    B23 (A**2) : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  ESTIMATED COORDINATE ERROR.                                         \\nREMARK   3   ESD FROM LUZZATI PLOT        (A) : NULL                            \\nREMARK   3   ESD FROM SIGMAA              (A) : NULL                            \\nREMARK   3   LOW RESOLUTION CUTOFF        (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  CROSS-VALIDATED ESTIMATED COORDINATE ERROR.                         \\nREMARK   3   ESD FROM C-V LUZZATI PLOT    (A) : NULL                            \\nREMARK   3   ESD FROM C-V SIGMAA          (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  RMS DEVIATIONS FROM IDEAL VALUES.                                   \\nREMARK   3   BOND LENGTHS                 (A) : NULL                            \\nREMARK   3   BOND ANGLES            (DEGREES) : NULL                            \\nREMARK   3   DIHEDRAL ANGLES        (DEGREES) : NULL                            \\nREMARK   3   IMPROPER ANGLES        (DEGREES) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL MODEL : NULL                                      \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL FACTOR RESTRAINTS.    RMS    SIGMA                \\nREMARK   3   MAIN-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   MAIN-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  NCS MODEL : NULL                                                    \\nREMARK   3                                                                      \\nREMARK   3  NCS RESTRAINTS.                         RMS   SIGMA/WEIGHT          \\nREMARK   3   GROUP  1  POSITIONAL            (A) : NULL  ; NULL                 \\nREMARK   3   GROUP  1  B-FACTOR           (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  PARAMETER FILE  1  : NULL                                           \\nREMARK   3  TOPOLOGY FILE  1   : NULL                                           \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS: NULL                                      \\nREMARK   4                                                                      \\nREMARK   4 1CYC COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY BNL.                                \\nREMARK 100 THE DEPOSITION ID IS D_1000172595.                                   \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : NULL                               \\nREMARK 200  TEMPERATURE           (KELVIN) : NULL                               \\nREMARK 200  PH                             : NULL                               \\nREMARK 200  NUMBER OF CRYSTALS USED        : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : NULL                               \\nREMARK 200  RADIATION SOURCE               : NULL                               \\nREMARK 200  BEAMLINE                       : NULL                               \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : NULL                               \\nREMARK 200  WAVELENGTH OR RANGE        (A) : NULL                               \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : NULL                               \\nREMARK 200  DETECTOR MANUFACTURER          : NULL                               \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : NULL                               \\nREMARK 200  DATA SCALING SOFTWARE          : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : NULL                               \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : NULL                               \\nREMARK 200  RESOLUTION RANGE LOW       (A) : NULL                               \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY                : NULL                               \\nREMARK 200  R MERGE                    (I) : NULL                               \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : NULL                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : NULL                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : NULL                               \\nREMARK 200  R MERGE FOR SHELL          (I) : NULL                               \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: NULL                                           \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: NULL                         \\nREMARK 200 SOFTWARE USED: NULL                                                  \\nREMARK 200 STARTING MODEL: NULL                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: NULL                                                         \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): 39.18                                     \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 2.02                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: NULL                                     \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: P 21 21 21                       \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -X+1/2,-Y,Z+1/2                                         \\nREMARK 290       3555   -X,Y+1/2,-Z+1/2                                         \\nREMARK 290       4555   X+1/2,-Y+1/2,-Z                                         \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -1.000000  0.000000  0.000000       28.84000            \\nREMARK 290   SMTRY2   2  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000       18.91500            \\nREMARK 290   SMTRY1   3 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   3  0.000000  1.000000  0.000000       42.29000            \\nREMARK 290   SMTRY3   3  0.000000  0.000000 -1.000000       18.91500            \\nREMARK 290   SMTRY1   4  1.000000  0.000000  0.000000       28.84000            \\nREMARK 290   SMTRY2   4  0.000000 -1.000000  0.000000       42.29000            \\nREMARK 290   SMTRY3   4  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1, 2                                                    \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: MONOMERIC                         \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 2                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: MONOMERIC                         \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: B                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 470                                                                      \\nREMARK 470 MISSING ATOM                                                         \\nREMARK 470 THE FOLLOWING RESIDUES HAVE MISSING ATOMS (M=MODEL NUMBER;           \\nREMARK 470 RES=RESIDUE NAME; C=CHAIN IDENTIFIER; SSEQ=SEQUENCE NUMBER;          \\nREMARK 470 I=INSERTION CODE):                                                   \\nREMARK 470   M RES CSSEQI  ATOMS                                                \\nREMARK 470     ASN A  22    CB   CG   OD1  ND2                                  \\nREMARK 470     LYS A  39    CE   NZ                                             \\nREMARK 470     ASN B  22    CB   CG   OD1  ND2                                  \\nREMARK 470     LYS B  39    CE   NZ                                             \\nREMARK 620                                                                      \\nREMARK 620 METAL COORDINATION                                                   \\nREMARK 620 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 620 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE):                             \\nREMARK 620                                                                      \\nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \\nREMARK 620                             HEC A 104  FE                            \\nREMARK 620 N RES CSSEQI ATOM                                                    \\nREMARK 620 1 HIS A  18   NE2                                                    \\nREMARK 620 2 HEC A 104   NA   87.0                                              \\nREMARK 620 3 HEC A 104   NB   87.8  90.6                                        \\nREMARK 620 4 HEC A 104   NC   92.0 178.7  90.1                                  \\nREMARK 620 5 HEC A 104   ND   91.1  86.9 177.4  92.3                            \\nREMARK 620 6 MET A  80   SD  170.9 102.1  93.0  79.0  88.6                      \\nREMARK 620 N                    1     2     3     4     5                       \\nREMARK 620                                                                      \\nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \\nREMARK 620                             HEC B 104  FE                            \\nREMARK 620 N RES CSSEQI ATOM                                                    \\nREMARK 620 1 HIS B  18   NE2                                                    \\nREMARK 620 2 HEC B 104   NA   87.0                                              \\nREMARK 620 3 HEC B 104   NB   87.8  90.6                                        \\nREMARK 620 4 HEC B 104   NC   92.0 178.7  90.1                                  \\nREMARK 620 5 HEC B 104   ND   91.1  86.9 177.4  92.3                            \\nREMARK 620 6 MET B  80   SD  170.9 102.1  93.0  79.0  88.6                      \\nREMARK 620 N                    1     2     3     4     5                       \\nREMARK 800                                                                      \\nREMARK 800 SITE                                                                 \\nREMARK 800 SITE_IDENTIFIER: AC1                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE HEC A 104                 \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: AC2                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE HEC B 104                 \\nDBREF  1CYC A    1   103  UNP    P00025   CYC_KATPE        1    103             \\nDBREF  1CYC B    1   103  UNP    P00025   CYC_KATPE        1    103             \\nSEQRES   1 A  103  GLY ASP VAL ALA LYS GLY LYS LYS THR PHE VAL GLN LYS          \\nSEQRES   2 A  103  CYS ALA GLN CYS HIS THR VAL GLU ASN GLY GLY LYS HIS          \\nSEQRES   3 A  103  LYS VAL GLY PRO ASN LEU TRP GLY LEU PHE GLY ARG LYS          \\nSEQRES   4 A  103  THR GLY GLN ALA GLU GLY TYR SER TYR THR ASP ALA ASN          \\nSEQRES   5 A  103  LYS SER LYS GLY ILE VAL TRP ASN GLU ASN THR LEU MET          \\nSEQRES   6 A  103  GLU TYR LEU GLU ASN PRO LYS LYS TYR ILE PRO GLY THR          \\nSEQRES   7 A  103  LYS MET ILE PHE ALA GLY ILE LYS LYS LYS GLY GLU ARG          \\nSEQRES   8 A  103  GLN ASP LEU VAL ALA TYR LEU LYS SER ALA THR SER              \\nSEQRES   1 B  103  GLY ASP VAL ALA LYS GLY LYS LYS THR PHE VAL GLN LYS          \\nSEQRES   2 B  103  CYS ALA GLN CYS HIS THR VAL GLU ASN GLY GLY LYS HIS          \\nSEQRES   3 B  103  LYS VAL GLY PRO ASN LEU TRP GLY LEU PHE GLY ARG LYS          \\nSEQRES   4 B  103  THR GLY GLN ALA GLU GLY TYR SER TYR THR ASP ALA ASN          \\nSEQRES   5 B  103  LYS SER LYS GLY ILE VAL TRP ASN GLU ASN THR LEU MET          \\nSEQRES   6 B  103  GLU TYR LEU GLU ASN PRO LYS LYS TYR ILE PRO GLY THR          \\nSEQRES   7 B  103  LYS MET ILE PHE ALA GLY ILE LYS LYS LYS GLY GLU ARG          \\nSEQRES   8 B  103  GLN ASP LEU VAL ALA TYR LEU LYS SER ALA THR SER              \\nHET    HEC  A 104      43                                                       \\nHET    HEC  B 104      43                                                       \\nHETNAM     HEC HEME C                                                           \\nFORMUL   3  HEC    2(C34 H34 FE N4 O4)                                          \\nFORMUL   5  HOH   *(H2 O)                                                       \\nHELIX    1  H1 GLY A    1  VAL A   11  1                                  11    \\nHELIX    2  H4 CYS A   14  CYS A   17  514 AND 17 BOUND TO HEME GROUP      4    \\nHELIX    3  H5 THR A   49  LYS A   55  5LOOSE FROM 49-53,3/10 53-55        7    \\nHELIX    4  H2 ASN A   60  GLU A   69  1                                  10    \\nHELIX    5  H3 GLU A   90  SER A  103  1                                  14    \\nHELIX    6  H6 GLY B    1  VAL B   11  1                                  11    \\nHELIX    7  H7 CYS B   14  CYS B   17  514 AND 17 BOUND TO HEME GROUP      4    \\nHELIX    8  H8 THR B   49  LYS B   55  5LOOSE FROM 49-53,3/10 53-55        7    \\nHELIX    9  H9 ASN B   60  GLU B   69  1                                  10    \\nHELIX   10 H10 GLU B   90  SER B  103  1                                  14    \\nLINK         SG  CYS A  14                 CAB HEC A 104     1555   1555  1.48  \\nLINK         SG  CYS A  17                 CAC HEC A 104     1555   1555  1.86  \\nLINK         CE2 TYR A  48                 O2D HEC A 104     1555   1555  2.02  \\nLINK         SG  CYS B  14                 CAB HEC B 104     1555   1555  1.48  \\nLINK         SG  CYS B  17                 CAC HEC B 104     1555   1555  1.86  \\nLINK         CE2 TYR B  48                 O2D HEC B 104     1555   1555  2.02  \\nLINK         NE2 HIS A  18                FE   HEC A 104     1555   1555  2.60  \\nLINK         SD  MET A  80                FE   HEC A 104     1555   1555  2.49  \\nLINK         NE2 HIS B  18                FE   HEC B 104     1555   1555  2.60  \\nLINK         SD  MET B  80                FE   HEC B 104     1555   1555  2.49  \\nSITE     1 AC1 17 LYS A  13  CYS A  14  CYS A  17  HIS A  18                    \\nSITE     2 AC1 17 LEU A  32  LEU A  35  ARG A  38  THR A  40                    \\nSITE     3 AC1 17 TYR A  46  TYR A  48  ASN A  52  TRP A  59                    \\nSITE     4 AC1 17 THR A  78  LYS A  79  MET A  80  PHE A  82                    \\nSITE     5 AC1 17 HOH A 105                                                     \\nSITE     1 AC2 16 LYS B  13  CYS B  14  CYS B  17  HIS B  18                    \\nSITE     2 AC2 16 LEU B  32  LEU B  35  ARG B  38  THR B  40                    \\nSITE     3 AC2 16 TYR B  46  TYR B  48  ASN B  52  TRP B  59                    \\nSITE     4 AC2 16 THR B  78  LYS B  79  MET B  80  PHE B  82                    \\nCRYST1   57.680   84.580   37.830  90.00  90.00  90.00 P 21 21 21    8          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.017337  0.000000  0.000000        0.00000                         \\nSCALE2      0.000000  0.011823  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.026434        0.00000                         \\nMTRIX1   1  1.000000  0.000000  0.000000       28.84000    1                    \\nMTRIX2   1  0.000000  1.000000  0.000000        0.00000    1                    \\nMTRIX3   1  0.000000  0.000000  1.000000        0.00000    1                    \\nATOM      1  N   GLY A   1     -21.138  13.774  -7.711  1.00  0.00           N  \\nATOM      2  CA  GLY A   1     -19.878  14.593  -7.963  1.00  0.00           C  \\nATOM      3  C   GLY A   1     -18.996  14.530  -6.703  1.00  0.00           C  \\nATOM      4  O   GLY A   1     -18.114  13.711  -6.640  1.00  0.00           O  \\nATOM      5  N   ASP A   2     -18.996  15.412  -5.821  1.00  0.00           N  \\nATOM      6  CA  ASP A   2     -17.610  15.097  -5.191  1.00  0.00           C  \\nATOM      7  C   ASP A   2     -17.862  14.152  -3.931  1.00  0.00           C  \\nATOM      8  O   ASP A   2     -17.106  13.333  -3.553  1.00  0.00           O  \\nATOM      9  CB  ASP A   2     -17.169  16.546  -4.435  1.00  0.00           C  \\nATOM     10  CG  ASP A   2     -17.988  17.365  -3.427  1.00  0.00           C  \\nATOM     11  OD1 ASP A   2     -19.500  17.365  -3.994  1.00  0.00           O  \\nATOM     12  OD2 ASP A   2     -17.484  18.373  -2.293  1.00  0.00           O  \\nATOM     13  N   VAL A   3     -19.059  14.089  -3.679  1.00  0.00           N  \\nATOM     14  CA  VAL A   3     -19.815  12.955  -2.860  1.00  0.00           C  \\nATOM     15  C   VAL A   3     -19.374  11.506  -3.805  1.00  0.00           C  \\nATOM     16  O   VAL A   3     -18.870  10.624  -3.427  1.00  0.00           O  \\nATOM     17  CB  VAL A   3     -22.587  13.270  -3.175  1.00  0.00           C  \\nATOM     18  CG1 VAL A   3     -21.957  11.947  -3.427  1.00  0.00           C  \\nATOM     19  CG2 VAL A   3     -21.705  13.963  -2.167  1.00  0.00           C  \\nATOM     20  N   ALA A   4     -19.437  11.380  -5.002  1.00  0.00           N  \\nATOM     21  CA  ALA A   4     -18.681  10.120  -5.569  1.00  0.00           C  \\nATOM     22  C   ALA A   4     -17.358   9.805  -4.939  1.00  0.00           C  \\nATOM     23  O   ALA A   4     -17.295   8.923  -4.183  1.00  0.00           O  \\nATOM     24  CB  ALA A   4     -18.429  10.435  -7.144  1.00  0.00           C  \\nATOM     25  N   LYS A   5     -16.476  10.687  -5.191  1.00  0.00           N  \\nATOM     26  CA  LYS A   5     -15.216  10.498  -4.372  1.00  0.00           C  \\nATOM     27  C   LYS A   5     -15.279  10.246  -2.797  1.00  0.00           C  \\nATOM     28  O   LYS A   5     -14.838   9.238  -2.293  1.00  0.00           O  \\nATOM     29  CB  LYS A   5     -14.208  11.695  -4.246  1.00  0.00           C  \\nATOM     30  CG  LYS A   5     -12.948  11.758  -3.553  1.00  0.00           C  \\nATOM     31  CD  LYS A   5     -11.814  13.081  -3.679  1.00  0.00           C  \\nATOM     32  CE  LYS A   5     -10.806  13.144  -2.545  1.00  0.00           C  \\nATOM     33  NZ  LYS A   5      -9.735  14.404  -2.293  1.00  0.00           N  \\nATOM     34  N   GLY A   6     -16.161  10.750  -1.915  1.00  0.00           N  \\nATOM     35  CA  GLY A   6     -16.350  10.435  -0.403  1.00  0.00           C  \\nATOM     36  C   GLY A   6     -16.539   8.923  -0.655  1.00  0.00           C  \\nATOM     37  O   GLY A   6     -15.909   8.041  -0.025  1.00  0.00           O  \\nATOM     38  N   LYS A   7     -17.421   8.419  -1.411  1.00  0.00           N  \\nATOM     39  CA  LYS A   7     -17.799   6.844  -1.033  1.00  0.00           C  \\nATOM     40  C   LYS A   7     -16.476   6.151  -1.474  1.00  0.00           C  \\nATOM     41  O   LYS A   7     -15.972   4.954  -0.970  1.00  0.00           O  \\nATOM     42  CB  LYS A   7     -18.870   6.403  -2.293  1.00  0.00           C  \\nATOM     43  CG  LYS A   7     -18.870   5.017  -2.734  1.00  0.00           C  \\nATOM     44  CD  LYS A   7     -20.193   4.765  -3.427  1.00  0.00           C  \\nATOM     45  CE  LYS A   7     -20.193   3.190  -4.057  1.00  0.00           C  \\nATOM     46  NZ  LYS A   7     -21.516   2.623  -4.309  1.00  0.00           N  \\nATOM     47  N   LYS A   8     -15.846   6.403  -2.545  1.00  0.00           N  \\nATOM     48  CA  LYS A   8     -14.460   5.899  -2.923  1.00  0.00           C  \\nATOM     49  C   LYS A   8     -13.452   5.899  -1.663  1.00  0.00           C  \\nATOM     50  O   LYS A   8     -13.011   5.017  -1.348  1.00  0.00           O  \\nATOM     51  CB  LYS A   8     -13.704   6.844  -3.931  1.00  0.00           C  \\nATOM     52  CG  LYS A   8     -12.318   6.529  -4.183  1.00  0.00           C  \\nATOM     53  CD  LYS A   8     -11.562   7.537  -5.065  1.00  0.00           C  \\nATOM     54  CE  LYS A   8     -10.176   7.222  -5.821  1.00  0.00           C  \\nATOM     55  NZ  LYS A   8      -9.357   8.419  -6.325  1.00  0.00           N  \\nATOM     56  N   THR A   9     -13.452   6.970  -1.159  1.00  0.00           N  \\nATOM     57  CA  THR A   9     -12.633   7.033   0.227  1.00  0.00           C  \\nATOM     58  C   THR A   9     -13.515   6.277   1.298  1.00  0.00           C  \\nATOM     59  O   THR A   9     -12.948   5.773   2.369  1.00  0.00           O  \\nATOM     60  CB  THR A   9     -12.381   8.419   0.920  1.00  0.00           C  \\nATOM     61  OG1 THR A   9     -11.058   8.923   0.542  1.00  0.00           O  \\nATOM     62  CG2 THR A   9     -12.444   8.734   2.558  1.00  0.00           C  \\nATOM     63  N   PHE A  10     -14.775   6.403   1.235  1.00  0.00           N  \\nATOM     64  CA  PHE A  10     -15.531   5.647   2.117  1.00  0.00           C  \\nATOM     65  C   PHE A  10     -15.216   4.135   1.739  1.00  0.00           C  \\nATOM     66  O   PHE A  10     -14.460   3.757   2.495  1.00  0.00           O  \\nATOM     67  CB  PHE A  10     -17.106   5.647   1.991  1.00  0.00           C  \\nATOM     68  CG  PHE A  10     -17.736   4.891   3.251  1.00  0.00           C  \\nATOM     69  CD1 PHE A  10     -18.492   3.757   2.747  1.00  0.00           C  \\nATOM     70  CD2 PHE A  10     -17.862   5.143   4.133  1.00  0.00           C  \\nATOM     71  CE1 PHE A  10     -18.996   3.001   3.818  1.00  0.00           C  \\nATOM     72  CE2 PHE A  10     -18.366   4.513   4.889  1.00  0.00           C  \\nATOM     73  CZ  PHE A  10     -18.996   3.379   5.015  1.00  0.00           C  \\nATOM     74  N   VAL A  11     -15.216   3.379   0.479  1.00  0.00           N  \\nATOM     75  CA  VAL A  11     -14.964   2.056   0.605  1.00  0.00           C  \\nATOM     76  C   VAL A  11     -13.326   1.867   1.109  1.00  0.00           C  \\nATOM     77  O   VAL A  11     -12.948   0.985   1.865  1.00  0.00           O  \\nATOM     78  CB  VAL A  11     -15.153   1.237  -0.529  1.00  0.00           C  \\nATOM     79  CG1 VAL A  11     -16.098   1.930  -1.537  1.00  0.00           C  \\nATOM     80  CG2 VAL A  11     -13.956   1.111  -1.285  1.00  0.00           C  \\nATOM     81  N   GLN A  12     -12.570   2.749   0.794  1.00  0.00           N  \\nATOM     82  CA  GLN A  12     -10.995   2.749   1.235  1.00  0.00           C  \\nATOM     83  C   GLN A  12     -10.680   2.938   2.873  1.00  0.00           C  \\nATOM     84  O   GLN A  12     -10.176   1.930   3.377  1.00  0.00           O  \\nATOM     85  CB  GLN A  12     -10.428   3.820   0.479  1.00  0.00           C  \\nATOM     86  CG  GLN A  12      -9.861   3.253  -0.718  1.00  0.00           C  \\nATOM     87  CD  GLN A  12      -9.168   4.135  -2.293  1.00  0.00           C  \\nATOM     88  OE1 GLN A  12      -7.593   3.820  -2.041  1.00  0.00           O  \\nATOM     89  NE2 GLN A  12      -9.672   5.143  -2.671  1.00  0.00           N  \\nATOM     90  N   LYS A  13     -10.995   3.757   3.629  1.00  0.00           N  \\nATOM     91  CA  LYS A  13     -10.869   3.757   5.015  1.00  0.00           C  \\nATOM     92  C   LYS A  13     -12.066   3.694   5.960  1.00  0.00           C  \\nATOM     93  O   LYS A  13     -12.003   3.127   6.779  1.00  0.00           O  \\nATOM     94  CB  LYS A  13     -10.239   5.017   5.141  1.00  0.00           C  \\nATOM     95  CG  LYS A  13      -9.672   5.521   3.881  1.00  0.00           C  \\nATOM     96  CD  LYS A  13      -8.286   6.277   4.007  1.00  0.00           C  \\nATOM     97  CE  LYS A  13      -8.160   6.907   2.621  1.00  0.00           C  \\nATOM     98  NZ  LYS A  13      -6.837   7.789   2.243  1.00  0.00           N  \\nATOM     99  N   CYS A  14     -13.137   4.072   5.519  1.00  0.00           N  \\nATOM    100  CA  CYS A  14     -14.460   4.072   6.464  1.00  0.00           C  \\nATOM    101  C   CYS A  14     -15.216   2.812   5.897  1.00  0.00           C  \\nATOM    102  O   CYS A  14     -15.342   1.804   6.716  1.00  0.00           O  \\nATOM    103  CB  CYS A  14     -15.468   5.332   5.897  1.00  0.00           C  \\nATOM    104  SG  CYS A  14     -14.712   6.844   6.464  1.00  0.00           S  \\nATOM    105  N   ALA A  15     -15.720   2.623   4.637  1.00  0.00           N  \\nATOM    106  CA  ALA A  15     -16.161   1.237   4.322  1.00  0.00           C  \\nATOM    107  C   ALA A  15     -16.035  -0.086   5.015  1.00  0.00           C  \\nATOM    108  O   ALA A  15     -17.043  -0.590   5.393  1.00  0.00           O  \\nATOM    109  CB  ALA A  15     -15.405   0.985   3.062  1.00  0.00           C  \\nATOM    110  N   GLN A  16     -14.964  -0.653   4.889  1.00  0.00           N  \\nATOM    111  CA  GLN A  16     -14.901  -2.039   5.897  1.00  0.00           C  \\nATOM    112  C   GLN A  16     -15.090  -1.787   7.283  1.00  0.00           C  \\nATOM    113  O   GLN A  16     -14.838  -2.669   8.291  1.00  0.00           O  \\nATOM    114  CB  GLN A  16     -13.578  -2.039   5.519  1.00  0.00           C  \\nATOM    115  CG  GLN A  16     -12.318  -1.724   6.275  1.00  0.00           C  \\nATOM    116  CD  GLN A  16     -11.625  -0.842   5.015  1.00  0.00           C  \\nATOM    117  OE1 GLN A  16     -10.365  -0.149   5.330  1.00  0.00           O  \\nATOM    118  NE2 GLN A  16     -12.444  -0.464   4.385  1.00  0.00           N  \\nATOM    119  N   CYS A  17     -15.216  -0.653   7.787  1.00  0.00           N  \\nATOM    120  CA  CYS A  17     -15.405  -0.905   9.299  1.00  0.00           C  \\nATOM    121  C   CYS A  17     -16.917  -0.401   9.173  1.00  0.00           C  \\nATOM    122  O   CYS A  17     -17.736  -1.283   9.173  1.00  0.00           O  \\nATOM    123  CB  CYS A  17     -14.460   0.355  10.055  1.00  0.00           C  \\nATOM    124  SG  CYS A  17     -13.326  -0.338  11.441  1.00  0.00           S  \\nATOM    125  N   HIS A  18     -17.232   0.733   9.551  1.00  0.00           N  \\nATOM    126  CA  HIS A  18     -18.870   0.985   9.803  1.00  0.00           C  \\nATOM    127  C   HIS A  18     -19.878   0.229   8.795  1.00  0.00           C  \\nATOM    128  O   HIS A  18     -19.626  -0.023   7.472  1.00  0.00           O  \\nATOM    129  CB  HIS A  18     -19.185   2.245   9.173  1.00  0.00           C  \\nATOM    130  CG  HIS A  18     -18.681   3.127  10.181  1.00  0.00           C  \\nATOM    131  ND1 HIS A  18     -18.618   3.253  11.567  1.00  0.00           N  \\nATOM    132  CD2 HIS A  18     -17.736   4.072  10.559  1.00  0.00           C  \\nATOM    133  CE1 HIS A  18     -18.051   3.694  12.827  1.00  0.00           C  \\nATOM    134  NE2 HIS A  18     -17.232   4.387  11.819  1.00  0.00           N  \\nATOM    135  N   THR A  19     -21.012   0.103   9.299  1.00  0.00           N  \\nATOM    136  CA  THR A  19     -22.272  -0.401   8.417  1.00  0.00           C  \\nATOM    137  C   THR A  19     -22.902   0.922   7.661  1.00  0.00           C  \\nATOM    138  O   THR A  19     -22.272   1.867   7.157  1.00  0.00           O  \\nATOM    139  CB  THR A  19     -23.280  -1.283   8.795  1.00  0.00           C  \\nATOM    140  OG1 THR A  19     -23.532  -0.905   9.740  1.00  0.00           O  \\nATOM    141  CG2 THR A  19     -22.776  -2.669   8.543  1.00  0.00           C  \\nATOM    142  N   VAL A  20     -24.036   1.111   7.472  1.00  0.00           N  \\nATOM    143  CA  VAL A  20     -24.414   2.371   6.779  1.00  0.00           C  \\nATOM    144  C   VAL A  20     -26.052   2.434   6.401  1.00  0.00           C  \\nATOM    145  O   VAL A  20     -26.556   3.442   5.771  1.00  0.00           O  \\nATOM    146  CB  VAL A  20     -23.784   2.560   5.519  1.00  0.00           C  \\nATOM    147  CG1 VAL A  20     -24.540   2.119   4.637  1.00  0.00           C  \\nATOM    148  CG2 VAL A  20     -23.658   4.009   5.267  1.00  0.00           C  \\nATOM    149  N   GLU A  21     -26.682   1.363   6.527  1.00  0.00           N  \\nATOM    150  CA  GLU A  21     -28.194   1.363   5.897  1.00  0.00           C  \\nATOM    151  C   GLU A  21     -29.013   1.741   7.283  1.00  0.00           C  \\nATOM    152  O   GLU A  21     -28.635   2.623   7.913  1.00  0.00           O  \\nATOM    153  CB  GLU A  21     -28.572  -0.086   5.645  1.00  0.00           C  \\nATOM    154  CG  GLU A  21     -28.698  -0.779   4.133  1.00  0.00           C  \\nATOM    155  CD  GLU A  21     -27.690  -1.913   4.007  1.00  0.00           C  \\nATOM    156  OE1 GLU A  21     -26.682  -2.039   5.141  1.00  0.00           O  \\nATOM    157  OE2 GLU A  21     -27.753  -2.921   2.873  1.00  0.00           O  \\nATOM    158  N   ASN A  22     -30.210   1.363   7.283  1.00  0.00           N  \\nATOM    159  CA  ASN A  22     -30.966   1.867   8.417  1.00  0.00           C  \\nATOM    160  C   ASN A  22     -30.714   1.237   9.677  1.00  0.00           C  \\nATOM    161  O   ASN A  22     -31.218   0.103  10.055  1.00  0.00           O  \\nATOM    162  N   GLY A  23     -29.958   1.552  10.433  1.00  0.00           N  \\nATOM    163  CA  GLY A  23     -29.517   0.859  11.819  1.00  0.00           C  \\nATOM    164  C   GLY A  23     -28.572  -0.338  11.252  1.00  0.00           C  \\nATOM    165  O   GLY A  23     -28.194  -1.283  12.071  1.00  0.00           O  \\nATOM    166  N   GLY A  24     -28.068  -0.149   9.929  1.00  0.00           N  \\nATOM    167  CA  GLY A  24     -27.060  -1.409   9.047  1.00  0.00           C  \\nATOM    168  C   GLY A  24     -26.871  -2.480  10.118  1.00  0.00           C  \\nATOM    169  O   GLY A  24     -27.186  -3.803   9.929  1.00  0.00           O  \\nATOM    170  N   LYS A  25     -26.304  -2.165  11.441  1.00  0.00           N  \\nATOM    171  CA  LYS A  25     -26.241  -3.047  12.701  1.00  0.00           C  \\nATOM    172  C   LYS A  25     -24.792  -2.795  13.079  1.00  0.00           C  \\nATOM    173  O   LYS A  25     -23.910  -2.417  12.323  1.00  0.00           O  \\nATOM    174  CB  LYS A  25     -26.241  -4.433  11.819  1.00  0.00           C  \\nATOM    175  CG  LYS A  25     -25.296  -4.496  10.433  1.00  0.00           C  \\nATOM    176  CD  LYS A  25     -24.918  -5.819   9.803  1.00  0.00           C  \\nATOM    177  CE  LYS A  25     -23.784  -5.819   8.543  1.00  0.00           C  \\nATOM    178  NZ  LYS A  25     -23.658  -6.827   7.661  1.00  0.00           N  \\nATOM    179  N   HIS A  26     -24.414  -2.921  14.465  1.00  0.00           N  \\nATOM    180  CA  HIS A  26     -23.028  -2.417  14.654  1.00  0.00           C  \\nATOM    181  C   HIS A  26     -21.894  -3.425  14.087  1.00  0.00           C  \\nATOM    182  O   HIS A  26     -22.083  -3.803  12.827  1.00  0.00           O  \\nATOM    183  CB  HIS A  26     -23.028  -2.543  16.103  1.00  0.00           C  \\nATOM    184  CG  HIS A  26     -24.036  -1.094  16.355  1.00  0.00           C  \\nATOM    185  ND1 HIS A  26     -24.540  -0.338  15.473  1.00  0.00           N  \\nATOM    186  CD2 HIS A  26     -24.666  -0.905  17.363  1.00  0.00           C  \\nATOM    187  CE1 HIS A  26     -25.422   0.355  16.103  1.00  0.00           C  \\nATOM    188  NE2 HIS A  26     -25.422   0.103  17.237  1.00  0.00           N  \\nATOM    189  N   LYS A  27     -20.886  -3.551  14.717  1.00  0.00           N  \\nATOM    190  CA  LYS A  27     -19.752  -4.433  14.213  1.00  0.00           C  \\nATOM    191  C   LYS A  27     -18.366  -3.677  14.276  1.00  0.00           C  \\nATOM    192  O   LYS A  27     -17.484  -3.677  13.331  1.00  0.00           O  \\nATOM    193  CB  LYS A  27     -20.004  -4.559  12.575  1.00  0.00           C  \\nATOM    194  CG  LYS A  27     -20.382  -3.425  11.882  1.00  0.00           C  \\nATOM    195  CD  LYS A  27     -19.500  -3.047  11.063  1.00  0.00           C  \\nATOM    196  CE  LYS A  27     -19.752  -3.677   9.551  1.00  0.00           C  \\nATOM    197  NZ  LYS A  27     -18.618  -3.677   8.858  1.00  0.00           N  \\nATOM    198  N   VAL A  28     -18.177  -3.047  15.410  1.00  0.00           N  \\nATOM    199  CA  VAL A  28     -16.854  -2.228  15.410  1.00  0.00           C  \\nATOM    200  C   VAL A  28     -17.358  -0.905  15.473  1.00  0.00           C  \\nATOM    201  O   VAL A  28     -17.610  -0.275  16.481  1.00  0.00           O  \\nATOM    202  CB  VAL A  28     -15.846  -2.291  14.780  1.00  0.00           C  \\nATOM    203  CG1 VAL A  28     -15.594  -1.094  14.087  1.00  0.00           C  \\nATOM    204  CG2 VAL A  28     -14.586  -2.858  15.095  1.00  0.00           C  \\nATOM    205  N   GLY A  29     -17.610  -0.023  14.591  1.00  0.00           N  \\nATOM    206  CA  GLY A  29     -18.240   1.111  14.843  1.00  0.00           C  \\nATOM    207  C   GLY A  29     -19.815   1.048  14.843  1.00  0.00           C  \\nATOM    208  O   GLY A  29     -20.193   0.103  13.961  1.00  0.00           O  \\nATOM    209  N   PRO A  30     -20.634   1.615  15.977  1.00  0.00           N  \\nATOM    210  CA  PRO A  30     -22.020   1.489  15.725  1.00  0.00           C  \\nATOM    211  C   PRO A  30     -22.398   1.804  14.339  1.00  0.00           C  \\nATOM    212  O   PRO A  30     -22.020   2.371  13.457  1.00  0.00           O  \\nATOM    213  CB  PRO A  30     -22.524   2.371  16.481  1.00  0.00           C  \\nATOM    214  CG  PRO A  30     -21.642   1.993  16.859  1.00  0.00           C  \\nATOM    215  CD  PRO A  30     -20.508   2.497  16.607  1.00  0.00           C  \\nATOM    216  N   ASN A  31     -23.910   1.552  13.961  1.00  0.00           N  \\nATOM    217  CA  ASN A  31     -24.414   2.119  12.701  1.00  0.00           C  \\nATOM    218  C   ASN A  31     -24.036   3.631  12.764  1.00  0.00           C  \\nATOM    219  O   ASN A  31     -24.036   4.387  13.835  1.00  0.00           O  \\nATOM    220  CB  ASN A  31     -25.926   1.993  12.701  1.00  0.00           C  \\nATOM    221  CG  ASN A  31     -26.052   2.056  11.063  1.00  0.00           C  \\nATOM    222  OD1 ASN A  31     -27.186   2.623  10.307  1.00  0.00           O  \\nATOM    223  ND2 ASN A  31     -25.359   1.363  10.181  1.00  0.00           N  \\nATOM    224  N   LEU A  32     -23.658   4.261  11.819  1.00  0.00           N  \\nATOM    225  CA  LEU A  32     -23.280   5.899  11.819  1.00  0.00           C  \\nATOM    226  C   LEU A  32     -24.666   6.403  11.315  1.00  0.00           C  \\nATOM    227  O   LEU A  32     -24.666   6.655   9.929  1.00  0.00           O  \\nATOM    228  CB  LEU A  32     -22.146   6.466  11.315  1.00  0.00           C  \\nATOM    229  CG  LEU A  32     -20.886   6.529  12.323  1.00  0.00           C  \\nATOM    230  CD1 LEU A  32     -20.382   7.978  12.323  1.00  0.00           C  \\nATOM    231  CD2 LEU A  32     -21.327   6.277  13.457  1.00  0.00           C  \\nATOM    232  N   TRP A  33     -25.611   6.151  11.693  1.00  0.00           N  \\nATOM    233  CA  TRP A  33     -27.186   6.529  11.189  1.00  0.00           C  \\nATOM    234  C   TRP A  33     -27.312   7.915  10.433  1.00  0.00           C  \\nATOM    235  O   TRP A  33     -27.312   7.852   9.299  1.00  0.00           O  \\nATOM    236  CB  TRP A  33     -28.446   6.403  12.071  1.00  0.00           C  \\nATOM    237  CG  TRP A  33     -28.572   5.773  13.205  1.00  0.00           C  \\nATOM    238  CD1 TRP A  33     -27.564   4.891  13.583  1.00  0.00           C  \\nATOM    239  CD2 TRP A  33     -29.328   5.647  14.213  1.00  0.00           C  \\nATOM    240  NE1 TRP A  33     -27.816   4.387  14.843  1.00  0.00           N  \\nATOM    241  CE2 TRP A  33     -29.013   4.828  15.158  1.00  0.00           C  \\nATOM    242  CE3 TRP A  33     -30.462   6.277  14.213  1.00  0.00           C  \\nATOM    243  CZ2 TRP A  33     -29.706   4.639  15.851  1.00  0.00           C  \\nATOM    244  CZ3 TRP A  33     -31.092   5.962  15.284  1.00  0.00           C  \\nATOM    245  CH2 TRP A  33     -30.714   5.143  15.914  1.00  0.00           C  \\nATOM    246  N   GLY A  34     -27.564   8.797  11.315  1.00  0.00           N  \\nATOM    247  CA  GLY A  34     -27.690  10.120  10.685  1.00  0.00           C  \\nATOM    248  C   GLY A  34     -26.934  11.191  11.315  1.00  0.00           C  \\nATOM    249  O   GLY A  34     -27.312  12.199  12.134  1.00  0.00           O  \\nATOM    250  N   LEU A  35     -25.548  11.191  11.252  1.00  0.00           N  \\nATOM    251  CA  LEU A  35     -24.666  12.199  11.945  1.00  0.00           C  \\nATOM    252  C   LEU A  35     -25.548  13.459  12.386  1.00  0.00           C  \\nATOM    253  O   LEU A  35     -25.800  13.963  13.457  1.00  0.00           O  \\nATOM    254  CB  LEU A  35     -23.154  12.640  11.567  1.00  0.00           C  \\nATOM    255  CG  LEU A  35     -22.335  11.506  11.189  1.00  0.00           C  \\nATOM    256  CD1 LEU A  35     -20.886  11.695  11.189  1.00  0.00           C  \\nATOM    257  CD2 LEU A  35     -22.650  10.183  11.819  1.00  0.00           C  \\nATOM    258  N   PHE A  36     -26.052  13.963  11.441  1.00  0.00           N  \\nATOM    259  CA  PHE A  36     -26.934  15.223  11.693  1.00  0.00           C  \\nATOM    260  C   PHE A  36     -28.131  15.223  12.701  1.00  0.00           C  \\nATOM    261  O   PHE A  36     -29.328  15.223  12.449  1.00  0.00           O  \\nATOM    262  CB  PHE A  36     -27.186  15.601  10.055  1.00  0.00           C  \\nATOM    263  CG  PHE A  36     -25.926  15.601   9.425  1.00  0.00           C  \\nATOM    264  CD1 PHE A  36     -25.674  15.727   8.102  1.00  0.00           C  \\nATOM    265  CD2 PHE A  36     -24.792  15.097   9.803  1.00  0.00           C  \\nATOM    266  CE1 PHE A  36     -24.540  15.475   7.409  1.00  0.00           C  \\nATOM    267  CE2 PHE A  36     -23.658  15.097   9.551  1.00  0.00           C  \\nATOM    268  CZ  PHE A  36     -23.406  15.097   8.291  1.00  0.00           C  \\nATOM    269  N   GLY A  37     -27.879  15.538  13.961  1.00  0.00           N  \\nATOM    270  CA  GLY A  37     -28.509  15.664  15.347  1.00  0.00           C  \\nATOM    271  C   GLY A  37     -28.005  14.971  16.355  1.00  0.00           C  \\nATOM    272  O   GLY A  37     -28.572  14.215  17.300  1.00  0.00           O  \\nATOM    273  N   ARG A  38     -26.682  15.097  16.859  1.00  0.00           N  \\nATOM    274  CA  ARG A  38     -26.556  14.341  18.119  1.00  0.00           C  \\nATOM    275  C   ARG A  38     -25.170  14.845  18.371  1.00  0.00           C  \\nATOM    276  O   ARG A  38     -24.414  15.349  17.741  1.00  0.00           O  \\nATOM    277  CB  ARG A  38     -26.178  13.081  17.741  1.00  0.00           C  \\nATOM    278  CG  ARG A  38     -25.863  12.703  16.355  1.00  0.00           C  \\nATOM    279  CD  ARG A  38     -25.422  11.191  16.040  1.00  0.00           C  \\nATOM    280  NE  ARG A  38     -24.666  10.561  16.985  1.00  0.00           N  \\nATOM    281  CZ  ARG A  38     -24.792   9.238  16.859  1.00  0.00           C  \\nATOM    282  NH1 ARG A  38     -24.351   8.545  17.489  1.00  0.00           N  \\nATOM    283  NH2 ARG A  38     -25.485   8.797  15.851  1.00  0.00           N  \\nATOM    284  N   LYS A  39     -25.044  14.593  19.757  1.00  0.00           N  \\nATOM    285  CA  LYS A  39     -24.036  15.160  20.639  1.00  0.00           C  \\nATOM    286  C   LYS A  39     -23.028  14.089  20.891  1.00  0.00           C  \\nATOM    287  O   LYS A  39     -23.406  13.207  20.009  1.00  0.00           O  \\nATOM    288  CB  LYS A  39     -24.540  15.601  22.151  1.00  0.00           C  \\nATOM    289  CG  LYS A  39     -24.414  15.475  23.663  1.00  0.00           C  \\nATOM    290  CD  LYS A  39     -25.170  16.231  24.923  1.00  0.00           C  \\nATOM    291  N   THR A  40     -22.020  13.837  21.521  1.00  0.00           N  \\nATOM    292  CA  THR A  40     -21.390  12.388  21.269  1.00  0.00           C  \\nATOM    293  C   THR A  40     -21.642  11.191  22.277  1.00  0.00           C  \\nATOM    294  O   THR A  40     -21.516  11.695  23.411  1.00  0.00           O  \\nATOM    295  CB  THR A  40     -19.752  12.577  21.269  1.00  0.00           C  \\nATOM    296  OG1 THR A  40     -19.374  11.884  19.883  1.00  0.00           O  \\nATOM    297  CG2 THR A  40     -19.626  13.837  20.765  1.00  0.00           C  \\nATOM    298  N   GLY A  41     -22.020  10.309  21.773  1.00  0.00           N  \\nATOM    299  CA  GLY A  41     -22.650   9.553  22.907  1.00  0.00           C  \\nATOM    300  C   GLY A  41     -23.910   8.797  22.403  1.00  0.00           C  \\nATOM    301  O   GLY A  41     -24.162   8.545  21.395  1.00  0.00           O  \\nATOM    302  N   GLN A  42     -24.918   8.419  23.033  1.00  0.00           N  \\nATOM    303  CA  GLN A  42     -26.241   7.915  23.033  1.00  0.00           C  \\nATOM    304  C   GLN A  42     -26.304   7.537  21.647  1.00  0.00           C  \\nATOM    305  O   GLN A  42     -26.556   8.545  20.765  1.00  0.00           O  \\nATOM    306  CB  GLN A  42     -27.438   8.545  23.537  1.00  0.00           C  \\nATOM    307  CG  GLN A  42     -28.068   7.789  24.419  1.00  0.00           C  \\nATOM    308  CD  GLN A  42     -29.076   7.096  24.230  1.00  0.00           C  \\nATOM    309  OE1 GLN A  42     -29.706   5.899  24.797  1.00  0.00           O  \\nATOM    310  NE2 GLN A  42     -29.706   7.411  22.781  1.00  0.00           N  \\nATOM    311  N   ALA A  43     -26.304   6.277  21.143  1.00  0.00           N  \\nATOM    312  CA  ALA A  43     -26.682   6.025  19.505  1.00  0.00           C  \\nATOM    313  C   ALA A  43     -27.186   4.639  19.442  1.00  0.00           C  \\nATOM    314  O   ALA A  43     -27.060   4.009  18.497  1.00  0.00           O  \\nATOM    315  CB  ALA A  43     -25.548   6.025  18.497  1.00  0.00           C  \\nATOM    316  N   GLU A  44     -27.438   3.883  20.450  1.00  0.00           N  \\nATOM    317  CA  GLU A  44     -28.131   2.497  20.261  1.00  0.00           C  \\nATOM    318  C   GLU A  44     -27.060   1.615  20.891  1.00  0.00           C  \\nATOM    319  O   GLU A  44     -26.556   0.481  20.324  1.00  0.00           O  \\nATOM    320  CB  GLU A  44     -28.635   2.371  18.749  1.00  0.00           C  \\nATOM    321  CG  GLU A  44     -30.210   0.993  18.749  1.00  0.00           C  \\nATOM    322  CD  GLU A  44     -30.840   2.875  19.379  1.00  0.00           C  \\nATOM    323  OE1 GLU A  44     -32.100   2.875  18.875  1.00  0.00           O  \\nATOM    324  OE2 GLU A  44     -30.084   4.135  19.505  1.00  0.00           O  \\nATOM    325  N   GLY A  45     -26.808   1.615  22.277  1.00  0.00           N  \\nATOM    326  CA  GLY A  45     -25.674   0.859  22.781  1.00  0.00           C  \\nATOM    327  C   GLY A  45     -24.162   1.363  22.907  1.00  0.00           C  \\nATOM    328  O   GLY A  45     -23.532   1.363  21.521  1.00  0.00           O  \\nATOM    329  N   TYR A  46     -23.532   1.993  23.789  1.00  0.00           N  \\nATOM    330  CA  TYR A  46     -22.146   2.875  23.537  1.00  0.00           C  \\nATOM    331  C   TYR A  46     -21.327   1.993  24.356  1.00  0.00           C  \\nATOM    332  O   TYR A  46     -21.327   0.733  24.167  1.00  0.00           O  \\nATOM    333  CB  TYR A  46     -22.524   4.198  23.159  1.00  0.00           C  \\nATOM    334  CG  TYR A  46     -22.146   5.143  22.151  1.00  0.00           C  \\nATOM    335  CD1 TYR A  46     -22.776   5.269  21.521  1.00  0.00           C  \\nATOM    336  CD2 TYR A  46     -20.886   5.647  21.647  1.00  0.00           C  \\nATOM    337  CE1 TYR A  46     -22.524   5.899  20.387  1.00  0.00           C  \\nATOM    338  CE2 TYR A  46     -20.382   6.277  20.513  1.00  0.00           C  \\nATOM    339  CZ  TYR A  46     -21.264   6.277  20.009  1.00  0.00           C  \\nATOM    340  OH  TYR A  46     -21.138   6.655  19.001  1.00  0.00           O  \\nATOM    341  N   SER A  47     -20.634   2.560  25.049  1.00  0.00           N  \\nATOM    342  CA  SER A  47     -19.374   1.741  25.679  1.00  0.00           C  \\nATOM    343  C   SER A  47     -18.114   2.623  25.679  1.00  0.00           C  \\nATOM    344  O   SER A  47     -18.114   3.379  24.797  1.00  0.00           O  \\nATOM    345  CB  SER A  47     -19.248   0.355  25.427  1.00  0.00           C  \\nATOM    346  OG  SER A  47     -18.870   0.481  23.789  1.00  0.00           O  \\nATOM    347  N   TYR A  48     -17.484   2.749  26.939  1.00  0.00           N  \\nATOM    348  CA  TYR A  48     -16.413   3.757  26.939  1.00  0.00           C  \\nATOM    349  C   TYR A  48     -16.602   4.891  27.695  1.00  0.00           C  \\nATOM    350  O   TYR A  48     -17.610   5.269  27.821  1.00  0.00           O  \\nATOM    351  CB  TYR A  48     -16.224   3.757  25.301  1.00  0.00           C  \\nATOM    352  CG  TYR A  48     -15.216   4.639  24.545  1.00  0.00           C  \\nATOM    353  CD1 TYR A  48     -14.082   5.143  24.797  1.00  0.00           C  \\nATOM    354  CD2 TYR A  48     -15.342   4.765  23.411  1.00  0.00           C  \\nATOM    355  CE1 TYR A  48     -13.326   6.025  24.167  1.00  0.00           C  \\nATOM    356  CE2 TYR A  48     -14.712   5.521  22.529  1.00  0.00           C  \\nATOM    357  CZ  TYR A  48     -13.704   6.088  23.033  1.00  0.00           C  \\nATOM    358  OH  TYR A  48     -13.074   6.907  22.151  1.00  0.00           O  \\nATOM    359  N   THR A  49     -15.594   5.647  28.199  1.00  0.00           N  \\nATOM    360  CA  THR A  49     -15.846   6.907  29.207  1.00  0.00           C  \\nATOM    361  C   THR A  49     -15.594   8.041  28.829  1.00  0.00           C  \\nATOM    362  O   THR A  49     -16.350   8.671  27.947  1.00  0.00           O  \\nATOM    363  CB  THR A  49     -15.531   6.403  30.845  1.00  0.00           C  \\nATOM    364  OG1 THR A  49     -14.208   5.899  30.845  1.00  0.00           O  \\nATOM    365  CG2 THR A  49     -16.224   5.395  31.223  1.00  0.00           C  \\nATOM    366  N   ASP A  50     -14.334   8.482  28.703  1.00  0.00           N  \\nATOM    367  CA  ASP A  50     -14.082   9.805  28.199  1.00  0.00           C  \\nATOM    368  C   ASP A  50     -13.704   9.679  26.813  1.00  0.00           C  \\nATOM    369  O   ASP A  50     -13.074   8.545  26.183  1.00  0.00           O  \\nATOM    370  CB  ASP A  50     -13.074  10.435  28.829  1.00  0.00           C  \\nATOM    371  CG  ASP A  50     -12.948  10.183  30.089  1.00  0.00           C  \\nATOM    372  OD1 ASP A  50     -12.066   9.049  30.593  1.00  0.00           O  \\nATOM    373  OD2 ASP A  50     -13.956  10.309  30.971  1.00  0.00           O  \\nATOM    374  N   ALA A  51     -13.830  10.309  25.805  1.00  0.00           N  \\nATOM    375  CA  ALA A  51     -13.704   9.931  24.419  1.00  0.00           C  \\nATOM    376  C   ALA A  51     -15.216   9.805  23.915  1.00  0.00           C  \\nATOM    377  O   ALA A  51     -16.098   9.301  24.167  1.00  0.00           O  \\nATOM    378  CB  ALA A  51     -13.200   8.671  24.041  1.00  0.00           C  \\nATOM    379  N   ASN A  52     -15.342  10.561  22.907  1.00  0.00           N  \\nATOM    380  CA  ASN A  52     -16.539  10.561  22.277  1.00  0.00           C  \\nATOM    381  C   ASN A  52     -17.232  11.380  22.907  1.00  0.00           C  \\nATOM    382  O   ASN A  52     -17.232  12.766  22.781  1.00  0.00           O  \\nATOM    383  CB  ASN A  52     -17.610   9.553  21.899  1.00  0.00           C  \\nATOM    384  CG  ASN A  52     -17.484   8.671  20.639  1.00  0.00           C  \\nATOM    385  OD1 ASN A  52     -16.980   7.789  20.135  1.00  0.00           O  \\nATOM    386  ND2 ASN A  52     -18.240   9.427  19.631  1.00  0.00           N  \\nATOM    387  N   LYS A  53     -17.862  10.939  24.041  1.00  0.00           N  \\nATOM    388  CA  LYS A  53     -18.240  11.947  25.301  1.00  0.00           C  \\nATOM    389  C   LYS A  53     -17.106  12.703  25.805  1.00  0.00           C  \\nATOM    390  O   LYS A  53     -16.476  13.711  25.175  1.00  0.00           O  \\nATOM    391  CB  LYS A  53     -18.870  11.317  26.561  1.00  0.00           C  \\nATOM    392  CG  LYS A  53     -20.130  10.435  26.057  1.00  0.00           C  \\nATOM    393  CD  LYS A  53     -20.634   9.301  27.065  1.00  0.00           C  \\nATOM    394  CE  LYS A  53     -22.146   8.671  26.372  1.00  0.00           C  \\nATOM    395  NZ  LYS A  53     -22.524   7.663  27.191  1.00  0.00           N  \\nATOM    396  N   SER A  54     -16.602  12.325  27.065  1.00  0.00           N  \\nATOM    397  CA  SER A  54     -15.594  13.081  27.821  1.00  0.00           C  \\nATOM    398  C   SER A  54     -15.594  14.593  27.443  1.00  0.00           C  \\nATOM    399  O   SER A  54     -16.476  15.223  27.695  1.00  0.00           O  \\nATOM    400  CB  SER A  54     -14.460  12.703  27.569  1.00  0.00           C  \\nATOM    401  OG  SER A  54     -13.452  12.955  28.703  1.00  0.00           O  \\nATOM    402  N   LYS A  55     -14.838  15.223  26.813  1.00  0.00           N  \\nATOM    403  CA  LYS A  55     -14.964  16.420  26.057  1.00  0.00           C  \\nATOM    404  C   LYS A  55     -16.602  16.987  25.931  1.00  0.00           C  \\nATOM    405  O   LYS A  55     -16.854  18.121  26.183  1.00  0.00           O  \\nATOM    406  CB  LYS A  55     -14.208  16.609  24.734  1.00  0.00           C  \\nATOM    407  CG  LYS A  55     -14.712  17.743  23.600  1.00  0.00           C  \\nATOM    408  CD  LYS A  55     -13.452  17.743  22.844  1.00  0.00           C  \\nATOM    409  CE  LYS A  55     -13.578  17.995  21.143  1.00  0.00           C  \\nATOM    410  NZ  LYS A  55     -12.444  17.743  20.135  1.00  0.00           N  \\nATOM    411  N   GLY A  56     -17.232  16.042  25.679  1.00  0.00           N  \\nATOM    412  CA  GLY A  56     -18.870  16.042  25.301  1.00  0.00           C  \\nATOM    413  C   GLY A  56     -19.122  17.239  24.293  1.00  0.00           C  \\nATOM    414  O   GLY A  56     -18.492  18.247  23.852  1.00  0.00           O  \\nATOM    415  N   ILE A  57     -20.130  17.113  23.285  1.00  0.00           N  \\nATOM    416  CA  ILE A  57     -20.004  17.869  22.025  1.00  0.00           C  \\nATOM    417  C   ILE A  57     -21.264  18.121  21.143  1.00  0.00           C  \\nATOM    418  O   ILE A  57     -22.398  18.499  21.395  1.00  0.00           O  \\nATOM    419  CB  ILE A  57     -18.996  17.491  21.395  1.00  0.00           C  \\nATOM    420  CG1 ILE A  57     -19.374  16.357  20.513  1.00  0.00           C  \\nATOM    421  CG2 ILE A  57     -17.862  17.050  22.025  1.00  0.00           C  \\nATOM    422  CD1 ILE A  57     -19.122  16.609  19.127  1.00  0.00           C  \\nATOM    423  N   VAL A  58     -21.264  17.743  19.631  1.00  0.00           N  \\nATOM    424  CA  VAL A  58     -22.524  17.995  19.001  1.00  0.00           C  \\nATOM    425  C   VAL A  58     -22.020  17.239  17.615  1.00  0.00           C  \\nATOM    426  O   VAL A  58     -21.642  18.121  16.985  1.00  0.00           O  \\nATOM    427  CB  VAL A  58     -23.280  19.129  19.127  1.00  0.00           C  \\nATOM    428  CG1 VAL A  58     -22.398  20.263  18.749  1.00  0.00           C  \\nATOM    429  CG2 VAL A  58     -24.540  19.192  18.371  1.00  0.00           C  \\nATOM    430  N   TRP A  59     -22.398  16.231  16.859  1.00  0.00           N  \\nATOM    431  CA  TRP A  59     -22.020  16.105  15.221  1.00  0.00           C  \\nATOM    432  C   TRP A  59     -22.524  17.239  14.213  1.00  0.00           C  \\nATOM    433  O   TRP A  59     -23.784  17.365  14.339  1.00  0.00           O  \\nATOM    434  CB  TRP A  59     -22.146  14.845  14.717  1.00  0.00           C  \\nATOM    435  CG  TRP A  59     -21.264  13.963  15.347  1.00  0.00           C  \\nATOM    436  CD1 TRP A  59     -21.390  12.703  15.851  1.00  0.00           C  \\nATOM    437  CD2 TRP A  59     -19.878  14.152  15.536  1.00  0.00           C  \\nATOM    438  NE1 TRP A  59     -20.382  12.451  16.607  1.00  0.00           N  \\nATOM    439  CE2 TRP A  59     -19.500  13.207  15.977  1.00  0.00           C  \\nATOM    440  CE3 TRP A  59     -19.248  15.160  15.095  1.00  0.00           C  \\nATOM    441  CZ2 TRP A  59     -18.114  13.459  16.607  1.00  0.00           C  \\nATOM    442  CZ3 TRP A  59     -18.051  15.223  15.473  1.00  0.00           C  \\nATOM    443  CH2 TRP A  59     -17.484  14.467  16.229  1.00  0.00           C  \\nATOM    444  N   ASN A  60     -21.768  17.869  13.772  1.00  0.00           N  \\nATOM    445  CA  ASN A  60     -22.398  19.129  12.953  1.00  0.00           C  \\nATOM    446  C   ASN A  60     -21.642  19.255  11.441  1.00  0.00           C  \\nATOM    447  O   ASN A  60     -20.634  18.751  11.189  1.00  0.00           O  \\nATOM    448  CB  ASN A  60     -22.524  20.515  13.583  1.00  0.00           C  \\nATOM    449  CG  ASN A  60     -22.776  21.649  12.701  1.00  0.00           C  \\nATOM    450  OD1 ASN A  60     -24.036  21.712  11.945  1.00  0.00           O  \\nATOM    451  ND2 ASN A  60     -21.894  22.783  12.701  1.00  0.00           N  \\nATOM    452  N   GLU A  61     -22.020  20.137  10.811  1.00  0.00           N  \\nATOM    453  CA  GLU A  61     -21.138  20.263   9.425  1.00  0.00           C  \\nATOM    454  C   GLU A  61     -19.626  20.389  10.181  1.00  0.00           C  \\nATOM    455  O   GLU A  61     -18.744  19.507   9.803  1.00  0.00           O  \\nATOM    456  CB  GLU A  61     -21.516  21.523   8.669  1.00  0.00           C  \\nATOM    457  CG  GLU A  61     -22.020  21.523   7.409  1.00  0.00           C  \\nATOM    458  CD  GLU A  61     -23.154  20.767   7.283  1.00  0.00           C  \\nATOM    459  OE1 GLU A  61     -24.540  20.767   7.535  1.00  0.00           O  \\nATOM    460  OE2 GLU A  61     -22.902  19.381   6.653  1.00  0.00           O  \\nATOM    461  N   ASN A  62     -19.248  21.397  10.685  1.00  0.00           N  \\nATOM    462  CA  ASN A  62     -17.736  21.586  11.441  1.00  0.00           C  \\nATOM    463  C   ASN A  62     -17.106  20.137  11.945  1.00  0.00           C  \\nATOM    464  O   ASN A  62     -15.972  20.515  12.071  1.00  0.00           O  \\nATOM    465  CB  ASN A  62     -17.736  22.783  11.945  1.00  0.00           C  \\nATOM    466  CG  ASN A  62     -17.610  24.043  11.441  1.00  0.00           C  \\nATOM    467  OD1 ASN A  62     -18.744  24.547  10.559  1.00  0.00           O  \\nATOM    468  ND2 ASN A  62     -16.539  24.799  11.063  1.00  0.00           N  \\nATOM    469  N   THR A  63     -17.610  20.263  13.268  1.00  0.00           N  \\nATOM    470  CA  THR A  63     -17.043  19.255  14.087  1.00  0.00           C  \\nATOM    471  C   THR A  63     -16.602  17.869  13.331  1.00  0.00           C  \\nATOM    472  O   THR A  63     -15.720  17.239  13.331  1.00  0.00           O  \\nATOM    473  CB  THR A  63     -18.051  18.751  15.095  1.00  0.00           C  \\nATOM    474  OG1 THR A  63     -17.358  18.499  15.851  1.00  0.00           O  \\nATOM    475  CG2 THR A  63     -18.492  17.365  14.528  1.00  0.00           C  \\nATOM    476  N   LEU A  64     -17.358  17.617  12.197  1.00  0.00           N  \\nATOM    477  CA  LEU A  64     -16.854  16.357  11.441  1.00  0.00           C  \\nATOM    478  C   LEU A  64     -15.468  16.861  10.685  1.00  0.00           C  \\nATOM    479  O   LEU A  64     -14.838  16.231  10.433  1.00  0.00           O  \\nATOM    480  CB  LEU A  64     -18.114  15.979  10.055  1.00  0.00           C  \\nATOM    481  CG  LEU A  64     -18.744  14.719  10.433  1.00  0.00           C  \\nATOM    482  CD1 LEU A  64     -19.122  14.719  11.567  1.00  0.00           C  \\nATOM    483  CD2 LEU A  64     -19.878  14.593   9.803  1.00  0.00           C  \\nATOM    484  N   MET A  65     -15.468  17.995  10.181  1.00  0.00           N  \\nATOM    485  CA  MET A  65     -13.956  18.499   9.803  1.00  0.00           C  \\nATOM    486  C   MET A  65     -13.200  18.373  11.189  1.00  0.00           C  \\nATOM    487  O   MET A  65     -12.192  17.869  11.189  1.00  0.00           O  \\nATOM    488  CB  MET A  65     -13.956  19.885   9.425  1.00  0.00           C  \\nATOM    489  CG  MET A  65     -13.452  20.137   7.913  1.00  0.00           C  \\nATOM    490  SD  MET A  65     -13.704  18.625   7.283  1.00  0.00           S  \\nATOM    491  CE  MET A  65     -13.452  19.192   5.519  1.00  0.00           C  \\nATOM    492  N   GLU A  66     -13.578  18.751  12.197  1.00  0.00           N  \\nATOM    493  CA  GLU A  66     -13.074  18.751  13.457  1.00  0.00           C  \\nATOM    494  C   GLU A  66     -12.444  17.239  13.646  1.00  0.00           C  \\nATOM    495  O   GLU A  66     -11.310  17.113  13.835  1.00  0.00           O  \\nATOM    496  CB  GLU A  66     -13.830  19.381  14.717  1.00  0.00           C  \\nATOM    497  CG  GLU A  66     -13.578  20.767  14.969  1.00  0.00           C  \\nATOM    498  CD  GLU A  66     -12.318  21.271  15.851  1.00  0.00           C  \\nATOM    499  OE1 GLU A  66     -12.192  22.657  16.103  1.00  0.00           O  \\nATOM    500  OE2 GLU A  66     -11.310  20.641  16.355  1.00  0.00           O  \\nATOM    501  N   TYR A  67     -13.263  16.483  13.457  1.00  0.00           N  \\nATOM    502  CA  TYR A  67     -12.822  14.971  13.457  1.00  0.00           C  \\nATOM    503  C   TYR A  67     -12.192  14.467  12.323  1.00  0.00           C  \\nATOM    504  O   TYR A  67     -11.184  14.089  11.945  1.00  0.00           O  \\nATOM    505  CB  TYR A  67     -14.082  14.341  13.835  1.00  0.00           C  \\nATOM    506  CG  TYR A  67     -14.082  12.955  14.465  1.00  0.00           C  \\nATOM    507  CD1 TYR A  67     -14.838  11.821  14.465  1.00  0.00           C  \\nATOM    508  CD2 TYR A  67     -13.326  13.081  15.599  1.00  0.00           C  \\nATOM    509  CE1 TYR A  67     -14.649  10.876  15.221  1.00  0.00           C  \\nATOM    510  CE2 TYR A  67     -13.200  11.947  16.103  1.00  0.00           C  \\nATOM    511  CZ  TYR A  67     -13.830  10.876  15.977  1.00  0.00           C  \\nATOM    512  OH  TYR A  67     -13.704   9.805  16.607  1.00  0.00           O  \\nATOM    513  N   LEU A  68     -12.822  14.719  11.315  1.00  0.00           N  \\nATOM    514  CA  LEU A  68     -12.192  14.341  10.055  1.00  0.00           C  \\nATOM    515  C   LEU A  68     -10.806  14.719  10.307  1.00  0.00           C  \\nATOM    516  O   LEU A  68      -9.798  13.963  10.307  1.00  0.00           O  \\nATOM    517  CB  LEU A  68     -13.074  14.845   9.173  1.00  0.00           C  \\nATOM    518  CG  LEU A  68     -14.334  13.837   8.795  1.00  0.00           C  \\nATOM    519  CD1 LEU A  68     -15.216  14.467   7.787  1.00  0.00           C  \\nATOM    520  CD2 LEU A  68     -13.704  12.829   7.787  1.00  0.00           C  \\nATOM    521  N   GLU A  69     -10.428  15.853  10.685  1.00  0.00           N  \\nATOM    522  CA  GLU A  69      -8.916  16.231  10.937  1.00  0.00           C  \\nATOM    523  C   GLU A  69      -8.097  14.971  11.441  1.00  0.00           C  \\nATOM    524  O   GLU A  69      -7.278  14.593  10.685  1.00  0.00           O  \\nATOM    525  CB  GLU A  69      -9.042  17.365  11.693  1.00  0.00           C  \\nATOM    526  CG  GLU A  69      -8.034  18.373  11.819  1.00  0.00           C  \\nATOM    527  CD  GLU A  69      -6.900  18.373  12.953  1.00  0.00           C  \\nATOM    528  OE1 GLU A  69      -5.388  18.499  12.827  1.00  0.00           O  \\nATOM    529  OE2 GLU A  69      -7.278  17.869  14.087  1.00  0.00           O  \\nATOM    530  N   ASN A  70      -8.097  14.845  12.575  1.00  0.00           N  \\nATOM    531  CA  ASN A  70      -7.278  13.585  12.827  1.00  0.00           C  \\nATOM    532  C   ASN A  70      -7.404  13.459  14.339  1.00  0.00           C  \\nATOM    533  O   ASN A  70      -7.152  14.215  15.221  1.00  0.00           O  \\nATOM    534  CB  ASN A  70      -5.892  13.837  12.449  1.00  0.00           C  \\nATOM    535  CG  ASN A  70      -4.758  14.089  13.457  1.00  0.00           C  \\nATOM    536  OD1 ASN A  70      -3.624  15.097  13.457  1.00  0.00           O  \\nATOM    537  ND2 ASN A  70      -4.758  13.333  14.717  1.00  0.00           N  \\nATOM    538  N   PRO A  71      -7.908  12.199  14.780  1.00  0.00           N  \\nATOM    539  CA  PRO A  71      -8.475  11.758  15.725  1.00  0.00           C  \\nATOM    540  C   PRO A  71      -7.593  11.758  16.859  1.00  0.00           C  \\nATOM    541  O   PRO A  71      -7.782  11.317  17.993  1.00  0.00           O  \\nATOM    542  CB  PRO A  71      -9.042  10.561  15.536  1.00  0.00           C  \\nATOM    543  CG  PRO A  71      -8.916  10.309  14.213  1.00  0.00           C  \\nATOM    544  CD  PRO A  71      -8.160  11.191  13.583  1.00  0.00           C  \\nATOM    545  N   LYS A  72      -6.522  12.325  16.733  1.00  0.00           N  \\nATOM    546  CA  LYS A  72      -5.388  12.703  17.615  1.00  0.00           C  \\nATOM    547  C   LYS A  72      -5.451  14.215  17.993  1.00  0.00           C  \\nATOM    548  O   LYS A  72      -5.262  14.971  17.111  1.00  0.00           O  \\nATOM    549  CB  LYS A  72      -3.876  12.451  17.111  1.00  0.00           C  \\nATOM    550  CG  LYS A  72      -3.624  11.065  17.363  1.00  0.00           C  \\nATOM    551  CD  LYS A  72      -2.112  10.435  17.237  1.00  0.00           C  \\nATOM    552  CE  LYS A  72      -1.860   8.923  17.741  1.00  0.00           C  \\nATOM    553  NZ  LYS A  72      -0.222   8.545  17.741  1.00  0.00           N  \\nATOM    554  N   LYS A  73      -6.081  14.719  19.127  1.00  0.00           N  \\nATOM    555  CA  LYS A  73      -6.396  16.231  19.127  1.00  0.00           C  \\nATOM    556  C   LYS A  73      -7.782  16.357  18.623  1.00  0.00           C  \\nATOM    557  O   LYS A  73      -8.160  16.231  17.363  1.00  0.00           O  \\nATOM    558  CB  LYS A  73      -5.640  17.113  17.993  1.00  0.00           C  \\nATOM    559  CG  LYS A  73      -4.758  18.247  18.623  1.00  0.00           C  \\nATOM    560  CD  LYS A  73      -3.876  19.003  17.993  1.00  0.00           C  \\nATOM    561  CE  LYS A  73      -2.616  20.389  18.812  1.00  0.00           C  \\nATOM    562  NZ  LYS A  73      -1.986  21.145  18.056  1.00  0.00           N  \\nATOM    563  N   TYR A  74      -8.664  16.231  19.631  1.00  0.00           N  \\nATOM    564  CA  TYR A  74     -10.050  16.105  19.379  1.00  0.00           C  \\nATOM    565  C   TYR A  74     -10.806  14.971  19.883  1.00  0.00           C  \\nATOM    566  O   TYR A  74     -10.554  13.963  19.379  1.00  0.00           O  \\nATOM    567  CB  TYR A  74     -10.554  16.105  17.741  1.00  0.00           C  \\nATOM    568  CG  TYR A  74     -11.814  16.735  17.363  1.00  0.00           C  \\nATOM    569  CD1 TYR A  74     -12.948  16.105  17.237  1.00  0.00           C  \\nATOM    570  CD2 TYR A  74     -11.940  17.995  16.985  1.00  0.00           C  \\nATOM    571  CE1 TYR A  74     -14.145  16.609  16.859  1.00  0.00           C  \\nATOM    572  CE2 TYR A  74     -13.074  18.625  16.607  1.00  0.00           C  \\nATOM    573  CZ  TYR A  74     -14.082  17.869  16.607  1.00  0.00           C  \\nATOM    574  OH  TYR A  74     -15.216  18.373  16.355  1.00  0.00           O  \\nATOM    575  N   ILE A  75     -11.310  15.097  21.017  1.00  0.00           N  \\nATOM    576  CA  ILE A  75     -11.940  14.089  21.647  1.00  0.00           C  \\nATOM    577  C   ILE A  75     -10.806  13.081  21.773  1.00  0.00           C  \\nATOM    578  O   ILE A  75     -10.932  11.821  21.521  1.00  0.00           O  \\nATOM    579  CB  ILE A  75     -13.200  13.585  21.143  1.00  0.00           C  \\nATOM    580  CG1 ILE A  75     -13.641  14.089  19.631  1.00  0.00           C  \\nATOM    581  CG2 ILE A  75     -14.271  13.585  22.025  1.00  0.00           C  \\nATOM    582  CD1 ILE A  75     -14.712  13.459  19.253  1.00  0.00           C  \\nATOM    583  N   PRO A  76      -9.420  13.585  21.899  1.00  0.00           N  \\nATOM    584  CA  PRO A  76      -8.160  12.955  21.647  1.00  0.00           C  \\nATOM    585  C   PRO A  76      -8.160  11.695  22.529  1.00  0.00           C  \\nATOM    586  O   PRO A  76      -8.538  11.695  23.789  1.00  0.00           O  \\nATOM    587  CB  PRO A  76      -6.900  13.837  21.836  1.00  0.00           C  \\nATOM    588  CG  PRO A  76      -7.152  14.845  22.277  1.00  0.00           C  \\nATOM    589  CD  PRO A  76      -8.538  14.845  22.151  1.00  0.00           C  \\nATOM    590  N   GLY A  77      -8.097  10.435  22.088  1.00  0.00           N  \\nATOM    591  CA  GLY A  77      -8.286   9.175  22.907  1.00  0.00           C  \\nATOM    592  C   GLY A  77      -9.546   8.167  22.529  1.00  0.00           C  \\nATOM    593  O   GLY A  77      -9.798   7.159  22.781  1.00  0.00           O  \\nATOM    594  N   THR A  78     -10.050   8.545  21.143  1.00  0.00           N  \\nATOM    595  CA  THR A  78     -11.310   7.789  20.639  1.00  0.00           C  \\nATOM    596  C   THR A  78     -10.995   6.529  20.513  1.00  0.00           C  \\nATOM    597  O   THR A  78      -9.798   6.151  20.009  1.00  0.00           O  \\nATOM    598  CB  THR A  78     -11.625   8.167  19.253  1.00  0.00           C  \\nATOM    599  OG1 THR A  78     -13.074   8.167  19.127  1.00  0.00           O  \\nATOM    600  CG2 THR A  78     -11.184   9.553  18.875  1.00  0.00           C  \\nATOM    601  N   LYS A  79     -11.562   5.521  20.765  1.00  0.00           N  \\nATOM    602  CA  LYS A  79     -11.058   4.072  20.639  1.00  0.00           C  \\nATOM    603  C   LYS A  79     -11.058   3.631  19.001  1.00  0.00           C  \\nATOM    604  O   LYS A  79     -10.932   2.497  18.371  1.00  0.00           O  \\nATOM    605  CB  LYS A  79     -11.562   3.064  21.395  1.00  0.00           C  \\nATOM    606  CG  LYS A  79     -12.570   2.560  20.513  1.00  0.00           C  \\nATOM    607  CD  LYS A  79     -13.704   1.615  20.891  1.00  0.00           C  \\nATOM    608  CE  LYS A  79     -14.838   1.363  20.135  1.00  0.00           C  \\nATOM    609  NZ  LYS A  79     -15.972   0.859  20.513  1.00  0.00           N  \\nATOM    610  N   MET A  80     -11.310   4.639  18.119  1.00  0.00           N  \\nATOM    611  CA  MET A  80     -11.436   4.135  16.607  1.00  0.00           C  \\nATOM    612  C   MET A  80     -10.050   4.198  16.103  1.00  0.00           C  \\nATOM    613  O   MET A  80      -9.294   5.017  16.103  1.00  0.00           O  \\nATOM    614  CB  MET A  80     -12.444   5.017  15.851  1.00  0.00           C  \\nATOM    615  CG  MET A  80     -11.940   6.088  14.969  1.00  0.00           C  \\nATOM    616  SD  MET A  80     -13.074   6.781  13.457  1.00  0.00           S  \\nATOM    617  CE  MET A  80     -12.192   7.852  13.079  1.00  0.00           C  \\nATOM    618  N   ILE A  81      -9.672   3.127  15.725  1.00  0.00           N  \\nATOM    619  CA  ILE A  81      -8.160   3.127  15.095  1.00  0.00           C  \\nATOM    620  C   ILE A  81      -7.908   4.009  13.835  1.00  0.00           C  \\nATOM    621  O   ILE A  81      -8.034   3.505  12.323  1.00  0.00           O  \\nATOM    622  CB  ILE A  81      -7.656   1.867  15.284  1.00  0.00           C  \\nATOM    623  CG1 ILE A  81      -6.522   1.867  16.103  1.00  0.00           C  \\nATOM    624  CG2 ILE A  81      -7.026   1.237  13.835  1.00  0.00           C  \\nATOM    625  CD1 ILE A  81      -6.900   2.245  17.615  1.00  0.00           C  \\nATOM    626  N   PHE A  82      -7.530   5.143  13.961  1.00  0.00           N  \\nATOM    627  CA  PHE A  82      -7.404   5.899  12.449  1.00  0.00           C  \\nATOM    628  C   PHE A  82      -6.144   6.529  12.197  1.00  0.00           C  \\nATOM    629  O   PHE A  82      -5.766   7.411  12.953  1.00  0.00           O  \\nATOM    630  CB  PHE A  82      -8.538   6.655  12.449  1.00  0.00           C  \\nATOM    631  CG  PHE A  82      -8.916   7.159  10.937  1.00  0.00           C  \\nATOM    632  CD1 PHE A  82      -8.916   8.293  10.433  1.00  0.00           C  \\nATOM    633  CD2 PHE A  82      -9.042   6.529  10.055  1.00  0.00           C  \\nATOM    634  CE1 PHE A  82      -9.042   8.671   9.173  1.00  0.00           C  \\nATOM    635  CE2 PHE A  82      -9.042   6.781   8.669  1.00  0.00           C  \\nATOM    636  CZ  PHE A  82      -9.042   7.852   8.291  1.00  0.00           C  \\nATOM    637  N   ALA A  83      -5.640   6.529  11.189  1.00  0.00           N  \\nATOM    638  CA  ALA A  83      -4.506   7.285  10.937  1.00  0.00           C  \\nATOM    639  C   ALA A  83      -4.758   8.797  10.811  1.00  0.00           C  \\nATOM    640  O   ALA A  83      -4.884   9.679  11.441  1.00  0.00           O  \\nATOM    641  CB  ALA A  83      -3.876   6.907   9.173  1.00  0.00           C  \\nATOM    642  N   GLY A  84      -5.136   9.301   9.677  1.00  0.00           N  \\nATOM    643  CA  GLY A  84      -5.325  10.687   9.299  1.00  0.00           C  \\nATOM    644  C   GLY A  84      -6.396  11.317   8.543  1.00  0.00           C  \\nATOM    645  O   GLY A  84      -7.467  11.569   8.921  1.00  0.00           O  \\nATOM    646  N   ILE A  85      -6.207  11.569   7.283  1.00  0.00           N  \\nATOM    647  CA  ILE A  85      -7.152  12.451   6.653  1.00  0.00           C  \\nATOM    648  C   ILE A  85      -6.774  13.963   6.905  1.00  0.00           C  \\nATOM    649  O   ILE A  85      -7.656  14.719   7.220  1.00  0.00           O  \\nATOM    650  CB  ILE A  85      -8.664  12.199   7.031  1.00  0.00           C  \\nATOM    651  CG1 ILE A  85      -9.168  10.813   6.149  1.00  0.00           C  \\nATOM    652  CG2 ILE A  85      -9.420  13.333   6.401  1.00  0.00           C  \\nATOM    653  CD1 ILE A  85     -10.176  11.191   5.204  1.00  0.00           C  \\nATOM    654  N   LYS A  86      -5.640  14.467   6.527  1.00  0.00           N  \\nATOM    655  CA  LYS A  86      -5.325  15.727   7.157  1.00  0.00           C  \\nATOM    656  C   LYS A  86      -5.136  16.546   5.771  1.00  0.00           C  \\nATOM    657  O   LYS A  86      -4.254  17.365   5.267  1.00  0.00           O  \\nATOM    658  CB  LYS A  86      -3.876  15.727   7.913  1.00  0.00           C  \\nATOM    659  CG  LYS A  86      -3.120  14.467   7.535  1.00  0.00           C  \\nATOM    660  CD  LYS A  86      -1.356  14.215   7.787  1.00  0.00           C  \\nATOM    661  CE  LYS A  86      -0.852  12.955   6.653  1.00  0.00           C  \\nATOM    662  NZ  LYS A  86       0.660  12.577   7.157  1.00  0.00           N  \\nATOM    663  N   LYS A  87      -5.892  16.357   5.015  1.00  0.00           N  \\nATOM    664  CA  LYS A  87      -5.892  17.239   3.755  1.00  0.00           C  \\nATOM    665  C   LYS A  87      -7.278  17.869   3.377  1.00  0.00           C  \\nATOM    666  O   LYS A  87      -8.160  17.239   2.873  1.00  0.00           O  \\nATOM    667  CB  LYS A  87      -5.514  16.294   2.558  1.00  0.00           C  \\nATOM    668  CG  LYS A  87      -4.632  16.735   1.235  1.00  0.00           C  \\nATOM    669  CD  LYS A  87      -4.254  15.727   0.353  1.00  0.00           C  \\nATOM    670  CE  LYS A  87      -3.120  16.231  -0.403  1.00  0.00           C  \\nATOM    671  NZ  LYS A  87      -2.868  15.727  -1.915  1.00  0.00           N  \\nATOM    672  N   LYS A  88      -7.278  19.129   3.251  1.00  0.00           N  \\nATOM    673  CA  LYS A  88      -8.412  19.885   2.621  1.00  0.00           C  \\nATOM    674  C   LYS A  88      -9.168  18.877   1.613  1.00  0.00           C  \\nATOM    675  O   LYS A  88     -10.428  18.499   1.613  1.00  0.00           O  \\nATOM    676  CB  LYS A  88      -7.782  21.019   1.991  1.00  0.00           C  \\nATOM    677  CG  LYS A  88      -6.270  20.767   1.865  1.00  0.00           C  \\nATOM    678  CD  LYS A  88      -5.514  21.271   0.605  1.00  0.00           C  \\nATOM    679  CE  LYS A  88      -5.766  20.767  -0.403  1.00  0.00           C  \\nATOM    680  NZ  LYS A  88      -5.388  21.523  -1.663  1.00  0.00           N  \\nATOM    681  N   GLY A  89      -8.538  18.499   0.353  1.00  0.00           N  \\nATOM    682  CA  GLY A  89      -9.168  17.491  -0.781  1.00  0.00           C  \\nATOM    683  C   GLY A  89     -10.050  16.357  -0.025  1.00  0.00           C  \\nATOM    684  O   GLY A  89     -11.184  16.357  -0.025  1.00  0.00           O  \\nATOM    685  N   GLU A  90      -9.546  15.475   0.479  1.00  0.00           N  \\nATOM    686  CA  GLU A  90     -10.050  14.089   1.109  1.00  0.00           C  \\nATOM    687  C   GLU A  90     -10.932  14.593   2.369  1.00  0.00           C  \\nATOM    688  O   GLU A  90     -12.192  14.341   2.621  1.00  0.00           O  \\nATOM    689  CB  GLU A  90      -9.042  13.207   1.613  1.00  0.00           C  \\nATOM    690  CG  GLU A  90      -8.286  12.577   0.605  1.00  0.00           C  \\nATOM    691  CD  GLU A  90      -6.900  11.821   0.857  1.00  0.00           C  \\nATOM    692  OE1 GLU A  90      -6.018  11.443  -0.277  1.00  0.00           O  \\nATOM    693  OE2 GLU A  90      -6.270  11.569   2.243  1.00  0.00           O  \\nATOM    694  N   ARG A  91     -10.554  15.475   2.999  1.00  0.00           N  \\nATOM    695  CA  ARG A  91     -11.184  15.916   4.511  1.00  0.00           C  \\nATOM    696  C   ARG A  91     -12.570  16.672   3.881  1.00  0.00           C  \\nATOM    697  O   ARG A  91     -13.578  16.357   4.133  1.00  0.00           O  \\nATOM    698  CB  ARG A  91     -10.302  16.987   5.141  1.00  0.00           C  \\nATOM    699  CG  ARG A  91     -10.554  17.113   6.527  1.00  0.00           C  \\nATOM    700  CD  ARG A  91      -9.420  18.121   7.283  1.00  0.00           C  \\nATOM    701  NE  ARG A  91      -9.294  19.381   6.527  1.00  0.00           N  \\nATOM    702  CZ  ARG A  91      -7.908  20.137   6.464  1.00  0.00           C  \\nATOM    703  NH1 ARG A  91      -6.900  19.507   7.535  1.00  0.00           N  \\nATOM    704  NH2 ARG A  91      -7.530  21.145   5.519  1.00  0.00           N  \\nATOM    705  N   GLN A  92     -12.444  17.743   3.251  1.00  0.00           N  \\nATOM    706  CA  GLN A  92     -13.830  18.373   2.369  1.00  0.00           C  \\nATOM    707  C   GLN A  92     -14.586  17.113   1.991  1.00  0.00           C  \\nATOM    708  O   GLN A  92     -15.720  16.735   2.243  1.00  0.00           O  \\nATOM    709  CB  GLN A  92     -13.704  19.633   1.739  1.00  0.00           C  \\nATOM    710  CG  GLN A  92     -13.830  19.255   0.164  1.00  0.00           C  \\nATOM    711  CD  GLN A  92     -13.200  20.326  -0.907  1.00  0.00           C  \\nATOM    712  OE1 GLN A  92     -11.688  19.885  -0.403  1.00  0.00           O  \\nATOM    713  NE2 GLN A  92     -13.452  21.460  -1.411  1.00  0.00           N  \\nATOM    714  N   ASP A  93     -13.956  16.357   0.983  1.00  0.00           N  \\nATOM    715  CA  ASP A  93     -14.838  15.223   0.605  1.00  0.00           C  \\nATOM    716  C   ASP A  93     -15.342  14.341   1.613  1.00  0.00           C  \\nATOM    717  O   ASP A  93     -16.602  13.837   1.487  1.00  0.00           O  \\nATOM    718  CB  ASP A  93     -13.830  14.593  -0.529  1.00  0.00           C  \\nATOM    719  CG  ASP A  93     -13.704  14.971  -1.915  1.00  0.00           C  \\nATOM    720  OD1 ASP A  93     -12.444  15.727  -2.419  1.00  0.00           O  \\nATOM    721  OD2 ASP A  93     -14.838  15.223  -2.797  1.00  0.00           O  \\nATOM    722  N   LEU A  94     -14.838  13.963   2.495  1.00  0.00           N  \\nATOM    723  CA  LEU A  94     -15.531  13.207   3.503  1.00  0.00           C  \\nATOM    724  C   LEU A  94     -16.728  13.963   3.881  1.00  0.00           C  \\nATOM    725  O   LEU A  94     -17.925  13.459   3.881  1.00  0.00           O  \\nATOM    726  CB  LEU A  94     -14.586  13.018   4.763  1.00  0.00           C  \\nATOM    727  CG  LEU A  94     -14.586  11.443   5.267  1.00  0.00           C  \\nATOM    728  CD1 LEU A  94     -15.783  10.813   4.637  1.00  0.00           C  \\nATOM    729  CD2 LEU A  94     -13.578  10.561   4.889  1.00  0.00           C  \\nATOM    730  N   VAL A  95     -16.539  15.097   4.637  1.00  0.00           N  \\nATOM    731  CA  VAL A  95     -17.736  15.979   5.141  1.00  0.00           C  \\nATOM    732  C   VAL A  95     -18.744  15.853   3.881  1.00  0.00           C  \\nATOM    733  O   VAL A  95     -20.004  15.601   4.322  1.00  0.00           O  \\nATOM    734  CB  VAL A  95     -17.358  17.365   5.393  1.00  0.00           C  \\nATOM    735  CG1 VAL A  95     -18.492  18.184   5.519  1.00  0.00           C  \\nATOM    736  CG2 VAL A  95     -16.476  17.491   6.401  1.00  0.00           C  \\nATOM    737  N   ALA A  96     -18.492  15.979   2.621  1.00  0.00           N  \\nATOM    738  CA  ALA A  96     -19.752  15.601   1.739  1.00  0.00           C  \\nATOM    739  C   ALA A  96     -20.130  14.215   1.802  1.00  0.00           C  \\nATOM    740  O   ALA A  96     -21.390  13.837   1.802  1.00  0.00           O  \\nATOM    741  CB  ALA A  96     -19.500  16.105   0.353  1.00  0.00           C  \\nATOM    742  N   TYR A  97     -19.248  13.333   1.865  1.00  0.00           N  \\nATOM    743  CA  TYR A  97     -19.626  11.884   1.865  1.00  0.00           C  \\nATOM    744  C   TYR A  97     -20.886  11.632   2.873  1.00  0.00           C  \\nATOM    745  O   TYR A  97     -22.146  11.317   2.621  1.00  0.00           O  \\nATOM    746  CB  TYR A  97     -18.744  10.813   2.243  1.00  0.00           C  \\nATOM    747  CG  TYR A  97     -19.374   9.427   1.739  1.00  0.00           C  \\nATOM    748  CD1 TYR A  97     -19.626   8.419   2.621  1.00  0.00           C  \\nATOM    749  CD2 TYR A  97     -19.626   9.175   0.353  1.00  0.00           C  \\nATOM    750  CE1 TYR A  97     -20.256   7.348   2.117  1.00  0.00           C  \\nATOM    751  CE2 TYR A  97     -20.256   8.041  -0.403  1.00  0.00           C  \\nATOM    752  CZ  TYR A  97     -20.508   7.285   0.857  1.00  0.00           C  \\nATOM    753  OH  TYR A  97     -21.264   6.403   0.353  1.00  0.00           O  \\nATOM    754  N   LEU A  98     -20.760  11.947   4.007  1.00  0.00           N  \\nATOM    755  CA  LEU A  98     -21.768  11.695   5.393  1.00  0.00           C  \\nATOM    756  C   LEU A  98     -23.154  12.703   5.015  1.00  0.00           C  \\nATOM    757  O   LEU A  98     -24.288  12.199   4.889  1.00  0.00           O  \\nATOM    758  CB  LEU A  98     -21.012  11.695   6.527  1.00  0.00           C  \\nATOM    759  CG  LEU A  98     -20.004  10.939   6.401  1.00  0.00           C  \\nATOM    760  CD1 LEU A  98     -19.374  10.687   7.661  1.00  0.00           C  \\nATOM    761  CD2 LEU A  98     -20.508   9.553   5.771  1.00  0.00           C  \\nATOM    762  N   LYS A  99     -23.028  13.963   4.889  1.00  0.00           N  \\nATOM    763  CA  LYS A  99     -24.288  14.593   4.385  1.00  0.00           C  \\nATOM    764  C   LYS A  99     -25.044  13.963   2.999  1.00  0.00           C  \\nATOM    765  O   LYS A  99     -26.304  13.837   3.125  1.00  0.00           O  \\nATOM    766  CB  LYS A  99     -23.910  16.105   4.259  1.00  0.00           C  \\nATOM    767  CG  LYS A  99     -24.918  17.113   4.259  1.00  0.00           C  \\nATOM    768  CD  LYS A  99     -24.666  18.625   3.881  1.00  0.00           C  \\nATOM    769  CE  LYS A  99     -25.674  19.633   4.133  1.00  0.00           C  \\nATOM    770  NZ  LYS A  99     -25.674  21.082   3.755  1.00  0.00           N  \\nATOM    771  N   SER A 100     -24.540  13.459   1.991  1.00  0.00           N  \\nATOM    772  CA  SER A 100     -25.296  12.577   0.857  1.00  0.00           C  \\nATOM    773  C   SER A 100     -25.548  11.191   1.361  1.00  0.00           C  \\nATOM    774  O   SER A 100     -26.934  10.813   1.361  1.00  0.00           O  \\nATOM    775  CB  SER A 100     -24.540  12.577  -0.655  1.00  0.00           C  \\nATOM    776  OG  SER A 100     -25.296  11.695  -1.663  1.00  0.00           O  \\nATOM    777  N   ALA A 101     -24.666  10.435   1.865  1.00  0.00           N  \\nATOM    778  CA  ALA A 101     -25.044   8.797   2.243  1.00  0.00           C  \\nATOM    779  C   ALA A 101     -25.800   8.923   3.629  1.00  0.00           C  \\nATOM    780  O   ALA A 101     -26.808   8.293   3.755  1.00  0.00           O  \\nATOM    781  CB  ALA A 101     -23.910   8.167   2.369  1.00  0.00           C  \\nATOM    782  N   THR A 102     -25.422   9.427   4.637  1.00  0.00           N  \\nATOM    783  CA  THR A 102     -26.052   9.301   6.149  1.00  0.00           C  \\nATOM    784  C   THR A 102     -27.690   9.805   5.897  1.00  0.00           C  \\nATOM    785  O   THR A 102     -28.572   9.049   6.149  1.00  0.00           O  \\nATOM    786  CB  THR A 102     -25.548  10.246   7.220  1.00  0.00           C  \\nATOM    787  OG1 THR A 102     -24.666   9.427   7.913  1.00  0.00           O  \\nATOM    788  CG2 THR A 102     -26.430  10.876   8.039  1.00  0.00           C  \\nATOM    789  N   SER A 103     -27.942  10.939   5.645  1.00  0.00           N  \\nATOM    790  CA  SER A 103     -29.454  11.317   5.519  1.00  0.00           C  \\nATOM    791  C   SER A 103     -30.084  10.813   4.385  1.00  0.00           C  \\nATOM    792  O   SER A 103     -29.454   9.931   3.377  1.00  0.00           O  \\nATOM    793  CB  SER A 103     -29.580  12.892   5.897  1.00  0.00           C  \\nATOM    794  OG  SER A 103     -28.320  13.585   5.393  1.00  0.00           O  \\nATOM    795  OXT SER A 103     -31.344  11.317   4.133  1.00  0.00           O  \\nTER     796      SER A 103                                                      \\nHETATM 1593 FE   HEC A 104     -15.216   5.773  12.701  1.00  0.00          FE  \\nHETATM 1594  CHA HEC A 104     -16.980   6.466  15.347  1.00  0.00           C  \\nHETATM 1595  CHB HEC A 104     -16.476   8.293  11.189  1.00  0.00           C  \\nHETATM 1596  CHC HEC A 104     -13.578   5.017   9.929  1.00  0.00           C  \\nHETATM 1597  CHD HEC A 104     -13.956   3.127  14.339  1.00  0.00           C  \\nHETATM 1598  NA  HEC A 104     -16.476   7.096  13.205  1.00  0.00           N  \\nHETATM 1599  C1A HEC A 104     -17.106   7.159  14.465  1.00  0.00           C  \\nHETATM 1600  C2A HEC A 104     -17.862   8.167  14.591  1.00  0.00           C  \\nHETATM 1601  C3A HEC A 104     -17.736   8.797  13.205  1.00  0.00           C  \\nHETATM 1602  C4A HEC A 104     -17.114   8.167  12.323  1.00  0.00           C  \\nHETATM 1603  CMA HEC A 104     -18.303  10.057  12.701  1.00  0.00           C  \\nHETATM 1604  CAA HEC A 104     -18.618   8.167  15.599  1.00  0.00           C  \\nHETATM 1605  CBA HEC A 104     -20.067   8.734  15.473  1.00  0.00           C  \\nHETATM 1606  CGA HEC A 104     -20.823   8.734  16.859  1.00  0.00           C  \\nHETATM 1607  O1A HEC A 104     -21.705   8.041  17.111  1.00  0.00           O  \\nHETATM 1608  O2A HEC A 104     -20.508   9.490  17.741  1.00  0.00           O  \\nHETATM 1609  NB  HEC A 104     -15.090   6.466  11.126  1.00  0.00           N  \\nHETATM 1610  C1B HEC A 104     -15.594   7.348  10.559  1.00  0.00           C  \\nHETATM 1611  C2B HEC A 104     -15.216   7.663   9.299  1.00  0.00           C  \\nHETATM 1612  C3B HEC A 104     -14.334   6.592   8.921  1.00  0.00           C  \\nHETATM 1613  C4B HEC A 104     -14.334   6.025   9.929  1.00  0.00           C  \\nHETATM 1614  CMB HEC A 104     -15.657   8.797   8.543  1.00  0.00           C  \\nHETATM 1615  CAB HEC A 104     -13.578   6.781   7.409  1.00  0.00           C  \\nHETATM 1616  CBB HEC A 104     -12.570   7.663   7.661  1.00  0.00           C  \\nHETATM 1617  NC  HEC A 104     -13.956   4.387  12.197  1.00  0.00           N  \\nHETATM 1618  C1C HEC A 104     -13.452   4.261  10.811  1.00  0.00           C  \\nHETATM 1619  C2C HEC A 104     -12.759   3.001  10.811  1.00  0.00           C  \\nHETATM 1620  C3C HEC A 104     -12.948   2.623  11.945  1.00  0.00           C  \\nHETATM 1621  C4C HEC A 104     -13.704   3.379  12.953  1.00  0.00           C  \\nHETATM 1622  CMC HEC A 104     -11.688   2.497   9.803  1.00  0.00           C  \\nHETATM 1623  CAC HEC A 104     -12.444   1.111  12.197  1.00  0.00           C  \\nHETATM 1624  CBC HEC A 104     -11.184   0.859  12.008  1.00  0.00           C  \\nHETATM 1625  ND  HEC A 104     -15.405   5.143  14.213  1.00  0.00           N  \\nHETATM 1626  C1D HEC A 104     -14.838   4.009  14.843  1.00  0.00           C  \\nHETATM 1627  C2D HEC A 104     -15.216   3.757  15.977  1.00  0.00           C  \\nHETATM 1628  C3D HEC A 104     -16.098   4.765  16.355  1.00  0.00           C  \\nHETATM 1629  C4D HEC A 104     -16.161   5.395  15.347  1.00  0.00           C  \\nHETATM 1630  CMD HEC A 104     -14.838   2.749  16.733  1.00  0.00           C  \\nHETATM 1631  CAD HEC A 104     -16.854   4.513  17.615  1.00  0.00           C  \\nHETATM 1632  CBD HEC A 104     -16.098   4.135  18.623  1.00  0.00           C  \\nHETATM 1633  CGD HEC A 104     -15.216   5.017  19.505  1.00  0.00           C  \\nHETATM 1634  O1D HEC A 104     -14.082   5.773  18.560  1.00  0.00           O  \\nHETATM 1635  O2D HEC A 104     -15.405   5.647  20.639  1.00  0.00           O  \\nHETATM 1679  O   HOH A 105     -16.496   9.304  17.402  1.00  0.00           O  \\nCONECT  104 1615                                                                \\nCONECT  124 1623                                                                \\nCONECT  134 1593                                                                \\nCONECT  356 1635                                                                \\nCONECT  616 1593                                                                \\nCONECT  900 1658                                                                \\nCONECT  920 1666                                                                \\nCONECT  930 1636                                                                \\nCONECT 1152 1678                                                                \\nCONECT 1412 1636                                                                \\nCONECT 1593  134  616 1598 1609                                                 \\nCONECT 1593 1617 1625                                                           \\nCONECT 1594 1599 1629                                                           \\nCONECT 1595 1602 1610                                                           \\nCONECT 1596 1613 1618                                                           \\nCONECT 1597 1621 1626                                                           \\nCONECT 1598 1593 1599 1602                                                      \\nCONECT 1599 1594 1598 1600                                                      \\nCONECT 1600 1599 1601 1604                                                      \\nCONECT 1601 1600 1602 1603                                                      \\nCONECT 1602 1595 1598 1601                                                      \\nCONECT 1603 1601                                                                \\nCONECT 1604 1600 1605                                                           \\nCONECT 1605 1604 1606                                                           \\nCONECT 1606 1605 1607 1608                                                      \\nCONECT 1607 1606                                                                \\nCONECT 1608 1606                                                                \\nCONECT 1609 1593 1610 1613                                                      \\nCONECT 1610 1595 1609 1611                                                      \\nCONECT 1611 1610 1612 1614                                                      \\nCONECT 1612 1611 1613 1615                                                      \\nCONECT 1613 1596 1609 1612                                                      \\nCONECT 1614 1611                                                                \\nCONECT 1615  104 1612 1616                                                      \\nCONECT 1616 1615                                                                \\nCONECT 1617 1593 1618 1621                                                      \\nCONECT 1618 1596 1617 1619                                                      \\nCONECT 1619 1618 1620 1622                                                      \\nCONECT 1620 1619 1621 1623                                                      \\nCONECT 1621 1597 1617 1620                                                      \\nCONECT 1622 1619                                                                \\nCONECT 1623  124 1620 1624                                                      \\nCONECT 1624 1623                                                                \\nCONECT 1625 1593 1626 1629                                                      \\nCONECT 1626 1597 1625 1627                                                      \\nCONECT 1627 1626 1628 1630                                                      \\nCONECT 1628 1627 1629 1631                                                      \\nCONECT 1629 1594 1625 1628                                                      \\nCONECT 1630 1627                                                                \\nCONECT 1631 1628 1632                                                           \\nCONECT 1632 1631 1633                                                           \\nCONECT 1633 1632 1634 1635                                                      \\nCONECT 1634 1633                                                                \\nCONECT 1635  356 1633                                                           \\nCONECT 1636  930 1412 1641 1652                                                 \\nCONECT 1636 1660 1668                                                           \\nCONECT 1637 1642 1672                                                           \\nCONECT 1638 1645 1653                                                           \\nCONECT 1639 1656 1661                                                           \\nCONECT 1640 1664 1669                                                           \\nCONECT 1641 1636 1642 1645                                                      \\nCONECT 1642 1637 1641 1643                                                      \\nCONECT 1643 1642 1644 1647                                                      \\nCONECT 1644 1643 1645 1646                                                      \\nCONECT 1645 1638 1641 1644                                                      \\nCONECT 1646 1644                                                                \\nCONECT 1647 1643 1648                                                           \\nCONECT 1648 1647 1649                                                           \\nCONECT 1649 1648 1650 1651                                                      \\nCONECT 1650 1649                                                                \\nCONECT 1651 1649                                                                \\nCONECT 1652 1636 1653 1656                                                      \\nCONECT 1653 1638 1652 1654                                                      \\nCONECT 1654 1653 1655 1657                                                      \\nCONECT 1655 1654 1656 1658                                                      \\nCONECT 1656 1639 1652 1655                                                      \\nCONECT 1657 1654                                                                \\nCONECT 1658  900 1655 1659                                                      \\nCONECT 1659 1658                                                                \\nCONECT 1660 1636 1661 1664                                                      \\nCONECT 1661 1639 1660 1662                                                      \\nCONECT 1662 1661 1663 1665                                                      \\nCONECT 1663 1662 1664 1666                                                      \\nCONECT 1664 1640 1660 1663                                                      \\nCONECT 1665 1662                                                                \\nCONECT 1666  920 1663 1667                                                      \\nCONECT 1667 1666                                                                \\nCONECT 1668 1636 1669 1672                                                      \\nCONECT 1669 1640 1668 1670                                                      \\nCONECT 1670 1669 1671 1673                                                      \\nCONECT 1671 1670 1672 1674                                                      \\nCONECT 1672 1637 1668 1671                                                      \\nCONECT 1673 1670                                                                \\nCONECT 1674 1671 1675                                                           \\nCONECT 1675 1674 1676                                                           \\nCONECT 1676 1675 1677 1678                                                      \\nCONECT 1677 1676                                                                \\nCONECT 1678 1152 1676                                                           \\nMASTER      287    0    2   10    0    0    9    9 1677    2   98   16          \\nEND                                                                             \\n\",\"pdb\");\n\tviewer_16742555283712852.setBackgroundColor(\"white\");\n\tviewer_16742555283712852.setStyle({\"cartoon\": {\"color\": \"white\"}});\n\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"5\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"6\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"7\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"8\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"9\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"10\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"11\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"12\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"13\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"14\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"15\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"16\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"17\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"18\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"19\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"20\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"21\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"22\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"23\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"24\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"25\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"26\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"27\"]},{\"cartoon\": {\"color\": \"grey\"}});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"5\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"5\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"5\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"6\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"6\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"6\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"6\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"6\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"95\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"6\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"7\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"7\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"15\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"7\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"97\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"7\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"100\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"8\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"12\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"8\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"8\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"97\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"9\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"85\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"9\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"13\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"15\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"16\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"11\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"13\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"13\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"15\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"13\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"16\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"13\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"13\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"83\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"13\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"32\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"15\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"15\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"16\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"18\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"28\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"29\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"80\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"81\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"82\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"18\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"28\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"18\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"32\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"19\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"32\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"22\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"23\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"24\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"32\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"101\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"21\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"25\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"21\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"33\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"22\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"31\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"22\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"33\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"23\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"25\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"23\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"26\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"23\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"32\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"23\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"33\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"25\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"30\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"25\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"31\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"26\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"28\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"27\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"30\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"27\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"31\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n\tviewer_16742555283712852.zoomTo();\nviewer_16742555283712852.render();\n});\n</script>",
      "text/html": [
       "<div id=\"3dmolviewer_16742555283712852\"  style=\"position: relative; width: 640px; height: 480px\">\n",
       "        <p id=\"3dmolwarning_16742555283712852\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n",
       "        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n",
       "        </div>\n",
       "<script>\n",
       "\n",
       "var loadScriptAsync = function(uri){\n",
       "  return new Promise((resolve, reject) => {\n",
       "    //this is to ignore the existence of requirejs amd\n",
       "    var savedexports, savedmodule;\n",
       "    if (typeof exports !== 'undefined') savedexports = exports;\n",
       "    else exports = {}\n",
       "    if (typeof module !== 'undefined') savedmodule = module;\n",
       "    else module = {}\n",
       "\n",
       "    var tag = document.createElement('script');\n",
       "    tag.src = uri;\n",
       "    tag.async = true;\n",
       "    tag.onload = () => {\n",
       "        exports = savedexports;\n",
       "        module = savedmodule;\n",
       "        resolve();\n",
       "    };\n",
       "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
       "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
       "});\n",
       "};\n",
       "\n",
       "if(typeof $3Dmolpromise === 'undefined') {\n",
       "$3Dmolpromise = null;\n",
       "  $3Dmolpromise = loadScriptAsync('https://3dmol.org/build/3Dmol.js');\n",
       "}\n",
       "\n",
       "var viewer_16742555283712852 = null;\n",
       "var warn = document.getElementById(\"3dmolwarning_16742555283712852\");\n",
       "if(warn) {\n",
       "    warn.parentNode.removeChild(warn);\n",
       "}\n",
       "$3Dmolpromise.then(function() {\n",
       "viewer_16742555283712852 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_16742555283712852\"),{backgroundColor:\"white\"});\n",
       "viewer_16742555283712852.zoomTo();\n",
       "\tviewer_16742555283712852.addModel(\"HEADER    ELECTRON TRANSPORT                      01-AUG-76   1CYC              \\nTITLE     THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C AT 2.3     \\nTITLE    2 ANGSTROMS RESOLUTION. II. STRUCTURE AND FUNCTION                     \\nCAVEAT     1CYC    THR A 40 HAS WRONG CHIRALITY AT ATOM CB THR A 78 HAS WRONG   \\nCAVEAT   2 1CYC    CHIRALITY AT ATOM CB THR B 40 HAS WRONG CHIRALITY AT ATOM    \\nCAVEAT   3 1CYC    CB THR B 78 HAS WRONG CHIRALITY AT ATOM CB                   \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: FERROCYTOCHROME C;                                         \\nCOMPND   3 CHAIN: A, B;                                                         \\nCOMPND   4 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 ORGANISM_SCIENTIFIC: KATSUWONUS PELAMIS;                             \\nSOURCE   3 ORGANISM_COMMON: SKIPJACK TUNA;                                      \\nSOURCE   4 ORGANISM_TAXID: 8226                                                 \\nKEYWDS    ELECTRON TRANSPORT                                                    \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    N.TANAKA,T.YAMANE,T.TSUKIHARA,T.ASHIDA,M.KAKUDO                       \\nREVDAT  17   03-MAR-21 1CYC    1       CAVEAT COMPND REMARK HET                 \\nREVDAT  17 2                   1       HETNAM HETSYN FORMUL LINK                \\nREVDAT  17 3                   1       ATOM                                     \\nREVDAT  16   29-NOV-17 1CYC    1       HELIX                                    \\nREVDAT  15   24-FEB-09 1CYC    1       VERSN                                    \\nREVDAT  14   01-APR-03 1CYC    1       JRNL                                     \\nREVDAT  13   27-JAN-84 1CYC    1       REMARK                                   \\nREVDAT  12   30-SEP-83 1CYC    1       REVDAT                                   \\nREVDAT  11   19-APR-83 1CYC    1       COMPND SOURCE REMARK                     \\nREVDAT  10   15-JAN-82 1CYC    1       REMARK                                   \\nREVDAT   9   06-JAN-82 1CYC    3       SCALE                                    \\nREVDAT   8   31-DEC-80 1CYC    1       REMARK                                   \\nREVDAT   7   07-APR-80 1CYC    3       SOURCE SEQRES ATOM                       \\nREVDAT   6   23-APR-79 1CYC    3       ATOM                                     \\nREVDAT   5   01-NOV-77 1CYC    1       AUTHOR REMARK FORMUL                     \\nREVDAT   4   09-SEP-77 1CYC    3       ATOM                                     \\nREVDAT   3   13-JUN-77 1CYC    1       HET                                      \\nREVDAT   2   25-OCT-76 1CYC    2       CONECT                                   \\nREVDAT   1   06-OCT-76 1CYC    0                                                \\nJRNL        AUTH   N.TANAKA,T.YAMANE,T.TSUKIHARA,T.ASHIDA,M.KAKUDO              \\nJRNL        TITL   THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C   \\nJRNL        TITL 2 AT 2.3 A RESOLUTION. II. STRUCTURE AND FUNCTION.             \\nJRNL        REF    J.BIOCHEM.(TOKYO)             V.  77   147 1975              \\nJRNL        REFN                   ISSN 0021-924X                               \\nJRNL        PMID   166072                                                       \\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   T.TSUKIHARA,T.YAMANE,N.TANAKA,T.ASHIDA,M.KAKUDO              \\nREMARK   1  TITL   OXIDATION OF A FERROCYTOCHROME C IN THE CRYSTALLINE          \\nREMARK   1  TITL 2 STATE-STRUCTURAL CHANGE AND ANION BINDING                    \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  73  1163 1973              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 2                                                          \\nREMARK   1  AUTH   T.ASHIDA,N.TANAKA,T.YAMANE,T.TSUKIHARA,M.KAKUDO              \\nREMARK   1  TITL   THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C   \\nREMARK   1  TITL 2 AT 2.3 ANGSTROMS RESOLUTION                                  \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  73   463 1973              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 3                                                          \\nREMARK   1  AUTH   T.ASHIDA,T.UEKI,T.TSUKIHARA,A.SUGIHARA,T.TAKANO,M.KAKUDO     \\nREMARK   1  TITL   THE CRYSTAL STRUCTURE OF BONITO (KATSUO) FERROCYTOCHROME C   \\nREMARK   1  TITL 2 AT 4 ANGSTROMS RESOLUTION                                    \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  70   913 1971              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 4                                                          \\nREMARK   1  AUTH   T.NAKAYAMA,K.TITANI,K.NARITA                                 \\nREMARK   1  TITL   THE AMINO ACID SEQUENCE OF CYTOCHROME C FROM BONITO          \\nREMARK   1  TITL 2 (KATSUWONUS PELAMIS, LINNAEUS)                               \\nREMARK   1  REF    J.BIOCHEM.(TOKYO)             V.  70   311 1971              \\nREMARK   1  REFN                   ISSN 0021-924X                               \\nREMARK   1 REFERENCE 5                                                          \\nREMARK   1  EDIT   R.J.FELDMANN                                                 \\nREMARK   1  REF    ATLAS OF MACROMOLECULAR                120 1976              \\nREMARK   1  REF  2 STRUCTURE ON MICROFICHE                                      \\nREMARK   1  PUBL   TRACOR JITCO INC.,ROCKVILLE,MD.                              \\nREMARK   1  REFN                                                                \\nREMARK   1 REFERENCE 6                                                          \\nREMARK   1  EDIT   M.O.DAYHOFF                                                  \\nREMARK   1  REF    ATLAS OF PROTEIN SEQUENCE     V.   5    39 1976              \\nREMARK   1  REF  2 AND STRUCTURE,SUPPLEMENT 2                                   \\nREMARK   1  PUBL   NATIONAL BIOMEDICAL RESEARCH FOUNDATION, SILVER SPRING,MD.   \\nREMARK   1  REFN                   ISSN 0-912466-05-7                           \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    2.30 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : NULL                                                 \\nREMARK   3   AUTHORS     : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 2.30                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : NULL                           \\nREMARK   3   DATA CUTOFF            (SIGMA(F)) : NULL                           \\nREMARK   3   DATA CUTOFF HIGH         (ABS(F)) : NULL                           \\nREMARK   3   DATA CUTOFF LOW          (ABS(F)) : NULL                           \\nREMARK   3   COMPLETENESS (WORKING+TEST)   (%) : NULL                           \\nREMARK   3   NUMBER OF REFLECTIONS             : NULL                           \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   CROSS-VALIDATION METHOD          : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SELECTION  : NULL                            \\nREMARK   3   R VALUE            (WORKING SET) : NULL                            \\nREMARK   3   FREE R VALUE                     : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : NULL                            \\nREMARK   3   FREE R VALUE TEST SET COUNT      : NULL                            \\nREMARK   3   ESTIMATED ERROR OF FREE R VALUE  : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  FIT IN THE HIGHEST RESOLUTION BIN.                                  \\nREMARK   3   TOTAL NUMBER OF BINS USED           : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE HIGH       (A) : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE LOW        (A) : NULL                         \\nREMARK   3   BIN COMPLETENESS (WORKING+TEST) (%) : NULL                         \\nREMARK   3   REFLECTIONS IN BIN    (WORKING SET) : NULL                         \\nREMARK   3   BIN R VALUE           (WORKING SET) : NULL                         \\nREMARK   3   BIN FREE R VALUE                    : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET SIZE  (%) : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET COUNT     : NULL                         \\nREMARK   3   ESTIMATED ERROR OF BIN FREE R VALUE : NULL                         \\nREMARK   3                                                                      \\nREMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \\nREMARK   3   PROTEIN ATOMS            : 1590                                    \\nREMARK   3   NUCLEIC ACID ATOMS       : 0                                       \\nREMARK   3   HETEROGEN ATOMS          : 86                                      \\nREMARK   3   SOLVENT ATOMS            : 1                                       \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : NULL                           \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : NULL                           \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : NULL                                                 \\nREMARK   3    B22 (A**2) : NULL                                                 \\nREMARK   3    B33 (A**2) : NULL                                                 \\nREMARK   3    B12 (A**2) : NULL                                                 \\nREMARK   3    B13 (A**2) : NULL                                                 \\nREMARK   3    B23 (A**2) : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  ESTIMATED COORDINATE ERROR.                                         \\nREMARK   3   ESD FROM LUZZATI PLOT        (A) : NULL                            \\nREMARK   3   ESD FROM SIGMAA              (A) : NULL                            \\nREMARK   3   LOW RESOLUTION CUTOFF        (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  CROSS-VALIDATED ESTIMATED COORDINATE ERROR.                         \\nREMARK   3   ESD FROM C-V LUZZATI PLOT    (A) : NULL                            \\nREMARK   3   ESD FROM C-V SIGMAA          (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  RMS DEVIATIONS FROM IDEAL VALUES.                                   \\nREMARK   3   BOND LENGTHS                 (A) : NULL                            \\nREMARK   3   BOND ANGLES            (DEGREES) : NULL                            \\nREMARK   3   DIHEDRAL ANGLES        (DEGREES) : NULL                            \\nREMARK   3   IMPROPER ANGLES        (DEGREES) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL MODEL : NULL                                      \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL FACTOR RESTRAINTS.    RMS    SIGMA                \\nREMARK   3   MAIN-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   MAIN-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  NCS MODEL : NULL                                                    \\nREMARK   3                                                                      \\nREMARK   3  NCS RESTRAINTS.                         RMS   SIGMA/WEIGHT          \\nREMARK   3   GROUP  1  POSITIONAL            (A) : NULL  ; NULL                 \\nREMARK   3   GROUP  1  B-FACTOR           (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  PARAMETER FILE  1  : NULL                                           \\nREMARK   3  TOPOLOGY FILE  1   : NULL                                           \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS: NULL                                      \\nREMARK   4                                                                      \\nREMARK   4 1CYC COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY BNL.                                \\nREMARK 100 THE DEPOSITION ID IS D_1000172595.                                   \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : NULL                               \\nREMARK 200  TEMPERATURE           (KELVIN) : NULL                               \\nREMARK 200  PH                             : NULL                               \\nREMARK 200  NUMBER OF CRYSTALS USED        : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : NULL                               \\nREMARK 200  RADIATION SOURCE               : NULL                               \\nREMARK 200  BEAMLINE                       : NULL                               \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : NULL                               \\nREMARK 200  WAVELENGTH OR RANGE        (A) : NULL                               \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : NULL                               \\nREMARK 200  DETECTOR MANUFACTURER          : NULL                               \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : NULL                               \\nREMARK 200  DATA SCALING SOFTWARE          : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : NULL                               \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : NULL                               \\nREMARK 200  RESOLUTION RANGE LOW       (A) : NULL                               \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY                : NULL                               \\nREMARK 200  R MERGE                    (I) : NULL                               \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : NULL                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : NULL                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : NULL                               \\nREMARK 200  R MERGE FOR SHELL          (I) : NULL                               \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: NULL                                           \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: NULL                         \\nREMARK 200 SOFTWARE USED: NULL                                                  \\nREMARK 200 STARTING MODEL: NULL                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: NULL                                                         \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): 39.18                                     \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 2.02                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: NULL                                     \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: P 21 21 21                       \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -X+1/2,-Y,Z+1/2                                         \\nREMARK 290       3555   -X,Y+1/2,-Z+1/2                                         \\nREMARK 290       4555   X+1/2,-Y+1/2,-Z                                         \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -1.000000  0.000000  0.000000       28.84000            \\nREMARK 290   SMTRY2   2  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000       18.91500            \\nREMARK 290   SMTRY1   3 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   3  0.000000  1.000000  0.000000       42.29000            \\nREMARK 290   SMTRY3   3  0.000000  0.000000 -1.000000       18.91500            \\nREMARK 290   SMTRY1   4  1.000000  0.000000  0.000000       28.84000            \\nREMARK 290   SMTRY2   4  0.000000 -1.000000  0.000000       42.29000            \\nREMARK 290   SMTRY3   4  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1, 2                                                    \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: MONOMERIC                         \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 2                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: MONOMERIC                         \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: B                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 470                                                                      \\nREMARK 470 MISSING ATOM                                                         \\nREMARK 470 THE FOLLOWING RESIDUES HAVE MISSING ATOMS (M=MODEL NUMBER;           \\nREMARK 470 RES=RESIDUE NAME; C=CHAIN IDENTIFIER; SSEQ=SEQUENCE NUMBER;          \\nREMARK 470 I=INSERTION CODE):                                                   \\nREMARK 470   M RES CSSEQI  ATOMS                                                \\nREMARK 470     ASN A  22    CB   CG   OD1  ND2                                  \\nREMARK 470     LYS A  39    CE   NZ                                             \\nREMARK 470     ASN B  22    CB   CG   OD1  ND2                                  \\nREMARK 470     LYS B  39    CE   NZ                                             \\nREMARK 620                                                                      \\nREMARK 620 METAL COORDINATION                                                   \\nREMARK 620 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 620 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE):                             \\nREMARK 620                                                                      \\nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \\nREMARK 620                             HEC A 104  FE                            \\nREMARK 620 N RES CSSEQI ATOM                                                    \\nREMARK 620 1 HIS A  18   NE2                                                    \\nREMARK 620 2 HEC A 104   NA   87.0                                              \\nREMARK 620 3 HEC A 104   NB   87.8  90.6                                        \\nREMARK 620 4 HEC A 104   NC   92.0 178.7  90.1                                  \\nREMARK 620 5 HEC A 104   ND   91.1  86.9 177.4  92.3                            \\nREMARK 620 6 MET A  80   SD  170.9 102.1  93.0  79.0  88.6                      \\nREMARK 620 N                    1     2     3     4     5                       \\nREMARK 620                                                                      \\nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \\nREMARK 620                             HEC B 104  FE                            \\nREMARK 620 N RES CSSEQI ATOM                                                    \\nREMARK 620 1 HIS B  18   NE2                                                    \\nREMARK 620 2 HEC B 104   NA   87.0                                              \\nREMARK 620 3 HEC B 104   NB   87.8  90.6                                        \\nREMARK 620 4 HEC B 104   NC   92.0 178.7  90.1                                  \\nREMARK 620 5 HEC B 104   ND   91.1  86.9 177.4  92.3                            \\nREMARK 620 6 MET B  80   SD  170.9 102.1  93.0  79.0  88.6                      \\nREMARK 620 N                    1     2     3     4     5                       \\nREMARK 800                                                                      \\nREMARK 800 SITE                                                                 \\nREMARK 800 SITE_IDENTIFIER: AC1                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE HEC A 104                 \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: AC2                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE HEC B 104                 \\nDBREF  1CYC A    1   103  UNP    P00025   CYC_KATPE        1    103             \\nDBREF  1CYC B    1   103  UNP    P00025   CYC_KATPE        1    103             \\nSEQRES   1 A  103  GLY ASP VAL ALA LYS GLY LYS LYS THR PHE VAL GLN LYS          \\nSEQRES   2 A  103  CYS ALA GLN CYS HIS THR VAL GLU ASN GLY GLY LYS HIS          \\nSEQRES   3 A  103  LYS VAL GLY PRO ASN LEU TRP GLY LEU PHE GLY ARG LYS          \\nSEQRES   4 A  103  THR GLY GLN ALA GLU GLY TYR SER TYR THR ASP ALA ASN          \\nSEQRES   5 A  103  LYS SER LYS GLY ILE VAL TRP ASN GLU ASN THR LEU MET          \\nSEQRES   6 A  103  GLU TYR LEU GLU ASN PRO LYS LYS TYR ILE PRO GLY THR          \\nSEQRES   7 A  103  LYS MET ILE PHE ALA GLY ILE LYS LYS LYS GLY GLU ARG          \\nSEQRES   8 A  103  GLN ASP LEU VAL ALA TYR LEU LYS SER ALA THR SER              \\nSEQRES   1 B  103  GLY ASP VAL ALA LYS GLY LYS LYS THR PHE VAL GLN LYS          \\nSEQRES   2 B  103  CYS ALA GLN CYS HIS THR VAL GLU ASN GLY GLY LYS HIS          \\nSEQRES   3 B  103  LYS VAL GLY PRO ASN LEU TRP GLY LEU PHE GLY ARG LYS          \\nSEQRES   4 B  103  THR GLY GLN ALA GLU GLY TYR SER TYR THR ASP ALA ASN          \\nSEQRES   5 B  103  LYS SER LYS GLY ILE VAL TRP ASN GLU ASN THR LEU MET          \\nSEQRES   6 B  103  GLU TYR LEU GLU ASN PRO LYS LYS TYR ILE PRO GLY THR          \\nSEQRES   7 B  103  LYS MET ILE PHE ALA GLY ILE LYS LYS LYS GLY GLU ARG          \\nSEQRES   8 B  103  GLN ASP LEU VAL ALA TYR LEU LYS SER ALA THR SER              \\nHET    HEC  A 104      43                                                       \\nHET    HEC  B 104      43                                                       \\nHETNAM     HEC HEME C                                                           \\nFORMUL   3  HEC    2(C34 H34 FE N4 O4)                                          \\nFORMUL   5  HOH   *(H2 O)                                                       \\nHELIX    1  H1 GLY A    1  VAL A   11  1                                  11    \\nHELIX    2  H4 CYS A   14  CYS A   17  514 AND 17 BOUND TO HEME GROUP      4    \\nHELIX    3  H5 THR A   49  LYS A   55  5LOOSE FROM 49-53,3/10 53-55        7    \\nHELIX    4  H2 ASN A   60  GLU A   69  1                                  10    \\nHELIX    5  H3 GLU A   90  SER A  103  1                                  14    \\nHELIX    6  H6 GLY B    1  VAL B   11  1                                  11    \\nHELIX    7  H7 CYS B   14  CYS B   17  514 AND 17 BOUND TO HEME GROUP      4    \\nHELIX    8  H8 THR B   49  LYS B   55  5LOOSE FROM 49-53,3/10 53-55        7    \\nHELIX    9  H9 ASN B   60  GLU B   69  1                                  10    \\nHELIX   10 H10 GLU B   90  SER B  103  1                                  14    \\nLINK         SG  CYS A  14                 CAB HEC A 104     1555   1555  1.48  \\nLINK         SG  CYS A  17                 CAC HEC A 104     1555   1555  1.86  \\nLINK         CE2 TYR A  48                 O2D HEC A 104     1555   1555  2.02  \\nLINK         SG  CYS B  14                 CAB HEC B 104     1555   1555  1.48  \\nLINK         SG  CYS B  17                 CAC HEC B 104     1555   1555  1.86  \\nLINK         CE2 TYR B  48                 O2D HEC B 104     1555   1555  2.02  \\nLINK         NE2 HIS A  18                FE   HEC A 104     1555   1555  2.60  \\nLINK         SD  MET A  80                FE   HEC A 104     1555   1555  2.49  \\nLINK         NE2 HIS B  18                FE   HEC B 104     1555   1555  2.60  \\nLINK         SD  MET B  80                FE   HEC B 104     1555   1555  2.49  \\nSITE     1 AC1 17 LYS A  13  CYS A  14  CYS A  17  HIS A  18                    \\nSITE     2 AC1 17 LEU A  32  LEU A  35  ARG A  38  THR A  40                    \\nSITE     3 AC1 17 TYR A  46  TYR A  48  ASN A  52  TRP A  59                    \\nSITE     4 AC1 17 THR A  78  LYS A  79  MET A  80  PHE A  82                    \\nSITE     5 AC1 17 HOH A 105                                                     \\nSITE     1 AC2 16 LYS B  13  CYS B  14  CYS B  17  HIS B  18                    \\nSITE     2 AC2 16 LEU B  32  LEU B  35  ARG B  38  THR B  40                    \\nSITE     3 AC2 16 TYR B  46  TYR B  48  ASN B  52  TRP B  59                    \\nSITE     4 AC2 16 THR B  78  LYS B  79  MET B  80  PHE B  82                    \\nCRYST1   57.680   84.580   37.830  90.00  90.00  90.00 P 21 21 21    8          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.017337  0.000000  0.000000        0.00000                         \\nSCALE2      0.000000  0.011823  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.026434        0.00000                         \\nMTRIX1   1  1.000000  0.000000  0.000000       28.84000    1                    \\nMTRIX2   1  0.000000  1.000000  0.000000        0.00000    1                    \\nMTRIX3   1  0.000000  0.000000  1.000000        0.00000    1                    \\nATOM      1  N   GLY A   1     -21.138  13.774  -7.711  1.00  0.00           N  \\nATOM      2  CA  GLY A   1     -19.878  14.593  -7.963  1.00  0.00           C  \\nATOM      3  C   GLY A   1     -18.996  14.530  -6.703  1.00  0.00           C  \\nATOM      4  O   GLY A   1     -18.114  13.711  -6.640  1.00  0.00           O  \\nATOM      5  N   ASP A   2     -18.996  15.412  -5.821  1.00  0.00           N  \\nATOM      6  CA  ASP A   2     -17.610  15.097  -5.191  1.00  0.00           C  \\nATOM      7  C   ASP A   2     -17.862  14.152  -3.931  1.00  0.00           C  \\nATOM      8  O   ASP A   2     -17.106  13.333  -3.553  1.00  0.00           O  \\nATOM      9  CB  ASP A   2     -17.169  16.546  -4.435  1.00  0.00           C  \\nATOM     10  CG  ASP A   2     -17.988  17.365  -3.427  1.00  0.00           C  \\nATOM     11  OD1 ASP A   2     -19.500  17.365  -3.994  1.00  0.00           O  \\nATOM     12  OD2 ASP A   2     -17.484  18.373  -2.293  1.00  0.00           O  \\nATOM     13  N   VAL A   3     -19.059  14.089  -3.679  1.00  0.00           N  \\nATOM     14  CA  VAL A   3     -19.815  12.955  -2.860  1.00  0.00           C  \\nATOM     15  C   VAL A   3     -19.374  11.506  -3.805  1.00  0.00           C  \\nATOM     16  O   VAL A   3     -18.870  10.624  -3.427  1.00  0.00           O  \\nATOM     17  CB  VAL A   3     -22.587  13.270  -3.175  1.00  0.00           C  \\nATOM     18  CG1 VAL A   3     -21.957  11.947  -3.427  1.00  0.00           C  \\nATOM     19  CG2 VAL A   3     -21.705  13.963  -2.167  1.00  0.00           C  \\nATOM     20  N   ALA A   4     -19.437  11.380  -5.002  1.00  0.00           N  \\nATOM     21  CA  ALA A   4     -18.681  10.120  -5.569  1.00  0.00           C  \\nATOM     22  C   ALA A   4     -17.358   9.805  -4.939  1.00  0.00           C  \\nATOM     23  O   ALA A   4     -17.295   8.923  -4.183  1.00  0.00           O  \\nATOM     24  CB  ALA A   4     -18.429  10.435  -7.144  1.00  0.00           C  \\nATOM     25  N   LYS A   5     -16.476  10.687  -5.191  1.00  0.00           N  \\nATOM     26  CA  LYS A   5     -15.216  10.498  -4.372  1.00  0.00           C  \\nATOM     27  C   LYS A   5     -15.279  10.246  -2.797  1.00  0.00           C  \\nATOM     28  O   LYS A   5     -14.838   9.238  -2.293  1.00  0.00           O  \\nATOM     29  CB  LYS A   5     -14.208  11.695  -4.246  1.00  0.00           C  \\nATOM     30  CG  LYS A   5     -12.948  11.758  -3.553  1.00  0.00           C  \\nATOM     31  CD  LYS A   5     -11.814  13.081  -3.679  1.00  0.00           C  \\nATOM     32  CE  LYS A   5     -10.806  13.144  -2.545  1.00  0.00           C  \\nATOM     33  NZ  LYS A   5      -9.735  14.404  -2.293  1.00  0.00           N  \\nATOM     34  N   GLY A   6     -16.161  10.750  -1.915  1.00  0.00           N  \\nATOM     35  CA  GLY A   6     -16.350  10.435  -0.403  1.00  0.00           C  \\nATOM     36  C   GLY A   6     -16.539   8.923  -0.655  1.00  0.00           C  \\nATOM     37  O   GLY A   6     -15.909   8.041  -0.025  1.00  0.00           O  \\nATOM     38  N   LYS A   7     -17.421   8.419  -1.411  1.00  0.00           N  \\nATOM     39  CA  LYS A   7     -17.799   6.844  -1.033  1.00  0.00           C  \\nATOM     40  C   LYS A   7     -16.476   6.151  -1.474  1.00  0.00           C  \\nATOM     41  O   LYS A   7     -15.972   4.954  -0.970  1.00  0.00           O  \\nATOM     42  CB  LYS A   7     -18.870   6.403  -2.293  1.00  0.00           C  \\nATOM     43  CG  LYS A   7     -18.870   5.017  -2.734  1.00  0.00           C  \\nATOM     44  CD  LYS A   7     -20.193   4.765  -3.427  1.00  0.00           C  \\nATOM     45  CE  LYS A   7     -20.193   3.190  -4.057  1.00  0.00           C  \\nATOM     46  NZ  LYS A   7     -21.516   2.623  -4.309  1.00  0.00           N  \\nATOM     47  N   LYS A   8     -15.846   6.403  -2.545  1.00  0.00           N  \\nATOM     48  CA  LYS A   8     -14.460   5.899  -2.923  1.00  0.00           C  \\nATOM     49  C   LYS A   8     -13.452   5.899  -1.663  1.00  0.00           C  \\nATOM     50  O   LYS A   8     -13.011   5.017  -1.348  1.00  0.00           O  \\nATOM     51  CB  LYS A   8     -13.704   6.844  -3.931  1.00  0.00           C  \\nATOM     52  CG  LYS A   8     -12.318   6.529  -4.183  1.00  0.00           C  \\nATOM     53  CD  LYS A   8     -11.562   7.537  -5.065  1.00  0.00           C  \\nATOM     54  CE  LYS A   8     -10.176   7.222  -5.821  1.00  0.00           C  \\nATOM     55  NZ  LYS A   8      -9.357   8.419  -6.325  1.00  0.00           N  \\nATOM     56  N   THR A   9     -13.452   6.970  -1.159  1.00  0.00           N  \\nATOM     57  CA  THR A   9     -12.633   7.033   0.227  1.00  0.00           C  \\nATOM     58  C   THR A   9     -13.515   6.277   1.298  1.00  0.00           C  \\nATOM     59  O   THR A   9     -12.948   5.773   2.369  1.00  0.00           O  \\nATOM     60  CB  THR A   9     -12.381   8.419   0.920  1.00  0.00           C  \\nATOM     61  OG1 THR A   9     -11.058   8.923   0.542  1.00  0.00           O  \\nATOM     62  CG2 THR A   9     -12.444   8.734   2.558  1.00  0.00           C  \\nATOM     63  N   PHE A  10     -14.775   6.403   1.235  1.00  0.00           N  \\nATOM     64  CA  PHE A  10     -15.531   5.647   2.117  1.00  0.00           C  \\nATOM     65  C   PHE A  10     -15.216   4.135   1.739  1.00  0.00           C  \\nATOM     66  O   PHE A  10     -14.460   3.757   2.495  1.00  0.00           O  \\nATOM     67  CB  PHE A  10     -17.106   5.647   1.991  1.00  0.00           C  \\nATOM     68  CG  PHE A  10     -17.736   4.891   3.251  1.00  0.00           C  \\nATOM     69  CD1 PHE A  10     -18.492   3.757   2.747  1.00  0.00           C  \\nATOM     70  CD2 PHE A  10     -17.862   5.143   4.133  1.00  0.00           C  \\nATOM     71  CE1 PHE A  10     -18.996   3.001   3.818  1.00  0.00           C  \\nATOM     72  CE2 PHE A  10     -18.366   4.513   4.889  1.00  0.00           C  \\nATOM     73  CZ  PHE A  10     -18.996   3.379   5.015  1.00  0.00           C  \\nATOM     74  N   VAL A  11     -15.216   3.379   0.479  1.00  0.00           N  \\nATOM     75  CA  VAL A  11     -14.964   2.056   0.605  1.00  0.00           C  \\nATOM     76  C   VAL A  11     -13.326   1.867   1.109  1.00  0.00           C  \\nATOM     77  O   VAL A  11     -12.948   0.985   1.865  1.00  0.00           O  \\nATOM     78  CB  VAL A  11     -15.153   1.237  -0.529  1.00  0.00           C  \\nATOM     79  CG1 VAL A  11     -16.098   1.930  -1.537  1.00  0.00           C  \\nATOM     80  CG2 VAL A  11     -13.956   1.111  -1.285  1.00  0.00           C  \\nATOM     81  N   GLN A  12     -12.570   2.749   0.794  1.00  0.00           N  \\nATOM     82  CA  GLN A  12     -10.995   2.749   1.235  1.00  0.00           C  \\nATOM     83  C   GLN A  12     -10.680   2.938   2.873  1.00  0.00           C  \\nATOM     84  O   GLN A  12     -10.176   1.930   3.377  1.00  0.00           O  \\nATOM     85  CB  GLN A  12     -10.428   3.820   0.479  1.00  0.00           C  \\nATOM     86  CG  GLN A  12      -9.861   3.253  -0.718  1.00  0.00           C  \\nATOM     87  CD  GLN A  12      -9.168   4.135  -2.293  1.00  0.00           C  \\nATOM     88  OE1 GLN A  12      -7.593   3.820  -2.041  1.00  0.00           O  \\nATOM     89  NE2 GLN A  12      -9.672   5.143  -2.671  1.00  0.00           N  \\nATOM     90  N   LYS A  13     -10.995   3.757   3.629  1.00  0.00           N  \\nATOM     91  CA  LYS A  13     -10.869   3.757   5.015  1.00  0.00           C  \\nATOM     92  C   LYS A  13     -12.066   3.694   5.960  1.00  0.00           C  \\nATOM     93  O   LYS A  13     -12.003   3.127   6.779  1.00  0.00           O  \\nATOM     94  CB  LYS A  13     -10.239   5.017   5.141  1.00  0.00           C  \\nATOM     95  CG  LYS A  13      -9.672   5.521   3.881  1.00  0.00           C  \\nATOM     96  CD  LYS A  13      -8.286   6.277   4.007  1.00  0.00           C  \\nATOM     97  CE  LYS A  13      -8.160   6.907   2.621  1.00  0.00           C  \\nATOM     98  NZ  LYS A  13      -6.837   7.789   2.243  1.00  0.00           N  \\nATOM     99  N   CYS A  14     -13.137   4.072   5.519  1.00  0.00           N  \\nATOM    100  CA  CYS A  14     -14.460   4.072   6.464  1.00  0.00           C  \\nATOM    101  C   CYS A  14     -15.216   2.812   5.897  1.00  0.00           C  \\nATOM    102  O   CYS A  14     -15.342   1.804   6.716  1.00  0.00           O  \\nATOM    103  CB  CYS A  14     -15.468   5.332   5.897  1.00  0.00           C  \\nATOM    104  SG  CYS A  14     -14.712   6.844   6.464  1.00  0.00           S  \\nATOM    105  N   ALA A  15     -15.720   2.623   4.637  1.00  0.00           N  \\nATOM    106  CA  ALA A  15     -16.161   1.237   4.322  1.00  0.00           C  \\nATOM    107  C   ALA A  15     -16.035  -0.086   5.015  1.00  0.00           C  \\nATOM    108  O   ALA A  15     -17.043  -0.590   5.393  1.00  0.00           O  \\nATOM    109  CB  ALA A  15     -15.405   0.985   3.062  1.00  0.00           C  \\nATOM    110  N   GLN A  16     -14.964  -0.653   4.889  1.00  0.00           N  \\nATOM    111  CA  GLN A  16     -14.901  -2.039   5.897  1.00  0.00           C  \\nATOM    112  C   GLN A  16     -15.090  -1.787   7.283  1.00  0.00           C  \\nATOM    113  O   GLN A  16     -14.838  -2.669   8.291  1.00  0.00           O  \\nATOM    114  CB  GLN A  16     -13.578  -2.039   5.519  1.00  0.00           C  \\nATOM    115  CG  GLN A  16     -12.318  -1.724   6.275  1.00  0.00           C  \\nATOM    116  CD  GLN A  16     -11.625  -0.842   5.015  1.00  0.00           C  \\nATOM    117  OE1 GLN A  16     -10.365  -0.149   5.330  1.00  0.00           O  \\nATOM    118  NE2 GLN A  16     -12.444  -0.464   4.385  1.00  0.00           N  \\nATOM    119  N   CYS A  17     -15.216  -0.653   7.787  1.00  0.00           N  \\nATOM    120  CA  CYS A  17     -15.405  -0.905   9.299  1.00  0.00           C  \\nATOM    121  C   CYS A  17     -16.917  -0.401   9.173  1.00  0.00           C  \\nATOM    122  O   CYS A  17     -17.736  -1.283   9.173  1.00  0.00           O  \\nATOM    123  CB  CYS A  17     -14.460   0.355  10.055  1.00  0.00           C  \\nATOM    124  SG  CYS A  17     -13.326  -0.338  11.441  1.00  0.00           S  \\nATOM    125  N   HIS A  18     -17.232   0.733   9.551  1.00  0.00           N  \\nATOM    126  CA  HIS A  18     -18.870   0.985   9.803  1.00  0.00           C  \\nATOM    127  C   HIS A  18     -19.878   0.229   8.795  1.00  0.00           C  \\nATOM    128  O   HIS A  18     -19.626  -0.023   7.472  1.00  0.00           O  \\nATOM    129  CB  HIS A  18     -19.185   2.245   9.173  1.00  0.00           C  \\nATOM    130  CG  HIS A  18     -18.681   3.127  10.181  1.00  0.00           C  \\nATOM    131  ND1 HIS A  18     -18.618   3.253  11.567  1.00  0.00           N  \\nATOM    132  CD2 HIS A  18     -17.736   4.072  10.559  1.00  0.00           C  \\nATOM    133  CE1 HIS A  18     -18.051   3.694  12.827  1.00  0.00           C  \\nATOM    134  NE2 HIS A  18     -17.232   4.387  11.819  1.00  0.00           N  \\nATOM    135  N   THR A  19     -21.012   0.103   9.299  1.00  0.00           N  \\nATOM    136  CA  THR A  19     -22.272  -0.401   8.417  1.00  0.00           C  \\nATOM    137  C   THR A  19     -22.902   0.922   7.661  1.00  0.00           C  \\nATOM    138  O   THR A  19     -22.272   1.867   7.157  1.00  0.00           O  \\nATOM    139  CB  THR A  19     -23.280  -1.283   8.795  1.00  0.00           C  \\nATOM    140  OG1 THR A  19     -23.532  -0.905   9.740  1.00  0.00           O  \\nATOM    141  CG2 THR A  19     -22.776  -2.669   8.543  1.00  0.00           C  \\nATOM    142  N   VAL A  20     -24.036   1.111   7.472  1.00  0.00           N  \\nATOM    143  CA  VAL A  20     -24.414   2.371   6.779  1.00  0.00           C  \\nATOM    144  C   VAL A  20     -26.052   2.434   6.401  1.00  0.00           C  \\nATOM    145  O   VAL A  20     -26.556   3.442   5.771  1.00  0.00           O  \\nATOM    146  CB  VAL A  20     -23.784   2.560   5.519  1.00  0.00           C  \\nATOM    147  CG1 VAL A  20     -24.540   2.119   4.637  1.00  0.00           C  \\nATOM    148  CG2 VAL A  20     -23.658   4.009   5.267  1.00  0.00           C  \\nATOM    149  N   GLU A  21     -26.682   1.363   6.527  1.00  0.00           N  \\nATOM    150  CA  GLU A  21     -28.194   1.363   5.897  1.00  0.00           C  \\nATOM    151  C   GLU A  21     -29.013   1.741   7.283  1.00  0.00           C  \\nATOM    152  O   GLU A  21     -28.635   2.623   7.913  1.00  0.00           O  \\nATOM    153  CB  GLU A  21     -28.572  -0.086   5.645  1.00  0.00           C  \\nATOM    154  CG  GLU A  21     -28.698  -0.779   4.133  1.00  0.00           C  \\nATOM    155  CD  GLU A  21     -27.690  -1.913   4.007  1.00  0.00           C  \\nATOM    156  OE1 GLU A  21     -26.682  -2.039   5.141  1.00  0.00           O  \\nATOM    157  OE2 GLU A  21     -27.753  -2.921   2.873  1.00  0.00           O  \\nATOM    158  N   ASN A  22     -30.210   1.363   7.283  1.00  0.00           N  \\nATOM    159  CA  ASN A  22     -30.966   1.867   8.417  1.00  0.00           C  \\nATOM    160  C   ASN A  22     -30.714   1.237   9.677  1.00  0.00           C  \\nATOM    161  O   ASN A  22     -31.218   0.103  10.055  1.00  0.00           O  \\nATOM    162  N   GLY A  23     -29.958   1.552  10.433  1.00  0.00           N  \\nATOM    163  CA  GLY A  23     -29.517   0.859  11.819  1.00  0.00           C  \\nATOM    164  C   GLY A  23     -28.572  -0.338  11.252  1.00  0.00           C  \\nATOM    165  O   GLY A  23     -28.194  -1.283  12.071  1.00  0.00           O  \\nATOM    166  N   GLY A  24     -28.068  -0.149   9.929  1.00  0.00           N  \\nATOM    167  CA  GLY A  24     -27.060  -1.409   9.047  1.00  0.00           C  \\nATOM    168  C   GLY A  24     -26.871  -2.480  10.118  1.00  0.00           C  \\nATOM    169  O   GLY A  24     -27.186  -3.803   9.929  1.00  0.00           O  \\nATOM    170  N   LYS A  25     -26.304  -2.165  11.441  1.00  0.00           N  \\nATOM    171  CA  LYS A  25     -26.241  -3.047  12.701  1.00  0.00           C  \\nATOM    172  C   LYS A  25     -24.792  -2.795  13.079  1.00  0.00           C  \\nATOM    173  O   LYS A  25     -23.910  -2.417  12.323  1.00  0.00           O  \\nATOM    174  CB  LYS A  25     -26.241  -4.433  11.819  1.00  0.00           C  \\nATOM    175  CG  LYS A  25     -25.296  -4.496  10.433  1.00  0.00           C  \\nATOM    176  CD  LYS A  25     -24.918  -5.819   9.803  1.00  0.00           C  \\nATOM    177  CE  LYS A  25     -23.784  -5.819   8.543  1.00  0.00           C  \\nATOM    178  NZ  LYS A  25     -23.658  -6.827   7.661  1.00  0.00           N  \\nATOM    179  N   HIS A  26     -24.414  -2.921  14.465  1.00  0.00           N  \\nATOM    180  CA  HIS A  26     -23.028  -2.417  14.654  1.00  0.00           C  \\nATOM    181  C   HIS A  26     -21.894  -3.425  14.087  1.00  0.00           C  \\nATOM    182  O   HIS A  26     -22.083  -3.803  12.827  1.00  0.00           O  \\nATOM    183  CB  HIS A  26     -23.028  -2.543  16.103  1.00  0.00           C  \\nATOM    184  CG  HIS A  26     -24.036  -1.094  16.355  1.00  0.00           C  \\nATOM    185  ND1 HIS A  26     -24.540  -0.338  15.473  1.00  0.00           N  \\nATOM    186  CD2 HIS A  26     -24.666  -0.905  17.363  1.00  0.00           C  \\nATOM    187  CE1 HIS A  26     -25.422   0.355  16.103  1.00  0.00           C  \\nATOM    188  NE2 HIS A  26     -25.422   0.103  17.237  1.00  0.00           N  \\nATOM    189  N   LYS A  27     -20.886  -3.551  14.717  1.00  0.00           N  \\nATOM    190  CA  LYS A  27     -19.752  -4.433  14.213  1.00  0.00           C  \\nATOM    191  C   LYS A  27     -18.366  -3.677  14.276  1.00  0.00           C  \\nATOM    192  O   LYS A  27     -17.484  -3.677  13.331  1.00  0.00           O  \\nATOM    193  CB  LYS A  27     -20.004  -4.559  12.575  1.00  0.00           C  \\nATOM    194  CG  LYS A  27     -20.382  -3.425  11.882  1.00  0.00           C  \\nATOM    195  CD  LYS A  27     -19.500  -3.047  11.063  1.00  0.00           C  \\nATOM    196  CE  LYS A  27     -19.752  -3.677   9.551  1.00  0.00           C  \\nATOM    197  NZ  LYS A  27     -18.618  -3.677   8.858  1.00  0.00           N  \\nATOM    198  N   VAL A  28     -18.177  -3.047  15.410  1.00  0.00           N  \\nATOM    199  CA  VAL A  28     -16.854  -2.228  15.410  1.00  0.00           C  \\nATOM    200  C   VAL A  28     -17.358  -0.905  15.473  1.00  0.00           C  \\nATOM    201  O   VAL A  28     -17.610  -0.275  16.481  1.00  0.00           O  \\nATOM    202  CB  VAL A  28     -15.846  -2.291  14.780  1.00  0.00           C  \\nATOM    203  CG1 VAL A  28     -15.594  -1.094  14.087  1.00  0.00           C  \\nATOM    204  CG2 VAL A  28     -14.586  -2.858  15.095  1.00  0.00           C  \\nATOM    205  N   GLY A  29     -17.610  -0.023  14.591  1.00  0.00           N  \\nATOM    206  CA  GLY A  29     -18.240   1.111  14.843  1.00  0.00           C  \\nATOM    207  C   GLY A  29     -19.815   1.048  14.843  1.00  0.00           C  \\nATOM    208  O   GLY A  29     -20.193   0.103  13.961  1.00  0.00           O  \\nATOM    209  N   PRO A  30     -20.634   1.615  15.977  1.00  0.00           N  \\nATOM    210  CA  PRO A  30     -22.020   1.489  15.725  1.00  0.00           C  \\nATOM    211  C   PRO A  30     -22.398   1.804  14.339  1.00  0.00           C  \\nATOM    212  O   PRO A  30     -22.020   2.371  13.457  1.00  0.00           O  \\nATOM    213  CB  PRO A  30     -22.524   2.371  16.481  1.00  0.00           C  \\nATOM    214  CG  PRO A  30     -21.642   1.993  16.859  1.00  0.00           C  \\nATOM    215  CD  PRO A  30     -20.508   2.497  16.607  1.00  0.00           C  \\nATOM    216  N   ASN A  31     -23.910   1.552  13.961  1.00  0.00           N  \\nATOM    217  CA  ASN A  31     -24.414   2.119  12.701  1.00  0.00           C  \\nATOM    218  C   ASN A  31     -24.036   3.631  12.764  1.00  0.00           C  \\nATOM    219  O   ASN A  31     -24.036   4.387  13.835  1.00  0.00           O  \\nATOM    220  CB  ASN A  31     -25.926   1.993  12.701  1.00  0.00           C  \\nATOM    221  CG  ASN A  31     -26.052   2.056  11.063  1.00  0.00           C  \\nATOM    222  OD1 ASN A  31     -27.186   2.623  10.307  1.00  0.00           O  \\nATOM    223  ND2 ASN A  31     -25.359   1.363  10.181  1.00  0.00           N  \\nATOM    224  N   LEU A  32     -23.658   4.261  11.819  1.00  0.00           N  \\nATOM    225  CA  LEU A  32     -23.280   5.899  11.819  1.00  0.00           C  \\nATOM    226  C   LEU A  32     -24.666   6.403  11.315  1.00  0.00           C  \\nATOM    227  O   LEU A  32     -24.666   6.655   9.929  1.00  0.00           O  \\nATOM    228  CB  LEU A  32     -22.146   6.466  11.315  1.00  0.00           C  \\nATOM    229  CG  LEU A  32     -20.886   6.529  12.323  1.00  0.00           C  \\nATOM    230  CD1 LEU A  32     -20.382   7.978  12.323  1.00  0.00           C  \\nATOM    231  CD2 LEU A  32     -21.327   6.277  13.457  1.00  0.00           C  \\nATOM    232  N   TRP A  33     -25.611   6.151  11.693  1.00  0.00           N  \\nATOM    233  CA  TRP A  33     -27.186   6.529  11.189  1.00  0.00           C  \\nATOM    234  C   TRP A  33     -27.312   7.915  10.433  1.00  0.00           C  \\nATOM    235  O   TRP A  33     -27.312   7.852   9.299  1.00  0.00           O  \\nATOM    236  CB  TRP A  33     -28.446   6.403  12.071  1.00  0.00           C  \\nATOM    237  CG  TRP A  33     -28.572   5.773  13.205  1.00  0.00           C  \\nATOM    238  CD1 TRP A  33     -27.564   4.891  13.583  1.00  0.00           C  \\nATOM    239  CD2 TRP A  33     -29.328   5.647  14.213  1.00  0.00           C  \\nATOM    240  NE1 TRP A  33     -27.816   4.387  14.843  1.00  0.00           N  \\nATOM    241  CE2 TRP A  33     -29.013   4.828  15.158  1.00  0.00           C  \\nATOM    242  CE3 TRP A  33     -30.462   6.277  14.213  1.00  0.00           C  \\nATOM    243  CZ2 TRP A  33     -29.706   4.639  15.851  1.00  0.00           C  \\nATOM    244  CZ3 TRP A  33     -31.092   5.962  15.284  1.00  0.00           C  \\nATOM    245  CH2 TRP A  33     -30.714   5.143  15.914  1.00  0.00           C  \\nATOM    246  N   GLY A  34     -27.564   8.797  11.315  1.00  0.00           N  \\nATOM    247  CA  GLY A  34     -27.690  10.120  10.685  1.00  0.00           C  \\nATOM    248  C   GLY A  34     -26.934  11.191  11.315  1.00  0.00           C  \\nATOM    249  O   GLY A  34     -27.312  12.199  12.134  1.00  0.00           O  \\nATOM    250  N   LEU A  35     -25.548  11.191  11.252  1.00  0.00           N  \\nATOM    251  CA  LEU A  35     -24.666  12.199  11.945  1.00  0.00           C  \\nATOM    252  C   LEU A  35     -25.548  13.459  12.386  1.00  0.00           C  \\nATOM    253  O   LEU A  35     -25.800  13.963  13.457  1.00  0.00           O  \\nATOM    254  CB  LEU A  35     -23.154  12.640  11.567  1.00  0.00           C  \\nATOM    255  CG  LEU A  35     -22.335  11.506  11.189  1.00  0.00           C  \\nATOM    256  CD1 LEU A  35     -20.886  11.695  11.189  1.00  0.00           C  \\nATOM    257  CD2 LEU A  35     -22.650  10.183  11.819  1.00  0.00           C  \\nATOM    258  N   PHE A  36     -26.052  13.963  11.441  1.00  0.00           N  \\nATOM    259  CA  PHE A  36     -26.934  15.223  11.693  1.00  0.00           C  \\nATOM    260  C   PHE A  36     -28.131  15.223  12.701  1.00  0.00           C  \\nATOM    261  O   PHE A  36     -29.328  15.223  12.449  1.00  0.00           O  \\nATOM    262  CB  PHE A  36     -27.186  15.601  10.055  1.00  0.00           C  \\nATOM    263  CG  PHE A  36     -25.926  15.601   9.425  1.00  0.00           C  \\nATOM    264  CD1 PHE A  36     -25.674  15.727   8.102  1.00  0.00           C  \\nATOM    265  CD2 PHE A  36     -24.792  15.097   9.803  1.00  0.00           C  \\nATOM    266  CE1 PHE A  36     -24.540  15.475   7.409  1.00  0.00           C  \\nATOM    267  CE2 PHE A  36     -23.658  15.097   9.551  1.00  0.00           C  \\nATOM    268  CZ  PHE A  36     -23.406  15.097   8.291  1.00  0.00           C  \\nATOM    269  N   GLY A  37     -27.879  15.538  13.961  1.00  0.00           N  \\nATOM    270  CA  GLY A  37     -28.509  15.664  15.347  1.00  0.00           C  \\nATOM    271  C   GLY A  37     -28.005  14.971  16.355  1.00  0.00           C  \\nATOM    272  O   GLY A  37     -28.572  14.215  17.300  1.00  0.00           O  \\nATOM    273  N   ARG A  38     -26.682  15.097  16.859  1.00  0.00           N  \\nATOM    274  CA  ARG A  38     -26.556  14.341  18.119  1.00  0.00           C  \\nATOM    275  C   ARG A  38     -25.170  14.845  18.371  1.00  0.00           C  \\nATOM    276  O   ARG A  38     -24.414  15.349  17.741  1.00  0.00           O  \\nATOM    277  CB  ARG A  38     -26.178  13.081  17.741  1.00  0.00           C  \\nATOM    278  CG  ARG A  38     -25.863  12.703  16.355  1.00  0.00           C  \\nATOM    279  CD  ARG A  38     -25.422  11.191  16.040  1.00  0.00           C  \\nATOM    280  NE  ARG A  38     -24.666  10.561  16.985  1.00  0.00           N  \\nATOM    281  CZ  ARG A  38     -24.792   9.238  16.859  1.00  0.00           C  \\nATOM    282  NH1 ARG A  38     -24.351   8.545  17.489  1.00  0.00           N  \\nATOM    283  NH2 ARG A  38     -25.485   8.797  15.851  1.00  0.00           N  \\nATOM    284  N   LYS A  39     -25.044  14.593  19.757  1.00  0.00           N  \\nATOM    285  CA  LYS A  39     -24.036  15.160  20.639  1.00  0.00           C  \\nATOM    286  C   LYS A  39     -23.028  14.089  20.891  1.00  0.00           C  \\nATOM    287  O   LYS A  39     -23.406  13.207  20.009  1.00  0.00           O  \\nATOM    288  CB  LYS A  39     -24.540  15.601  22.151  1.00  0.00           C  \\nATOM    289  CG  LYS A  39     -24.414  15.475  23.663  1.00  0.00           C  \\nATOM    290  CD  LYS A  39     -25.170  16.231  24.923  1.00  0.00           C  \\nATOM    291  N   THR A  40     -22.020  13.837  21.521  1.00  0.00           N  \\nATOM    292  CA  THR A  40     -21.390  12.388  21.269  1.00  0.00           C  \\nATOM    293  C   THR A  40     -21.642  11.191  22.277  1.00  0.00           C  \\nATOM    294  O   THR A  40     -21.516  11.695  23.411  1.00  0.00           O  \\nATOM    295  CB  THR A  40     -19.752  12.577  21.269  1.00  0.00           C  \\nATOM    296  OG1 THR A  40     -19.374  11.884  19.883  1.00  0.00           O  \\nATOM    297  CG2 THR A  40     -19.626  13.837  20.765  1.00  0.00           C  \\nATOM    298  N   GLY A  41     -22.020  10.309  21.773  1.00  0.00           N  \\nATOM    299  CA  GLY A  41     -22.650   9.553  22.907  1.00  0.00           C  \\nATOM    300  C   GLY A  41     -23.910   8.797  22.403  1.00  0.00           C  \\nATOM    301  O   GLY A  41     -24.162   8.545  21.395  1.00  0.00           O  \\nATOM    302  N   GLN A  42     -24.918   8.419  23.033  1.00  0.00           N  \\nATOM    303  CA  GLN A  42     -26.241   7.915  23.033  1.00  0.00           C  \\nATOM    304  C   GLN A  42     -26.304   7.537  21.647  1.00  0.00           C  \\nATOM    305  O   GLN A  42     -26.556   8.545  20.765  1.00  0.00           O  \\nATOM    306  CB  GLN A  42     -27.438   8.545  23.537  1.00  0.00           C  \\nATOM    307  CG  GLN A  42     -28.068   7.789  24.419  1.00  0.00           C  \\nATOM    308  CD  GLN A  42     -29.076   7.096  24.230  1.00  0.00           C  \\nATOM    309  OE1 GLN A  42     -29.706   5.899  24.797  1.00  0.00           O  \\nATOM    310  NE2 GLN A  42     -29.706   7.411  22.781  1.00  0.00           N  \\nATOM    311  N   ALA A  43     -26.304   6.277  21.143  1.00  0.00           N  \\nATOM    312  CA  ALA A  43     -26.682   6.025  19.505  1.00  0.00           C  \\nATOM    313  C   ALA A  43     -27.186   4.639  19.442  1.00  0.00           C  \\nATOM    314  O   ALA A  43     -27.060   4.009  18.497  1.00  0.00           O  \\nATOM    315  CB  ALA A  43     -25.548   6.025  18.497  1.00  0.00           C  \\nATOM    316  N   GLU A  44     -27.438   3.883  20.450  1.00  0.00           N  \\nATOM    317  CA  GLU A  44     -28.131   2.497  20.261  1.00  0.00           C  \\nATOM    318  C   GLU A  44     -27.060   1.615  20.891  1.00  0.00           C  \\nATOM    319  O   GLU A  44     -26.556   0.481  20.324  1.00  0.00           O  \\nATOM    320  CB  GLU A  44     -28.635   2.371  18.749  1.00  0.00           C  \\nATOM    321  CG  GLU A  44     -30.210   0.993  18.749  1.00  0.00           C  \\nATOM    322  CD  GLU A  44     -30.840   2.875  19.379  1.00  0.00           C  \\nATOM    323  OE1 GLU A  44     -32.100   2.875  18.875  1.00  0.00           O  \\nATOM    324  OE2 GLU A  44     -30.084   4.135  19.505  1.00  0.00           O  \\nATOM    325  N   GLY A  45     -26.808   1.615  22.277  1.00  0.00           N  \\nATOM    326  CA  GLY A  45     -25.674   0.859  22.781  1.00  0.00           C  \\nATOM    327  C   GLY A  45     -24.162   1.363  22.907  1.00  0.00           C  \\nATOM    328  O   GLY A  45     -23.532   1.363  21.521  1.00  0.00           O  \\nATOM    329  N   TYR A  46     -23.532   1.993  23.789  1.00  0.00           N  \\nATOM    330  CA  TYR A  46     -22.146   2.875  23.537  1.00  0.00           C  \\nATOM    331  C   TYR A  46     -21.327   1.993  24.356  1.00  0.00           C  \\nATOM    332  O   TYR A  46     -21.327   0.733  24.167  1.00  0.00           O  \\nATOM    333  CB  TYR A  46     -22.524   4.198  23.159  1.00  0.00           C  \\nATOM    334  CG  TYR A  46     -22.146   5.143  22.151  1.00  0.00           C  \\nATOM    335  CD1 TYR A  46     -22.776   5.269  21.521  1.00  0.00           C  \\nATOM    336  CD2 TYR A  46     -20.886   5.647  21.647  1.00  0.00           C  \\nATOM    337  CE1 TYR A  46     -22.524   5.899  20.387  1.00  0.00           C  \\nATOM    338  CE2 TYR A  46     -20.382   6.277  20.513  1.00  0.00           C  \\nATOM    339  CZ  TYR A  46     -21.264   6.277  20.009  1.00  0.00           C  \\nATOM    340  OH  TYR A  46     -21.138   6.655  19.001  1.00  0.00           O  \\nATOM    341  N   SER A  47     -20.634   2.560  25.049  1.00  0.00           N  \\nATOM    342  CA  SER A  47     -19.374   1.741  25.679  1.00  0.00           C  \\nATOM    343  C   SER A  47     -18.114   2.623  25.679  1.00  0.00           C  \\nATOM    344  O   SER A  47     -18.114   3.379  24.797  1.00  0.00           O  \\nATOM    345  CB  SER A  47     -19.248   0.355  25.427  1.00  0.00           C  \\nATOM    346  OG  SER A  47     -18.870   0.481  23.789  1.00  0.00           O  \\nATOM    347  N   TYR A  48     -17.484   2.749  26.939  1.00  0.00           N  \\nATOM    348  CA  TYR A  48     -16.413   3.757  26.939  1.00  0.00           C  \\nATOM    349  C   TYR A  48     -16.602   4.891  27.695  1.00  0.00           C  \\nATOM    350  O   TYR A  48     -17.610   5.269  27.821  1.00  0.00           O  \\nATOM    351  CB  TYR A  48     -16.224   3.757  25.301  1.00  0.00           C  \\nATOM    352  CG  TYR A  48     -15.216   4.639  24.545  1.00  0.00           C  \\nATOM    353  CD1 TYR A  48     -14.082   5.143  24.797  1.00  0.00           C  \\nATOM    354  CD2 TYR A  48     -15.342   4.765  23.411  1.00  0.00           C  \\nATOM    355  CE1 TYR A  48     -13.326   6.025  24.167  1.00  0.00           C  \\nATOM    356  CE2 TYR A  48     -14.712   5.521  22.529  1.00  0.00           C  \\nATOM    357  CZ  TYR A  48     -13.704   6.088  23.033  1.00  0.00           C  \\nATOM    358  OH  TYR A  48     -13.074   6.907  22.151  1.00  0.00           O  \\nATOM    359  N   THR A  49     -15.594   5.647  28.199  1.00  0.00           N  \\nATOM    360  CA  THR A  49     -15.846   6.907  29.207  1.00  0.00           C  \\nATOM    361  C   THR A  49     -15.594   8.041  28.829  1.00  0.00           C  \\nATOM    362  O   THR A  49     -16.350   8.671  27.947  1.00  0.00           O  \\nATOM    363  CB  THR A  49     -15.531   6.403  30.845  1.00  0.00           C  \\nATOM    364  OG1 THR A  49     -14.208   5.899  30.845  1.00  0.00           O  \\nATOM    365  CG2 THR A  49     -16.224   5.395  31.223  1.00  0.00           C  \\nATOM    366  N   ASP A  50     -14.334   8.482  28.703  1.00  0.00           N  \\nATOM    367  CA  ASP A  50     -14.082   9.805  28.199  1.00  0.00           C  \\nATOM    368  C   ASP A  50     -13.704   9.679  26.813  1.00  0.00           C  \\nATOM    369  O   ASP A  50     -13.074   8.545  26.183  1.00  0.00           O  \\nATOM    370  CB  ASP A  50     -13.074  10.435  28.829  1.00  0.00           C  \\nATOM    371  CG  ASP A  50     -12.948  10.183  30.089  1.00  0.00           C  \\nATOM    372  OD1 ASP A  50     -12.066   9.049  30.593  1.00  0.00           O  \\nATOM    373  OD2 ASP A  50     -13.956  10.309  30.971  1.00  0.00           O  \\nATOM    374  N   ALA A  51     -13.830  10.309  25.805  1.00  0.00           N  \\nATOM    375  CA  ALA A  51     -13.704   9.931  24.419  1.00  0.00           C  \\nATOM    376  C   ALA A  51     -15.216   9.805  23.915  1.00  0.00           C  \\nATOM    377  O   ALA A  51     -16.098   9.301  24.167  1.00  0.00           O  \\nATOM    378  CB  ALA A  51     -13.200   8.671  24.041  1.00  0.00           C  \\nATOM    379  N   ASN A  52     -15.342  10.561  22.907  1.00  0.00           N  \\nATOM    380  CA  ASN A  52     -16.539  10.561  22.277  1.00  0.00           C  \\nATOM    381  C   ASN A  52     -17.232  11.380  22.907  1.00  0.00           C  \\nATOM    382  O   ASN A  52     -17.232  12.766  22.781  1.00  0.00           O  \\nATOM    383  CB  ASN A  52     -17.610   9.553  21.899  1.00  0.00           C  \\nATOM    384  CG  ASN A  52     -17.484   8.671  20.639  1.00  0.00           C  \\nATOM    385  OD1 ASN A  52     -16.980   7.789  20.135  1.00  0.00           O  \\nATOM    386  ND2 ASN A  52     -18.240   9.427  19.631  1.00  0.00           N  \\nATOM    387  N   LYS A  53     -17.862  10.939  24.041  1.00  0.00           N  \\nATOM    388  CA  LYS A  53     -18.240  11.947  25.301  1.00  0.00           C  \\nATOM    389  C   LYS A  53     -17.106  12.703  25.805  1.00  0.00           C  \\nATOM    390  O   LYS A  53     -16.476  13.711  25.175  1.00  0.00           O  \\nATOM    391  CB  LYS A  53     -18.870  11.317  26.561  1.00  0.00           C  \\nATOM    392  CG  LYS A  53     -20.130  10.435  26.057  1.00  0.00           C  \\nATOM    393  CD  LYS A  53     -20.634   9.301  27.065  1.00  0.00           C  \\nATOM    394  CE  LYS A  53     -22.146   8.671  26.372  1.00  0.00           C  \\nATOM    395  NZ  LYS A  53     -22.524   7.663  27.191  1.00  0.00           N  \\nATOM    396  N   SER A  54     -16.602  12.325  27.065  1.00  0.00           N  \\nATOM    397  CA  SER A  54     -15.594  13.081  27.821  1.00  0.00           C  \\nATOM    398  C   SER A  54     -15.594  14.593  27.443  1.00  0.00           C  \\nATOM    399  O   SER A  54     -16.476  15.223  27.695  1.00  0.00           O  \\nATOM    400  CB  SER A  54     -14.460  12.703  27.569  1.00  0.00           C  \\nATOM    401  OG  SER A  54     -13.452  12.955  28.703  1.00  0.00           O  \\nATOM    402  N   LYS A  55     -14.838  15.223  26.813  1.00  0.00           N  \\nATOM    403  CA  LYS A  55     -14.964  16.420  26.057  1.00  0.00           C  \\nATOM    404  C   LYS A  55     -16.602  16.987  25.931  1.00  0.00           C  \\nATOM    405  O   LYS A  55     -16.854  18.121  26.183  1.00  0.00           O  \\nATOM    406  CB  LYS A  55     -14.208  16.609  24.734  1.00  0.00           C  \\nATOM    407  CG  LYS A  55     -14.712  17.743  23.600  1.00  0.00           C  \\nATOM    408  CD  LYS A  55     -13.452  17.743  22.844  1.00  0.00           C  \\nATOM    409  CE  LYS A  55     -13.578  17.995  21.143  1.00  0.00           C  \\nATOM    410  NZ  LYS A  55     -12.444  17.743  20.135  1.00  0.00           N  \\nATOM    411  N   GLY A  56     -17.232  16.042  25.679  1.00  0.00           N  \\nATOM    412  CA  GLY A  56     -18.870  16.042  25.301  1.00  0.00           C  \\nATOM    413  C   GLY A  56     -19.122  17.239  24.293  1.00  0.00           C  \\nATOM    414  O   GLY A  56     -18.492  18.247  23.852  1.00  0.00           O  \\nATOM    415  N   ILE A  57     -20.130  17.113  23.285  1.00  0.00           N  \\nATOM    416  CA  ILE A  57     -20.004  17.869  22.025  1.00  0.00           C  \\nATOM    417  C   ILE A  57     -21.264  18.121  21.143  1.00  0.00           C  \\nATOM    418  O   ILE A  57     -22.398  18.499  21.395  1.00  0.00           O  \\nATOM    419  CB  ILE A  57     -18.996  17.491  21.395  1.00  0.00           C  \\nATOM    420  CG1 ILE A  57     -19.374  16.357  20.513  1.00  0.00           C  \\nATOM    421  CG2 ILE A  57     -17.862  17.050  22.025  1.00  0.00           C  \\nATOM    422  CD1 ILE A  57     -19.122  16.609  19.127  1.00  0.00           C  \\nATOM    423  N   VAL A  58     -21.264  17.743  19.631  1.00  0.00           N  \\nATOM    424  CA  VAL A  58     -22.524  17.995  19.001  1.00  0.00           C  \\nATOM    425  C   VAL A  58     -22.020  17.239  17.615  1.00  0.00           C  \\nATOM    426  O   VAL A  58     -21.642  18.121  16.985  1.00  0.00           O  \\nATOM    427  CB  VAL A  58     -23.280  19.129  19.127  1.00  0.00           C  \\nATOM    428  CG1 VAL A  58     -22.398  20.263  18.749  1.00  0.00           C  \\nATOM    429  CG2 VAL A  58     -24.540  19.192  18.371  1.00  0.00           C  \\nATOM    430  N   TRP A  59     -22.398  16.231  16.859  1.00  0.00           N  \\nATOM    431  CA  TRP A  59     -22.020  16.105  15.221  1.00  0.00           C  \\nATOM    432  C   TRP A  59     -22.524  17.239  14.213  1.00  0.00           C  \\nATOM    433  O   TRP A  59     -23.784  17.365  14.339  1.00  0.00           O  \\nATOM    434  CB  TRP A  59     -22.146  14.845  14.717  1.00  0.00           C  \\nATOM    435  CG  TRP A  59     -21.264  13.963  15.347  1.00  0.00           C  \\nATOM    436  CD1 TRP A  59     -21.390  12.703  15.851  1.00  0.00           C  \\nATOM    437  CD2 TRP A  59     -19.878  14.152  15.536  1.00  0.00           C  \\nATOM    438  NE1 TRP A  59     -20.382  12.451  16.607  1.00  0.00           N  \\nATOM    439  CE2 TRP A  59     -19.500  13.207  15.977  1.00  0.00           C  \\nATOM    440  CE3 TRP A  59     -19.248  15.160  15.095  1.00  0.00           C  \\nATOM    441  CZ2 TRP A  59     -18.114  13.459  16.607  1.00  0.00           C  \\nATOM    442  CZ3 TRP A  59     -18.051  15.223  15.473  1.00  0.00           C  \\nATOM    443  CH2 TRP A  59     -17.484  14.467  16.229  1.00  0.00           C  \\nATOM    444  N   ASN A  60     -21.768  17.869  13.772  1.00  0.00           N  \\nATOM    445  CA  ASN A  60     -22.398  19.129  12.953  1.00  0.00           C  \\nATOM    446  C   ASN A  60     -21.642  19.255  11.441  1.00  0.00           C  \\nATOM    447  O   ASN A  60     -20.634  18.751  11.189  1.00  0.00           O  \\nATOM    448  CB  ASN A  60     -22.524  20.515  13.583  1.00  0.00           C  \\nATOM    449  CG  ASN A  60     -22.776  21.649  12.701  1.00  0.00           C  \\nATOM    450  OD1 ASN A  60     -24.036  21.712  11.945  1.00  0.00           O  \\nATOM    451  ND2 ASN A  60     -21.894  22.783  12.701  1.00  0.00           N  \\nATOM    452  N   GLU A  61     -22.020  20.137  10.811  1.00  0.00           N  \\nATOM    453  CA  GLU A  61     -21.138  20.263   9.425  1.00  0.00           C  \\nATOM    454  C   GLU A  61     -19.626  20.389  10.181  1.00  0.00           C  \\nATOM    455  O   GLU A  61     -18.744  19.507   9.803  1.00  0.00           O  \\nATOM    456  CB  GLU A  61     -21.516  21.523   8.669  1.00  0.00           C  \\nATOM    457  CG  GLU A  61     -22.020  21.523   7.409  1.00  0.00           C  \\nATOM    458  CD  GLU A  61     -23.154  20.767   7.283  1.00  0.00           C  \\nATOM    459  OE1 GLU A  61     -24.540  20.767   7.535  1.00  0.00           O  \\nATOM    460  OE2 GLU A  61     -22.902  19.381   6.653  1.00  0.00           O  \\nATOM    461  N   ASN A  62     -19.248  21.397  10.685  1.00  0.00           N  \\nATOM    462  CA  ASN A  62     -17.736  21.586  11.441  1.00  0.00           C  \\nATOM    463  C   ASN A  62     -17.106  20.137  11.945  1.00  0.00           C  \\nATOM    464  O   ASN A  62     -15.972  20.515  12.071  1.00  0.00           O  \\nATOM    465  CB  ASN A  62     -17.736  22.783  11.945  1.00  0.00           C  \\nATOM    466  CG  ASN A  62     -17.610  24.043  11.441  1.00  0.00           C  \\nATOM    467  OD1 ASN A  62     -18.744  24.547  10.559  1.00  0.00           O  \\nATOM    468  ND2 ASN A  62     -16.539  24.799  11.063  1.00  0.00           N  \\nATOM    469  N   THR A  63     -17.610  20.263  13.268  1.00  0.00           N  \\nATOM    470  CA  THR A  63     -17.043  19.255  14.087  1.00  0.00           C  \\nATOM    471  C   THR A  63     -16.602  17.869  13.331  1.00  0.00           C  \\nATOM    472  O   THR A  63     -15.720  17.239  13.331  1.00  0.00           O  \\nATOM    473  CB  THR A  63     -18.051  18.751  15.095  1.00  0.00           C  \\nATOM    474  OG1 THR A  63     -17.358  18.499  15.851  1.00  0.00           O  \\nATOM    475  CG2 THR A  63     -18.492  17.365  14.528  1.00  0.00           C  \\nATOM    476  N   LEU A  64     -17.358  17.617  12.197  1.00  0.00           N  \\nATOM    477  CA  LEU A  64     -16.854  16.357  11.441  1.00  0.00           C  \\nATOM    478  C   LEU A  64     -15.468  16.861  10.685  1.00  0.00           C  \\nATOM    479  O   LEU A  64     -14.838  16.231  10.433  1.00  0.00           O  \\nATOM    480  CB  LEU A  64     -18.114  15.979  10.055  1.00  0.00           C  \\nATOM    481  CG  LEU A  64     -18.744  14.719  10.433  1.00  0.00           C  \\nATOM    482  CD1 LEU A  64     -19.122  14.719  11.567  1.00  0.00           C  \\nATOM    483  CD2 LEU A  64     -19.878  14.593   9.803  1.00  0.00           C  \\nATOM    484  N   MET A  65     -15.468  17.995  10.181  1.00  0.00           N  \\nATOM    485  CA  MET A  65     -13.956  18.499   9.803  1.00  0.00           C  \\nATOM    486  C   MET A  65     -13.200  18.373  11.189  1.00  0.00           C  \\nATOM    487  O   MET A  65     -12.192  17.869  11.189  1.00  0.00           O  \\nATOM    488  CB  MET A  65     -13.956  19.885   9.425  1.00  0.00           C  \\nATOM    489  CG  MET A  65     -13.452  20.137   7.913  1.00  0.00           C  \\nATOM    490  SD  MET A  65     -13.704  18.625   7.283  1.00  0.00           S  \\nATOM    491  CE  MET A  65     -13.452  19.192   5.519  1.00  0.00           C  \\nATOM    492  N   GLU A  66     -13.578  18.751  12.197  1.00  0.00           N  \\nATOM    493  CA  GLU A  66     -13.074  18.751  13.457  1.00  0.00           C  \\nATOM    494  C   GLU A  66     -12.444  17.239  13.646  1.00  0.00           C  \\nATOM    495  O   GLU A  66     -11.310  17.113  13.835  1.00  0.00           O  \\nATOM    496  CB  GLU A  66     -13.830  19.381  14.717  1.00  0.00           C  \\nATOM    497  CG  GLU A  66     -13.578  20.767  14.969  1.00  0.00           C  \\nATOM    498  CD  GLU A  66     -12.318  21.271  15.851  1.00  0.00           C  \\nATOM    499  OE1 GLU A  66     -12.192  22.657  16.103  1.00  0.00           O  \\nATOM    500  OE2 GLU A  66     -11.310  20.641  16.355  1.00  0.00           O  \\nATOM    501  N   TYR A  67     -13.263  16.483  13.457  1.00  0.00           N  \\nATOM    502  CA  TYR A  67     -12.822  14.971  13.457  1.00  0.00           C  \\nATOM    503  C   TYR A  67     -12.192  14.467  12.323  1.00  0.00           C  \\nATOM    504  O   TYR A  67     -11.184  14.089  11.945  1.00  0.00           O  \\nATOM    505  CB  TYR A  67     -14.082  14.341  13.835  1.00  0.00           C  \\nATOM    506  CG  TYR A  67     -14.082  12.955  14.465  1.00  0.00           C  \\nATOM    507  CD1 TYR A  67     -14.838  11.821  14.465  1.00  0.00           C  \\nATOM    508  CD2 TYR A  67     -13.326  13.081  15.599  1.00  0.00           C  \\nATOM    509  CE1 TYR A  67     -14.649  10.876  15.221  1.00  0.00           C  \\nATOM    510  CE2 TYR A  67     -13.200  11.947  16.103  1.00  0.00           C  \\nATOM    511  CZ  TYR A  67     -13.830  10.876  15.977  1.00  0.00           C  \\nATOM    512  OH  TYR A  67     -13.704   9.805  16.607  1.00  0.00           O  \\nATOM    513  N   LEU A  68     -12.822  14.719  11.315  1.00  0.00           N  \\nATOM    514  CA  LEU A  68     -12.192  14.341  10.055  1.00  0.00           C  \\nATOM    515  C   LEU A  68     -10.806  14.719  10.307  1.00  0.00           C  \\nATOM    516  O   LEU A  68      -9.798  13.963  10.307  1.00  0.00           O  \\nATOM    517  CB  LEU A  68     -13.074  14.845   9.173  1.00  0.00           C  \\nATOM    518  CG  LEU A  68     -14.334  13.837   8.795  1.00  0.00           C  \\nATOM    519  CD1 LEU A  68     -15.216  14.467   7.787  1.00  0.00           C  \\nATOM    520  CD2 LEU A  68     -13.704  12.829   7.787  1.00  0.00           C  \\nATOM    521  N   GLU A  69     -10.428  15.853  10.685  1.00  0.00           N  \\nATOM    522  CA  GLU A  69      -8.916  16.231  10.937  1.00  0.00           C  \\nATOM    523  C   GLU A  69      -8.097  14.971  11.441  1.00  0.00           C  \\nATOM    524  O   GLU A  69      -7.278  14.593  10.685  1.00  0.00           O  \\nATOM    525  CB  GLU A  69      -9.042  17.365  11.693  1.00  0.00           C  \\nATOM    526  CG  GLU A  69      -8.034  18.373  11.819  1.00  0.00           C  \\nATOM    527  CD  GLU A  69      -6.900  18.373  12.953  1.00  0.00           C  \\nATOM    528  OE1 GLU A  69      -5.388  18.499  12.827  1.00  0.00           O  \\nATOM    529  OE2 GLU A  69      -7.278  17.869  14.087  1.00  0.00           O  \\nATOM    530  N   ASN A  70      -8.097  14.845  12.575  1.00  0.00           N  \\nATOM    531  CA  ASN A  70      -7.278  13.585  12.827  1.00  0.00           C  \\nATOM    532  C   ASN A  70      -7.404  13.459  14.339  1.00  0.00           C  \\nATOM    533  O   ASN A  70      -7.152  14.215  15.221  1.00  0.00           O  \\nATOM    534  CB  ASN A  70      -5.892  13.837  12.449  1.00  0.00           C  \\nATOM    535  CG  ASN A  70      -4.758  14.089  13.457  1.00  0.00           C  \\nATOM    536  OD1 ASN A  70      -3.624  15.097  13.457  1.00  0.00           O  \\nATOM    537  ND2 ASN A  70      -4.758  13.333  14.717  1.00  0.00           N  \\nATOM    538  N   PRO A  71      -7.908  12.199  14.780  1.00  0.00           N  \\nATOM    539  CA  PRO A  71      -8.475  11.758  15.725  1.00  0.00           C  \\nATOM    540  C   PRO A  71      -7.593  11.758  16.859  1.00  0.00           C  \\nATOM    541  O   PRO A  71      -7.782  11.317  17.993  1.00  0.00           O  \\nATOM    542  CB  PRO A  71      -9.042  10.561  15.536  1.00  0.00           C  \\nATOM    543  CG  PRO A  71      -8.916  10.309  14.213  1.00  0.00           C  \\nATOM    544  CD  PRO A  71      -8.160  11.191  13.583  1.00  0.00           C  \\nATOM    545  N   LYS A  72      -6.522  12.325  16.733  1.00  0.00           N  \\nATOM    546  CA  LYS A  72      -5.388  12.703  17.615  1.00  0.00           C  \\nATOM    547  C   LYS A  72      -5.451  14.215  17.993  1.00  0.00           C  \\nATOM    548  O   LYS A  72      -5.262  14.971  17.111  1.00  0.00           O  \\nATOM    549  CB  LYS A  72      -3.876  12.451  17.111  1.00  0.00           C  \\nATOM    550  CG  LYS A  72      -3.624  11.065  17.363  1.00  0.00           C  \\nATOM    551  CD  LYS A  72      -2.112  10.435  17.237  1.00  0.00           C  \\nATOM    552  CE  LYS A  72      -1.860   8.923  17.741  1.00  0.00           C  \\nATOM    553  NZ  LYS A  72      -0.222   8.545  17.741  1.00  0.00           N  \\nATOM    554  N   LYS A  73      -6.081  14.719  19.127  1.00  0.00           N  \\nATOM    555  CA  LYS A  73      -6.396  16.231  19.127  1.00  0.00           C  \\nATOM    556  C   LYS A  73      -7.782  16.357  18.623  1.00  0.00           C  \\nATOM    557  O   LYS A  73      -8.160  16.231  17.363  1.00  0.00           O  \\nATOM    558  CB  LYS A  73      -5.640  17.113  17.993  1.00  0.00           C  \\nATOM    559  CG  LYS A  73      -4.758  18.247  18.623  1.00  0.00           C  \\nATOM    560  CD  LYS A  73      -3.876  19.003  17.993  1.00  0.00           C  \\nATOM    561  CE  LYS A  73      -2.616  20.389  18.812  1.00  0.00           C  \\nATOM    562  NZ  LYS A  73      -1.986  21.145  18.056  1.00  0.00           N  \\nATOM    563  N   TYR A  74      -8.664  16.231  19.631  1.00  0.00           N  \\nATOM    564  CA  TYR A  74     -10.050  16.105  19.379  1.00  0.00           C  \\nATOM    565  C   TYR A  74     -10.806  14.971  19.883  1.00  0.00           C  \\nATOM    566  O   TYR A  74     -10.554  13.963  19.379  1.00  0.00           O  \\nATOM    567  CB  TYR A  74     -10.554  16.105  17.741  1.00  0.00           C  \\nATOM    568  CG  TYR A  74     -11.814  16.735  17.363  1.00  0.00           C  \\nATOM    569  CD1 TYR A  74     -12.948  16.105  17.237  1.00  0.00           C  \\nATOM    570  CD2 TYR A  74     -11.940  17.995  16.985  1.00  0.00           C  \\nATOM    571  CE1 TYR A  74     -14.145  16.609  16.859  1.00  0.00           C  \\nATOM    572  CE2 TYR A  74     -13.074  18.625  16.607  1.00  0.00           C  \\nATOM    573  CZ  TYR A  74     -14.082  17.869  16.607  1.00  0.00           C  \\nATOM    574  OH  TYR A  74     -15.216  18.373  16.355  1.00  0.00           O  \\nATOM    575  N   ILE A  75     -11.310  15.097  21.017  1.00  0.00           N  \\nATOM    576  CA  ILE A  75     -11.940  14.089  21.647  1.00  0.00           C  \\nATOM    577  C   ILE A  75     -10.806  13.081  21.773  1.00  0.00           C  \\nATOM    578  O   ILE A  75     -10.932  11.821  21.521  1.00  0.00           O  \\nATOM    579  CB  ILE A  75     -13.200  13.585  21.143  1.00  0.00           C  \\nATOM    580  CG1 ILE A  75     -13.641  14.089  19.631  1.00  0.00           C  \\nATOM    581  CG2 ILE A  75     -14.271  13.585  22.025  1.00  0.00           C  \\nATOM    582  CD1 ILE A  75     -14.712  13.459  19.253  1.00  0.00           C  \\nATOM    583  N   PRO A  76      -9.420  13.585  21.899  1.00  0.00           N  \\nATOM    584  CA  PRO A  76      -8.160  12.955  21.647  1.00  0.00           C  \\nATOM    585  C   PRO A  76      -8.160  11.695  22.529  1.00  0.00           C  \\nATOM    586  O   PRO A  76      -8.538  11.695  23.789  1.00  0.00           O  \\nATOM    587  CB  PRO A  76      -6.900  13.837  21.836  1.00  0.00           C  \\nATOM    588  CG  PRO A  76      -7.152  14.845  22.277  1.00  0.00           C  \\nATOM    589  CD  PRO A  76      -8.538  14.845  22.151  1.00  0.00           C  \\nATOM    590  N   GLY A  77      -8.097  10.435  22.088  1.00  0.00           N  \\nATOM    591  CA  GLY A  77      -8.286   9.175  22.907  1.00  0.00           C  \\nATOM    592  C   GLY A  77      -9.546   8.167  22.529  1.00  0.00           C  \\nATOM    593  O   GLY A  77      -9.798   7.159  22.781  1.00  0.00           O  \\nATOM    594  N   THR A  78     -10.050   8.545  21.143  1.00  0.00           N  \\nATOM    595  CA  THR A  78     -11.310   7.789  20.639  1.00  0.00           C  \\nATOM    596  C   THR A  78     -10.995   6.529  20.513  1.00  0.00           C  \\nATOM    597  O   THR A  78      -9.798   6.151  20.009  1.00  0.00           O  \\nATOM    598  CB  THR A  78     -11.625   8.167  19.253  1.00  0.00           C  \\nATOM    599  OG1 THR A  78     -13.074   8.167  19.127  1.00  0.00           O  \\nATOM    600  CG2 THR A  78     -11.184   9.553  18.875  1.00  0.00           C  \\nATOM    601  N   LYS A  79     -11.562   5.521  20.765  1.00  0.00           N  \\nATOM    602  CA  LYS A  79     -11.058   4.072  20.639  1.00  0.00           C  \\nATOM    603  C   LYS A  79     -11.058   3.631  19.001  1.00  0.00           C  \\nATOM    604  O   LYS A  79     -10.932   2.497  18.371  1.00  0.00           O  \\nATOM    605  CB  LYS A  79     -11.562   3.064  21.395  1.00  0.00           C  \\nATOM    606  CG  LYS A  79     -12.570   2.560  20.513  1.00  0.00           C  \\nATOM    607  CD  LYS A  79     -13.704   1.615  20.891  1.00  0.00           C  \\nATOM    608  CE  LYS A  79     -14.838   1.363  20.135  1.00  0.00           C  \\nATOM    609  NZ  LYS A  79     -15.972   0.859  20.513  1.00  0.00           N  \\nATOM    610  N   MET A  80     -11.310   4.639  18.119  1.00  0.00           N  \\nATOM    611  CA  MET A  80     -11.436   4.135  16.607  1.00  0.00           C  \\nATOM    612  C   MET A  80     -10.050   4.198  16.103  1.00  0.00           C  \\nATOM    613  O   MET A  80      -9.294   5.017  16.103  1.00  0.00           O  \\nATOM    614  CB  MET A  80     -12.444   5.017  15.851  1.00  0.00           C  \\nATOM    615  CG  MET A  80     -11.940   6.088  14.969  1.00  0.00           C  \\nATOM    616  SD  MET A  80     -13.074   6.781  13.457  1.00  0.00           S  \\nATOM    617  CE  MET A  80     -12.192   7.852  13.079  1.00  0.00           C  \\nATOM    618  N   ILE A  81      -9.672   3.127  15.725  1.00  0.00           N  \\nATOM    619  CA  ILE A  81      -8.160   3.127  15.095  1.00  0.00           C  \\nATOM    620  C   ILE A  81      -7.908   4.009  13.835  1.00  0.00           C  \\nATOM    621  O   ILE A  81      -8.034   3.505  12.323  1.00  0.00           O  \\nATOM    622  CB  ILE A  81      -7.656   1.867  15.284  1.00  0.00           C  \\nATOM    623  CG1 ILE A  81      -6.522   1.867  16.103  1.00  0.00           C  \\nATOM    624  CG2 ILE A  81      -7.026   1.237  13.835  1.00  0.00           C  \\nATOM    625  CD1 ILE A  81      -6.900   2.245  17.615  1.00  0.00           C  \\nATOM    626  N   PHE A  82      -7.530   5.143  13.961  1.00  0.00           N  \\nATOM    627  CA  PHE A  82      -7.404   5.899  12.449  1.00  0.00           C  \\nATOM    628  C   PHE A  82      -6.144   6.529  12.197  1.00  0.00           C  \\nATOM    629  O   PHE A  82      -5.766   7.411  12.953  1.00  0.00           O  \\nATOM    630  CB  PHE A  82      -8.538   6.655  12.449  1.00  0.00           C  \\nATOM    631  CG  PHE A  82      -8.916   7.159  10.937  1.00  0.00           C  \\nATOM    632  CD1 PHE A  82      -8.916   8.293  10.433  1.00  0.00           C  \\nATOM    633  CD2 PHE A  82      -9.042   6.529  10.055  1.00  0.00           C  \\nATOM    634  CE1 PHE A  82      -9.042   8.671   9.173  1.00  0.00           C  \\nATOM    635  CE2 PHE A  82      -9.042   6.781   8.669  1.00  0.00           C  \\nATOM    636  CZ  PHE A  82      -9.042   7.852   8.291  1.00  0.00           C  \\nATOM    637  N   ALA A  83      -5.640   6.529  11.189  1.00  0.00           N  \\nATOM    638  CA  ALA A  83      -4.506   7.285  10.937  1.00  0.00           C  \\nATOM    639  C   ALA A  83      -4.758   8.797  10.811  1.00  0.00           C  \\nATOM    640  O   ALA A  83      -4.884   9.679  11.441  1.00  0.00           O  \\nATOM    641  CB  ALA A  83      -3.876   6.907   9.173  1.00  0.00           C  \\nATOM    642  N   GLY A  84      -5.136   9.301   9.677  1.00  0.00           N  \\nATOM    643  CA  GLY A  84      -5.325  10.687   9.299  1.00  0.00           C  \\nATOM    644  C   GLY A  84      -6.396  11.317   8.543  1.00  0.00           C  \\nATOM    645  O   GLY A  84      -7.467  11.569   8.921  1.00  0.00           O  \\nATOM    646  N   ILE A  85      -6.207  11.569   7.283  1.00  0.00           N  \\nATOM    647  CA  ILE A  85      -7.152  12.451   6.653  1.00  0.00           C  \\nATOM    648  C   ILE A  85      -6.774  13.963   6.905  1.00  0.00           C  \\nATOM    649  O   ILE A  85      -7.656  14.719   7.220  1.00  0.00           O  \\nATOM    650  CB  ILE A  85      -8.664  12.199   7.031  1.00  0.00           C  \\nATOM    651  CG1 ILE A  85      -9.168  10.813   6.149  1.00  0.00           C  \\nATOM    652  CG2 ILE A  85      -9.420  13.333   6.401  1.00  0.00           C  \\nATOM    653  CD1 ILE A  85     -10.176  11.191   5.204  1.00  0.00           C  \\nATOM    654  N   LYS A  86      -5.640  14.467   6.527  1.00  0.00           N  \\nATOM    655  CA  LYS A  86      -5.325  15.727   7.157  1.00  0.00           C  \\nATOM    656  C   LYS A  86      -5.136  16.546   5.771  1.00  0.00           C  \\nATOM    657  O   LYS A  86      -4.254  17.365   5.267  1.00  0.00           O  \\nATOM    658  CB  LYS A  86      -3.876  15.727   7.913  1.00  0.00           C  \\nATOM    659  CG  LYS A  86      -3.120  14.467   7.535  1.00  0.00           C  \\nATOM    660  CD  LYS A  86      -1.356  14.215   7.787  1.00  0.00           C  \\nATOM    661  CE  LYS A  86      -0.852  12.955   6.653  1.00  0.00           C  \\nATOM    662  NZ  LYS A  86       0.660  12.577   7.157  1.00  0.00           N  \\nATOM    663  N   LYS A  87      -5.892  16.357   5.015  1.00  0.00           N  \\nATOM    664  CA  LYS A  87      -5.892  17.239   3.755  1.00  0.00           C  \\nATOM    665  C   LYS A  87      -7.278  17.869   3.377  1.00  0.00           C  \\nATOM    666  O   LYS A  87      -8.160  17.239   2.873  1.00  0.00           O  \\nATOM    667  CB  LYS A  87      -5.514  16.294   2.558  1.00  0.00           C  \\nATOM    668  CG  LYS A  87      -4.632  16.735   1.235  1.00  0.00           C  \\nATOM    669  CD  LYS A  87      -4.254  15.727   0.353  1.00  0.00           C  \\nATOM    670  CE  LYS A  87      -3.120  16.231  -0.403  1.00  0.00           C  \\nATOM    671  NZ  LYS A  87      -2.868  15.727  -1.915  1.00  0.00           N  \\nATOM    672  N   LYS A  88      -7.278  19.129   3.251  1.00  0.00           N  \\nATOM    673  CA  LYS A  88      -8.412  19.885   2.621  1.00  0.00           C  \\nATOM    674  C   LYS A  88      -9.168  18.877   1.613  1.00  0.00           C  \\nATOM    675  O   LYS A  88     -10.428  18.499   1.613  1.00  0.00           O  \\nATOM    676  CB  LYS A  88      -7.782  21.019   1.991  1.00  0.00           C  \\nATOM    677  CG  LYS A  88      -6.270  20.767   1.865  1.00  0.00           C  \\nATOM    678  CD  LYS A  88      -5.514  21.271   0.605  1.00  0.00           C  \\nATOM    679  CE  LYS A  88      -5.766  20.767  -0.403  1.00  0.00           C  \\nATOM    680  NZ  LYS A  88      -5.388  21.523  -1.663  1.00  0.00           N  \\nATOM    681  N   GLY A  89      -8.538  18.499   0.353  1.00  0.00           N  \\nATOM    682  CA  GLY A  89      -9.168  17.491  -0.781  1.00  0.00           C  \\nATOM    683  C   GLY A  89     -10.050  16.357  -0.025  1.00  0.00           C  \\nATOM    684  O   GLY A  89     -11.184  16.357  -0.025  1.00  0.00           O  \\nATOM    685  N   GLU A  90      -9.546  15.475   0.479  1.00  0.00           N  \\nATOM    686  CA  GLU A  90     -10.050  14.089   1.109  1.00  0.00           C  \\nATOM    687  C   GLU A  90     -10.932  14.593   2.369  1.00  0.00           C  \\nATOM    688  O   GLU A  90     -12.192  14.341   2.621  1.00  0.00           O  \\nATOM    689  CB  GLU A  90      -9.042  13.207   1.613  1.00  0.00           C  \\nATOM    690  CG  GLU A  90      -8.286  12.577   0.605  1.00  0.00           C  \\nATOM    691  CD  GLU A  90      -6.900  11.821   0.857  1.00  0.00           C  \\nATOM    692  OE1 GLU A  90      -6.018  11.443  -0.277  1.00  0.00           O  \\nATOM    693  OE2 GLU A  90      -6.270  11.569   2.243  1.00  0.00           O  \\nATOM    694  N   ARG A  91     -10.554  15.475   2.999  1.00  0.00           N  \\nATOM    695  CA  ARG A  91     -11.184  15.916   4.511  1.00  0.00           C  \\nATOM    696  C   ARG A  91     -12.570  16.672   3.881  1.00  0.00           C  \\nATOM    697  O   ARG A  91     -13.578  16.357   4.133  1.00  0.00           O  \\nATOM    698  CB  ARG A  91     -10.302  16.987   5.141  1.00  0.00           C  \\nATOM    699  CG  ARG A  91     -10.554  17.113   6.527  1.00  0.00           C  \\nATOM    700  CD  ARG A  91      -9.420  18.121   7.283  1.00  0.00           C  \\nATOM    701  NE  ARG A  91      -9.294  19.381   6.527  1.00  0.00           N  \\nATOM    702  CZ  ARG A  91      -7.908  20.137   6.464  1.00  0.00           C  \\nATOM    703  NH1 ARG A  91      -6.900  19.507   7.535  1.00  0.00           N  \\nATOM    704  NH2 ARG A  91      -7.530  21.145   5.519  1.00  0.00           N  \\nATOM    705  N   GLN A  92     -12.444  17.743   3.251  1.00  0.00           N  \\nATOM    706  CA  GLN A  92     -13.830  18.373   2.369  1.00  0.00           C  \\nATOM    707  C   GLN A  92     -14.586  17.113   1.991  1.00  0.00           C  \\nATOM    708  O   GLN A  92     -15.720  16.735   2.243  1.00  0.00           O  \\nATOM    709  CB  GLN A  92     -13.704  19.633   1.739  1.00  0.00           C  \\nATOM    710  CG  GLN A  92     -13.830  19.255   0.164  1.00  0.00           C  \\nATOM    711  CD  GLN A  92     -13.200  20.326  -0.907  1.00  0.00           C  \\nATOM    712  OE1 GLN A  92     -11.688  19.885  -0.403  1.00  0.00           O  \\nATOM    713  NE2 GLN A  92     -13.452  21.460  -1.411  1.00  0.00           N  \\nATOM    714  N   ASP A  93     -13.956  16.357   0.983  1.00  0.00           N  \\nATOM    715  CA  ASP A  93     -14.838  15.223   0.605  1.00  0.00           C  \\nATOM    716  C   ASP A  93     -15.342  14.341   1.613  1.00  0.00           C  \\nATOM    717  O   ASP A  93     -16.602  13.837   1.487  1.00  0.00           O  \\nATOM    718  CB  ASP A  93     -13.830  14.593  -0.529  1.00  0.00           C  \\nATOM    719  CG  ASP A  93     -13.704  14.971  -1.915  1.00  0.00           C  \\nATOM    720  OD1 ASP A  93     -12.444  15.727  -2.419  1.00  0.00           O  \\nATOM    721  OD2 ASP A  93     -14.838  15.223  -2.797  1.00  0.00           O  \\nATOM    722  N   LEU A  94     -14.838  13.963   2.495  1.00  0.00           N  \\nATOM    723  CA  LEU A  94     -15.531  13.207   3.503  1.00  0.00           C  \\nATOM    724  C   LEU A  94     -16.728  13.963   3.881  1.00  0.00           C  \\nATOM    725  O   LEU A  94     -17.925  13.459   3.881  1.00  0.00           O  \\nATOM    726  CB  LEU A  94     -14.586  13.018   4.763  1.00  0.00           C  \\nATOM    727  CG  LEU A  94     -14.586  11.443   5.267  1.00  0.00           C  \\nATOM    728  CD1 LEU A  94     -15.783  10.813   4.637  1.00  0.00           C  \\nATOM    729  CD2 LEU A  94     -13.578  10.561   4.889  1.00  0.00           C  \\nATOM    730  N   VAL A  95     -16.539  15.097   4.637  1.00  0.00           N  \\nATOM    731  CA  VAL A  95     -17.736  15.979   5.141  1.00  0.00           C  \\nATOM    732  C   VAL A  95     -18.744  15.853   3.881  1.00  0.00           C  \\nATOM    733  O   VAL A  95     -20.004  15.601   4.322  1.00  0.00           O  \\nATOM    734  CB  VAL A  95     -17.358  17.365   5.393  1.00  0.00           C  \\nATOM    735  CG1 VAL A  95     -18.492  18.184   5.519  1.00  0.00           C  \\nATOM    736  CG2 VAL A  95     -16.476  17.491   6.401  1.00  0.00           C  \\nATOM    737  N   ALA A  96     -18.492  15.979   2.621  1.00  0.00           N  \\nATOM    738  CA  ALA A  96     -19.752  15.601   1.739  1.00  0.00           C  \\nATOM    739  C   ALA A  96     -20.130  14.215   1.802  1.00  0.00           C  \\nATOM    740  O   ALA A  96     -21.390  13.837   1.802  1.00  0.00           O  \\nATOM    741  CB  ALA A  96     -19.500  16.105   0.353  1.00  0.00           C  \\nATOM    742  N   TYR A  97     -19.248  13.333   1.865  1.00  0.00           N  \\nATOM    743  CA  TYR A  97     -19.626  11.884   1.865  1.00  0.00           C  \\nATOM    744  C   TYR A  97     -20.886  11.632   2.873  1.00  0.00           C  \\nATOM    745  O   TYR A  97     -22.146  11.317   2.621  1.00  0.00           O  \\nATOM    746  CB  TYR A  97     -18.744  10.813   2.243  1.00  0.00           C  \\nATOM    747  CG  TYR A  97     -19.374   9.427   1.739  1.00  0.00           C  \\nATOM    748  CD1 TYR A  97     -19.626   8.419   2.621  1.00  0.00           C  \\nATOM    749  CD2 TYR A  97     -19.626   9.175   0.353  1.00  0.00           C  \\nATOM    750  CE1 TYR A  97     -20.256   7.348   2.117  1.00  0.00           C  \\nATOM    751  CE2 TYR A  97     -20.256   8.041  -0.403  1.00  0.00           C  \\nATOM    752  CZ  TYR A  97     -20.508   7.285   0.857  1.00  0.00           C  \\nATOM    753  OH  TYR A  97     -21.264   6.403   0.353  1.00  0.00           O  \\nATOM    754  N   LEU A  98     -20.760  11.947   4.007  1.00  0.00           N  \\nATOM    755  CA  LEU A  98     -21.768  11.695   5.393  1.00  0.00           C  \\nATOM    756  C   LEU A  98     -23.154  12.703   5.015  1.00  0.00           C  \\nATOM    757  O   LEU A  98     -24.288  12.199   4.889  1.00  0.00           O  \\nATOM    758  CB  LEU A  98     -21.012  11.695   6.527  1.00  0.00           C  \\nATOM    759  CG  LEU A  98     -20.004  10.939   6.401  1.00  0.00           C  \\nATOM    760  CD1 LEU A  98     -19.374  10.687   7.661  1.00  0.00           C  \\nATOM    761  CD2 LEU A  98     -20.508   9.553   5.771  1.00  0.00           C  \\nATOM    762  N   LYS A  99     -23.028  13.963   4.889  1.00  0.00           N  \\nATOM    763  CA  LYS A  99     -24.288  14.593   4.385  1.00  0.00           C  \\nATOM    764  C   LYS A  99     -25.044  13.963   2.999  1.00  0.00           C  \\nATOM    765  O   LYS A  99     -26.304  13.837   3.125  1.00  0.00           O  \\nATOM    766  CB  LYS A  99     -23.910  16.105   4.259  1.00  0.00           C  \\nATOM    767  CG  LYS A  99     -24.918  17.113   4.259  1.00  0.00           C  \\nATOM    768  CD  LYS A  99     -24.666  18.625   3.881  1.00  0.00           C  \\nATOM    769  CE  LYS A  99     -25.674  19.633   4.133  1.00  0.00           C  \\nATOM    770  NZ  LYS A  99     -25.674  21.082   3.755  1.00  0.00           N  \\nATOM    771  N   SER A 100     -24.540  13.459   1.991  1.00  0.00           N  \\nATOM    772  CA  SER A 100     -25.296  12.577   0.857  1.00  0.00           C  \\nATOM    773  C   SER A 100     -25.548  11.191   1.361  1.00  0.00           C  \\nATOM    774  O   SER A 100     -26.934  10.813   1.361  1.00  0.00           O  \\nATOM    775  CB  SER A 100     -24.540  12.577  -0.655  1.00  0.00           C  \\nATOM    776  OG  SER A 100     -25.296  11.695  -1.663  1.00  0.00           O  \\nATOM    777  N   ALA A 101     -24.666  10.435   1.865  1.00  0.00           N  \\nATOM    778  CA  ALA A 101     -25.044   8.797   2.243  1.00  0.00           C  \\nATOM    779  C   ALA A 101     -25.800   8.923   3.629  1.00  0.00           C  \\nATOM    780  O   ALA A 101     -26.808   8.293   3.755  1.00  0.00           O  \\nATOM    781  CB  ALA A 101     -23.910   8.167   2.369  1.00  0.00           C  \\nATOM    782  N   THR A 102     -25.422   9.427   4.637  1.00  0.00           N  \\nATOM    783  CA  THR A 102     -26.052   9.301   6.149  1.00  0.00           C  \\nATOM    784  C   THR A 102     -27.690   9.805   5.897  1.00  0.00           C  \\nATOM    785  O   THR A 102     -28.572   9.049   6.149  1.00  0.00           O  \\nATOM    786  CB  THR A 102     -25.548  10.246   7.220  1.00  0.00           C  \\nATOM    787  OG1 THR A 102     -24.666   9.427   7.913  1.00  0.00           O  \\nATOM    788  CG2 THR A 102     -26.430  10.876   8.039  1.00  0.00           C  \\nATOM    789  N   SER A 103     -27.942  10.939   5.645  1.00  0.00           N  \\nATOM    790  CA  SER A 103     -29.454  11.317   5.519  1.00  0.00           C  \\nATOM    791  C   SER A 103     -30.084  10.813   4.385  1.00  0.00           C  \\nATOM    792  O   SER A 103     -29.454   9.931   3.377  1.00  0.00           O  \\nATOM    793  CB  SER A 103     -29.580  12.892   5.897  1.00  0.00           C  \\nATOM    794  OG  SER A 103     -28.320  13.585   5.393  1.00  0.00           O  \\nATOM    795  OXT SER A 103     -31.344  11.317   4.133  1.00  0.00           O  \\nTER     796      SER A 103                                                      \\nHETATM 1593 FE   HEC A 104     -15.216   5.773  12.701  1.00  0.00          FE  \\nHETATM 1594  CHA HEC A 104     -16.980   6.466  15.347  1.00  0.00           C  \\nHETATM 1595  CHB HEC A 104     -16.476   8.293  11.189  1.00  0.00           C  \\nHETATM 1596  CHC HEC A 104     -13.578   5.017   9.929  1.00  0.00           C  \\nHETATM 1597  CHD HEC A 104     -13.956   3.127  14.339  1.00  0.00           C  \\nHETATM 1598  NA  HEC A 104     -16.476   7.096  13.205  1.00  0.00           N  \\nHETATM 1599  C1A HEC A 104     -17.106   7.159  14.465  1.00  0.00           C  \\nHETATM 1600  C2A HEC A 104     -17.862   8.167  14.591  1.00  0.00           C  \\nHETATM 1601  C3A HEC A 104     -17.736   8.797  13.205  1.00  0.00           C  \\nHETATM 1602  C4A HEC A 104     -17.114   8.167  12.323  1.00  0.00           C  \\nHETATM 1603  CMA HEC A 104     -18.303  10.057  12.701  1.00  0.00           C  \\nHETATM 1604  CAA HEC A 104     -18.618   8.167  15.599  1.00  0.00           C  \\nHETATM 1605  CBA HEC A 104     -20.067   8.734  15.473  1.00  0.00           C  \\nHETATM 1606  CGA HEC A 104     -20.823   8.734  16.859  1.00  0.00           C  \\nHETATM 1607  O1A HEC A 104     -21.705   8.041  17.111  1.00  0.00           O  \\nHETATM 1608  O2A HEC A 104     -20.508   9.490  17.741  1.00  0.00           O  \\nHETATM 1609  NB  HEC A 104     -15.090   6.466  11.126  1.00  0.00           N  \\nHETATM 1610  C1B HEC A 104     -15.594   7.348  10.559  1.00  0.00           C  \\nHETATM 1611  C2B HEC A 104     -15.216   7.663   9.299  1.00  0.00           C  \\nHETATM 1612  C3B HEC A 104     -14.334   6.592   8.921  1.00  0.00           C  \\nHETATM 1613  C4B HEC A 104     -14.334   6.025   9.929  1.00  0.00           C  \\nHETATM 1614  CMB HEC A 104     -15.657   8.797   8.543  1.00  0.00           C  \\nHETATM 1615  CAB HEC A 104     -13.578   6.781   7.409  1.00  0.00           C  \\nHETATM 1616  CBB HEC A 104     -12.570   7.663   7.661  1.00  0.00           C  \\nHETATM 1617  NC  HEC A 104     -13.956   4.387  12.197  1.00  0.00           N  \\nHETATM 1618  C1C HEC A 104     -13.452   4.261  10.811  1.00  0.00           C  \\nHETATM 1619  C2C HEC A 104     -12.759   3.001  10.811  1.00  0.00           C  \\nHETATM 1620  C3C HEC A 104     -12.948   2.623  11.945  1.00  0.00           C  \\nHETATM 1621  C4C HEC A 104     -13.704   3.379  12.953  1.00  0.00           C  \\nHETATM 1622  CMC HEC A 104     -11.688   2.497   9.803  1.00  0.00           C  \\nHETATM 1623  CAC HEC A 104     -12.444   1.111  12.197  1.00  0.00           C  \\nHETATM 1624  CBC HEC A 104     -11.184   0.859  12.008  1.00  0.00           C  \\nHETATM 1625  ND  HEC A 104     -15.405   5.143  14.213  1.00  0.00           N  \\nHETATM 1626  C1D HEC A 104     -14.838   4.009  14.843  1.00  0.00           C  \\nHETATM 1627  C2D HEC A 104     -15.216   3.757  15.977  1.00  0.00           C  \\nHETATM 1628  C3D HEC A 104     -16.098   4.765  16.355  1.00  0.00           C  \\nHETATM 1629  C4D HEC A 104     -16.161   5.395  15.347  1.00  0.00           C  \\nHETATM 1630  CMD HEC A 104     -14.838   2.749  16.733  1.00  0.00           C  \\nHETATM 1631  CAD HEC A 104     -16.854   4.513  17.615  1.00  0.00           C  \\nHETATM 1632  CBD HEC A 104     -16.098   4.135  18.623  1.00  0.00           C  \\nHETATM 1633  CGD HEC A 104     -15.216   5.017  19.505  1.00  0.00           C  \\nHETATM 1634  O1D HEC A 104     -14.082   5.773  18.560  1.00  0.00           O  \\nHETATM 1635  O2D HEC A 104     -15.405   5.647  20.639  1.00  0.00           O  \\nHETATM 1679  O   HOH A 105     -16.496   9.304  17.402  1.00  0.00           O  \\nCONECT  104 1615                                                                \\nCONECT  124 1623                                                                \\nCONECT  134 1593                                                                \\nCONECT  356 1635                                                                \\nCONECT  616 1593                                                                \\nCONECT  900 1658                                                                \\nCONECT  920 1666                                                                \\nCONECT  930 1636                                                                \\nCONECT 1152 1678                                                                \\nCONECT 1412 1636                                                                \\nCONECT 1593  134  616 1598 1609                                                 \\nCONECT 1593 1617 1625                                                           \\nCONECT 1594 1599 1629                                                           \\nCONECT 1595 1602 1610                                                           \\nCONECT 1596 1613 1618                                                           \\nCONECT 1597 1621 1626                                                           \\nCONECT 1598 1593 1599 1602                                                      \\nCONECT 1599 1594 1598 1600                                                      \\nCONECT 1600 1599 1601 1604                                                      \\nCONECT 1601 1600 1602 1603                                                      \\nCONECT 1602 1595 1598 1601                                                      \\nCONECT 1603 1601                                                                \\nCONECT 1604 1600 1605                                                           \\nCONECT 1605 1604 1606                                                           \\nCONECT 1606 1605 1607 1608                                                      \\nCONECT 1607 1606                                                                \\nCONECT 1608 1606                                                                \\nCONECT 1609 1593 1610 1613                                                      \\nCONECT 1610 1595 1609 1611                                                      \\nCONECT 1611 1610 1612 1614                                                      \\nCONECT 1612 1611 1613 1615                                                      \\nCONECT 1613 1596 1609 1612                                                      \\nCONECT 1614 1611                                                                \\nCONECT 1615  104 1612 1616                                                      \\nCONECT 1616 1615                                                                \\nCONECT 1617 1593 1618 1621                                                      \\nCONECT 1618 1596 1617 1619                                                      \\nCONECT 1619 1618 1620 1622                                                      \\nCONECT 1620 1619 1621 1623                                                      \\nCONECT 1621 1597 1617 1620                                                      \\nCONECT 1622 1619                                                                \\nCONECT 1623  124 1620 1624                                                      \\nCONECT 1624 1623                                                                \\nCONECT 1625 1593 1626 1629                                                      \\nCONECT 1626 1597 1625 1627                                                      \\nCONECT 1627 1626 1628 1630                                                      \\nCONECT 1628 1627 1629 1631                                                      \\nCONECT 1629 1594 1625 1628                                                      \\nCONECT 1630 1627                                                                \\nCONECT 1631 1628 1632                                                           \\nCONECT 1632 1631 1633                                                           \\nCONECT 1633 1632 1634 1635                                                      \\nCONECT 1634 1633                                                                \\nCONECT 1635  356 1633                                                           \\nCONECT 1636  930 1412 1641 1652                                                 \\nCONECT 1636 1660 1668                                                           \\nCONECT 1637 1642 1672                                                           \\nCONECT 1638 1645 1653                                                           \\nCONECT 1639 1656 1661                                                           \\nCONECT 1640 1664 1669                                                           \\nCONECT 1641 1636 1642 1645                                                      \\nCONECT 1642 1637 1641 1643                                                      \\nCONECT 1643 1642 1644 1647                                                      \\nCONECT 1644 1643 1645 1646                                                      \\nCONECT 1645 1638 1641 1644                                                      \\nCONECT 1646 1644                                                                \\nCONECT 1647 1643 1648                                                           \\nCONECT 1648 1647 1649                                                           \\nCONECT 1649 1648 1650 1651                                                      \\nCONECT 1650 1649                                                                \\nCONECT 1651 1649                                                                \\nCONECT 1652 1636 1653 1656                                                      \\nCONECT 1653 1638 1652 1654                                                      \\nCONECT 1654 1653 1655 1657                                                      \\nCONECT 1655 1654 1656 1658                                                      \\nCONECT 1656 1639 1652 1655                                                      \\nCONECT 1657 1654                                                                \\nCONECT 1658  900 1655 1659                                                      \\nCONECT 1659 1658                                                                \\nCONECT 1660 1636 1661 1664                                                      \\nCONECT 1661 1639 1660 1662                                                      \\nCONECT 1662 1661 1663 1665                                                      \\nCONECT 1663 1662 1664 1666                                                      \\nCONECT 1664 1640 1660 1663                                                      \\nCONECT 1665 1662                                                                \\nCONECT 1666  920 1663 1667                                                      \\nCONECT 1667 1666                                                                \\nCONECT 1668 1636 1669 1672                                                      \\nCONECT 1669 1640 1668 1670                                                      \\nCONECT 1670 1669 1671 1673                                                      \\nCONECT 1671 1670 1672 1674                                                      \\nCONECT 1672 1637 1668 1671                                                      \\nCONECT 1673 1670                                                                \\nCONECT 1674 1671 1675                                                           \\nCONECT 1675 1674 1676                                                           \\nCONECT 1676 1675 1677 1678                                                      \\nCONECT 1677 1676                                                                \\nCONECT 1678 1152 1676                                                           \\nMASTER      287    0    2   10    0    0    9    9 1677    2   98   16          \\nEND                                                                             \\n\",\"pdb\");\n",
       "\tviewer_16742555283712852.setBackgroundColor(\"white\");\n",
       "\tviewer_16742555283712852.setStyle({\"cartoon\": {\"color\": \"white\"}});\n",
       "\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"5\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"6\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"7\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"8\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"9\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"10\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"11\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"12\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"13\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"14\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"15\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"16\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"17\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"18\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"19\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"20\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"21\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"22\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"23\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"24\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"25\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"26\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555283712852.addStyle({\"chain\": \"A\", \"resi\": [\"27\"]},{\"cartoon\": {\"color\": \"grey\"}});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"5\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"5\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"5\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"6\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"6\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"6\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"6\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"6\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"95\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"6\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"7\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"7\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"15\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"7\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"97\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"7\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"100\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"8\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"12\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"8\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"93\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"8\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"97\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"9\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"85\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"9\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"13\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"15\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"16\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"10\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"11\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"13\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"13\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"15\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"13\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"16\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"13\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"13\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"83\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"13\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"32\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"94\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"14\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"98\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"15\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"15\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"16\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"18\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"28\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"29\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"80\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"81\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"17\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"82\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"18\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"28\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"18\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"32\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"19\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"32\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"22\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"23\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"24\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"32\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"20\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"101\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"21\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"25\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"21\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"33\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"22\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"31\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"22\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"33\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"23\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"25\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"23\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"26\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"23\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"32\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"23\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"33\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"25\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"30\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"25\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"31\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"26\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"28\"]}, \"color\": \"red\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"27\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"30\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.addLine({\"start\": {\"chain\": \"A\", \"resi\": [\"27\"]}, \"end\": {\"chain\": \"A\", \"resi\": [\"31\"]}, \"color\": \"green\", \"dashed\": false, \"linewidth\": 3});\n",
       "\tviewer_16742555283712852.zoomTo();\n",
       "viewer_16742555283712852.render();\n",
       "});\n",
       "</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "<py3Dmol.view at 0x7f127a2d2910>"
      ]
     },
     "execution_count": 55,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "view_frustration_dif(pdb_file_chain,path_bs+'exon_0.pdb',fr_dif2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
